<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="applicable-device" content="pc,mobile">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3 | Genome Biology | Full Text</title>

        

    <meta name="citation_abstract" content="Nucleosomes are present throughout the genome and must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms. Despite the development of protocols and techniques that have enabled us to map nucleosome occupancy genome-wide, the dynamic properties of nucleosomes remain poorly understood, particularly in mammalian cells. The histone variant H3.3 is incorporated into chromatin independently of DNA replication and requires displacement of existing nucleosomes for its deposition. Here, we measure H3.3 turnover at high resolution in the mammalian genome in order to present a genome-wide characterization of replication-independent H3.3-nucleosome dynamics. We developed a system to study the DNA replication-independent turnover of nucleosomes containing the histone variant H3.3 in mammalian cells. By measuring the genome-wide incorporation of H3.3 at different time points following epitope-tagged H3.3 expression, we find three categories of H3.3-nucleosome turnover in vivo: rapid turnover, intermediate turnover and, specifically at telomeres, slow turnover. Our data indicate that H3.3-containing nucleosomes at enhancers and promoters undergo rapid turnover that is associated with active histone modification marks including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z. The rate of turnover is negatively correlated with H3K27me3 at regulatory regions and with H3K36me3 at gene bodies. We have established a reliable approach to measure turnover rates of H3.3-containing nucleosomes on a genome-wide level in mammalian cells. Our results suggest that distinct mechanisms control the dynamics of H3.3 incorporation at functionally different genomic regions.">

    <meta name="journal_id" content="13059">

    <meta name="dc.title" content="Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3">

    <meta name="dc.source" content="Genome Biology 2013 14:10">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2013-12-10">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2013 Kraushaar et al.; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="Nucleosomes are present throughout the genome and must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms. Despite the development of protocols and techniques that have enabled us to map nucleosome occupancy genome-wide, the dynamic properties of nucleosomes remain poorly understood, particularly in mammalian cells. The histone variant H3.3 is incorporated into chromatin independently of DNA replication and requires displacement of existing nucleosomes for its deposition. Here, we measure H3.3 turnover at high resolution in the mammalian genome in order to present a genome-wide characterization of replication-independent H3.3-nucleosome dynamics. We developed a system to study the DNA replication-independent turnover of nucleosomes containing the histone variant H3.3 in mammalian cells. By measuring the genome-wide incorporation of H3.3 at different time points following epitope-tagged H3.3 expression, we find three categories of H3.3-nucleosome turnover in vivo: rapid turnover, intermediate turnover and, specifically at telomeres, slow turnover. Our data indicate that H3.3-containing nucleosomes at enhancers and promoters undergo rapid turnover that is associated with active histone modification marks including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z. The rate of turnover is negatively correlated with H3K27me3 at regulatory regions and with H3K36me3 at gene bodies. We have established a reliable approach to measure turnover rates of H3.3-containing nucleosomes on a genome-wide level in mammalian cells. Our results suggest that distinct mechanisms control the dynamics of H3.3 incorporation at functionally different genomic regions.">

    <meta name="prism.issn" content="1474-760X">

    <meta name="prism.publicationName" content="Genome Biology">

    <meta name="prism.publicationDate" content="2013-12-10">

    <meta name="prism.volume" content="14">

    <meta name="prism.number" content="10">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="1">

    <meta name="prism.endingPage" content="15">

    <meta name="prism.copyright" content="2013 Kraushaar et al.; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r121">

    <meta name="prism.doi" content="doi:10.1186/gb-2013-14-10-r121">

    <meta name="citation_pdf_url" content="https://genomebiology.biomedcentral.com/track/pdf/10.1186/gb-2013-14-10-r121">

    <meta name="citation_fulltext_html_url" content="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r121">

    <meta name="citation_journal_title" content="Genome Biology">

    <meta name="citation_journal_abbrev" content="Genome Biol">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1474-760X">

    <meta name="citation_title" content="Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3">

    <meta name="citation_volume" content="14">

    <meta name="citation_issue" content="10">

    <meta name="citation_publication_date" content="2013/10">

    <meta name="citation_online_date" content="2013/12/10">

    <meta name="citation_firstpage" content="1">

    <meta name="citation_lastpage" content="15">

    <meta name="citation_article_type" content="Research">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="citation_language" content="en">

    <meta name="dc.identifier" content="doi:10.1186/gb-2013-14-10-r121">

    <meta name="DOI" content="10.1186/gb-2013-14-10-r121">

    <meta name="citation_doi" content="10.1186/gb-2013-14-10-r121">

    <meta name="description" content="Nucleosomes are present throughout the genome and must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms. Despite the development of protocols and techniques that have enabled us to map nucleosome occupancy genome-wide, the dynamic properties of nucleosomes remain poorly understood, particularly in mammalian cells. The histone variant H3.3 is incorporated into chromatin independently of DNA replication and requires displacement of existing nucleosomes for its deposition. Here, we measure H3.3 turnover at high resolution in the mammalian genome in order to present a genome-wide characterization of replication-independent H3.3-nucleosome dynamics. We developed a system to study the DNA replication-independent turnover of nucleosomes containing the histone variant H3.3 in mammalian cells. By measuring the genome-wide incorporation of H3.3 at different time points following epitope-tagged H3.3 expression, we find three categories of H3.3-nucleosome turnover in vivo: rapid turnover, intermediate turnover and, specifically at telomeres, slow turnover. Our data indicate that H3.3-containing nucleosomes at enhancers and promoters undergo rapid turnover that is associated with active histone modification marks including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z. The rate of turnover is negatively correlated with H3K27me3 at regulatory regions and with H3K36me3 at gene bodies. We have established a reliable approach to measure turnover rates of H3.3-containing nucleosomes on a genome-wide level in mammalian cells. Our results suggest that distinct mechanisms control the dynamics of H3.3 incorporation at functionally different genomic regions.">

    <meta name="dc.creator" content="Kraushaar, Daniel C">

    <meta name="dc.creator" content="Jin, Wenfei">

    <meta name="dc.creator" content="Maunakea, Alika">

    <meta name="dc.creator" content="Abraham, Brian">

    <meta name="dc.creator" content="Ha, Misook">

    <meta name="dc.creator" content="Zhao, Keji">

    <meta name="dc.subject" content="Animal Genetics and Genomics">

    <meta name="dc.subject" content="Human Genetics">

    <meta name="dc.subject" content="Plant Genetics and Genomics">

    <meta name="dc.subject" content="Microbial Genetics and Genomics">

    <meta name="dc.subject" content="Bioinformatics">

    <meta name="dc.subject" content="Evolutionary Biology">

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=H3.3/H2A.Z double variant-containing nucleosomes mark &#39;nucleosome-free regions&#39; of active promoters and other regulatory regions; citation_author=C Jin, C Zang, G Wei, K Cui, W Peng, K Zhao, G Felsenfeld; citation_volume=41; citation_publication_date=2009; citation_pages=941-945; citation_doi=10.1038/ng.409; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Genes Dev; citation_title=Nucleosome stability mediated by histone variants H3.3 and H2A.Z; citation_author=C Jin, G Felsenfeld; citation_volume=21; citation_publication_date=2007; citation_pages=1519-1529; citation_doi=10.1101/gad.1547707; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Nat Cell Biol; citation_title=Epigenetic memory of an active gene state depends on histone H3.3 incorporation into chromatin in the absence of transcription; citation_author=RK Ng, JB Gurdon; citation_volume=10; citation_publication_date=2008; citation_pages=102-109; citation_doi=10.1038/ncb1674; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Cell Stem Cell; citation_title=H2A.Z facilitates access of active and repressive complexes to chromatin in embryonic stem cell self-renewal and differentiation; citation_author=G Hu, K Cui, D Northrup, C Liu, C Wang, Q Tang, K Ge, D Levens, C Crane-Robinson, K Zhao; citation_volume=12; citation_publication_date=2013; citation_pages=180-192; citation_doi=10.1016/j.stem.2012.11.003; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=Trends Genet; citation_title=Histone variants, nucleosome assembly and epigenetic inheritance; citation_author=S Henikoff, T Furuyama, K Ahmad; citation_volume=20; citation_publication_date=2004; citation_pages=320-326; citation_doi=10.1016/j.tig.2004.05.004; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=Curr Opin Genet Dev; citation_title=New functions for an old variant: no substitute for histone H3.3; citation_author=SJ Elsaesser, AD Goldberg, CD Allis; citation_volume=20; citation_publication_date=2010; citation_pages=110-117; citation_doi=10.1016/j.gde.2010.01.003; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=The histone variant H3.3 marks active chromatin by replication-independent nucleosome assembly; citation_author=K Ahmad, S Henikoff; citation_volume=9; citation_publication_date=2002; citation_pages=1191-1200; citation_doi=10.1016/S1097-2765(02)00542-7; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=HIRA is critical for a nucleosome assembly pathway independent of DNA synthesis; citation_author=D Ray-Gallet, JP Quivy, C Scamps, EM Martini, M Lipinski, G Almouzni; citation_volume=9; citation_publication_date=2002; citation_pages=1091-1100; citation_doi=10.1016/S1097-2765(02)00526-9; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Distinct factors control histone variant H3.3 localization at specific genomic regions; citation_author=AD Goldberg, LA Banaszynski, KM Noh, PW Lewis, SJ Elsaesser, S Stadler, S Dewell, M Law, X Guo, X Li, D Wen, A Chapgier, RC DeKelver, JC Miller, YL Lee, EA Boydston, MC Holmes, PD Gregory, JM Greally, S Rafii, C Yang, PJ Scambler, D Garrick, RJ Gibbons, DR Higgs, IM Cristea, FD Urnov, D Zheng, CD Allis; citation_volume=140; citation_publication_date=2010; citation_pages=678-691; citation_doi=10.1016/j.cell.2010.01.003; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=ATRX interacts with H3.3 in maintaining telomere structural integrity in pluripotent embryonic stem cells; citation_author=LH Wong, JD McGhie, M Sim, MA Anderson, S Ahn, RD Hannan, AJ George, KA Morgan, JR Mann, KH Choo; citation_volume=20; citation_publication_date=2010; citation_pages=351-360; citation_doi=10.1101/gr.101477.109; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Daxx is an H3.3-specific histone chaperone and cooperates with ATRX in replication-independent chromatin assembly at telomeres; citation_author=PW Lewis, SJ Elsaesser, KM Noh, SC Stadler, CD Allis; citation_volume=107; citation_publication_date=2010; citation_pages=14075-14080; citation_doi=10.1073/pnas.1008850107; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Cell Rep; citation_title=A developmental requirement for HIRA-dependent H3.3 deposition revealed at gastrulation in Xenopus; citation_author=E Szenker, N Lacoste, G Almouzni; citation_volume=1; citation_publication_date=2012; citation_pages=730-740; citation_doi=10.1016/j.celrep.2012.05.006; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=J Biol Chem; citation_title=Expression patterns and post-translational modifications associated with mammalian histone H3 variants; citation_author=SB Hake, BA Garcia, EM Duncan, M Kauer, G Dellaire, J Shabanowitz, DP Bazett-Jones, CD Allis, DF Hunt; citation_volume=281; citation_publication_date=2006; citation_pages=559-568; citation_doi=10.1074/jbc.M509266200; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Genes Dev; citation_title=Transcriptional activation triggers deposition and removal of the histone variant H3.3; citation_author=BE Schwartz, K Ahmad; citation_volume=19; citation_publication_date=2005; citation_pages=804-814; citation_doi=10.1101/gad.1259805; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Genome-scale profiling of histone H3.3 replacement patterns; citation_author=Y Mito, JG Henikoff, S Henikoff; citation_volume=37; citation_publication_date=2005; citation_pages=1090-1097; citation_doi=10.1038/ng1637; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=J Biol Chem; citation_title=Inducible deposition of the histone variant H3.3 in interferon-stimulated genes; citation_author=T Tamura, M Smith, T Kanno, H Dasenbrock, A Nishiyama, K Ozato; citation_volume=284; citation_publication_date=2009; citation_pages=12217-12225; citation_doi=10.1074/jbc.M805651200; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Genome-wide analysis of histone H3.1 and H3.3 variants in Arabidopsis thaliana; citation_author=H Stroud, S Otero, B Desvoyes, E Ramirez-Parra, SE Jacobsen, C Gutierrez; citation_volume=109; citation_publication_date=2012; citation_pages=5370-5375; citation_doi=10.1073/pnas.1203145109; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Trends Genet; citation_title=Gene regulation by nucleosome positioning; citation_author=L Bai, AV Morozov; citation_volume=26; citation_publication_date=2010; citation_pages=476-483; citation_doi=10.1016/j.tig.2010.08.003; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Trends Biochem Sci; citation_title=Transcription factors vs nucleosomes: regulation of the PHO5 promoter in yeast; citation_author=J Svaren, W Horz; citation_volume=22; citation_publication_date=1997; citation_pages=93-97; citation_doi=10.1016/S0968-0004(97)01001-3; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Dynamic regulation of nucleosome positioning in the human genome; citation_author=DE Schones, K Cui, S Cuddapah, TY Roh, A Barski, Z Wang, G Wei, K Zhao; citation_volume=132; citation_publication_date=2008; citation_pages=887-898; citation_doi=10.1016/j.cell.2008.02.022; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=Cell Biosci; citation_title=SWI/SNF-mediated chromatin remodeling induces Z-DNA formation on a nucleosome; citation_author=N Mulholland, Y Xu, H Sugiyama, K Zhao; citation_volume=2; citation_publication_date=2012; citation_pages=3; citation_doi=10.1186/2045-3701-2-3; citation_id=CR21">

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Human SWI/SNF interconverts a nucleosome between its base state and a stable remodeled state; citation_author=G Schnitzler, S Sif, RE Kingston; citation_volume=94; citation_publication_date=1998; citation_pages=17-27; citation_doi=10.1016/S0092-8674(00)81217-9; citation_id=CR22">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Removal of promoter nucleosomes by disassembly rather than sliding in vivo; citation_author=H Boeger, J Griesenbeck, JS Strattan, RD Kornberg; citation_volume=14; citation_publication_date=2004; citation_pages=667-673; citation_doi=10.1016/j.molcel.2004.05.013; citation_id=CR23">

    <meta name="citation_reference" content="citation_journal_title=Nat Struct Mol Biol; citation_title=Determinants of nucleosome positioning; citation_author=K Struhl, E Segal; citation_volume=20; citation_publication_date=2013; citation_pages=267-273; citation_doi=10.1038/nsmb.2506; citation_id=CR24">

    <meta name="citation_reference" content="citation_journal_title=Curr Opin Genet Dev; citation_title=Regulation of nucleosome stability as a mediator of chromatin function; citation_author=PG Giresi, M Gupta, JD Lieb; citation_volume=16; citation_publication_date=2006; citation_pages=171-176; citation_doi=10.1016/j.gde.2006.02.003; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Dynamics of replication-independent histone turnover in budding yeast; citation_author=MF Dion, T Kaplan, M Kim, S Buratowski, N Friedman, OJ Rando; citation_volume=315; citation_publication_date=2007; citation_pages=1405-1408; citation_doi=10.1126/science.1134053; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Continuous histone H2B and transcription-dependent histone H3 exchange in yeast cells outside of replication; citation_author=A Jamai, RM Imoberdorf, M Strubin; citation_volume=25; citation_publication_date=2007; citation_pages=345-355; citation_doi=10.1016/j.molcel.2007.01.019; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1; citation_author=A Rufiange, PE Jacques, W Bhat, F Robert, A Nourani; citation_volume=27; citation_publication_date=2007; citation_pages=393-405; citation_doi=10.1016/j.molcel.2007.07.011; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Capturing the dynamic epigenome; citation_author=RB Deal, S Henikoff; citation_volume=11; citation_publication_date=2010; citation_pages=218; citation_doi=10.1186/gb-2010-11-10-218; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Genome-wide kinetics of nucleosome turnover determined by metabolic labeling of histones; citation_author=RB Deal, JG Henikoff, S Henikoff; citation_volume=328; citation_publication_date=2010; citation_pages=1161-1164; citation_doi=10.1126/science.1186777; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=J Cell Biol; citation_title=Propagation of centromeric chromatin requires exit from mitosis; citation_author=LE Jansen, BE Black, DR Foltz, DW Cleveland; citation_volume=176; citation_publication_date=2007; citation_pages=795-805; citation_doi=10.1083/jcb.200701066; citation_id=CR31">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Dynamics of histone H3 deposition in vivo reveal a nucleosome gap-filling mechanism for H3.3 to maintain chromatin integrity; citation_author=D Ray-Gallet, A Woolfe, I Vassias, C Pellentz, N Lacoste, A Puri, DC Schultz, NA Pchelintsev, PD Adams, LE Jansen, G Almouzni; citation_volume=44; citation_publication_date=2011; citation_pages=928-941; citation_doi=10.1016/j.molcel.2011.12.006; citation_id=CR32">

    <meta name="citation_reference" content="citation_journal_title=J Cell Biol; citation_title=Kinetics of core histones in living human cells: little exchange of H3 and H4 and some rapid exchange of H2B; citation_author=H Kimura, PR Cook; citation_volume=153; citation_publication_date=2001; citation_pages=1341-1353; citation_doi=10.1083/jcb.153.7.1341; citation_id=CR33">

    <meta name="citation_reference" content="citation_journal_title=Dev Cell; citation_title=Hyperdynamic plasticity of chromatin proteins in pluripotent embryonic stem cells; citation_author=E Meshorer, D Yellajoshula, E George, PJ Scambler, DT Brown, T Misteli; citation_volume=10; citation_publication_date=2006; citation_pages=105-116; citation_doi=10.1016/j.devcel.2005.10.017; citation_id=CR34">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Histone H3K27ac separates active from poised enhancers and predicts developmental state; citation_author=MP Creyghton, AW Cheng, GG Welstead, T Kooistra, BW Carey, EJ Steine, J Hanna, MA Lodato, GM Frampton, PA Sharp, LA Boyer, RA Young, R Jaenisch; citation_volume=107; citation_publication_date=2010; citation_pages=21931-21936; citation_doi=10.1073/pnas.1016071107; citation_id=CR35">

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A unique chromatin signature uncovers early developmental enhancers in humans; citation_author=A Rada-Iglesias, R Bajpai, T Swigut, SA Brugmann, RA Flynn, J Wysocka; citation_volume=470; citation_publication_date=2011; citation_pages=279-283; citation_doi=10.1038/nature09692; citation_id=CR36">

    <meta name="citation_reference" content="citation_journal_title=J Biol Chem; citation_title=Histone H3 lysine 36 dimethylation (H3K36me2) is sufficient to recruit the Rpd3s histone deacetylase complex and to repress spurious transcription; citation_author=B Li, J Jackson, MD Simon, B Fleharty, M Gogol, C Seidel, JL Workman, A Shilatifard; citation_volume=284; citation_publication_date=2009; citation_pages=7970-7976; citation_doi=10.1074/jbc.M808220200; citation_id=CR37">

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription; citation_author=MJ Carrozza, B Li, L Florens, T Suganuma, SK Swanson, KK Lee, WJ Shia, S Anderson, J Yates, MP Washburn, JL Workman; citation_volume=123; citation_publication_date=2005; citation_pages=581-592; citation_doi=10.1016/j.cell.2005.10.023; citation_id=CR38">

    <meta name="citation_reference" content="citation_journal_title=Nat Struct Mol Biol; citation_title=Pol II and its associated epigenetic marks are present at Pol III-transcribed noncoding RNA genes; citation_author=A Barski, I Chepelev, D Liko, S Cuddapah, AB Fleming, J Birch, K Cui, RJ White, K Zhao; citation_volume=17; citation_publication_date=2010; citation_pages=629-634; citation_doi=10.1038/nsmb.1806; citation_id=CR39">

    <meta name="citation_reference" content="citation_journal_title=FEBS Lett; citation_title=In vitro low propensity to form nucleosomes of four telomeric sequences; citation_author=S Cacchione, MA Cerone, M Savino; citation_volume=400; citation_publication_date=1997; citation_pages=37-41; citation_doi=10.1016/S0014-5793(96)01318-X; citation_id=CR40">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Transcriptional activators are dispensable for transcription in the absence of Spt6-mediated chromatin reassembly of promoter regions; citation_author=MW Adkins, JK Tyler; citation_volume=21; citation_publication_date=2006; citation_pages=405-416; citation_doi=10.1016/j.molcel.2005.12.010; citation_id=CR41">

    <meta name="citation_reference" content="citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_title=Nuclear mobility and mitotic chromosome binding: similarities between pioneer transcription factor FoxA and linker histone H1; citation_author=KS Zaret, JM Caravaca, A Tulin, T Sekiya; citation_volume=75; citation_publication_date=2010; citation_pages=219-226; citation_doi=10.1101/sqb.2010.75.061; citation_id=CR42">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Nucleosome-driven transcription factor binding and gene regulation; citation_author=C Ballare, G Castellano, L Gaveglia, S Althammer, J Gonzalez-Vallinas, E Eyras, F Le Dily, R Zaurin, D Soronellas, GP Vicent, M Beato; citation_volume=49; citation_publication_date=2013; citation_pages=67-79; citation_doi=10.1016/j.molcel.2012.10.019; citation_id=CR43">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell Biol; citation_title=The Swi/Snf complex is important for histone eviction during transcriptional activation and RNA polymerase II elongation in vivo; citation_author=MA Schwabish, K Struhl; citation_volume=27; citation_publication_date=2007; citation_pages=6987-6995; citation_doi=10.1128/MCB.00717-07; citation_id=CR44">

    <meta name="citation_reference" content="citation_journal_title=Genes Dev; citation_title=Heat shock reduces stalled RNA polymerase II and nucleosome turnover genome-wide; citation_author=SS Teves, S Henikoff; citation_volume=25; citation_publication_date=2011; citation_pages=2387-2397; citation_doi=10.1101/gad.177675.111; citation_id=CR45">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Transcription elongation factors repress transcription initiation from cryptic sites; citation_author=CD Kaplan, L Laprade, F Winston; citation_volume=301; citation_publication_date=2003; citation_pages=1096-1099; citation_doi=10.1126/science.1087374; citation_id=CR46">

    <meta name="citation_reference" content="citation_journal_title=Mol Cell Biol; citation_title=A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation; citation_author=KO Kizer, HP Phatnani, Y Shibata, H Hall, AL Greenleaf, BD Strahl; citation_volume=25; citation_publication_date=2005; citation_pages=3305-3316; citation_doi=10.1128/MCB.25.8.3305-3316.2005; citation_id=CR47">

    <meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=Extraction, purification and analysis of histones; citation_author=D Shechter, HL Dormann, CD Allis, SB Hake; citation_volume=2; citation_publication_date=2007; citation_pages=1445-1457; citation_doi=10.1038/nprot.2007.202; citation_id=CR48">

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=High-resolution profiling of histone methylations in the human genome; citation_author=A Barski, S Cuddapah, K Cui, TY Roh, DE Schones, Z Wang, G Wei, I Chepelev, K Zhao; citation_volume=129; citation_publication_date=2007; citation_pages=823-837; citation_doi=10.1016/j.cell.2007.05.009; citation_id=CR49">

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B Langmead, C Trapnell, M Pop, SL Salzberg; citation_volume=10; citation_publication_date=2009; citation_pages=R25; citation_doi=10.1186/gb-2009-10-3-r25; citation_id=CR50">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=A clustering approach for identification of enriched domains from histone modification ChIP-Seq data; citation_author=C Zang, DE Schones, C Zeng, K Cui, K Zhao, W Peng; citation_volume=25; citation_publication_date=2009; citation_pages=1952-1958; citation_doi=10.1093/bioinformatics/btp340; citation_id=CR51">

    <meta name="citation_reference" content="citation_journal_title=Cytogenet Genome Res; citation_title=Repbase Update, a database of eukaryotic repetitive elements; citation_author=J Jurka, VV Kapitonov, A Pavlicek, P Klonowski, O Kohany, J Walichiewicz; citation_volume=110; citation_publication_date=2005; citation_pages=462-467; citation_doi=10.1159/000084979; citation_id=CR52">

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions; citation_author=D Kim, G Pertea, C Trapnell, H Pimentel, R Kelley, SL Salzberg; citation_volume=14; citation_publication_date=2013; citation_pages=R36; citation_doi=10.1186/gb-2013-14-4-r36; citation_id=CR53">

    <meta name="citation_author" content="Kraushaar, Daniel C">

    <meta name="citation_author_institution" content="Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA">

    <meta name="citation_author" content="Jin, Wenfei">

    <meta name="citation_author_institution" content="Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA">

    <meta name="citation_author" content="Maunakea, Alika">

    <meta name="citation_author_institution" content="Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA">

    <meta name="citation_author" content="Abraham, Brian">

    <meta name="citation_author_institution" content="Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA">

    <meta name="citation_author" content="Ha, Misook">

    <meta name="citation_author_institution" content="Samsung Advanced Institute of Technology, Samsung Electronics Corporation, Yongin-Si, South Korea">

    <meta name="citation_author" content="Zhao, Keji">

    <meta name="citation_author_institution" content="Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA">



        <meta name="format-detection" content="telephone=no">

        
<link rel="apple-touch-icon" sizes="180x180" href=/static/img/favicons/bmc/apple-touch-icon-582ef1d0f5.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/img/favicons/bmc/favicon-32x32-5d7879efe1.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/img/favicons/bmc/favicon-16x16-c241ac1a2f.png>
<link rel="manifest" href=/static/app-bmc/manifest-3bb9ad383b.json>
<link rel="mask-icon" href=/static/img/favicons/bmc/safari-pinned-tab-427338756f.svg color="#e6e6e6">
<link rel="shortcut icon" data-test="shortcut-icon" href=/static/img/favicons/bmc/favicon-1886637b78.ico>
<meta name="msapplication-config" content=/static/app-bmc/browserconfig-ecd9aa8205.xml>
<meta name="msapplication-TileColor"
      content="#1b3051">
<meta name="msapplication-TileImage"
      content=/static/img/favicons/bmc/app-icon-144x144.png>
<meta name="theme-color"
      content="#1b3051">


        
        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>

        


    
        <link rel="stylesheet" media="screen" href=/static/app-bmc/css/core-article-cd4a428820.css>
    

    <link rel="stylesheet" media="screen" href=/static/app-bmc/css/core-b076aec3c2.css>
    <link rel="stylesheet" media="print" href=/static/app-bmc/css/print-b8af42253b.css>
    

    

    
        <style>@media only screen and (-webkit-min-device-pixel-ratio: 0) and (min-color-index: 0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio: 0) and (min-resolution: 3 e1dpcm) { button{line-height:inherit}html,label{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}html{-webkit-font-smoothing:subpixel-antialiased;box-sizing:border-box;color:#333;font-size:100%;height:100%;line-height:1.61803;overflow-y:scroll}*{box-sizing:inherit}body{background:#fff;font-size:1.8em;margin:0;max-width:100%;min-height:100%}button,div,form,input{margin:0;padding:0}body,p{padding:0}a{color:#004b83;text-decoration:underline;text-decoration-skip-ink:auto}a>img{vertical-align:middle}h1,h2,h3{margin-top:0}h1{font-size:28px;font-weight:700}html.webfonts-loaded h1,html.webfonts-loaded h2,html.webfonts-loaded h3{font-family:Europa,Trebuchet MS}h1,h2,h3{color:#1b3051;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-style:normal;line-height:1.4}h2{font-size:26px;font-weight:700;margin-bottom:.5em}h3{font-size:18px;margin-bottom:.7em}p{margin:0}ul{margin-top:0}p{margin-bottom:1.5em}p:last-child{margin-bottom:0}p:empty{display:none}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}.c-ad--728x90 iframe{height:90px;max-width:970px}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad iframe{border:0;overflow:auto;vertical-align:top}.c-ad__label{color:#333;font-weight:400;line-height:1.5;margin-bottom:4px}.c-ad__label,.c-skip-link{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem}.c-skip-link{background:#dceaf6;bottom:auto;color:#004b83;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#004b83}.c-dropdown__button:after{border-color:transparent transparent transparent #fff;border-style:solid;border-width:4px 0 4px 14px;content:"";display:block;height:0;margin-left:3px;width:0}.c-dropdown{display:inline-block;position:relative}.c-dropdown__button{background-color:transparent;border:0;display:inline-block;padding:0;white-space:nowrap}.c-dropdown__button:after{border-color:currentcolor transparent transparent;border-width:5px 4px 0 5px;display:inline-block;margin-left:8px;vertical-align:middle}.c-dropdown__menu{background-color:#fff;border:1px solid #d9d9d9;border-radius:3px;box-shadow:0 2px 6px rgba(0,0,0,.1);font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:14px;line-height:1.4;list-style:none;margin:0;padding:8px 0;position:absolute;top:100%;transform:translateY(8px);width:180px;z-index:100}.c-dropdown__menu:after,.c-dropdown__menu:before{border-style:solid;bottom:100%;content:"";display:block;height:0;left:16px;position:absolute;width:0}.c-dropdown__menu:before{border-color:transparent transparent #d9d9d9;border-width:0 9px 9px;transform:translateX(-1px)}.c-dropdown__menu:after{border-color:transparent transparent #fff;border-width:0 8px 8px}.c-dropdown__menu--right{left:auto;right:0}.c-dropdown__menu--right:after,.c-dropdown__menu--right:before{left:auto;right:16px}.c-dropdown__menu--right:before{transform:translateX(1px)}.c-dropdown__link{background-color:transparent;color:#004b83;display:block;padding:4px 16px;text-decoration:none}.c-header{background-color:#1b3051;border-bottom:4px solid #1b3051;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:14px;padding:16px 0}.c-header__container{-webkit-box-align:center;-webkit-box-pack:justify;align-items:center;display:flex;justify-content:space-between;margin:0 auto;max-width:1280px;padding:0 16px}.c-header__brand{margin-right:48px}.c-header__brand a,.c-journal-title a{text-decoration:none}.c-header__menu,.c-header__navigation{display:flex}.c-header__navigation{-webkit-box-align:center;align-items:center}.c-header__menu{list-style:none;margin:0;padding:0}.c-header__item{color:#fff;margin-right:24px}.c-header__item:last-child{margin-right:0}.c-header__link{color:#fff;text-decoration:none}html.webfonts-loaded .c-journal-title{font-family:Europa,Trebuchet MS}.c-form-field{margin-bottom:1em}.c-form-field__label{color:#666;display:block;font-size:14px;margin-bottom:.4em}.c-form-field__input{border:1px solid #a6a6a6;border-radius:3px;box-shadow:inset 0 1px 3px 0 rgba(0,0,0,.21);font-size:14px;line-height:1.28571;padding:.75em 1em;vertical-align:middle;width:100%}.c-popup-search{background-color:#25426f;box-shadow:0 3px 3px -3px rgba(0,0,0,.21);padding:16px 0;position:relative;z-index:10}@media only screen and (min-width:1024px){.js .c-popup-search{position:absolute;top:100%;width:100%}.c-popup-search__container{margin:auto;max-width:70%}}.c-logo img{display:block}.ctx-search .c-form-field{margin-bottom:0}.ctx-search .c-form-field__label{color:#fff}.ctx-search .c-form-field__input{border-bottom-right-radius:0;border-top-right-radius:0;margin-right:0}.c-journal-header{overflow:hidden}.c-journal-header__inner{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-journal-header__inner{-webkit-box-pack:justify;display:flex;justify-content:space-between}}.c-journal-header__identity{background-size:auto 80px;min-height:16px;padding:6px 0}@media only screen and (min-width:540px){.c-journal-header__identity{background-position:0}}.c-journal-header__identity--default{background:url(/static/images/bmc/identities/patterns/default-7e3a6b1388.svg) 0 0 no-repeat #04caa8}.c-journal-title{color:#1b3051;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:26px;font-style:normal;font-weight:700;line-height:1.4;margin-bottom:0}.c-journal-title>a{color:#0097ef;color:#1b3051;display:flex;text-decoration:none}.c-journal-title__text{-webkit-box-flex:0;align-self:center;display:block;flex:0 1 auto}.c-logo,.c-logo>a{-webkit-box-align:baseline;align-items:baseline;display:flex}.c-logo__strapline{display:none}@media only screen and (min-width:540px){.c-logo__strapline{display:block;margin:0 0 0 16px;transform:translateY(1px)}}.c-navbar{background:#f2f2f2;border-bottom:1px solid #d9d9d9;border-top:1px solid #d9d9d9;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:14px;line-height:1.42857;padding:16px 0}.c-navbar__container{-webkit-box-pack:justify;flex-wrap:wrap;justify-content:space-between;margin:0 auto;max-width:1280px;padding:0 16px}.c-navbar__container,.c-navbar__content{-webkit-box-align:center;align-items:center;display:flex}.c-navbar__content{-webkit-box-flex:0;flex:0 1 auto}@media only screen and (min-width:1024px){.c-navbar__content{display:flex!important}}.c-navbar__nav{display:flex;list-style:none;margin:0;padding:0}@media only screen and (min-width:1024px){.c-navbar__nav{transform:translateY(1px)}}.c-navbar__item:not(:first-child){margin-left:16px}.c-navbar__link{background:0 0;border:0;color:currentcolor;display:block;text-decoration:none;text-transform:capitalize}.c-navbar__link--is-shown{text-decoration:underline}.c-logo{display:block}html.webfonts-loaded .u-h3,html.webfonts-loaded .u-h4{font-family:Europa,Trebuchet MS}.u-button{-webkit-box-align:center;-webkit-box-pack:center;align-items:center;background-color:#f2f2f2;background-image:linear-gradient(#fff,#f2f2f2);border:1px solid #ccc;border-radius:2px;cursor:pointer;display:inline-flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button svg{fill:currentcolor}.u-button{color:#004b83}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-display-flex{display:flex;width:100%}.u-align-items-center{-webkit-box-align:center;align-items:center}.u-flex-static{-webkit-box-flex:0;flex:0 1 auto;flex:0 0 auto}.u-display-none{display:none}.js .u-js-hide{display:none;visibility:hidden}@media print{.u-hide-print{display:none}}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-list-reset{list-style:none;margin:0;padding:0}.u-position-relative{position:relative}.u-mt-32{margin-top:32px}.u-mr-24{margin-right:24px}.u-mr-48{margin-right:48px}.u-mb-32{margin-bottom:32px}.u-ml-8{margin-left:8px}.u-button-reset{background-color:transparent;border:0;padding:0}.u-text-sm{font-size:14px}.u-h3,.u-h4{color:#1b3051;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:18px;font-style:normal;line-height:1.4;margin-bottom:.7em}.u-h4{font-style:italic;font-weight:700}.u-vh-full{min-height:100vh}.u-hide{display:none;visibility:hidden}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}}@media only screen and (max-width:1023px){.u-hide-at-lt-lg{display:none;visibility:hidden}}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}body{font-size:1.125rem} }</style>

        <link rel="stylesheet" data-inline-css-source="critical-css" href="/static/app-bmc/css/enhanced-2b5751e1cf.css"
              media="print" onload="this.media='only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)';this.onload=null">

        <style>@media only screen and (-webkit-min-device-pixel-ratio: 0) and (min-color-index: 0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio: 0) and (min-resolution: 3 e1dpcm) { .c-reading-companion__figure-title,.u-h3,.u-h4{font-family:Georgia,Palatino,serif;font-style:normal;line-height:1.4}.u-h3{margin-bottom:1em}.u-h4{margin-bottom:1em}button{line-height:inherit}html,label{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}html{-webkit-font-smoothing:subpixel-antialiased;box-sizing:border-box;color:#333;font-size:100%;height:100%;line-height:1.61803;overflow-y:scroll}*{box-sizing:inherit}body{background:#fff;font-size:1.8em;margin:0;max-width:100%;min-height:100%}button,div,form,input{margin:0;padding:0}body,p{padding:0}a{color:#004b83;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}a>img{vertical-align:middle}h1,h2,h3{margin-top:0}h1{font-size:28px;font-weight:700}html.webfonts-loaded h1,html.webfonts-loaded h2,html.webfonts-loaded h3{font-family:Europa,Trebuchet MS}h1,h2,h3{color:#1b3051;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-style:normal;line-height:1.4}h2{font-size:26px;font-weight:700;margin-bottom:.5em}h3{font-size:18px;margin-bottom:.7em}p{margin:0}ul{margin-top:0}p{margin-bottom:1.5em;overflow-wrap:break-word;word-break:break-word}p:last-child{margin-bottom:0}p:empty{display:none}.u-h3{font-size:1.75rem;font-size:1.5rem;font-weight:700}.c-reading-companion__figure-title{font-size:1.25rem;font-weight:700}.u-h4{font-size:1.25rem;font-weight:700}body{font-size:1.125rem}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-identifiers__open{color:#cc4b14}.c-article-title{font-size:1.5rem;line-height:1.25;margin-bottom:16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{margin-left:0}.c-article-author-list li,.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list__item svg{margin-left:4px}.c-article-author-list__show-more{margin-right:4px}@media only screen and (max-width:539px){html.js .js-small-screen-show-inline{display:inline!important;visibility:visible!important}.js .js-smaller-author-etal{display:none;visibility:hidden}}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-button-author-list svg{margin:1px 4px 0 0}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:0 0 16px}.c-article-metrics-bar__item{-webkit-box-align:baseline;align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Georgia,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;line-height:1.3;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article__sub-heading{color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;font-style:normal;font-weight:400;line-height:1.3;margin:0 0 8px}@media only screen and (min-width:768px){.c-article__sub-heading{font-size:1.5rem;line-height:1.24}}.c-reading-companion__figure-full-link svg{margin-left:4px}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;display:inline-block;font-size:.875rem;font-weight:700;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#0067c5;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title,.c-nature-box{line-height:1.3}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-reading-companion{clear:both}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{-webkit-box-orient:horizontal;-webkit-box-direction:normal;display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{-webkit-box-flex:1;flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#0067c5;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__figures-list,.c-reading-companion__references-list,.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1em;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{-webkit-box-pack:justify;display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{display:inline-block}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-article-associated-content__container .c-article-associated-content__collection.collection .c-article-associated-content__collection-label,.c-article-associated-content__container .c-article-associated-content__collection.section .c-article-associated-content__collection-label,.c-reading-companion__panel--active{display:block}.c-pdf-download{display:flex;margin-bottom:24px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{-webkit-box-pack:justify;-webkit-box-flex:1;background:#173962;border:1px solid transparent;border-radius:2px;color:#fff;display:flex;flex:1 1 0%;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;justify-content:space-between;line-height:1.3;padding:13px 24px;text-decoration:none}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-list-reset{list-style:none;margin:0;padding:0}.hide{display:none;visibility:hidden}@media only screen and (max-width:30em){.js .js-smaller-author-etal{display:none;visibility:hidden}}.c-article-associated-content__container a{text-decoration:underline}.c-article-associated-content__container .c-article-associated-content__collection.collection~.c-article-associated-content__collection.collection .c-article-associated-content__collection-label,.c-article-associated-content__container .c-article-associated-content__collection.section~.c-article-associated-content__collection.section .c-article-associated-content__collection-label,.c-article-associated-content__container .c-article-associated-content__title{display:none}.c-article-associated-content__container .c-article-associated-content__collection.collection,.c-article-associated-content__container .c-article-associated-content__collection.section{margin-bottom:5px}.c-article-associated-content__container .c-article-associated-content__collection.section~.c-article-associated-content__collection.collection{margin-top:28px}.c-article-associated-content__container .c-article-associated-content__collection:first-child{margin-top:0}.c-article-associated-content__container .c-article-associated-content__collection:last-child{margin-bottom:2.4rem}.c-article-associated-content__container .c-article-associated-content__collection-label{color:#1b3051}.c-article-associated-content__container .c-article-associated-content__collection-title{font-size:1.063rem;font-weight:400}.c-article-main-column .c-pdf-download{display:none}@media only screen and (max-width:1023px){.c-article-main-column .c-pdf-download{display:block}}.webfonts-loaded .c-article__sub-heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif} }</style>
        <link rel="stylesheet" data-inline-css-source="critical-css" href="/static/app-bmc/css/enhanced-article-415bd1e493.css"
              media="print" onload="this.media='only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)';this.onload=null">
    



        

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'genomebiology.biomedcentral.com',
            siteWithPath: 'genomebiology.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/gb-2013-14-10-r121',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            hasFigshareInvoked: false,
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        
    <script type="text/javascript" data-test="dataLayer">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/gb-2013-14-10-r121","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Transcription Start Site;Histone Modification;Gene Body;Histone Variant;Nucleosome Occupancy"},"contentInfo":{"imprint":"BioMed Central","title":"Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3","publishedAt":1386633600000,"publishedAtDate":"2013-12-10","author":["Daniel C Kraushaar","Wenfei Jin","Alika Maunakea","Brian Abraham","Misook Ha","Keji Zhao"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"genomebiology.biomedcentral.com","volume":"14","issue":"10","title":"Genome Biology","type":"FLAGSHIP","journalID":13059,"section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"BMC Flagships"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"},"attributes":{"featureFlags":[],"environment":"live"}},"event":"dataLayerCreated"}];
    </script>

        

        
            <script>
                window.dataLayer.push({
                    cmpAndNewGtmFeatureFlag: true
                });
            </script>
        

        
        
            <script>
                window.dataLayer.push({
                    ga4MeasurementId: 'G-PJCTJWPV25',
                    ga360TrackingId: 'UA-54492316-9',
                    twitterId: 'o47a2',
                    ga4ServerUrl: 'https://collect.biomedcentral.com',
                    imprint: 'biomedcentral'
                });
            </script>
        
        

        
    <script>
        (function(w, d) {
            w.config = w.config || {};
            w.config.mustardcut = false;

            
            if (w.matchMedia && w.matchMedia('only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)').matches) {
                w.config.mustardcut = true;
                d.classList.add('js');
                d.classList.remove('grade-c');
            }
        })(window, document.documentElement);
    </script>



        
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>

        
    <script class="js-entry">
        if (window.config.mustardcut) {
            (function(w, d) {
                
                
                
                    window.Component = {};
                    window.suppressShareButton = true;
                

                var currentScript = d.currentScript || d.head.querySelector('script.js-entry');

                
                function catchNoModuleSupport() {
                    var scriptEl = d.createElement('script');
                    return (!('noModule' in scriptEl) && 'onbeforeload' in scriptEl)
                }

                var headScripts = [
                    {'src': '/static/js/polyfill-es5-bundle-95f6277fcf.js', 'async': false},
                    {'src': '/static/js/airbrake-es5-bundle-80569d2d62.js', 'async': false},
                ];

                var bodyScripts = [
                    {'src': '/static/js/app-es5-bundle-5306b345a4.js', 'async': false, 'module': false},
                    {'src': '/static/js/app-es6-bundle-8425be8e90.js', 'async': false, 'module': true}
                    
                    
                        , {'src': '/static/js/global-article-es5-bundle-d18afc649a.js', 'async': false, 'module': false},
                        {'src': '/static/js/global-article-es6-bundle-cef3b249aa.js', 'async': false, 'module': true}
                    
                ];

                function createScript(script) {
                    var scriptEl = d.createElement('script');
                    scriptEl.src = script.src;
                    scriptEl.async = script.async;
                    if (script.module === true) {
                        scriptEl.type = "module";
                        if (catchNoModuleSupport()) {
                            scriptEl.src = '';
                        }
                    } else if (script.module === false) {
                        scriptEl.setAttribute('nomodule', true)
                    }
                    if (script.charset) {
                        scriptEl.setAttribute('charset', script.charset);
                    }

                    return scriptEl;
                }

                for (var i = 0; i < headScripts.length; ++i) {
                    var scriptEl = createScript(headScripts[i]);
                    currentScript.parentNode.insertBefore(scriptEl, currentScript.nextSibling);
                }

                d.addEventListener('DOMContentLoaded', function() {
                    for (var i = 0; i < bodyScripts.length; ++i) {
                        var scriptEl = createScript(bodyScripts[i]);
                        d.body.appendChild(scriptEl);
                    }
                });

                // Webfont repeat view
                var config = w.config;
                if (config && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    d.documentElement.className += ' webfonts-loaded';
                }
            })(window, document);
        }
    </script>




        
            
            
                <script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                
                    j.src = 'https://collect.biomedcentral.com/gtm.js?id=' + i + dl;
                
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>

            
            
        

        <script>
    (function (w, d, t) {
        function cc() {
            var h = w.location.hostname;
            var e = d.createElement(t),
            s = d.getElementsByTagName(t)[0];

            if (h.indexOf('springer.com') > -1) {
                e.src = 'https://push-content.springernature.io/pcf_sb_5_1617714720898560639/production_live/consent-bundle-17-9.js';
                e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
            } else if (h.indexOf('biomedcentral.com') > -1) {
                e.src = 'https://push-content.springernature.io/pcf_sb_5_1617714720898560639/production_live/consent-bundle-15-9.js';
                e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
            } else if (h.indexOf('springeropen.com') > -1) {
                e.src = 'https://push-content.springernature.io/pcf_sb_5_1617714720898560639/production_live/consent-bundle-16-9.js';
                e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-444fb65518.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window, document, 'script');
</script>


        
    
        
    


        
    
    <link rel="canonical" href="//genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r121"/>
    

        
        
        
        <meta property="og:url" content="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r121">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BioMed Central">
        <meta property="og:title" content="Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3 - Genome Biology">
        <meta property="og:description" content="Background Nucleosomes are present throughout the genome and must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms. Despite the development of protocols and techniques that have enabled us to map nucleosome occupancy genome-wide, the dynamic properties of nucleosomes remain poorly understood, particularly in mammalian cells. The histone variant H3.3 is incorporated into chromatin independently of DNA replication and requires displacement of existing nucleosomes for its deposition. Here, we measure H3.3 turnover at high resolution in the mammalian genome in order to present a genome-wide characterization of replication-independent H3.3-nucleosome dynamics. Results We developed a system to study the DNA replication-independent turnover of nucleosomes containing the histone variant H3.3 in mammalian cells. By measuring the genome-wide incorporation of H3.3 at different time points following epitope-tagged H3.3 expression, we find three categories of H3.3-nucleosome turnover in vivo: rapid turnover, intermediate turnover and, specifically at telomeres, slow turnover. Our data indicate that H3.3-containing nucleosomes at enhancers and promoters undergo rapid turnover that is associated with active histone modification marks including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z. The rate of turnover is negatively correlated with H3K27me3 at regulatory regions and with H3K36me3 at gene bodies. Conclusions We have established a reliable approach to measure turnover rates of H3.3-containing nucleosomes on a genome-wide level in mammalian cells. Our results suggest that distinct mechanisms control the dynamics of H3.3 incorporation at functionally different genomic regions.">
        <meta property="og:image" content="https://media.springernature.com/w200/springer-static/cover/journal/13059.jpg">
    
        

        
    <script type="application/ld+json">{"mainEntity":{"headline":"Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3","description":"Nucleosomes are present throughout the genome and must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms. Despite the development of protocols and techniques that have enabled us to map nucleosome occupancy genome-wide, the dynamic properties of nucleosomes remain poorly understood, particularly in mammalian cells. The histone variant H3.3 is incorporated into chromatin independently of DNA replication and requires displacement of existing nucleosomes for its deposition. Here, we measure H3.3 turnover at high resolution in the mammalian genome in order to present a genome-wide characterization of replication-independent H3.3-nucleosome dynamics. We developed a system to study the DNA replication-independent turnover of nucleosomes containing the histone variant H3.3 in mammalian cells. By measuring the genome-wide incorporation of H3.3 at different time points following epitope-tagged H3.3 expression, we find three categories of H3.3-nucleosome turnover in vivo: rapid turnover, intermediate turnover and, specifically at telomeres, slow turnover. Our data indicate that H3.3-containing nucleosomes at enhancers and promoters undergo rapid turnover that is associated with active histone modification marks including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z. The rate of turnover is negatively correlated with H3K27me3 at regulatory regions and with H3K36me3 at gene bodies. We have established a reliable approach to measure turnover rates of H3.3-containing nucleosomes on a genome-wide level in mammalian cells. Our results suggest that distinct mechanisms control the dynamics of H3.3 incorporation at functionally different genomic regions.","datePublished":"2013-12-10","dateModified":"2013-12-10","pageStart":"1","pageEnd":"15","sameAs":"https://doi.org/10.1186/gb-2013-14-10-r121","keywords":"Animal Genetics and Genomics,Human Genetics,Plant Genetics and Genomics,Microbial Genetics and Genomics,Bioinformatics,Evolutionary Biology","image":"https://static-content.springer.com/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig1_HTML.jpg","isPartOf":{"name":"Genome Biology","issn":["1474-760X"],"volumeNumber":"14","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"BioMed Central","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Kraushaar, Daniel C","affiliation":[{"name":"National Heart, Lung, and Blood Institute, NIH","address":{"name":"Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jin, Wenfei","affiliation":[{"name":"National Heart, Lung, and Blood Institute, NIH","address":{"name":"Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Maunakea, Alika","affiliation":[{"name":"National Heart, Lung, and Blood Institute, NIH","address":{"name":"Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Abraham, Brian","affiliation":[{"name":"National Heart, Lung, and Blood Institute, NIH","address":{"name":"Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ha, Misook","affiliation":[{"name":"Samsung Electronics Corporation","address":{"name":"Samsung Advanced Institute of Technology, Samsung Electronics Corporation, Yongin-Si, South Korea","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Zhao, Keji","affiliation":[{"name":"National Heart, Lung, and Blood Institute, NIH","address":{"name":"Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"zhaok@nhlbi.nih.gov","@type":"Person"}],"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>

        
    </head>

    <body class="journal journal-fulltext"
    
          >
        
    
        
            <!-- Google Tag Manager (noscript) -->
            <noscript>
                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                        height="0" width="0" style="display:none;visibility:hidden"></iframe>
            </noscript>
            <!-- End Google Tag Manager (noscript) -->
        
    

        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m1.00087166 7h5.99912834v-5.99912834c0-.55276615.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v5.99912834h5.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-5.9991283v5.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-5.9991283h-5.99912834c-.55276615 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812,263.868c-8.837,0-16,7.163-16,16s7.163,16,16,16s16-7.163,16-16S488.649,263.868,479.812,263.868z
	 M479.812,293.868c-7.732,0-14-6.269-14-14s6.268-14,14-14s14,6.269,14,14S487.545,293.868,479.812,293.868z"/><path d="M483.025,280.48l0.32-2.477h-2.453v-1.582c0-0.715,0.199-1.207,1.227-1.207h1.311v-2.213
	c-0.227-0.029-1.003-0.098-1.907-0.098c-1.894,0-3.186,1.154-3.186,3.271v1.826h-2.142v2.477h2.142v6.354h2.557v-6.354
	L483.025,280.48L483.025,280.48z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><g><path d="M486.416,276.191c-0.483,0.215-1.007,0.357-1.554,0.429c0.558-0.338,0.991-0.868,1.19-1.502
		c-0.521,0.308-1.104,0.536-1.72,0.657c-0.494-0.526-1.2-0.854-1.979-0.854c-1.496,0-2.711,1.213-2.711,2.71
		c0,0.212,0.023,0.419,0.069,0.616c-2.252-0.111-4.25-1.19-5.586-2.831c-0.231,0.398-0.365,0.866-0.365,1.361
		c0,0.94,0.479,1.772,1.204,2.257c-0.441-0.015-0.861-0.138-1.227-0.339v0.031c0,1.314,0.937,2.41,2.174,2.656
		c-0.227,0.062-0.47,0.098-0.718,0.098c-0.171,0-0.343-0.018-0.511-0.049c0.35,1.074,1.347,1.859,2.531,1.883
		c-0.928,0.726-2.095,1.16-3.366,1.16c-0.22,0-0.433-0.014-0.644-0.037c1.2,0.768,2.621,1.215,4.155,1.215
		c4.983,0,7.71-4.129,7.71-7.711c0-0.115-0.004-0.232-0.006-0.351C485.592,277.212,486.054,276.734,486.416,276.191z"/></g><path d="M479.812,263.868c-8.837,0-16,7.163-16,16s7.163,16,16,16s16-7.163,16-16S488.649,263.868,479.812,263.868z
	 M479.812,293.868c-7.732,0-14-6.269-14-14s6.268-14,14-14s14,6.269,14,14S487.545,293.868,479.812,293.868z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812,263.868c-8.838,0-16,7.163-16,16s7.162,16,16,16c8.837,0,16-7.163,16-16S488.649,263.868,479.812,263.868z
	 M479.812,293.868c-7.732,0-14-6.269-14-14s6.268-14,14-14c7.731,0,14,6.269,14,14S487.545,293.868,479.812,293.868z"/><g><path d="M478.552,285.348c-2.616,0.261-4.876-0.926-5.044-2.649c-0.167-1.722,1.814-3.33,4.433-3.588
		c2.609-0.263,4.871,0.926,5.041,2.647C483.147,283.479,481.164,285.089,478.552,285.348 M483.782,279.63
		c-0.226-0.065-0.374-0.109-0.259-0.403c0.25-0.639,0.276-1.188,0.005-1.581c-0.515-0.734-1.915-0.693-3.521-0.021
		c0,0-0.508,0.224-0.378-0.181c0.247-0.798,0.209-1.468-0.178-1.852c-0.87-0.878-3.194,0.032-5.183,2.027
		c-1.489,1.494-2.357,3.082-2.357,4.453c0,2.619,3.354,4.213,6.631,4.213c4.297,0,7.154-2.504,7.154-4.493
		C485.697,280.594,484.689,279.911,483.782,279.63"/><path d="M486.637,274.833c-1.039-1.154-2.57-1.592-3.982-1.291l0,0c-0.325,0.068-0.532,0.391-0.465,0.72
		c0.068,0.328,0.391,0.537,0.72,0.466c1.005-0.215,2.092,0.104,2.827,0.92c0.736,0.818,0.938,1.939,0.625,2.918l0,0
		c-0.102,0.318,0.068,0.661,0.39,0.762c0.32,0.104,0.658-0.069,0.763-0.391v-0.001C487.953,277.558,487.674,275.985,486.637,274.833
		"/><path d="M485.041,276.276c-0.504-0.562-1.25-0.774-1.938-0.63c-0.279,0.06-0.461,0.339-0.396,0.621
		c0.062,0.281,0.335,0.461,0.617,0.398l0,0c0.336-0.071,0.702,0.03,0.947,0.307s0.312,0.649,0.207,0.979l0,0
		c-0.089,0.271,0.062,0.565,0.336,0.654c0.274,0.09,0.564-0.062,0.657-0.336C485.686,277.604,485.549,276.837,485.041,276.276"/><path d="M478.694,282.227c-0.09,0.156-0.293,0.233-0.451,0.166c-0.151-0.062-0.204-0.235-0.115-0.389
		c0.093-0.155,0.284-0.229,0.44-0.168C478.725,281.892,478.782,282.071,478.694,282.227 M477.862,283.301
		c-0.253,0.405-0.795,0.58-1.202,0.396c-0.403-0.186-0.521-0.655-0.27-1.051c0.248-0.39,0.771-0.566,1.176-0.393
		C477.979,282.423,478.109,282.889,477.862,283.301 M478.812,280.437c-1.244-0.326-2.65,0.294-3.19,1.396
		c-0.553,1.119-0.021,2.369,1.236,2.775c1.303,0.42,2.84-0.225,3.374-1.436C480.758,281.989,480.1,280.77,478.812,280.437"/></g></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 3)"/></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H1.002A1.006 1.006 0 010 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L1.115 6.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-rss" viewBox="0 0 18 18"><path d="m.97480857 6.01583891.11675372.00378391c5.75903295.51984988 10.34261021 5.10537458 10.85988231 10.86480098.0494035.5500707-.3564674 1.0360406-.906538 1.0854441-.5500707.0494036-1.0360406-.3564673-1.08544412-.906538-.43079083-4.7965248-4.25151132-8.61886853-9.04770289-9.05180573-.55004837-.04965115-.95570047-.53580366-.90604933-1.08585203.04610464-.5107592.46858035-.89701345.96909831-.90983323zm1.52519143 6.95474179c1.38071187 0 2.5 1.1192881 2.5 2.5s-1.11928813 2.5-2.5 2.5-2.5-1.1192881-2.5-2.5 1.11928813-2.5 2.5-2.5zm-1.43253846-12.96884168c9.09581416.53242539 16.37540296 7.8163886 16.90205336 16.91294558.0319214.5513615-.389168 1.0242056-.9405294 1.056127-.5513615.0319214-1.0242057-.389168-1.0561271-.9405294-.4679958-8.08344784-6.93949306-14.55883389-15.02226722-15.03196077-.55134101-.03227286-.97212889-.50538538-.93985602-1.05672639.03227286-.551341.50538538-.97212888 1.05672638-.93985602z" fill-rule="evenodd"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-springer-collections" viewBox="3 3 32 32"><path fill-rule="evenodd" d="M25.583333,30.1249997 L25.583333,7.1207574 C25.583333,7.10772495 25.579812,7.10416665 25.5859851,7.10416665 L6.10400517,7.10571021 L6.10400517,30.1355179 C6.10400517,31.1064087 6.89406744,31.8958329 7.86448169,31.8958329 L26.057145,31.8958329 C25.7558021,31.374901 25.583333,30.7700915 25.583333,30.1249997 Z M4.33333333,30.1355179 L4.33333333,7.10571021 C4.33333333,6.12070047 5.12497502,5.33333333 6.10151452,5.33333333 L25.5859851,5.33333333 C26.5617372,5.33333333 27.3541664,6.13359035 27.3541664,7.1207574 L27.3541664,12.4166666 L32.6666663,12.4166666 L32.6666663,30.1098941 C32.6666663,32.0694626 31.0857174,33.6666663 29.1355179,33.6666663 L7.86448169,33.6666663 C5.91736809,33.6666663 4.33333333,32.0857174 4.33333333,30.1355179 Z M27.3541664,14.1874999 L27.3541664,30.1249997 C27.3541664,31.1030039 28.1469954,31.8958329 29.1249997,31.8958329 C30.1030039,31.8958329 30.8958329,31.1030039 30.8958329,30.1249997 L30.8958329,14.1874999 L27.3541664,14.1874999 Z M9.64583326,10.6458333 L22.0416665,10.6458333 L22.0416665,17.7291665 L9.64583326,17.7291665 L9.64583326,10.6458333 Z M11.4166666,12.4166666 L11.4166666,15.9583331 L20.2708331,15.9583331 L20.2708331,12.4166666 L11.4166666,12.4166666 Z M9.64583326,19.4999998 L22.0416665,19.4999998 L22.0416665,21.2708331 L9.64583326,21.2708331 L9.64583326,19.4999998 Z M9.64583326,23.0416665 L22.0416665,23.0416665 L22.0416665,24.8124997 L9.64583326,24.8124997 L9.64583326,23.0416665 Z M9.64583326,26.583333 L22.0416665,26.583333 L22.0416665,28.3541664 L9.64583326,28.3541664 L9.64583326,26.583333 Z"/></symbol><symbol id="icon-springer-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-springer-info" viewBox="0 0 24 24"><!--Generator: Sketch 63.1 (92452) - https://sketch.com--><g id="V&amp;I" stroke="none" stroke-width="1" fill-rule="evenodd"><g id="info" fill-rule="nonzero"><path d="M12,0 C18.627417,0 24,5.372583 24,12 C24,18.627417 18.627417,24 12,24 C5.372583,24 0,18.627417 0,12 C0,5.372583 5.372583,0 12,0 Z M12.5540543,9.1 L11.5540543,9.1 C11.0017696,9.1 10.5540543,9.54771525 10.5540543,10.1 L10.5540543,10.1 L10.5540543,18.1 C10.5540543,18.6522847 11.0017696,19.1 11.5540543,19.1 L11.5540543,19.1 L12.5540543,19.1 C13.1063391,19.1 13.5540543,18.6522847 13.5540543,18.1 L13.5540543,18.1 L13.5540543,10.1 C13.5540543,9.54771525 13.1063391,9.1 12.5540543,9.1 L12.5540543,9.1 Z M12,5 C11.5356863,5 11.1529412,5.14640523 10.8517647,5.43921569 C10.5505882,5.73202614 10.4,6.11546841 10.4,6.58954248 C10.4,7.06361656 10.5505882,7.45054466 10.8517647,7.7503268 C11.1529412,8.05010893 11.5356863,8.2 12,8.2 C12.4768627,8.2 12.8627451,8.05010893 13.1576471,7.7503268 C13.452549,7.45054466 13.6,7.06361656 13.6,6.58954248 C13.6,6.11546841 13.452549,5.73202614 13.1576471,5.43921569 C12.8627451,5.14640523 12.4768627,5 12,5 Z" id="Combined-Shape"/></g></g></symbol><symbol id="icon-springer-tick-circle" viewBox="0 0 24 24"><g id="Page-1" stroke="none" stroke-width="1" fill-rule="evenodd"><g id="springer-tick-circle" fill-rule="nonzero"><path d="M12,24 C5.372583,24 0,18.627417 0,12 C0,5.372583 5.372583,0 12,0 C18.627417,0 24,5.372583 24,12 C24,18.627417 18.627417,24 12,24 Z M7.657,10.79 C7.45285634,10.6137568 7.18569967,10.5283283 6.91717333,10.5534259 C6.648647,10.5785236 6.40194824,10.7119794 6.234,10.923 C5.87705269,11.3666969 5.93445559,12.0131419 6.364,12.387 L10.261,15.754 C10.6765468,16.112859 11.3037113,16.0695601 11.666,15.657 L17.759,8.713 C18.120307,8.27302248 18.0695334,7.62621189 17.644,7.248 C17.4414817,7.06995024 17.1751516,6.9821166 16.9064461,7.00476032 C16.6377406,7.02740404 16.3898655,7.15856958 16.22,7.368 L10.768,13.489 L7.657,10.79 Z" id="path-1"/></g></g></symbol><symbol id="icon-updates" viewBox="0 0 18 18"><g fill-rule="nonzero"><path d="M16.98 3.484h-.48c-2.52-.058-5.04 1.161-7.44 2.903-2.46-1.8-4.74-2.903-8.04-2.903-.3 0-.54.29-.54.58v9.813c0 .29.24.523.54.581 2.76.348 4.86 1.045 7.62 2.903.24.116.54.116.72 0 2.76-1.858 4.86-2.555 7.62-2.903.3-.058.54-.29.54-.58V4.064c0-.29-.24-.523-.54-.581zm-15.3 1.22c2.34 0 4.86 1.509 6.72 2.786v8.478c-2.34-1.394-4.38-2.09-6.72-2.439V4.703zm14.58 8.767c-2.34.348-4.38 1.045-6.72 2.439V7.374C12 5.632 14.1 4.645 16.26 4.645v8.826z"/><path d="M9 .058c-1.56 0-2.76 1.22-2.76 2.671C6.24 4.181 7.5 5.4 9 5.4c1.5 0 2.76-1.22 2.76-2.671 0-1.452-1.2-2.67-2.76-2.67zm0 4.413c-.96 0-1.8-.755-1.8-1.742C7.2 1.8 7.98.987 9 .987s1.8.755 1.8 1.742c0 .93-.84 1.742-1.8 1.742z"/></g></symbol></svg>
</div>


        

        

    <div class="u-vh-full">
        <a class="c-skip-link" href="#main-content">Skip to main content</a>
        
            
    
        <div class="u-hide u-show-following-ad"></div>
    
    <aside class="adsbox c-ad c-ad--728x90" data-component-mpu>
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                data-ad-type="LB1"
                data-pa11y-ignore
                data-gpt
                data-gpt-unitpath="/270604982/bmc/genomebiology/articles"
                data-gpt-sizes="728x90,970x90"
                data-gpt-targeting="pos=LB1;doi=10.1186/gb-2013-14-10-r121;type=article;kwrd=Transcription Start Site,Histone Modification,Gene Body,Histone Variant,Nucleosome Occupancy;pmc=L32030,B12008,L32020,L32010,L15001,L21001;"
            >
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/genomebiology/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/gb-2013-14-10-r121&amp;type=article&amp;kwrd=Transcription Start Site,Histone Modification,Gene Body,Histone Variant,Nucleosome Occupancy&amp;pmc=L32030,B12008,L32020,L32010,L15001,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                            src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/genomebiology/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/gb-2013-14-10-r121&amp;type=article&amp;kwrd=Transcription Start Site,Histone Modification,Gene Body,Histone Variant,Nucleosome Occupancy&amp;pmc=L32030,B12008,L32020,L32010,L15001,L21001&amp;"
                            alt="Advertisement"
                            width="728"
                            height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </aside>

        
        

         
    
        <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
    
    
        <div id="top" class="u-position-relative">
    <header class="c-header" data-test="publisher-header">
        <div class="c-header__container">
            <div class="c-header__brand u-mr-48" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img alt="BMC" itemprop="logo" width="76" height="18" role="img" src=/static/images/bmc/logos/logo-bmc-white-flagship-6c54473310.svg>
        <div class="c-logo__strapline">
            <img alt="Part of Springer Nature" width="173" height="16" role="img" src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-f224388d67.svg>
        </div>
    </a>
</div>

            </div>
            <div class="c-header__navigation">
                <button
                    type="button"
                    class="c-header__link u-button-reset u-mr-24"
                    data-expander
                    data-expander-target="#publisher-header-search"
                    data-expander-autofocus="firstTabbable"
                    data-test="header-search-button"
                    aria-controls="publisher-header-search"
                    aria-expanded="false">
                    <span class="u-display-flex u-align-items-center">
                        <span class="u-text-sm">Search</span>
                        <svg class="u-icon u-flex-static u-ml-8" aria-hidden="true" focusable="false">
                            <use xlink:href="#global-icon-search"></use>
                        </svg>
                    </span>
                </button>
                <nav>
                    <ul class="c-header__menu" data-enhanced-menu data-test="publisher-navigation">
                        
                            
                                <li class="c-header__item u-hide-at-lt-lg">
                                    <a class="c-header__link"
                                       href="//www.biomedcentral.com/journals">
                                        Explore journals
                                    </a>
                                </li>
                            
                                <li class="c-header__item u-hide-at-lt-lg">
                                    <a class="c-header__link"
                                       href="//www.biomedcentral.com/getpublished">
                                        Get published
                                    </a>
                                </li>
                            
                                <li class="c-header__item u-hide-at-lt-lg">
                                    <a class="c-header__link"
                                       href="//www.biomedcentral.com/about">
                                        About BMC
                                    </a>
                                </li>
                            
                        
                        <li class="c-header__item">
                            <a data-enhanced-account
                               class="c-header__link"
                               href="https://www.biomedcentral.com/account"
                               data-test="login-link">
                                My account
                            </a>
                        </li>
                    </ul>
                </nav>
            </div>
        </div>
    </header>
    <div class="c-popup-search u-js-hide" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="u-display-flex">
            <input id="publisherSearch" class="c-form-field__input" data-search-input autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""/>
            <div>
                <button class="u-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="u-icon u-flex-static" width="16" height="16" aria-hidden="true" focusable="false">
        <use xlink:href="#global-icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</div>

    

        
            <header class="c-journal-header c-journal-header--genome-biology ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner c-journal-header__inner--featured-journal">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
    <span class="c-journal-title__text  u-visually-hidden">Genome Biology</span>
    
        <div class="c-journal-title__featured-journal-lines c-journal-title__featured-journal-lines--genome-biology">
            <img src=/static/images/bmc/logos/genome-biology-logo-cf72c0ec54.svg class="c-journal-title__featured-journal-logo" alt="Genome Biology logo" itemprop="logo" role="img">
        </div>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="c-navbar__container">
                        
                            <div class="c-navbar__content">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" data-track="click" data-track-category="Home" data-track-action="Clicked journal navigation link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" data-track="click" data-track-category="About" data-track-action="Clicked journal navigation link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" data-track="click" data-track-category="Articles" data-track-action="Clicked journal navigation link" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" data-track="click" data-track-category="Submission Guidelines" data-track-action="Clicked journal navigation link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--featured-journal">
                    
                </div>
            </header>
            
        

        <div class="u-container u-mt-32 u-mb-32 u-clearfix" id="main-content" data-component="article-container">

            <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
                
    <div class="c-pdf-download u-clear-both">
        <a href="//genomebiology.biomedcentral.com/track/pdf/10.1186/gb-2013-14-10-r121.pdf" class="c-pdf-download__link" data-article-pdf="true"  data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external>
            <span>Download PDF</span>
            <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
        </a>
    </div>

                <article lang="en">
                    <div class="c-article-header">
                        
    


                        <ul class="c-article-identifiers" data-test="article-identifier">
                            
    
        <li class="c-article-identifiers__item" data-test="article-category">Research</li>
    
    
        <li class="c-article-identifiers__item">
            <span class="c-article-identifiers__open" data-test="open-access">Open Access</span>
        </li>
    
    

                            <li class="c-article-identifiers__item"><a href="#article-info" data-track="click" data-track-action="publication date" data-track-label="link">Published: <time datetime="2013-12-10">10 December 2013</time></a></li>
                        </ul>

                        <h1 class="c-article-title" data-test="article-title" data-article-title="">Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3</h1>
                        <ul class="c-article-author-list js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daniel_C-Kraushaar" data-author-popup="auth-Daniel_C-Kraushaar">Daniel C Kraushaar</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Wenfei-Jin" data-author-popup="auth-Wenfei-Jin">Wenfei Jin</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alika-Maunakea" data-author-popup="auth-Alika-Maunakea">Alika Maunakea</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Brian-Abraham" data-author-popup="auth-Brian-Abraham">Brian Abraham</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Misook-Ha" data-author-popup="auth-Misook-Ha">Misook Ha</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Keji-Zhao" data-author-popup="auth-Keji-Zhao" data-corresp-id="c1">Keji Zhao<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-email"></use></svg></a><sup class="u-js-hide"><a href="#Aff1">1</a></sup> </li></ul>
                        <p class="c-article-info-details" data-container-section="info">
                            
    <a data-test="journal-link" href="/"><i data-test="journal-title">Genome Biology</i></a>

                            <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 14</b>, Article number: <span data-test="article-number">R121</span> (<span data-test="article-publication-year">2013</span>)
            <a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                        </p>
                        
    <div class="c-article-metrics-bar__wrapper u-clear-both">
        <ul class="c-article-metrics-bar u-list-reset">
            
                <li class=" c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__count">6989 <span class="c-article-metrics-bar__label">Accesses</span></p>
                </li>
            
            
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__count">94 <span class="c-article-metrics-bar__label">Citations</span></p>
                </li>
            
            
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">3 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                    </li>
                
            
            <li class="c-article-metrics-bar__item">
                <p class="c-article-metrics-bar__details"><a href="/articles/10.1186/gb-2013-14-10-r121/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
            </li>
        </ul>
    </div>

                        
                        
    
        <div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
            <span class="c-status-message__icon">
                <svg class="u-icon" width="18" height="18" aria-hidden="true" focusable="false">
                    <use xlink:href="#icon-info"></use>
                </svg>
            </span>
            
                
                    <p>An <a href="/articles/10.1186/s13059-016-0886-3" class="relation-link" data-track="click" data-track-action="view linked article" data-track-label="link">Erratum</a> to this article was published on 04 February 2016</p>
                
            
        </div>
    

    

                        
                    </div>

                    <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Background</h3><p>Nucleosomes are present throughout the genome and must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms. Despite the development of protocols and techniques that have enabled us to map nucleosome occupancy genome-wide, the dynamic properties of nucleosomes remain poorly understood, particularly in mammalian cells. The histone variant H3.3 is incorporated into chromatin independently of DNA replication and requires displacement of existing nucleosomes for its deposition. Here, we measure H3.3 turnover at high resolution in the mammalian genome in order to present a genome-wide characterization of replication-independent H3.3-nucleosome dynamics.</p><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Results</h3><p>We developed a system to study the DNA replication-independent turnover of nucleosomes containing the histone variant H3.3 in mammalian cells. By measuring the genome-wide incorporation of H3.3 at different time points following epitope-tagged H3.3 expression, we find three categories of H3.3-nucleosome turnover <i>in vivo</i>: rapid turnover, intermediate turnover and, specifically at telomeres, slow turnover. Our data indicate that H3.3-containing nucleosomes at enhancers and promoters undergo rapid turnover that is associated with active histone modification marks including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z. The rate of turnover is negatively correlated with H3K27me3 at regulatory regions and with H3K36me3 at gene bodies.</p><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Conclusions</h3><p>We have established a reliable approach to measure turnover rates of H3.3-containing nucleosomes on a genome-wide level in mammalian cells. Our results suggest that distinct mechanisms control the dynamics of H3.3 incorporation at functionally different genomic regions.</p></div></div></section>
                    
    


                    <section data-title="Background"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Background</h2><div class="c-article-section__content" id="Sec1-content"><p>Incorporation of histone variants into chromatin critically influences the properties of nucleosomes that play important roles in regulating transcription and epigenetic memories [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e608">1</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Elsaesser SJ, Goldberg AD, Allis CD: New functions for an old variant: no substitute for histone H3.3. Curr Opin Genet Dev. 2010, 20: 110-117. 10.1016/j.gde.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR6" id="ref-link-section-d151128192e611">6</a>]. The histone variant H3.3 differs from canonical H3 by a few amino acids and is ubiquitously expressed in eukaryotes. Different from canonical histone H3, which is expressed in S phase and is incorporated into chromatin during DNA replication, H3.3 can be incorporated into chromatin independent of DNA replication [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Ahmad K, Henikoff S: The histone variant H3.3 marks active chromatin by replication-independent nucleosome assembly. Mol Cell. 2002, 9: 1191-1200. 10.1016/S1097-2765(02)00542-7." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR7" id="ref-link-section-d151128192e614">7</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Ray-Gallet D, Quivy JP, Scamps C, Martini EM, Lipinski M, Almouzni G: HIRA is critical for a nucleosome assembly pathway independent of DNA synthesis. Mol Cell. 2002, 9: 1091-1100. 10.1016/S1097-2765(02)00526-9." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR8" id="ref-link-section-d151128192e617">8</a>]. The incorporation of histone variants is tightly regulated by histone chaperones. The H3.3-specific chaperones Atrx-Daxx and HIRA deposit H3.3 primarily at telomeres and non-heterochromatic regions, respectively [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e620">9</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Szenker E, Lacoste N, Almouzni G: A developmental requirement for HIRA-dependent H3.3 deposition revealed at gastrulation in Xenopus. Cell Rep. 2012, 1: 730-740. 10.1016/j.celrep.2012.05.006." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR12" id="ref-link-section-d151128192e624">12</a>]. Despite small differences in amino acid sequence between H3.1, H3.2 and H3.3, each is distributed differentially across the genome and carries its own characteristic histone modification signature, strongly suggesting distinct functional roles for each H3 variant [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e627">9</a>]. Histone marks that are associated with gene activation, such as acetylation marks and H3K4me3, are typically found on H3.3 whereas H3K27me2 and H3K9me3 are found on H3.2 [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Hake SB, Garcia BA, Duncan EM, Kauer M, Dellaire G, Shabanowitz J, Bazett-Jones DP, Allis CD, Hunt DF: Expression patterns and post-translational modifications associated with mammalian histone H3 variants. J Biol Chem. 2006, 281: 559-568. 10.1074/jbc.M509266200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR13" id="ref-link-section-d151128192e630">13</a>]. Marks associated with gene silencing are predominantly found on H3.1. However, the precise relationship between H3.3 deposition and transcription are not well understood. In <i>Drosophila</i>, H3.3 replaces H3 during gene activation and becomes deposited in active chromatin and especially very highly expressed ribosomal DNA [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Schwartz BE, Ahmad K: Transcriptional activation triggers deposition and removal of the histone variant H3.3. Genes Dev. 2005, 19: 804-814. 10.1101/gad.1259805." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR14" id="ref-link-section-d151128192e636">14</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Mito Y, Henikoff JG, Henikoff S: Genome-scale profiling of histone H3.3 replacement patterns. Nat Genet. 2005, 37: 1090-1097. 10.1038/ng1637." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR15" id="ref-link-section-d151128192e639">15</a>]. In mammalian tissues the pattern of H3.3 enrichment is also associated with gene activity and H3.3 is generally associated with the transcription start site (TSS), transcription end site (TES) and gene bodies of active genes [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Tamura T, Smith M, Kanno T, Dasenbrock H, Nishiyama A, Ozato K: Inducible deposition of the histone variant H3.3 in interferon-stimulated genes. J Biol Chem. 2009, 284: 12217-12225. 10.1074/jbc.M805651200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR16" id="ref-link-section-d151128192e643">16</a>], although the unique chromatin of embryonic stem cells (ESCs) also carries H3.3 at promoters of certain inactive genes [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e646">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e649">9</a>]. In spite of their independent evolution, plant H3.3 and H3.1 display a broadly similar distribution to animal H3 variants, which indicates a conserved function for H3 variants [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Stroud H, Otero S, Desvoyes B, Ramirez-Parra E, Jacobsen SE, Gutierrez C: Genome-wide analysis of histone H3.1 and H3.3 variants in Arabidopsis thaliana. Proc Natl Acad Sci U S A. 2012, 109: 5370-5375. 10.1073/pnas.1203145109." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR17" id="ref-link-section-d151128192e652">17</a>]. Nucleosome occupancy and positioning are critical to regulating gene transcription and epigenome maintenance [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Bai L, Morozov AV: Gene regulation by nucleosome positioning. Trends Genet. 2010, 26: 476-483. 10.1016/j.tig.2010.08.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR18" id="ref-link-section-d151128192e655">18</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Svaren J, Horz W: Transcription factors vs nucleosomes: regulation of the PHO5 promoter in yeast. Trends Biochem Sci. 1997, 22: 93-97. 10.1016/S0968-0004(97)01001-3." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR19" id="ref-link-section-d151128192e658">19</a>]. In mammalian cells, although nucleosomes are present at promoters and enhancers, they must be dynamically regulated to accommodate binding of transcription factors and RNA polymerase machineries by various mechanisms [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Schones DE, Cui K, Cuddapah S, Roh TY, Barski A, Wang Z, Wei G, Zhao K: Dynamic regulation of nucleosome positioning in the human genome. Cell. 2008, 132: 887-898. 10.1016/j.cell.2008.02.022." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR20" id="ref-link-section-d151128192e662">20</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Boeger H, Griesenbeck J, Strattan JS, Kornberg RD: Removal of promoter nucleosomes by disassembly rather than sliding in vivo. Mol Cell. 2004, 14: 667-673. 10.1016/j.molcel.2004.05.013." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR23" id="ref-link-section-d151128192e665">23</a>].</p><p>Intrinsic nucleosome characteristics such as inclusion of histone variants as well as extrinsic factors such as ATP-dependent nucleosome remodeling, the transcriptional machinery and various other factors shape the dynamic profile of nucleosomes [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Struhl K, Segal E: Determinants of nucleosome positioning. Nat Struct Mol Biol. 2013, 20: 267-273. 10.1038/nsmb.2506." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR24" id="ref-link-section-d151128192e671">24</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Giresi PG, Gupta M, Lieb JD: Regulation of nucleosome stability as a mediator of chromatin function. Curr Opin Genet Dev. 2006, 16: 171-176. 10.1016/j.gde.2006.02.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR25" id="ref-link-section-d151128192e674">25</a>]. Despite the development of protocols and techniques that have enabled us to map the genome-wide nucleosome occupancy, their dynamic properties are only poorly understood. Our current understanding of the dynamics of nucleosomes comes from studies performed in yeast and <i>Drosophila</i>. Transgenic epitope-tagged histones can be inducibly expressed to estimate nucleosome turnover and allow detection of specific histone incorporation [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Dion MF, Kaplan T, Kim M, Buratowski S, Friedman N, Rando OJ: Dynamics of replication-independent histone turnover in budding yeast. Science. 2007, 315: 1405-1408. 10.1126/science.1134053." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR26" id="ref-link-section-d151128192e680">26</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Rufiange A, Jacques PE, Bhat W, Robert F, Nourani A: Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol Cell. 2007, 27: 393-405. 10.1016/j.molcel.2007.07.011." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR28" id="ref-link-section-d151128192e683">28</a>]. Alternatively, newly synthesized, native histones can be metabolically labeled with an amino acid analogue that is coupled to an affinity tag (CATCH-IT), which allows for detection of H3/H4 tetramers [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Deal RB, Henikoff S: Capturing the dynamic epigenome. Genome Biol. 2010, 11: 218-10.1186/gb-2010-11-10-218." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR29" id="ref-link-section-d151128192e687">29</a>]. Using these strategies, it has been shown that nucleosome exchange is rapid at promoters and coding regions, and relatively slower at heterochromatic regions. Furthermore, differential turnover can be quite localized. For instance, faster nucleosome turnover has been detected at Trithorax-group binding sites than at polycomb group protein binding sites [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Dion MF, Kaplan T, Kim M, Buratowski S, Friedman N, Rando OJ: Dynamics of replication-independent histone turnover in budding yeast. Science. 2007, 315: 1405-1408. 10.1126/science.1134053." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR26" id="ref-link-section-d151128192e690">26</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Deal RB, Henikoff JG, Henikoff S: Genome-wide kinetics of nucleosome turnover determined by metabolic labeling of histones. Science. 2010, 328: 1161-1164. 10.1126/science.1186777." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR30" id="ref-link-section-d151128192e693">30</a>].</p><p>Apart from the measurement of turnover, inducible expression systems with tagged histones have also contributed to our understanding of mechanistic aspects that pertain to histone deposition. For instance, studies from yeast have shown that Asf1 is required for the deposition of H3 and that the amino termini of both H2B and H3 are not required for their incorporation into nucleosomes [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Jamai A, Imoberdorf RM, Strubin M: Continuous histone H2B and transcription-dependent histone H3 exchange in yeast cells outside of replication. Mol Cell. 2007, 25: 345-355. 10.1016/j.molcel.2007.01.019." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR27" id="ref-link-section-d151128192e699">27</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Rufiange A, Jacques PE, Bhat W, Robert F, Nourani A: Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol Cell. 2007, 27: 393-405. 10.1016/j.molcel.2007.07.011." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR28" id="ref-link-section-d151128192e702">28</a>].</p><p>While these techniques measure average histone deposition rates across cell populations, alternative techniques such as FRAP and SNAP-tag have allowed the deposition of histones in individual cells to be visualized [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Jansen LE, Black BE, Foltz DR, Cleveland DW: Propagation of centromeric chromatin requires exit from mitosis. J Cell Biol. 2007, 176: 795-805. 10.1083/jcb.200701066." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR31" id="ref-link-section-d151128192e708">31</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Kimura H, Cook PR: Kinetics of core histones in living human cells: little exchange of H3 and H4 and some rapid exchange of H2B. J Cell Biol. 2001, 153: 1341-1353. 10.1083/jcb.153.7.1341." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR33" id="ref-link-section-d151128192e711">33</a>].</p><p>The genome-wide turnover of the histone variant H3.3 in mammals has not been studied up until now. Global FRAP studies in HeLa cells with green fluorescent protein (GFP)-tagged histones revealed only cycling of H2B. Canonical H3 and H4, in contrast, exhibited very slow cycling, and the majority of H3 remained permanently bound outside S phase [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Kimura H, Cook PR: Kinetics of core histones in living human cells: little exchange of H3 and H4 and some rapid exchange of H2B. J Cell Biol. 2001, 153: 1341-1353. 10.1083/jcb.153.7.1341." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR33" id="ref-link-section-d151128192e718">33</a>]. Slow turnover of core histones may be a feature of somatic cells since core histone exchange is substantially more rapid in pluripotent ESCs than in differentiated cell types [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Meshorer E, Yellajoshula D, George E, Scambler PJ, Brown DT, Misteli T: Hyperdynamic plasticity of chromatin proteins in pluripotent embryonic stem cells. Dev Cell. 2006, 10: 105-116. 10.1016/j.devcel.2005.10.017." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR34" id="ref-link-section-d151128192e721">34</a>]. Hence, fast turnover may be inherently linked to cell plasticity.</p><p>In this study, we developed a versatile technique to map the dynamics of histone variant incorporation into chromatin in mammalian cells. Using this technique, we mapped the replication-independent incorporation of the histone variant H3.3 in mouse embryonic fibroblasts (MEFs). We were able to track H3.3 incorporation across a relatively short time window of several hours after induction of H3.3 as well as over a longer time frame of up to 72 hours. By combining our chromatin immunoprecipitation (ChIP)-based technique with high-throughput sequencing, we measured the H3.3-nucleosome turnover kinetics at the genome-wide level. Our results reveal three major categories of H3.3 nucleosome turnover: (1) rapid turnover at enhancers and promoters; (2) intermediate turnover at gene bodies; and (3) slow turnover at heterochromatic regions. We find faster H3.3 turnover at enhancers and promoters is positively correlated with active histone modifications, including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z, whereas slower turnover is negatively correlated with H3K27me3 and H3K36me3 modifications. These results show that distinct mechanisms of histone deposition and eviction pertain to the dynamics of nucleosomes at different functional chromatin regions. We also show that turnover is related to the presence of specific histone marks, strongly suggesting that histone modifications are important determinants of nucleosome stability.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">An ectopic expression system to measure turnover of H3.3</h3><p>In order to track histone incorporation and thereby assay the genome-wide dynamics of the histone variant H3.3, we generated MEFs that carry a cytomegalovirus-controlled tetracycline (TET) transactivator and hemagglutinin (HA)/FLAG-tagged H3.3 expression cassette controlled by tetracycline response elements. This TET-ON expression system allowed us to induce the expression of a HA/FLAG-tagged version of H3.3 by addition of the tetracycline analog doxycycline (DOX) (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig1">1</a>A). In our tetracycline inducible-HA/FLAG-H3.3 MEF cell line, we detected robust H3.3 expression as early as 2 hours after DOX addition that continued to increase until 48 hours after DOX addition. No tagged H3.3 expression was detected in the absence of DOX (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig1">1</a>B). Immunoblotting against H3.3 revealed that transgenic H3.3 expression levels were only a small fraction of those of endogenous H3.3. Furthermore we verified that the HA/FLAG tags did not interfere with the H3K4me3 modification of H3.3 (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig1">1</a>C).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Figure 1"><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig1_HTML.jpg?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig1_HTML.jpg" alt="figure 1" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>A versatile system to study replication-independent nucleosome dynamics in mammals. (A)</b> Schematic of TET-inducible expression system to study H3.3 turnover. CMV, cytomegalovirus; rtTA, reverse tetracycline-controlled transactivator; TRE, tetracycline responsive elements. <b>(B)</b> Western blot showing protein levels of transgenic HA/FLAG-H3.3 compared to endogenous H3.3. HA/FLAG-H3.3 expression 24 hours after DOX addition. The band marked with an asterisk is non-specific. The arrow marks transgenic HA/FLAG-H3.3. <b>(C)</b> Time course western blots of HA/FLAG-H3.3 expression. <b>(D)</b> Bromodeoxyuridine (BrdU) immunostaining of NIH/3 T3 cells treated with DNA polymerase inhibitor aphidicolin and DOX across time points of H3.3 induction. DMSO, dimethylsulfoxide. <b>(E)</b> Cell cycle analysis of cells treated with aphidicolin/DOX. Cells were stained with propidium iodide and analyzed by flow cytometry.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>In order to minimize the effect of replication-coupled histone disassembly, we arrested the cell cycle of confluent NIH/3 T3 MEF cells by treatment with the DNA polymerase (Pol) inhibitor aphidicolin. After 18 hours of aphidicolin treatment (time point 0) and across the time course of HA/FLAG-H3.3 induction, the MEF cell population was essentially devoid of cells in S phase and arrested at the G1/S phase boundary, as indicated by bromodeoxyuridine and propidium iodide staining (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig1">1</a>D,E). Thus, to monitor the genome-wide dynamics of replication-independent H3.3 incorporation, we induced HA/FLAG-H3.3 expression in cells arrested by aphidicolin, followed by ChIP-Seq analysis using the HA antibody at various time points. We took a ‘high-resolution’ approach by tracking histone incorporation across hourly time points of early protein expression and across a longer time frame of up to 48 hours.</p><h3 class="c-article__sub-heading" id="Sec4">Genome-wide characterization of H3.3 incorporation</h3><p>In order to characterize the genome-wide deposition of HA-H3.3, we sought to map the H3.3 distribution 72 hours post-induction. Consistent with previous reports from HeLa and mouse ESCs [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e793">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e796">9</a>], we found that H3.3 is strongly enriched in promoters, 5’ UTRs, as well as the 3’ ends of genes (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>A,B,F). In terms of read density, promoters and TES regions were overrepresented in the H3.3-enriched genomic regions (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>A). The majority of H3.3 peaks were detected at introns followed by peaks in intergenic regions (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>B). In line with the idea that H3.3 carries primarily active histone modifications [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Hake SB, Garcia BA, Duncan EM, Kauer M, Dellaire G, Shabanowitz J, Bazett-Jones DP, Allis CD, Hunt DF: Expression patterns and post-translational modifications associated with mammalian histone H3 variants. J Biol Chem. 2006, 281: 559-568. 10.1074/jbc.M509266200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR13" id="ref-link-section-d151128192e809">13</a>], we found strong overlap of H3.3 with the distribution profiles of histone modifications such as H3K4me1, H3K4me3, H3K9ac and H3K27ac. In contrast we detected substantially fewer regions of H3.3 incorporation at sites enriched with H3K27me3 (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>D, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>E). Since the vast majority of sharp H3.3 peaks in introns and intergenic regions were associated with enhancer marks, including H3K4me1, H3K27ac and H2A.Z, we considered them as putative enhancers in the following analyses.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Figure 2"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig2_HTML.jpg?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig2_HTML.jpg" alt="figure 2" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>Distribution and enrichment of H3.3 in the genome. (A)</b> Enrichment of HA-H3.3 at various genomic regions of different annotation. The total read count of each category was normalized over its total length. <b>(B)</b> Pie chart showing the proportions and numbers of peak centers that fall into various genomic categories. <b>(C)</b> Enrichment of HA-H3.3 at various repetitive elements. All_Rep, all repetitive elements; SAT, satellite regions; SINE, short interspersed nuclear element. <b>(D)</b> Count of histone mark peaks that overlap with HA-H3.3 peaks. Overlapped peaks were defined as overlapping when one peak (H3.3 or histone mark) shared 10% of reads with the other. ChIP-Seq libraries for various histone marks were prepared from cells treated with aphidicolin. <b>(E)</b> UCSC Genome browser view illustrating overlap of HA-H3.3 with various histone marks and H2AZ. <b>(F)</b> Distribution profile of HA-H3.3 based on mRNA expression levels as indicated by different colors. RNA-seq libraries were prepared from cells treated with aphidicolin. <b>(G)</b> Distribution profile of HA-H3.1 based on mRNA expression levels as indicated by different colors. ChIP-Seq and RNA-Seq libraries were prepared from untreated (no aphidicolin) cells.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>Although most of the repetitive elements showed obvious depletion of H3.3, high levels of H3.3 were detected at telomeres, rRNA and tRNA repeats (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>C). H3.3 co-localizes with pericentromeres in several differentiated cell types [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Wong LH, McGhie JD, Sim M, Anderson MA, Ahn S, Hannan RD, George AJ, Morgan KA, Mann JR, Choo KH: ATRX interacts with H3.3 in maintaining telomere structural integrity in pluripotent embryonic stem cells. Genome Res. 2010, 20: 351-360. 10.1101/gr.101477.109." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR10" id="ref-link-section-d151128192e863">10</a>]. We, however, did not find significant deposition of HA-H3.3 at pericentromeric regions. This strongly suggests that H3.3 deposition in these regions is a strictly replication-coupled process as we profiled the distribution of HA-H3.3 from cell cycle arrested and non-dividing cells.</p><p>To test the relationship between H3.3 incorporation and gene expression, we plotted the average profiles of H3.3 across several groups of genes sorted according to their expression levels (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>F). Unlike in mouse ESCs, we found that H3.3 is associated mostly with active genes and not inactive genes.</p><p>Enrichment levels of H3.3 at TSSs, gene bodies and TESs were positively correlated with the transcription level of the associated genes (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>F). Interestingly, we find that the relation between HA-H3.3 enrichment and gene expression follows a bimodal distribution at promoters. Only promoters of expressed genes carry substantial amounts of H3.3, and H3.3 enrichment levels do not increase further at even higher expression levels. In contrast, HA-H3.3 enrichment in gene body and TES regions was strongly positively correlated with gene expression levels. This indicates that H3.3 in gene bodies may be directly transcription coupled, whereas other mechanisms of H3.3 deposition may exist upstream of TSSs. As a control alongside H3.3, we generated a MEF cell line that expressed canonical HA-H3.1 instead of HA-H3.3, which differs in five amino acids from H3.3, and mapped its replication-coupled deposition. Changing H3.3 to H3.1 abolished the defined enrichment of H3.3 to a homogenous distribution similar to reads mapped from input samples, confirming its universal deposition throughout the mammalian genome (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>E). Furthermore, the pronounced pattern of H3.3 enrichment around the TSS, gene body and TES was lost. Slightly higher levels of H3.1 were detected around the TSSs of inactive and low-expressed promoters, indicating prevalence of H3.1 at inactive genes (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>G). Similar genome-wide ChIP-Seq profiles were obtained by Goldberg <i>et al.</i>[<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e887">9</a>] with zinc finger-targeted, heterozygously tagged H3.3B in ESCs: mutating HA-H3.3 to HA-H3.2 or to a HA-H3.1S31 hybrid altered the H3 distribution to a genome-wide pattern similar to that of input samples or ChIP-Seq profiles obtained with antibodies against general H3.</p><h3 class="c-article__sub-heading" id="Sec5">H3.3 displays early and late chromatin incorporation</h3><p>The above results indicated the steady-state enrichment of H3.3 across the genome. In order to measure turnover rates of H3.3, we profiled the dynamics of HA-H3.3 incorporation after 0, 1, 2, 3, 4, 5, 6, 12, 18, 24 and 48 hours of DOX treatment (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig3">3</a>A). Using a linear regression model, we calculated a turnover index for each individual H3.3 peak that was identified using SICER and found that the turnover indices are highly reproducible from two independent experiments (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig3">3</a>B).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Figure 3"><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig3_HTML.jpg?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig3_HTML.jpg" alt="figure 3" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>H3.3 nucleosomes display fast and slow turnover rates. (A)</b> Genome browser view of HA-H3.3 profiles at several genomic regions over a 6-hour time frame. Examples of HA-H3.3 peaks that exhibit slow (left two panels), intermediate (center two panels) and fast turnover (right two panels) are displayed. <b>(B)</b> Dot plot of turnover indices calculated from two independent experiments. <b>(C)</b> Heat map showing relative HA-H3.3 enrichment (red = low, green = high) across all time points sorted by turnover index from low to high. <b>(D)</b> Distribution of turnover indices reveals two populations of H3.3 nucleosome exchange.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>We observed very different H3.3 incorporation kinetics across the genome. A substantial number of sharp peaks appeared within 2 to 3 hours of DOX induction and reached their maximum enrichment well before the end point of the induction (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig3">3</a>A,C). The subsequent decline in read numbers that was observed at peaks with rapid H3.3 deposition may be a result of ‘read shifting’. As the total read coverage becomes saturated, more reads may shift to peaks, which appear at later stages of induction. Importantly, this way, sites of enrichment may be identified, which would otherwise go undetected in a steady state. Meanwhile, we also observed many broad peaks, mainly in transcribed gene body regions, which took 12 to 24 hours to be detected (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig3">3</a>A,C). Plotting the distribution of turnover rates revealed the existence of two populations of H3.3 peaks with slow turnover and fast turnover, respectively (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig3">3</a>D).</p><h3 class="c-article__sub-heading" id="Sec6">Distinct H3.3 exchange rates in promoters, enhancers and gene bodies</h3><p>To examine the H3.3 turnover rates at different genomic regions, we calculated the average mean and median turnover indices at each region (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>A). The analysis revealed that 5’ UTR and promoter regions generally have the highest turnover indices; enhancers have the second highest, and 3’ UTR and TES regions have the lowest turnover indices (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>A). To examine the timing of H3.3 incorporation in these different regions, we compared the average profiles of H3.3 density of either all genes or active genes for the first 12 hours or after 12 hours till 72 hours of induction (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>B-E). Notable incorporation of HA-H3.3 was detected at the promoter region within 3 hours. Further increases in deposition of HA-H3.3 were observed within 6 hours at the TSS and the majority of genes reached their maximum by 12 hours before starting to decline (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>B,D). Remarkably, the onset of HA-H3.3 deposition at gene bodies was not detected any time before the 6-hour time point (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>C,E). H3.3 deposition at gene bodies continued to increase from 12 hours to 72 hours of induction, when it reached the steady-state level. It is interesting to note that the signal of HA-H3.3 at the TSS exhibited decreases from 12 hours to 72 hours.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Figure 4"><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig4_HTML.jpg?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig4_HTML.jpg" alt="figure 4" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>Promoters and enhancers are associated with fast H3.3 nucleosome turnover whereas gene body and transcription end site regions are associated with slow H3.3 nucleosome turnover. (A)</b> Median and mean turnover indices (TIs) calculated for various genomic regions. Intergenic enhancers were defined as peaks that are located in intergenic regions at least 1 kb from the TSS. Intronic enhancers were defined as peaks located in introns only. Gene body regions were defined as broad peaks that spanned both intronic and exonic regions and shared significant overlap with H3K36me3. <b>(B-E)</b> Distribution profiles of HA-H3.3 from TSS to TES at different time points. <b>(F)</b> Distribution plot illustrating the range of turnover indices within various genomic categories. Turnover rates were log-normalized. RPKM, reads per kilobase of exon model per million reads. <b>(G)</b> Distribution plot illustrating the range of turnover indices of H3.3 nucleosomes within various genomic categories. Log of turnover indices was calculated and re-scaled from 0 to 1.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>High turnover of H3.3 in promoter regions of active genes suggests that nucleosome turnover may be correlated with transcriptional activation. Further examination of turnover rates at promoters indeed showed a modest but positive correlation with gene expression levels (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>F). The relationship between promoter turnover and gene transcription suggests additional mechanisms by which nucleosome exchange facilitates and/or is facilitated by gene activation.</p><p>To further examine variation of the H3.3 turnover rates within each genomic category, we plotted the distribution of turnover indices separated into these categories (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>G). The analysis revealed a relatively narrow range for the high turnover at promoters and 5’ UTRs and slow turnover at gene bodies and 3’ UTRs, respectively, suggesting that the H3.3 nucleosome exchange in these regions are controlled by distinct mechanisms at these respective sites (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>G). Interestingly, enhancer regions exhibited broad variability with regards to their turnover rates, indicating that not all regulatory regions are marked by high nucleosome turnover. Instead additional factors such as histone variants or histone modifications may contribute to nucleosome stability and turnover.</p><h3 class="c-article__sub-heading" id="Sec7">Rapid H3.3 nucleosome turnover is associated with active histone marks at promoters and enhancers</h3><p>Several studies have shown that H3.3 is enriched in transcriptional regulatory regions such as promoters and enhancers [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e1022">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e1025">9</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Mito Y, Henikoff JG, Henikoff S: Genome-scale profiling of histone H3.3 replacement patterns. Nat Genet. 2005, 37: 1090-1097. 10.1038/ng1637." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR15" id="ref-link-section-d151128192e1028">15</a>]. Consistent with these observations, we found that many H3.3 peaks are co-localized with active histone modification marks, including H3K4me1, H3K4me3, H3K9ac, H3K27ac and the histone variant H2A.Z, that are often associated with promoters and enhancers (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>D,E) [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e1034">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Creyghton MP, Cheng AW, Welstead GG, Kooistra T, Carey BW, Steine EJ, Hanna J, Lodato MA, Frampton GM, Sharp PA, Boyer LA, Young RA, Jaenisch R: Histone H3K27ac separates active from poised enhancers and predicts developmental state. Proc Natl Acad Sci U S A. 2010, 107: 21931-21936. 10.1073/pnas.1016071107." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR35" id="ref-link-section-d151128192e1038">35</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Rada-Iglesias A, Bajpai R, Swigut T, Brugmann SA, Flynn RA, Wysocka J: A unique chromatin signature uncovers early developmental enhancers in humans. Nature. 2011, 470: 279-283. 10.1038/nature09692." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR36" id="ref-link-section-d151128192e1041">36</a>]. To elucidate the relationship between H3.3 nucleosome turnover and histone modifications, we sorted all H3.3 peaks based on their turnover rates and displayed histone modification signals using heatmaps. The analysis indicated that faster turnover rates are generally associated with higher levels of H3K4me1, H3K4me3, H3K9ac, H3K27ac and H2A.Z, whereas slower turnover is associated with higher levels of H3K27me3 (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig5">5</a>A). More quantitative analyses confirmed that indeed H3.3 peaks with higher levels of active modifications are turned over faster. In contrast, H3.3 peaks associated with heterochromatic marks such as H3K9me2 and repressive marks such as H3K27me3 are turned over slower (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig5">5</a>B).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Figure 5"><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 5</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig5_HTML.jpg?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig5_HTML.jpg" alt="figure 5" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>Fast turnover rates are associated with active histone modifications. (A)</b> Heat map illustrating the relation between turnover rates and histone marks. Turnover indices were sorted from low to high (top to bottom). Relative enrichment (red = low, green = high) levels of various histone marks is shown. <b>(B)</b> Enrichment values for various histone marks were grouped into four bins with increasing enrichment levels and mean turnover indices were calculated for each bin. <b>(C)</b> Missing any of the five ‘active’ modifications decreases the H3.3 turnover index of enhancer nucleosomes. The turnover indices for enhancer H3.3 nucleosomes containing all five active modifications or missing one single modification are indicated on the y-axis. <b>(D)</b> Association with any of the five active modifications increases the turnover index of H3.3-nucleosomes at enhancers. The turnover indices for enhancer H3.3 nucleosomes containing none of the five active modifications or only one modification are indicated on the y-axis. <b>(E)</b> Association with either H3K27me3 or H3K36me3 decreases the turnover index of H3.3 nucleosomes. The turnover indices are indicated on the y-axis for H3.3 nucleosomes containing all five active modifications (only active) or five active modifications plus either H3K27me3 or H3K36me3 or both.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>To understand the differences between the enhancers with fast H3.3 turnover and those with slow H3.3 turnover (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>G, black line), we compared the histone modifications associated with these two groups of enhancers (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/gb-2013-14-10-r121#MOESM1">1</a>). The analysis revealed that enhancers that displayed high turnover carried significantly higher median levels of active chromatin marks whereas enhancers of low turnover carried higher levels of H3K27me3 (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/gb-2013-14-10-r121#MOESM1">1</a>). Unexpectedly, our data indicated a negative correlation between H3.3 turnover and H3K36me3 levels (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig5">5</a>A,B), in line with the idea that H3K36me3 stabilizes nucleosomes in order to prevent cryptic transcription [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Li B, Jackson J, Simon MD, Fleharty B, Gogol M, Seidel C, Workman JL, Shilatifard A: Histone H3 lysine 36 dimethylation (H3K36me2) is sufficient to recruit the Rpd3s histone deacetylase complex and to repress spurious transcription. J Biol Chem. 2009, 284: 7970-7976. 10.1074/jbc.M808220200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR37" id="ref-link-section-d151128192e1097">37</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Carrozza MJ, Li B, Florens L, Suganuma T, Swanson SK, Lee KK, Shia WJ, Anderson S, Yates J, Washburn MP, Workman JL: Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription. Cell. 2005, 123: 581-592. 10.1016/j.cell.2005.10.023." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR38" id="ref-link-section-d151128192e1101">38</a>].</p><p>To evaluate the relative contribution of individual histone modifications to turnover rates, we compared the group of enhancers associated with all five active modifications to groups missing single histone marks. The results indicate that the turnover rates decreased significantly by missing any of the active modifications (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig5">5</a>C). Consistent with this observation, association with only one of these modifications increased the turnover rates compared to the enhancers without these modifications (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig5">5</a>D). In contrast to these active modifications, inclusion of either H3K27me3 or H3K36me3 substantially decreased the turnover index of H3.3 nucleosomes (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig5">5</a>E).</p><p>In summary, we find that H3.3 exchange rate is positively correlated with active chromatin modifications (H3K4me1, H3K4me3, H3K9ac, H3K27ac and H2A.Z) and negatively correlated with repressive marks (H3K9me2 and H3K27me3). While the steady-state level of H3.3 is positively correlated with H3K36me3, its turnover rate is negatively correlated with H3K36me3.</p><h3 class="c-article__sub-heading" id="Sec8">The kinetics of HA-H3.3 incorporation at tRNA and rRNAs genes</h3><p>We found that HA-H3.3 is enriched in tRNAs and rRNA genes (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig2">2</a>C). The incorporation of HA-H3.3 was apparent from 6 hours of induction and continued to increase till 24 hours before it started to decline at 48 hours (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig6">6</a>A). The incorporation of HA-H3.3 at rRNA genes was apparent at 12 hours and reached its maximum at 48 hours (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig6">6</a>B). Both tRNA and rRNA exist as tandem repeat regions, are highly expressed and are transcribed by Pol III and Pol I, respectively. Pol III-transcribed genes share many epigenetic characteristics with Pol II transcribed genes, such as presence of H3K4me1/2/3 and absence of H3K27me3 [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Barski A, Chepelev I, Liko D, Cuddapah S, Fleming AB, Birch J, Cui K, White RJ, Zhao K: Pol II and its associated epigenetic marks are present at Pol III-transcribed noncoding RNA genes. Nat Struct Mol Biol. 2010, 17: 629-634. 10.1038/nsmb.1806." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR39" id="ref-link-section-d151128192e1135">39</a>]. Hence, high turnover at tRNA and rRNA repeats is consistent with our previous result that demonstrated high H3.3-nucleosome turnover at sites marked with active histone marks. High nucleosome exchange may directly facilitate accessibility of Pol III-specific transcription factors TFIIIB and TFIIIC to provide functional promoters and assist in transcriptional initiation.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Figure 6"><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 6</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig6_HTML.jpg?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_Fig6_HTML.jpg" alt="figure 6" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>Different repetitive sequences display distinct H3.3 turnover rates. (A-C)</b> Mapping of HA-H3.3 to repetitive elements across all time points showing turnover of H3.3 at tRNA <b>(A)</b>, rRNA <b>(B)</b> and telomeric repeats <b>(C)</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/gb-2013-14-10-r121/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec9">Slow H3.3 replacement in telomeres</h3><p>Compared to promoters, enhancers, gene body regions, tRNAs and rRNAs, the incorporation of HA-H3.3 at telomeres exhibited the slowest rate. It was detected at 48 hours and reached a maximum at 72 hours after induction (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig6">6</a>C), suggesting that a different mechanism for the HA-H3.3 deposition may take effect at telomeres. Heterochromatic regions are expected to have stable nucleosomes and most of their histone exchange is expected to take place during S phase. Yet, we observed slow rates of HA-H3.3 replacement at telomeres outside S phase, indicating slow replication-independent histone exchange that may be necessary for maintenance of telomeric stability. It is known that repetitive features of telomeres and pericentromeres have a relatively low affinity for nucleosomes and this may contribute to histone displacement [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Cacchione S, Cerone MA, Savino M: In vitro low propensity to form nucleosomes of four telomeric sequences. FEBS Lett. 1997, 400: 37-41. 10.1016/S0014-5793(96)01318-X." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR40" id="ref-link-section-d151128192e1178">40</a>].</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Discussion</h2><div class="c-article-section__content" id="Sec10-content"><p>Nucleosome dynamics and histone turnover are not well understood in mammals. In this report, we have developed a robust, TET-inducible system to study the kinetics of genome-wide deposition of the histone variant H3.3 in MEF cells. Using this system, we were able to measure turnover rates of genome-wide H3.3 target sites and were able to infer important differences between turnover rates of various genomic regions. Several categories of H3.3 deposition kinetics were observed: invariably high turnover at promoters, a broad range of turnover rates at enhancers, and slow turnover at gene bodies. In addition, repeat elements displayed dramatically different turnover rates with relatively high turnover at tRNA and rRNA elements and very slow H3.3 exchange at telomeres. These results provide novel insights into the genome-wide turnover of H3.3-containing nucleosomes and suggest distinct mechanisms of nucleosome assembly, stability and eviction in order to fulfill their function in regulating transcription and maintaining chromatin stability and integrity.</p><h3 class="c-article__sub-heading" id="Sec11">Genome-wide incorporation of H3.3</h3><p>Unlike canonical histones, the histone variant H3.3 can be deposited onto chromatin in a replication-independent manner [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Ahmad K, Henikoff S: The histone variant H3.3 marks active chromatin by replication-independent nucleosome assembly. Mol Cell. 2002, 9: 1191-1200. 10.1016/S1097-2765(02)00542-7." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR7" id="ref-link-section-d151128192e1196">7</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Ray-Gallet D, Quivy JP, Scamps C, Martini EM, Lipinski M, Almouzni G: HIRA is critical for a nucleosome assembly pathway independent of DNA synthesis. Mol Cell. 2002, 9: 1091-1100. 10.1016/S1097-2765(02)00526-9." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR8" id="ref-link-section-d151128192e1199">8</a>]. It is enriched in transcriptionally active regions, such as gene bodies and promoters and enhancers of mammalian cells [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e1202">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e1205">9</a>]. Incorporation of H3.3 into chromatin destabilizes the nucleosome structure and may facilitate transcriptional activation by generating a more accessible chromatin configuration [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e1208">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Jin C, Felsenfeld G: Nucleosome stability mediated by histone variants H3.3 and H2A.Z. Genes Dev. 2007, 21: 1519-1529. 10.1101/gad.1547707." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR2" id="ref-link-section-d151128192e1212">2</a>]. Its deposition at promoters as well as in gene bodies is associated with epigenetic inheritance and may contribute to epigenetic memories [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Ng RK, Gurdon JB: Epigenetic memory of an active gene state depends on histone H3.3 incorporation into chromatin in the absence of transcription. Nat Cell Biol. 2008, 10: 102-109. 10.1038/ncb1674." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR3" id="ref-link-section-d151128192e1215">3</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Tamura T, Smith M, Kanno T, Dasenbrock H, Nishiyama A, Ozato K: Inducible deposition of the histone variant H3.3 in interferon-stimulated genes. J Biol Chem. 2009, 284: 12217-12225. 10.1074/jbc.M805651200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR16" id="ref-link-section-d151128192e1218">16</a>].</p><p>Our analysis of H3.3 distribution at the steady state revealed its presence at intergenic regions that overlap with histone modifications that typically mark enhancers, such has H3K4me1, H3K27ac and H2A.Z, in addition to the enrichment at TSS regions. In coding regions, we find that H3.3 is broadly distributed within the gene body of actively transcribed genes as well as TESs and regions immediately after. This association is well correlated with gene expression levels as has been reported from other cell lines [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR1" id="ref-link-section-d151128192e1224">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR9" id="ref-link-section-d151128192e1227">9</a>]. At promoter regions we find strong differences in H3.3 enrichment between inactive genes and active genes but relatively moderate differences between different expression groups, which follows an almost bimodal distribution. Based on H3.3 incorporation studies that followed H3.3 incorporation over time it was suggested that H3.3 incorporation at promoters was constitutive as only minor increases in H3.3 enrichment were observed upon interferon stimulation [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Tamura T, Smith M, Kanno T, Dasenbrock H, Nishiyama A, Ozato K: Inducible deposition of the histone variant H3.3 in interferon-stimulated genes. J Biol Chem. 2009, 284: 12217-12225. 10.1074/jbc.M805651200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR16" id="ref-link-section-d151128192e1230">16</a>]. Our data argue against a direct transcription-coupled mechanism of incorporation at promoters as expression levels and enrichment levels are bimodal, which further suggests that the H3.3 incorporation may function upstream of the transcriptional activation. Enrichment of H3.3 within gene body regions and at the TES showed strong correlations with transcription levels, in line with the notion that H3.3 incorporation in gene bodies is directly coupled to elongation. The ChIP-Seq profiles between gene body H3.3 and Pol II are highly similar. Furthermore, a physical interaction between HIRA and Pol II has been demonstrated and provides further testament for a direct link between H3.3 deposition and transcriptional elongation [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Ray-Gallet D, Woolfe A, Vassias I, Pellentz C, Lacoste N, Puri A, Schultz DC, Pchelintsev NA, Adams PD, Jansen LE, Almouzni G: Dynamics of histone H3 deposition in vivo reveal a nucleosome gap-filling mechanism for H3.3 to maintain chromatin integrity. Mol Cell. 2011, 44: 928-941. 10.1016/j.molcel.2011.12.006." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR32" id="ref-link-section-d151128192e1233">32</a>]. The role of H3.3 deposition in gene bodies remains unclear but may be implicated in conferring epigenetic inheritance in order to facilitate constitutive transcription of genes after cell replication has taken place.</p><h3 class="c-article__sub-heading" id="Sec12">Dynamics of H3.3 incorporation and differential turnover</h3><p>Our results on the dynamics of H3.3 incorporation revealed three basic modes of H3.3 deposition kinetics. The most dynamic exchange of H3.3 was observed at promoters and enhancers. Robust signals of induced H3.3 were detected within 2 to 3 hours of induction at these regulatory regions, which is consistent with the rapid nucleosome turnover at chromatin boundaries observed in yeast [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Dion MF, Kaplan T, Kim M, Buratowski S, Friedman N, Rando OJ: Dynamics of replication-independent histone turnover in budding yeast. Science. 2007, 315: 1405-1408. 10.1126/science.1134053." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR26" id="ref-link-section-d151128192e1244">26</a>]. The second category of intermediate-rate incorporation of newly induced H3.3 was found at gene body regions of active genes. The incorporation of H3.3 to gene bodies became apparent at 12 hours and reached a maximum at 72 hours. The slowest incorporation of H3.3 was detected at telomeres, with detectable H3.3 deposition at around 24 to 48 hours post-induction. Such differential turnover suggests that multiple mechanisms of H3.3 deposition and displacement occur, which no doubt will become subject to further investigation.</p><p>The fast turnover at enhancers and promoters may involve histone chaperones such as HIRA and Atrx-Daxx and other chromatin remodeling enzymes as well as sequence-specific transcription factors. High turnover at promoters suggests that high nucleosome turnover is a pre-requisite for binding of the transcriptional machinery as well as transcription factors.</p><p>The concept of nucleosome versus transcription factor competition has been well established from the Pho5 promoter in yeast [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Svaren J, Horz W: Transcription factors vs nucleosomes: regulation of the PHO5 promoter in yeast. Trends Biochem Sci. 1997, 22: 93-97. 10.1016/S0968-0004(97)01001-3." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR19" id="ref-link-section-d151128192e1252">19</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Boeger H, Griesenbeck J, Strattan JS, Kornberg RD: Removal of promoter nucleosomes by disassembly rather than sliding in vivo. Mol Cell. 2004, 14: 667-673. 10.1016/j.molcel.2004.05.013." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR23" id="ref-link-section-d151128192e1255">23</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Adkins MW, Tyler JK: Transcriptional activators are dispensable for transcription in the absence of Spt6-mediated chromatin reassembly of promoter regions. Mol Cell. 2006, 21: 405-416. 10.1016/j.molcel.2005.12.010." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR41" id="ref-link-section-d151128192e1258">41</a>]. Without stimulation, nucleosomes are bound to the binding sites of the transactivator Pho4 and effectively repress expression of the gene. Only upon release of nucleosomes at the promoter can Pho4 bind to its target site and gene activation effectively occur [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Boeger H, Griesenbeck J, Strattan JS, Kornberg RD: Removal of promoter nucleosomes by disassembly rather than sliding in vivo. Mol Cell. 2004, 14: 667-673. 10.1016/j.molcel.2004.05.013." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR23" id="ref-link-section-d151128192e1261">23</a>]. Unlike in yeast, active promoters of mammalian cells are occupied by nucleosomes [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Schones DE, Cui K, Cuddapah S, Roh TY, Barski A, Wang Z, Wei G, Zhao K: Dynamic regulation of nucleosome positioning in the human genome. Cell. 2008, 132: 887-898. 10.1016/j.cell.2008.02.022." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR20" id="ref-link-section-d151128192e1264">20</a>]. So what is the role of their turnover? It is attractive to speculate that nucleosomes sterically compete with DNA binding factors and that accessibility can only be granted when nucleosome turnover is high [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Dion MF, Kaplan T, Kim M, Buratowski S, Friedman N, Rando OJ: Dynamics of replication-independent histone turnover in budding yeast. Science. 2007, 315: 1405-1408. 10.1126/science.1134053." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR26" id="ref-link-section-d151128192e1268">26</a>].</p><p>Although simple nucleosome depletion could accomplish accessibility, ‘transient’ nucleosome deposition may stabilize DNA integrity, which may be necessary for accurate transcription factor binding. Several families of transcription factors, so-called ‘pioneer factors’ [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Zaret KS, Caravaca JM, Tulin A, Sekiya T: Nuclear mobility and mitotic chromosome binding: similarities between pioneer transcription factor FoxA and linker histone H1. Cold Spring Harb Symp Quant Biol. 2010, 75: 219-226. 10.1101/sqb.2010.75.061." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR42" id="ref-link-section-d151128192e1274">42</a>], require the presence and ‘guidance’ of nucleosomes for proper binding and subsequent nucleosome remodeling [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Ballare C, Castellano G, Gaveglia L, Althammer S, Gonzalez-Vallinas J, Eyras E, Le Dily F, Zaurin R, Soronellas D, Vicent GP, Beato M: Nucleosome-driven transcription factor binding and gene regulation. Mol Cell. 2013, 49: 67-79." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR43" id="ref-link-section-d151128192e1277">43</a>]. Secondly, nucleosomes and their histone modifications provide docking stations for a variety of transcription factors and chromatin remodeling factors, which may require continuous recruitment for initiation of transcription. Continuous nucleosome turnover may allow for accessibility and nucleosome binding for chromatin factors at the same time, which could be facilitated by the presence of multiple active modifications at enhancers or promoters. In contrast, repression of transcription and epigenetic inheritance may require greater stability of nucleosome organization and thus slower turnover at these sites, which, in turn, could be facilitated by association with polycomb group proteins and H3K27me3 modification [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Deal RB, Henikoff JG, Henikoff S: Genome-wide kinetics of nucleosome turnover determined by metabolic labeling of histones. Science. 2010, 328: 1161-1164. 10.1126/science.1186777." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR30" id="ref-link-section-d151128192e1280">30</a>].</p><p>The modest rate of H3.3 incorporation in gene bodies and TES regions is closely related to active transcription, and the accumulative signals of H3.3 in these regions may require multiple rounds of transcription of a gene unit. It is generally thought that RNA polymerase displaces nucleosomes during elongation, as it cannot pass by nucleosomes ahead of it [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Schwabish MA, Struhl K: The Swi/Snf complex is important for histone eviction during transcriptional activation and RNA polymerase II elongation in vivo. Mol Cell Biol. 2007, 27: 6987-6995. 10.1128/MCB.00717-07." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR44" id="ref-link-section-d151128192e1287">44</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Teves SS, Henikoff S: Heat shock reduces stalled RNA polymerase II and nucleosome turnover genome-wide. Genes Dev. 2011, 25: 2387-2397. 10.1101/gad.177675.111." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR45" id="ref-link-section-d151128192e1290">45</a>]. Nucleosomes must be replaced behind passing RNA polymerase in order to maintain the fidelity of transcription and avoid cryptic transcription [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Kaplan CD, Laprade L, Winston F: Transcription elongation factors repress transcription initiation from cryptic sites. Science. 2003, 301: 1096-1099. 10.1126/science.1087374." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR46" id="ref-link-section-d151128192e1293">46</a>]. Unlike other histone chaperones, HIRA possesses unique DNA binding properties that allow it to bind to naked DNA. Due to its physical association with Pol II, a ‘gap filling’ mechanism has been proposed by which HIRA under the guidance of PolII will fill protective H3.3-nucleosomes into spaces of naked DNA [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Ray-Gallet D, Woolfe A, Vassias I, Pellentz C, Lacoste N, Puri A, Schultz DC, Pchelintsev NA, Adams PD, Jansen LE, Almouzni G: Dynamics of histone H3 deposition in vivo reveal a nucleosome gap-filling mechanism for H3.3 to maintain chromatin integrity. Mol Cell. 2011, 44: 928-941. 10.1016/j.molcel.2011.12.006." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR32" id="ref-link-section-d151128192e1296">32</a>]. Our study reaffirms the continuous eviction and replacement of H3.3-containing nucleosomes during transcription. Slow exchange of these nucleosomes likely enhances chromatin stability before nucleosomes are evicted during a new round of transcription. H3K36me3 is placed by SET2-related enzymes following active transcription and plays a role in facilitating nucleosome deacetylation [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Li B, Jackson J, Simon MD, Fleharty B, Gogol M, Seidel C, Workman JL, Shilatifard A: Histone H3 lysine 36 dimethylation (H3K36me2) is sufficient to recruit the Rpd3s histone deacetylase complex and to repress spurious transcription. J Biol Chem. 2009, 284: 7970-7976. 10.1074/jbc.M808220200." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR37" id="ref-link-section-d151128192e1299">37</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Kizer KO, Phatnani HP, Shibata Y, Hall H, Greenleaf AL, Strahl BD: A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation. Mol Cell Biol. 2005, 25: 3305-3316. 10.1128/MCB.25.8.3305-3316.2005." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR47" id="ref-link-section-d151128192e1303">47</a>]. Histone deacetylation stabilizes nucleosomes and thus may result in the observed negative correlation between H3K36me3 level and H3.3 turnover in gene body regions.</p><p>Our analyses suggest that heterochromatic regions turn over more slowly than euchromatic regions. We observed very slow turnover at telomeres and no turnover at all at pericentromeric regions, both of which are known to be enriched in H3.3. This suggests that pericentromeric nucleosomes are replaced only during replication, whereas telomeres undergo continuous exchange of their nucleosomes. Continuous exchange of nucleosomes at telomeres may be necessary for telomeric maintenance, although bulk telomeric nucleosomes may be exchanged during replication.</p><p>In summary, we have provided the basis for our understanding of nucleosome dynamics in mammals by providing a detailed picture of genome-wide replacement of the histone variant H3.3. This study will provide the platform for further studies into the determinants and mechanisms of nucleosome exchange.</p></div></div></section><section data-title="Conclusion"><div class="c-article-section" id="Sec13-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec13">Conclusion</h2><div class="c-article-section__content" id="Sec13-content"><p>In this study, we mapped the genome-wide H3.3-specific nucleosome occupancy and the dynamic turnover of this histone variant in mammalian cells. We found that H3.3 turnover rates vary dramatically across functionally distinct genomic regions: turnover was highest at promoters and enhancers, intermediate at gene bodies and slowest at telomeres. Furthermore, we delineated striking correlations between turnover rates, histone modifications and H2A.Z, suggesting that intrinsic nucleosome properties such as histone modifications and histone variant inclusions are important properties of nucleosome stability.</p></div></div></section><section data-title="Materials and methods"><div class="c-article-section" id="Sec14-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec14">Materials and methods</h2><div class="c-article-section__content" id="Sec14-content"><h3 class="c-article__sub-heading" id="Sec15">Cell line derivation</h3><p>cDNA of human H3.3B or mouse H3.1 (Hist1h3h) was subcloned in-frame with a HA and FLAG tag at the carboxyl terminus and cloned into the lentiviral plvx-Tight-Puro Vector (Clontech, Mountain View, CA, USA). Lentiviral particles were packaged in 293 T cells with the psPAX2 packaging plasmid. Subsequently, we transduced NIH/3 T3 Tet-On® 3G cells (Clontech) and drug-selected with puromycin for stable integration.</p><h3 class="c-article__sub-heading" id="Sec16">Western blotting</h3><p>Cells were lysed with RIPA buffer, and whole-cell lysates were resolved on SDS-PAGE, transferred onto nitrocellulose membranes, and blotted with anti-HA (Roche, Basel, Switzerland), anti-FLAG (M2, Sigma, St. Louis, MO, USA) antibodies at 1:1,000. For anti-H3.3 western blotting, histones were isolated by acid extraction as described in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Shechter D, Dormann HL, Allis CD, Hake SB: Extraction, purification and analysis of histones. Nat Protoc. 2007, 2: 1445-1457. 10.1038/nprot.2007.202." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR48" id="ref-link-section-d151128192e1337">48</a>] prior to immunoblotting with an anti-H3.3 antibody (Abcam, ab62642, Cambridge, UK). The anti-H3.3 antibody recognizes serine 31 of H3.3, which is not present on either H3.1 or H3.2, but cross-reactivity with other histone variants has not been tested experimentally (Abcam).</p><h3 class="c-article__sub-heading" id="Sec17">Cell cycle analysis</h3><p>Cells were examined for BrdU incorporation with the BrdU Labeling and Detection kit (Roche). For further cell cycle and DNA content analysis, cells were fixed with 70% ethanol, and subsequently stained with 50 μg/ml propidium iodide in 0.1% PBS-Triton-X with 0.2 mg/ml RNase A before analysis by flow cytometry.</p><h3 class="c-article__sub-heading" id="Sec18">Time course induction and preparation of ChIP-Seq/RNA-Seq libraries</h3><p>NIH/3 T3 MEFs were cultured in standard conditions with medium containing TET-compatible fetal bovine serum. MEFs were grown to 100% confluence and treated with 1 μg/ml aphidicolin for 18 hours. Adding 3 μg/ml doxycycline hyclate before crosslinking with formaldehyde at various time points induced HA/FLAG-H3.3 expression. ChIP-Seq experiments were performed as described previously [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Barski A, Cuddapah S, Cui K, Roh TY, Schones DE, Wang Z, Wei G, Chepelev I, Zhao K: High-resolution profiling of histone methylations in the human genome. Cell. 2007, 129: 823-837. 10.1016/j.cell.2007.05.009." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR49" id="ref-link-section-d151128192e1355">49</a>] with an antibody against HA. In order to correlate HA/FLAG-H3.3 turnover with the presence of histone modifications and/or gene expression levels, we prepared ChIP-Seq and RNA-Seq from cells that were treated with aphidicolin for 18 hours prior to crosslinking and cell lysis.</p><h3 class="c-article__sub-heading" id="Sec19">Data analysis</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">ChIP-Seq read mapping and peak calling</h4><p>ChIP-Seq reads were mapped to the mouse genome (mm9, or NCBI 37) using Bowtie [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Langmead B, Trapnell C, Pop M, Salzberg SL: Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. 2009, 10: R25-10.1186/gb-2009-10-3-r25." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR50" id="ref-link-section-d151128192e1371">50</a>], only allowing ‘one read per position’ matching. Redundant reads were removed from each ChIP-Seq library. We identified ChIP-Seq tag-enriched peaks using SICER [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Zang C, Schones DE, Zeng C, Cui K, Zhao K, Peng W: A clustering approach for identification of enriched domains from histone modification ChIP-Seq data. Bioinformatics. 2009, 25: 1952-1958. 10.1093/bioinformatics/btp340." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR51" id="ref-link-section-d151128192e1374">51</a>], which takes advantage of the enrichment information from neighboring bins to identify spatial clusters of signals that are unlikely to appear by chance. In this way, we could identify the dispersed epigenomic domains such as broad H3.3 domains on gene bodies. For calling of H3.3 peaks, we set the window size to 200 bp and gap size to 600 bp. Peaks with a false discovery rate higher than 0.01 were excluded from the analysis, compared with the input control libraries.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Analysis of H3.3 enrichment in genic regions and repetitive elements</h4><p>The RefSeq gene annotation (mm9 or NCBI37) was downloaded from the UCSC browser. The mouse genome was classified into 5’ UTR, exon, intron, 3’ UTR and intergenic based on the UCSC annotation. Then we defined another two categories: promoter (3 kb upstream of the TSS) and TES (from TES to 5 kb downstream). The total mapped reads in each category were normalized to its total length. All mouse consensus repetitive element sequences, including pericentromeric element, L1, SINE1 and telomeric element, were obtained from Repbase Update (RU) [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Jurka J, Kapitonov VV, Pavlicek A, Klonowski P, Kohany O, Walichiewicz J: Repbase Update, a database of eukaryotic repetitive elements. Cytogenet Genome Res. 2005, 110: 462-467. 10.1159/000084979." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR52" id="ref-link-section-d151128192e1385">52</a>]. The relative enrichment of H3.3 in each category of repetitive element was calculated by the proportions of reads mapped to repetitive elements normalized over input.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">H3.3 peak annotation and distribution profiling</h4><p>We used the position of the center of each peak for peak annotation. Two peaks from different libraries were considered as overlapping if the overlapped region accounts for 10% of the length of one peak. We classified genes with RPKM (reads per kilobase of exon model per million reads) &lt;1 as lowest (inactive) genes and the other genes as active genes. We grouped genes into five categories with equal numbers of genes (highest, high, intermediate, low, lowest) based on their RPKM values. For each gene, reads falling in peaks were counted according to their shifted positions in 50 bp windows for the regions from 3 kb upstream of the TSS to the TSS and from the TES to 5 kb downstream of the TES. Within gene bodies, reads falling in peaks were counted according to their shifted positions in windows equal to 1% length of each gene. The number of reads in each window was normalized by the total number of bases within the window, and the total number of peak-filtered reads in the corresponding sample to obtain a normalized read-tag density.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">RNA-Seq data analysis</h4><p>The reads from RNA-Seq libraries were mapped to the mouse genome (mm9, or NCBI37) using TopHat [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL: TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol. 2013, 14: R36-10.1186/gb-2013-14-4-r36." href="/articles/10.1186/gb-2013-14-10-r121#ref-CR53" id="ref-link-section-d151128192e1403">53</a>], a fast splice junction mapper. The gene expression level was measured by RPKM.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">Quantifying peak dynamics</h4><p>The total number of H3.3 peaks was identified by peak-calling at the 72-hour time point when both early appearing and late appearing peaks were readily detected. The number of reads in each of these peak regions was recorded and normalized over the total mapped reads for each ChIP-Seq library. The relative H3.3 enrichment of each peak was calculated by normalizing the normalized reads in the peak over the normalized reads in input. A linear regression model was used to calculate turnover indices (<i>TI</i>) for each individual peak. Assuming that enrichment of H3.3 at 0 hour is E<sub>0</sub>, then E<sub>t</sub> = <i>TI</i> × <i>t</i> + E<sub>0</sub>, where E<sub>t</sub> equals H3.3 enrichment at each time point (E<sub>1</sub>, E<sub>2</sub>, E<sub>3</sub>,…E<sub>n</sub>), t = time point (1, 2, 3,…n). For peaks that reached their maximum enrichment before the end time point of analysis (<i>t</i> = 72 h), multiple linear regression coefficients were calculated by fitting the end time points from time point of maximum enrichment to 72 h and <i>t</i> was adjusted correspondingly. We adopted the regression coefficient with the best fit (smallest <i>P</i>-value) as the turnover index of the peaks. The turnover index was scaled from 0 and 1 in order to compare the reproducibility between duplicate experiments.</p><h3 class="c-article__sub-heading" id="Sec25">Accession numbers</h3><p>Our ChIP-seq and RNA-seq data sets have been deposited in the Gene Expression Omnibus data base with accession number GSE51505.</p></div></div></section>
                    <section data-title="Abbreviations"><div class="c-article-section" id="abbreviations-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="abbreviations">Abbreviations</h2><div class="c-article-section__content" id="abbreviations-content"><dl class="c-abbreviation_list"><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>BP:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Base pair</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>ChIP:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Chromatin immunoprecipitation</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>DOX:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Doxycycline</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>ESC:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Embryonic stem cell</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>HA:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Hemagglutinin</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>MEF:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Mouse embryonic fibroblast</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>Pol:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Polymerase</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>RPKM:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Reads per kilobase of exon model per million reads</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>TES:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Transcription end site</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>TET:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Tetracycline</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>TSS:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Transcription start site</p>
                  </dd><dt class="c-abbreviation_list__term u-text-bold u-float-left u-pr-16"><dfn>UTR:</dfn></dt><dd class="c-abbreviation_list__description u-mb-24">
                    <p>Untranslated region.</p>
                  </dd></dl></div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Jin C, Zang C, Wei G, Cui K, Peng W, Zhao K, Felsenfeld G: H3.3/H2A.Z double variant-containing nucleosomes mark 'nucleosome-free regions' of active promoters and other regulatory regions. Nat Genet. 2009, 41: 941-945. 10.1038/ng.409.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXovFymu7o%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fng.409" aria-label="Article reference 1">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19633671" aria-label="PubMed reference 1" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125718" aria-label="PubMed Central reference 1" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=H3.3%2FH2A.Z%20double%20variant-containing%20nucleosomes%20mark%20%27nucleosome-free%20regions%27%20of%20active%20promoters%20and%20other%20regulatory%20regions&amp;journal=Nat%20Genet&amp;doi=10.1038%2Fng.409&amp;volume=41&amp;pages=941-945&amp;publication_year=2009&amp;author=Jin%2CC&amp;author=Zang%2CC&amp;author=Wei%2CG&amp;author=Cui%2CK&amp;author=Peng%2CW&amp;author=Zhao%2CK&amp;author=Felsenfeld%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Jin C, Felsenfeld G: Nucleosome stability mediated by histone variants H3.3 and H2A.Z. Genes Dev. 2007, 21: 1519-1529. 10.1101/gad.1547707.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXntlCrurw%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgad.1547707" aria-label="Article reference 2">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17575053" aria-label="PubMed reference 2" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1891429" aria-label="PubMed Central reference 2" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleosome%20stability%20mediated%20by%20histone%20variants%20H3.3%20and%20H2A.Z&amp;journal=Genes%20Dev&amp;doi=10.1101%2Fgad.1547707&amp;volume=21&amp;pages=1519-1529&amp;publication_year=2007&amp;author=Jin%2CC&amp;author=Felsenfeld%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Ng RK, Gurdon JB: Epigenetic memory of an active gene state depends on histone H3.3 incorporation into chromatin in the absence of transcription. Nat Cell Biol. 2008, 10: 102-109. 10.1038/ncb1674.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXkvVan" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fncb1674" aria-label="Article reference 3">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18066050" aria-label="PubMed reference 3" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenetic%20memory%20of%20an%20active%20gene%20state%20depends%20on%20histone%20H3.3%20incorporation%20into%20chromatin%20in%20the%20absence%20of%20transcription&amp;journal=Nat%20Cell%20Biol&amp;doi=10.1038%2Fncb1674&amp;volume=10&amp;pages=102-109&amp;publication_year=2008&amp;author=Ng%2CRK&amp;author=Gurdon%2CJB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Hu G, Cui K, Northrup D, Liu C, Wang C, Tang Q, Ge K, Levens D, Crane-Robinson C, Zhao K: H2A.Z facilitates access of active and repressive complexes to chromatin in embryonic stem cell self-renewal and differentiation. Cell Stem Cell. 2013, 12: 180-192. 10.1016/j.stem.2012.11.003.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVymurrE" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.stem.2012.11.003" aria-label="Article reference 4">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23260488" aria-label="PubMed reference 4" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=H2A.Z%20facilitates%20access%20of%20active%20and%20repressive%20complexes%20to%20chromatin%20in%20embryonic%20stem%20cell%20self-renewal%20and%20differentiation&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2012.11.003&amp;volume=12&amp;pages=180-192&amp;publication_year=2013&amp;author=Hu%2CG&amp;author=Cui%2CK&amp;author=Northrup%2CD&amp;author=Liu%2CC&amp;author=Wang%2CC&amp;author=Tang%2CQ&amp;author=Ge%2CK&amp;author=Levens%2CD&amp;author=Crane-Robinson%2CC&amp;author=Zhao%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Henikoff S, Furuyama T, Ahmad K: Histone variants, nucleosome assembly and epigenetic inheritance. Trends Genet. 2004, 20: 320-326. 10.1016/j.tig.2004.05.004.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXlt1ansrg%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.tig.2004.05.004" aria-label="Article reference 5">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15219397" aria-label="PubMed reference 5" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20variants%2C%20nucleosome%20assembly%20and%20epigenetic%20inheritance&amp;journal=Trends%20Genet&amp;doi=10.1016%2Fj.tig.2004.05.004&amp;volume=20&amp;pages=320-326&amp;publication_year=2004&amp;author=Henikoff%2CS&amp;author=Furuyama%2CT&amp;author=Ahmad%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Elsaesser SJ, Goldberg AD, Allis CD: New functions for an old variant: no substitute for histone H3.3. Curr Opin Genet Dev. 2010, 20: 110-117. 10.1016/j.gde.2010.01.003.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXltF2gsrw%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.gde.2010.01.003" aria-label="Article reference 6">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20153629" aria-label="PubMed reference 6" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860041" aria-label="PubMed Central reference 6" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20functions%20for%20an%20old%20variant%3A%20no%20substitute%20for%20histone%20H3.3&amp;journal=Curr%20Opin%20Genet%20Dev&amp;doi=10.1016%2Fj.gde.2010.01.003&amp;volume=20&amp;pages=110-117&amp;publication_year=2010&amp;author=Elsaesser%2CSJ&amp;author=Goldberg%2CAD&amp;author=Allis%2CCD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Ahmad K, Henikoff S: The histone variant H3.3 marks active chromatin by replication-independent nucleosome assembly. Mol Cell. 2002, 9: 1191-1200. 10.1016/S1097-2765(02)00542-7.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD38Xlt1Sksbs%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS1097-2765%2802%2900542-7" aria-label="Article reference 7">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12086617" aria-label="PubMed reference 7" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20histone%20variant%20H3.3%20marks%20active%20chromatin%20by%20replication-independent%20nucleosome%20assembly&amp;journal=Mol%20Cell&amp;doi=10.1016%2FS1097-2765%2802%2900542-7&amp;volume=9&amp;pages=1191-1200&amp;publication_year=2002&amp;author=Ahmad%2CK&amp;author=Henikoff%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Ray-Gallet D, Quivy JP, Scamps C, Martini EM, Lipinski M, Almouzni G: HIRA is critical for a nucleosome assembly pathway independent of DNA synthesis. Mol Cell. 2002, 9: 1091-1100. 10.1016/S1097-2765(02)00526-9.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XksFKjtbo%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS1097-2765%2802%2900526-9" aria-label="Article reference 8">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12049744" aria-label="PubMed reference 8" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=HIRA%20is%20critical%20for%20a%20nucleosome%20assembly%20pathway%20independent%20of%20DNA%20synthesis&amp;journal=Mol%20Cell&amp;doi=10.1016%2FS1097-2765%2802%2900526-9&amp;volume=9&amp;pages=1091-1100&amp;publication_year=2002&amp;author=Ray-Gallet%2CD&amp;author=Quivy%2CJP&amp;author=Scamps%2CC&amp;author=Martini%2CEM&amp;author=Lipinski%2CM&amp;author=Almouzni%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Goldberg AD, Banaszynski LA, Noh KM, Lewis PW, Elsaesser SJ, Stadler S, Dewell S, Law M, Guo X, Li X, Wen D, Chapgier A, DeKelver RC, Miller JC, Lee YL, Boydston EA, Holmes MC, Gregory PD, Greally JM, Rafii S, Yang C, Scambler PJ, Garrick D, Gibbons RJ, Higgs DR, Cristea IM, Urnov FD, Zheng D, Allis CD: Distinct factors control histone variant H3.3 localization at specific genomic regions. Cell. 2010, 140: 678-691. 10.1016/j.cell.2010.01.003.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXlsVSgtrs%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2010.01.003" aria-label="Article reference 9">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20211137" aria-label="PubMed reference 9" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2885838" aria-label="PubMed Central reference 9" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20factors%20control%20histone%20variant%20H3.3%20localization%20at%20specific%20genomic%20regions&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2010.01.003&amp;volume=140&amp;pages=678-691&amp;publication_year=2010&amp;author=Goldberg%2CAD&amp;author=Banaszynski%2CLA&amp;author=Noh%2CKM&amp;author=Lewis%2CPW&amp;author=Elsaesser%2CSJ&amp;author=Stadler%2CS&amp;author=Dewell%2CS&amp;author=Law%2CM&amp;author=Guo%2CX&amp;author=Li%2CX&amp;author=Wen%2CD&amp;author=Chapgier%2CA&amp;author=DeKelver%2CRC&amp;author=Miller%2CJC&amp;author=Lee%2CYL&amp;author=Boydston%2CEA&amp;author=Holmes%2CMC&amp;author=Gregory%2CPD&amp;author=Greally%2CJM&amp;author=Rafii%2CS&amp;author=Yang%2CC&amp;author=Scambler%2CPJ&amp;author=Garrick%2CD&amp;author=Gibbons%2CRJ&amp;author=Higgs%2CDR&amp;author=Cristea%2CIM&amp;author=Urnov%2CFD&amp;author=Zheng%2CD&amp;author=Allis%2CCD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Wong LH, McGhie JD, Sim M, Anderson MA, Ahn S, Hannan RD, George AJ, Morgan KA, Mann JR, Choo KH: ATRX interacts with H3.3 in maintaining telomere structural integrity in pluripotent embryonic stem cells. Genome Res. 2010, 20: 351-360. 10.1101/gr.101477.109.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXjtlCktbk%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgr.101477.109" aria-label="Article reference 10">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20110566" aria-label="PubMed reference 10" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2840985" aria-label="PubMed Central reference 10" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=ATRX%20interacts%20with%20H3.3%20in%20maintaining%20telomere%20structural%20integrity%20in%20pluripotent%20embryonic%20stem%20cells&amp;journal=Genome%20Res&amp;doi=10.1101%2Fgr.101477.109&amp;volume=20&amp;pages=351-360&amp;publication_year=2010&amp;author=Wong%2CLH&amp;author=McGhie%2CJD&amp;author=Sim%2CM&amp;author=Anderson%2CMA&amp;author=Ahn%2CS&amp;author=Hannan%2CRD&amp;author=George%2CAJ&amp;author=Morgan%2CKA&amp;author=Mann%2CJR&amp;author=Choo%2CKH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Lewis PW, Elsaesser SJ, Noh KM, Stadler SC, Allis CD: Daxx is an H3.3-specific histone chaperone and cooperates with ATRX in replication-independent chromatin assembly at telomeres. Proc Natl Acad Sci U S A. 2010, 107: 14075-14080. 10.1073/pnas.1008850107.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVCktrfO" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1008850107" aria-label="Article reference 11">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20651253" aria-label="PubMed reference 11" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922592" aria-label="PubMed Central reference 11" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Daxx%20is%20an%20H3.3-specific%20histone%20chaperone%20and%20cooperates%20with%20ATRX%20in%20replication-independent%20chromatin%20assembly%20at%20telomeres&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;doi=10.1073%2Fpnas.1008850107&amp;volume=107&amp;pages=14075-14080&amp;publication_year=2010&amp;author=Lewis%2CPW&amp;author=Elsaesser%2CSJ&amp;author=Noh%2CKM&amp;author=Stadler%2CSC&amp;author=Allis%2CCD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Szenker E, Lacoste N, Almouzni G: A developmental requirement for HIRA-dependent H3.3 deposition revealed at gastrulation in Xenopus. Cell Rep. 2012, 1: 730-740. 10.1016/j.celrep.2012.05.006.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVOrsrnP" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.celrep.2012.05.006" aria-label="Article reference 12">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22813747" aria-label="PubMed reference 12" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20developmental%20requirement%20for%20HIRA-dependent%20H3.3%20deposition%20revealed%20at%20gastrulation%20in%20Xenopus&amp;journal=Cell%20Rep&amp;doi=10.1016%2Fj.celrep.2012.05.006&amp;volume=1&amp;pages=730-740&amp;publication_year=2012&amp;author=Szenker%2CE&amp;author=Lacoste%2CN&amp;author=Almouzni%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Hake SB, Garcia BA, Duncan EM, Kauer M, Dellaire G, Shabanowitz J, Bazett-Jones DP, Allis CD, Hunt DF: Expression patterns and post-translational modifications associated with mammalian histone H3 variants. J Biol Chem. 2006, 281: 559-568. 10.1074/jbc.M509266200.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1Ciug%3D%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1074%2Fjbc.M509266200" aria-label="Article reference 13">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16267050" aria-label="PubMed reference 13" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Expression%20patterns%20and%20post-translational%20modifications%20associated%20with%20mammalian%20histone%20H3%20variants&amp;journal=J%20Biol%20Chem&amp;doi=10.1074%2Fjbc.M509266200&amp;volume=281&amp;pages=559-568&amp;publication_year=2006&amp;author=Hake%2CSB&amp;author=Garcia%2CBA&amp;author=Duncan%2CEM&amp;author=Kauer%2CM&amp;author=Dellaire%2CG&amp;author=Shabanowitz%2CJ&amp;author=Bazett-Jones%2CDP&amp;author=Allis%2CCD&amp;author=Hunt%2CDF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Schwartz BE, Ahmad K: Transcriptional activation triggers deposition and removal of the histone variant H3.3. Genes Dev. 2005, 19: 804-814. 10.1101/gad.1259805.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXjt1OjtLY%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgad.1259805" aria-label="Article reference 14">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15774717" aria-label="PubMed reference 14" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1074318" aria-label="PubMed Central reference 14" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20activation%20triggers%20deposition%20and%20removal%20of%20the%20histone%20variant%20H3.3&amp;journal=Genes%20Dev&amp;doi=10.1101%2Fgad.1259805&amp;volume=19&amp;pages=804-814&amp;publication_year=2005&amp;author=Schwartz%2CBE&amp;author=Ahmad%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Mito Y, Henikoff JG, Henikoff S: Genome-scale profiling of histone H3.3 replacement patterns. Nat Genet. 2005, 37: 1090-1097. 10.1038/ng1637.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVCntL7F" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fng1637" aria-label="Article reference 15">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16155569" aria-label="PubMed reference 15" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20profiling%20of%20histone%20H3.3%20replacement%20patterns&amp;journal=Nat%20Genet&amp;doi=10.1038%2Fng1637&amp;volume=37&amp;pages=1090-1097&amp;publication_year=2005&amp;author=Mito%2CY&amp;author=Henikoff%2CJG&amp;author=Henikoff%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Tamura T, Smith M, Kanno T, Dasenbrock H, Nishiyama A, Ozato K: Inducible deposition of the histone variant H3.3 in interferon-stimulated genes. J Biol Chem. 2009, 284: 12217-12225. 10.1074/jbc.M805651200.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXltVCmu78%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1074%2Fjbc.M805651200" aria-label="Article reference 16">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19244243" aria-label="PubMed reference 16" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673290" aria-label="PubMed Central reference 16" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Inducible%20deposition%20of%20the%20histone%20variant%20H3.3%20in%20interferon-stimulated%20genes&amp;journal=J%20Biol%20Chem&amp;doi=10.1074%2Fjbc.M805651200&amp;volume=284&amp;pages=12217-12225&amp;publication_year=2009&amp;author=Tamura%2CT&amp;author=Smith%2CM&amp;author=Kanno%2CT&amp;author=Dasenbrock%2CH&amp;author=Nishiyama%2CA&amp;author=Ozato%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Stroud H, Otero S, Desvoyes B, Ramirez-Parra E, Jacobsen SE, Gutierrez C: Genome-wide analysis of histone H3.1 and H3.3 variants in Arabidopsis thaliana. Proc Natl Acad Sci U S A. 2012, 109: 5370-5375. 10.1073/pnas.1203145109.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XlslOjsL8%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1203145109" aria-label="Article reference 17">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22431625" aria-label="PubMed reference 17" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325649" aria-label="PubMed Central reference 17" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20analysis%20of%20histone%20H3.1%20and%20H3.3%20variants%20in%20Arabidopsis%20thaliana&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;doi=10.1073%2Fpnas.1203145109&amp;volume=109&amp;pages=5370-5375&amp;publication_year=2012&amp;author=Stroud%2CH&amp;author=Otero%2CS&amp;author=Desvoyes%2CB&amp;author=Ramirez-Parra%2CE&amp;author=Jacobsen%2CSE&amp;author=Gutierrez%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Bai L, Morozov AV: Gene regulation by nucleosome positioning. Trends Genet. 2010, 26: 476-483. 10.1016/j.tig.2010.08.003.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXht12qurjN" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.tig.2010.08.003" aria-label="Article reference 18">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20832136" aria-label="PubMed reference 18" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20regulation%20by%20nucleosome%20positioning&amp;journal=Trends%20Genet&amp;doi=10.1016%2Fj.tig.2010.08.003&amp;volume=26&amp;pages=476-483&amp;publication_year=2010&amp;author=Bai%2CL&amp;author=Morozov%2CAV">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Svaren J, Horz W: Transcription factors vs nucleosomes: regulation of the PHO5 promoter in yeast. Trends Biochem Sci. 1997, 22: 93-97. 10.1016/S0968-0004(97)01001-3.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaK2sXhsl2rt74%3D" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0968-0004%2897%2901001-3" aria-label="Article reference 19">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9066259" aria-label="PubMed reference 19" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20factors%20vs%20nucleosomes%3A%20regulation%20of%20the%20PHO5%20promoter%20in%20yeast&amp;journal=Trends%20Biochem%20Sci&amp;doi=10.1016%2FS0968-0004%2897%2901001-3&amp;volume=22&amp;pages=93-97&amp;publication_year=1997&amp;author=Svaren%2CJ&amp;author=Horz%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Schones DE, Cui K, Cuddapah S, Roh TY, Barski A, Wang Z, Wei G, Zhao K: Dynamic regulation of nucleosome positioning in the human genome. Cell. 2008, 132: 887-898. 10.1016/j.cell.2008.02.022.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXjs1ait7s%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2008.02.022" aria-label="Article reference 20">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18329373" aria-label="PubMed reference 20" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20regulation%20of%20nucleosome%20positioning%20in%20the%20human%20genome&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2008.02.022&amp;volume=132&amp;pages=887-898&amp;publication_year=2008&amp;author=Schones%2CDE&amp;author=Cui%2CK&amp;author=Cuddapah%2CS&amp;author=Roh%2CTY&amp;author=Barski%2CA&amp;author=Wang%2CZ&amp;author=Wei%2CG&amp;author=Zhao%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Mulholland N, Xu Y, Sugiyama H, Zhao K: SWI/SNF-mediated chromatin remodeling induces Z-DNA formation on a nucleosome. Cell Biosci. 2012, 2: 3-10.1186/2045-3701-2-3.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xlsl2qsro%3D" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2F2045-3701-2-3" aria-label="Article reference 21">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22264354" aria-label="PubMed reference 21" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3293710" aria-label="PubMed Central reference 21" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=SWI%2FSNF-mediated%20chromatin%20remodeling%20induces%20Z-DNA%20formation%20on%20a%20nucleosome&amp;journal=Cell%20Biosci&amp;doi=10.1186%2F2045-3701-2-3&amp;volume=2&amp;publication_year=2012&amp;author=Mulholland%2CN&amp;author=Xu%2CY&amp;author=Sugiyama%2CH&amp;author=Zhao%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Schnitzler G, Sif S, Kingston RE: Human SWI/SNF interconverts a nucleosome between its base state and a stable remodeled state. Cell. 1998, 94: 17-27. 10.1016/S0092-8674(00)81217-9.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaK1cXkslSqtbo%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0092-8674%2800%2981217-9" aria-label="Article reference 22">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9674423" aria-label="PubMed reference 22" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20SWI%2FSNF%20interconverts%20a%20nucleosome%20between%20its%20base%20state%20and%20a%20stable%20remodeled%20state&amp;journal=Cell&amp;doi=10.1016%2FS0092-8674%2800%2981217-9&amp;volume=94&amp;pages=17-27&amp;publication_year=1998&amp;author=Schnitzler%2CG&amp;author=Sif%2CS&amp;author=Kingston%2CRE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Boeger H, Griesenbeck J, Strattan JS, Kornberg RD: Removal of promoter nucleosomes by disassembly rather than sliding in vivo. Mol Cell. 2004, 14: 667-673. 10.1016/j.molcel.2004.05.013.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXltlGntLw%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2004.05.013" aria-label="Article reference 23">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15175161" aria-label="PubMed reference 23" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Removal%20of%20promoter%20nucleosomes%20by%20disassembly%20rather%20than%20sliding%20in%20vivo&amp;journal=Mol%20Cell&amp;doi=10.1016%2Fj.molcel.2004.05.013&amp;volume=14&amp;pages=667-673&amp;publication_year=2004&amp;author=Boeger%2CH&amp;author=Griesenbeck%2CJ&amp;author=Strattan%2CJS&amp;author=Kornberg%2CRD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Struhl K, Segal E: Determinants of nucleosome positioning. Nat Struct Mol Biol. 2013, 20: 267-273. 10.1038/nsmb.2506.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXjsFOms7w%3D" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnsmb.2506" aria-label="Article reference 24">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23463311" aria-label="PubMed reference 24" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3740156" aria-label="PubMed Central reference 24" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Determinants%20of%20nucleosome%20positioning&amp;journal=Nat%20Struct%20Mol%20Biol&amp;doi=10.1038%2Fnsmb.2506&amp;volume=20&amp;pages=267-273&amp;publication_year=2013&amp;author=Struhl%2CK&amp;author=Segal%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Giresi PG, Gupta M, Lieb JD: Regulation of nucleosome stability as a mediator of chromatin function. Curr Opin Genet Dev. 2006, 16: 171-176. 10.1016/j.gde.2006.02.003.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xis1eht70%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.gde.2006.02.003" aria-label="Article reference 25">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16503136" aria-label="PubMed reference 25" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20nucleosome%20stability%20as%20a%20mediator%20of%20chromatin%20function&amp;journal=Curr%20Opin%20Genet%20Dev&amp;doi=10.1016%2Fj.gde.2006.02.003&amp;volume=16&amp;pages=171-176&amp;publication_year=2006&amp;author=Giresi%2CPG&amp;author=Gupta%2CM&amp;author=Lieb%2CJD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Dion MF, Kaplan T, Kim M, Buratowski S, Friedman N, Rando OJ: Dynamics of replication-independent histone turnover in budding yeast. Science. 2007, 315: 1405-1408. 10.1126/science.1134053.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXisVKrsrs%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1126%2Fscience.1134053" aria-label="Article reference 26">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17347438" aria-label="PubMed reference 26" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamics%20of%20replication-independent%20histone%20turnover%20in%20budding%20yeast&amp;journal=Science&amp;doi=10.1126%2Fscience.1134053&amp;volume=315&amp;pages=1405-1408&amp;publication_year=2007&amp;author=Dion%2CMF&amp;author=Kaplan%2CT&amp;author=Kim%2CM&amp;author=Buratowski%2CS&amp;author=Friedman%2CN&amp;author=Rando%2COJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Jamai A, Imoberdorf RM, Strubin M: Continuous histone H2B and transcription-dependent histone H3 exchange in yeast cells outside of replication. Mol Cell. 2007, 25: 345-355. 10.1016/j.molcel.2007.01.019.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXitl2jtLk%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2007.01.019" aria-label="Article reference 27">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17289583" aria-label="PubMed reference 27" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Continuous%20histone%20H2B%20and%20transcription-dependent%20histone%20H3%20exchange%20in%20yeast%20cells%20outside%20of%20replication&amp;journal=Mol%20Cell&amp;doi=10.1016%2Fj.molcel.2007.01.019&amp;volume=25&amp;pages=345-355&amp;publication_year=2007&amp;author=Jamai%2CA&amp;author=Imoberdorf%2CRM&amp;author=Strubin%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Rufiange A, Jacques PE, Bhat W, Robert F, Nourani A: Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol Cell. 2007, 27: 393-405. 10.1016/j.molcel.2007.07.011.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXptlymsrw%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2007.07.011" aria-label="Article reference 28">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17679090" aria-label="PubMed reference 28" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20replication-independent%20histone%20H3%20exchange%20occurs%20predominantly%20at%20promoters%20and%20implicates%20H3%20K56%20acetylation%20and%20Asf1&amp;journal=Mol%20Cell&amp;doi=10.1016%2Fj.molcel.2007.07.011&amp;volume=27&amp;pages=393-405&amp;publication_year=2007&amp;author=Rufiange%2CA&amp;author=Jacques%2CPE&amp;author=Bhat%2CW&amp;author=Robert%2CF&amp;author=Nourani%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Deal RB, Henikoff S: Capturing the dynamic epigenome. Genome Biol. 2010, 11: 218-10.1186/gb-2010-11-10-218.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVahs7nK" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fgb-2010-11-10-218" aria-label="Article reference 29">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20959022" aria-label="PubMed reference 29" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218653" aria-label="PubMed Central reference 29" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Capturing%20the%20dynamic%20epigenome&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fgb-2010-11-10-218&amp;volume=11&amp;publication_year=2010&amp;author=Deal%2CRB&amp;author=Henikoff%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Deal RB, Henikoff JG, Henikoff S: Genome-wide kinetics of nucleosome turnover determined by metabolic labeling of histones. Science. 2010, 328: 1161-1164. 10.1126/science.1186777.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXmsVGjsb8%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1126%2Fscience.1186777" aria-label="Article reference 30">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20508129" aria-label="PubMed reference 30" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879085" aria-label="PubMed Central reference 30" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20kinetics%20of%20nucleosome%20turnover%20determined%20by%20metabolic%20labeling%20of%20histones&amp;journal=Science&amp;doi=10.1126%2Fscience.1186777&amp;volume=328&amp;pages=1161-1164&amp;publication_year=2010&amp;author=Deal%2CRB&amp;author=Henikoff%2CJG&amp;author=Henikoff%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Jansen LE, Black BE, Foltz DR, Cleveland DW: Propagation of centromeric chromatin requires exit from mitosis. J Cell Biol. 2007, 176: 795-805. 10.1083/jcb.200701066.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXjt1GitL0%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1083%2Fjcb.200701066" aria-label="Article reference 31">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17339380" aria-label="PubMed reference 31" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2064054" aria-label="PubMed Central reference 31" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Propagation%20of%20centromeric%20chromatin%20requires%20exit%20from%20mitosis&amp;journal=J%20Cell%20Biol&amp;doi=10.1083%2Fjcb.200701066&amp;volume=176&amp;pages=795-805&amp;publication_year=2007&amp;author=Jansen%2CLE&amp;author=Black%2CBE&amp;author=Foltz%2CDR&amp;author=Cleveland%2CDW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Ray-Gallet D, Woolfe A, Vassias I, Pellentz C, Lacoste N, Puri A, Schultz DC, Pchelintsev NA, Adams PD, Jansen LE, Almouzni G: Dynamics of histone H3 deposition in vivo reveal a nucleosome gap-filling mechanism for H3.3 to maintain chromatin integrity. Mol Cell. 2011, 44: 928-941. 10.1016/j.molcel.2011.12.006.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1OqsL%2FO" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2011.12.006" aria-label="Article reference 32">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22195966" aria-label="PubMed reference 32" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamics%20of%20histone%20H3%20deposition%20in%20vivo%20reveal%20a%20nucleosome%20gap-filling%20mechanism%20for%20H3.3%20to%20maintain%20chromatin%20integrity&amp;journal=Mol%20Cell&amp;doi=10.1016%2Fj.molcel.2011.12.006&amp;volume=44&amp;pages=928-941&amp;publication_year=2011&amp;author=Ray-Gallet%2CD&amp;author=Woolfe%2CA&amp;author=Vassias%2CI&amp;author=Pellentz%2CC&amp;author=Lacoste%2CN&amp;author=Puri%2CA&amp;author=Schultz%2CDC&amp;author=Pchelintsev%2CNA&amp;author=Adams%2CPD&amp;author=Jansen%2CLE&amp;author=Almouzni%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Kimura H, Cook PR: Kinetics of core histones in living human cells: little exchange of H3 and H4 and some rapid exchange of H2B. J Cell Biol. 2001, 153: 1341-1353. 10.1083/jcb.153.7.1341.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslensL4%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1083%2Fjcb.153.7.1341" aria-label="Article reference 33">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11425866" aria-label="PubMed reference 33" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2150718" aria-label="PubMed Central reference 33" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Kinetics%20of%20core%20histones%20in%20living%20human%20cells%3A%20little%20exchange%20of%20H3%20and%20H4%20and%20some%20rapid%20exchange%20of%20H2B&amp;journal=J%20Cell%20Biol&amp;doi=10.1083%2Fjcb.153.7.1341&amp;volume=153&amp;pages=1341-1353&amp;publication_year=2001&amp;author=Kimura%2CH&amp;author=Cook%2CPR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Meshorer E, Yellajoshula D, George E, Scambler PJ, Brown DT, Misteli T: Hyperdynamic plasticity of chromatin proteins in pluripotent embryonic stem cells. Dev Cell. 2006, 10: 105-116. 10.1016/j.devcel.2005.10.017.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1SqtL4%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.devcel.2005.10.017" aria-label="Article reference 34">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16399082" aria-label="PubMed reference 34" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1868458" aria-label="PubMed Central reference 34" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Hyperdynamic%20plasticity%20of%20chromatin%20proteins%20in%20pluripotent%20embryonic%20stem%20cells&amp;journal=Dev%20Cell&amp;doi=10.1016%2Fj.devcel.2005.10.017&amp;volume=10&amp;pages=105-116&amp;publication_year=2006&amp;author=Meshorer%2CE&amp;author=Yellajoshula%2CD&amp;author=George%2CE&amp;author=Scambler%2CPJ&amp;author=Brown%2CDT&amp;author=Misteli%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Creyghton MP, Cheng AW, Welstead GG, Kooistra T, Carey BW, Steine EJ, Hanna J, Lodato MA, Frampton GM, Sharp PA, Boyer LA, Young RA, Jaenisch R: Histone H3K27ac separates active from poised enhancers and predicts developmental state. Proc Natl Acad Sci U S A. 2010, 107: 21931-21936. 10.1073/pnas.1016071107.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1Wjs7bP" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1016071107" aria-label="Article reference 35">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21106759" aria-label="PubMed reference 35" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003124" aria-label="PubMed Central reference 35" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20H3K27ac%20separates%20active%20from%20poised%20enhancers%20and%20predicts%20developmental%20state&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;doi=10.1073%2Fpnas.1016071107&amp;volume=107&amp;pages=21931-21936&amp;publication_year=2010&amp;author=Creyghton%2CMP&amp;author=Cheng%2CAW&amp;author=Welstead%2CGG&amp;author=Kooistra%2CT&amp;author=Carey%2CBW&amp;author=Steine%2CEJ&amp;author=Hanna%2CJ&amp;author=Lodato%2CMA&amp;author=Frampton%2CGM&amp;author=Sharp%2CPA&amp;author=Boyer%2CLA&amp;author=Young%2CRA&amp;author=Jaenisch%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Rada-Iglesias A, Bajpai R, Swigut T, Brugmann SA, Flynn RA, Wysocka J: A unique chromatin signature uncovers early developmental enhancers in humans. Nature. 2011, 470: 279-283. 10.1038/nature09692.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFKlsr%2FO" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnature09692" aria-label="Article reference 36">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21160473" aria-label="PubMed reference 36" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20unique%20chromatin%20signature%20uncovers%20early%20developmental%20enhancers%20in%20humans&amp;journal=Nature&amp;doi=10.1038%2Fnature09692&amp;volume=470&amp;pages=279-283&amp;publication_year=2011&amp;author=Rada-Iglesias%2CA&amp;author=Bajpai%2CR&amp;author=Swigut%2CT&amp;author=Brugmann%2CSA&amp;author=Flynn%2CRA&amp;author=Wysocka%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Li B, Jackson J, Simon MD, Fleharty B, Gogol M, Seidel C, Workman JL, Shilatifard A: Histone H3 lysine 36 dimethylation (H3K36me2) is sufficient to recruit the Rpd3s histone deacetylase complex and to repress spurious transcription. J Biol Chem. 2009, 284: 7970-7976. 10.1074/jbc.M808220200.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXjtVSmtLw%3D" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1074%2Fjbc.M808220200" aria-label="Article reference 37">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19155214" aria-label="PubMed reference 37" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2658090" aria-label="PubMed Central reference 37" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20H3%20lysine%2036%20dimethylation%20%28H3K36me2%29%20is%20sufficient%20to%20recruit%20the%20Rpd3s%20histone%20deacetylase%20complex%20and%20to%20repress%20spurious%20transcription&amp;journal=J%20Biol%20Chem&amp;doi=10.1074%2Fjbc.M808220200&amp;volume=284&amp;pages=7970-7976&amp;publication_year=2009&amp;author=Li%2CB&amp;author=Jackson%2CJ&amp;author=Simon%2CMD&amp;author=Fleharty%2CB&amp;author=Gogol%2CM&amp;author=Seidel%2CC&amp;author=Workman%2CJL&amp;author=Shilatifard%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Carrozza MJ, Li B, Florens L, Suganuma T, Swanson SK, Lee KK, Shia WJ, Anderson S, Yates J, Washburn MP, Workman JL: Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription. Cell. 2005, 123: 581-592. 10.1016/j.cell.2005.10.023.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1yktrvL" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2005.10.023" aria-label="Article reference 38">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16286007" aria-label="PubMed reference 38" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20H3%20methylation%20by%20Set2%20directs%20deacetylation%20of%20coding%20regions%20by%20Rpd3S%20to%20suppress%20spurious%20intragenic%20transcription&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2005.10.023&amp;volume=123&amp;pages=581-592&amp;publication_year=2005&amp;author=Carrozza%2CMJ&amp;author=Li%2CB&amp;author=Florens%2CL&amp;author=Suganuma%2CT&amp;author=Swanson%2CSK&amp;author=Lee%2CKK&amp;author=Shia%2CWJ&amp;author=Anderson%2CS&amp;author=Yates%2CJ&amp;author=Washburn%2CMP&amp;author=Workman%2CJL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Barski A, Chepelev I, Liko D, Cuddapah S, Fleming AB, Birch J, Cui K, White RJ, Zhao K: Pol II and its associated epigenetic marks are present at Pol III-transcribed noncoding RNA genes. Nat Struct Mol Biol. 2010, 17: 629-634. 10.1038/nsmb.1806.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXltFKlsrY%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnsmb.1806" aria-label="Article reference 39">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20418881" aria-label="PubMed reference 39" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917008" aria-label="PubMed Central reference 39" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Pol%20II%20and%20its%20associated%20epigenetic%20marks%20are%20present%20at%20Pol%20III-transcribed%20noncoding%20RNA%20genes&amp;journal=Nat%20Struct%20Mol%20Biol&amp;doi=10.1038%2Fnsmb.1806&amp;volume=17&amp;pages=629-634&amp;publication_year=2010&amp;author=Barski%2CA&amp;author=Chepelev%2CI&amp;author=Liko%2CD&amp;author=Cuddapah%2CS&amp;author=Fleming%2CAB&amp;author=Birch%2CJ&amp;author=Cui%2CK&amp;author=White%2CRJ&amp;author=Zhao%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Cacchione S, Cerone MA, Savino M: In vitro low propensity to form nucleosomes of four telomeric sequences. FEBS Lett. 1997, 400: 37-41. 10.1016/S0014-5793(96)01318-X.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaK28XnsFSmtL0%3D" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0014-5793%2896%2901318-X" aria-label="Article reference 40">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9000509" aria-label="PubMed reference 40" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vitro%20low%20propensity%20to%20form%20nucleosomes%20of%20four%20telomeric%20sequences&amp;journal=FEBS%20Lett&amp;doi=10.1016%2FS0014-5793%2896%2901318-X&amp;volume=400&amp;pages=37-41&amp;publication_year=1997&amp;author=Cacchione%2CS&amp;author=Cerone%2CMA&amp;author=Savino%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Adkins MW, Tyler JK: Transcriptional activators are dispensable for transcription in the absence of Spt6-mediated chromatin reassembly of promoter regions. Mol Cell. 2006, 21: 405-416. 10.1016/j.molcel.2005.12.010.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xhslehu7k%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2005.12.010" aria-label="Article reference 41">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16455495" aria-label="PubMed reference 41" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20activators%20are%20dispensable%20for%20transcription%20in%20the%20absence%20of%20Spt6-mediated%20chromatin%20reassembly%20of%20promoter%20regions&amp;journal=Mol%20Cell&amp;doi=10.1016%2Fj.molcel.2005.12.010&amp;volume=21&amp;pages=405-416&amp;publication_year=2006&amp;author=Adkins%2CMW&amp;author=Tyler%2CJK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Zaret KS, Caravaca JM, Tulin A, Sekiya T: Nuclear mobility and mitotic chromosome binding: similarities between pioneer transcription factor FoxA and linker histone H1. Cold Spring Harb Symp Quant Biol. 2010, 75: 219-226. 10.1101/sqb.2010.75.061.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XnsVOjt7g%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fsqb.2010.75.061" aria-label="Article reference 42">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21502411" aria-label="PubMed reference 42" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Nuclear%20mobility%20and%20mitotic%20chromosome%20binding%3A%20similarities%20between%20pioneer%20transcription%20factor%20FoxA%20and%20linker%20histone%20H1&amp;journal=Cold%20Spring%20Harb%20Symp%20Quant%20Biol&amp;doi=10.1101%2Fsqb.2010.75.061&amp;volume=75&amp;pages=219-226&amp;publication_year=2010&amp;author=Zaret%2CKS&amp;author=Caravaca%2CJM&amp;author=Tulin%2CA&amp;author=Sekiya%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Ballare C, Castellano G, Gaveglia L, Althammer S, Gonzalez-Vallinas J, Eyras E, Le Dily F, Zaurin R, Soronellas D, Vicent GP, Beato M: Nucleosome-driven transcription factor binding and gene regulation. Mol Cell. 2013, 49: 67-79.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhslWqur7J" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2012.10.019" aria-label="Article reference 43">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23177737" aria-label="PubMed reference 43" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleosome-driven%20transcription%20factor%20binding%20and%20gene%20regulation&amp;journal=Mol%20Cell&amp;volume=49&amp;pages=67-79&amp;publication_year=2013&amp;author=Ballare%2CC&amp;author=Castellano%2CG&amp;author=Gaveglia%2CL&amp;author=Althammer%2CS&amp;author=Gonzalez-Vallinas%2CJ&amp;author=Eyras%2CE&amp;author=Le%20Dily%2CF&amp;author=Zaurin%2CR&amp;author=Soronellas%2CD&amp;author=Vicent%2CGP&amp;author=Beato%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Schwabish MA, Struhl K: The Swi/Snf complex is important for histone eviction during transcriptional activation and RNA polymerase II elongation in vivo. Mol Cell Biol. 2007, 27: 6987-6995. 10.1128/MCB.00717-07.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFKjsb7P" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1128%2FMCB.00717-07" aria-label="Article reference 44">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17709398" aria-label="PubMed reference 44" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2168902" aria-label="PubMed Central reference 44" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Swi%2FSnf%20complex%20is%20important%20for%20histone%20eviction%20during%20transcriptional%20activation%20and%20RNA%20polymerase%20II%20elongation%20in%20vivo&amp;journal=Mol%20Cell%20Biol&amp;doi=10.1128%2FMCB.00717-07&amp;volume=27&amp;pages=6987-6995&amp;publication_year=2007&amp;author=Schwabish%2CMA&amp;author=Struhl%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Teves SS, Henikoff S: Heat shock reduces stalled RNA polymerase II and nucleosome turnover genome-wide. Genes Dev. 2011, 25: 2387-2397. 10.1101/gad.177675.111.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsFCru7zP" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgad.177675.111" aria-label="Article reference 45">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22085965" aria-label="PubMed reference 45" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3222904" aria-label="PubMed Central reference 45" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Heat%20shock%20reduces%20stalled%20RNA%20polymerase%20II%20and%20nucleosome%20turnover%20genome-wide&amp;journal=Genes%20Dev&amp;doi=10.1101%2Fgad.177675.111&amp;volume=25&amp;pages=2387-2397&amp;publication_year=2011&amp;author=Teves%2CSS&amp;author=Henikoff%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Kaplan CD, Laprade L, Winston F: Transcription elongation factors repress transcription initiation from cryptic sites. Science. 2003, 301: 1096-1099. 10.1126/science.1087374.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXmsFWisbo%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1126%2Fscience.1087374" aria-label="Article reference 46">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12934008" aria-label="PubMed reference 46" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20elongation%20factors%20repress%20transcription%20initiation%20from%20cryptic%20sites&amp;journal=Science&amp;doi=10.1126%2Fscience.1087374&amp;volume=301&amp;pages=1096-1099&amp;publication_year=2003&amp;author=Kaplan%2CCD&amp;author=Laprade%2CL&amp;author=Winston%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Kizer KO, Phatnani HP, Shibata Y, Hall H, Greenleaf AL, Strahl BD: A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation. Mol Cell Biol. 2005, 25: 3305-3316. 10.1128/MCB.25.8.3305-3316.2005.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXjtlSjs7k%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1128%2FMCB.25.8.3305-3316.2005" aria-label="Article reference 47">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15798214" aria-label="PubMed reference 47" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1069628" aria-label="PubMed Central reference 47" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20novel%20domain%20in%20Set2%20mediates%20RNA%20polymerase%20II%20interaction%20and%20couples%20histone%20H3%20K36%20methylation%20with%20transcript%20elongation&amp;journal=Mol%20Cell%20Biol&amp;doi=10.1128%2FMCB.25.8.3305-3316.2005&amp;volume=25&amp;pages=3305-3316&amp;publication_year=2005&amp;author=Kizer%2CKO&amp;author=Phatnani%2CHP&amp;author=Shibata%2CY&amp;author=Hall%2CH&amp;author=Greenleaf%2CAL&amp;author=Strahl%2CBD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Shechter D, Dormann HL, Allis CD, Hake SB: Extraction, purification and analysis of histones. Nat Protoc. 2007, 2: 1445-1457. 10.1038/nprot.2007.202.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFagsLnL" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnprot.2007.202" aria-label="Article reference 48">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17545981" aria-label="PubMed reference 48" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Extraction%2C%20purification%20and%20analysis%20of%20histones&amp;journal=Nat%20Protoc&amp;doi=10.1038%2Fnprot.2007.202&amp;volume=2&amp;pages=1445-1457&amp;publication_year=2007&amp;author=Shechter%2CD&amp;author=Dormann%2CHL&amp;author=Allis%2CCD&amp;author=Hake%2CSB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Barski A, Cuddapah S, Cui K, Roh TY, Schones DE, Wang Z, Wei G, Chepelev I, Zhao K: High-resolution profiling of histone methylations in the human genome. Cell. 2007, 129: 823-837. 10.1016/j.cell.2007.05.009.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXmtFKjsro%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2007.05.009" aria-label="Article reference 49">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17512414" aria-label="PubMed reference 49" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20profiling%20of%20histone%20methylations%20in%20the%20human%20genome&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2007.05.009&amp;volume=129&amp;pages=823-837&amp;publication_year=2007&amp;author=Barski%2CA&amp;author=Cuddapah%2CS&amp;author=Cui%2CK&amp;author=Roh%2CTY&amp;author=Schones%2CDE&amp;author=Wang%2CZ&amp;author=Wei%2CG&amp;author=Chepelev%2CI&amp;author=Zhao%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Langmead B, Trapnell C, Pop M, Salzberg SL: Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. 2009, 10: R25-10.1186/gb-2009-10-3-r25.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fgb-2009-10-3-r25" aria-label="Article reference 50">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19261174" aria-label="PubMed reference 50" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2690996" aria-label="PubMed Central reference 50" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fgb-2009-10-3-r25&amp;volume=10&amp;publication_year=2009&amp;author=Langmead%2CB&amp;author=Trapnell%2CC&amp;author=Pop%2CM&amp;author=Salzberg%2CSL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Zang C, Schones DE, Zeng C, Cui K, Zhao K, Peng W: A clustering approach for identification of enriched domains from histone modification ChIP-Seq data. Bioinformatics. 2009, 25: 1952-1958. 10.1093/bioinformatics/btp340.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXovVektbk%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp340" aria-label="Article reference 51">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505939" aria-label="PubMed reference 51" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2732366" aria-label="PubMed Central reference 51" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20clustering%20approach%20for%20identification%20of%20enriched%20domains%20from%20histone%20modification%20ChIP-Seq%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp340&amp;volume=25&amp;pages=1952-1958&amp;publication_year=2009&amp;author=Zang%2CC&amp;author=Schones%2CDE&amp;author=Zeng%2CC&amp;author=Cui%2CK&amp;author=Zhao%2CK&amp;author=Peng%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Jurka J, Kapitonov VV, Pavlicek A, Klonowski P, Kohany O, Walichiewicz J: Repbase Update, a database of eukaryotic repetitive elements. Cytogenet Genome Res. 2005, 110: 462-467. 10.1159/000084979.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1:STN:280:DC%2BD2Mvislertw%3D%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1159%2F000084979" aria-label="Article reference 52">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16093699" aria-label="PubMed reference 52" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;journal=Cytogenet%20Genome%20Res&amp;doi=10.1159%2F000084979&amp;volume=110&amp;pages=462-467&amp;publication_year=2005&amp;author=Jurka%2CJ&amp;author=Kapitonov%2CVV&amp;author=Pavlicek%2CA&amp;author=Klonowski%2CP&amp;author=Kohany%2CO&amp;author=Walichiewicz%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL: TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol. 2013, 14: R36-10.1186/gb-2013-14-4-r36.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fgb-2013-14-4-r36" aria-label="Article reference 53">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23618408" aria-label="PubMed reference 53" rel="nofollow">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053844" aria-label="PubMed Central reference 53" rel="nofollow">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fgb-2013-14-4-r36&amp;volume=14&amp;publication_year=2013&amp;author=Kim%2CD&amp;author=Pertea%2CG&amp;author=Trapnell%2CC&amp;author=Pimentel%2CH&amp;author=Kelley%2CR&amp;author=Salzberg%2CSL">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1186/gb-2013-14-10-r121?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank the NHLBI DNA Sequencing Core facility for sequencing the ChIP-Seq and RNA-Seq libraries. The work is supported by the Division of Intramural Research, NHLBI, NIH, USA.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Systems Biology Center, National Heart, Lung, and Blood Institute, NIH, Bethesda, MD, 20892, USA</p><p class="c-article-author-affiliation__authors-list">Daniel C Kraushaar, Wenfei Jin, Alika Maunakea, Brian Abraham &amp; Keji Zhao</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Samsung Advanced Institute of Technology, Samsung Electronics Corporation, Yongin-Si, Gyeonggi-Do, 446-712, South Korea</p><p class="c-article-author-affiliation__authors-list">Misook Ha</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Daniel_C-Kraushaar"><span class="c-article-authors-search__title u-h3 js-search-name">Daniel C Kraushaar</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23%22Daniel%20C%20Kraushaar%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daniel%20C%20Kraushaar" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daniel%20C%20Kraushaar%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Wenfei-Jin"><span class="c-article-authors-search__title u-h3 js-search-name">Wenfei Jin</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23%22Wenfei%20Jin%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Wenfei%20Jin" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Wenfei%20Jin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Alika-Maunakea"><span class="c-article-authors-search__title u-h3 js-search-name">Alika Maunakea</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23%22Alika%20Maunakea%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alika%20Maunakea" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alika%20Maunakea%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Brian-Abraham"><span class="c-article-authors-search__title u-h3 js-search-name">Brian Abraham</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23%22Brian%20Abraham%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Brian%20Abraham" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Brian%20Abraham%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Misook-Ha"><span class="c-article-authors-search__title u-h3 js-search-name">Misook Ha</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23%22Misook%20Ha%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Misook%20Ha" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Misook%20Ha%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Keji-Zhao"><span class="c-article-authors-search__title u-h3 js-search-name">Keji Zhao</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23%22Keji%20Zhao%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Keji%20Zhao" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Keji%20Zhao%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:zhaok@nhlbi.nih.gov">Keji Zhao</a>.</p></div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><h3 class="c-article__sub-heading">Competing interests</h3><p>The authors declare that they have no competing interests.</p><h3 class="c-article__sub-heading">Authors’ contributions</h3><p>DK participated in the design of the study, carried out experiments and drafted the manuscript. WJ carried out the data analysis and drafted the manuscript. AM participated in the construct generation. AB participated in the data analysis. MH participated in the data analysis. KZ conceived of the study, and participated in its design and coordination and helped to draft the manuscript. All authors read and approved the final manuscript.</p><p>Daniel C Kraushaar, Wenfei Jin contributed equally to this work.</p><p>An Erratum for this chapter can be found at <a href="http://dx.doi.org/10.1186/s13059-016-0886-3">
                    <span class="c-literal">http://dx.doi.org/10.1186/s13059-016-0886-3</span>
                  </a>
</p><p>An erratum to this article is available at <a href="http://dx.doi.org/10.1186/s13059-016-0886-3">http://dx.doi.org/10.1186/s13059-016-0886-3</a>.</p></div></div></section><section data-title="Electronic supplementary material"><div class="c-article-section" id="Sec26-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec26">Electronic supplementary material</h2><div class="c-article-section__content" id="Sec26-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div>
                  
                <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM1_ESM.jpeg" data-supp-info-image="">13059_2013_3533_MOESM1_ESM.jpeg</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Additional file 1: Figure S1: Fast turnover of enhancer H3.3 nucleosomes are associated with higher active histone modification and lower repressive modification. The H3.3-containing nucleosomes at enhancers were separated into two groups (fast turnover and slow turnover) based on the distribution of turnover indices in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/gb-2013-14-10-r121#Fig4">4</a>G. The histone modification levels of the two groups are displayed and compared using box plots. (JPEG 480 KB)</p></div></div></div></div></div></section><section data-title="Authors’ original submitted files for images"><div class="c-article-section" id="Sec27-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec27">Authors’ original submitted files for images</h2><div class="c-article-section__content" id="Sec27-content"><div data-test="supplementary-info"><p>Below are the links to the authors’ original submitted files for images.</p><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM2_ESM.tif" data-supp-info-image="">Authors’ original file for figure 1</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM3_ESM.tif" data-supp-info-image="">Authors’ original file for figure 2</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM4_ESM.tif" data-supp-info-image="">Authors’ original file for figure 3</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM5_ESM.tif" data-supp-info-image="">Authors’ original file for figure 4</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM6_ESM.tif" data-supp-info-image="">Authors’ original file for figure 5</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM7_ESM.tiff" data-supp-info-image="">Authors’ original file for figure 6</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fgb-2013-14-10-r121/MediaObjects/13059_2013_3533_MOESM8_ESM.jpeg" data-supp-info-image="">Authors’ original file for figure 7</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Genome-wide%20incorporation%20dynamics%20reveal%20distinct%20categories%20of%20turnover%20for%20the%20histone%20variant%20H3.3&amp;author=Daniel%20C%20Kraushaar%20et%20al&amp;contentID=10.1186%2Fgb-2013-14-10-r121&amp;copyright=Kraushaar%20et%20al.%3B%20licensee%20BioMed%20Central%20Ltd.&amp;publication=1474-760X&amp;publicationDate=2013-12-10&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Kraushaar, D.C., Jin, W., Maunakea, A. <i>et al.</i> Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3.
                    <i>Genome Biol</i> <b>14, </b>R121 (2013). https://doi.org/10.1186/gb-2013-14-10-r121</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" href="https://citation-needed.springer.com/v2/references/10.1186/gb-2013-14-10-r121?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2013-06-05">05 June 2013</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2013-10-31">31 October 2013</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2013-12-10">10 December 2013</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--doi"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1186/gb-2013-14-10-r121</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><h3 class="c-article__sub-heading">Keywords</h3><ul class="c-article-subject-list"><li class="c-article-subject-list__subject"><span>Transcription Start Site</span></li><li class="c-article-subject-list__subject"><span>Histone Modification</span></li><li class="c-article-subject-list__subject"><span>Gene Body</span></li><li class="c-article-subject-list__subject"><span>Histone Variant</span></li><li class="c-article-subject-list__subject"><span>Nucleosome Occupancy</span></li></ul><div data-component="article-info-list"></div></div></div></div></div></section>

                    


                </article>
            </main>

            <div class="c-article-extras u-text-sm u-hide-print" data-track-component="reading companion">
                <aside>
                    
    <div class="c-pdf-download u-clear-both">
        <a href="//genomebiology.biomedcentral.com/track/pdf/10.1186/gb-2013-14-10-r121.pdf" class="c-pdf-download__link" data-article-pdf="true"  data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external>
            <span>Download PDF</span>
            <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
        </a>
    </div>

                    

                    
    


                    <div class="c-reading-companion">
                        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
                            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                                <div class="js-ad">
    
    <aside class="adsbox c-ad c-ad--300x250 u-mt-16" data-component-mpu>
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-MPU1"
                data-ad-type="MPU1"
                data-pa11y-ignore
                data-gpt
                data-gpt-unitpath="/270604982/bmc/genomebiology/articles"
                data-gpt-sizes="300x250"
                data-gpt-targeting="pos=MPU1;doi=10.1186/gb-2013-14-10-r121;type=article;kwrd=Transcription Start Site,Histone Modification,Gene Body,Histone Variant,Nucleosome Occupancy;pmc=L32030,B12008,L32020,L32010,L15001,L21001;"
            >
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/genomebiology/articles&amp;sz=300x250&amp;pos=MPU1&amp;doi=10.1186/gb-2013-14-10-r121&amp;type=article&amp;kwrd=Transcription Start Site,Histone Modification,Gene Body,Histone Variant,Nucleosome Occupancy&amp;pmc=L32030,B12008,L32020,L32010,L15001,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                            src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/genomebiology/articles&amp;sz=300x250&amp;pos=MPU1&amp;doi=10.1186/gb-2013-14-10-r121&amp;type=article&amp;kwrd=Transcription Start Site,Histone Modification,Gene Body,Histone Variant,Nucleosome Occupancy&amp;pmc=L32030,B12008,L32020,L32010,L15001,L21001&amp;"
                            alt="Advertisement"
                            width="300"
                            height="250">
                    </a>
                </noscript>
            </div>
        </div>
    </aside>
</div>
                            </div>
                            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
                            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
                        </div>
                    </div>
                </aside>
            </div>

        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Genome Biology</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1474-760X</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:editorial@genomebiology.com">editorial@genomebiology.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/gb-2013-14-10-r121' alt=""/>

         
    <footer>
        
            <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between" data-test="publisher-footer-menu">
            <div class="u-display-flex">
                
                    
                        <ul class="c-list-group c-list-group--sm u-mr-24 u-mb-16">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-mr-24 u-mb-16">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-mr-24 u-mb-16">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://support.biomedcentral.com/support/home">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-mb-24">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="u-display-flex u-list-reset" data-test="footer-social-links">
                    
                        <li class="u-mt-8 u-mr-8">
                            <a href="https://twitter.com/biomedcentral"
                               data-track="click"
                               data-track-category="Social"
                               data-track-action="Clicked BMC Twitter"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="u-icon u-text-lg" width="24" height="24" aria-hidden="true" focusable="false">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-mt-8 u-mr-8">
                            <a href="https://www.facebook.com/BioMedCentral"
                               data-track="click"
                               data-track-category="Social"
                               data-track-action="Clicked BMC Facebook"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="u-icon u-text-lg" width="24" height="24" aria-hidden="true" focusable="false">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-mt-8 u-mr-8">
                            <a href="http://www.weibo.com/biomedcentral"
                               data-track="click"
                               data-track-category="Social"
                               data-track-action="Clicked BMC Weibo"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="u-icon u-text-lg" width="24" height="24" aria-hidden="true" focusable="false">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="https://www.springernature.com/ccpa">California Privacy Statement</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" data-cc-action="preferences" href="javascript:void(0);">Manage cookies/Do not sell my data</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        
        <div class="c-corporate-footer">
    <div class="u-container">
        <img src=/static/images/logo-springernature-acb40b85fb.svg class="c-corporate-footer__logo" alt="Springer Nature" itemprop="logo" role="img">
        <p class="c-corporate-footer__legal" data-test="copyright"> &#169; 2022 BioMed Central Ltd unless otherwise stated. Part of
            <a class="c-corporate-footer__link" href="https://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>



        
    <svg class="u-hide hide">
        <symbol id="global-icon-chevron-right" viewBox="0 0 16 16">
            <path d="M7.782 7L5.3 4.518c-.393-.392-.4-1.022-.02-1.403a1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L7.782 9l1.013-.998z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-download" viewBox="0 0 16 16">
            <path d="M2 14c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H3.002A1.006 1.006 0 012 14zM9 2v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L3.115 7.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L7 8.8V2c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-email" viewBox="0 0 18 18">
            <path d="M1.995 2h14.01A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM1 13.994A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3H1.995A1 1 0 001 4.006zM9 11L2 7V5.557l7 4 7-4V7z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-institution" viewBox="0 0 18 18">
            <path d="M14 8a1 1 0 011 1v6h1.5a.5.5 0 01.5.5v.5h.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H1v-.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v6h8V9a1 1 0 011-1zM6 8l2 1v4l-2 1zm6 0v6l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-search" viewBox="0 0 22 22">
            <path fill-rule="evenodd" d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z"/>
        </symbol>
        <symbol id="icon-info" viewBox="0 0 18 18">
            <path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="icon-success" viewBox="0 0 18 18">
            <path d="M9 0a9 9 0 110 18A9 9 0 019 0zm3.486 4.982l-4.718 5.506L5.14 8.465a.991.991 0 00-1.423.133 1.06 1.06 0 00.13 1.463l3.407 2.733a1 1 0 001.387-.133l5.385-6.334a1.06 1.06 0 00-.116-1.464.991.991 0 00-1.424.119z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="icon-chevron-down" viewBox="0 0 16 16">
            <path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/>
        </symbol>
        <symbol id="icon-warning" viewBox="0 0 18 18">
            <path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="icon-plus" viewBox="0 0 16 16">
            <path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="icon-minus" viewBox="0 0 16 16">
            <path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="icon-error" viewBox="0 0 18 18">
            <path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill="currentColor" fill-rule="evenodd"/>
        </symbol>
        <symbol id="icon-springer-arrow-left">
            <path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/>
        </symbol>
        <symbol id="icon-springer-arrow-right">
            <path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/>
        </symbol>
    </svg>

    </body>
</html>

