
<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="50b91d5e-dfec-470d-9f06-51749e301777"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";page:string:Article/Chapter View;article:article:doi\:10.1128/IAI.00552-13;ctype:string:Journal Content;requestedJournal:journal:iai;issue:issue:doi\:10.1128/iai.2013.81.issue-9;website:website:asmj;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text;journal:journal:iai"/>
<link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"></link><meta name="citation_journal_title" content="Infection and Immunity"></meta><meta name="dc.Title" content="Poison Domains Block Transit of Translocated Substrates via the Legionella pneumophila Icm/Dot System"></meta><meta name="dc.Creator" content="Whitney M. Amyot"></meta><meta name="dc.Creator" content="Dennise deJesus"></meta><meta name="dc.Creator" content="Ralph R. Isberg"></meta><meta name="dc.Description" content="ABSTRACT Legionella pneumophila uses the Icm/Dot type 4B secretion system (T4BSS) to deliver translocated protein substrates to the host cell, promoting replication vacuole formation. The conformat..."></meta><meta name="Description" content="ABSTRACT Legionella pneumophila uses the Icm/Dot type 4B secretion system (T4BSS) to deliver translocated protein substrates to the host cell, promoting replication vacuole formation. The conformat..."></meta><meta name="dc.Publisher" content="&#xA;American Society for Microbiology&#xA;1752 N St., N.W., Washington, DC&#xA;"></meta><meta name="dc.Contributor" content="A. J. Bäumler"></meta><meta name="dc.Date" scheme="WTN8601" content="2013-6-24"></meta><meta name="dc.Type" content="research-article"></meta><meta name="dc.Format" content="text/HTML"></meta><meta name="dc.Identifier" scheme="publisher-id" content="00552-13"></meta><meta name="dc.Identifier" scheme="doi" content="10.1128/IAI.00552-13"></meta><meta name="dc.Language" content="EN"></meta><meta name="dc.Coverage" content="1752 N St., N.W., Washington, DC"></meta><meta name="dc.Rights" content="Copyright © 2013, American Society for Microbiology. All Rights Reserved."></meta><!-- ${citation_fulltext_world_readable:10.1128/IAI.00552-13} -->




<link rel="meta" type="application/atom+xml" href="https://doi.org/10.1128%2FIAI.00552-13"></link>
<link rel="meta" type="application/rdf+json" href="https://doi.org/10.1128%2FIAI.00552-13"></link>
<link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1128%2FIAI.00552-13"></link>



<meta charset="UTF-8">
















        <meta property="og:title" content="Poison Domains Block Transit of Translocated Substrates via the Legionella pneumophila Icm/Dot System | Infection and Immunity"/>
    


        <meta property="og:type" content="Article"/>
    


        <meta property="og:url" content="https://journals.asm.org/doi/abs/10.1128/IAI.00552-13"/>
    

        <meta property="og:image" content="https://journals.asm.org/cms/asset/e7f36cfa-c21c-4778-a87f-864ae93eed43/iai.2013.81.issue-9.cover.gif" />
        <meta property="og:image:width" content="113" />
        <meta property="og:image:height" content="150" />
    


        <meta property="og:site_name" content="Infection and Immunity"/>
    


        <meta property="og:description" content="Legionella pneumophila uses the Icm/Dot type 4B secretion system (T4BSS) to deliver translocated protein
substrates to the host cell, promoting replication vacuole formation. The conformational
state of the translocated substrates within the ..."/>
    
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=0"/>
<title>Poison Domains Block Transit of Translocated Substrates via the Legionella pneumophila Icm/Dot System | Infection and Immunity</title>










            <link rel="stylesheet" type="text/css" href="/wro/l0of~article-metrics-phase3.css">
        
<link rel="stylesheet" type="text/css" href="/wro/l0of~product.css">



    
    
                <link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-6742698955b5929bab3a.css" media="all" id="build-style-sheet">
                <script>
                    function isIE() { //won't check for edge browser
                        var ua = navigator.userAgent;
                        /* MSIE used to detect old browsers and Trident used to newer ones*/
                        var is_ie = ua.indexOf("MSIE ") > -1 || ua.indexOf("Trident/") > -1;
          
                        return is_ie; 
                    }
                    
                    if(isIE()) {
                        document.write('<link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-style-fallback-0e2f8eb0a5d072bb5fa3.css" media="all" />')
                        var buildCss = document.getElementById('build-style-sheet');
                        buildCss.parentNode.removeChild(buildCss);
                    }
                </script>
        <link rel="stylesheet" href="/products/asmj/releasedAssets/css/print-6742698955b5929bab3a.css"media="print">
    
        <script>
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    





<meta http-equiv="X-UA-Compatible" content="IE=edge">
<style>
:root{
	--journal-image: url('/pb-assets/images/Homepage Header Images/header-iai-desktop-1602282256940.png');
	--journal-mobile-image: url("/pb-assets/images/Homepage Header Images/header-iai-mobile-1605057095637.png");
	--landing-image: url("/pb-assets/images/Homepage Header Images/header-global-desktop-1603901826210.png");
	--landing-mobile-image: url("/pb-assets/images/Homepage Header Images/header-global-mobile-1605057223667.png");


}
</style>
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/Journals-Atypon-1641917642317.css">
<script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
<script>
	window.googletag = window.googletag || {cmd: []};
	googletag.cmd.push(function() {
    	googletag.defineSlot('/13678947/asm_leader-mob', [[728, 90], [320, 50]], 'div-gpt-ad-1594664599282-0').addService(googletag.pubads());
    	googletag.defineSlot('/13678947/asm_box', [300, 250], 'div-gpt-ad-1594664702378-0').addService(googletag.pubads());	
		googletag.pubads().enableSingleRequest();
		
			
				googletag.pubads().setTargeting('asmjournals', 'iai');
			
		
		googletag.enableServices();
	});
</script>



                    <link rel="canonical" href="https://journals.asm.org/doi/10.1128/IAI.00552-13">
                





    <script src="//scholar.google.com/scholar_js/casa.js" async></script>








    
    

    
    
    
            <script type="text/javascript" src="/wro/l0of~article-metrics-phase2.js"></script>
        


    <!-- Google Tag Manager -->
    













<script>
    var dataLayer = dataLayer ||[];
    
    dataLayer.push({"event":"pageview","page_domain":"journals.asm.org","page_path":"/doi/10.1128/IAI.00552-13","page_name":"Poison Domains Block Transit of Translocated Substrates via the Legionella pneumophila Icm/Dot System | Infection and Immunity","page_type":"article","article_title":"Poison Domains Block Transit of Translocated Substrates via the Legionella pneumophila Icm/Dot System","article_type":"Research Article","article_category":"Cellular Microbiology: Pathogen-Host Cell Molecular Interactions","article_doi":"10.1128/IAI.00552-13","article_access":"free","article_received_date":"02 May 2013","article_accepted_date":"13 June 2013","article_published_online_date":"13 August 2013","article_issue":"volume 81, issue 9","article_issue_date":"September 2013","journal":"Infection and Immunity","user_type":"Guest","basic_groups":["All users group","Unknown Users","All users no webcrawlers"],"smart_groups":["Institution- IP authentication"]});
    
</script>


    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MTLLM76');</script>
    <!-- End Google Tag Manager -->

</head>
<body class="pb-ui">


    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MTLLM76" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->




















<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>





    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">




        
        <div>



        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input type="hidden" name="requestUsername" value="requestUsername" />
                <div class="input-group">
                    <div class="label">
                        <label for="5a26c991-102b-4ded-a542-61c81c149bd5.email">Email*</label>
                    </div>
                    <input id="5a26c991-102b-4ded-a542-61c81c149bd5.email" class="email" type="text" name="email" value="" size="15" />
                </div>
                
                    <div class="username-recaptcha-ajax"></div>
                
                <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" />
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        





<div class="popup registration-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Create a new account</h2>
        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.00552-13" />
            <div class="input-group">
                <div class="label">
                    <label for="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email">Email</label>
                </div>
                <input id="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email" class="email" type="email" name="email" value="" />
            </div>
            <div class="submit">
                <input class="button submit primary" type="submit" value="Register" />
            </div></form>
        <div class="center">
            <a href="/action/showLogin" class="show-login">
                
                Returning user
            </a>
        </div>
    </div>
</div>




        
        



<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="form legacy-form">
            






<h2>Change Password </h2>
<form action="/action/changePassword" method="post"><div class="message error"></div>
    <input type="hidden" name="submit" value="submit" />
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-old">Old Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-old" class="old" type="password" name="old" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
    </div>
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-new">New Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-new" class="pass-hint new" type="password" name="new" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
        
            







    


    


    


<div class="password-strength-indicator" data-min="8" data-max="20" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
        
    </div>
    <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
        </div>
        <div class="success-template hidden">
            <h2>Password Changed Successfully</h2>
            <p>Your password has been changed</p>
        </div>
    </div>
</div>





        
        















<div class="popup login-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        

        <div class="login-form-container ">
            <div class="tab tab--flex">
                <ul class="rlist tab__nav" role="tablist">
                    <li class="active" role="presentation">
                        <a class="active " id="pane-m1x01con" href="#pane-m1x01" aria-controls="pane-m1x01" role="tab"
                           data-toggle="tab" title="login" aria-selected="true" data-simple-tab-id=""
                           aria-expanded="true">
                            <span>Login</span>
                        </a>
                    </li>
                    <li role="presentation">
                        <a id="pane-m1x11con" href="#pane-m1x11" aria-controls="pane-m1x11" role="tab"
                           data-toggle="tab" title="register" aria-selected="false">
                            <span>Register</span>
                        </a>
                    </li>
                </ul>
                <ul class="rlist tab__content">
                    <li class="tab__pane active" id="pane-m1x01" aria-labelledby="pane-m1x01con" role="tabpanel">
                        




























<div class="login-form">
    
    <form action="/action/doLogin" method="post"><input type="hidden" name="id" value="016a54f6-925d-4632-a5f6-22977f0dce11" />
        





        <input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.00552-13" />
    

<input type="hidden" name="loginUri" value="/doi/10.1128/IAI.00552-13" />
        
            <input type="hidden" name="popup" value="true" />
        
        
                

























    
        
    
    
        
    
    
    
        
    

        

        
    <div class="input-group">
        <div class="label ">

            
                    <label for="login">Email*</label>
                
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="login" class="login" type="text" name="login" value="" size="15" placeholder="Enter your email address" /><div class="icon-mail"></div></div>
        
                    <div class="actions">
                        
                    </div>
                
    </div>

                





















    



    
    
    <div class="input-group">
        <div class="label ">
            <label for="password">Password*</label>
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="password" class="password" type="password" name="password" value="" autocomplete="off" placeholder="Enter your password" /><div class="icon-lock"></div></div>
        <span class="password-eye-icon icon-eye hidden"></span>
        
                    <div class="actions reset-password">
                        
                        
                        <i class="icon-lock"></i>
                        <span>
                            Forgot password?
                            Reset it <a href="/action/requestResetPassword" class="link show-request-reset-password uppercase styled-link">here</a>
                        </span>
                    </div>
                
    </div>

            

        

        <div class="remember">
            <label class="checkbox--primary remember-me" for="016a54f6-925d-4632-a5f6-22977f0dce11-remember">
                <input id="016a54f6-925d-4632-a5f6-22977f0dce11-remember" class="cmn-toggle cmn-toggle-round-flat" type="checkbox" name="remember" value="true" />
                <span class="label-txt">Keep me logged in</span>
            </label>
        </div>
        
                <div class="submit-section">
                    <input class="button submit primary" type="submit" name="submit" value="Login" />
                    <span class="required-info">Fields with * are mandatory</span>
                </div>
            
        

























        

        <div class="login-tabs-footer">
            <div class="actions">
                
                
                <span>
                    Don&apos;t have an account? Create one <a href="/action/registration" class="uppercase styled-link">here</a>
                </span>
            </div>
        </div></form>
</div>
                    </li>
                    <li class="tab__pane" id="pane-m1x11" aria-labelledby="pane-m1x11con" role="tabpanel">
                        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.00552-13" />
                            <div class="input-group">
                                <div class="label">
                                    <label for="016a54f6-925d-4632-a5f6-22977f0dce11.email">Email*</label>
                                </div>
                                <div class="iconable__input">
                                    <input id="016a54f6-925d-4632-a5f6-22977f0dce11.email" class="email" type="email" name="email" value="" placeholder="Enter your email address" />
                                    <div class="icon-mail"></div>
                                </div>
                            </div>
                            <div class="submit-section">
                                <input class="button submit primary" type="submit" value="Register" />
                                <span class="required-info">Fields with * are mandatory</span>
                            </div>
                            <div class="login-tabs-footer">
                                <div class="actions">
                                    
                                    
                                    <span>
                                        Already have an account? Login <a href="/action/showLogin" class="uppercase styled-link">here</a>
                                    </span>
                                </div>
                            </div></form>
                    </li>
                </ul>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <a href="#" class="close" tabindex="0"><i class="icon-close_thin"></i></a>
        <div class="form">
            





<p>Can't sign in? Forgot your password?</p>
<p class="sub">Enter your email address below and we will send you the reset instructions</p>
<div class="message error"></div>
<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input type="hidden" name="requestResetPassword" value="true" />
    
    <div class="input-group">
        <div class="label">
            <label for="7660a830-973e-446c-89b4-6e6ce315b3e0.email">Email*</label>
        </div>
        <input id="7660a830-973e-446c-89b4-6e6ce315b3e0.email" class="email" type="text" name="email" value="" size="15" />
    </div>
    
        








    

    
    
    
    

    
            <div class="clearfix"></div>
<div class="label required">
    <label for="captchaCode" class="prompt" style="white-space: nowrap">Retype the code from the picture:</label>
</div>
<link type="text/css" rel="stylesheet" href="/botdetectcaptcha?get=layoutStyleSheet" />

<div class="LBD_CaptchaDiv" id="sampleCaptcha" style="width: 280px !important; height: 50px !important"><!--
   --><div class="LBD_CaptchaImageDiv" id="sampleCaptcha_CaptchaImageDiv" style="width: 250px !important; height: 50px !important"><!--
   --><a target="_blank" href="http://captcha.com/java-captcha-info.html" onclick="sampleCaptcha.OnHelpLinkClick(); return sampleCaptcha.FollowHelpLink;"><img class="LBD_CaptchaImage" id="sampleCaptcha_CaptchaImage" src="/botdetectcaptcha?get=image&amp;c=sampleCaptcha&amp;t=e5e19bf92cf54077b7acc28169417a42" alt="CAPTCHA" /></a></div><!--
   --><div class="LBD_CaptchaIconsDiv" id="sampleCaptcha_CaptchaIconsDiv" style="width: 24px !important;"><!--
   --><a class="LBD_ReloadLink" id="sampleCaptcha_ReloadLink" href="#" onclick="sampleCaptcha.ReloadImage(); this.blur(); return false;" title="Change the CAPTCHA code"> <img class="LBD_ReloadIcon" id="sampleCaptcha_ReloadIcon" src="/botdetectcaptcha?get=ReloadIcon" alt="Change the CAPTCHA code" /></a><!--
   --><a id="sampleCaptcha_SoundLink" class="LBD_SoundLink" href="/botdetectcaptcha?get=sound&amp;c=sampleCaptcha&amp;t=e5e19bf92cf54077b7acc28169417a42" onclick="sampleCaptcha.PlaySound(); this.blur(); return false;" title="Speak the CAPTCHA code"> <img class="LBD_SoundIcon" id="sampleCaptcha_SoundIcon" src="/botdetectcaptcha?get=SoundIcon" alt="Speak the CAPTCHA code" /></a><!--
   --><div class="LBD_Placeholder" id="sampleCaptcha_AudioPlaceholder">&nbsp;</div><!--
   --></div>

<script type="text/javascript" src="/botdetectcaptcha?get=clientScriptInclude"></script>
<script type="text/javascript">
    //<![CDATA[
     BotDetect.Init('sampleCaptcha', 'e5e19bf92cf54077b7acc28169417a42', 'captchaCode', true, true, true, true, 1200, 7200, 0);
    //]]>
    </script>


<script type="text/javascript">
    //<![CDATA[
      try{(function(){var bdrsn = document.createElement('script'); bdrsn.type = 'text/javascript'; bdrsn.async = true; bdrsn.src = document.location.protocol + '//remote.captcha.com/include.js?i=ATABMwExATMBMAJBMhMRP3E_PygHKT8_Pz9AbXw_Yj8i'; var fsn = document.getElementsByTagName('script')[0]; fsn.parentNode.insertBefore(bdrsn, fsn);})();} catch(err){}
    //]]>
    </script>

   <input type="hidden" id="LBD_VCID_sampleCaptcha" name="LBD_VCID_sampleCaptcha" value="e5e19bf92cf54077b7acc28169417a42" />
</div>

<input type="hidden" value="0" id="LBD_BackWorkaround_sampleCaptcha">
<div class="validationDiv">
    <input type="hidden" name="loginBotCaptcha" value="">
    <input id="captchaCode" type="text" name="captchaCode"/>
</div>
        

    

    

    
    <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" /></form>
            
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <p>If the address matches an existing account you will receive an email with instructions to reset your password.</p>
            <div class="center">
                <a href="#" class="cancel">Close</a>
            </div>
        </div>
    </div>
</div>




        
        




<div class="top-drawer verify-phone-drawer hidden">
    <div class="content">
        <div class="form">
            <h2>Verify Phone</h2>
            <form action="/action/verifyPhoneNumber" method="post"><input type="hidden" name="submit" value="submit" />
                <div class="message error"></div>
                <div class="input-group">
                    <div class="label">
                        <label for="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code">Enter the verification code</label>
                    </div>
                    <input id="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code" class="verificationCode" type="text" name="verificationCode" value="" size="15" />
                </div>
                <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <h2>Congrats!</h2>
            <p>Your Phone has been verified</p>
            <div class="center">
                <a href="/action/showPreferences" class="button primary submit">close</a>
            </div>
        </div>
    </div>
</div>





        
        <div id="search-overlay" class="header__search-overlay search-overlay js--hidden"><div class="search-overlay__close"><a href="#" title="Close search panel" data-hide-target-id="search-overlay" class="search-overlay__close__btn"><span>CLOSE </span><i class="icon-close"></i></a></div><div class="container">



        
        <div class="quick-search quick-search--all"><div class="full-width quickSearchFormContainer"><div class="tab"><div class="quick-search__searchbox"><ul role="tablist" class="rlist tab__nav dropblock--tab"><li role="presentation" class="tab__nav__item active"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-controls="pane-0" role="tab" data-toggle="tab" title="This Journal" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" aria-expanded="false" class="tab__nav__item__link">This Journal</a></li><li role="presentation" class="tab__nav__item"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-controls="pane-1" role="tab" data-toggle="tab" title="Anywhere" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" aria-expanded="false" class="tab__nav__item__link">Anywhere</a></li></ul><div class="quick-search__search-items tab__content"><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-0 option-journal"><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Search here within this Journal" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/><input type="hidden" name="SeriesKey" value="iai"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-1 "><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Enter words / phrases / DOI / ISBN / keywords / authors / etc" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div>



        
        <div class="quick-search__links">
    <a href="/search/advanced" class="quick-search__advanced-link">Advanced search</a>
</div>
</div></div></div></div></div>
<div class="search-overlay__autocomplete"><div id="autocompleted-terms"></div></div><div class="search-overlay__suggestions"><div class="search-overlay__suggestions__title">Suggested Terms:</div><ul class="rlist search-overlay__suggestions__list">



        
        <li><a href="/asm-author-center" class="btn btn-secondary">author information</a></li>
<li><a href="/migration-information" class="btn btn-secondary">migration information</a></li>
<li><a href="/publication-fees" class="btn btn-secondary">publication fees</a></li>
<li><a href="/asm-covid-19" class="btn btn-secondary">COVID-19 collection</a></li>
<li><a href="/collection" class="btn btn-secondary">asm journals collections</a></li>
</ul></div></div></div><header class="header header--journal header--journal-page"><div class="header__row-1">



        
        <div class="journals-menu">
	<a data-toggle-target-id="journals-menu-content" href="#" title="Journals Menu" class="journals-menu__trigger js--target-hidden">
		<i class="icon-dehaze"></i>
		<i class="icon-close"></i>
	</a>
	<div id="journals-menu-content" class="journals-menu__content js--hidden" aria-expanded="true">
		<a href="#" data-hide-target-id="journals-menu-content" class="journals-menu__back">
			<i class="icon-arrow_left"></i>
			<span>BACK</span>
		</a>
		<ul class="rlist journals-menu__list">
			<li><a href="/journal/aac"><img src="/pb-assets/images/thumbnails/thumb-aac-1602006917653.jpg" alt="Antimicrobial Agents and Chemotherapy"><span>Antimicrobial Agents and Chemotherapy</span></a></li>
			<li><a href="/journal/aem"><img src="/pb-assets/images/thumbnails/thumb-aem-1602006917677.jpg" alt="Applied and Environmental Microbiology"><span>Applied and Environmental Microbiology</span></a></li>
			<li><a href="/journal/cvi"><img src="/pb-assets/images/thumbnails/thumb-cvi-1602006922190.jpg" alt="Clinical and Vaccine Immunology"><span>Clinical and Vaccine Immunology</span></a></li>
			<li><a href="/journal/cmr"><img src="/pb-assets/images/thumbnails/thumb-cmr-1602006918000.jpg" alt="Clinical Microbiology Reviews"><span>Clinical Microbiology Reviews</span></a></li>
			<li><a href="/journal/ecosalplus"><img src="/pb-assets/images/thumbnails/thumb-ecosalplus-1602678711627.jpg" alt="EcoSal Plus"><span>EcoSal Plus</span></a></li>
			<li><a href="/journal/ec"><img src="/pb-assets/images/thumbnails/thumb-ec-1602006922537.jpg" alt="Eukaryotic Cell"><span>Eukaryotic Cell</span></a></li>
			<li><a href="/journal/iai"><img src="/pb-assets/images/thumbnails/thumb-iai-1602006922193.jpg" alt="Infection and Immunity"><span>Infection and Immunity</span></a></li>
			<li><a href="/journal/jb"><img src="/pb-assets/images/thumbnails/thumb-jb-1602006930407.jpg" alt="Journal of Bacteriology"><span>Journal of Bacteriology</span></a></li>
			<li><a href="/journal/jcm"><img src="/pb-assets/images/thumbnails/thumb-jcm-1602006930870.jpg" alt="Journal of Clinical Microbiology"><span>Journal of Clinical Microbiology</span></a></li>
			<li><a href="/journal/jmbe"><img src="/pb-assets/images/thumbnails/thumb-jmbe-1606273579503.jpg" alt="Journal of Microbiology &amp; Biology Education"><span>Journal of Microbiology &amp; Biology Education</span></a></li>
			<li><a href="/journal/jvi"><img src="/pb-assets/images/thumbnails/thumb-jvi-1602006937397.jpg" alt="Journal of Virology"><span>Journal of Virology</span></a></li>
			<li><a href="/journal/mbio"><img src="/pb-assets/images/thumbnails/thumb-mbio-1602006937397.jpg" alt="mBio"><span>mBio</span></a></li>
			<li><a href="/journal/mmbr"><img src="/pb-assets/images/thumbnails/thumb-mmbr-1602006945363.jpg" alt="Microbiology and Molecular Biology Reviews"><span>Microbiology and Molecular Biology Reviews</span></a></li>
			<li><a href="/journal/mra"><img src="/pb-assets/images/thumbnails/thumb-mra-1602006946427.jpg" alt="Microbiology Resource Announcements"><span>Microbiology Resource Announcements</span></a></li>
			<li><a href="/journal/spectrum"><img src="/pb-assets/images/thumbnails/thumb-spectrum-1602006955410.jpg" alt="Microbiology Spectrum"><span>Microbiology Spectrum</span></a></li>
			<li><a href="/journal/mcb"><img src="/pb-assets/images/thumbnails/thumb-mcb-1602006937413.jpg" alt="Molecular and Cellular Biology"><span>Molecular and Cellular Biology</span></a></li>
			<li><a href="/journal/msphere"><img src="/pb-assets/images/thumbnails/thumb-msphere-1602006946427.jpg" alt="mSphere"><span>mSphere</span></a></li>
			<li><a href="/journal/msystems"><img src="/pb-assets/images/thumbnails/thumb-msystems-1602006955407.jpg" alt="mSystems"><span>mSystems</span></a></li>
		</ul>
	</div>
</div>
<div class="header__logos">



        
        <a href="https://asm.org/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div data-truncate="1" class="header__feed mx-3 js--truncate"></div><div class="header__attribution"></div><div class="header__cart">



        
        















<div class="eCommerceCartIndicatorWidget">
    
            <a href="/action/showCart?FlowID=1&amp;aria-label=Shopping+Cart" id="cartLabel" class="quick-menu__item" aria-label="Shopping Cart">
                <div class="cartLabel" title="Cart">
                    <i class="icon-shopping_cart" aria-hidden="true"></i>
                    <span class="filter-count filter-count-bubble shopping-cart hidden"
                          data-id="cart-size">0</span>
                        
                </div>
            </a>
        
</div>
</div><div class="header__login">



        
        




    








        




    













<div class="user-login-bar">
    <div title="Sign in" class="loginBar user-login-bar">
        <div class="user-login-bar">
            <div class="loginBar">
                <div class="loginBar__login">
                    <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FIAI.00552-13&amp;data-toggle-target-id=loginOptions" class="btn btn-primary btn--light btn--darker-content btn--borderless" redirectUri="/doi/10.1128/IAI.00552-13" data-toggle-target-id="loginOptions">
                        Login
                    </a>
                    <div id="loginOptions" class="loginBar__login__options">
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FIAI.00552-13">
                            
                            <img src="/specs/products/asmj/releasedAssets/images/logo-textless.svg" class="mr-2">
                            <span>ASM Member Login</span>
                            <i class="icon-open_in_new" aria-hidden="true"></i>
                        </a>
                        <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FIAI.00552-13">
                            <i class="ml-1 mr-2 icon-account_circle"></i>
                            Non-Member Login
                        </a>
                        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1128%2FIAI.00552-13">
                            <i class="ml-1 mr-2 icon-institution"></i>
                            Institutional Login
                        </a>
                        <a href="/action/registration?redirectUri=%2Fdoi%2F10.1128%2FIAI.00552-13" class="btn btn-secondary btn--inverse btn--borderless btn--darker-content btn-register" title="Register">
                            Register
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>



    

</div>



        
        <a href="#" data-show-target-id="search-overlay" class="hidden-md hidden-sm hidden-xs search-trigger js--target-shown"><i title="Open Search Panel" class="icon-search"></i></a>
</div><div class="header__row-2"><a data-toggle-target-id="responsive-menu" href="#" title="Journals Menu" class="header__mobile-trigger"><i class="icon-dehaze"></i><i class="icon-close"></i></a><div class="header__logos">



        
        <a href="/journal/iai" title="Infection and Immunity logo"><img id="" alt="Infection and Immunity logo" src="/pb-assets/images/pub-nav/IAI-logo-1601672963217.png"/></a>
</div><div class="hidden-sm hidden-xs hidden-md">



        
        <a href="#afterNav-ha0" class="sr-only-focusable">Skip main navigation</a><nav class="main-nav"><ul id="menubar" role="menubar" class="menubar rlist--inline"><li role="menuitem" aria-label="JOURNAL HOME" id="menu-item-pub_nav-0" class="menu-item is--current"><a href="/journal/iai"><span>JOURNAL HOME</span></a></li><li role="menuitem" aria-haspopup="true" aria-label="ARTICLES" id="menu-item-pub_nav-1" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-1" data-toggle="dropdown" class="dropdown__toggle"><span>ARTICLES</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-1" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="CURRENT ISSUE" class="menu-item is--current"><a href="/toc/iai/current"><span>CURRENT ISSUE</span></a></li><li role="menuitem" tabindex="-1" aria-label="ACCEPTED MANUSCRIPTS" class="menu-item is--current"><a href="/toc/iai/0/ja"><span>ACCEPTED MANUSCRIPTS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="COLLECTIONS" class="menu-item is--current"><a href="/journal/iai/collections"><span>COLLECTIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARCHIVE" class="menu-item is--current"><a href="/loi/iai"><span>ARCHIVE</span></a></li></ul></li><li role="menuitem" id="menu-item-pub_nav-2" class="menu-item">



        
        <li role="menuitem" aria-haspopup="true" aria-label="FOR AUTHORS" id="menu-item-pub_nav-custom" class="dropdown menu-parent is--current">
	<a href="#" id="pub_nav-custom" data-toggle="dropdown" class="dropdown__toggle"><span>FOR AUTHORS</span></a>
	<ul aria-labelledby="pub_nav-custom" role="menu" class="rlist dropdown__menu">
		<li role="menuitem" tabindex="-1" aria-label="SUBMITTING" class="menu-item is--current">
			<a href="/submitting?journal=iai"><span>SUBMITTING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="WRITING YOUR PAPER" class="menu-item is--current">
			<a href="/writing-your-paper?journal=iai"><span>WRITING YOUR PAPER</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="TRANSFERRING" class="menu-item is--current">
			<a href="/transferring?journal=iai"><span>TRANSFERRING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="ACCEPTED PAPERS" class="menu-item is--current">
			<a href="/accepted-papers?journal=iai"><span>ACCEPTED PAPERS</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="EDITORIAL POLICIES" class="menu-item is--current">
			<a href="/editorial-policies?journal=iai"><span>EDITORIAL POLICIES</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="PUBLISHING ETHICS" class="menu-item is--current">
			<a href="/publishing-ethics?journal=iai"><span>PUBLISHING ETHICS</span></a>
		</li>
	</ul>
</li>

</li><li role="menuitem" aria-haspopup="true" aria-label="ABOUT THE JOURNAL" id="menu-item-pub_nav-3" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-2" data-toggle="dropdown" class="dropdown__toggle"><span>ABOUT THE JOURNAL</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-2" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="ABOUT IAI" class="menu-item is--current"><a href="/journal/iai/about"><span>ABOUT IAI</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMIT" class="menu-item is--current"><a href="/journal/iai/submit"><span>SUBMIT</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="SCOPE" class="menu-item is--current"><a href="/journal/iai/scope"><span>SCOPE</span></a></li><li role="menuitem" tabindex="-1" aria-label="EDITORIAL BOARD" class="menu-item is--current"><a href="/journal/iai/board-editors"><span>EDITORIAL BOARD</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARTICLE TYPES" class="menu-item is--current"><a href="/journal/iai/article-types"><span>ARTICLE TYPES</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMISSION AND REVIEW PROCESS" class="menu-item is--current"><a href="/journal/iai/submission-review-process"><span>SUBMISSION AND REVIEW PROCESS</span></a></li><li role="menuitem" tabindex="-1" aria-label="EBM SUBMISSIONS" class="menu-item is--current"><a href="/journal/iai/ebm-submission"><span>EBM SUBMISSIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FOR REVIEWERS" class="menu-item is--current"><a href="/reviewers"><span>FOR REVIEWERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FAQ" class="menu-item is--current"><a href="/journal/iai/faq"><span>FAQ</span></a></li><li role="menuitem" tabindex="-1" aria-label="CONTACT US" class="menu-item is--current"><a href="/journal/iai/contact-us"><span>CONTACT US</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="SUBSCRIBE" id="menu-item-pub_nav-4" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-3" data-toggle="dropdown" class="dropdown__toggle"><span>SUBSCRIBE</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-3" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="MEMBERS" class="menu-item is--current"><a href="/members"><span>MEMBERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="INSTITUTIONS" class="menu-item is--current"><a href="/institutions"><span>INSTITUTIONS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="ALERTS" class="menu-item is--current"><a href="/action/showPreferences?menuTab=Alerts"><span>ALERTS</span></a></li><li role="menuitem" tabindex="-1" aria-label="RSS" class="menu-item is--current"><a href="/asm-journals-rss-feeds"><span>RSS</span></a></li></ul></li></ul><span id="afterNav-ha0" tabindex="-1"></span></nav>
</div><a href="#" data-show-target-id="search-overlay" class="hidden-lg search-trigger"><i title="Open Search Panel" class="icon-search"></i></a></div></header>




        
        <main class="content article-page">



        
        <div>









        <div data-widget-def="ux3-layout-widget" data-widget-id="62435c6d-56e5-45d8-8726-1c5a8298a1a6" class="article-container">
        



        
        <div class="mb-3">



        
        <ol aria-label="Breadcrumb" role="navigation" vocab="https://schema.org/" typeof="BreadcrumbList" class="breadcrumbs rlist--inline"><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="https://journals.asm.org/"><span property="name">ASM Journals</span></a><meta property="position" content="1"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/journal/iai"><span property="name">Infection and Immunity</span></a><meta property="position" content="2"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/toc/iai/81/9"><span property="name">Vol. 81, No. 9</span></a><meta property="position" content="3"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/doi/10.1128/IAI.00552-13" aria-current="page" class="breadcrumbs__current"><span property="name">Poison Domains Block Transit of Translocated Substrates via the <named-content content-type='genus-species' xlink:type='simple'>Legionella pneumophila</named-content> Icm/Dot System</span></a><meta property="position" content="4"/></li></ol>










        <div data-widget-def="general-html-asset" data-widget-id="b28fb475-91bf-4257-b86f-8120484c7481" class="container--center-content">
        



        
        <!-- /13678947/asm_leader-mob -->
<div class="leaderboard-wrapper">
<span class="leaderboard-label">Advertisement</span>
<div id='div-gpt-ad-1594664599282-0'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664599282-0'); });
  </script>
</div>
</div>

        </div>
    
</div>

        </div>
    




        
        <link rel="stylesheet" type="text/css" href="/specs/products/asmj/releasedAssets/css/build-article.css"/><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle"><section id="frontmatter" data-extent="frontmatter"><header><div class="core-container"><div data-article-access="free" data-article-access-type="limited" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__type">Research Article</div><div class="meta-panel__publication-data">13 August 2013</div></div><div class="meta-panel__right-content"><div class="meta-panel__share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share__block share__inline-links"><span class="sr-only">Share on</span><ul class="rlist--inline addthis addthis_toolbox addthis_default_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper  icon-facebook"></i></a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i></a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper  icon-linkedin"></i></a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper  icon-mail"></i></a></li></ul></div>
</div></div></div><h1 property="name">Poison Domains Block Transit of Translocated Substrates via the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> Icm/Dot System</h1><div class="core-relations my-3"><div class="relation mb-3"><div class="relation--head"><i class="icon-info"></i><div>This article has been corrected.</div><a href="/doi/10.1128/IAI.02356-14">VIEW CORRECTION</a></div></div></div><div class="contributors"><span class="authors"><span class="heading">Authors</span>: <span role="list"><span property="author" typeof="Person" role="listitem"><a href="#con1"><span property="givenName">Whitney M.</span> <span property="familyName">Amyot</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con2"><span property="givenName">Dennise</span> <span property="familyName">deJesus</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con3"><span property="givenName">Ralph R.</span> <span property="familyName">Isberg</span></a></span></span></span></div><div class="self-citation"><span class="heading">DOI</span>: <a href="https://doi.org/10.1128/IAI.00552-13" property="sameAs">https://doi.org/10.1128/IAI.00552-13</a><span>&nbsp;&#8226;&nbsp;</span><span class="crossmark"><a data-target="crossmark" href="#" data-doi="10.1128/IAI.00552-13" class="crossmark__link"><img alt="Check for updates on crossmark" src="/specs/products/asmj/releasedAssets/images/cross-mark.png" class="crossmark__logo"/></a></span></div><div class="info-panel"><div class="info-panel__metrics info-panel__item">



        
        







        <div class="toolbar-metric-container">
            <input class="toolbar-metric__data" type="hidden" name="json" value='{"data":{"menuSections":[{"total":"116","id":"downloads","title":"downloads"},{"total":"14","id":"citations","title":"citations"}],"doi":"10.1128/IAI.00552-13"},"thisWidget":{"properties":{"identifier":"","headingLvl":"h2","wrapperClass":"","citationMetricId":"2","downloadMetricId":"1","headingClass":"","ux3/template":"","wrapperTag":"div","title":"","linkToFullPage":""}}}'>
        </div>
    

</div><div class="info-panel__favorite info-panel__item">



        
        <div class="article-tools"><a href="/action/addCitationAlert?doi=10.1128%2FIAI.00552-13" target="_blank" title="Track Citations" class="article-tools__citation btn btn--slim btn-secondary btn--inverse btn--transparent"><i class="icon-alert-bell"></i></a></div>
</div><div class="info-panel__alerts info-panel__item"></div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" title="View Citations" class="btn btn--slim btn--inverse"><i aria-hidden="true" class="icon-format_quote"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://journals.asm.org/doi/full/10.1128/IAI.00552-13" class="btn btn--slim btn--inverse"><span>Full Text</span></a><a href="/doi/reader/10.1128/IAI.00552-13" class="btn btn--slim btn--pdf"><span>PDF/EPUB</span></a></div></div></div></header><nav data-core-nav="article"><header><div class="core-self-citation">



        
        <a href="/journal/iai" class="sections-navbar__title">IAI</a>
<div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">81</span></span><span class="delimiter">, </span><span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">9</span></span><br /><span property="datePublished">September 2013</span></div></div><nav></nav></header><ul xmlns="http://www.w3.org/1999/xhtml"><li><a href="#abstract">ABSTRACT</a></li><li><a href="#sec-1">INTRODUCTION</a></li><li><a href="#sec-2">MATERIALS AND METHODS</a></li><li><a href="#sec-3">RESULTS</a></li><li><a href="#sec-4">DISCUSSION</a></li><li><a href="#acknowledgments">ACKNOWLEDGMENTS</a></li><li><a href="#bibliography">REFERENCES</a></li></ul><footer></footer></nav><div id="abstracts"><div class="core-container"><section id="abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">ABSTRACT</h2><div role="paragraph"><span class="named-content" data-type="genus-species">Legionella pneumophila</span> uses the Icm/Dot type 4B secretion system (T4BSS) to deliver translocated protein substrates to the host cell, promoting replication vacuole formation. The conformational state of the translocated substrates within the bacterial cell is unknown, so we sought to determine if folded substrates could be translocated via this system. Fusions of <span class="named-content" data-type="genus-species">L. pneumophila</span> Icm/Dot-translocated substrates (IDTS) to dihydrofolate reductase (DHFR) or ubiquitin (Ub), small proteins known to fold rapidly, resulted in proteins with low translocation efficiencies. The folded moieties did not cause increased aggregation of the IDTS and did not impede interaction with the adaptor protein complex IcmS/IcmW, which is thought to form a soluble complex that promotes translocation. The translocation defect was alleviated with a Ub moiety harboring mutations known to destabilize its structure, indicating that unfolded proteins are preferred substrates. Real-time analysis of translocation, following movement during the first 30 min after bacterial contact with host cells, revealed that the folded moiety caused a kinetic defect in IDTS translocation. Expression of an IDTS fused to a folded moiety interfered with the translocation of other IDTS, consistent with it causing a blockage of the translocation channel. Furthermore, the folded protein fusions also interfered with intracellular growth, consistent with inefficient or impaired translocation of proteins critical for <span class="named-content" data-type="genus-species">L. pneumophila</span> intracellular growth. These studies indicate that substrates of the Icm/Dot T4SS are translocated to the host cytosol in an unfolded conformation and that folded proteins are stalled within the translocation channel, impairing the function of the secretion system.</div></section></div></div></section><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1" data-type="intro"><h2>INTRODUCTION</h2><div role="paragraph"><span class="named-content" data-type="genus-species">Legionella pneumophila</span> is a pathogen of amoebae that grows in an intravacuolar niche in these cells (<a href="#B1" role="doc-biblioref" data-xml-rid="B1">1</a><a href="#B2" role="doc-biblioref" data-xml-rid="B2 B3">–</a><a href="#B4" role="doc-biblioref" data-xml-rid="B4">4</a>). Upon aerosolization of contaminated water supplies, inhalation of <span class="named-content" data-type="genus-species">L. pneumophila</span> and subsequent internalization by alveolar macrophages lead to disease (<a href="#B2" role="doc-biblioref" data-xml-rid="B2">2</a>, <a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>, <a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>). The bacterium replicates within a membrane-bound vacuole in macrophages that is identical to the replication compartment found in amoebae (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>). Formation of the <span class="named-content" data-type="genus-species">Legionella</span>-containing vacuole (LCV) requires a functional type IV secretion system (T4SS) known as the Icm/Dot complex (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>, <a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>), which transports a massive collection of effectors into the host cytosol. To date, over 275 of these protein substrates have been identified (<a href="#B10" role="doc-biblioref" data-xml-rid="B10">10</a><a href="#B11" role="doc-biblioref" data-xml-rid="B11 B12 B13 B14 B15 B16 B17 B18">–</a><a href="#B19" role="doc-biblioref" data-xml-rid="B19">19</a>). Shortly after internalization of the bacteria into either macrophages or amoebae, mitochondria and markers of the endoplasmic reticulum are recruited to the LCV to establish a vacuole that allows replication before the cell lyses, releasing bacteria into the extracellular milieu for a second round of infection (<a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a><a href="#B21" role="doc-biblioref" data-xml-rid="B21">–</a><a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a>).</div><div role="paragraph">Type IV secretion systems (T4SS) are utilized by a wide variety of Gram-negative and Gram-positive bacteria to translocate effector molecules across the bacterial membrane. This diverse group of secretion complexes is defined by ancestral relatedness to bacterial conjugation machines that are able to translocate nucleic acids, proteins, and nucleoprotein complexes (<a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a><a href="#B24" role="doc-biblioref" data-xml-rid="B24 B25">–</a><a href="#B26" role="doc-biblioref" data-xml-rid="B26">26</a>). T4SS can be broadly classified into two subgroups based on similarity to the archetypal <span class="named-content" data-type="genus-species">Agrobacterium tumefaciens</span> VirB/VirD4 (VirB/D4) complex (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>). Type IVA systems (T4ASS) share significant similarity to the VirB/D4 system and can be found in pathogens such as <span class="named-content" data-type="genus-species">Helicobacter pylori</span> and <span class="named-content" data-type="genus-species">Brucella abortus</span> (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>, <a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>). The type IVB systems (T4BSS) are distinct from the VirB/D4 system and are best represented by <span class="named-content" data-type="genus-species">Legionella</span> species and <span class="named-content" data-type="genus-species">Coxiella burnetii</span> (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>, <a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>, <a href="#B27" role="doc-biblioref" data-xml-rid="B27">27</a>, <a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>). Although sequence similarities may be limited between the T4ASS and T4BSS members, there is evidence for functional conservation between these two subclasses as both appear to have core protein components that form a complex spanning the inner and outer membranes of Gram-negative organisms (<a href="#B29" role="doc-biblioref" data-xml-rid="B29">29</a><a href="#B30" role="doc-biblioref" data-xml-rid="B30">–</a><a href="#B31" role="doc-biblioref" data-xml-rid="B31">31</a>). In addition, each has a coupling ATPase complex, which is thought to act as a receptor for translocated substrates, transferring them to other components of the secretion system as a first step in transit to target cells (<a href="#B32" role="doc-biblioref" data-xml-rid="B32">32</a>).</div><div role="paragraph">In <span class="named-content" data-type="genus-species">L. pneumophila</span>, the earliest steps in the translocation process involve interactions between the Icm/Dot-translocated substrate (IDTS) and the IcmS/IcmW chaperone (<a href="#B33" role="doc-biblioref" data-xml-rid="B33">33</a>, <a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>), which is a soluble heteromeric component of the (IcmS/W) apparatus. As IcmS/W associates with the coupling ATPase complex DotL/DotM/DotN, it is likely that this five-member multiprotein complex presents translocated substrates to core secretion system components that span the bacterial envelope. Inactivating mutations in IcmS or IcmW result in highly reduced efficiencies of translocation of most IDTSs although a subset appears to translocate into host cells independently of this chaperone. It is likely that structural determinants control the dependence on IcmS/W for translocation of a protein and that IDTSs are capable of interacting directly with the secretion system in the absence of the chaperone. The critical components of the translocated substrates that allow Icm/Dot recognition are still somewhat obscure although it is clear that each IDTS contains a carboxyl-terminal signal of less than 30 amino acids that targets proteins into the secretion complex (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>, <a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>). Both short polar amino acids and a glutamate-rich region appear to be common sequence determinants for this signal (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>, <a href="#B19" role="doc-biblioref" data-xml-rid="B19">19</a>, <a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>). The proteins that recognize this signal and target the protein to the complex are not known.</div><div role="paragraph">In many secretion systems, there are determinants other than the recognition signal that are required for successful movement through the membrane. In <span class="named-content" data-type="genus-species">Escherichia coli</span>, chimeric proteins that fused secretion signal sequences onto normally cytoplasmic proteins resulted in “jamming” of the secretion apparatus, presumably by entry of an impassable substrate into the secretion channel (<a href="#B36" role="doc-biblioref" data-xml-rid="B36">36</a>). Similarly, translocation of proteins from the cytosol into mitochondria, chloroplasts, and lysosomes can all be blocked by introduction of tightly folded protein moieties into coding sequences, consistent with there being a secretion-competent conformation required for translocation (<a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a><a href="#B38" role="doc-biblioref" data-xml-rid="B38">–</a><a href="#B39" role="doc-biblioref" data-xml-rid="B39">39</a>). The necessity of a secretion-competent conformation, however, does not appear evident in many systems, such as the bacterial twin-arginine translocation (Tat) pathway and the eukaryotic peroxisomal import pathway, in which the insertion of folded domains has no effect on translocation (<a href="#B40" role="doc-biblioref" data-xml-rid="B40">40</a><a href="#B41" role="doc-biblioref" data-xml-rid="B41">–</a><a href="#B42" role="doc-biblioref" data-xml-rid="B42">42</a>).</div><div role="paragraph">Specialized bacterial secretion systems that deliver proteins to target cells provide a very special problem for maintaining a secretion-competent conformation. The translocated substrates of these systems are often synthesized and stored prior to contact of bacteria with their secretion targets, and a large bolus of protein must be injected via these systems quickly. In spite of these issues, there is evidence that a tightly folded moiety interferes with translocation of type III secretion substrates, either because it is rejected by the complex or jams the apparatus (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B44" role="doc-biblioref" data-xml-rid="B44">44</a>). There is also some evidence that in the case of some proteins, the secretion apparatus itself can unfold a substrate, converting a nontranslocatable form into a secretion-competent conformation (<a href="#B45" role="doc-biblioref" data-xml-rid="B45">45</a>, <a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>). In contrast, little is known regarding the tertiary structure of T4SS substrates, and structural analysis of the T4ASS core complex does not necessarily demand that substrates be unfolded prior to entering into the channel (<a href="#B30" role="doc-biblioref" data-xml-rid="B30">30</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). In this study, we investigate whether the <span class="named-content" data-type="genus-species">L. pneumophila</span> Icm/Dot system can tolerate substrates containing domains known to poison secretion by other translocation apparatuses. The results obtained are consistent with the presence of conformational states that interfere with efficient translocation.</div></section><section id="sec-2" data-type="materials methods"><h2>MATERIALS AND METHODS</h2><section id="sec-2-1"><h3>Bacterial strains, plasmids, and cell culture.</h3><div role="paragraph">All bacterial strains and plasmids used in this work are listed in <a href="#T1">Table 1</a>. All PCR primers used in this work are available from the authors. Growth medium for <span class="named-content" data-type="genus-species">L. pneumophila</span> was as described previously (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>, <a href="#B48" role="doc-biblioref" data-xml-rid="B48">48</a>). For <span class="named-content" data-type="genus-species">L. pneumophila</span>, chloramphenicol was used at a concentration of 20 μg/ml. For <span class="named-content" data-type="genus-species">E. coli</span> strains, antibiotics were used at the following concentrations: kanamycin, 40 μg/ml; ampicillin, 100 μg/ml; and chloramphenicol, 30 μg/ml.</div><div class="figure-wrap"><header><div class="label">Table 1</div></header><figure id="T1" class="table"><figcaption><span class="heading">Table 1</span> Strains and plasmids used in this study</figcaption><div class="table-wrap"><table><thead><tr data-xml-align="left" data-xml-valign="bottom"><th>Strain or plasmid</th><th>Genotype or relevant characteristic(s)<sup><i><a href="#TF1-1" role="doc-noteref">a</a></i></sup></th><th>Oligonucleotide(s)</th><th>Reference or source</th></tr></thead><tbody><tr data-xml-valign="top"><td data-xml-align="left">Strains</td><td> </td><td> </td><td> </td></tr><tr data-xml-valign="top"><td data-xml-align="left">    <span class="named-content" data-type="genus-species">L. pneumophila</span></td><td> </td><td> </td><td> </td></tr><tr data-xml-valign="top"><td data-xml-align="left">        Lp01</td><td data-xml-align="left">Philadelphia-1 <i>rpsL hsdR</i></td><td> </td><td data-xml-align="left"><a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a></td></tr><tr data-xml-valign="top"><td data-xml-align="left">        Lp02</td><td data-xml-align="left">Philadelphia-1 <i>rpsL hsdR thyA</i> mutant</td><td> </td><td data-xml-align="left"><a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a></td></tr><tr data-xml-valign="top"><td data-xml-align="left">        Lp03</td><td data-xml-align="left">Lp02 <i>dotA03</i></td><td> </td><td data-xml-align="left"><a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a></td></tr><tr data-xml-valign="top"><td data-xml-align="left">        Lp02 Δ<i>icmS</i></td><td> </td><td> </td><td data-xml-align="left"><a href="#B62" role="doc-biblioref" data-xml-rid="B62">62</a></td></tr><tr data-xml-valign="top"><td data-xml-align="left">    <span class="named-content" data-type="genus-species">E. coli</span></td><td> </td><td> </td><td> </td></tr><tr data-xml-valign="top"><td data-xml-align="left">        DH5α</td><td data-xml-align="left">λ<sup>−</sup> ϕ80d<i>lacZ</i>ΔM15 Δ(<i>lacZYA-argF</i>)<i>U169 recA1 endA1 hsdR17</i>(r<sub>K</sub><sup>−</sup> m<sub>K</sub><sup>−</sup>) <i>supE44 thi-1 gyrA relA1</i></td><td> </td><td> </td></tr><tr data-xml-valign="top"><td data-xml-align="left">Plasmids</td><td> </td><td> </td><td> </td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pJB908</td><td data-xml-align="left">pMMB66EH <i>oriRSF1010 oriT bla tdΔi</i></td><td> </td><td data-xml-align="left"><a href="#B66" role="doc-biblioref" data-xml-rid="B66">66</a></td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pJB2581</td><td data-xml-align="left"><i>cyaA</i> in pJB908-<i>tdΔi</i>, Cm<sup>r</sup></td><td> </td><td data-xml-align="left"><a href="#B49" role="doc-biblioref" data-xml-rid="B49">49</a></td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR</td><td data-xml-align="left">pJB2581 <i>dhfr<sup>+</sup></i></td><td data-xml-align="left">35, 36</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-MavU</td><td data-xml-align="left">pJB2581 <i>mavU</i></td><td data-xml-align="left">63, 58</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-MavU</td><td data-xml-align="left">pCyaA <i>dhfr mavU</i></td><td data-xml-align="left">63, 64, 65, 58</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-MavU(C50)</td><td data-xml-align="left">pJB2581 <i>mavU</i>(348-end)</td><td data-xml-align="left">44, 45</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-MavU(C50)</td><td data-xml-align="left">pCyaA <i>dhfr mavU</i>(348-end)</td><td data-xml-align="left">44, 45</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-MavU(C186)</td><td data-xml-align="left">pJB2581 <i>mavU</i>(212-end)</td><td> </td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-MavU(C186)</td><td data-xml-align="left">pCyaA <i>dhfr mavU</i>(212-end)</td><td data-xml-align="left">63, 58, 71, 72</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-MavU(C118)</td><td data-xml-align="left">pJB2581 <i>mavU</i>(280-end)</td><td> </td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-MavU(C118)</td><td data-xml-align="left">pCyaA <i>dhfr mavU</i>(280-end)</td><td data-xml-align="left">63, 58, 73, 74</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-MavU(C78)</td><td data-xml-align="left">pJB2581 <i>mavU</i>(320-end)</td><td> </td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-Mavu(C78)</td><td data-xml-align="left">pCyaA <i>dhfr mavU</i>(320-end)</td><td data-xml-align="left">64, 58, 75, 76</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pMavU</td><td data-xml-align="left">pJB908 <i>mavU</i></td><td data-xml-align="left">99, 58</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-SidG</td><td data-xml-align="left">pJB2581 <i>sidG</i></td><td data-xml-align="left">60, 54b</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-SidG</td><td data-xml-align="left">pCyaA <i>dhfr sidG</i></td><td data-xml-align="left">60, 54b</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pJB-CyaA-RalF</td><td data-xml-align="left">pJB2581 <i>ralF</i></td><td data-xml-align="left">66, 67</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-RalF</td><td data-xml-align="left">pCyaA <i>dhfr ralF</i></td><td data-xml-align="left">66, 67</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-LidA</td><td data-xml-align="left">pJB2581 <i>lidA</i></td><td data-xml-align="left">25, 31</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-LidA</td><td data-xml-align="left">pCyaA <i>dhfr lidA</i></td><td data-xml-align="left">25, 31, 17, 18, 20, 21</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-SidC</td><td data-xml-align="left">pJB2581 <i>sidC</i></td><td data-xml-align="left">7, 10</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-SidC</td><td data-xml-align="left">pCyaA <i>dhfr sidC</i></td><td data-xml-align="left">7, 10</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-VIpD</td><td data-xml-align="left">pJB2581 <i>vipD</i></td><td data-xml-align="left">22, 24</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-DHFR-VipD</td><td data-xml-align="left">pCyaA <i>dhfr vipD</i></td><td data-xml-align="left">22, 24</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-Ub</td><td data-xml-align="left">pJB2581 <i>ubi4p</i></td><td data-xml-align="left">93, 95</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-Ub(G3,13)</td><td data-xml-align="left">pJB2581 <i>ubi4p</i>(G3,13)</td><td data-xml-align="left">94, 95</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-Ub-RalF</td><td data-xml-align="left">pJB2581 <i>ubi4p ralF</i></td><td data-xml-align="left">93, 95, 66, 67</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-Ub(G3,13)-RalF</td><td data-xml-align="left">pJB2581 <i>ubi4p</i>(G3,13) <i>ralF</i></td><td data-xml-align="left">94, 95, 66, 67</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pCyaA-Ub(V76)-RalF</td><td data-xml-align="left">pJB2581 <i>ubi4p</i>(V76) <i>ralF</i></td><td data-xml-align="left">108, 95</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pFLAG-SidG</td><td data-xml-align="left">pJB908 FLAG-<i>sidG</i></td><td data-xml-align="left">104b, 43</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pFLAG-SidG(Cterm)</td><td data-xml-align="left">pJB908 FLAG-<i>sidG</i>(936-end)</td><td data-xml-align="left">43</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pFLAG-MavU</td><td data-xml-align="left">pJB908 FLAG-<i>mavU</i></td><td data-xml-align="left">110</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pFLAG-RalF</td><td data-xml-align="left">pJB908 FLAG-<i>ralF</i></td><td data-xml-align="left">109</td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pJB5809</td><td data-xml-align="left">pGST-IcmS/W</td><td> </td><td data-xml-align="left">J. Vogel</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pGST</td><td data-xml-align="left">pJB908 GST</td><td> </td><td data-xml-align="left">This study</td></tr><tr data-xml-valign="top"><td data-xml-align="left">    pSR47s</td><td data-xml-align="left"><i>ori</i>TRP4 <i>ori</i>PR6K <i>sacB</i> Kan<sup>r</sup></td><td> </td><td data-xml-align="left"><a href="#B14" role="doc-biblioref" data-xml-rid="B14">14</a></td></tr></tbody></table></div><div class="notes"><div class="labeled" role="doc-footnote"><div class="label"><sup>a</sup></div><div id="TF1-1" role="paragraph">For the plasmids, amino acid positions or substitutions are given in parentheses with the gene designations.</div></div></div></figure></div><div role="paragraph"><span class="named-content" data-type="genus-species">L. pneumophila</span> strain Lp02 (<i>thyA-hsdR rpsL</i>) (<a href="#B49" role="doc-biblioref" data-xml-rid="B49">49</a>), a thymine (Thy)-auxotrophic derivative of the Philadelphia-1 isolate, was used as the wild type in experiments. Lp03 (Lp02 <i>dotA03</i>) was used as a translocation-deficient control where indicated (<a href="#B49" role="doc-biblioref" data-xml-rid="B49">49</a>). The <i>icmS</i> deletion mutant (Lp02 Δ<i>icmS</i>) was generated as previously described (<a href="#B50" role="doc-biblioref" data-xml-rid="B50">50</a>).</div><div role="paragraph">All CyaA fusions were constructed by cloning IDTS open reading frames (ORFs) into pJB2581 (<a href="#B51" role="doc-biblioref" data-xml-rid="B51">51</a>). Dihydrofolate reductase (DHFR) fusions were constructed by inserting the coding sequence of <i>Mus musculus dhfr</i> (GenBank NM_010049.3) into pJB2581, in between the <i>cyaA</i> and IDTS sequences unless otherwise indicated. Fusions were generated using splicing by overlap extension PCR (SOE-PCR). Ubiquitin fusions were constructed similarly to those with DHFR, using the 67-amino-acid monomer of <i>ubi4p</i> from <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span> S288c (NM_001181859.1). FLAG-tagged fusions were constructed by adding FLAG sequence (ATG-GATTACAAGGACGACGATGACAAG) to the amino terminus of IDTS ORFs by PCR and cloning into pJB908. The glutathione <i>S</i>-transferase (GST)-IcmS/IcmW plasmid (pJB5809) was a generous gift of J. Vogel, Washington University, St. Louis, MO.</div><div role="paragraph">The U937 cell line (ATCC CRL-1593.2) was passaged and differentiated as described previously (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>). HEK293T cells (ATCC CRL-11268) were grown in Dulbecco's modified Eagle's medium (DMEM) containing 4.5g/liter <span data-style="small-caps">d</span>-glucose (catalog number 11995; Invitrogen-Gibco) supplemented with 10% heat-inactivated HyClone fetal bovine serum (FBS; Thermo Scientific). Mouse bone marrow-derived macrophages (BMDM) were prepared from the femurs of female A/J mice (Jackson Laboratories) according to a previously published protocol (<a href="#B52" role="doc-biblioref" data-xml-rid="B52">52</a>) and cultivated in RPMI 1640 medium supplemented with glutamate (Invitrogen-Gibco) and 10% heat-inactivated FBS (Gibco). All manipulations with animals were approved by the Tufts University IACUC, and the research approach was approved the Tufts University Institutional Biosafety Review Board.</div></section><section id="sec-2-2"><h3>CyaA translocation assay.</h3><div role="paragraph">Translocation assays using CyaA protein fusions were adapted from previous protocols (<a href="#B51" role="doc-biblioref" data-xml-rid="B51">51</a>). Unless otherwise indicated, <span class="named-content" data-type="genus-species">L. pneumophila</span> was grown overnight in AYE [<i>N</i>-(2-acetamido)-2-aminoethanesulfonic acid-buffered yeast extract] broth with appropriate additives to exponential phase (<i>A</i><sub>600</sub> of ∼2.2 to 2.4, with less than 10% of the bacteria motile). CyaA protein expression was induced with 200 μM isopropyl-β-<span data-style="small-caps">d</span>-thiogalactopyranoside (IPTG) and incubated for an additional 3 to 5 h until late exponential phase (<i>A</i><sub>600</sub> ∼3.2 to 3.6 with more than 50% motility). A total of 2.5 × 10<sup>6</sup> phorbol ester-differentiated U937 cells were incubated in 24-well plates and challenged with <span class="named-content" data-type="genus-species">L. pneumophila</span> strains at a multiplicity of infection (MOI) of 1. After a 1-h incubation with bacteria (unless otherwise indicated), U937 cells were washed three times with warm phosphate-buffered saline (PBS), and extracts were prepared by the addition of 200 μl of lysis buffer (50 mM HCl, 0.1% Triton X-100) and incubation on ice for 10 min. Lysates were collected, boiled for 5 min, and then neutralized by the addition of 12 μl of 0.5 M NaOH. Protein was precipitated by adding 400 μl of cold 95% ethanol (EtOH) and incubated at −20°C for at least 5 min. Insoluble material was removed by centrifugation for 5 min at 13,000 rpm. Supernatants were dried and resuspended in assay buffer, and the cyclic AMP (cAMP) concentration was measured in extracts diluted 1:500 using a GE Healthcare Life Sciences Biotrak cAMP enzyme-linked immunosorbent assay (ELISA) kit (RPN225). Data were normalized to the amount of the housekeeping protein isocitrate dehydrogenase (ICDH) or the Icm/Dot system protein DotF to determine bacterial load, and data were expressed as a ratio of the cyclase reaction rate relative to the steady-state levels of CyaA reporter fusions in each well, as determined by Western blotting of bacterial lysates. Data are presented as fmol/well/[fusion protein].</div></section><section id="sec-2-3"><h3>Antibodies used in this work.</h3><div role="paragraph">For detection of cyclase fusions by Western blotting probing, anti-CyaA (3D1) mouse monoclonal IgG (Santa Cruz) was used at a concentration of 1:15,000 diluted in 5% milk with a goat anti-mouse IgG conjugated to horseradish peroxidase (HRP) secondary antibody used at a concentration of 1:10,000. For detection using the FLA-9000 developer, anti-CyaA was used at a concentration of 1:5,000 diluted in NAP (non-animal protein) Blocker (GBiosciences) with a Cy5-goat anti-mouse IgG at a concentration of 1:10,000 or Cy3-goat anti-mouse IgG at a concentration of 1:5,000 (Invitrogen). Serum specific for <span class="named-content" data-type="genus-species">Bacillus subtilis</span> isocitrate dehydrogenase (ICDH) was generously provided by A. L. Sonenshein, Tufts University Medical School, Boston, MA.</div></section><section id="sec-2-4"><h3>Real-time assay of translocation rate.</h3><div role="paragraph">A GloSensor cAMP assay (TM076; Promega) was adapted for the real-time analysis of translocation using <span class="named-content" data-type="genus-species">L. pneumophila</span> harboring CyaA fusions. Two days prior to infection, 293T cells were resuspended at a density of 1.5 × 10<sup>5</sup> cells/ml in a solid white 96-well plate (Costar 3917) and incubated overnight. The next day, cells were transfected with pGloSensor20F cAMP plasmid (E1171; Promega) at a concentration of 12.5 ng/μl, as well as with a plasmid expressing FcγRIIA at a concentration of 18.75 ng/μl, using FuGENE HD, according to the manufacturer's instructions. After 24 h of incubation, the medium was carefully removed from the 293T cells and replaced with 200 μl of GloSensor cAMP reagent (E1290; Promega) equilibration medium (88% CO<sub>2</sub>-independent medium [Invitrogen 18045], 10% FBS, 2 to 3% [vol/vol] freshly thawed cAMP reagent) supplemented with 100 μg/ml thymidine and 200 μM IPTG, as necessary. The cells were incubated for 1 h further at 37°C prior to challenge at an MOI of 10 with bacteria that were diluted in opsonizing medium (CO<sub>2</sub>-independent medium, 10% FBS, 100 μg/ml Thy, 200 μM IPTG, and anti-<span class="named-content" data-type="genus-species">Legionella</span> antibody [1:1,000]). The wells containing the opsonized bacteria were subjected to 5 min of centrifugation at 1,000 rpm and immediately placed in a luminescence plate reader preheated to 37°C. Luminescence measurements were taken every 2 min for up to 1 h. As a positive control, forskolin, an activator of adenylyl cyclase, was added at a concentration of 10 μM. Data were recorded as relative light units, after the subtraction of background from wells that were not incubated with the cAMP reagent.</div></section><section id="sec-2-5"><h3>Determination of DHFR enzymatic activity.</h3><div role="paragraph">DHFR activity of the CyaA-IDTS fusions was measured with a colorimetric assay based on the NADPH-dependent reduction of dihydrofolic acid to tetrahydrofolic acid. For measurement of DHFR activity, 100-ml overnight cultures of <span class="named-content" data-type="genus-species">L. pneumophila</span> expressing IDTS fusions to murine DHFR were pelleted, resuspended in 0.5 ml of ice-cold lysis buffer (50 mM Tris, pH 7.5, 1 mM EDTA, 50 μg/ml DNase, 200 μg/ml lysozyme, protease inhibitor cocktail [Roche]), and incubated on ice for 30 min. Samples were lysed by three freeze-thaw cycles, alternating 1- to 2-min incubations in liquid nitrogen, and a 37°C water bath. Cell debris was cleared by centrifugation at 15,000 × <i>g</i> at 4°C for 30 min. Total protein in the extracts was quantified using a Bradford protein assay (Bio-Rad) according to standard protocols.</div><div role="paragraph">For each reaction, 100 μl of protein extracts (0.5 to 1.0 mg of total protein) was incubated in assay buffer (50 mM Tris-HCl, pH 7.5, 500 mM NaCl, 5 mM β-mercaptoethanol) containing 100 μM β-2′-NADPH (Sigma) and 100 μM dihydrofolic acid (Sigma). In each reaction mixture, 100 μM trimethoprim (TMP) or 100 μM methotrexate (MTX) was added. The substrate (dihydrofolic acid) was then added, the reaction solution was mixed by inverting the cuvette several times; the <i>A</i><sub>340</sub> was measured immediately and every 15 s thereafter for 3 min. Purified mammalian DHFR (1.5 × 10<sup>−3</sup> units) was used as a positive control. Specific activity in μmol/min/mg of protein is calculated by the following formula: units/mg protein = [(ΔOD/min<sub>sample</sub>) − (ΔOD/min<sub>blank</sub>)]/ε(mM<sup>−1</sup> cm<sup>−1</sup>)<i>V</i> (mg protein/ml), where ε is 12.3, <i>V</i> is the volume of enzyme (ml), and mg protein/ml is the enzyme concentration of the original sample. To normalize specific activity to the DHFR present in the fusions, steady-state levels of DHFR were determined by Western blotting to obtain protein amounts. Quantification by densitometry compared to a standard curve made from serial dilutions of pure DHFR on the same immunoblot was used to determine protein concentrations.</div></section><section id="sec-2-6"><h3>Cyclase assay on bacterial cultures.</h3><div role="paragraph">A total of 1 × 10<sup>8</sup> bacteria of each culture were pelleted at 8,000 rpm for 5 min, resuspended in 190 μl of lysis buffer (60 mM Tris [pH 8], 10 mM MgCl<sub>2</sub>, 50 μM CaCl<sub>2</sub>, 50 μg/ml DNase, 200 μg/ml lysozyme, protease inhibitor cocktail [Roche]) and incubated on ice for 45 min. Samples were lysed by three freeze-thaw cycles, alternating 1 to 2 min incubations in liquid nitrogen, and a 37°C water bath. In both lysed and unlysed samples, cell debris was cleared by centrifugation at 15,000 × <i>g</i> at 4°C for 5 min. For the reaction mixture, 100 μl of supernatant was mixed with 1 mM ATP in 50 mM Tris and 50 μM CaCl<sub>2</sub>; 10 μM calmodulin (CaM) was added to each tube, and reaction mixtures were incubated at 37°C for 5 to 30 min before reactions were stopped with 2× stop buffer (100 μM HCl–0.2% Triton X-100) and boiling for 5 min, followed by neutralization with 12 μl of 0.5 M NaOH. Protein was precipitated with 400 μl of ice-cold 95% EtOH and incubated at −20°C for at least 5 min. Extracts were dried under vacuum and mixed with assay buffer, and cAMP concentration was measured as for all cyclase assays. Data were normalized to the expression of CyaA fusions as determined by steady-state levels of lysates on a Western blot and displayed as fmol/[fusion protein].</div></section><section id="sec-2-7"><h3>Quantification of protein aggregation.</h3><div role="paragraph">Overnight cultures of <span class="named-content" data-type="genus-species">L. pneumophila</span> harboring fusion proteins were grown as for cyclase assays. A total of 2.5 × 10<sup>9</sup> bacteria were pelleted at 10,000 × <i>g</i> for 5 min, resuspended in 1 ml of lysis buffer (50 mM Tris, pH 7.5, 10 mM EDTA [pH 8.0], 50 μg/ml DNase, 200 μg/ml lysozyme, and protease inhibitor cocktail [Roche]), and incubated on ice for 30 min. Samples were lysed as described for bacterial cyclase assays. Cellular debris was separated by a slow centrifugation at 3,000 × <i>g</i> for 1 min. The pellets were resuspended in 500 μl 2× sample buffer and saved. Total protein samples were obtained by mixing 80 μl of supernatant with 20 μl of 5× sample buffer. To precipitate proteins, 800 μl of supernatant was subjected to centrifugation at ∼100,000 × <i>g</i> for 1 h. Pellets (aggregated proteins) were resuspended in 400 μl of 2× sample buffer, and 160 μl of supernatant was mixed with 40 μl of 5× sample buffer. All samples were boiled, and the equivalent of 2.5 × 10<sup>7</sup> cells for each sample was separated by SDS-PAGE. CyaA-IDTS protein in each sample was detected using antibody against CyaA and quantified by densitometry. Percent aggregation was expressed as the amount of protein in the pellet fractions relative to the total protein.</div></section><section id="sec-2-8"><h3>Copurification of the IDTS and IcmS/W complex.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">E. coli</span> DH5α was transformed with the plasmid pJB5809, encoding GST-IcmS-IcmW (J. Vogel, Washington University, St. Louis, MO), and a second plasmid that encodes a FLAG-tagged IDTS. A plasmid expressing GST without IcmS/IcmW was used as a control. Overnight cultures (500 ml) were grown in LB medium supplemented with kanamycin and carbenicillin. Protein expression was induced by the addition of 0.5 M IPTG, and cells were incubated at 30°C for 3 h with slow shaking. Cultures were harvested by centrifugation at 10,000 × <i>g</i> for 15 min at 4°C, and pellets were resuspended in 10 ml of PBS (pH 7.4), 5 mM MgCl<sub>2</sub>, 50 μg/ml DNase, 1 mM dithiothreitol (DTT), and protease inhibitor cocktail (Roche), followed by lysing with a French press. Lysates were clarified by centrifugation at 15,000 × <i>g</i> for 15 min at 4°C, and cleared lysates were incubated with 0.5 ml of glutathione beads for 2 h at 4°C. Columns were washed three times with cold wash buffer (PBS [pH 7.4], 5 mM MgCl<sub>2</sub>, 1 mM DTT, and protease inhibitor cocktail [Roche]). Bound GST-IcmS/IcmW and associated proteins were eluted with four 0.75-ml aliquots of 10 mM reduced glutathione in 50 mM Tris (pH 8.0). Elutions were combined and concentrated by trichloroacetic acid (TCA) precipitation. All fractions were mixed with 2× sample buffer and probed after Western blotting, with eluate fractions enriched 20×.</div></section><section id="sec-2-9"><h3>Translocation assayed by bacterial fractionation.</h3><div role="paragraph">Differentiated U937 cells were plated in a six-well tissue culture dish at a density of 1.5 × 10<sup>7</sup> cells/well. <span class="named-content" data-type="genus-species">L. pneumophila</span> strains were cultured as for other translocation assays and used to challenge U937 cells at an MOI of 5. After 1.5 h, wells were washed three times with warm medium to remove nonassociated bacteria. Cells were lifted by pipetting up and down in 2 to 3 ml of PBS, pelleted by centrifugation for 5 min at 1,000 rpm, resuspended in 200 μl of PBS containing 0.5% digitonin and protease inhibitor cocktail (Roche), and incubated for 20 min with rotation at room temperature. The detergent lysate was separated by centrifugation at 10,000 rpm for 15 min at 4°C. Supernatant, representing the detergent soluble fraction, was mixed with 5× sample buffer and boiled. Pellets were further solubilized in 2% SDS and mixed with 2× sample buffer.</div></section><section id="sec-2-10"><h3>Intracellular growth assay.</h3><div role="paragraph">Bone marrow-derived macrophages (BMDM) from A/J mice were replated on glass coverslips in a 24-well plate at 2 × 10<sup>5</sup> cells per well. Postexponential phase <span class="named-content" data-type="genus-species">L. pneumophila</span> was used to challenge cells at an MOI of 1, and plates were subjected to centrifugation for 5 min at 1,000 rpm to promote bacterial association with cells. After 2 h, cells were washed with RPMI medium, and growth was allowed to proceed for an additional 12 h. Coverslips were washed in PBS, fixed with 3.7% formaldehyde for 10 min, and blocked in 4% goat serum in PBS. After samples were permeabilized with 0.1% Triton X-100, bacteria were stained by anti-<span class="named-content" data-type="genus-species">L. pneumophila</span> rat serum and secondary anti-rat Texas Red (1:500) (Invitrogen). The number of intracellular bacteria in individual phagosomes was counted by visual inspection under a microscope for 150 to 300 cells.</div></section></section><section id="sec-3" data-type="results"><h2>RESULTS</h2><section id="sec-3-1"><h3>Fusion of dihydrofolate reductase (DHFR) to Icm/Dot-translocated substrates interferes with translocation.</h3><div role="paragraph">The fusion of secreted proteins to DHFR, a rapidly folding enzyme that can be stabilized in its folded conformation with folate analogs, is commonly used to monitor whether proteins can be secreted or translocated in a folded state (<a href="#B33" role="doc-biblioref" data-xml-rid="B33">33</a>, <a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a><a href="#B38" role="doc-biblioref" data-xml-rid="B38">–</a><a href="#B39" role="doc-biblioref" data-xml-rid="B39">39</a>, <a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B53" role="doc-biblioref" data-xml-rid="B53">53</a>). To analyze if translocation via the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> Icm/Dot type IVB secretion system can occur in the presence of folded substrates, fusions of several Icm/Dot-translocated substrates (IDTSs) to the carboxyl terminus of the murine DHFR were constructed. We chose four different substrates to analyze. RalF and SidG were interrogated because there is considerable information regarding the nature of the C-terminal translocation signals of these proteins as well as the effects of the mature protein sequences on translocation efficiency (<a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>, <a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>). MavU was chosen because in a previous study we found that its C-terminal signal was one of the most effective targeting sequences identified among the IDTSs, so we wanted to determine if a high-efficiency signal could overcome deficits caused by the presence of a folded domain (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>). Finally, we wanted to investigate whether the introduction of a folded domain internal to an IDTS could have different effects from placing the domain upstream from the complete coding sequence. For this reason, we used LidA, because in previous studies we had identified important junctions between domains in the protein that we thought would likely tolerate added sequences (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>).</div><div role="paragraph">To detect translocation, each construction was tagged with the calmodulin-dependent adenylyl cyclase enzymatic domain encoded by the <i>Bordetella pertussis cyaA</i> gene (<a href="#F1">Fig. 1A</a>; <a href="#T1">Table 1</a>). Production of cAMP by these constructions requires translocation into eukaryotic cells as bacteria do not express calmodulin (<a href="#B56" role="doc-biblioref" data-xml-rid="B56">56</a>). Expression of these CyaA constructs was induced by the addition of IPTG to cultures of the wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> strain Lp02, and the bacteria were used to challenge the human macrophage-like cell line U937. The translocation efficiency of each protein fusion into target U937 cells was quantified as intracellular cAMP produced by the CyaA-IDTS fusions, normalized to relative steady-state levels of the fusion within each strain (see Materials and Methods) (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>).</div><div class="figure-wrap"><header><div class="label">Fig 1</div></header><figure id="F1" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/e5c2c30c-ce08-4f3e-a720-9fc0615f7cb4/assets/graphic/zii9990902850001.jpeg" /><figcaption><span class="heading">Fig 1</span> Insertion of DHFR interferes with translocation. (A) Schematic representation of CyaA-IDTS fusion proteins used in translocation assays. For fusions containing the DHFR moiety, DHFR was fused to the carboxyl terminus of CyaA immediately upstream of an IDTS or translocation signal. (B) The presence of DHFR inhibits translocation. Translocation levels of the IDTS were quantified by measuring intracellular cAMP levels resulting from CyaA. The wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> strain Lp02 (black and gray bars) or the translocation-defective <i>dotA</i> strain (striped bar) expressing the indicated CyaA protein fusions was used to challenge U937 cells. After 1 h, intracellular cAMP levels were determined by ELISA in cell lysates (see Materials and Methods). The values represent the mean cAMP determined in seven separate experiments (21 independent infections), normalized to the amount of the T4SS protein DotF and steady-state levels of CyaA in the vector-alone control absent the IDTS, as determined by immunoblotting of bacterial lysates. Black bars, fusions to CyaA alone, as indicated; gray bars, the same IDTS fused to DHFR as shown in panel A; striped bar, Cya-RalF expressed in a <i>dotA</i> mutant strain. Error bars indicate ± standard errors. *, <i>P</i> &lt; 0.02; **, <i>P</i> &lt; 0.004; ***, <i>P</i> &lt; 0.001.</figcaption></figure></div><div role="paragraph">Each of the IDTSs tested exhibited translocation defects when fused to DHFR (<a href="#F1">Fig. 1B</a>), with each DHFR fusion showing translocation levels that were between 8-fold (for DHFR-MavU) and 40-fold (for DHFR-LidA) lower than the comparable fusions lacking the DHFR moiety. In the case of DHFR-LidA, it appeared that DHFR totally blocked translocation; its translocation efficiency was no greater than that observed with a mutant strain (<i>dotA</i>) containing a defective type IV system unable to translocate IDTS (<a href="#F1">Fig. 1B</a>).</div><div role="paragraph">Interestingly, the CyaA protein that was not fused to any IDTS still translocated at levels ∼5-fold higher in a wild-type strain than in a <i>dotA</i> mutant background (<a href="#F1">Fig. 1B</a>), and this level was higher than what we observed with wild-type strains harboring the DHFR-LidA fusion. The CyaA construction had a short tail added to its carboxyl-terminal end to facilitate construction of the fusions, and this may have acted as a cryptic signal since the identical construction lacking this short carboxyl-terminal sequence showed translocation levels as low as those of the <i>dotA</i> control (J. Vogel, personal communication). This leakage through the Icm/Dot complex was abolished when CyaA was fused to DHFR, indicating that the DHFR moiety alone is inhibitory to translocation through the Icm/Dot complex even when a cryptic translocation signal is present in a protein (<a href="#F1">Fig. 1B</a>).</div><div role="paragraph">There was considerable translocation of the protein having a DHFR fusion to MavU, presumably due to the extremely effective C-terminal signal, so we tested whether the relatively high levels of translocation were due to protein structure or the nature of the signal. To this end, serial amino-terminal deletions of the <span class="named-content" data-type="genus-species">L. pneumophila</span> IDTS MavU (Lpg1798) were constructed to selectively remove domains in the protein. MavU is a 398-amino-acid protein predicted to contain a Rho-GTPase-activating protein (RhoGAP) domain and a coiled-coil region (<a href="#F2">Fig. 2A</a>). Therefore, truncations were made immediately downstream of the RhoGAP domain (<a href="#F2">Fig. 2</a>, C186), flanking the predicted coiled-coil domain (C118 and C78), and just upstream of the predicted disordered carboxyl terminus (C50) (<a href="#F2">Fig. 2A</a>; <a href="#T1">Table 1</a>). CyaA fusions to these truncated proteins were all similarly expressed, and all exhibited translocation efficiencies that were similar to the translocation efficiency of the full-length MavU (<a href="#F2">Fig. 2B</a> and <a href="#F2">C</a>, black bars). Translocation of each of the truncated proteins fused to DHFR was then determined relative to the amount of full-length steady-state protein found on gels (see Materials and Methods). All of these DHFR fusion proteins were translocated at similar levels and showed similar levels of defects, consistent with the idea that a specific protein structure cannot ameliorate DHFR-mediated translocation defects (<a href="#F2">Fig. 2C</a>). The observation that translocation of CyaA-DHFR-MavU(C50), encoding only the C-terminal 50 amino acids of MavU, was similar to that of the full-length MavU-DHFR fusion indicates that the sensitivity of an IDTS to translocation inhibition resulting from insertion of the DHFR moiety does not depend on the size of the hybrid protein but may be connected to the efficiency at which the C-terminal signal can allow recognition by the Icm/Dot translocation system.</div><div class="figure-wrap"><header><div class="label">Fig 2</div></header><figure id="F2" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/df777c8e-9005-4e49-9361-286dfb56101c/assets/graphic/zii9990902850002.jpeg" /><figcaption><span class="heading">Fig 2</span> Serial truncations of MavU fused to DHFR exhibit equivalent defects in translocation competence. (A) A schematic of MavU fusions to CyaA and DHFR (top) and of the MavU protein with the RhoGAP domain (light gray), coiled-coil domain (white box), and predicted disordered carboxyl terminus (black) (bottom) is shown. Arrows indicate individual truncation sites for CyaA fusions. (B) Steady-state levels of MavU protein fusions. Fusion protein expression was detected using a monoclonal antibody against CyaA. Asterisks note full-length fusion bands. Bacterial load was quantified using antibody against the housekeeping protein ICDH. The blot is a representative of strains used in one typical CyaA translocation assay. (C) The presence of the DHFR moiety interferes with truncated translocation-competent proteins. U937 cells were challenged with the wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> strain Lp02 harboring truncations (black and gray bars) or a translocation-defective <i>dotA3</i> strain harboring full-length CyaA-MavU (striped bars). After 1 h, intracellular cAMP levels in host cells were determined (see Materials and Methods). The values represent the average amount of cAMP determined in seven separate experiments (21 independent infections) normalized to the amount of the housekeeping protein ICDH in bacteria and then normalized to the amount of CyaA in the fusion protein, as determined by immunoblotting of bacterial lysates. α, anti; FL, full-length.</figcaption></figure></div></section><section id="sec-3-2"><h3>The DHFR moiety in IDTS fusions is enzymatically active.</h3><div role="paragraph">In many of the previous studies examining the effects of protein folding on secretion, a DHFR inhibitor such as methotrexate (MTX), which inhibits both mammalian and bacterial proteins, was necessary to stabilize the DHFR in a folded conformation (<a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a>). As DHFR blocked IDTS translocation in the absence of MTX, we tested whether the inhibitor would cause further depression of translocation and whether the enzyme was folded in an active confirmation, consistent with its ability to block translocation. As expected, MTX had no effect on the growth in culture of the <i>thy</i> mutant strain used in these studies (data not shown). Furthermore, the addition of MTX did not depress further the already low levels of translocation of the CyaA-DHFR-MavU fusion (data not shown). This raised the possibility that the DHFR moiety was not assuming the conformation of the native active protein in these protein fusions. Therefore, to determine whether DHFR placed within the chimeric proteins was in its native folded form, we tested if the derivatives were enzymatically active.</div><div role="paragraph">DHFR activity of the CyaA-IDTS fusions was measured using a colorimetric assay that detects the NADPH-dependent reduction of dihydrofolic acid to tetrahydrofolic acid (<a href="#F3">Fig. 3A</a>) (see Materials and Methods). Cell lysates of <span class="named-content" data-type="genus-species">L. pneumophila</span> expressing CyaA-DHFR-IDTS fusions were incubated with NADPH and dihydrofolic acid, and the accumulation of tetrahydrofolate was measured (see Materials and Methods). To determine the specific activity, the rate of hydrolysis as a function of protein concentration was determined by quantitating the amount of fusion protein expressed for each sample (see Materials and Methods) (<a href="#F3">Fig. 3B</a>). Trimethoprim was added to eliminate the endogenous <span class="named-content" data-type="genus-species">L. pneumophila</span> DHFR activity as, unlike MTX (<a href="#B57" role="doc-biblioref" data-xml-rid="B57">57</a>), this antibiotic inhibits the prokaryotic protein specifically and has no effect on the mammalian DHFR present in the fusions (<a href="#F2">Fig. 2C</a>). The specific activities of DHFR in the protein fusions were comparable to those of purified DHFR, consistent with proper folding of this moiety (<a href="#F2">Fig. 2C</a>). As expected, no activity was seen in a CyaA-IDTS fusion that did not have DHFR, and the enzymatic activity in all reactions was abolished by MTX (<a href="#F2">Fig. 2C</a>). The data are consistent with a folded protein moiety causing defective substrate translocation.</div><div class="figure-wrap"><header><div class="label">Fig 3</div></header><figure id="F3" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/b2433a9b-6e42-467a-b837-8cae8a461f59/assets/graphic/zii9990902850003.jpeg" /><figcaption><span class="heading">Fig 3</span> The DHFR moiety within the DHFR-IDTS fusions is catalytically active. (A) NADPH-dependent reduction of dihydrofolic acid (DHFA) to tetrahydrofolic acid (THFA) catalyzed by DHFR. (B) Immunoblot of <span class="named-content" data-type="genus-species">L. pneumophila</span> soluble extracts from strains overexpressing CyaA-MavU (lane 1) or two different cultures of CyaA-DHFR-MavU proteins (lanes 2 and 3). Antibody against DHFR was used to determine expression of the CyaA-DHFR-IDTS or endogenous DHFR. Densitometry of the CyaA-DHFR-IDTS bands allowed for quantification of protein concentration for determining specific activity. (C) DHFR fusion proteins are enzymatically active. Bacterial cultures were grown in the presence of IPTG to induce expression of the indicated protein fusions. Cells were lysed, and the soluble protein extracts were combined with 100 μM NADPH, 100 μM dihydrofolic acid, and either 50 μM TMP or MTX to inhibit endogenous bacterial DHFR activity or all DHFR activity, respectively. Purified human DHFR was used as a positive control. Activity was normalized to DHFR protein levels as determined by Western blotting. <i>P</i> values are determined by Student's <i>t</i> test, comparing DHFR activity of fusions in the presence of TMP to the purified human DHFR control.</figcaption></figure></div></section><section id="sec-3-3"><h3>DHFR does not interfere with cyclase activity or increase aggregation of the chimeric proteins.</h3><div role="paragraph">One complication of using CyaA fusions is that the assay is contingent on the enzymatic activity of the CyaA moiety being unaffected by the IDTS sequence. Therefore, it was necessary to exclude the possibility that the presence of DHFR inhibits adenylate cyclase activity. A modified, <i>in vitro</i> version of the CyaA translocation assay was established to quantify the enzymatic activity of CyaA-DHFR-IDTS fusions relative to the respective CyaA-IDTS. <span class="named-content" data-type="genus-species">L. pneumophila</span> cultures were grown to late exponential phase, induced to express the CyaA constructs, and lysed. The supernatants of the bacterial lysates were incubated for 30 min in the presence of calmodulin and the adenylate cyclase substrate ATP, and the resulting cAMP production was measured via ELISA (see Materials and Methods). No significant difference was observed in the CyaA activities of any of the protein fusions, regardless of DHFR, confirming that DHFR is not inhibiting the activity of the marker enzyme (<a href="#F4">Fig. 4A</a>). Similar data were seen when the incubation period was shortened to 5 min, indicating that the longer reaction time did not overload the system (data not shown). Background levels of cAMP produced by <span class="named-content" data-type="genus-species">L. pneumophila</span> not expressing any CyaA fusion and a reaction mixture lacking calmodulin confirmed that adenylate cyclase activity was due to the <span class="named-content" data-type="genus-species">B. pertussis</span> CyaA. The culture supernatant from a strain that was not lysed also did not produce significant levels of cAMP, consistent with previous research showing that Icm/Dot secretion occurs only after contact with a host cell (<a href="#B58" role="doc-biblioref" data-xml-rid="B58">58</a>). These data are consistent with the DHFR moiety having no effect on cyclase activity, with the caveat that it is possible that after translocation into host cells, the folding properties of the translocated substrate may differ significantly from that observed in bacterial extracts.</div><div class="figure-wrap"><header><div class="label">Fig 4</div></header><figure id="F4" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/42b90928-c45f-4071-86de-3a36aa8a9af3/assets/graphic/zii9990902850004.jpeg" /><figcaption><span class="heading">Fig 4</span> Translocation deficiency is not due to loss of adenylate cyclase activity or aggregation of fusion proteins. (A) Adenylate cyclase activity is unaffected by DHFR. Bacterial lysates were incubated in the presence of ATP and calmodulin for 30 min at 37°C to allow for the production of cAMP. Samples were harvested and assayed as described in Materials and Methods. (B) Aggregation of IDTS protein fusions does not correlate with translocation defects of DHFR-IDTS fusions. <span class="named-content" data-type="genus-species">L. pneumophila</span> wild-type strains expressing the fusions were lysed, and cell debris was cleared by low-speed centrifugation. The resulting supernatant fraction was subjected to centrifugation at 90,000 × <i>g</i> for 1 h to precipitate any aggregated protein. Supernatant and pellet fractions after centrifugation were analyzed by Western blotting with anti-CyaA and anti-ICDH antibody probes. D, low-speed pellet; T, low-speed supernatant; S, high-speed supernatant; P, high-speed pellet. (C) The intensity of the fusion proteins in panel B was determined by densitometry, and protein expression is graphed as a percentage of the total protein after high-speed centrifugation.</figcaption></figure></div><div role="paragraph">We next determined if DHFR hinders translocation by causing aggregation of the fusions. To compare aggregation of DHFR fusions and IDTS alone, we lysed cells and compared the relative amounts of soluble and pelleted protein after high-speed centrifugation. The presence of substrate proteins in postcentrifugation fractions was visualized via immunoblots probed with anti-CyaA (<a href="#F4">Fig. 4B</a>) (see Materials and Methods). The percentage of aggregation was determined as the amount of CyaA fusion protein that precipitated after high-speed centrifugation relative to the total protein retrieved, as quantified by densitometry of the immunoblots. In all strains tested, no significant difference in aggregation was seen between the IDTS fusions in the presence or absence of DHFR, demonstrating that DHFR does not cause protein aggregation (<a href="#F4">Fig. 4C</a>). Interestingly, a majority of the protein remained soluble after centrifugation in the strains expressing RalF and MavU fusions, but in the strains expressing SidG fusions, nearly all protein recovered from the high-speed centrifugation was seen in the pellets. Although a relatively small amount of soluble SidG protein appears to be available, our results show that SidG is still readily translocated. These results are consistent with the model that the DHFR moiety does not cause aggregation and that aggregation does not correlate with decreased translocation.</div></section><section id="sec-3-4"><h3>Interference of IDTS translocation by ubiquitin requires folding of the interfering moiety.</h3><div role="paragraph">Although the data presented argue that the DHFR moiety was folded and did not interfere with the reporter readout, we could not analyze the consequences of conditional folding of an interfering moiety. Therefore, we analyzed ubiquitin (Ub) fusions to IDTSs as mutant Ub derivatives exist that are poorly folded, allowing us to compare folded and unfolded internal domains. As with DHFR, ubiquitin (Ub) is a compact, rapidly folding protein that can serve as a disruptive domain for secreted substrates in multiple systems (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a>, <a href="#B60" role="doc-biblioref" data-xml-rid="B60">60</a>). Furthermore, simple mutations can be used to destabilize the protein and yield an unfolded control for secretion assays (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a>). To this end, we constructed a fusion of RalF tagged with CyaA to a Ub protein in which two residues in the hydrophobic core (Ile3 and Ile13) are replaced with Gly (Ub<sup>G3,13</sup>) to destabilize the protein conformation (<a href="#F5">Fig. 5A</a>) (<a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a>).</div><div class="figure-wrap"><header><div class="label">Fig 5</div></header><figure id="F5" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/c3925f04-2e29-4367-b6c5-7a93b583cb35/assets/graphic/zii9990902850005.jpeg" /><figcaption><span class="heading">Fig 5</span> Folded ubiquitin inhibits translocation of fused Icm/Dot substrates. U937 cells were challenged with the wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> strain Lp02 or a translocation-defective <i>dotA</i> strain expressing CyaA-tagged RalF fused to either ubiquitin, a nonfolding variant of ubiquitin (Ub<sup>G3,13</sup>), or a variant that cannot be cleaved by DUBs (Ub<sup>V76</sup>). (A) Schematic of the RalF-Ub fusions. Lightning bolts indicate locations of the mutations used to create an uncleavable (Ub<sup>V76</sup>) or destabilized (Ub<sup>G3,13</sup>) ubiquitin. (B) Wild-type Ub is cleaved in <span class="named-content" data-type="genus-species">L. pneumophila</span>. Immunoblotting of fusion protein expression was performed using a monoclonal antibody against CyaA. Bacterial load was quantified using antibody against the Icm/Dot complex protein DotF. The blot is a representative of strains used in one typical CyaA translocation assay. (C) Folded Ub inhibits translocation. A CyaA translocation assay was performed as described in the legend of <a href="#F1">Fig. 1</a>. The values represent the average cAMP amounts produced from four independent experiments (9 to 12 independent infections) normalized to DotF and then normalized to the expression of CyaA in the vector. Asterisks mark significance based on Student's <i>t</i> test (*, <i>P</i> ≤ 0.02; **, <i>P</i> ≤ 0.0002; ***, <i>P</i> ≤ 0.0009).</figcaption></figure></div><div role="paragraph">Interestingly, we found that the CyaA-RalF fusion to wild-type Ub was cleaved in the cytosol of <span class="named-content" data-type="genus-species">L. pneumophila</span> (<a href="#F5">Fig. 5B</a>). This may be due to a deubiquitinating protease (DUB) that was predicted bioinformatically to be encoded by <span class="named-content" data-type="genus-species">L. pneumophila</span> (Lpg2907; MavW) (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>, <a href="#B61" role="doc-biblioref" data-xml-rid="B61">61</a>). In contrast, the Ub<sup>G3,13</sup> protein was not cleaved, consistent with the interpretation that these mutations prevent folding of the Ub domain. As cleaved Ub fusion proteins would be uninformative in our experiments, we constructed an additional Ub construct by replacing the terminal residue (Gly76) with a Val residue (Ub<sup>V76</sup>). This mutation alters the site of proteolytic cleavage by DUBs without perturbing the folding of Ub (<a href="#B62" role="doc-biblioref" data-xml-rid="B62">62</a>).</div><div role="paragraph">CyaA-RalF fused to the folded, uncleavable Ub<sup>V76</sup> demonstrated very low translocation efficiency (<a href="#F5">Fig. 5C</a>), resulting in cAMP levels similar to those of the <i>dotA</i> strain. In contrast, the fusion protein having the unfolded Ub<sup>G3,13</sup> derivative was translocated at a small but significantly (<i>P</i> = 0.02) higher efficiency than CyaA-RalF alone. Taking these results together with the DHFR data, we conclude that the extent to which an IDTS maintains a region with a folded conformation inversely correlates with its translocation efficiency.</div></section><section id="sec-3-5"><h3>The DHFR-MavU fusion protein exhibits a kinetic defect in translocation.</h3><div role="paragraph">The above analysis shows that accumulation of the fusion proteins is lowered by the presence of a poison domain but does not investigate whether there is premature termination of secretion or whether the domain slows the rate of translocation. To carry out real-time, kinetic analysis of translocation to answer this question, we adapted the GloSensor cAMP assay (see Materials and Methods). The system uses a modified <span class="named-content" data-type="genus-species">Photinus pyralis</span> luciferase that contains a cAMP-binding domain. Binding of cAMP causes a conformational change that allows the luciferase to interact with its substrate to produce light. For our purposes, monolayers of 293T cells transiently transfected with a plasmid expressing the modified luciferase and the FcγRIIA receptor (to facilitate uptake of IgG-opsonized <span class="named-content" data-type="genus-species">L. pneumophila</span>) were preincubated with the luciferase substrate and then challenged with <span class="named-content" data-type="genus-species">L. pneumophila</span> strains harboring IDTSs fused to CyaA. Intracellular cAMP resulting from translocation of the adenylate cyclase was monitored by the luminescent output over the course of the infection (see Materials and Methods). The kinetic analysis agreed strongly with the endpoint cAMP assay. DHFR fusions to RalF and SidG, as well as the CyaA gene itself, resulted in a substantial kinetic delay in luminescence signal (decreased rate of transfer as represented by depressed slope and increased time-to-half-maximal signal). (<a href="#F6">Fig. 6A</a> and <a href="#F6">B</a>). In each case, the time to half-maximal luminescence for the wild-type proteins fused to Cya was approximately 5 min (<a href="#F6">Fig. 6A</a> to <a href="#F6">C</a>). In contrast, for the DHFR fusions to RalF and SidG, the time to half-maximal luminescence was at least 30 min (<a href="#F6">Fig. 6A</a> and <a href="#F6">B</a>). The MavU-DHFR fusion, however, exhibited a more modest kinetic delay, with a time to half-maximal luminescence of approximately 18 min although the total accumulation of luminescence was clearly lower than that of the wild type (<a href="#F6">Fig. 6C</a>). Similar results were obtained with the quantitative endpoint CyaA translocation assay using individual time points over the course of infection (<a href="#F6">Fig. 6D</a>). Both assays are consistent with an initial kinetic delay caused by DHFR fusion to MavU of about 15 to 20 min. This indicates that for at least some IDTSs, a folded moiety within the protein interferes with the efficiency of its passage through the Icm/Dot channel, lowering the yields of translocated product due to depressed rates of translocation and possibly delayed initiation of transit.</div><div class="figure-wrap"><header><div class="label">Fig 6</div></header><figure id="F6" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/3954dae0-903c-47d1-8203-d37d491aa0b6/assets/graphic/zii9990902850006.jpeg" /><figcaption><span class="heading">Fig 6</span> The DHFR-IDTS fusion proteins exhibit kinetic defects in translocation. Results of real-time translocation assays of CyaA and Cya-DHFR moieties fused to IDTS RalF (A), SidG (B), and MavU (C) are shown. HEK293 cells transiently transfected with the pGloSensor cAMP biosensor plasmid and FcγRIIA were incubated with the pGloSensor luciferase substrate reagent for 1 h (see Materials and Methods). Cells were challenged with bacterial strains expressing the indicated proteins that had been opsonized with anti-<span class="named-content" data-type="genus-species">Legionella</span> IgG. The fluorescence was measured in a luminometer every 2 min for up to 1 h postinfection. Data represent the average of four independent replicates from a typical experiment ± standard error. (D) Direct quantitation of cAMP demonstrates kinetic defect of CyaA-DHFR-MavU fusion. CyaA translocation assays of CyaA-MavU and CyaA-DHFR-MavU fusion proteins were performed as described in the legend of <a href="#F1">Fig. 1</a> (see also Materials and Methods). Data represent the average of three replicates ± standard error from a typical experiment. RLU, relative light units.</figcaption></figure></div></section><section id="sec-3-6"><h3>Low-level translocation of DHFR-MavU requires IcmS.</h3><div role="paragraph">The IcmS/W adaptor complex is necessary for translocation of a large subset of <span class="named-content" data-type="genus-species">L. pneumophila</span> effectors (<a href="#B63" role="doc-biblioref" data-xml-rid="B63">63</a>, <a href="#B64" role="doc-biblioref" data-xml-rid="B64">64</a>). Previous work indicated that the IcmS/W complex binds to several different IDTSs, and in the case of SidG, this appears to alter the conformational state of the protein (<a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>). Furthermore, IcmS is necessary for the stability of the T4BSS coupling protein DotL and binds directly to DotL (<a href="#B32" role="doc-biblioref" data-xml-rid="B32">32</a>, <a href="#B64" role="doc-biblioref" data-xml-rid="B64">64</a>). Therefore, if folded DHFR precludes IcmS from binding the IDTS fusions, the folded moiety could restrict the substrate from forming a translocation-competent conformation with IcmS/W.</div><div role="paragraph">To investigate whether the presence of DHFR precludes IcmS from facilitating translocation, we first compared translocation of DHFR-MavU in the wild type (Lp02) to an <i>icmS</i> deletion mutant of <span class="named-content" data-type="genus-species">L. pneumophila</span>. The translocation efficiency of DHFR-MavU was more than 10-fold lower in the Δ<i>icmS</i> strain than in the wild type, similar to the magnitude of defect found with the parental MavU fusion (<a href="#F7">Fig. 7A</a>). This indicates that the IcmS/IcmW complex is still necessary and capable of facilitating translocation of the DHFR fusion and is consistent with DHFR interfering with translocation at some level other than IcmS/W interactions.</div><div class="figure-wrap"><header><div class="label">Fig 7</div></header><figure id="F7" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/76073067-b3f0-4cd0-9d8d-87042f1f093a/assets/graphic/zii9990902850007.jpeg" /><figcaption><span class="heading">Fig 7</span> DHFR does not preclude interaction of the IcmS adapter protein with the IDTS protein fusions. (A) Lowered translocation of DHFR-MavU fusions to MavU in the absence of IcmS. CyaA-MavU and CyaA-DHFR-MavU were expressed in the wild-type strain (Lp02) and an IcmS-deficient strain (Δ<i>icmS</i> strain) and used to challenge U937 cells. A CyaA translocation assay was performed as described in the legend of <a href="#F2">Fig. 2</a>. The values represent the average amount of cAMP determined by ELISA in five independent experiments (15 independent infections) normalized to the amount of the T4SS protein DotF expressed and then normalized to the amount of CyaA in the vector. (B) IcmS copurifies with MavU-DHFR fusions. Copurification of CyaA-MavU and CyaA-DHFR-MavU with GST-IcmS/W was performed. Lysates from <span class="named-content" data-type="genus-species">E. coli</span> encoding the indicated fusion with or without a second plasmid expressing GST or GST-IcmS/W were incubated with glutathione-conjugated resin for 1 h. Bound GST-IcmS and complexed proteins were eluted by reduced glutathione and concentrated by TCA precipitation. Protein samples were analyzed via Western blotting using anti-CyaA. Shown are the clarified lysate (IN) and the eluate concentrated 20-fold (E). The experiment was performed twice. The IN and E lanes from each sample were analyzed on a single blot but were loaded on noncontiguous lanes, as noted by gaps between lanes.</figcaption></figure></div><div role="paragraph">To further determine if DHFR prevented association of the IcmS/W complex with IDTSs, we determined the relative fraction of fusions that coassociate with the chaperone in the presence or absence of the DHFR moiety (<a href="#F7">Fig. 7B</a>). Either CyaA-MavU or CyaA-DHFR-MavU was coexpressed in <span class="named-content" data-type="genus-species">E. coli</span> with a GST-IcmS/IcmW complex (see Materials and Methods) (<a href="#T1">Table 1</a>). Lysates were incubated with immobilized glutathione resin, bound proteins were eluted with reduced glutathione, and CyaA-tagged proteins bound to GST-IcmS/IcmW were detected by immunoblotting. There was no significant difference in binding to GST-IcmS/W between CyaA-MavU and CyaA-DHFR-MavU (<a href="#F7">Fig. 7B</a>). In contrast, little fusion protein was associated with the resin in the absence of coexpression of the GST-IcmS/W complex or if GST alone was coexpressed with the fusions (<a href="#F7">Fig. 7B</a>). We thus confirm that the folded DHFR moiety does not interfere with the interaction of IcmS/W with IDTSs.</div></section><section id="sec-3-7"><h3>IDTS fused to folded DHFR or Ub interferes with Icm/Dot secretion channel function.</h3><div role="paragraph">It is well documented that secretion signals fused to secretion-incompetent proteins often result in blockage of the translocation channel, resulting in a severe attenuation of channel function (<a href="#B36" role="doc-biblioref" data-xml-rid="B36">36</a>). Most relevant to this study, introduction of DHFR and Ub to type III secretion system-translocated substrates in <span class="named-content" data-type="genus-species">Yersinia enterocolitica</span> has been shown to interfere with transport of other proteins through this channel (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>). As the DHFR and Ub fusions interfered with translocation in the Icm/Dot T4SS, we tested whether these constructions could occlude translocation of other IDTSs or slow the rate of translocation to the extent that there would be defective intracellular growth due to a blockage the Icm/Dot system, which is known to be essential for intracellular growth.</div><div role="paragraph">To determine if introducing a folded moiety into RalF causes translocation inhibition, we monitored export of an endogenously expressed IDTS, SidC, in strains overproducing RalF fusions having either folded or unfolded domains inserted. U937 cells were challenged for 1.5 h with <span class="named-content" data-type="genus-species">L. pneumophila</span> that expresses either RalF or one of the RalF fusions (<a href="#F8">Fig. 8A</a> and <a href="#F8">B</a>), and cells were lysed with digitonin. The detergent-soluble and -insoluble fractions were then analyzed by immunoblotting to identify exported and bacterium-associated SidC, respectively (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>) (see Materials and Methods). Strains expressing either the Ub<sup>V76</sup> or DHFR interfered with translocation of SidC as the translocation efficiency was less than 25% of that observed in strains harboring either RalF or the unfolded moiety-containing Ub<sup>G3,13</sup>. Therefore, overproduction of the translocation-incompetent proteins interferes with function of the Icm/Dot translocon.</div><div class="figure-wrap"><header><div class="label">Fig 8</div></header><figure id="F8" class="graphic"><img src="/cms/10.1128/IAI.00552-13/asset/be6cd60d-4b06-4464-9b93-43cafd760b10/assets/graphic/zii9990902850008.jpeg" /><figcaption><span class="heading">Fig 8</span> Folded protein fusions cause a partial blockage of the Icm/Dot translocon. (A) Translocation of SidC is decreased in strains harboring folded fusions. Secretion of IDTSs in strains overexpressing folded protein fusions was visualized by detergent fractionation of infected cells (see Materials and Methods). U937 cells were challenged with <span class="named-content" data-type="genus-species">Legionella</span> expressing CyaA-RalF fusions to DHFR, a folded Ub (Ub<sup>V76</sup>), or the unfolded Ub (Ub<sup>G3,13</sup>). After 1.5 h, cells were washed and lysed with 0.5% digitonin, and the soluble and insoluble fractions were separated by centrifugation. Translocation of the IDTS SidC into the cytoplasm of infected cells (digitonin-soluble extract) was analyzed by probing via immunoblotting. Antibodies to the bacterial T4SS protein DotF and the cytoplasmic protein RhoGDI were used to confirm that lysis does not release bacterial proteins as well as to show separation of the fractions. L.p., <span class="named-content" data-type="genus-species">L. pneumophila</span>. (B) The graph depicts the percentage of the SidC protein translocated, considered as the amount in the soluble fraction relative to the total protein in the sample, as determined by densitometry. The experiment was performed twice. Shown is a representative experiment. (C) Intracellular growth of strains harboring folded fusion proteins is inhibited. Murine bone marrow macrophages were challenged with either the wild-type strain (Lp02) or <i>dotA</i> strains harboring RalF fusions containing DHFR, a folded Ub (Ub<sup>V76</sup>), or the unfolded Ub (Ub<sup>G3,13</sup>). Growth was allowed to proceed for 14 h, at which time the infected macrophages were fixed and stained with an antibody against <span class="named-content" data-type="genus-species">Legionella</span>. The number of intracellular bacteria in individual vacuoles was determined for 100 cells each on three individual replicates for each sample. Data indicate the percentage of vacuoles with 1, 2 to 3, 4 to 7, 8 to 10, and 11 or more bacteria per vacuole ± standard error.</figcaption></figure></div><div role="paragraph">To determine if interference with translocation also inhibits intracellular bacterial replication, an intracellular growth assay of <span class="named-content" data-type="genus-species">L. pneumophila</span> overexpressing RalF fused to DHFR, Ub<sup>G3,13</sup>, or Ub<sup>V76</sup> was performed. Monolayers of bone marrow-derived macrophages from A/J mice were challenged with wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> expressing the reporter protein fusions for 14 h, which is approximately one round of infection. Cells were fixed, and the number of bacteria in individual <span class="named-content" data-type="genus-species">Legionella</span>-containing vacuoles (LCVs) was determined by microscopy. In strains harboring RalF alone or RalF fused to the unfolded Ub<sup>G3,13</sup> variant, approximately 60% of LCVs were large vacuoles with 11 or more bacteria, consistent with efficient growth in the presence of these proteins. In the strain expressing the DHFR fusion, the presence of DHFR restricted growth, as evidenced by an increase in the percentage of vacuoles that had small numbers of bacteria (<a href="#B1" role="doc-biblioref" data-xml-rid="B1">1</a><a href="#B2" role="doc-biblioref" data-xml-rid="B2">–</a><a href="#B3" role="doc-biblioref" data-xml-rid="B3">3</a>), with a corresponding decrease in vacuoles having more than 11 bacteria (<a href="#F8">Fig. 8C</a>). The growth defect was most pronounced in the strain expressing the folded ubiquitin (Ub<sup>V76</sup>), with the percentage of large vacuoles of 11 or more bacteria decreasing approximately 4-fold compared to the strain expressing the unfolded Ub fusion (<a href="#F8">Fig. 8C</a>). Furthermore, there was a significant increase in the percentage of small and medium-sized vacuoles, consistent with decreased growth in the presence of this fusion. This phenotype is indicative of a partial blockage of the translocon by folded proteins, reducing the rate of translocation of IDTSs needed for bacterial replication. Therefore, occlusion of the channel by the introduction of a folded domain interferes with the ability of <span class="named-content" data-type="genus-species">L. pneumophila</span> to grow intracellularly.</div></section></section><section id="sec-4" data-type="discussion"><h2>DISCUSSION</h2><div role="paragraph">In this study, we show that translocation of substrates of the Icm/Dot T4SS into host cells is depressed by the presence of moieties that assume folded conformations. Fusion proteins of DHFR and Ub to IDTSs demonstrate a defect in translocation compared to their unfolded counterparts (<a href="#F1">Fig. 1</a> and <a href="#F5">5</a>). Furthermore, there was a small but significant increase in translocation of the IDTS RalF fused to a destabilized variant of ubiquitin (Ub<sup>G3,13</sup>) relative to RalF alone, consistent with an unfolded region stimulating translocation (<a href="#F5">Fig. 5</a>). If the unstructured moiety increases the amount of unfolding of the entire IDTS, translocation competence may be defined by the ability of the IDTS to maintain an unfolded conformation.</div><div role="paragraph">Many IDTSs are translated prior to infection, as indicated by the fact that protein synthesis inhibitors do not affect translocation rates (<a href="#B58" role="doc-biblioref" data-xml-rid="B58">58</a>). If we consider this in context with our data, the system would likely require one or more chaperone proteins to perpetuate an unfolded, translocation-competent state of the effectors. These may be heat shock proteins or uncharacterized chaperones but also could include the adapter protein complexes of IcmS/W and IcmS/LvgA (<a href="#B64" role="doc-biblioref" data-xml-rid="B64">64</a>). These small proteins are hypothesized to resemble the type III secretion system chaperones which act to hold proteins in a partially unfolded conformation for delivery to the translocon ATPase for subsequent unfolding and secretion (<a href="#B45" role="doc-biblioref" data-xml-rid="B45">45</a>, <a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B66" role="doc-biblioref" data-xml-rid="B66">66</a>). Consistent with a role for these proteins forming a pretranslocation complex, IcmS/W can associate with translocated substrates only when coexpressed in cells (<a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>). This pretranslocation complex appears to be the form that is presented to the secretion channel as IcmS binds directly to DotL, the secretion ATPase in the Icm/Dot system (<a href="#B32" role="doc-biblioref" data-xml-rid="B32">32</a>). Although insertion of DHFR interfered with translocation, IcmS/W was still able to interact with MavU- and SidG-DHFR fusions, and IcmS was necessary to facilitate the translocation of the DHFR-MavU fusion (<a href="#F7">Fig. 7</a>). These results indicate that the defect in translocation observed in this work is independent of IcmS/W action.</div><div role="paragraph">Using two different assays, we were able to demonstrate that the DHFR moiety decreases the rate of protein transport (<a href="#F6">Fig. 6</a>). Two models can explain this result. The first possibility is that a folded domain does not cause an absolute block in translocation, either because the Icm/Dot system has some intrinsic ability to translocate folded proteins or because the secretion system is able to convert translocation-incompetent substrates into transferrable conformations. The second possibility is that there is a small subpopulation of proteins that are partially unfolded. In either model, the extent to which a protein is in a translocation-competent conformation directly correlates with the rate that it can be transported through the Icm/Dot translocon.</div><div role="paragraph">In addition to the translocation defect of a tightly folded protein, the effector fusions interfere with translocation of other IDTSs (<a href="#F8">Fig. 8</a>), consistent with at least partial occlusion of the translocon. In the case of the <span class="named-content" data-type="genus-species">Yersinia</span> T3SS, the presence of folded domains in translocated substrates interferes with the assembly of specific components necessary for the function of the translocon (<a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>). Although we cannot exclude this possibility, the fact that the DHFR fusions are very poorly translocated but cause only partial interference with intracellular growth (<a href="#F8">Fig. 8</a>) is consistent with the Icm/Dot complex being largely intact. We propose two models for the mechanism of this blockage. First, the interfering protein fusion could disrupt transport of only a subset of translocated effectors that are critical for intracellular growth. The second model is that the fusion protein is stalled within the secretion channel, causing a global decrease in the rate of all IDTSs. The low rate of transport of the fusion proteins could generate a situation akin to a traffic jam, in which a group of slow-moving individuals causes a general reduction in the rate of movement of all individuals moving through the secretion channel.</div><div role="paragraph">If stalled or slow-moving substrates cause the generalized translocation defect, then there must be some explanation for why such interference occurs. Although the structure of the T4BSS has not been fully elucidated, cryo-electron microscopic and crystal structures of core components of a T4SS from plasmid pKM101 have been solved. While the Icm/Dot system constitutes a different class of secretion apparatus (T4BSS), some insight can be gained from considering the known structure. Cryo-electron microscopy of the T4SS reveals a unique structure in which the core forms a large, bulbous chamber between the inner and outer membranes (<a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). Importantly, the channel is constricted at the capped outer membrane face, which the crystal structure suggests undergoes a conformational change that could restrict the channel at the outer membrane, allowing opening or closing of the secretion pore (<a href="#B30" role="doc-biblioref" data-xml-rid="B30">30</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). At the base of the structure is a large entryway of 55 nm although it is believed that other components of the secretion system could fit within this entryway and make for a much narrower passage site (<a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). Based upon this configuration, at the very least, the transport of folded proteins through the Icm/Dot system may be hindered while exiting the translocon. This situation would likely lead to the accumulation of proteins within the secretion system core.</div><div role="paragraph">Blocks could also occur at the entry site of the secretion complex. In this scenario, initiation of movement into the secretion complex would result in a frustrated intermediate in which there is a physical block at the entrance. Furthermore, occlusion of IDTSs could occur at a step that is prior to entry into the complex. In the T3SS, folded effector fusions prevent translocation of other IDTSs by binding to and inhibiting the ATPase associated with the secretion system (<a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>). Normally, chaperones carry their cognate substrates to the T3SS in a partially unfolded conformation, thus “priming” them for complete unfolding by the ATPase before translocation (<a href="#B45" role="doc-biblioref" data-xml-rid="B45">45</a>, <a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>). In the Icm/Dot system it has been proposed that DotL, as part of a subcomplex with the ATPases DotM and DotN, acts as a regulator of IDTS entry (<a href="#B67" role="doc-biblioref" data-xml-rid="B67">67</a>). DotL has sequence similarity to type IV secretion system coupling proteins, which are proposed to link substrates to the rest of the secretion complex (<a href="#B68" role="doc-biblioref" data-xml-rid="B68">68</a>). The fact that DotL associates with IcmS could be key to linking the substrate to the secretion system. Folded proteins may be blocked from entry by an inappropriate interaction with DotL, thus inhibiting further translocation of effectors.</div><div role="paragraph">These studies demonstrate that small folded domains can generate impassable substrates through the <span class="named-content" data-type="genus-species">L. pneumophila</span> T4SS, consistent with a strong selection against such moieties being present in IDTSs. Therefore, it would appear that most IDTSs are either easily unfolded, maintained with chaperones in an unfolded conformation, or have determinants that allow recognition by unfoldases associated with the Icm/Dot complex. Future work will require identifying the features of the substrates that allow them to be secretion competent and determining which bacterial proteins contribute to generating structures that maintain this conformation.</div></section><section id="acknowledgments" role="doc-acknowledgments"><h2>ACKNOWLEDGMENTS</h2><div data-type="financial-disclosure" role="paragraph">R.R.I. is an investigator of HHMI, while W.M.A. and D.D. were supported by NIH training grant T32AI00742. D.D. was also supported by NIAID fellowship F31 AI098423. This work was also supported by Howard Hughes Medical Institute.</div><div role="paragraph">We thank Joseph Vogel for plasmid gifts as well as numerous suggestions and support, Linc Sonenshein for ICDH antibody, and Elizabeth Creasey, Sina Mohammadi, Edward Geisinger, Tamara O'Connor, and Kim Davis for review of the manuscript.</div></section></div></section><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="bibliography" role="doc-bibliography"><h2>REFERENCES</h2><div class="labeled" role="doc-biblioentry"><div class="label">1.</div><div id="B1" class="citations"><div class="citation"><div class="citation-content">Fields BS. 1996. The molecular ecology of legionellae. <em>Trends Microbiol.</em> 4:286–290.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0966-842X(96)10041-X">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8829338">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1996UY30500010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+molecular+ecology+of+legionellae&amp;author=BS+Fields&amp;publication_year=1996&amp;pages=286-290&amp;doi=10.1016%2F0966-842X%2896%2910041-X&amp;pmid=8829338">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">2.</div><div id="B2" class="citations"><div class="citation"><div class="citation-content">Fields BS, Benson RF, and Besser RE. 2002. <span class="named-content" data-type="genus-species">Legionella</span> and Legionnaires' disease: 25 years of investigation. <em>Clin. Microbiol. Rev.</em> 15:506–526.</div><div class="external-links"><div><a href="https://doi.org/10.1128/CMR.15.3.506-526.2002">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12097254">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000176791300014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+and+Legionnaires%27+disease%3A+25+years+of+investigation&amp;author=BS+Fields&amp;author=RF+Benson&amp;author=RE+Besser&amp;publication_year=2002&amp;pages=506-526&amp;doi=10.1128%2FCMR.15.3.506-526.2002&amp;pmid=12097254">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">3.</div><div id="B3" class="citations"><div class="citation"><div class="citation-content">Fields BS, Shotts EB Jr, Feeley JC, Gorman GW, and Martin WT. 1984. Proliferation of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> as an intracellular parasite of the ciliated protozoan <span class="named-content" data-type="genus-species">Tetrahymena pyriformis</span>. <em>Appl. Environ. Microbiol.</em> 47:467–471.</div><div class="external-links"><div><a href="https://doi.org/10.1128/aem.47.3.467-471.1984">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/6424568">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1984SG70300004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proliferation+of+Legionella+pneumophila+as+an+intracellular+parasite+of+the+ciliated+protozoan+Tetrahymena+pyriformis&amp;author=BS+Fields&amp;author=EB+Shotts&amp;author=JC+Feeley&amp;author=GW+Gorman&amp;author=WT+Martin&amp;publication_year=1984&amp;pages=467-471&amp;doi=10.1128%2Faem.47.3.467-471.1984&amp;pmid=6424568">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">4.</div><div id="B4" class="citations"><div class="citation"><div class="citation-content">Wadowsky RM, Butler LJ, Cook MK, Verma SM, Paul MA, Fields BS, Keleti G, Sykora JL, and Yee RB. 1988. Growth-supporting activity for <span class="named-content" data-type="genus-species">Legionella pneumophila</span> in tap water cultures and implication of hartmannellid amoebae as growth factors. <em>Appl. Environ. Microbiol.</em> 54:2677–2682.</div><div class="external-links"><div><a href="https://doi.org/10.1128/aem.54.11.2677-2682.1988">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3214153">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1988Q820000015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Growth-supporting+activity+for+Legionella+pneumophila+in+tap+water+cultures+and+implication+of+hartmannellid+amoebae+as+growth+factors&amp;author=RM+Wadowsky&amp;author=LJ+Butler&amp;author=MK+Cook&amp;author=SM+Verma&amp;author=MA+Paul&amp;author=BS+Fields&amp;author=G+Keleti&amp;author=JL+Sykora&amp;author=RB+Yee&amp;publication_year=1988&amp;pages=2677-2682&amp;doi=10.1128%2Faem.54.11.2677-2682.1988&amp;pmid=3214153">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">5.</div><div id="B5" class="citations"><div class="citation"><div class="citation-content">Fraser DW, Tsai TR, Orenstein W, Parkin WE, Beecham HJ, Sharrar RG, Harris J, Mallison GF, Martin SM, McDade JE, Shepard CC, and Brachman PS. 1977. Legionnaires' disease: description of an epidemic of pneumonia. <em>N. Engl. J. Med.</em> 297:1189–1197.</div><div class="external-links"><div><a href="https://doi.org/10.1056/NEJM197712012972201">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/335244">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1977EC22000001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionnaires%27+disease%3A+description+of+an+epidemic+of+pneumonia&amp;author=DW+Fraser&amp;author=TR+Tsai&amp;author=W+Orenstein&amp;author=WE+Parkin&amp;author=HJ+Beecham&amp;author=RG+Sharrar&amp;author=J+Harris&amp;author=GF+Mallison&amp;author=SM+Martin&amp;author=JE+McDade&amp;author=CC+Shepard&amp;author=PS+Brachman&amp;publication_year=1977&amp;pages=1189-1197&amp;doi=10.1056%2FNEJM197712012972201&amp;pmid=335244">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">6.</div><div id="B6" class="citations"><div class="citation"><div class="citation-content">Horwitz MA and Silverstein SC. 1980. Legionnaires' disease bacterium (<span class="named-content" data-type="genus-species">Legionella pneumophila</span>) multiples intracellularly in human monocytes. <em>J. Clin. Invest.</em> 66:441–450.</div><div class="external-links"><div><a href="https://doi.org/10.1172/JCI109874">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7190579">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1980KE70700006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionnaires%27+disease+bacterium+%28Legionella+pneumophila%29+multiples+intracellularly+in+human+monocytes&amp;author=MA+Horwitz&amp;author=SC+Silverstein&amp;publication_year=1980&amp;pages=441-450&amp;doi=10.1172%2FJCI109874&amp;pmid=7190579">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">7.</div><div id="B7" class="citations"><div class="citation"><div class="citation-content">Segal G and Shuman HA. 1999. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> utilizes the same genes to multiply within <span class="named-content" data-type="genus-species">Acanthamoeba castellanii</span> and human macrophages. <em>Infect. Immun.</em> 67:2117–2124.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.67.5.2117-2124.1999">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10225863">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000079909300010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+utilizes+the+same+genes+to+multiply+within+Acanthamoeba+castellanii+and+human+macrophages&amp;author=G+Segal&amp;author=HA+Shuman&amp;publication_year=1999&amp;pages=2117-2124&amp;doi=10.1128%2FIAI.67.5.2117-2124.1999&amp;pmid=10225863">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">8.</div><div id="B8" class="citations"><div class="citation"><div class="citation-content">Berger KH and Isberg RR. 1993. Two distinct defects in intracellular growth complemented by a single genetic locus in <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Mol. Microbiol.</em> 7:7–19.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.1993.tb01092.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8382332">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1993KE35300002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Two+distinct+defects+in+intracellular+growth+complemented+by+a+single+genetic+locus+in+Legionella+pneumophila&amp;author=KH+Berger&amp;author=RR+Isberg&amp;publication_year=1993&amp;pages=7-19&amp;doi=10.1111%2Fj.1365-2958.1993.tb01092.x&amp;pmid=8382332">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">9.</div><div id="B9" class="citations"><div class="citation"><div class="citation-content">Segal G and Shuman HA. 1997. Characterization of a new region required for macrophage killing by <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Infect. Immun.</em> 65:5057–5066.</div><div class="external-links"><div><a href="https://doi.org/10.1128/iai.65.12.5057-5066.1997">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9393796">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1997YJ22700024">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Characterization+of+a+new+region+required+for+macrophage+killing+by+Legionella+pneumophila&amp;author=G+Segal&amp;author=HA+Shuman&amp;publication_year=1997&amp;pages=5057-5066&amp;doi=10.1128%2Fiai.65.12.5057-5066.1997&amp;pmid=9393796">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">10.</div><div id="B10" class="citations"><div class="citation"><div class="citation-content">Altman E and Segal G. 2008. The response regulator CpxR directly regulates expression of several <i>Legionella pneumophila icm</i>/<i>dot</i> components as well as new translocated substrates. <em>J. Bacteriol.</em> 190:1985–1996.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.01493-07">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18192394">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+response+regulator+CpxR+directly+regulates+expression+of+several+Legionella+pneumophila+icm%2Fdot+components+as+well+as+new+translocated+substrates&amp;author=E+Altman&amp;author=G+Segal&amp;publication_year=2008&amp;pages=1985-1996&amp;doi=10.1128%2FJB.01493-07&amp;pmid=18192394">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">11.</div><div id="B11" class="citations"><div class="citation"><div class="citation-content">de Felipe KS, Glover RT, Charpentier X, Anderson OR, Reyes M, Pericone CD, and Shuman HA. 2008. <span class="named-content" data-type="genus-species">Legionella</span> eukaryotic-like type IV substrates interfere with organelle trafficking. <em>PLoS Pathog.</em> 4:e1000117.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000117">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18670632">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000259783100005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+eukaryotic-like+type+IV+substrates+interfere+with+organelle+trafficking&amp;author=KS+de+Felipe&amp;author=RT+Glover&amp;author=X+Charpentier&amp;author=OR+Anderson&amp;author=M+Reyes&amp;author=CD+Pericone&amp;author=HA+Shuman&amp;publication_year=2008&amp;pages=e1000117&amp;doi=10.1371%2Fjournal.ppat.1000117&amp;pmid=18670632">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">12.</div><div id="B12" class="citations"><div class="citation"><div class="citation-content">de Felipe KS, Pampou S, Jovanovic OS, Pericone CD, Ye SF, Kalachikov S, and Shuman HA. 2005. Evidence for acquisition of <span class="named-content" data-type="genus-species">Legionella</span> type IV secretion substrates via interdomain horizontal gene transfer. <em>J. Bacteriol.</em> 187:7716–7726.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.187.22.7716-7726.2005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16267296">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Evidence+for+acquisition+of+Legionella+type+IV+secretion+substrates+via+interdomain+horizontal+gene+transfer&amp;author=KS+de+Felipe&amp;author=S+Pampou&amp;author=OS+Jovanovic&amp;author=CD+Pericone&amp;author=SF+Ye&amp;author=S+Kalachikov&amp;author=HA+Shuman&amp;publication_year=2005&amp;pages=7716-7726&amp;doi=10.1128%2FJB.187.22.7716-7726.2005&amp;pmid=16267296">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">13.</div><div id="B13" class="citations"><div class="citation"><div class="citation-content">Huang L, Boyd D, Amyot WM, Hempstead AD, Luo ZQ, O'Connor TJ, Chen C, Machner M, Montminy T, and Isberg RR. 2011. The E block motif is associated with <span class="named-content" data-type="genus-species">Legionella pneumophila</span> translocated substrates. <em>Cell Microbiol.</em> 13:227–245.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2010.01531.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20880356">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000286206100006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+E+block+motif+is+associated+with+Legionella+pneumophila+translocated+substrates&amp;author=L+Huang&amp;author=D+Boyd&amp;author=WM+Amyot&amp;author=AD+Hempstead&amp;author=ZQ+Luo&amp;author=TJ+O%27Connor&amp;author=C+Chen&amp;author=M+Machner&amp;author=T+Montminy&amp;author=RR+Isberg&amp;publication_year=2011&amp;pages=227-245&amp;doi=10.1111%2Fj.1462-5822.2010.01531.x&amp;pmid=20880356">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">14.</div><div id="B14" class="citations"><div class="citation"><div class="citation-content">Luo ZQ and Isberg RR. 2004. Multiple substrates of the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> Dot/Icm system identified by interbacterial protein transfer. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 101:841–846.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0304916101">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14715899">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000188555400028">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Multiple+substrates+of+the+Legionella+pneumophila+Dot%2FIcm+system+identified+by+interbacterial+protein+transfer&amp;author=ZQ+Luo&amp;author=RR+Isberg&amp;publication_year=2004&amp;pages=841-846&amp;doi=10.1073%2Fpnas.0304916101&amp;pmid=14715899">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">15.</div><div id="B15" class="citations"><div class="citation"><div class="citation-content">Nagai H, Kagan JC, Zhu X, Kahn RA, and Roy CR. 2002. A bacterial guanine nucleotide exchange factor activates ARF on <span class="named-content" data-type="genus-species">Legionella</span> phagosomes. <em>Science</em> 295:679–682.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1067025">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11809974">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000173560900046">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+bacterial+guanine+nucleotide+exchange+factor+activates+ARF+on+Legionella+phagosomes&amp;author=H+Nagai&amp;author=JC+Kagan&amp;author=X+Zhu&amp;author=RA+Kahn&amp;author=CR+Roy&amp;publication_year=2002&amp;pages=679-682&amp;doi=10.1126%2Fscience.1067025&amp;pmid=11809974">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">16.</div><div id="B16" class="citations"><div class="citation"><div class="citation-content">Zusman T, Degtyar E, and Segal G. 2008. Identification of a hypervariable region containing new <span class="named-content" data-type="genus-species">Legionella pneumophila</span> Icm/Dot translocated substrates by using the conserved <i>icmQ</i> regulatory signature. <em>Infect. Immun.</em> 76:4581–4591.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.00337-08">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18694969">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+a+hypervariable+region+containing+new+Legionella+pneumophila+Icm%2FDot+translocated+substrates+by+using+the+conserved+icmQ+regulatory+signature&amp;author=T+Zusman&amp;author=E+Degtyar&amp;author=G+Segal&amp;publication_year=2008&amp;pages=4581-4591&amp;doi=10.1128%2FIAI.00337-08&amp;pmid=18694969">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">17.</div><div id="B17" class="citations"><div class="citation"><div class="citation-content">Zhu W, Banga S, Tan Y, Zheng C, Stephenson R, Gately J, and Luo ZQ. 2011. Comprehensive identification of protein substrates of the Dot/Icm type IV transporter of <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>PLoS One</em> 6:e17638.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.pone.0017638">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21408005">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000288170900034">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive+identification+of+protein+substrates+of+the+Dot%2FIcm+type+IV+transporter+of+Legionella+pneumophila&amp;author=W+Zhu&amp;author=S+Banga&amp;author=Y+Tan&amp;author=C+Zheng&amp;author=R+Stephenson&amp;author=J+Gately&amp;author=ZQ+Luo&amp;publication_year=2011&amp;pages=e17638&amp;doi=10.1371%2Fjournal.pone.0017638&amp;pmid=21408005">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">18.</div><div id="B18" class="citations"><div class="citation"><div class="citation-content">Nagai H and Kubori T. 2011. Type IVB secretion systems of <span class="named-content" data-type="genus-species">Legionella</span> and other Gram-negative bacteria. <em>Front. Microbiol.</em> 2:136.</div><div class="external-links"><div><a href="https://doi.org/10.3389/fmicb.2011.00136">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21743810">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000208863500146">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+IVB+secretion+systems+of+Legionella+and+other+Gram-negative+bacteria&amp;author=H+Nagai&amp;author=T+Kubori&amp;publication_year=2011&amp;pages=136&amp;doi=10.3389%2Ffmicb.2011.00136&amp;pmid=21743810">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">19.</div><div id="B19" class="citations"><div class="citation"><div class="citation-content">Lifshitz Z, Burstein D, Peeri M, Zusman T, Schwartz K, Shuman HA, Pupko T, and Segal G. 2013. Computational modeling and experimental validation of the <span class="named-content" data-type="genus-species">Legionella</span> and <span class="named-content" data-type="genus-species">Coxiella</span> virulence-related type-IVB secretion signal. <em>Proc. Nat. Acad. Sci., U. S. A.</em> 110:E707–E715.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.1215278110">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/23382224">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Computational+modeling+and+experimental+validation+of+the+Legionella+and+Coxiella+virulence-related+type-IVB+secretion+signal&amp;author=Z+Lifshitz&amp;author=D+Burstein&amp;author=M+Peeri&amp;author=T+Zusman&amp;author=K+Schwartz&amp;author=HA+Shuman&amp;author=T+Pupko&amp;author=G+Segal&amp;publication_year=2013&amp;pages=E707-E715&amp;doi=10.1073%2Fpnas.1215278110&amp;pmid=23382224">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">20.</div><div id="B20" class="citations"><div class="citation"><div class="citation-content">Kagan JC, Stein MP, Pypaert M, and Roy CR. 2004. <span class="named-content" data-type="genus-species">Legionella</span> subvert the functions of Rab1 and Sec22b to create a replicative organelle. <em>J. Exp. Med.</em> 199:1201–1211.</div><div class="external-links"><div><a href="https://doi.org/10.1084/jem.20031706">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15117975">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000221274600004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+subvert+the+functions+of+Rab1+and+Sec22b+to+create+a+replicative+organelle&amp;author=JC+Kagan&amp;author=MP+Stein&amp;author=M+Pypaert&amp;author=CR+Roy&amp;publication_year=2004&amp;pages=1201-1211&amp;doi=10.1084%2Fjem.20031706&amp;pmid=15117975">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">21.</div><div id="B21" class="citations"><div class="citation"><div class="citation-content">Derre I and Isberg RR. 2004. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> replication vacuole formation involves rapid recruitment of proteins of the early secretory system. <em>Infect. Immun.</em> 72:3048–3053.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.72.5.3048-3053.2004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15102819">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000221120100069">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+replication+vacuole+formation+involves+rapid+recruitment+of+proteins+of+the+early+secretory+system&amp;author=I+Derre&amp;author=RR+Isberg&amp;publication_year=2004&amp;pages=3048-3053&amp;doi=10.1128%2FIAI.72.5.3048-3053.2004&amp;pmid=15102819">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">22.</div><div id="B22" class="citations"><div class="citation"><div class="citation-content">Molmeret M, Santic M, Asare R, Carabeo RA, and Abu Kwaik Y. 2007. Rapid escape of the dot/icm mutants of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> into the cytosol of mammalian and protozoan cells. <em>Infect. Immun.</em> 75:3290–3304.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.00292-07">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17438033">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000247707600009">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rapid+escape+of+the+dot%2Ficm+mutants+of+Legionella+pneumophila+into+the+cytosol+of+mammalian+and+protozoan+cells&amp;author=M+Molmeret&amp;author=M+Santic&amp;author=R+Asare&amp;author=RA+Carabeo&amp;author=Y+Abu+Kwaik&amp;publication_year=2007&amp;pages=3290-3304&amp;doi=10.1128%2FIAI.00292-07&amp;pmid=17438033">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">23.</div><div id="B23" class="citations"><div class="citation"><div class="citation-content">Christie PJ. 2001. Type IV secretion: intercellular transfer of macromolecules by systems ancestrally related to conjugation machines. <em>Mol. Microbiol.</em> 40:294–305.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.2001.02302.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11309113">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000168473700003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+IV+secretion%3A+intercellular+transfer+of+macromolecules+by+systems+ancestrally+related+to+conjugation+machines&amp;author=PJ+Christie&amp;publication_year=2001&amp;pages=294-305&amp;doi=10.1046%2Fj.1365-2958.2001.02302.x&amp;pmid=11309113">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">24.</div><div id="B24" class="citations"><div class="citation"><div class="citation-content">Christie PJ and Vogel JP. 2000. Bacterial type IV secretion: conjugation systems adapted to deliver effector molecules to host cells. <em>Trends Microbiol.</em> 8:354–360.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0966-842X(00)01792-3">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10920394">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000088880500011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Bacterial+type+IV+secretion%3A+conjugation+systems+adapted+to+deliver+effector+molecules+to+host+cells&amp;author=PJ+Christie&amp;author=JP+Vogel&amp;publication_year=2000&amp;pages=354-360&amp;doi=10.1016%2FS0966-842X%2800%2901792-3&amp;pmid=10920394">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">25.</div><div id="B25" class="citations"><div class="citation"><div class="citation-content">Wallden K, Rivera-Calzada A, and Waksman G. 2010. Type IV secretion systems: versatility and diversity in function. <em>Cell Microbiol.</em> 12:1203–1212.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2010.01499.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20642798">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000280972600002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+IV+secretion+systems%3A+versatility+and+diversity+in+function&amp;author=K+Wallden&amp;author=A+Rivera-Calzada&amp;author=G+Waksman&amp;publication_year=2010&amp;pages=1203-1212&amp;doi=10.1111%2Fj.1462-5822.2010.01499.x&amp;pmid=20642798">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">26.</div><div id="B26" class="citations"><div class="citation"><div class="citation-content">Nagai H and Roy CR. 2003. Show me the substrates: modulation of host cell function by type IV secretion systems. <em>Cell Microbiol.</em> 5:373–383.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1462-5822.2003.00285.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12780775">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Show+me+the+substrates%3A+modulation+of+host+cell+function+by+type+IV+secretion+systems&amp;author=H+Nagai&amp;author=CR+Roy&amp;publication_year=2003&amp;pages=373-383&amp;doi=10.1046%2Fj.1462-5822.2003.00285.x&amp;pmid=12780775">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">27.</div><div id="B27" class="citations"><div class="citation"><div class="citation-content">Beare PA, Gilk SD, Larson CL, Hill J, Stead CM, Omsland A, Cockrell DC, Howe D, Voth DE, and Heinzen RA. 2011. Dot/Icm type IVB secretion system requirements for <span class="named-content" data-type="genus-species">Coxiella burnetii</span> growth in human macrophages. <em>mBio</em> 2(4):e00175-00111.</div><div class="external-links"><div><a href="https://doi.org/10.1128/mBio.00175-11">Crossref</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Dot%2FIcm+type+IVB+secretion+system+requirements+for+Coxiella+burnetii+growth+in+human+macrophages&amp;author=PA+Beare&amp;author=SD+Gilk&amp;author=CL+Larson&amp;author=J+Hill&amp;author=CM+Stead&amp;author=A+Omsland&amp;author=DC+Cockrell&amp;author=D+Howe&amp;author=DE+Voth&amp;author=RA+Heinzen&amp;publication_year=2011&amp;pages=e00175-00111&amp;doi=10.1128%2FmBio.00175-11">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">28.</div><div id="B28" class="citations"><div class="citation"><div class="citation-content">Leclerque A and Kleespies RG. 2008. Type IV secretion system components as phylogenetic markers of entomopathogenic bacteria of the genus <span class="named-content" data-type="genus-species">Rickettsiella</span>. <em>FEMS Microbiol. Lett.</em> 279:167–173.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1574-6968.2007.01025.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18179586">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+IV+secretion+system+components+as+phylogenetic+markers+of+entomopathogenic+bacteria+of+the+genus+Rickettsiella&amp;author=A+Leclerque&amp;author=RG+Kleespies&amp;publication_year=2008&amp;pages=167-173&amp;doi=10.1111%2Fj.1574-6968.2007.01025.x&amp;pmid=18179586">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">29.</div><div id="B29" class="citations"><div class="citation"><div class="citation-content">Fronzes R, Christie PJ, and Waksman G. 2009. The structural biology of type IV secretion systems. <em>Nat. Rev. Microbiol.</em> 7:703–714.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nrmicro2218">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19756009">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269831000015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+structural+biology+of+type+IV+secretion+systems&amp;author=R+Fronzes&amp;author=PJ+Christie&amp;author=G+Waksman&amp;publication_year=2009&amp;pages=703-714&amp;doi=10.1038%2Fnrmicro2218&amp;pmid=19756009">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">30.</div><div id="B30" class="citations"><div class="citation"><div class="citation-content">Chandran V, Fronzes R, Duquerroy S, Cronin N, Navaza J, and Waksman G. 2009. Structure of the outer membrane complex of a type IV secretion system. <em>Nature</em> 462:1011–1015.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nature08588">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19946264">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000272996000037">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Structure+of+the+outer+membrane+complex+of+a+type+IV+secretion+system&amp;author=V+Chandran&amp;author=R+Fronzes&amp;author=S+Duquerroy&amp;author=N+Cronin&amp;author=J+Navaza&amp;author=G+Waksman&amp;publication_year=2009&amp;pages=1011-1015&amp;doi=10.1038%2Fnature08588&amp;pmid=19946264">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">31.</div><div id="B31" class="citations"><div class="citation"><div class="citation-content">Vincent CD, Friedman JR, Jeong KC, Buford EC, Miller JL, and Vogel JP. 2006. Identification of the core transmembrane complex of the <span class="named-content" data-type="genus-species">Legionella</span> Dot/Icm type IV secretion system. <em>Mol. Microbiol.</em> 62:1278–1291.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2006.05446.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17040490">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000241832000007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+the+core+transmembrane+complex+of+the+Legionella+Dot%2FIcm+type+IV+secretion+system&amp;author=CD+Vincent&amp;author=JR+Friedman&amp;author=KC+Jeong&amp;author=EC+Buford&amp;author=JL+Miller&amp;author=JP+Vogel&amp;publication_year=2006&amp;pages=1278-1291&amp;doi=10.1111%2Fj.1365-2958.2006.05446.x&amp;pmid=17040490">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">32.</div><div id="B32" class="citations"><div class="citation"><div class="citation-content">Vincent CD, Friedman JR, Jeong KC, Sutherland MC, and Vogel JP. 2012. Identification of the DotL coupling protein subcomplex of the <span class="named-content" data-type="genus-species">Legionella</span> Dot/Icm type IV secretion system. <em>Mol. Microbiol.</em> 85:378–391.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2012.08118.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22694730">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+the+DotL+coupling+protein+subcomplex+of+the+Legionella+Dot%2FIcm+type+IV+secretion+system&amp;author=CD+Vincent&amp;author=JR+Friedman&amp;author=KC+Jeong&amp;author=MC+Sutherland&amp;author=JP+Vogel&amp;publication_year=2012&amp;pages=378-391&amp;doi=10.1111%2Fj.1365-2958.2012.08118.x&amp;pmid=22694730">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">33.</div><div id="B33" class="citations"><div class="citation"><div class="citation-content">Klingenberg O and Olsnes S. 1996. Ability of methotrexate to inhibit translocation to the cytosol of dihydrofolate reductase fused to diphtheria toxin. <em>Biochem. J.</em> 313:647–653.</div><div class="external-links"><div><a href="https://doi.org/10.1042/bj3130647">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8573105">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Ability+of+methotrexate+to+inhibit+translocation+to+the+cytosol+of+dihydrofolate+reductase+fused+to+diphtheria+toxin&amp;author=O+Klingenberg&amp;author=S+Olsnes&amp;publication_year=1996&amp;pages=647-653&amp;doi=10.1042%2Fbj3130647&amp;pmid=8573105">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">34.</div><div id="B34" class="citations"><div class="citation"><div class="citation-content">Cambronne ED and Roy CR. 2007. The <span class="named-content" data-type="genus-species">Legionella pneumophila</span> IcmSW complex interacts with multiple Dot/Icm effectors to facilitate type IV translocation. <em>PLoS Pathog.</em> 3:e188.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.0030188">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18069892">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000253296400002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+pneumophila+IcmSW+complex+interacts+with+multiple+Dot%2FIcm+effectors+to+facilitate+type+IV+translocation&amp;author=ED+Cambronne&amp;author=CR+Roy&amp;publication_year=2007&amp;pages=e188&amp;doi=10.1371%2Fjournal.ppat.0030188&amp;pmid=18069892">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">35.</div><div id="B35" class="citations"><div class="citation"><div class="citation-content">Burstein D, Zusman T, Degtyar E, Viner R, Segal G, and Pupko T. 2009. Genome-scale identification of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> effectors using a machine learning approach. <em>PLoS Pathog.</em> 5:e1000508.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000508">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19593377">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269224500025">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale+identification+of+Legionella+pneumophila+effectors+using+a+machine+learning+approach&amp;author=D+Burstein&amp;author=T+Zusman&amp;author=E+Degtyar&amp;author=R+Viner&amp;author=G+Segal&amp;author=T+Pupko&amp;publication_year=2009&amp;pages=e1000508&amp;doi=10.1371%2Fjournal.ppat.1000508&amp;pmid=19593377">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">36.</div><div id="B36" class="citations"><div class="citation"><div class="citation-content">Bieker KL and Silhavy TJ. 1989. PrlA is important for the translocation of exported proteins across the cytoplasmic membrane of <span class="named-content" data-type="genus-species">Escherichia coli</span>. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 86:968–972.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.86.3.968">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/2536939">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=PrlA+is+important+for+the+translocation+of+exported+proteins+across+the+cytoplasmic+membrane+of+Escherichia+coli&amp;author=KL+Bieker&amp;author=TJ+Silhavy&amp;publication_year=1989&amp;pages=968-972&amp;doi=10.1073%2Fpnas.86.3.968&amp;pmid=2536939">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">37.</div><div id="B37" class="citations"><div class="citation"><div class="citation-content">Eilers M and Schatz G. 1986. Binding of a specific ligand inhibits import of a purified precursor protein into mitochondria. <em>Nature</em> 322:228–232.</div><div class="external-links"><div><a href="https://doi.org/10.1038/322228a0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3016548">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1986D250300040">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Binding+of+a+specific+ligand+inhibits+import+of+a+purified+precursor+protein+into+mitochondria&amp;author=M+Eilers&amp;author=G+Schatz&amp;publication_year=1986&amp;pages=228-232&amp;doi=10.1038%2F322228a0&amp;pmid=3016548">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">38.</div><div id="B38" class="citations"><div class="citation"><div class="citation-content">Endo T, Kawakami M, Goto A, America T, Weisbeek P, and Nakai M. 1994. Chloroplast protein import. Chloroplast envelopes and thylakoids have different abilities to unfold proteins. <em>Eur. J. Biochem.</em> 225:403–409.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1432-1033.1994.00403.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7925462">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Chloroplast+protein+import.+Chloroplast+envelopes+and+thylakoids+have+different+abilities+to+unfold+proteins&amp;author=T+Endo&amp;author=M+Kawakami&amp;author=A+Goto&amp;author=T+America&amp;author=P+Weisbeek&amp;author=M+Nakai&amp;publication_year=1994&amp;pages=403-409&amp;doi=10.1111%2Fj.1432-1033.1994.00403.x&amp;pmid=7925462">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">39.</div><div id="B39" class="citations"><div class="citation"><div class="citation-content">Salvador N, Aguado C, Horst M, and Knecht E. 2000. Import of a cytosolic protein into lysosomes by chaperone-mediated autophagy depends on its folding state. <em>J. Biol. Chem.</em> 275:27447–27456.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0021-9258(19)61529-2">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10862611">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Import+of+a+cytosolic+protein+into+lysosomes+by+chaperone-mediated+autophagy+depends+on+its+folding+state&amp;author=N+Salvador&amp;author=C+Aguado&amp;author=M+Horst&amp;author=E+Knecht&amp;publication_year=2000&amp;pages=27447-27456&amp;doi=10.1016%2FS0021-9258%2819%2961529-2&amp;pmid=10862611">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">40.</div><div id="B40" class="citations"><div class="citation"><div class="citation-content">McNew JA and Goodman JM. 1994. An oligomeric protein is imported into peroxisomes in vivo. <em>J. Cell Biol.</em> 127:1245–1257.</div><div class="external-links"><div><a href="https://doi.org/10.1083/jcb.127.5.1245">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7962087">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=An+oligomeric+protein+is+imported+into+peroxisomes+in+vivo&amp;author=JA+McNew&amp;author=JM+Goodman&amp;publication_year=1994&amp;pages=1245-1257&amp;doi=10.1083%2Fjcb.127.5.1245&amp;pmid=7962087">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">41.</div><div id="B41" class="citations"><div class="citation"><div class="citation-content">Walton PA, Hill PE, and Subramani S. 1995. Import of stably folded proteins into peroxisomes. <em>Mol. Biol. Cell</em> 6:675–683.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.6.6.675">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7579687">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Import+of+stably+folded+proteins+into+peroxisomes&amp;author=PA+Walton&amp;author=PE+Hill&amp;author=S+Subramani&amp;publication_year=1995&amp;pages=675-683&amp;doi=10.1091%2Fmbc.6.6.675&amp;pmid=7579687">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">42.</div><div id="B42" class="citations"><div class="citation"><div class="citation-content">Wickner W and Schekman R. 2005. Protein translocation across biological membranes. <em>Science</em> 310:1452–1456.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1113752">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16322447">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000233756500033">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Protein+translocation+across+biological+membranes&amp;author=W+Wickner&amp;author=R+Schekman&amp;publication_year=2005&amp;pages=1452-1456&amp;doi=10.1126%2Fscience.1113752&amp;pmid=16322447">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">43.</div><div id="B43" class="citations"><div class="citation"><div class="citation-content">Lee VT and Schneewind O. 2002. Yop fusions to tightly folded protein domains and their effects on <span class="named-content" data-type="genus-species">Yersinia enterocolitica</span> type III secretion. <em>J. Bacteriol.</em> 184:3740–3745.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.184.13.3740-3745.2002">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12057971">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Yop+fusions+to+tightly+folded+protein+domains+and+their+effects+on+Yersinia+enterocolitica+type+III+secretion&amp;author=VT+Lee&amp;author=O+Schneewind&amp;publication_year=2002&amp;pages=3740-3745&amp;doi=10.1128%2FJB.184.13.3740-3745.2002&amp;pmid=12057971">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">44.</div><div id="B44" class="citations"><div class="citation"><div class="citation-content">Sorg JA, Miller NC, Marketon MM, and Schneewind O. 2005. Rejection of impassable substrates by <span class="named-content" data-type="genus-species">Yersinia</span> type III secretion machines. <em>J. Bacteriol.</em> 187:7090–7102.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.187.20.7090-7102.2005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16199580">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000232509500027">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rejection+of+impassable+substrates+by+Yersinia+type+III+secretion+machines&amp;author=JA+Sorg&amp;author=NC+Miller&amp;author=MM+Marketon&amp;author=O+Schneewind&amp;publication_year=2005&amp;pages=7090-7102&amp;doi=10.1128%2FJB.187.20.7090-7102.2005&amp;pmid=16199580">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">45.</div><div id="B45" class="citations"><div class="citation"><div class="citation-content">Akeda Y and Galan JE. 2005. Chaperone release and unfolding of substrates in type III secretion. <em>Nature</em> 437:911–915.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nature03992">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16208377">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000232338600052">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Chaperone+release+and+unfolding+of+substrates+in+type+III+secretion&amp;author=Y+Akeda&amp;author=JE+Galan&amp;publication_year=2005&amp;pages=911-915&amp;doi=10.1038%2Fnature03992&amp;pmid=16208377">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">46.</div><div id="B46" class="citations"><div class="citation"><div class="citation-content">Feldman MF, Muller S, Wuest E, and Cornelis GR. 2002. SycE allows secretion of YopE-DHFR hybrids by the <span class="named-content" data-type="genus-species">Yersinia enterocolitica</span> type III Ysc system. <em>Mol. Microbiol.</em> 46:1183–1197.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.2002.03241.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12421321">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000179400900023">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=SycE+allows+secretion+of+YopE-DHFR+hybrids+by+the+Yersinia+enterocolitica+type+III+Ysc+system&amp;author=MF+Feldman&amp;author=S+Muller&amp;author=E+Wuest&amp;author=GR+Cornelis&amp;publication_year=2002&amp;pages=1183-1197&amp;doi=10.1046%2Fj.1365-2958.2002.03241.x&amp;pmid=12421321">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">47.</div><div id="B47" class="citations"><div class="citation"><div class="citation-content">Fronzes R, Schafer E, Wang L, Saibil HR, Orlova EV, and Waksman G. 2009. Structure of a type IV secretion system core complex. <em>Science</em> 323:266–268.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1166101">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19131631">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000262290600042">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Structure+of+a+type+IV+secretion+system+core+complex&amp;author=R+Fronzes&amp;author=E+Schafer&amp;author=L+Wang&amp;author=HR+Saibil&amp;author=EV+Orlova&amp;author=G+Waksman&amp;publication_year=2009&amp;pages=266-268&amp;doi=10.1126%2Fscience.1166101&amp;pmid=19131631">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">48.</div><div id="B48" class="citations"><div class="citation"><div class="citation-content">Feeley JC, Gibson RJ, Gorman GW, Langford NC, Rasheed JK, Mackel DC, and Baine WB. 1979. Charcoal-yeast extract agar: primary isolation medium for <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>J. Clin. Microbiol.</em> 10:437–441.</div><div class="external-links"><div><a href="https://doi.org/10.1128/jcm.10.4.437-441.1979">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/393713">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1979HP33700008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Charcoal-yeast+extract+agar%3A+primary+isolation+medium+for+Legionella+pneumophila&amp;author=JC+Feeley&amp;author=RJ+Gibson&amp;author=GW+Gorman&amp;author=NC+Langford&amp;author=JK+Rasheed&amp;author=DC+Mackel&amp;author=WB+Baine&amp;publication_year=1979&amp;pages=437-441&amp;doi=10.1128%2Fjcm.10.4.437-441.1979&amp;pmid=393713">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">49.</div><div id="B49" class="citations"><div class="citation"><div class="citation-content">Berger KH and Isberg RR. 1994. Intracellular survival by <span class="named-content" data-type="genus-species">Legionella</span>. <em>Methods Cell. Biol.</em> 45:247–259.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0091-679X(08)61855-5">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7707989">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1994BD01Y00013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Intracellular+survival+by+Legionella&amp;author=KH+Berger&amp;author=RR+Isberg&amp;publication_year=1994&amp;pages=247-259&amp;doi=10.1016%2FS0091-679X%2808%2961855-5&amp;pmid=7707989">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">50.</div><div id="B50" class="citations"><div class="citation"><div class="citation-content">Coers J, Kagan JC, Matthews M, Nagai H, Zuckman DM, and Roy CR. 2000. Identification of Icm protein complexes that play distinct roles in the biogenesis of an organelle permissive for <span class="named-content" data-type="genus-species">Legionella pneumophila</span> intracellular growth. <em>Mol. Microbiol.</em> 38:719–736.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.2000.02176.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11115108">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+Icm+protein+complexes+that+play+distinct+roles+in+the+biogenesis+of+an+organelle+permissive+for+Legionella+pneumophila+intracellular+growth&amp;author=J+Coers&amp;author=JC+Kagan&amp;author=M+Matthews&amp;author=H+Nagai&amp;author=DM+Zuckman&amp;author=CR+Roy&amp;publication_year=2000&amp;pages=719-736&amp;doi=10.1046%2Fj.1365-2958.2000.02176.x&amp;pmid=11115108">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">51.</div><div id="B51" class="citations"><div class="citation"><div class="citation-content">Bardill JP, Miller JL, and Vogel JP. 2005. IcmS-dependent translocation of SdeA into macrophages by the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> type IV secretion system. <em>Mol. Microbiol.</em> 56:90–103.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2005.04539.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15773981">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000227499000008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=IcmS-dependent+translocation+of+SdeA+into+macrophages+by+the+Legionella+pneumophila+type+IV+secretion+system&amp;author=JP+Bardill&amp;author=JL+Miller&amp;author=JP+Vogel&amp;publication_year=2005&amp;pages=90-103&amp;doi=10.1111%2Fj.1365-2958.2005.04539.x&amp;pmid=15773981">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">52.</div><div id="B52" class="citations"><div class="citation"><div class="citation-content">Swanson MS and Isberg RR. 1995. Association of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> with the macrophage endoplasmic reticulum. <em>Infect. Immun.</em> 63:3609–3620.</div><div class="external-links"><div><a href="https://doi.org/10.1128/iai.63.9.3609-3620.1995">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7642298">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1995RQ79200056">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Association+of+Legionella+pneumophila+with+the+macrophage+endoplasmic+reticulum&amp;author=MS+Swanson&amp;author=RR+Isberg&amp;publication_year=1995&amp;pages=3609-3620&amp;doi=10.1128%2Fiai.63.9.3609-3620.1995&amp;pmid=7642298">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">53.</div><div id="B53" class="citations"><div class="citation"><div class="citation-content">Haug G, Wilde C, Leemhuis J, Meyer DK, Aktories K, and Barth H. 2003. Cellular uptake of <span class="named-content" data-type="genus-species">Clostridium botulinum</span> C2 toxin: membrane translocation of a fusion toxin requires unfolding of its dihydrofolate reductase domain. <em>Biochemistry</em> 42:15284–15291.</div><div class="external-links"><div><a href="https://doi.org/10.1021/bi0354278">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14690438">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Cellular+uptake+of+Clostridium+botulinum+C2+toxin%3A+membrane+translocation+of+a+fusion+toxin+requires+unfolding+of+its+dihydrofolate+reductase+domain&amp;author=G+Haug&amp;author=C+Wilde&amp;author=J+Leemhuis&amp;author=DK+Meyer&amp;author=K+Aktories&amp;author=H+Barth&amp;publication_year=2003&amp;pages=15284-15291&amp;doi=10.1021%2Fbi0354278&amp;pmid=14690438">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">54.</div><div id="B54" class="citations"><div class="citation"><div class="citation-content">Nagai H, Cambronne ED, Kagan JC, Amor JC, Kahn RA, and Roy CR. 2005. A C-terminal translocation signal required for Dot/Icm-dependent delivery of the <span class="named-content" data-type="genus-species">Legionella</span> RalF protein to host cells. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 102:826–831.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0406239101">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15613486">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000226436000055">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+C-terminal+translocation+signal+required+for+Dot%2FIcm-dependent+delivery+of+the+Legionella+RalF+protein+to+host+cells&amp;author=H+Nagai&amp;author=ED+Cambronne&amp;author=JC+Kagan&amp;author=JC+Amor&amp;author=RA+Kahn&amp;author=CR+Roy&amp;publication_year=2005&amp;pages=826-831&amp;doi=10.1073%2Fpnas.0406239101&amp;pmid=15613486">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">55.</div><div id="B55" class="citations"><div class="citation"><div class="citation-content">Derre I and Isberg RR. 2005. LidA, a translocated substrate of the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> type IV secretion system, interferes with the early secretory pathway. <em>Infect. Immun.</em> 73:4370–4380.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.73.7.4370-4380.2005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15972532">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000230151100066">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=LidA%2C+a+translocated+substrate+of+the+Legionella+pneumophila+type+IV+secretion+system%2C+interferes+with+the+early+secretory+pathway&amp;author=I+Derre&amp;author=RR+Isberg&amp;publication_year=2005&amp;pages=4370-4380&amp;doi=10.1128%2FIAI.73.7.4370-4380.2005&amp;pmid=15972532">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">56.</div><div id="B56" class="citations"><div class="citation"><div class="citation-content">Sory MP and Cornelis GR. 1994. Translocation of a hybrid YopE-adenylate cyclase from <span class="named-content" data-type="genus-species">Yersinia enterocolitica</span> into HeLa cells. <em>Mol. Microbiol.</em> 14:583–594.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.1994.tb02191.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7885236">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1994PP94700018">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Translocation+of+a+hybrid+YopE-adenylate+cyclase+from+Yersinia+enterocolitica+into+HeLa+cells&amp;author=MP+Sory&amp;author=GR+Cornelis&amp;publication_year=1994&amp;pages=583-594&amp;doi=10.1111%2Fj.1365-2958.1994.tb02191.x&amp;pmid=7885236">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">57.</div><div id="B57" class="citations"><div class="citation"><div class="citation-content">Williams MN, Poe M, Greenfield NJ, Hirshfield JM, and Hoogsteen K. 1973. Methotrexate binding to dihydrofolate reductase from a methotrexate-resistant strain of <span class="named-content" data-type="genus-species">Escherichia coli</span>. <em>J. Biol. Chem.</em> 248:6375–6379.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0021-9258(19)43456-X">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/4147159">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Methotrexate+binding+to+dihydrofolate+reductase+from+a+methotrexate-resistant+strain+of+Escherichia+coli&amp;author=MN+Williams&amp;author=M+Poe&amp;author=NJ+Greenfield&amp;author=JM+Hirshfield&amp;author=K+Hoogsteen&amp;publication_year=1973&amp;pages=6375-6379&amp;doi=10.1016%2FS0021-9258%2819%2943456-X&amp;pmid=4147159">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">58.</div><div id="B58" class="citations"><div class="citation"><div class="citation-content">Charpentier X, Gabay JE, Reyes M, Zhu JW, Weiss A, and Shuman HA. 2009. Chemical genetics reveals bacterial and host cell functions critical for type IV effector translocation by <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>PLoS Pathog.</em> 5:e1000501.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000501">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19578436">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269224500034">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Chemical+genetics+reveals+bacterial+and+host+cell+functions+critical+for+type+IV+effector+translocation+by+Legionella+pneumophila&amp;author=X+Charpentier&amp;author=JE+Gabay&amp;author=M+Reyes&amp;author=JW+Zhu&amp;author=A+Weiss&amp;author=HA+Shuman&amp;publication_year=2009&amp;pages=e1000501&amp;doi=10.1371%2Fjournal.ppat.1000501&amp;pmid=19578436">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">59.</div><div id="B59" class="citations"><div class="citation"><div class="citation-content">Johnsson N and Varshavsky A. 1994. Ubiquitin-assisted dissection of protein transport across membranes. <em>EMBO J.</em> 13:2686–2698.</div><div class="external-links"><div><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06559.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8013467">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1994NU20500023">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Ubiquitin-assisted+dissection+of+protein+transport+across+membranes&amp;author=N+Johnsson&amp;author=A+Varshavsky&amp;publication_year=1994&amp;pages=2686-2698&amp;doi=10.1002%2Fj.1460-2075.1994.tb06559.x&amp;pmid=8013467">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">60.</div><div id="B60" class="citations"><div class="citation"><div class="citation-content">Vijay-Kumar S, Bugg CE, and Cook WJ. 1987. Structure of ubiquitin refined at 1.8 Å resolution. <em>J. Mol. Biol.</em> 194:531–544.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0022-2836(87)90679-6">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3041007">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Structure+of+ubiquitin+refined+at+1.8+%E2%84%AB+resolution&amp;author=S+Vijay-Kumar&amp;author=CE+Bugg&amp;author=WJ+Cook&amp;publication_year=1987&amp;pages=531-544&amp;doi=10.1016%2F0022-2836%2887%2990679-6&amp;pmid=3041007">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">61.</div><div id="B61" class="citations"><div class="citation"><div class="citation-content">Catic A, Fiebiger E, Korbel GA, Blom D, Galardy PJ, and Ploegh HL. 2007. Screen for ISG15-crossreactive deubiquitinases. <em>PLoS One</em> 2:e679.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.pone.0000679">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17653289">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Screen+for+ISG15-crossreactive+deubiquitinases&amp;author=A+Catic&amp;author=E+Fiebiger&amp;author=GA+Korbel&amp;author=D+Blom&amp;author=PJ+Galardy&amp;author=HL+Ploegh&amp;publication_year=2007&amp;pages=e679&amp;doi=10.1371%2Fjournal.pone.0000679&amp;pmid=17653289">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">62.</div><div id="B62" class="citations"><div class="citation"><div class="citation-content">Johnson ES, Bartel B, Seufert W, and Varshavsky A. 1992. Ubiquitin as a degradation signal. <em>EMBO J.</em> 11:497–505.</div><div class="external-links"><div><a href="https://doi.org/10.1002/j.1460-2075.1992.tb05080.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/1311250">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1992HE19300015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Ubiquitin+as+a+degradation+signal&amp;author=ES+Johnson&amp;author=B+Bartel&amp;author=W+Seufert&amp;author=A+Varshavsky&amp;publication_year=1992&amp;pages=497-505&amp;doi=10.1002%2Fj.1460-2075.1992.tb05080.x&amp;pmid=1311250">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">63.</div><div id="B63" class="citations"><div class="citation"><div class="citation-content">Ninio S, Zuckman-Cholon DM, Cambronne ED, and Roy CR. 2005. The <span class="named-content" data-type="genus-species">Legionella</span> IcmS-IcmW protein complex is important for Dot/Icm-mediated protein translocation. <em>Mol. Microbiol.</em> 55:912–926.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2004.04435.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15661013">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+IcmS-IcmW+protein+complex+is+important+for+Dot%2FIcm-mediated+protein+translocation&amp;author=S+Ninio&amp;author=DM+Zuckman-Cholon&amp;author=ED+Cambronne&amp;author=CR+Roy&amp;publication_year=2005&amp;pages=912-926&amp;doi=10.1111%2Fj.1365-2958.2004.04435.x&amp;pmid=15661013">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">64.</div><div id="B64" class="citations"><div class="citation"><div class="citation-content">Vincent CD and Vogel JP. 2006. The <span class="named-content" data-type="genus-species">Legionella pneumophila</span> IcmS-LvgA protein complex is important for Dot/Icm-dependent intracellular growth. <em>Mol. Microbiol.</em> 61:596–613.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2006.05243.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16803597">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+pneumophila+IcmS-LvgA+protein+complex+is+important+for+Dot%2FIcm-dependent+intracellular+growth&amp;author=CD+Vincent&amp;author=JP+Vogel&amp;publication_year=2006&amp;pages=596-613&amp;doi=10.1111%2Fj.1365-2958.2006.05243.x&amp;pmid=16803597">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">65.</div><div id="B65" class="citations"><div class="citation"><div class="citation-content">Sorg JA, Blaylock B, and Schneewind O. 2006. Secretion signal recognition by YscN, the <span class="named-content" data-type="genus-species">Yersinia</span> type III secretion ATPase. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 103:16490–16495.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0605974103">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17050689">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000241879500076">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Secretion+signal+recognition+by+YscN%2C+the+Yersinia+type+III+secretion+ATPase&amp;author=JA+Sorg&amp;author=B+Blaylock&amp;author=O+Schneewind&amp;publication_year=2006&amp;pages=16490-16495&amp;doi=10.1073%2Fpnas.0605974103&amp;pmid=17050689">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">66.</div><div id="B66" class="citations"><div class="citation"><div class="citation-content">Stebbins CE and Galan JE. 2001. Maintenance of an unfolded polypeptide by a cognate chaperone in bacterial type III secretion. <em>Nature</em> 414:77–81.</div><div class="external-links"><div><a href="https://doi.org/10.1038/35102073">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11689946">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000171898900045">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Maintenance+of+an+unfolded+polypeptide+by+a+cognate+chaperone+in+bacterial+type+III+secretion&amp;author=CE+Stebbins&amp;author=JE+Galan&amp;publication_year=2001&amp;pages=77-81&amp;doi=10.1038%2F35102073&amp;pmid=11689946">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">67.</div><div id="B67" class="citations"><div class="citation"><div class="citation-content">Sutherland MC, Nguyen TL, Tseng V, and Vogel JP. 2012. The <span class="named-content" data-type="genus-species">Legionella</span> IcmSW complex directly interacts with DotL to mediate translocation of adaptor-dependent substrates. <em>PLoS Pathog.</em> 8:e1002910.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1002910">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/23028312">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000309816500017">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+IcmSW+complex+directly+interacts+with+DotL+to+mediate+translocation+of+adaptor-dependent+substrates&amp;author=MC+Sutherland&amp;author=TL+Nguyen&amp;author=V+Tseng&amp;author=JP+Vogel&amp;publication_year=2012&amp;pages=e1002910&amp;doi=10.1371%2Fjournal.ppat.1002910&amp;pmid=23028312">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">68.</div><div id="B68" class="citations"><div class="citation"><div class="citation-content">Buscher BA, Conover GM, Miller JL, Vogel SA, Meyers SN, Isberg RR, and Vogel JP. 2005. The DotL protein, a member of the TraG-coupling protein family, is essential for Viability of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> strain Lp02. <em>J. Bacteriol.</em> 187:2927–2938.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.187.9.2927-2938.2005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15838018">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000228633800001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+DotL+protein%2C+a+member+of+the+TraG-coupling+protein+family%2C+is+essential+for+Viability+of+Legionella+pneumophila+strain+Lp02&amp;author=BA+Buscher&amp;author=GM+Conover&amp;author=JL+Miller&amp;author=SA+Vogel&amp;author=SN+Meyers&amp;author=RR+Isberg&amp;author=JP+Vogel&amp;publication_year=2005&amp;pages=2927-2938&amp;doi=10.1128%2FJB.187.9.2927-2938.2005&amp;pmid=15838018">Google Scholar</a></div></div></div></div></div></section></div></section><div class="core-collateral"><div id="core-collateral-info" role="tabpanel"><header><h2><i class="icon-info" aria-hidden="true"></i>Information &amp; Contributors</h2></header><section id="tab-information" aria-labelledby="information" role="tabpanel"><h3>Information</h3><section class="core-self-citation"><h4>Published In</h4><div class="core-journal-presentation">



        
        <div class="cover-image"><div class="cover-image__image"><img src="/cms/asset/e7f36cfa-c21c-4778-a87f-864ae93eed43/iai.2013.81.issue-9.cover.gif" alt="Infection and Immunity cover image"/></div></div>
<div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">Infection and Immunity</span></div><div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">81</span></span> • <span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">9</span></span> • <span property="datePublished">September 2013</span></div><div class="core-pagination"><span class="heading">Pages</span>: <span class="content"><span property="pageStart">3239</span> - <span property="pageEnd">3252</span></span></div><div class="core-editors"><span class="heading">Editor</span>: <span role="list"><span role="listitem" property="editor" typeof="Person"><span property="givenName">A. J.</span> <span property="familyName">Bäumler</span></span></span></div><div class="core-pmid"><span class="heading">PubMed</span>: <a class="content" href="https://pubmed.ncbi.nlm.nih.gov/23798536/" property="sameAs">23798536</a></div></div></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">© 2013 American Society for Microbiology.</div></section><section class="core-history"><h4>History</h4><div>Received: 2 May 2013</div><div>Returned for modification: 2 June 2013</div><div>Accepted: 13 June 2013</div><div>Published online: 13 August 2013</div></section><section class="core-permissions"><h4>Permissions</h4><div>Request permissions for this article.</div><div><a href="https://marketplace.copyright.com/rs-ui-web/mp/search/all/10.1128%2FIAI.00552-13">Request Permissions</a></div></section><div class="core-download"><a href="/doi/pdf/10.1128/IAI.00552-13?download=true" class="btn btn-primary"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></div></section><section id="tab-contributors" aria-labelledby="contributors" role="tabpanel"><h3>Contributors</h3><section class="core-authors"><h4>Authors</h4><div id="con1" property="author" typeof="Person"><div class="heading"><span property="givenName">Whitney M.</span> <span property="familyName">Amyot</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Biology and Microbiology, Sackler School of Graduate Biomedical Science, Tufts University School of Medicine, Boston, Massachusetts, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Graduate Program in Molecular Microbiology, Sackler School of Graduate Biomedical Science, Tufts University School of Medicine, Boston, Massachusetts, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Amyot/Whitney+M">View all articles by this author</a></div></div></div><div id="con2" property="author" typeof="Person"><div class="heading"><span property="givenName">Dennise</span> <span property="familyName">deJesus</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Biology and Microbiology, Sackler School of Graduate Biomedical Science, Tufts University School of Medicine, Boston, Massachusetts, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Graduate Program in Molecular Microbiology, Sackler School of Graduate Biomedical Science, Tufts University School of Medicine, Boston, Massachusetts, USA</span></div></div><div class="core-author-link"><a href="/authored-by/deJesus/Dennise">View all articles by this author</a></div></div></div><div id="con3" property="author" typeof="Person"><div class="heading"><span property="givenName">Ralph R.</span> <span property="familyName">Isberg</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Sackler School of Graduate Biomedical Science, Tufts University School of Medicine, Boston, Massachusetts, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Biology and Microbiology, Sackler School of Graduate Biomedical Science, Tufts University School of Medicine, Boston, Massachusetts, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Isberg/Ralph+R">View all articles by this author</a></div></div></div></section><section class="core-editors"><h4>Editor</h4><div id="editor" property="editor" typeof="Person"><div class="heading"><span property="givenName">A. J.</span> <span property="familyName">Bäumler</span></div><div class="content"><div property="jobTitle">Editor</div></div></div></section><section class="core-authors-notes"><h4>Notes</h4><div role="doc-footnote">Address correspondence to Ralph R. Isberg, <a href="/cdn-cgi/l/email-protection#84f6e5e8f4ecaaedf7e6e1f6e3c4f0f1e2f0f7aae1e0f1"><span class="__cf_email__" data-cfemail="d6a4b7baa6bef8bfa5b4b3a4b196a2a3b0a2a5f8b3b2a3">[email&#160;protected]</span></a>.</div></section></section></div><div id="core-collateral-metrics" role="tabpanel"><header><h2><i class="icon-timeline" aria-hidden="true"></i>Metrics &amp; Citations</h2></header><section id="tab-metrics-inner" aria-labelledby="metrics-inner" role="tabpanel"><h3>Metrics</h3><section>



        
        







                <div class="sidebar-metric-container">
                        <input class="sidebar-metric__data" type="hidden" name="json" value='{"data":{"dataSections":[{"total":"116","months":[{"month":"2022-2-1","count":11},{"month":"2022-1-1","count":5},{"month":"2021-12-1","count":13},{"month":"2021-11-1","count":25},{"month":"2021-10-1","count":20},{"month":"2021-9-1","count":10},{"month":"2021-8-1","count":13},{"month":"2021-7-1","count":15},{"month":"2021-6-1","count":4}],"color":"color-4","firstMonth":[],"id":"download","title":"Download","startDate":"2021-6-1","displayDefault":true},{"total":"14","months":[{"month":"2022-2-1","count":0},{"month":"2022-1-1","count":0},{"month":"2021-12-1","count":0},{"month":"2021-11-1","count":0},{"month":"2021-10-1","count":0},{"month":"2021-9-1","count":2},{"month":"2021-8-1","count":0},{"month":"2021-7-1","count":0},{"month":"2021-6-1","count":0},{"month":"2021-5-1","count":1},{"month":"2021-4-1","count":0},{"month":"2021-3-1","count":0},{"month":"2021-2-1","count":0},{"month":"2021-1-1","count":0},{"month":"2020-12-1","count":2},{"month":"2020-11-1","count":0},{"month":"2020-10-1","count":0},{"month":"2020-9-1","count":0},{"month":"2020-8-1","count":0},{"month":"2020-7-1","count":0},{"month":"2020-6-1","count":0},{"month":"2020-5-1","count":0},{"month":"2020-4-1","count":0},{"month":"2020-3-1","count":0},{"month":"2020-2-1","count":0},{"month":"2020-1-1","count":0},{"month":"2019-12-1","count":0},{"month":"2019-11-1","count":0},{"month":"2019-10-1","count":0},{"month":"2019-9-1","count":0},{"month":"2019-8-1","count":0},{"month":"2019-7-1","count":0},{"month":"2019-6-1","count":0},{"month":"2019-5-1","count":2},{"month":"2019-4-1","count":0},{"month":"2019-3-1","count":0},{"month":"2019-2-1","count":1},{"month":"2019-1-1","count":0},{"month":"2018-12-1","count":0},{"month":"2018-11-1","count":0},{"month":"2018-10-1","count":0},{"month":"2018-9-1","count":0},{"month":"2018-8-1","count":0},{"month":"2018-7-1","count":0},{"month":"2018-6-1","count":0},{"month":"2018-5-1","count":0},{"month":"2018-4-1","count":0},{"month":"2018-3-1","count":1},{"month":"2018-2-1","count":0},{"month":"2018-1-1","count":2},{"month":"2017-12-1","count":0},{"month":"2017-11-1","count":0},{"month":"2017-10-1","count":0},{"month":"2017-9-1","count":0},{"month":"2017-8-1","count":0},{"month":"2017-7-1","count":0},{"month":"2017-6-1","count":0},{"month":"2017-5-1","count":0},{"month":"2017-4-1","count":0},{"month":"2017-3-1","count":0},{"month":"2017-2-1","count":0},{"month":"2017-1-1","count":0},{"month":"2016-12-1","count":0},{"month":"2016-11-1","count":0},{"month":"2016-10-1","count":0},{"month":"2016-9-1","count":0},{"month":"2016-8-1","count":0},{"month":"2016-7-1","count":0},{"month":"2016-6-1","count":0},{"month":"2016-5-1","count":0},{"month":"2016-4-1","count":0},{"month":"2016-3-1","count":0},{"month":"2016-2-1","count":1},{"month":"2016-1-1","count":0},{"month":"2015-12-1","count":0},{"month":"2015-11-1","count":0},{"month":"2015-10-1","count":0},{"month":"2015-9-1","count":0},{"month":"2015-8-1","count":0},{"month":"2015-7-1","count":0},{"month":"2015-6-1","count":0},{"month":"2015-5-1","count":0},{"month":"2015-4-1","count":1},{"month":"2015-3-1","count":0},{"month":"2015-2-1","count":0},{"month":"2015-1-1","count":0},{"month":"2014-12-1","count":0},{"month":"2014-11-1","count":0},{"month":"2014-10-1","count":0},{"month":"2014-9-1","count":1}],"color":"color-0","id":"citation","title":"Citation","startDate":"2014-9-1","displayDefault":true}],"doi":"10.1128/IAI.00552-13","widgetTitle":"Article Metrics"}}'>
                </div>
        
</section><section></section><section class="text-right"><a href="/doi/metrics/10.1128/IAI.00552-13" class="btn btn--inverse open-in-viewer"><span>VIEW ALL METRICS</span><i class="icon-open_in_new"></i></a></section></section><section id="tab-citations" aria-labelledby="citations" role="tabpanel"><h3>Citations</h3><section></section><section>



        
        <div class="citation-download">
	<p class="citation-msg">
		If you have the appropriate software installed, you can download article citation data to the citation manager of your choice. Simply select your manager software from the list below and click Download.
	</p>
	<!-- download options -->
	<form action="/action/downloadCitation" name="frmCitmgr" class="citation-form" method="post" target="_self"><input type="hidden" name="doi" value="10.1128/IAI.00552-13"><input type="hidden" name="downloadFileName" value="csp_81_"><input type="hidden" name="include" value="abs">
		<fieldset class="format-select">
			<label for="slct_format">Format</label>
			<div class="select-container">
				<select id="slct_format" name="format" class="js__slcInclude">
					<option value="" selected="selected">Please select one from the list</option>
					<option value="ris">RIS (ProCite, Reference Manager)</option>
					<option value="endnote">EndNote</option>
					<option value="bibtex">BibTex</option>
					<option value="medlars">Medlars</option>
					<option value="refworks">RefWorks</option>
				</select>
			</div>
			<label class="label-direct" for="direct">
				<input id="direct" type="checkbox" name="direct" value="" checked="checked">
				<span class="round-check"><span class="check"></span></span>
				Direct import
			</label>
		</fieldset>           
		<footer class="form-footer">
			<input onclick="onCitMgrSubmit()" class="formbutton" type="submit" name="submit" value="Download">
		</footer>
	</form>
</div>

</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel"><header><h2><i class="icon-eye" aria-hidden="true"></i>View Options</h2></header><div tabindex="0" class="section--wrapper"><h3>View options</h3><section><h4>ePub</h4><a href="https://journals.asm.org/doi/epub/10.1128/IAI.00552-13" class="btn"><i aria-hidden="true" class="icon-epub"></i><span>Open ePub</span></a></section><section><h4>PDF</h4><a href="/doi/pdf/10.1128/IAI.00552-13?download=true" download="true" class="btn btn--pdf"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></section><section><h4>Full Text</h4><a href="https://journals.asm.org/doi/full/10.1128/IAI.00552-13" class="btn"><i aria-hidden="true" class="icon-subject"></i><span>Open Full Text</span></a></section></div><!-- Its needed to duplicate this for PB check Pill.js as well--><div tabindex="-1" class="section--wrapper"><h3>Get Access</h3><section>



        
        












    <div class="purchaseArea pill__item" id="purchaseArea">
        <div class="login-options" data-pb-dropzone="login-options">
            
        </div>
        <div class="purchaseOptions">
            <div class="purchaseOptions--header">
                <h4 class="purchaseArea--section-heading">
                    PURCHASE OPTIONS
                </h4>
                
                        <div class="save-for-later">
                            <a data-bind="saveItem" data-doi="10.1128/IAI.00552-13" class="purchaseOptions--save" href="/action/saveItem?doi=10.1128/IAI.00552-13">
                                <i aria-hidden="true" class="icon-heart mx-1"></i>
                                <span>Save for later</span>
                            </a>
                            <a class="saved-go-cart hidden"  href="/action/showCart?FlowID=2"><span class="icon-cart-icon-material"></span> Item saved, go to cart </a>
                        </div>
                    
            </div>
            <p class="purchaseArea--section-info">
                Subscribe and get full access to this article
            </p>

            <div class="purchaseOptions--options-list">
                











<div class="purchase-options-container">
    

   
        <div class="savedItem-info hidden">
            <p></p>
        </div>
    

     
            
                
                
                        <div class="purchaseMessage hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00552-13-500044">
                            <p class="errorMsgBox"></p>
                        </div>
                        <div class="add-journal-to-cart">
                            <div class="purchaseOptions--option mb-3">
                                 
                                    <h5 class="option-title">Buy Article</h5>
                                    <span class="option-info mb-1">Infection and Immunity</span>
                                    <span class="option-info">Vol.81 &bull; Issue 9 &bull; 
                                            ASM Journals Pay Per View, PPV 25
                                        </span>
                                    <div class="option-actions mt-3 pt-3">
                                        <div class="option-price">
                                            <b>$25.00</b>
                                        </div>
                                        <a href="/action/addToCart?id=ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00552-13-500044" data-bind="addItem"
                                           data-key="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00552-13-500044" data-entity="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00552-13-500044"
                                           class="btn btn-primary btn--borderless">
                                            <i aria-hidden="true" class="mx-2 icon-shopping_cart"> </i>
                                            <span>ADD TO CART</span>
                                        </a>
                                    </div>
                                 
                            <div class="option-actions mt-3 pt-3 hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00552-13-500044" style="justify-content: flex-end">
                                <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                    <span>Checkout</span>
                                </a>
                            </div>
                        </div>

                        <div class="purchaseOptions--option mb-3">
                                                             <h5 class="option-title">Buy Article</h5>
                                <span class="option-info mb-1">Infection and Immunity</span>
                                <span class="option-info">Vol.81 &bull; Issue 9 &bull; 
                                        ASM Journals Pay Per View, PPV 25
                                    
                                </span>
                             
                             <div class="option-actions mt-3 pt-3 hidden"
                                     data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00552-13-500044" style="justify-content: flex-end">
                                    <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                        <span>Checkout</span>
                                    </a>
                             </div>
                        </div>
                        </div>
                    
            
        

    
</div>

                <div class="purchaseOptions--option">
                    <h5 class="option-title">Journal Subscription</h5>
                    <span class="option-info mb-1">Infection and Immunity</span>
                    <div class="option-actions mt-3 pt-3">
                        <div class="option-actions--info">ASM members can purchase subscriptions to journals.</div>
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FIAI.00552-13" redirectUri="/doi/10.1128/IAI.00552-13" class="btn btn-primary btn--borderless">
                            
                            <span>Join or renew</span>
                            <i aria-hidden="true" class="mx-2 icon-open_in_new"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>




</section></div></div><div id="core-collateral-media" role="tabpanel"><header><h2><i class="icon-photo" aria-hidden="true"></i>Figures and Media</h2></header><section id="tab-figures" aria-labelledby="figures" role="tabpanel"><h3>Figures</h3></section><section id="tab-other" aria-labelledby="other" role="tabpanel"><h3>Media</h3><section>



        
        
</section></section></div><div id="core-collateral-tables" role="tabpanel"><header><h2><i class="icon-tables" aria-hidden="true"></i>Tables</h2></header></div><div id="core-collateral-share" role="tabpanel"><header><h2><i class="icon-share" aria-hidden="true"></i>Share</h2></header><h3>Share</h3><div class="section--wrapper"><section><h4>Share the article link</h4><div id="share-self"><p><a href="#" target="_blank" class="share-self__source"></a></p><button class="btn btn--inverse share-self__action"><i aria-hidden="true" class="icon-copy-o"></i><span>Copy Link</span></button><div class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section><h4>Share with email</h4><p><a href="#" target="_blank" class="btn btn-primary share-email__source"><i class="icon-copy-o"></i><span>EMAIL A COLLEAGUE</span></a></p></section><section><h4>Share on social media</h4><div class="share-buttons"><a role="link" target="_blank" href="#" class="btn btn--facebook addthis_button_facebook"><i aria-hidden="true" class="icon-facebook-2"></i><span>facebook</span></a><a role="link" target="_blank" href="#" class="btn btn--twitter addthis_button_twitter"><i aria-hidden="true" class="icon-twitter"></i><span>twitter</span></a><a role="link" target="_blank" href="#" class="btn btn--linkedin addthis_button_linkedin"><i aria-hidden="true" class="icon-linkedin"></i><span>linkedin</span></a><a role="link" target="_blank" href="#" class="btn btn--email addthis_button_email"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a></div></section></div></div></div></article><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script id="contribution-meta" type="application/json">{
    "doi": "10.1128/IAI.00552-13",
    "toc": "/toc/iai/81/9",
    "issn": "0019-9567"
}</script><div class="article__side-ad hidden">



        
        <!-- /13678947/asm_box -->
<div class="boombox-wrapper">
	<span class="boombox-label">Advertisement</span>
<div id='div-gpt-ad-1594664702378-0' style='width: 300px; height: 250px;'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664702378-0'); });
  </script>
</div>
</div>
</div>




        
        <div class="article-footer-element">
    <div class="row">
        <div class="col-12">
            <section aria-label="we recommend" class="we-recommend">
               <div id="trendmd-suggestions"></div>
<script defer src='//js.trendmd.com/trendmd.min.js' data-trendmdconfig='{"journal_id":"35290", "element":"#trendmd-suggestions"}'></script>
            </section>
        </div>
    </div>
</div>










        <div data-widget-def="ux3-layout-widget" data-widget-id="9f4096e6-614d-46c7-a704-f706dfcdc172" class="related-tabs swipe-with-arrows">
        



        
        <div class="article-footer-element">









        <div data-widget-def="UX3Tabs" data-widget-id="3c1dc793-20ba-4336-a3d4-bb86a6fc2c43" class="tab  tab--flex" data-ctrl-res="screen-sm">
        



        
        <div data-ctrl-res="screen-md" class="tab"><ul role="tablist" class="tab__nav rlist"><li role="presentation" class="tab__nav__item  active "><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" aria-selected="true" class="tab__nav__item__link active"><span>Related Articles</span></a></li><li role="presentation" class="tab__nav__item"><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" aria-selected="false" class="tab__nav__item__link"><span>Related Articles From ASM</span></a></li></ul><div class="tab__content"><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" role="tabpanel" class="tab__pane active">



        
        <section class="related-articles"><ul class="rlist related-articles__items"><li class="related-articles__item"><a href="/doi/10.1128/IAI.02356-14" class="related-articles__item__title">Correction for Amyot et al., Poison Domains Block Transit of Translocated Substrates via the Legionella pneumophila Icm/Dot System</a><ul class="meta__authors rlist--inline loa mobile-authors" title="list of authors">
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Amyot%2C+Whitney+M" title="Whitney M. Amyot">Whitney M. Amyot</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=DeJesus%2C+Dennise" title="Dennise deJesus">Dennise deJesus</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Isberg%2C+Ralph+R" title="Ralph R. Isberg">Ralph R. Isberg</a></span></li>
</ul>
</li></ul></section>
</div><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" role="tabpanel" class="tab__pane">



        
        <!--content loaded via ajax-->
<div class="asmj-org general-rss-feed-reader aog" id="80036a5d-0f41-4828-9cd1-19399fe2560a" data-rss-url="https://asm.org/rss/content/">
    <div class="rss-body"></div>
</div>
</div></div></div>

        </div>
    
</div>

        </div>
    




        
        
</div>
</main>




        
        <footer class="footer"><div class="footer__wrapper container container--lg"><div class="footer__top row"><div class="sitemap"><div class="sitemap__column sitemap__column--media col-xs-12"><div class="footer__logos">



        
        <a href="https://asm.org/" title="American Society for Microbiology logo"><img id="" alt="American Society for Microbiology logo" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title="ASM Journals logo"><img id="" alt="ASM Journals logo" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div class="footer__social-media">



        
        <div class="footer__social-media">
    <ul class="rlist--inline">
        <li><a href="https://www.facebook.com/asmfan" target="_blank" rel="noopener"><i class="icon-facebook"></i></a></li>
        <li><a href="https://twitter.com/asmicrobiology" target="_blank" rel="noopener"><i class="icon-twitter"></i></a></li>
        <li><a href="https://www.linkedin.com/companies/45861" target="_blank" rel="noopener"><i class="icon-linkedin"></i></a></li>
        <li><a href="https://www.youtube.com/channel/UCBxWzf49gVHz0Ksgw_N8brw" target="_blank" rel="noopener"><i class="icon-youtube"></i></a></li>
    </ul>
</div>
</div></div><div class="sitemap__column sitemap__column--journals col-xs-12">



        
        <a href="#" data-toggle-target-id="footerJournals"><h6 class="footer__list-title">JOURNALS</h6></a>
<ul id="footerJournals" class="rlist footer__list footer__list--columns">
	<li><a href="/journal/aac">Antimicrobial Agents and Chemotherapy</a></li>
	<li><a href="/journal/aem">Applied and Environmental Microbiology</a></li>
	<li><a href="/journal/cvi">Clinical and Vaccine Immunology</a></li>
	<li><a href="/journal/cmr">Clinical Microbiology Reviews</a></li>
	<li><a href="/journal/ecosalplus">EcoSal Plus</a></li>
	<li><a href="/journal/ec">Eukaryotic Cell</a></li>
	<li><a href="/journal/iai">Infection and Immunity</a></li>
	<li><a href="/journal/jb">Journal of Bacteriology</a></li>
	<li><a href="/journal/jcm">Journal of Clinical Microbiology</a></li>
	<li><a href="/journal/jmbe">Journal of Microbiology &amp; Biology Education</a></li>
	<li><a href="/journal/jvi">Journal of Virology</a></li>
	<li><a href="/journal/mbio">mBio</a></li>
	<li><a href="/journal/mmbr">Microbiology and Molecular Biology Reviews</a></li>
	<li><a href="/journal/mra">Microbiology Resource Announcements</a></li>
	<li><a href="/journal/spectrum">Microbiology Spectrum</a></li>
	<li><a href="/journal/mcb">Molecular and Cellular Biology</a></li>
	<li><a href="/journal/msphere">mSphere</a></li>
	<li><a href="/journal/msystems">mSystems</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--connect col-xs-12">



        
        <ul class="rlist footer__list">
    <li><h6>RESOURCES</h6></li>
    <li><a href="/about-asm-journals" target="_blank" rel="noopener">About Us</a></li>
    <li><a href="/announcements" target="_blank" rel="noopener">Announcements</a></li>
    <li><a href="/asm-author-center" target="_blank" rel="noopener">Author Center</a></li>
    <li><a href="/collection" target="_blank" rel="noopener">ASM Journal Collections</a></li>
     <li><a href="/publishing-ethics" target="_blank" rel="noopener">Publishing Ethics</a></li>
    <li><a href="/migration-information" target="_blank" rel="noopener">Migration Information</a></li>
    <li><a href="/advertising" target="_blank" rel="noopener">Advertising</a></li>
    <li><a href="https://asm.org/Membership/Home" target="_blank" rel="noopener">ASM Membership</a></li>
    <li><a href="https://asm.org/cookie-policy" target="_blank" rel="noopener">Privacy Policy</a></li>
    <li><a href="/terms" target="_blank" rel="noopener">Terms &amp; Conditions</a></li>
    <li><a href="/proprietary-rights" target="_blank" rel="noopener">Proprietary Rights Notices</a></li>
    <li><a href="/non-commercial-tdm-license" target="_blank" rel="noopener">Non-Commercial TDM License</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--news col-xs-12">



        
        <a href="#" data-toggle-target-id="footerConnected"><h6 class="footer__list-title">STAY CONNECTED</h6></a>
<ul id="footerConnected" class="rlist footer__list">
    <li><a href="/action/showPreferences?menuTab=Alerts" target="_blank" rel="noopener">Alerts</a></li>
    <li><a href="/asm-journals-rss-feeds" target="_blank" rel="noopener">RSS Feeds</a></li>
    <li><a href="/contact-us" target="_blank" rel="noopener">Contact Us</a></li>
    <li><a href=" https://connect.asm.org/ " target="_blank" rel="noopener">ASM Communities</a></li>
    
</ul>
</div><div class="sitemap__column sitemap__column--other col-xs-12 ">



        
        <h6>NEWS</h6>
<ul class="rlist footer__list">
    <li><a href="https://asm.org/newsroom" target="_blank" rel="noopener">Press Releases</a></li>
    <li><a href="https://asm.org/Browse-By-Content-Type/Policy-Statement" target="_blank" rel="noopener">Policy Statements</a></li>
    
</ul>
</div></div></div><div class="footer__bottom row"><div class="col-xs-12"><div class="footer__copyright">



        
        Copyright © 2022 American Society for Microbiology
</div></div></div></div></footer>
</div>
        </div>
    </div>



        
        

            <script src="/products/asmj/releasedAssets/js/build.lazyload.bundle-9eae693cc49bd405a4bc.js"></script>

        <script src="/products/asmj/releasedAssets/js/main.bundle-f0f6c9d7da4e7fd5b03e.js" ></script>
        

















<script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"6e7ac9eadbf17ea1","token":"edc08be6f1e64730992376d7e16c0615","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>
</body>
</html>
