
<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="6e44ae25-636f-4881-9f74-ddee62288ca4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";page:string:Article/Chapter View;ctype:string:Journal Content;requestedJournal:journal:iai;website:website:asmj;issue:issue:doi\:10.1128/iai.2011.79.issue-5;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:doi\:10.1128/IAI.00805-10;journal:journal:iai"/>
<link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"></link><meta name="citation_journal_title" content="Infection and Immunity"></meta><meta name="dc.Title" content="Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of Legionella pneumophila"></meta><meta name="dc.Creator" content="Eva Hervet"></meta><meta name="dc.Creator" content="Xavier Charpentier"></meta><meta name="dc.Creator" content="Anne Vianney"></meta><meta name="dc.Creator" content="Jean-Claude Lazzaroni"></meta><meta name="dc.Creator" content="Christophe Gilbert"></meta><meta name="dc.Creator" content="Danièle Atlan"></meta><meta name="dc.Creator" content="Patricia Doublet"></meta><meta name="dc.Description" content="ABSTRACT Legionella pneumophila is the etiological agent of Legionnaires' disease. Crucial to the pathogenesis of this intracellular pathogen is its ability to subvert host cell defenses, permittin..."></meta><meta name="Description" content="ABSTRACT Legionella pneumophila is the etiological agent of Legionnaires' disease. Crucial to the pathogenesis of this intracellular pathogen is its ability to subvert host cell defenses, permittin..."></meta><meta name="dc.Publisher" content="&#xA;American Society for Microbiology&#xA;1752 N St., N.W., Washington, DC&#xA;"></meta><meta name="dc.Contributor" content="J. B. Bliska"></meta><meta name="dc.Date" scheme="WTN8601" content="2011-2-14"></meta><meta name="dc.Type" content="research-article"></meta><meta name="dc.Format" content="text/HTML"></meta><meta name="dc.Identifier" scheme="publisher-id" content="0805-10"></meta><meta name="dc.Identifier" scheme="doi" content="10.1128/IAI.00805-10"></meta><meta name="dc.Language" content="EN"></meta><meta name="dc.Coverage" content="1752 N St., N.W., Washington, DC"></meta><meta name="dc.Rights" content="Copyright © 2011, American Society for Microbiology"></meta><!-- ${citation_fulltext_world_readable:10.1128/IAI.00805-10} -->




<link rel="meta" type="application/atom+xml" href="https://doi.org/10.1128%2FIAI.00805-10"></link>
<link rel="meta" type="application/rdf+json" href="https://doi.org/10.1128%2FIAI.00805-10"></link>
<link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1128%2FIAI.00805-10"></link>



<meta charset="UTF-8">
















        <meta property="og:title" content="Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of Legionella pneumophila | Infection and Immunity"/>
    


        <meta property="og:type" content="Article"/>
    


        <meta property="og:url" content="https://journals.asm.org/doi/abs/10.1128/IAI.00805-10"/>
    

        <meta property="og:image" content="https://journals.asm.org/cms/asset/353df96d-410c-47e2-bebe-342ca3a4f2d8/iai.2011.79.issue-5.cover.gif" />
        <meta property="og:image:width" content="113" />
        <meta property="og:image:height" content="150" />
    


        <meta property="og:site_name" content="Infection and Immunity"/>
    


        <meta property="og:description" content="Legionella pneumophila is the etiological agent of Legionnaires' disease. Crucial to the pathogenesis of
this intracellular pathogen is its ability to subvert host cell defenses, permitting
intracellular replication in specialized vacuoles within ..."/>
    
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=0"/>
<title>Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of Legionella pneumophila | Infection and Immunity</title>










            <link rel="stylesheet" type="text/css" href="/wro/l0of~article-metrics-phase3.css">
        
<link rel="stylesheet" type="text/css" href="/wro/l0of~product.css">



    
    
                <link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-6742698955b5929bab3a.css" media="all" id="build-style-sheet">
                <script>
                    function isIE() { //won't check for edge browser
                        var ua = navigator.userAgent;
                        /* MSIE used to detect old browsers and Trident used to newer ones*/
                        var is_ie = ua.indexOf("MSIE ") > -1 || ua.indexOf("Trident/") > -1;
          
                        return is_ie; 
                    }
                    
                    if(isIE()) {
                        document.write('<link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-style-fallback-0e2f8eb0a5d072bb5fa3.css" media="all" />')
                        var buildCss = document.getElementById('build-style-sheet');
                        buildCss.parentNode.removeChild(buildCss);
                    }
                </script>
        <link rel="stylesheet" href="/products/asmj/releasedAssets/css/print-6742698955b5929bab3a.css"media="print">
    
        <script>
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    





<meta http-equiv="X-UA-Compatible" content="IE=edge">
<style>
:root{
	--journal-image: url('/pb-assets/images/Homepage Header Images/header-iai-desktop-1602282256940.png');
	--journal-mobile-image: url("/pb-assets/images/Homepage Header Images/header-iai-mobile-1605057095637.png");
	--landing-image: url("/pb-assets/images/Homepage Header Images/header-global-desktop-1603901826210.png");
	--landing-mobile-image: url("/pb-assets/images/Homepage Header Images/header-global-mobile-1605057223667.png");


}
</style>
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/Journals-Atypon-1641917642317.css">
<script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
<script>
	window.googletag = window.googletag || {cmd: []};
	googletag.cmd.push(function() {
    	googletag.defineSlot('/13678947/asm_leader-mob', [[728, 90], [320, 50]], 'div-gpt-ad-1594664599282-0').addService(googletag.pubads());
    	googletag.defineSlot('/13678947/asm_box', [300, 250], 'div-gpt-ad-1594664702378-0').addService(googletag.pubads());	
		googletag.pubads().enableSingleRequest();
		
			
				googletag.pubads().setTargeting('asmjournals', 'iai');
			
		
		googletag.enableServices();
	});
</script>



                    <link rel="canonical" href="https://journals.asm.org/doi/10.1128/IAI.00805-10">
                





    <script src="//scholar.google.com/scholar_js/casa.js" async></script>








    
    

    
    
    
            <script type="text/javascript" src="/wro/l0of~article-metrics-phase2.js"></script>
        


    <!-- Google Tag Manager -->
    













<script>
    var dataLayer = dataLayer ||[];
    
    dataLayer.push({"event":"pageview","page_domain":"journals.asm.org","page_path":"/doi/10.1128/IAI.00805-10","page_name":"Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of Legionella pneumophila | Infection and Immunity","page_type":"article","article_title":"Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of Legionella pneumophila","article_type":"Research Article","article_category":"Molecular Pathogenesis","article_doi":"10.1128/IAI.00805-10","article_access":"free","article_received_date":"23 July 2010","article_accepted_date":"03 February 2011","article_published_online_date":"19 April 2011","article_issue":"volume 79, issue 5","article_issue_date":"May 2011","journal":"Infection and Immunity","user_type":"Guest","basic_groups":["All users group","Unknown Users","All users no webcrawlers"],"smart_groups":["Institution- IP authentication"]});
    
</script>


    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MTLLM76');</script>
    <!-- End Google Tag Manager -->

</head>
<body class="pb-ui">


    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MTLLM76" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->




















<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>





    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">




        
        <div>



        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input type="hidden" name="requestUsername" value="requestUsername" />
                <div class="input-group">
                    <div class="label">
                        <label for="5a26c991-102b-4ded-a542-61c81c149bd5.email">Email*</label>
                    </div>
                    <input id="5a26c991-102b-4ded-a542-61c81c149bd5.email" class="email" type="text" name="email" value="" size="15" />
                </div>
                
                    <div class="username-recaptcha-ajax"></div>
                
                <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" />
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        





<div class="popup registration-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Create a new account</h2>
        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.00805-10" />
            <div class="input-group">
                <div class="label">
                    <label for="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email">Email</label>
                </div>
                <input id="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email" class="email" type="email" name="email" value="" />
            </div>
            <div class="submit">
                <input class="button submit primary" type="submit" value="Register" />
            </div></form>
        <div class="center">
            <a href="/action/showLogin" class="show-login">
                
                Returning user
            </a>
        </div>
    </div>
</div>




        
        



<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="form legacy-form">
            






<h2>Change Password </h2>
<form action="/action/changePassword" method="post"><div class="message error"></div>
    <input type="hidden" name="submit" value="submit" />
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-old">Old Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-old" class="old" type="password" name="old" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
    </div>
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-new">New Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-new" class="pass-hint new" type="password" name="new" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
        
            







    


    


    


<div class="password-strength-indicator" data-min="8" data-max="20" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
        
    </div>
    <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
        </div>
        <div class="success-template hidden">
            <h2>Password Changed Successfully</h2>
            <p>Your password has been changed</p>
        </div>
    </div>
</div>





        
        















<div class="popup login-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        

        <div class="login-form-container ">
            <div class="tab tab--flex">
                <ul class="rlist tab__nav" role="tablist">
                    <li class="active" role="presentation">
                        <a class="active " id="pane-m1x01con" href="#pane-m1x01" aria-controls="pane-m1x01" role="tab"
                           data-toggle="tab" title="login" aria-selected="true" data-simple-tab-id=""
                           aria-expanded="true">
                            <span>Login</span>
                        </a>
                    </li>
                    <li role="presentation">
                        <a id="pane-m1x11con" href="#pane-m1x11" aria-controls="pane-m1x11" role="tab"
                           data-toggle="tab" title="register" aria-selected="false">
                            <span>Register</span>
                        </a>
                    </li>
                </ul>
                <ul class="rlist tab__content">
                    <li class="tab__pane active" id="pane-m1x01" aria-labelledby="pane-m1x01con" role="tabpanel">
                        




























<div class="login-form">
    
    <form action="/action/doLogin" method="post"><input type="hidden" name="id" value="016a54f6-925d-4632-a5f6-22977f0dce11" />
        





        <input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.00805-10" />
    

<input type="hidden" name="loginUri" value="/doi/10.1128/IAI.00805-10" />
        
            <input type="hidden" name="popup" value="true" />
        
        
                

























    
        
    
    
        
    
    
    
        
    

        

        
    <div class="input-group">
        <div class="label ">

            
                    <label for="login">Email*</label>
                
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="login" class="login" type="text" name="login" value="" size="15" placeholder="Enter your email address" /><div class="icon-mail"></div></div>
        
                    <div class="actions">
                        
                    </div>
                
    </div>

                





















    



    
    
    <div class="input-group">
        <div class="label ">
            <label for="password">Password*</label>
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="password" class="password" type="password" name="password" value="" autocomplete="off" placeholder="Enter your password" /><div class="icon-lock"></div></div>
        <span class="password-eye-icon icon-eye hidden"></span>
        
                    <div class="actions reset-password">
                        
                        
                        <i class="icon-lock"></i>
                        <span>
                            Forgot password?
                            Reset it <a href="/action/requestResetPassword" class="link show-request-reset-password uppercase styled-link">here</a>
                        </span>
                    </div>
                
    </div>

            

        

        <div class="remember">
            <label class="checkbox--primary remember-me" for="016a54f6-925d-4632-a5f6-22977f0dce11-remember">
                <input id="016a54f6-925d-4632-a5f6-22977f0dce11-remember" class="cmn-toggle cmn-toggle-round-flat" type="checkbox" name="remember" value="true" />
                <span class="label-txt">Keep me logged in</span>
            </label>
        </div>
        
                <div class="submit-section">
                    <input class="button submit primary" type="submit" name="submit" value="Login" />
                    <span class="required-info">Fields with * are mandatory</span>
                </div>
            
        

























        

        <div class="login-tabs-footer">
            <div class="actions">
                
                
                <span>
                    Don&apos;t have an account? Create one <a href="/action/registration" class="uppercase styled-link">here</a>
                </span>
            </div>
        </div></form>
</div>
                    </li>
                    <li class="tab__pane" id="pane-m1x11" aria-labelledby="pane-m1x11con" role="tabpanel">
                        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.00805-10" />
                            <div class="input-group">
                                <div class="label">
                                    <label for="016a54f6-925d-4632-a5f6-22977f0dce11.email">Email*</label>
                                </div>
                                <div class="iconable__input">
                                    <input id="016a54f6-925d-4632-a5f6-22977f0dce11.email" class="email" type="email" name="email" value="" placeholder="Enter your email address" />
                                    <div class="icon-mail"></div>
                                </div>
                            </div>
                            <div class="submit-section">
                                <input class="button submit primary" type="submit" value="Register" />
                                <span class="required-info">Fields with * are mandatory</span>
                            </div>
                            <div class="login-tabs-footer">
                                <div class="actions">
                                    
                                    
                                    <span>
                                        Already have an account? Login <a href="/action/showLogin" class="uppercase styled-link">here</a>
                                    </span>
                                </div>
                            </div></form>
                    </li>
                </ul>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <a href="#" class="close" tabindex="0"><i class="icon-close_thin"></i></a>
        <div class="form">
            





<p>Can't sign in? Forgot your password?</p>
<p class="sub">Enter your email address below and we will send you the reset instructions</p>
<div class="message error"></div>
<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input type="hidden" name="requestResetPassword" value="true" />
    
    <div class="input-group">
        <div class="label">
            <label for="7660a830-973e-446c-89b4-6e6ce315b3e0.email">Email*</label>
        </div>
        <input id="7660a830-973e-446c-89b4-6e6ce315b3e0.email" class="email" type="text" name="email" value="" size="15" />
    </div>
    
        








    

    
    
    
    

    
            <div class="clearfix"></div>
<div class="label required">
    <label for="captchaCode" class="prompt" style="white-space: nowrap">Retype the code from the picture:</label>
</div>
<link type="text/css" rel="stylesheet" href="/botdetectcaptcha?get=layoutStyleSheet" />

<div class="LBD_CaptchaDiv" id="sampleCaptcha" style="width: 280px !important; height: 50px !important"><!--
   --><div class="LBD_CaptchaImageDiv" id="sampleCaptcha_CaptchaImageDiv" style="width: 250px !important; height: 50px !important"><!--
   --><a target="_blank" href="http://captcha.com/java-captcha-info.html" onclick="sampleCaptcha.OnHelpLinkClick(); return sampleCaptcha.FollowHelpLink;"><img class="LBD_CaptchaImage" id="sampleCaptcha_CaptchaImage" src="/botdetectcaptcha?get=image&amp;c=sampleCaptcha&amp;t=8244ad78b6434cd99733625d8dd19ecc" alt="CAPTCHA" /></a></div><!--
   --><div class="LBD_CaptchaIconsDiv" id="sampleCaptcha_CaptchaIconsDiv" style="width: 24px !important;"><!--
   --><a class="LBD_ReloadLink" id="sampleCaptcha_ReloadLink" href="#" onclick="sampleCaptcha.ReloadImage(); this.blur(); return false;" title="Change the CAPTCHA code"> <img class="LBD_ReloadIcon" id="sampleCaptcha_ReloadIcon" src="/botdetectcaptcha?get=ReloadIcon" alt="Change the CAPTCHA code" /></a><!--
   --><a id="sampleCaptcha_SoundLink" class="LBD_SoundLink" href="/botdetectcaptcha?get=sound&amp;c=sampleCaptcha&amp;t=8244ad78b6434cd99733625d8dd19ecc" onclick="sampleCaptcha.PlaySound(); this.blur(); return false;" title="Speak the CAPTCHA code"> <img class="LBD_SoundIcon" id="sampleCaptcha_SoundIcon" src="/botdetectcaptcha?get=SoundIcon" alt="Speak the CAPTCHA code" /></a><!--
   --><div class="LBD_Placeholder" id="sampleCaptcha_AudioPlaceholder">&nbsp;</div><!--
   --></div>

<script type="text/javascript" src="/botdetectcaptcha?get=clientScriptInclude"></script>
<script type="text/javascript">
    //<![CDATA[
     BotDetect.Init('sampleCaptcha', '8244ad78b6434cd99733625d8dd19ecc', 'captchaCode', true, true, true, true, 1200, 7200, 0);
    //]]>
    </script>


<script type="text/javascript">
    //<![CDATA[
      try{(function(){var bdrsn = document.createElement('script'); bdrsn.type = 'text/javascript'; bdrsn.async = true; bdrsn.src = document.location.protocol + '//remote.captcha.com/include.js?i=ATABMwExATMBMAJBMhMRP3E_PygHKT8_Pz9AbXw_Yj8i'; var fsn = document.getElementsByTagName('script')[0]; fsn.parentNode.insertBefore(bdrsn, fsn);})();} catch(err){}
    //]]>
    </script>

   <input type="hidden" id="LBD_VCID_sampleCaptcha" name="LBD_VCID_sampleCaptcha" value="8244ad78b6434cd99733625d8dd19ecc" />
</div>

<input type="hidden" value="0" id="LBD_BackWorkaround_sampleCaptcha">
<div class="validationDiv">
    <input type="hidden" name="loginBotCaptcha" value="">
    <input id="captchaCode" type="text" name="captchaCode"/>
</div>
        

    

    

    
    <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" /></form>
            
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <p>If the address matches an existing account you will receive an email with instructions to reset your password.</p>
            <div class="center">
                <a href="#" class="cancel">Close</a>
            </div>
        </div>
    </div>
</div>




        
        




<div class="top-drawer verify-phone-drawer hidden">
    <div class="content">
        <div class="form">
            <h2>Verify Phone</h2>
            <form action="/action/verifyPhoneNumber" method="post"><input type="hidden" name="submit" value="submit" />
                <div class="message error"></div>
                <div class="input-group">
                    <div class="label">
                        <label for="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code">Enter the verification code</label>
                    </div>
                    <input id="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code" class="verificationCode" type="text" name="verificationCode" value="" size="15" />
                </div>
                <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <h2>Congrats!</h2>
            <p>Your Phone has been verified</p>
            <div class="center">
                <a href="/action/showPreferences" class="button primary submit">close</a>
            </div>
        </div>
    </div>
</div>





        
        <div id="search-overlay" class="header__search-overlay search-overlay js--hidden"><div class="search-overlay__close"><a href="#" title="Close search panel" data-hide-target-id="search-overlay" class="search-overlay__close__btn"><span>CLOSE </span><i class="icon-close"></i></a></div><div class="container">



        
        <div class="quick-search quick-search--all"><div class="full-width quickSearchFormContainer"><div class="tab"><div class="quick-search__searchbox"><ul role="tablist" class="rlist tab__nav dropblock--tab"><li role="presentation" class="tab__nav__item active"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-controls="pane-0" role="tab" data-toggle="tab" title="This Journal" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" aria-expanded="false" class="tab__nav__item__link">This Journal</a></li><li role="presentation" class="tab__nav__item"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-controls="pane-1" role="tab" data-toggle="tab" title="Anywhere" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" aria-expanded="false" class="tab__nav__item__link">Anywhere</a></li></ul><div class="quick-search__search-items tab__content"><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-0 option-journal"><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Search here within this Journal" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/><input type="hidden" name="SeriesKey" value="iai"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-1 "><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Enter words / phrases / DOI / ISBN / keywords / authors / etc" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div>



        
        <div class="quick-search__links">
    <a href="/search/advanced" class="quick-search__advanced-link">Advanced search</a>
</div>
</div></div></div></div></div>
<div class="search-overlay__autocomplete"><div id="autocompleted-terms"></div></div><div class="search-overlay__suggestions"><div class="search-overlay__suggestions__title">Suggested Terms:</div><ul class="rlist search-overlay__suggestions__list">



        
        <li><a href="/asm-author-center" class="btn btn-secondary">author information</a></li>
<li><a href="/migration-information" class="btn btn-secondary">migration information</a></li>
<li><a href="/publication-fees" class="btn btn-secondary">publication fees</a></li>
<li><a href="/asm-covid-19" class="btn btn-secondary">COVID-19 collection</a></li>
<li><a href="/collection" class="btn btn-secondary">asm journals collections</a></li>
</ul></div></div></div><header class="header header--journal header--journal-page"><div class="header__row-1">



        
        <div class="journals-menu">
	<a data-toggle-target-id="journals-menu-content" href="#" title="Journals Menu" class="journals-menu__trigger js--target-hidden">
		<i class="icon-dehaze"></i>
		<i class="icon-close"></i>
	</a>
	<div id="journals-menu-content" class="journals-menu__content js--hidden" aria-expanded="true">
		<a href="#" data-hide-target-id="journals-menu-content" class="journals-menu__back">
			<i class="icon-arrow_left"></i>
			<span>BACK</span>
		</a>
		<ul class="rlist journals-menu__list">
			<li><a href="/journal/aac"><img src="/pb-assets/images/thumbnails/thumb-aac-1602006917653.jpg" alt="Antimicrobial Agents and Chemotherapy"><span>Antimicrobial Agents and Chemotherapy</span></a></li>
			<li><a href="/journal/aem"><img src="/pb-assets/images/thumbnails/thumb-aem-1602006917677.jpg" alt="Applied and Environmental Microbiology"><span>Applied and Environmental Microbiology</span></a></li>
			<li><a href="/journal/cvi"><img src="/pb-assets/images/thumbnails/thumb-cvi-1602006922190.jpg" alt="Clinical and Vaccine Immunology"><span>Clinical and Vaccine Immunology</span></a></li>
			<li><a href="/journal/cmr"><img src="/pb-assets/images/thumbnails/thumb-cmr-1602006918000.jpg" alt="Clinical Microbiology Reviews"><span>Clinical Microbiology Reviews</span></a></li>
			<li><a href="/journal/ecosalplus"><img src="/pb-assets/images/thumbnails/thumb-ecosalplus-1602678711627.jpg" alt="EcoSal Plus"><span>EcoSal Plus</span></a></li>
			<li><a href="/journal/ec"><img src="/pb-assets/images/thumbnails/thumb-ec-1602006922537.jpg" alt="Eukaryotic Cell"><span>Eukaryotic Cell</span></a></li>
			<li><a href="/journal/iai"><img src="/pb-assets/images/thumbnails/thumb-iai-1602006922193.jpg" alt="Infection and Immunity"><span>Infection and Immunity</span></a></li>
			<li><a href="/journal/jb"><img src="/pb-assets/images/thumbnails/thumb-jb-1602006930407.jpg" alt="Journal of Bacteriology"><span>Journal of Bacteriology</span></a></li>
			<li><a href="/journal/jcm"><img src="/pb-assets/images/thumbnails/thumb-jcm-1602006930870.jpg" alt="Journal of Clinical Microbiology"><span>Journal of Clinical Microbiology</span></a></li>
			<li><a href="/journal/jmbe"><img src="/pb-assets/images/thumbnails/thumb-jmbe-1606273579503.jpg" alt="Journal of Microbiology &amp; Biology Education"><span>Journal of Microbiology &amp; Biology Education</span></a></li>
			<li><a href="/journal/jvi"><img src="/pb-assets/images/thumbnails/thumb-jvi-1602006937397.jpg" alt="Journal of Virology"><span>Journal of Virology</span></a></li>
			<li><a href="/journal/mbio"><img src="/pb-assets/images/thumbnails/thumb-mbio-1602006937397.jpg" alt="mBio"><span>mBio</span></a></li>
			<li><a href="/journal/mmbr"><img src="/pb-assets/images/thumbnails/thumb-mmbr-1602006945363.jpg" alt="Microbiology and Molecular Biology Reviews"><span>Microbiology and Molecular Biology Reviews</span></a></li>
			<li><a href="/journal/mra"><img src="/pb-assets/images/thumbnails/thumb-mra-1602006946427.jpg" alt="Microbiology Resource Announcements"><span>Microbiology Resource Announcements</span></a></li>
			<li><a href="/journal/spectrum"><img src="/pb-assets/images/thumbnails/thumb-spectrum-1602006955410.jpg" alt="Microbiology Spectrum"><span>Microbiology Spectrum</span></a></li>
			<li><a href="/journal/mcb"><img src="/pb-assets/images/thumbnails/thumb-mcb-1602006937413.jpg" alt="Molecular and Cellular Biology"><span>Molecular and Cellular Biology</span></a></li>
			<li><a href="/journal/msphere"><img src="/pb-assets/images/thumbnails/thumb-msphere-1602006946427.jpg" alt="mSphere"><span>mSphere</span></a></li>
			<li><a href="/journal/msystems"><img src="/pb-assets/images/thumbnails/thumb-msystems-1602006955407.jpg" alt="mSystems"><span>mSystems</span></a></li>
		</ul>
	</div>
</div>
<div class="header__logos">



        
        <a href="https://asm.org/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div data-truncate="1" class="header__feed mx-3 js--truncate"></div><div class="header__attribution"></div><div class="header__cart">



        
        















<div class="eCommerceCartIndicatorWidget">
    
            <a href="/action/showCart?FlowID=1&amp;aria-label=Shopping+Cart" id="cartLabel" class="quick-menu__item" aria-label="Shopping Cart">
                <div class="cartLabel" title="Cart">
                    <i class="icon-shopping_cart" aria-hidden="true"></i>
                    <span class="filter-count filter-count-bubble shopping-cart hidden"
                          data-id="cart-size">0</span>
                        
                </div>
            </a>
        
</div>
</div><div class="header__login">



        
        




    








        




    













<div class="user-login-bar">
    <div title="Sign in" class="loginBar user-login-bar">
        <div class="user-login-bar">
            <div class="loginBar">
                <div class="loginBar__login">
                    <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FIAI.00805-10&amp;data-toggle-target-id=loginOptions" class="btn btn-primary btn--light btn--darker-content btn--borderless" redirectUri="/doi/10.1128/IAI.00805-10" data-toggle-target-id="loginOptions">
                        Login
                    </a>
                    <div id="loginOptions" class="loginBar__login__options">
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FIAI.00805-10">
                            
                            <img src="/specs/products/asmj/releasedAssets/images/logo-textless.svg" class="mr-2">
                            <span>ASM Member Login</span>
                            <i class="icon-open_in_new" aria-hidden="true"></i>
                        </a>
                        <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FIAI.00805-10">
                            <i class="ml-1 mr-2 icon-account_circle"></i>
                            Non-Member Login
                        </a>
                        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1128%2FIAI.00805-10">
                            <i class="ml-1 mr-2 icon-institution"></i>
                            Institutional Login
                        </a>
                        <a href="/action/registration?redirectUri=%2Fdoi%2F10.1128%2FIAI.00805-10" class="btn btn-secondary btn--inverse btn--borderless btn--darker-content btn-register" title="Register">
                            Register
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>



    

</div>



        
        <a href="#" data-show-target-id="search-overlay" class="hidden-md hidden-sm hidden-xs search-trigger js--target-shown"><i title="Open Search Panel" class="icon-search"></i></a>
</div><div class="header__row-2"><a data-toggle-target-id="responsive-menu" href="#" title="Journals Menu" class="header__mobile-trigger"><i class="icon-dehaze"></i><i class="icon-close"></i></a><div class="header__logos">



        
        <a href="/journal/iai" title="Infection and Immunity logo"><img id="" alt="Infection and Immunity logo" src="/pb-assets/images/pub-nav/IAI-logo-1601672963217.png"/></a>
</div><div class="hidden-sm hidden-xs hidden-md">



        
        <a href="#afterNav-a5t" class="sr-only-focusable">Skip main navigation</a><nav class="main-nav"><ul id="menubar" role="menubar" class="menubar rlist--inline"><li role="menuitem" aria-label="JOURNAL HOME" id="menu-item-pub_nav-0" class="menu-item is--current"><a href="/journal/iai"><span>JOURNAL HOME</span></a></li><li role="menuitem" aria-haspopup="true" aria-label="ARTICLES" id="menu-item-pub_nav-1" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-1" data-toggle="dropdown" class="dropdown__toggle"><span>ARTICLES</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-1" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="CURRENT ISSUE" class="menu-item is--current"><a href="/toc/iai/current"><span>CURRENT ISSUE</span></a></li><li role="menuitem" tabindex="-1" aria-label="ACCEPTED MANUSCRIPTS" class="menu-item is--current"><a href="/toc/iai/0/ja"><span>ACCEPTED MANUSCRIPTS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="COLLECTIONS" class="menu-item is--current"><a href="/journal/iai/collections"><span>COLLECTIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARCHIVE" class="menu-item is--current"><a href="/loi/iai"><span>ARCHIVE</span></a></li></ul></li><li role="menuitem" id="menu-item-pub_nav-2" class="menu-item">



        
        <li role="menuitem" aria-haspopup="true" aria-label="FOR AUTHORS" id="menu-item-pub_nav-custom" class="dropdown menu-parent is--current">
	<a href="#" id="pub_nav-custom" data-toggle="dropdown" class="dropdown__toggle"><span>FOR AUTHORS</span></a>
	<ul aria-labelledby="pub_nav-custom" role="menu" class="rlist dropdown__menu">
		<li role="menuitem" tabindex="-1" aria-label="SUBMITTING" class="menu-item is--current">
			<a href="/submitting?journal=iai"><span>SUBMITTING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="WRITING YOUR PAPER" class="menu-item is--current">
			<a href="/writing-your-paper?journal=iai"><span>WRITING YOUR PAPER</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="TRANSFERRING" class="menu-item is--current">
			<a href="/transferring?journal=iai"><span>TRANSFERRING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="ACCEPTED PAPERS" class="menu-item is--current">
			<a href="/accepted-papers?journal=iai"><span>ACCEPTED PAPERS</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="EDITORIAL POLICIES" class="menu-item is--current">
			<a href="/editorial-policies?journal=iai"><span>EDITORIAL POLICIES</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="PUBLISHING ETHICS" class="menu-item is--current">
			<a href="/publishing-ethics?journal=iai"><span>PUBLISHING ETHICS</span></a>
		</li>
	</ul>
</li>

</li><li role="menuitem" aria-haspopup="true" aria-label="ABOUT THE JOURNAL" id="menu-item-pub_nav-3" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-2" data-toggle="dropdown" class="dropdown__toggle"><span>ABOUT THE JOURNAL</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-2" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="ABOUT IAI" class="menu-item is--current"><a href="/journal/iai/about"><span>ABOUT IAI</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMIT" class="menu-item is--current"><a href="/journal/iai/submit"><span>SUBMIT</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="SCOPE" class="menu-item is--current"><a href="/journal/iai/scope"><span>SCOPE</span></a></li><li role="menuitem" tabindex="-1" aria-label="EDITORIAL BOARD" class="menu-item is--current"><a href="/journal/iai/board-editors"><span>EDITORIAL BOARD</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARTICLE TYPES" class="menu-item is--current"><a href="/journal/iai/article-types"><span>ARTICLE TYPES</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMISSION AND REVIEW PROCESS" class="menu-item is--current"><a href="/journal/iai/submission-review-process"><span>SUBMISSION AND REVIEW PROCESS</span></a></li><li role="menuitem" tabindex="-1" aria-label="EBM SUBMISSIONS" class="menu-item is--current"><a href="/journal/iai/ebm-submission"><span>EBM SUBMISSIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FOR REVIEWERS" class="menu-item is--current"><a href="/reviewers"><span>FOR REVIEWERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FAQ" class="menu-item is--current"><a href="/journal/iai/faq"><span>FAQ</span></a></li><li role="menuitem" tabindex="-1" aria-label="CONTACT US" class="menu-item is--current"><a href="/journal/iai/contact-us"><span>CONTACT US</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="SUBSCRIBE" id="menu-item-pub_nav-4" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-3" data-toggle="dropdown" class="dropdown__toggle"><span>SUBSCRIBE</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-3" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="MEMBERS" class="menu-item is--current"><a href="/members"><span>MEMBERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="INSTITUTIONS" class="menu-item is--current"><a href="/institutions"><span>INSTITUTIONS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="ALERTS" class="menu-item is--current"><a href="/action/showPreferences?menuTab=Alerts"><span>ALERTS</span></a></li><li role="menuitem" tabindex="-1" aria-label="RSS" class="menu-item is--current"><a href="/asm-journals-rss-feeds"><span>RSS</span></a></li></ul></li></ul><span id="afterNav-a5t" tabindex="-1"></span></nav>
</div><a href="#" data-show-target-id="search-overlay" class="hidden-lg search-trigger"><i title="Open Search Panel" class="icon-search"></i></a></div></header>




        
        <main class="content article-page">



        
        <div>









        <div data-widget-def="ux3-layout-widget" data-widget-id="62435c6d-56e5-45d8-8726-1c5a8298a1a6" class="article-container">
        



        
        <div class="mb-3">



        
        <ol aria-label="Breadcrumb" role="navigation" vocab="https://schema.org/" typeof="BreadcrumbList" class="breadcrumbs rlist--inline"><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="https://journals.asm.org/"><span property="name">ASM Journals</span></a><meta property="position" content="1"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/journal/iai"><span property="name">Infection and Immunity</span></a><meta property="position" content="2"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/toc/iai/79/5"><span property="name">Vol. 79, No. 5</span></a><meta property="position" content="3"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/doi/10.1128/IAI.00805-10" aria-current="page" class="breadcrumbs__current"><span property="name">Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of <named-content content-type='genus-species' xlink:type='simple'>Legionella pneumophila</named-content></span></a><meta property="position" content="4"/></li></ol>










        <div data-widget-def="general-html-asset" data-widget-id="b28fb475-91bf-4257-b86f-8120484c7481" class="container--center-content">
        



        
        <!-- /13678947/asm_leader-mob -->
<div class="leaderboard-wrapper">
<span class="leaderboard-label">Advertisement</span>
<div id='div-gpt-ad-1594664599282-0'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664599282-0'); });
  </script>
</div>
</div>

        </div>
    
</div>

        </div>
    




        
        <link rel="stylesheet" type="text/css" href="/specs/products/asmj/releasedAssets/css/build-article.css"/><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle"><section id="frontmatter" data-extent="frontmatter"><header><div class="core-container"><div data-article-access="free" data-article-access-type="limited" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__type">Research Article</div><div class="meta-panel__publication-data">19 April 2011</div></div><div class="meta-panel__right-content"><div class="meta-panel__share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share__block share__inline-links"><span class="sr-only">Share on</span><ul class="rlist--inline addthis addthis_toolbox addthis_default_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper  icon-facebook"></i></a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i></a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper  icon-linkedin"></i></a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper  icon-mail"></i></a></li></ul></div>
</div></div></div><h1 property="name">Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of <span class="named-content" data-type="genus-species">Legionella pneumophila</span></h1><div class="core-relations my-3"><div class="relation mb-3"><div class="relation--head"><i class="icon-info"></i><div>This article has been corrected.</div><a href="/doi/10.1128/IAI.00587-15">VIEW CORRECTION</a></div></div></div><div class="contributors"><span class="authors"><span class="heading">Authors</span>: <span role="list"><span property="author" typeof="Person" role="listitem"><a href="#con1"><span property="givenName">Eva</span> <span property="familyName">Hervet</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con2"><span property="givenName">Xavier</span> <span property="familyName">Charpentier</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con3"><span property="givenName">Anne</span> <span property="familyName">Vianney</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con4"><span property="givenName">Jean-Claude</span> <span property="familyName">Lazzaroni</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con5"><span property="givenName">Christophe</span> <span property="familyName">Gilbert</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con6"><span property="givenName">Danièle</span> <span property="familyName">Atlan</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con7"><span property="givenName">Patricia</span> <span property="familyName">Doublet</span></a> <a href="/cdn-cgi/l/email-protection#bfcfdecbcdd6dcd6de91dbd0caddd3dacbffcad1d6c992d3c6d0d18e91d9cd" property="email"><span class="__cf_email__" data-cfemail="d6a6b7a2a4bfb5bfb7f8b2b9a3b4bab3a296a3b8bfa0fbbaafb9b8e7f8b0a4">[email&#160;protected]</span></a></span></span></span></div><div class="self-citation"><span class="heading">DOI</span>: <a href="https://doi.org/10.1128/IAI.00805-10" property="sameAs">https://doi.org/10.1128/IAI.00805-10</a><span>&nbsp;&#8226;&nbsp;</span><span class="crossmark"><a data-target="crossmark" href="#" data-doi="10.1128/IAI.00805-10" class="crossmark__link"><img alt="Check for updates on crossmark" src="/specs/products/asmj/releasedAssets/images/cross-mark.png" class="crossmark__logo"/></a></span></div><div class="info-panel"><div class="info-panel__metrics info-panel__item">



        
        







        <div class="toolbar-metric-container">
            <input class="toolbar-metric__data" type="hidden" name="json" value='{"data":{"menuSections":[{"total":"158","id":"downloads","title":"downloads"},{"total":"57","id":"citations","title":"citations"}],"doi":"10.1128/IAI.00805-10"},"thisWidget":{"properties":{"identifier":"","headingLvl":"h2","wrapperClass":"","citationMetricId":"2","downloadMetricId":"1","headingClass":"","ux3/template":"","wrapperTag":"div","title":"","linkToFullPage":""}}}'>
        </div>
    

</div><div class="info-panel__favorite info-panel__item">



        
        <div class="article-tools"><a href="/action/addCitationAlert?doi=10.1128%2FIAI.00805-10" target="_blank" title="Track Citations" class="article-tools__citation btn btn--slim btn-secondary btn--inverse btn--transparent"><i class="icon-alert-bell"></i></a></div>
</div><div class="info-panel__alerts info-panel__item"></div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" title="View Citations" class="btn btn--slim btn--inverse"><i aria-hidden="true" class="icon-format_quote"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://journals.asm.org/doi/full/10.1128/IAI.00805-10" class="btn btn--slim btn--inverse"><span>Full Text</span></a><a href="/doi/reader/10.1128/IAI.00805-10" class="btn btn--slim btn--pdf"><span>PDF/EPUB</span></a></div></div></div></header><nav data-core-nav="article"><header><div class="core-self-citation">



        
        <a href="/journal/iai" class="sections-navbar__title">IAI</a>
<div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">79</span></span><span class="delimiter">, </span><span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">5</span></span><br /><span property="datePublished">May 2011</span></div></div><nav></nav></header><ul xmlns="http://www.w3.org/1999/xhtml"><li><a href="#abstract">ABSTRACT</a></li><li><a href="#sec-1">INTRODUCTION</a></li><li><a href="#sec-2">MATERIALS AND METHODS</a></li><li><a href="#sec-3">RESULTS</a></li><li><a href="#sec-4">DISCUSSION</a></li><li><a href="#acknowledgments">ACKNOWLEDGMENTS</a></li><li><a href="#bibliography">REFERENCES</a></li></ul><footer></footer></nav><div id="abstracts"><div class="core-container"><section id="abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">ABSTRACT</h2><div role="paragraph"><span class="named-content" data-type="genus-species">Legionella pneumophila</span> is the etiological agent of Legionnaires' disease. Crucial to the pathogenesis of this intracellular pathogen is its ability to subvert host cell defenses, permitting intracellular replication in specialized vacuoles within host cells. The Dot/Icm type IV secretion system (T4SS), which translocates a large number of bacterial effectors into host cell, is absolutely required for rerouting the <span class="named-content" data-type="genus-species">Legionella</span> phagosome. Many <span class="named-content" data-type="genus-species">Legionella</span> effectors display distinctive eukaryotic domains, among which are protein kinase domains. <span class="named-content" data-type="genus-species">In silico</span> analysis and <i>in vitro</i> phosphorylation assays identified five functional protein kinases, LegK1 to LegK5, encoded by the epidemic <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens strain. Except for LegK5, the <span class="named-content" data-type="genus-species">Legionella</span> protein kinases are all T4SS effectors. LegK2 plays a key role in bacterial virulence, as demonstrated by gene inactivation. The <i>legK2</i> mutant containing vacuoles displays less-efficient recruitment of endoplasmic reticulum markers, which results in delayed intracellular replication. Considering that a kinase-dead substitution mutant of <i>legK2</i> exhibits the same virulence defects, we highlight here a new molecular mechanism, namely, protein phosphorylation, developed by <span class="named-content" data-type="genus-species">L. pneumophila</span> to establish a replicative niche and evade host cell defenses.</div></section></div></div></section><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1" data-type="intro"><h2>INTRODUCTION</h2><div role="paragraph"><span class="named-content" data-type="genus-species">Legionella pneumophila</span> is the most common causative agent of the atypical and severe pneumonia legionellosis. Pathogenic <span class="named-content" data-type="genus-species">Legionella</span> strains emerge from the environment after intracellular multiplication in phagocytic protozoans. It has been proposed that its ability to exploit the basic cellular mechanisms of numerous protozoal hosts also enables <span class="named-content" data-type="genus-species">Legionella</span> to infect human macrophages of alveolar lungs and consequently to cause disease (<a href="#B29" role="doc-biblioref" data-xml-rid="B29">29</a>, <a href="#B63" role="doc-biblioref" data-xml-rid="B63">63</a>).</div><div role="paragraph">Mechanisms of infection of protozoans and macrophages by <span class="named-content" data-type="genus-species">L. pneumophila</span> are quite similar (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>). Upon internalization of <span class="named-content" data-type="genus-species">L. pneumophila</span>, the bacteria containing phagosome avoids fusion with lysosomes and recruits mitochondria and endoplasmic reticulum (ER) to establish a replication niche. The Dot/Icm type IV secretion system (T4SS), which translocates bacterial proteins called “effectors” into host cell, is absolutely required to reprogram the endosomal-lysosomal degradation pathway of the phagocytic cell and to lead to bacterial intracellular survival and successful replication (<a href="#B3" role="doc-biblioref" data-xml-rid="B3">3</a>, <a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>, <a href="#B64" role="doc-biblioref" data-xml-rid="B64">64</a>, <a href="#B67" role="doc-biblioref" data-xml-rid="B67">67</a>). Multiple approaches have successfully identified more than 100 effector proteins (reviewed by reference <a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a>). Many <span class="named-content" data-type="genus-species">Legionella</span> effectors display distinctive eukaryotic domains, such as those involved in protein-protein interactions (e.g., ankyrin repeats, leucine-rich repeats, and coiled coils), and domains with more defined activities (protein kinases, ubiquitin ligases, sphingosine-1-phosphate lyases, phospholipases, and guanine nucleotide exchange factors). In spite of the fact that <span class="named-content" data-type="genus-species">L. pneumophila</span> seems well equipped with proteins adapted to interfere with a wide variety of host cell processes, only a minority of these effectors have been functionally characterized (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>, <a href="#B18" role="doc-biblioref" data-xml-rid="B18">18</a>, <a href="#B21" role="doc-biblioref" data-xml-rid="B21">21</a>, <a href="#B30" role="doc-biblioref" data-xml-rid="B30">30</a>, <a href="#B40" role="doc-biblioref" data-xml-rid="B40">40</a>, <a href="#B44" role="doc-biblioref" data-xml-rid="B44">44</a>, <a href="#B48" role="doc-biblioref" data-xml-rid="B48">48</a>, <a href="#B49" role="doc-biblioref" data-xml-rid="B49">49</a>, <a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>, <a href="#B56" role="doc-biblioref" data-xml-rid="B56">56</a>, <a href="#B61" role="doc-biblioref" data-xml-rid="B61">61</a>, <a href="#B72" role="doc-biblioref" data-xml-rid="B72">72</a>, <a href="#B73" role="doc-biblioref" data-xml-rid="B73">73</a>).</div><div role="paragraph">The genomes of the five sequenced <span class="named-content" data-type="genus-species">L. pneumophila</span> strains—Philadelphia, Lens, Paris, Corby, and Alcoy—have been reported to encode four eukaryote-like serine/threonine kinases (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>, <a href="#B15" role="doc-biblioref" data-xml-rid="B15">15</a>, <a href="#B16" role="doc-biblioref" data-xml-rid="B16">16</a>, <a href="#B26" role="doc-biblioref" data-xml-rid="B26">26</a>). Phosphorylation-dephosphorylation of proteins at serine/threonine/tyrosine residues represents a powerful regulatory mechanism of cellular activity. Indeed, intensive research has revealed that eukaryotes contain numerous interconnected signal transduction networks in which protein phosphorylation plays a dominant role. For many years, however, phosphorylation at serine/threonine/tyrosine has been considered a new addition to the cell's regulatory arsenal and believed to be exclusive to eukaryotes. Due in large part to genomic sequencing programs, some “classical” serine/threonine/tyrosine protein kinases have been identified in several bacterial genomes, but the function of most of them remains unknown (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>, <a href="#B19" role="doc-biblioref" data-xml-rid="B19">19</a>). Interestingly, some of these enzymes have been recently described to play a critical role in virulence of several pathogenic bacteria: PknG of <span class="named-content" data-type="genus-species">Mycobacterium tuberculosis</span> (<a href="#B71" role="doc-biblioref" data-xml-rid="B71">71</a>), YpkA of <span class="named-content" data-type="genus-species">Yersinia pseudotuberculosis</span> (<a href="#B4" role="doc-biblioref" data-xml-rid="B4">4</a>), and SteC of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium (<a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a>) are translocated into the host cell cytoplasm to modulate eukaryotic signal transduction to the bacteria advantage and thus contribute to virulence. Moreover, StkP of <span class="named-content" data-type="genus-species">Streptococcus pneumoniae</span> is a global regulator of gene expression that positively controls virulence for lung infection and bloodstream invasion (<a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a>, <a href="#B66" role="doc-biblioref" data-xml-rid="B66">66</a>), SP-STK of <span class="named-content" data-type="genus-species">Streptococcus pyogenes</span> has been reported to have pleiotropic effect, including the expression of major virulence factors (<a href="#B32" role="doc-biblioref" data-xml-rid="B32">32</a>), and PrkC of <span class="named-content" data-type="genus-species">Enterococcus faecalis</span> mediates antimicrobial resistance and intestinal persistence (<a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a>).</div><div role="paragraph">We investigated the role of bacterial protein kinases in the virulence of <span class="named-content" data-type="genus-species">L. pneumophila</span>. We report that the five putative protein kinases identified by <i>in silico</i> analysis of the <span class="named-content" data-type="genus-species">L. pneumophila</span> genomes are functional in terms of phosphorylation; actually, they can autophosphorylate and undergo <i>in vitro</i> phosphorylation of classical eukaryotic protein kinase substrates. Interestingly, we demonstrate that the protein kinase LegK2 plays a key role in the virulence of <span class="named-content" data-type="genus-species">Legionella</span> toward amoebae. More precisely, we observed that the <i>legK2</i> mutant is impaired in ER recruitment and intracellular replication. Since LegK2 is translocated into the host cell cytoplasm during infection, we assume that this protein kinase would interfere with host cell signal transduction pathways to subvert host cell defenses to the bacteria benefit, thus resulting in an effective infectious cycle and virulence.</div></section><section id="sec-2" data-type="materials methods"><h2>MATERIALS AND METHODS</h2><section id="sec-2-1"><h3>Growth of bacteria and phagocytes.</h3><div role="paragraph">The bacterial strains and plasmids used in the present study are summarized in <a href="#T1">Table 1</a>.</div><div class="figure-wrap"><header><div class="label">Table 1.</div></header><figure id="T1" class="table"><figcaption><span class="heading">Table 1</span>. Strains and plasmids used in this study</figcaption><div class="table-wrap"><table><thead data-xml-valign="bottom"><tr data-xml-align="center"><th>Strain or plasmid</th><th>Relevant properties</th><th>Source or reference</th></tr></thead><tbody data-xml-valign="top"><tr data-xml-align="left"><td>Strains</td><td> </td><td> </td></tr><tr data-xml-align="left"><td>    <span class="named-content" data-type="genus-species">E. coli</span></td><td> </td><td> </td></tr><tr data-xml-align="left"><td>        XL1-Blue</td><td><i>endA1 gyrA96</i>(Nal<sup>r</sup>) <i>thi-1 recA1 relA1 lac glnV44</i> F′[::Tn<i>10 proAB<sup>+</sup></i> <i>lacI</i><sup>q</sup> <i>Δ</i>(<i>lacZ</i>)M15] <i>hsdR17</i>(r<sub>K</sub><sup>+</sup> m<sub>K</sub><sup>−</sup>)</td><td>Stratagene</td></tr><tr data-xml-align="left"><td>        BL21(DE3)(pREP4-<i>groESL</i>)</td><td>F<sup>−</sup> <i>ompT gal dcm lon hsdS</i><i><sub>B</sub></i>(r<sub>B</sub><sup>−</sup> m<sub>B</sub><sup>−</sup>) <i>λ</i>(DE3 [<i>lacI lacUV5-T7</i> gene 1 <i>ind1 sam7 nin5</i>])</td><td><a href="#B2" role="doc-biblioref" data-xml-rid="B2">2</a></td></tr><tr data-xml-align="left"><td>    <span class="named-content" data-type="genus-species">L. pneumophila</span></td><td> </td><td> </td></tr><tr data-xml-align="left"><td>    CIP 108286</td><td>Virulent <span class="named-content" data-type="genus-species">L. pneumophila</span> serogroup 1, strain Lens</td><td><a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a></td></tr><tr data-xml-align="left"><td>        Δ<i>dotA</i> mutant</td><td>Lens <i>lpl2613</i>::Km</td><td><a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a></td></tr><tr data-xml-align="left"><td>        Δ<i>legK1</i> mutant</td><td>Lens <i>lpl1545</i>::Km</td><td>This study</td></tr><tr data-xml-align="left"><td>        Δ<i>legK2</i> mutant</td><td>Lens <i>lpl2066</i>::Km</td><td>This study</td></tr><tr data-xml-align="left"><td>        Δ<i>legK3</i> mutant</td><td>Lens <i>lpl2481</i>::Km</td><td>This study</td></tr><tr data-xml-align="left"><td>        Δ<i>legK4</i> mutant</td><td>Lens <i>lpl0262</i>::Km</td><td>This study</td></tr><tr data-xml-align="left"><td>        Δ<i>legK5</i> mutant</td><td>Lens <i>lpl2476</i>::Km</td><td>This study</td></tr><tr data-xml-align="left"><td>    <span class="named-content" data-type="genus-species">A. castellanii</span></td><td>Environmental isolate</td><td>P. Pernin, Faculty of Pharmacy, Université Lyon 1</td></tr><tr data-xml-align="left"><td>    <span class="named-content" data-type="genus-species">D. discoideum</span></td><td> </td><td> </td></tr><tr data-xml-align="left"><td>        DBS0302388</td><td>Wild-type DH1 strain</td><td><a href="#B12" role="doc-biblioref" data-xml-rid="B12">12</a></td></tr><tr data-xml-align="left"><td>        DBS0236184</td><td>[act15]:<i>cnxA</i>:GFP</td><td><a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a></td></tr><tr data-xml-align="left"><td>        Dd03</td><td>[act15]:<i>legK2</i>(<i>K112M</i>):c-<i>myc</i></td><td>This study</td></tr><tr data-xml-align="left"><td>        J774A.1</td><td> </td><td>ATCC TIB-67</td></tr><tr data-xml-align="left"><td>Plasmids</td><td> </td><td> </td></tr><tr data-xml-align="left"><td>    pGEX-6P-3</td><td><span class="named-content" data-type="genus-species">E. coli</span> expression vector</td><td><a href="#B69" role="doc-biblioref" data-xml-rid="B69">69</a></td></tr><tr data-xml-align="left"><td>    pGEX-<i>legK1</i></td><td><i>lpl1545</i> inserted in BamHI-SalI of pGEX-6P-3 for GST-LegK1 overproduction</td><td>This study</td></tr><tr data-xml-align="left"><td>    pGEX-<i>legK2</i></td><td><i>lpl2066</i> inserted in BamHI-SalI of pGEX-6P-3 for GST-LegK2 overproduction</td><td>This study</td></tr><tr data-xml-align="left"><td>    pGEX-<i>legK3</i></td><td><i>lpl2481</i> inserted in BamHI-SalI of pGEX-6P-3 for GST-LegK3 overproduction</td><td>This study</td></tr><tr data-xml-align="left"><td>    pQE30</td><td><span class="named-content" data-type="genus-species">E. coli</span> expression vector</td><td>Qiagen</td></tr><tr data-xml-align="left"><td>    pQE30-<i>legK4</i></td><td><i>lpl0262</i> inserted in BamHI-SalI of pQE30 for 6His-LegK4 overproduction</td><td>This study</td></tr><tr data-xml-align="left"><td>    pQE30-<i>legK5</i></td><td><i>lpl2476</i> inserted in PstI-SacI of pQE30 for 6His-LegK5 overproduction</td><td>This study</td></tr><tr data-xml-align="left"><td>    pKD13</td><td>Used for amplification of <i>Km</i></td><td><a href="#B14" role="doc-biblioref" data-xml-rid="B14">14</a></td></tr><tr data-xml-align="left"><td>    pAV695</td><td><i>sacB</i>, OriT OriV, <i>cml</i> (pCDP05 with a deletion of a 4.3-kb fragment)</td><td>Modified from reference <a href="#B60" role="doc-biblioref" data-xml-rid="B60">60</a></td></tr><tr data-xml-align="left"><td>    p695-<i>legK1</i>::<i>Km</i></td><td>pAV695 derivative carrying <i>lpl1545</i> interrupted by <i>Km</i></td><td>This study</td></tr><tr data-xml-align="left"><td>    p695-<i>legK2</i>::<i>Km</i></td><td>pAV695 derivative carrying <i>lpl2066</i> interrupted by <i>Km</i></td><td>This study</td></tr><tr data-xml-align="left"><td>    p695-<i>legK3</i>::<i>Km</i></td><td>pAV695 derivative carrying <i>lpl2481</i> interrupted by <i>Km</i></td><td>This study</td></tr><tr data-xml-align="left"><td>    p695-<i>legK4</i>::<i>Km</i></td><td>pAV695 derivative carrying <i>lpl0262</i> interrupted by <i>Km</i></td><td>This study</td></tr><tr data-xml-align="left"><td>    p695-<i>legK5</i>::<i>Km</i></td><td>pAV695 derivative carrying <i>lpl2476</i> interrupted by <i>Km</i></td><td>This study</td></tr><tr data-xml-align="left"><td>    pXDC50</td><td><span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying <i>mCherry</i></td><td>X. Charpentier</td></tr><tr data-xml-align="left"><td>    p<i>legK1</i></td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry; expression of <i>legK1</i> under the control of <i>legK2</i> promoter (400 bp)</td><td>This study</td></tr><tr data-xml-align="left"><td>    p<i>legK2</i></td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry; expression of <i>legK2</i> under its promoter (400 bp)</td><td>This study</td></tr><tr data-xml-align="left"><td>    p<i>legK2</i>(<i>K112M</i>)</td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry; expression of <i>legK2</i>(<i>K112M</i>) under its promoter (400 bp)</td><td>This study</td></tr><tr data-xml-align="left"><td>    p<i>legK3</i></td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry; expression of <i>legK3</i> under the control of <i>legK2</i> promoter (400 bp)</td><td>This study</td></tr><tr data-xml-align="left"><td>    p<i>legK4</i></td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry; expression of <i>legK4</i> under the control of <i>legK2</i> promoter (400 bp)</td><td>This study</td></tr><tr data-xml-align="left"><td>    p<i>legK5</i></td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry; expression of <i>legK5</i> under the control of <i>legK2</i> promoter (400 bp)</td><td>This study</td></tr><tr data-xml-align="left"><td>    p3041</td><td>pXDC50 derivative <span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying mCherry and gentamicin resistance gene</td><td>This study</td></tr><tr data-xml-align="left"><td>    pEP46</td><td><span class="named-content" data-type="genus-species">D. discoideum</span> expression vector carrying <i>legK2</i>(<i>K112M</i>)-c-<i>myc</i> under the control of the actin promoter</td><td>This study</td></tr><tr data-xml-align="left"><td>    pEP73</td><td><span class="named-content" data-type="genus-species">Legionella</span> expression vector carrying <i>gfp-legK2</i>; production of the GFP-LegK2 hybrid protein</td><td>This study</td></tr></tbody></table></div></figure></div><div role="paragraph"><span class="named-content" data-type="genus-species">L. pneumophila</span> strains were grown at 30°C either on buffered charcoal yeast extract (BCYE) agar or in BYE liquid medium; each medium was supplemented with kanamycin at 10 μg ml<sup>−1</sup> or chloramphenicol at 5 μg ml<sup>−1</sup> when appropriate. <span class="named-content" data-type="genus-species">Escherichia coli</span> strains were grown at 37°C in LB medium supplemented with ampicillin at 100 μg ml<sup>−1</sup>, kanamycin at 20 μg ml<sup>−1</sup>, or chloramphenicol at 5 μg ml<sup>−1</sup>. <span class="named-content" data-type="genus-species">E. coli</span> XL1-Blue was used to maintain plasmids and strain <span class="named-content" data-type="genus-species">E. coli</span> BL21(DE3)(pREP4-<i>groESL</i>) was used for recombinant protein overproduction.</div><div role="paragraph">Axenic <span class="named-content" data-type="genus-species">Acanthamoeba castellanii</span> cells were grown on PYG medium (proteose-yeast extract-glucose medium) at 30°C and split once a week. <span class="named-content" data-type="genus-species">Dictyostelium discoideum</span> expressing calnexin-green fluorescent protein (GFP) (DBS0236184) was obtained from the Dicty Stock Center (depositor A. Müller-Taubenberger [<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>]). <span class="named-content" data-type="genus-species">D. discoideum</span> cells were axenically grown in HL5 medium at 22°C, supplemented with G418 at 20 μg ml<sup>−1</sup> when necessary.</div><div role="paragraph">J774A.1 macrophages were maintained at 37°C in 5% CO<sub>2</sub> in RPMI 1640 medium supplemented with 10% heat-inactivated fetal calf serum.</div></section><section id="sec-2-2"><h3>General DNA techniques.</h3><div role="paragraph">The oligonucleotides used in the present study are shown in <a href="#T2">Table 2</a>.</div><div class="figure-wrap"><header><div class="label">Table 2.</div></header><figure id="T2" class="table"><figcaption><span class="heading">Table 2</span>. Primers used in this study</figcaption><div class="table-wrap"><table><thead data-xml-valign="bottom"><tr data-xml-align="center"><th>No.</th><th>Name</th><th>Sequence (5′–3′)<a href="#TF2-1" role="doc-noteref"><i><sup>a</sup></i></a></th><th>Description</th></tr></thead><tbody data-xml-valign="top"><tr data-xml-align="left"><td>1</td><td>N-lpl1545-BamHI</td><td>ATA<b>GGATCC</b>CCTCGTACAATGTTTTTTTCC</td><td>GST-LegK1 overproduction</td></tr><tr data-xml-align="left"><td>2</td><td>C-lpl1545-SalI</td><td>ATA<b>GTCGAC</b>TTACTCAGCCACTAACCATAAGG</td><td>GST-LegK1 overproduction</td></tr><tr data-xml-align="left"><td>3</td><td>N-lpl2066-BamHI</td><td>ATA<b>GGATCC</b>GTTTATTACATAAATTTGAAGGAAC</td><td>GST-LegK2 overproduction</td></tr><tr data-xml-align="left"><td>4</td><td>C-lpl2066-SalI</td><td>ATA<b>GTCGAC</b>TTAGCTTGGGCCTCGCATC</td><td>GST-LegK2 overproduction</td></tr><tr data-xml-align="left"><td>5</td><td>N-lpl2481-BamHI</td><td>ATA<b>GGATCC</b>TTTGATAGAAATATAAAAGAAATAATC</td><td>GST-LegK3 overproduction</td></tr><tr data-xml-align="left"><td>6</td><td>C-lpl2481-SalI</td><td>ATA<b>GTCGAC</b>TTATAATTCAAAGCCTGAAT</td><td>GST-LegK3 overproduction</td></tr><tr data-xml-align="left"><td>7</td><td>N-lpl0262-BamHI</td><td>ATA<b>GGATCC</b>AAATTGCTTCGGTTTCATGAATT</td><td>6His-LegK4 overproduction</td></tr><tr data-xml-align="left"><td>8</td><td>C-lpl0262-SalI</td><td>ATA<b>GTCGAC</b>TTAATATGGCAAAATGATGACGT</td><td>6His-LegK4 overproduction</td></tr><tr data-xml-align="left"><td>9</td><td>N-lpl2476-SacI</td><td>ATA<b>GAGCTC</b>GGGATTATCATGGCTACAGT</td><td>6His-LegK5 overproduction</td></tr><tr data-xml-align="left"><td>10</td><td>C-lpl2476-PstI</td><td>ATA<b>CTGCAG</b>TTATTTTATGAAATCGGCCTTTA</td><td>6His-LegK5 overproduction</td></tr><tr data-xml-align="left"><td>11</td><td>Kan-FRTS</td><td>CAC<b>GTCGAC</b>AGCGATTGTGTAGGCTGGAGC</td><td>Km amplification</td></tr><tr data-xml-align="left"><td>12</td><td>Kan-FRTR</td><td>GGGGATCC<b>GTCGAC</b>CTGC</td><td>Km amplification</td></tr><tr data-xml-align="left"><td>13</td><td>P1-dotA-NotI</td><td>AAAA<b>GCGGCCGC</b>GCTCTCGCTGAAAGTGGCTC</td><td><i>lpl2613</i> deletion</td></tr><tr data-xml-align="left"><td>14</td><td>P2-dotA-SalI</td><td>CGC<b>GTCGAC</b>TGCTTGCAAGCTCTTGGTTG</td><td><i>lpl2613</i> deletion</td></tr><tr data-xml-align="left"><td>15</td><td>P3-dotA-SalI</td><td>CGC<b>GTCGAC</b>GCCATTTCCTACATCCAATCG</td><td><i>lpl2613</i> deletion</td></tr><tr data-xml-align="left"><td>16</td><td>P4-dotA-NotI</td><td>AAAA<b>GCGGCCGC</b>CCGGTTTAGAGCTTGGTCCA</td><td><i>lpl2613</i> deletion</td></tr><tr data-xml-align="left"><td>17</td><td>P1-legK1-NotI</td><td>AAAA<b>GCGGCCGC</b>CGTTGATGCCGCTAATCTCC</td><td><i>lpl1545</i> deletion</td></tr><tr data-xml-align="left"><td>18</td><td>P2-legK1-SalI</td><td>CGC<b>GTCGAC</b>CGGTTTAACCGCTATATGCCC</td><td><i>lpl1545</i> deletion</td></tr><tr data-xml-align="left"><td>19</td><td>P3-legK1-SalI</td><td>CGC<b>GTCGAC</b>AAGGCTATCAAGCAGTTTTCCC</td><td><i>lpl1545</i> deletion</td></tr><tr data-xml-align="left"><td>20</td><td>P4-legK1-NotI</td><td>AAAA<b>GCGGCCGC</b>TTTGAGAAAATAATCCCAGGCG</td><td><i>lpl1545</i> deletion</td></tr><tr data-xml-align="left"><td>21</td><td>P1-legK2-NotI</td><td>AAAA<b>GCGGCCGC</b>AATTTGAAGGAACAACCCTTACCTC</td><td><i>lpl2066</i> deletion</td></tr><tr data-xml-align="left"><td>22</td><td>P2-legK2-SalI</td><td>CGC<b>GTCGAC</b>AAGTTTTTCCAGGACACATCCCT</td><td><i>lpl2066</i> deletion</td></tr><tr data-xml-align="left"><td>23</td><td>P3-legK2-SalI</td><td>CGC<b>GTCGAC</b>TTCGAATTTACAGGCTTACAAGGATC</td><td><i>lpl2066</i> deletion</td></tr><tr data-xml-align="left"><td>24</td><td>P4-legK2-NotI</td><td>AAAA<b>GCGGCC</b>GCGCCTCGCATCAATGAAGGTG</td><td><i>lpl2066</i> deletion</td></tr><tr data-xml-align="left"><td>25</td><td>P1-legK3-NotI</td><td>AAAA<b>GCGGCC</b>GATTTACTTCCGGGCACTGG</td><td><i>lpl2481</i> deletion</td></tr><tr data-xml-align="left"><td>26</td><td>P2-legK3-SalI</td><td>CGC<b>GTCGAC</b>GGCATTCGTTTCATCGTCAG</td><td><i>lpl2481</i> deletion</td></tr><tr data-xml-align="left"><td>27</td><td>P3-legK3-SalI</td><td>CGC<b>GTCGAC</b>AGCAACTTGCGTCCATTACG</td><td><i>lpl2481</i> deletion</td></tr><tr data-xml-align="left"><td>28</td><td>P4-legK3-NotI</td><td>AAAA<b>GCGGCC</b>AGCTTCGCTTGCATGCAAA</td><td><i>lpl2481</i> deletion</td></tr><tr data-xml-align="left"><td>29</td><td>P1-legK4-NotI</td><td>AAAA<b>GCGGCCGC</b>AAGCCAATCATCGTTCCCAC</td><td><i>lpl0262</i> deletion</td></tr><tr data-xml-align="left"><td>30</td><td>P2-legK4-SalI</td><td>CGC<b>GTCGAC</b>GTTCTGGTGCTAAATAGCTTGCG</td><td><i>lpl0262</i> deletion</td></tr><tr data-xml-align="left"><td>31</td><td>P3-legK4-SalI</td><td>CGC<b>GTCGAC</b>CTGCCACATCAAGTCCCCTC</td><td><i>lpl0262</i> deletion</td></tr><tr data-xml-align="left"><td>32</td><td>P4-legK4-NotI</td><td>AAAA<b>GCGGCCGC</b>TGGCAAAATGATGACGTTGC</td><td><i>lpl0262</i> deletion</td></tr><tr data-xml-align="left"><td>33</td><td>P1-legK5-NotI</td><td>AAAA<b>GCGGCCGC</b>CATGGCTACAGTAGATTCCG</td><td><i>lpl2476</i> deletion</td></tr><tr data-xml-align="left"><td>34</td><td>P2-legK5-SalI</td><td>CGC<b>GTCGAC</b>GAACGTTAGCTTCACGCTCT</td><td><i>lpl2476</i> deletion</td></tr><tr data-xml-align="left"><td>35</td><td>P3-legK5-SalI</td><td>CGC<b>GTCGAC</b>CACTGAAAATCGCGGATATCG</td><td><i>lpl2476</i> deletion</td></tr><tr data-xml-align="left"><td>36</td><td>P4-legK5-NotI</td><td>AAAA<b>GCGGCCGC</b>GTTCCATGTCAATTTTAGGGC</td><td><i>lpl2476</i> deletion</td></tr><tr data-xml-align="left"><td>37</td><td>5-promo<i>lpl2066</i>-SacI</td><td>ATA<b>GAGCTC</b>CAGGGTAACTGAATAAGCCC</td><td><i>lpl2066</i> complementation</td></tr><tr data-xml-align="left"><td>38</td><td>3-promo<i>lpl2066</i>-KpnI</td><td>CGG<b>GGTACC</b>TCTCCTACAAATCAATTGCC</td><td><i>lpl2066</i> complementation</td></tr><tr data-xml-align="left"><td>39</td><td>5′SphI-promo<i>lpl2066</i></td><td>ATA<b>GCATGC</b>CAGGGTAACTGAATAAGCCC</td><td><i>lpl2066</i> complementation</td></tr><tr data-xml-align="left"><td>40</td><td><i>lpl2066</i>(<i>K112M</i>)-sens</td><td>CCCAAAGAGAATATACTGATGGTTTTATATCAAAATTTTAGTAATGTCG</td><td><i>lpl2066</i> mutagenesis</td></tr><tr data-xml-align="left"><td>41</td><td><i>lpl2066</i>(<i>K112M</i>)-rev</td><td>CGACATTACTAAAATTTTGATATAAAACCATCAGTATATTCTCTTTGGG</td><td><i>lpl2066</i> mutagenesis</td></tr></tbody></table></div><div class="notes"><div class="labeled" role="doc-footnote"><div class="label"><sup>a</sup></div><div id="TF2-1" role="paragraph">Restriction enzyme sites are indicated in boldface.</div></div></div></figure></div><div role="paragraph">Standard techniques were used for nucleic acid cloning and restriction analysis. Restriction enzymes and T4 DNA ligase were purchased from Fermentas (Saint Rémy-les-Chevreuses, France). Plasmid DNA from <span class="named-content" data-type="genus-species">E. coli</span> was prepared by rapid alkaline lysis (<a href="#B31" role="doc-biblioref" data-xml-rid="B31">31</a>). PCR amplifications were carried out with Phusion polymerase as recommended by the manufacturer (Finnzymes, Espoo, Finland). Purification of DNA fragments from agarose gels for subcloning was carried out with a QIAquick gel purification kit (Qiagen, Courtaboeuf, France).</div></section><section id="sec-2-3"><h3>Expression and purification of recombinant proteins.</h3><div role="paragraph">DNA fragments corresponding to the coding sequences of <i>legK1</i> (<i>lpl1545</i>), <i>legK2</i> (<i>lpl2066</i>), <i>legK3</i> (<i>lpl2481</i>), <i>legK4</i> (<i>lpl0262</i>), and <i>legK5</i> (<i>lpl2476</i>) were PCR amplified using genomic DNA of <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens as the template and the following oligonucleotide pairs: for <i>legK1</i>, 1/2; for <i>legK2</i>, 3/4; for <i>legK3</i>, 5/6; for <i>legK4</i>, 7/8; and for <i>legK5</i>, 9/10. The amplified DNA fragments were digested with BamHI and SalI or with SacI and PstI for <i>legK5</i> and then inserted into PGEX-6P-3 for <i>legK1</i>, <i>legK2</i>, and <i>legK3</i> or pQE30 for <i>legK4</i> and <i>legK5</i>. The resulting plasmids—pGEX-<i>legK1</i>, pGEX-<i>legK2</i>, pGEX-<i>legK3</i>, pQE30-<i>legK4</i>, and pQE30-<i>legK5</i>—were introduced into <span class="named-content" data-type="genus-species">E. coli</span> strain BL21(DE3)(pREP4-<i>groESL</i>). Transformants were grown at 37°C until cultures reached an optical density at 600 nm (OD<sub>600</sub>) of 0.7. Gene expressions were induced with 1 mM IPTG (isopropyl-β-<span data-style="small-caps">d</span>-thiogalactopyranoside) for 4 h at 20°C, and cell lysates were prepared by using a French pressure cell (SLM, Urbana, IL). GST and 6His recombinant proteins were purified under native conditions by affinity chromatography on glutathione-Sepharose beads (GE Healthcare, Orsay, France) and a Ni-NTA column (Qiagen), respectively, according to the manufacturers' recommendations. The purity of the eluted protein was analyzed by SDS-PAGE, and protein concentrations were determined by using a protein assay dye reagent concentrate (Bio-Rad, Marnes-la-Coquette, France).</div></section><section id="sec-2-4"><h3><i>In vitro</i> phosphorylation assay.</h3><div role="paragraph"><i>In vitro</i> phosphorylation of about 2 μg of purified recombinant protein was performed for 30 min at 37°C in 20 μl of a buffer containing 25 mM Tris-HCl (pH 7.5), 5 mM MgCl<sub>2</sub>, 1 mM dithiothreitol, 10 μM ATP, and 5 μCi of [γ-<sup>32</sup>P]ATP (Perkin-Elmer, Courtaboeuf, France). In some assays, MgCl<sub>2</sub> was replaced with MnCl<sub>2</sub>. In eukaryotic substrate phosphorylation assays, 1 μg of myelin basic protein (MBP) was added. In each case, the reaction was stopped by the addition of an equal volume of 2× Laemmli loading buffer (<a href="#B41" role="doc-biblioref" data-xml-rid="B41">41</a>).</div></section><section id="sec-2-5"><h3>TEM translocation assays.</h3><div role="paragraph">J774A.1 cells grown in RPMI 1640 containing 10% fetal calf serum were seeded in a black, clear-bottom 96-well plate at 10<sup>5</sup> cells per well 24 h prior to infection. <span class="named-content" data-type="genus-species">L. pneumophila</span> strains carrying the various TEM β-lactamase fusions were grown on BCYE plates containing chloramphenicol. Single colonies were then streaked onto BCYE plates containing chloramphenicol and 0.5 mM IPTG and grown for 24 h to induce production of the hybrid proteins. Then, 10 μl of bacteria resuspended in RPMI at 5 × 10<sup>8</sup> cells ml<sup>−1</sup> was used to infect J774A.1 cells (multiplicity of infection [MOI] = 50). After centrifugation (600 × <i>g</i>, 10 min) to initiate bacterium-cell contact, the plate was shifted to 37°C, followed by incubation for 1 h with CO<sub>2</sub> exchange. Cell monolayers were loaded with the fluorescent substrate by adding 20 μl of 6× CCF4/AM solution (LiveBLAzer-FRET B/G loading kit; Invitrogen, Cergy Pontoise, France) containing 15 mM Probenecid (Sigma, St. Quentin Fallavier, France). The cells were incubated for an additional 2 h at room temperature. Fluorescence was quantified on a Victor microplate reader (Perkin-Elmer) with excitation at 405 nm (10-nm band-pass), and emission was detected via 460 nm (40-nm band-pass, blue fluorescence) and 530 nm (30-nm band-pass, green fluorescence) filters. The cells were visualized by fluorescence microscopy using an inverted microscope equipped with the β-lactamase ratiometric filter set (Chroma, Bellows Falls, VT) (<a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>).</div></section><section id="sec-2-6"><h3>Gene inactivation in <span class="named-content" data-type="genus-species">L. pneumophila</span>.</h3><div role="paragraph">To obtain <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens mutants defective for <i>legK</i> genes, a homologous recombination strategy was chosen as previously described (<a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a>). The 400-bp upstream and downstream regions of the gene of interest were amplified by PCR, digested by NotI and SalI, and then cloned into pAV695. A kanamycin resistance cassette was amplified from pKD13 (with the primers 11 and 12), digested by SalI, and cloned inside the encoding sequence of each <i>legK</i> gene in pAV695. The resulting construct was introduced into <span class="named-content" data-type="genus-species">L. pneumophila</span> for chromosomal recombination.</div></section><section id="sec-2-7"><h3>Construction of a kinase-dead LegK2-producing strain.</h3><div role="paragraph">In order to obtain the catalytic mutant of LegK2, namely, LegK2(K112M) defective in phosphate donor ATP binding, substitution in the <i>legK2</i> gene was performed with a QuikChange II site-directed mutagenesis kit (Stratagene) using primers 40 and 41 listed in <a href="#T2">Table 2</a>, and the pXDC50 derivative vector in which <i>legK2</i> is cloned under its own promoter, namely, p<i>legK2</i> plasmid, as a template (<a href="#T1">Table 1</a>). The final construct was used to transform the <span class="named-content" data-type="genus-species">L. pneumophila</span> <i>ΔlegK2</i> mutant strain.</div></section><section id="sec-2-8"><h3>Pigment production.</h3><div role="paragraph">For quantifying pigment accumulation, 1-ml portions of the samples, obtained from 5-day-old BYE cultures grown at 30°C, were centrifuged at 16,000 × <i>g</i> for 10 min, and the OD<sub>550</sub> of the supernatants was measured (<a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>).</div></section><section id="sec-2-9"><h3>Cytotoxicity to <span class="named-content" data-type="genus-species">A. castellanii</span>.</h3><div role="paragraph">For measurement of the number of viable <span class="named-content" data-type="genus-species">A. castellanii</span> cells remaining after infection, the monolayers were treated with 10% Alamar blue (Invitrogen, Cergy Pontoise, France). At the time point indicated, the monolayers were washed four times with protease-yeast extract (PY), and then 100 μl of PY containing 10% (vol/vol) of Alamar blue was added to each well. After an overnight incubation, the OD<sub>570</sub> values were determined. The relative degree of amoeba mortality was expressed as the ratio of the OD value of infected monolayer to that of uninfected one, calculated as [1 − (mean OD<sub>infected</sub>/mean OD<sub>uninfected</sub>)] × 100.</div></section><section id="sec-2-10"><h3>Measurement of released extracellular bacteria.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">L. pneumophila</span> was grown on BCYE agar for 5 days at 30°C prior to infection. <span class="named-content" data-type="genus-species">A. castellanii</span> cells were seeded in multiwell plates to a final concentration of 5 × 10<sup>5</sup> cells ml<sup>−1</sup>. After adhesion, the cells were infected at an MOI of 10 with <span class="named-content" data-type="genus-species">L. pneumophila</span>. The plates were centrifuged at 880 × <i>g</i> for 10 min, followed by incubation for 1 h at 30°C. Monolayers were then washed four times with PY to remove extracellular bacteria. The time at the end of the final wash was the initial time point. After 24 h, extracellular bacteria were numbered by plating them on BCYE.</div></section><section id="sec-2-11"><h3>Intracellular growth in <span class="named-content" data-type="genus-species">A. castellanii</span>.</h3><div role="paragraph">Intracellular growth was monitored as previously described (<a href="#B36" role="doc-biblioref" data-xml-rid="B36">36</a>). <span class="named-content" data-type="genus-species">L. pneumophila</span> cells harboring a fluorescent mCherry protein-producing plasmid were grown on BCYE agar containing 0.5 mM IPTG and chloramphenicol for 5 days at 30°C. <span class="named-content" data-type="genus-species">A. castellanii</span> cells were grown as adherent cells in 96-well microplates (10<sup>5</sup> cells/well) and then infected with 10<sup>6</sup> fluorescent legionellae (MOI = 10). Infection was synchronized by spinning the bacteria at 880 × <i>g</i> for 10 min. Intracellular growth was automatically monitored by measuring the fluorescence of mCherry at an excitation of 587 nm and an emission of 610 nm in a Xenius Safas plate reader every hour for 66 h (Safas, Monaco). Fluorescence data were subjected to background subtractions (uninfected cells).</div></section><section id="sec-2-12"><h3>Bacteria uptake assay by trypan blue quenching.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">L. pneumophila</span> phagocytosis was measured by trypan blue quenching as previously described (<a href="#B10" role="doc-biblioref" data-xml-rid="B10">10</a>). Briefly, bacteria from overnight cultures in BYE were labeled with 5,6-carboxyfluorescein succinimidyl ester. <span class="named-content" data-type="genus-species">A. castellanii</span> cells were grown as adherent cells in 96-well microplates (5 × 10<sup>5</sup> cells/well) and then infected with 10<sup>7</sup> fluorescent legionellae (MOI = 20). After centrifugation (880 × <i>g</i> for 10 min) to initiate cell-bacterium contact, the plate was incubated at 30°C for 30 min. The medium was then replaced by 50 μl per well of trypan blue solution to quench the fluorescence of noninternalized bacteria. After 1 min of incubation, the fluorescence of internalized bacteria was measured on a Xenius Safas plate reader (Safas, Monaco) with an excitation of 485 nm and an emission of 530 nm.</div></section><section id="sec-2-13"><h3>Recruitment of the ER to LCV in <span class="named-content" data-type="genus-species">D. discoideum</span>.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">D. discoideum</span> cells producing calnexin-GFP were used to analyze by fluorescence microscopy the recruitment of ER to <span class="named-content" data-type="genus-species">Legionella</span>-containing vacuole (LCV) that harbor mCherry-labeled <span class="named-content" data-type="genus-species">L. pneumophila</span>. <span class="named-content" data-type="genus-species">D. discoideum</span> cells were seeded onto sterile glass coverslips in 6-well plates at 5 × 10<sup>6</sup> per well in HL5 and allowed to adhere overnight. Monolayers were infected at an MOI of 100 with bacteria grown for 5 days at 30°C. The plates were spun at 880 × <i>g</i> for 10 min, followed by incubation at 25°C. The monolayers were then washed two times with 2 ml of SorC buffer (2 mM Na<sub>2</sub>HPO<sub>4</sub>, 15 mM KH<sub>2</sub>PO<sub>4</sub>, 50 mM CaCl<sub>2</sub>; pH 6.0) and fixed with 3.7% paraformaldehyde (30 min, 4°C). Coverslips were examined with an inverted confocal microscope (Axiovert 200M; Zeiss, Thornwood, NJ) equipped with a ×63 phase-contrast objective lens (Plan Neofluar [Zeiss]; aperture, 1.4, oil).</div></section><section id="sec-2-14"><h3>Localization of LegK2 within the host cell.</h3><div role="paragraph">The Δ<i>legK2</i> mutant strain was cotransformed with p3041 and pEP73 plasmids (<a href="#T1">Table 1</a>), which encode mCherry and the GFP-LegK2 hybrid protein, respectively. The obtained strain was used to infect <span class="named-content" data-type="genus-species">D. discoideum</span> DH1 at an MOI of 100 for 1 h. Fluorescent GFP-LegK2 protein and mCherry-labeled bacteria were observed with an inverted confocal microscope (Axiovert 200M; Zeiss) equipped with a ×63 phase-contrast objective (Plan Neofluar). Alternatively, <span class="named-content" data-type="genus-species">D. discoideum</span> DH1 harboring pEP46 vector (<a href="#T1">Table 1</a>), which ectopically express <i>legK2</i>(<i>K112M</i>)<i>-c-myc</i>, were seeded onto coverslips and infected with mCherry-labeled <span class="named-content" data-type="genus-species">L. pneumophila</span> <i>legK2</i> and <i>dotA</i> mutant strains or <span class="named-content" data-type="genus-species">E. coli</span> XL1-Blue as described above. After 1 h of infection, the cells were fixed, permeabilized, and blocked with phosphate-buffered saline supplemented with 0.2% bovine serum albumin. The coverslips were then stained with α-c-Myc monoclonal antibody (1/5,000, clone 9E10; Sigma) for 1 h and visualized with Alexa Fluor 488-conjugated secondary antibody (Molecular Probes, Eugene, OR) at a 1/300 dilution for 30 min. Microscopic observations of mCherry-labeled bacteria and LegK2 protein were performed with an inverted confocal microscope (Axiovert 200M).</div></section><section id="sec-2-15"><h3>Statistical analysis.</h3><div role="paragraph">The results were statistically analyzed by using a Student <i>t</i> test. The <i>t</i> test results obtained correspond to the comparison between <i>legK</i> mutants with the parental strain values in the same conditions.</div></section></section><section id="sec-3" data-type="results"><h2>RESULTS</h2><section id="sec-3-1"><h3><span class="named-content" data-type="genus-species">L. pneumophila</span> Lens genome encodes five putative protein kinases.</h3><div role="paragraph"><i>In silico</i> analysis of the epidemic <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens strain genome revealed five genes encoding putative protein kinases. These genes, which we named <i>legK1</i> to <i>legK5</i> for <span data-style="underline"><i>L</i></span><i>egionella</i> <span data-style="underline">e</span>ukaryotic <span data-style="underline">g</span>ene <span data-style="underline">k</span>inases, as proposed earlier (<a href="#B16" role="doc-biblioref" data-xml-rid="B16">16</a>), encode 529, 545, 462, 961, and 453 amino acids proteins with calculated molecular masses of 61, 62, 53, 109, and 51 kDa, respectively (<a href="#T3">Table 3</a>). Alignment with several prokaryotic and eukaryotic protein kinases revealed residues that are highly conserved in the Hanks subdomains characterizing the eukaryote-like protein kinase family (<a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>). These include the glycine-rich loop and the invariant lysine in subdomains I and II, which are essential for binding and correct orientation of the phosphate donor ATP (<a href="#F1">Fig. 1</a> A). Consensus sequences in subdomains VI, VII, and VIII, involved in phosphotransfer and substrate recognition are also conserved in LegK1, LegK3, and LegK4; LegK2 and LegK5 lack subdomain VIII (<a href="#F1">Fig. 1</a>A). Further primary sequence analysis predicted an EF-hand domain in LegK5, which is known to be involved in calcium binding (<a href="#B53" role="doc-biblioref" data-xml-rid="B53">53</a>). Moreover, hydropathy profiles suggest a transmembrane helix between amino acids 478 and 498 of LegK1, which is consistent with a membrane location of LegK1, whereas LegK2 to LegK5 are likely cytoplasmic (<a href="#T3">Table 3</a>).</div><div class="figure-wrap"><header><div class="label">Table 3.</div></header><figure id="T3" class="table"><figcaption><span class="heading">Table 3</span>. <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens putative protein kinases</figcaption><img src="/cms/10.1128/IAI.00805-10/asset/4527fc32-c2ec-47aa-83f7-7c4ba9742022/assets/graphic/zii9990991060007.jpeg" /><div class="notes"><div class="labeled" role="doc-footnote"><div class="label"><sup>a</sup></div><div id="TF3-1" role="paragraph">Domains inferred from electronic annotations: TM, transmembrane helix; STPK, serine/threonine protein kinase domain; YPK, tyrosine protein kinase domain; Ca-<i>depPK</i>, calmodulin-dependent protein kinase domain; EF-HAND-1, specific protein domain; AA, amino acid position of the first PK domain.</div></div></div></figure></div><div class="figure-wrap"><header><div class="label">Fig. 1.</div></header><figure id="F1" class="graphic"><img src="/cms/10.1128/IAI.00805-10/asset/d6bbbf63-6e2b-423b-a3a2-51e3ef4d401e/assets/graphic/zii9990991060001.jpeg" /><figcaption><span class="heading">Fig. 1</span>. Putative protein kinases encoded by the <span class="named-content" data-type="genus-species">L. pneumophila</span> genome. (A) Multiple sequence alignment of the protein kinase domains of LegK1 to LegK5. The highly conserved amino acid residues of the Hanks subdomains I, II, VIB, VII, and VIII are indicated in boldface. (B) Schematic representations of the <i>legK5</i> genomic region. The GC content of each gene of the region is indicated in the upper scheme. The genes conserved in the five <i>sg1</i> <span class="named-content" data-type="genus-species">L. pneumophila</span> strains Lens, Paris, Philadelphia, Corby, and Alcoy are represented in gray, while the genes specific to <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens are represented in black. Encoded proteins are named below.</figcaption></figure></div><div role="paragraph">The protein kinases encoding genes <i>legK1</i> to <i>legK5</i> are distributed in various regions of the <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens chromosome. Organization of the <i>legK4</i> region is remarkable: <i>legK4</i> is located immediately upstream a gene (<i>lpl0263</i>) that encodes a protein with a high degree of similarity (39% identity) with the C-terminal domain (from amino acid 323 to amino acid 910) of LegK4. This genetic organization is conserved in the five sequenced <span class="named-content" data-type="genus-species">L. pneumophila</span> strains and suggests a partial duplication of <i>legK4</i> homologous genes. On the other hand, whereas genes <i>legK1</i> to <i>legK4</i> are conserved in all <span class="named-content" data-type="genus-species">L. pneumophila</span> sequenced strains, <i>legK5</i> is specific of strain Lens. It is located in a chromosomal region containing four genes that are absent in Philadelphia, Paris, Corby, and Alcoy genomes (<a href="#F1">Fig. 1</a>B). Compared to the average genomic GC content of 38.4% for Lens strain (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>), <i>legK5</i> and its neighboring genes have a significantly lower GC content of 35% (<i>legK5</i>), 34% (<i>lpl2477</i>), and 36% (<i>lpl2478</i>). Moreover, <i>lpl2478</i> gene encodes a putative transposase, and the region is flanked by two homologous genes, namely, <i>lpl2475</i> and <i>lpl2479</i> (<a href="#F1">Fig. 1</a>B). These observations suggest that the <i>legK5</i> chromosomal region has been acquired in Lens genome by a small insertion event associated with horizontal transfer.</div></section><section id="sec-3-2"><h3>LegK2 to LegK5 display autokinase and protein kinase activities.</h3><div role="paragraph">To determine whether LegK1 to LegK5 are functional protein kinases, each protein was overproduced and purified as a recombinant protein fused to either six-histidine or glutathione <i>S</i>-transferase tags. The <i>legK4</i> and <i>legK5</i> genes were PCR amplified using genomic DNA of <span class="named-content" data-type="genus-species">L. pneumophila</span> strain Lens and cloned into the expression vector pQE30 to allow the synthesis of N-terminal six-histidine-tagged proteins. The recombinant proteins, with respective molecular masses of 110 and 52 kDa, were overproduced in the <span class="named-content" data-type="genus-species">E. coli</span> strain BL21(DE3)(pREP4-<i>groESL</i>) and highly enriched by a single-step chromatography on Ni-NTA matrix (<a href="#F2">Fig. 2</a> A). The full-length <i>legK2</i> and <i>legK3</i> genes were synthesized by PCR amplification and cloned into pGEX-6P-3 plasmid to produce GST N-terminal fused proteins. The truncated <i>legK1</i> gene (nucleotides 4 to 1431) lacking the DNA fragment encoding the putative transmembrane helix was cloned into the same expression vector. The GST-tagged recombinant LegK1 (87 kDa), LegK2 (88 kDa), and LegK3 (79 kDa) proteins were purified to partial homogeneity from the <span class="named-content" data-type="genus-species">E. coli</span> strain BL21(DE3) (pREP4-<i>groESL</i>) by using a glutathione-Sepharose 4B matrix (<a href="#F2">Fig. 2</a>A).</div><div class="figure-wrap"><header><div class="label">Fig. 2.</div></header><figure id="F2" class="graphic"><img src="/cms/10.1128/IAI.00805-10/asset/82894cc2-4381-4fdd-98e2-04e753abe58d/assets/graphic/zii9990991060002.jpeg" /><figcaption><span class="heading">Fig. 2</span>. Biochemical activities of recombinant LegK proteins. (A) SDS-PAGE analysis of purified GST-LegK1, GST-LegK2, GST-LegK3, 6His-LegK4, and 6His-LegK5 after staining with Coomassie blue. Molecular mass standards are indicated on the left. (B) Effects of cations on autokinase activities of GST-LegK2, GST-LegK3, 6His-LegK4, and 6His-LegK5. Purified LegK proteins were subjected to <i>in vitro</i> autophosphorylation assays in the presence of [γ-<sup>32</sup>P]ATP and Mg<sup>2+</sup> or Mn<sup>2+</sup>. Phosphoproteins were separated by SDS-PAGE and then revealed by autoradiography. (C) Protein kinase activities of LegK proteins. The eukaryotic substrate myelin basic protein (MBP) was incubated with each LegK recombinant protein in the presence of [γ-<sup>32</sup>P]ATP. Phosphoproteins were visualized by autoradiography after SDS-PAGE separation.</figcaption></figure></div><div role="paragraph">The purified tagged LegK1-LegK5 proteins were then separately assayed for autokinase activity. Each recombinant protein was incubated with the phosphate donor [γ-<sup>32</sup>P]ATP and then separated by SDS-PAGE and analyzed by autoradiography (<a href="#F2">Fig. 2</a>B). Except for GST-LegK1, all <span class="named-content" data-type="genus-species">Legionella</span> recombinant kinases were significantly radiolabeled and thus undergo autophosphorylation. Their autokinase activities are differently dependent of magnesium or manganese ions. Although LegK4 used these ions as cofactor with the same efficiency, LegK2 and LegK5 autophosphorylated more efficiently in the presence of manganese; conversely, the radiolabeling of LegK3 was higher in the presence of magnesium (<a href="#F2">Fig. 2</a>B).</div><div role="paragraph">The recombinant LegK1 to LegK5 proteins were further characterized by studying their ability to phosphorylate proteins. They were assayed for <i>in vitro</i> phosphorylation of the general eukaryotic protein kinase substrate, myelin basic protein (MBP), in the presence of [γ-<sup>32</sup>P]ATP. Except for GST-LegK1, a radiolabeled signal corresponding to the expected 18-kDa molecular mass of MBP was detected, thus demonstrating that LegK2, LegK3, LegK4, and LegK5 are able to phosphorylate protein substrates such as MBP (<a href="#F2">Fig. 2</a>C).</div></section><section id="sec-3-3"><h3>LegK1 to LegK4 proteins are effectors of the Dot/Icm T4SS.</h3><div role="paragraph">Most of the eukaryote-like proteins encoded by <span class="named-content" data-type="genus-species">L. pneumophila</span> genome are translocated into the host cell cytoplasm, where they may interfere with host cell functions, thus allowing bacterial evasion of the endosomal pathway (<a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>). Although LegK1, LegK2, and LegK3 have been recently identified as Dot/Icm T4SS effectors (<a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>, <a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>, <a href="#B68" role="doc-biblioref" data-xml-rid="B68">68</a>), the translocation of LegK4 and LegK5 has not been documented yet. In order to determine whether these protein kinases are translocated during the infectious cycle, the TEM-1 reporter system (β-lactamase) was used (<a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>). Briefly, the TEM-1 was fused to the N-terminal of LegK4 and LegK5, and <span class="named-content" data-type="genus-species">L. pneumophila</span> strains efficiently expressing these fusions (<a href="#F3">Fig. 3</a> A) were used to infect J774 macrophages. Host cells were then loaded with CCF4, which emits green fluorescence (520 nm) due to fluorescence resonance energy transfer between the two fluorophores of this substrate when excited at 409 nm (<a href="#F3">Fig. 3</a>B, uninfected). When the hybrid protein was translocated, as expected for the fusion with the known Dot/Icm effector VipA, it cleaved the β-lactam ring of CCF4, thus resulting in fluorescence changing from green to blue (447 nm), when excited at the same wavelength (<a href="#F3">Fig. 3</a>B, TEM-VipA). Conversely, when TEM-1 was fused to the housekeeping protein enoyl-acyl coenzyme A reductase FabI, cells still developed green fluorescence (<a href="#F3">Fig. 3</a>B, TEM-FabI). Emission of blue fluorescence was observed when cells were infected with wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> strains expressing the TEM-LegK4 fusion protein, while green fluorescence was still emitted when cells were infected with bacteria containing TEM-LegK5 fusion. These analyses establish that LegK4, but not LegK5, is translocated into host macrophages during <span class="named-content" data-type="genus-species">L. pneumophila</span> infection. When a <i>dotA</i> mutant <span class="named-content" data-type="genus-species">L. pneumophila</span> strain producing TEM-LegK4 infected macrophages, no change in fluorescence was observed [<a href="#F3">Fig. 3</a>B, <i>ΔdotA</i>(TEM-LegK4)]. This result demonstrates that translocation of the LegK4 protein kinase is Dot/Icm T4SS dependent.</div><div class="figure-wrap"><header><div class="label">Fig. 3.</div></header><figure id="F3" class="graphic"><img src="/cms/10.1128/IAI.00805-10/asset/e48c53bb-06b4-4644-a959-a1986cdded58/assets/graphic/zii9990991060003.jpeg" /><figcaption><span class="heading">Fig. 3</span>. Dot/Icm-dependent translocation of LegK4 into J774 cells. (A) Western blot analysis of TEM fusion expression detected with an α-TEM antibody. (B) J774 cells were infected with <span class="named-content" data-type="genus-species">L. pneumophila</span> wild-type or <i>dotA</i> mutant strains harboring TEM-FabI, TEM-VipA, TEM-LegK4, and TEM-LegK5 expression plasmids at an MOI of 50. Infected cells were loaded with CCF4/AM, and translocation was determined by a comparison of cleaved to uncleaved CCF4 that gives blue and green fluorescence, respectively. Images were obtained by using epifluorescence microscopy on individual assay wells.</figcaption></figure></div></section><section id="sec-3-4"><h3>Inactivation of <i>legK1</i> to <i>legK5</i> genes has no effect on medium growth neither intracellular life cycle switch of <span class="named-content" data-type="genus-species">Legionella</span>.</h3><div role="paragraph">To identify the physiological role of <span class="named-content" data-type="genus-species">Legionella</span> protein kinases, each <i>legK</i> gene was separately inactivated, as described in Materials and Methods. Genes were partially deleted and replaced by a unique insertion of a kanamycin cassette (<i>legK</i>::<i>kan</i>), verified by PCR and Southern hybridization (data not shown). The five isogenic mutants displayed similar growth characteristics in liquid medium compared to the parental strain <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens, and analysis of mutant cell morphologies by light microscopy did not reveal any major differences (data not shown).</div><div role="paragraph">Successful infection of host cells is strictly linked to a precise timing of the life cycle of <span class="named-content" data-type="genus-species">Legionella</span>, namely, to the switch from the replicative to transmissive bacteria. Intracellular acquired transmissive traits can be mimicked by stationary medium growth phase (<a href="#B52" role="doc-biblioref" data-xml-rid="B52">52</a>). To check whether the LegK protein kinases are involved in the regulatory processes that control the intracellular life cycle of <span class="named-content" data-type="genus-species">Legionella</span>, each <i>legK</i> mutant was assayed for the characteristic trait of the postexponential growth phase, pigment production (<a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>). The five <i>legK</i> mutants accumulated pigment during growth, as did the wild-type strain (data not shown).</div></section><section id="sec-3-5"><h3>LegK2 protein kinase activity plays a key role in the virulence of <span class="named-content" data-type="genus-species">Legionella</span> toward amoeba.</h3><div role="paragraph">In order to assess the virulence of each <i>legK</i> mutant strain, <span class="named-content" data-type="genus-species">Acanthamoeba castellanii</span> amoebae were infected at an MOI of 10 with the wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens or the isogenic <i>legK</i>::<i>kan</i> mutants, and the viability of amoeba cells present in infected monolayers at 48 h postinfection was quantified by using the Alamar blue dye. The cytotoxicity of the <i>legK2</i> mutant strain toward amoeba, estimated at 30%, was significantly lower (<i>P</i> &lt; 0.01) than the value of 90% measured for the parental strain. Conversely, <i>legK1</i>, <i>legK3</i>, <i>legK4</i>, and <i>legK5</i> mutants displayed similar cytotoxicity toward amoebae compared to the wild-type strain (<a href="#F4">Fig. 4</a> A).</div><div class="figure-wrap"><header><div class="label">Fig. 4.</div></header><figure id="F4" class="graphic"><img src="/cms/10.1128/IAI.00805-10/asset/bb2b6610-9517-4619-b732-57b240559f8d/assets/graphic/zii9990991060004.jpeg" /><figcaption><span class="heading">Fig. 4</span>. Role of LegK proteins in virulence toward <span class="named-content" data-type="genus-species">A. castellanii</span>. (A) Cytotoxicity of <span class="named-content" data-type="genus-species">L. pneumophila</span>. Cells were infected at an MOI of 10. The viability of amoeba cells present in infected monolayers at 48 h postinfection was quantified by using the Alamar blue dye. These data are representative of two independent experiments done in triplicate. (B) Release of bacteria from amoebae infected with <span class="named-content" data-type="genus-species">L. pneumophila</span>. After 24 h of infection, the number of extracellular bacteria was evaluated by the standard plate count assay. The results are expressed in CFU ml<sup>−1</sup> and are representative of two independent experiments performed in triplicate. Error bars represent the standard deviations. (C) Intracellular growth of <span class="named-content" data-type="genus-species">L. pneumophila</span> in amoebae. Amoebae were infected at an MOI of 10 by <span class="named-content" data-type="genus-species">L. pneumophila</span> cells expressing the mCherry gene on a plasmid. Bacteria multiplication was automatically monitored by measuring the fluorescence of mCherry at an excitation of 587 nm and an emission of 610 nm every 2 h for 66 h. Fluorescence data were subjected to background subtractions (uninfected cells).</figcaption></figure></div><div role="paragraph">To further evaluate the virulence of each <span class="named-content" data-type="genus-species">Legionella</span> strain, extracellular released bacteria were numbered after one infectious cycle, i.e., at 24 h postinfection. Compared to the parental strain, 1.5-log fewer bacteria of the mutant strain <i>legK2</i> (<i>P</i> = 0.04) were released from <span class="named-content" data-type="genus-species">A. castellanii</span> at 24 h postinfection. In contrast, extracellular bacteria quantified for the <i>legK1</i>, <i>legK3</i>, <i>legK4</i>, and <i>legK5</i> mutants were similar to the wild-type strain (<a href="#F4">Fig. 4</a>B).</div><div role="paragraph">To check whether the <i>legK2</i> phenotype is related to a defect of bacteria egress or intracellular multiplication, the Lens strain or its <i>legK</i> mutant derivatives expressing the fluorescent mCherry protein on a plasmid were used to infect amoebas at an MOI of 10. Bacterial intracellular growth was monitored by fluorescence measurement during a 66 h-time course (<a href="#F4">Fig. 4</a>C). As expected, wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> started efficient intracellular growth (an ∼40-fold increase in fluorescence) from 15 h postinfection, whereas the T4SS <i>dotA</i> mutant failed to replicate. The <i>legK1</i>, <i>legK3</i>, <i>legK4</i>, and <i>legK5</i> mutants showed intracellular multiplication from the same time, with the same or lightly reduced growth rates compared to Lens strain. On the contrary, <i>legK2</i> mutant was delayed for intracellular multiplication since it started replicating 48 h after infection. After that, its growth rate was not significantly different (<i>P</i> = 0.02) from that measured for the wild-type strain.</div><div role="paragraph">To confirm the link between <i>legK2</i> deletion and the virulence defect observed, complementation experiments were carried out. Complementation of the <i>legK2</i> mutant by introducing a Δ<i>mobA</i> RSF1010 derivative vector expressing the intact <i>legK2</i> gene restored the cytotoxicity and the released extracellular bacteria of the parental strain (<a href="#F5">Fig. 5</a> A and B). Complementation assays of the <i>legK2</i> intracellular multiplication defect were performed by introducing the same vector, producing both the intact LegK2 and the fluorescent mCherry proteins. The intracellular replication kinetics observed for the complemented <i>legK2</i> strain was similar to that observed for the Lens strain (<a href="#F5">Fig. 5</a>C).</div><div class="figure-wrap"><header><div class="label">Fig. 5.</div></header><figure id="F5" class="graphic"><img src="/cms/10.1128/IAI.00805-10/asset/d6dd8c33-84b4-4822-ae02-220b573e6cf6/assets/graphic/zii9990991060005.jpeg" /><figcaption><span class="heading">Fig. 5</span>. Complementation of the <i>legK2</i> mutant virulence defect by other <i>legK</i> genes. (A) Cytotoxicity of <span class="named-content" data-type="genus-species">L. pneumophila</span> measured as described in <a href="#F4">Fig. 4</a>A. (B) Release of bacteria from amoebae evaluated as described in <a href="#F4">Fig. 4</a>B. (C) Intracellular growth of <span class="named-content" data-type="genus-species">L. pneumophila</span> in amoebae monitored as reported in <a href="#F4">Fig. 4</a>C.</figcaption></figure></div><div role="paragraph">To determine whether the protein kinase activity of LegK2 is critical for <span class="named-content" data-type="genus-species">Legionella</span> intracellular multiplication, a substitution kinase-dead mutant of LegK2, namely, LegK2(K112M) defective in phosphate donor ATP binding, was generated by site-directed mutagenesis of the <i>legK2</i> gene cloned on the RSF1010 derivative vector. This construct, which stably produces the same level of kinase-dead LegK2 compared to the wild-type protein (as checked by Western blotting experiments [data not shown]), was used to complement the <i>ΔlegK2</i> mutant defects in cytotoxicity (<a href="#F5">Fig. 5</a>A), extracellular bacteria release (<a href="#F5">Fig. 5</a>B), and intracellular multiplication (<a href="#F5">Fig. 5</a>C). The substitution <i>legK2</i> mutant is not able to complement the defects of the deletion <i>legK2</i> mutant strain, suggesting that the protein kinase activity of LegK2 plays a key role in the virulence of <span class="named-content" data-type="genus-species">L. pneumophila</span>.</div><div role="paragraph">Thus, <span class="named-content" data-type="genus-species">L. pneumophila</span> encodes five functional protein kinases that play a variable role in bacterial virulence toward amoebae: a nondetectable role for LegK1, LegK3, LegK4, and LegK5, but a key role for LegK2. These results suggest that <span class="named-content" data-type="genus-species">Legionella</span> protein kinases are differentially expressed during the <span class="named-content" data-type="genus-species">Legionella</span> infectious cycle or that they are not functionally equal, and they would control various host or bacterial functions. To answer this question, <i>legK</i> gene expression during the exponential and stationary bacterial growth phases, which mimic the replicative and transmissive bacterial forms, respectively, was established by performing real-time quantitative PCR experiments. All the five <i>legK</i> genes were expressed in laboratory growth conditions, and expression patterns displayed a similar slight downregulation of <i>legK1</i>, <i>legK2</i>, <i>legK3</i>, and <i>legK4</i> genes when bacteria entered the postexponential growth phase (data not shown). Moreover, the <i>legK2</i> virulence defect was assayed for complementation by vectors expressing <i>legK1</i>, <i>legK3</i>, <i>legK4</i>, or <i>legK5</i> under the control of the endogenic promoter of the <i>legK2</i> gene. None of these <i>legK</i> genes expressing plasmids can reverse the virulence phenotype of <i>legK2</i> mutant, as observed with the cytotoxicity of the corresponding strains (<a href="#F5">Fig. 5</a>A), the extracellular bacteria numeration (<a href="#F5">Fig. 5</a>B), and the intracellular replication (<a href="#F5">Fig. 5</a>C). Surprisingly, expression of these genes even led to an additional loss of virulence. To check whether this observation is restricted to a <i>legK2</i> genetic background, the same <i>legK</i> genes expressing vectors were transformed into the wild-type strain Lens. Expression of each <i>legK</i> gene on this plasmid resulted in a loss of virulence (data not shown). Further experiments are under way to mechanistically examine this result, but we assume that this effect most likely results from nonspecific factors, such as the elevated expression level of <i>legK</i> genes, which may interfere with the translocation of other T4SS substrates. Taken together, these data demonstrate that LegK2 plays a unique role in the virulence of <span class="named-content" data-type="genus-species">L. pneumophila</span> and is not functionally redundant with the four tested <span class="named-content" data-type="genus-species">Legionella</span> protein kinases.</div></section><section id="sec-3-6"><h3>LegK2 protein kinase activity is required for ER recruitment on the LCV.</h3><div role="paragraph">To further characterize the virulence defect of <i>legK2</i> mutant strain, the infectious cycle step altered was investigated. Intracellular survival at the onset of the <span class="named-content" data-type="genus-species">A. castellanii</span> infection was assessed by counting the viable intracellular bacteria at 1 h 20 min, 2 h, and 4 h postinfection. The level of viable intracellular <i>legK2</i> cells in amoebae corresponds to &lt;10% of the parental strain (<i>P</i> = 0.02), as early as 1 h 20 postinfection (<a href="#F6">Fig. 6</a> A). This result points out the major role of the protein kinase LegK2 in the early steps of the infectious cycle of <span class="named-content" data-type="genus-species">L. pneumophila</span>, namely, in bacterial uptake or in bacterial intracellular survival.</div><div class="figure-wrap"><header><div class="label">Fig. 6.</div></header><figure id="F6" class="graphic"><img src="/cms/10.1128/IAI.00805-10/asset/9e7f250a-2ab3-47ed-a1ec-29dec144847a/assets/graphic/zii9990991060006.jpeg" /><figcaption><span class="heading">Fig. 6</span>. LegK2 is required for ER recruitment on LCV. (A) Uptake and survival ability of <i>legK2</i> mutant strain. <span class="named-content" data-type="genus-species">A. castellanii</span> cells were infected at an MOI of 10 with wild-type, <i>dotA</i> mutant, and <i>legK2</i> mutant strains. After different periods of contact with <span class="named-content" data-type="genus-species">L. pneumophila</span>, monolayers were treated for 1 h with gentamicin to kill adherent bacteria and disrupted with 0.04% Triton X-100. Viable intracellular bacteria were diluted and plated onto BCYE agar plates for colony enumeration. The results are expressed as a relative value (%) compared to a control invasion experiment with the wild-type strain. These data are representative of three independent experiments performed in triplicate; error bars represent the standard deviations. (B) Bacterial uptake assay. <span class="named-content" data-type="genus-species">A. castellanii</span> cells were infected with fluorescein-labeled <span class="named-content" data-type="genus-species">Legionella</span> at an MOI of 20, in the presence of cytochalasin D when indicated (+ CytoD). After 30 min of incubation, the medium was replaced by trypan blue solution to quench the fluorescence of noninternalized bacteria. The fluorescence of internalized bacteria was measured using an excitation of 485 nm and an emission of 530 nm. Fluorescence data were corrected for differences in labeling efficiency between the tested strains. Labeling efficiencies between strains varied by ca. 10%. (C) Recruitment of calnexin-GFP. Fifty <span class="named-content" data-type="genus-species">Legionella</span> containing vacuoles were scored from each sample by confocal laser scanning micrographs of calnexin-GFP-labeled <span class="named-content" data-type="genus-species">D. discoideum</span> AX3 infected at an MOI of 100 with mCherry-labeled <span class="named-content" data-type="genus-species">L. pneumophila.</span> Calnexin-positive vacuoles were numbered for amoeba cells infected by the wild-type <span class="named-content" data-type="genus-species">L. pneumophila</span> Lens strain, its derivative <i>dotA</i> and <i>legK2</i> mutant strains, and the complemented <i>legK2</i>(p<i>legK2</i>) and <i>legK2</i>(p<i>legK2<sub>K112M</sub></i>) mutant strains. The data are representative of three independent experiments; each error bar represents the standard deviation. (D) Localization of ectopically produced LegK2(K112M)<i>-</i>c-Myc in <span class="named-content" data-type="genus-species">D. discoideum</span> cells during infection. Confocal laser scanning micrographs of DH1 cells expressing <i>legK2</i>(<i>K112M</i>)<i>-c-myc</i>, either uninfected or infected with mCherry-labeled <span class="named-content" data-type="genus-species">L. pneumophila</span> <i>legK2</i> mutant and <i>dotA</i> mutant strains or XL1-Blue <span class="named-content" data-type="genus-species">E. coli</span>. LegK2(K112M)<i>-</i>c-Myc was detected by an α-c-Myc antibody. The experiments were reproduced twice with similar results.</figcaption></figure></div><div role="paragraph">Bacteria uptake in amoebae was measured by trypan blue quenching of fluorescein-labeled <span class="named-content" data-type="genus-species">L. pneumophila</span>. Wild-type strain or <i>legK2</i> derivative were fluorescein-labeled and used to infect <span class="named-content" data-type="genus-species">A. castellanii.</span> After contact, trypan blue was added to quench the fluorescence of noninternalized bacteria. Measured fluorescence is representative of bacterial uptake, as demonstrated by the effect of the actin polymerization inhibitor, cytochalasin D (<a href="#F6">Fig. 6</a>B). The fluorescence of internalized <i>legK2</i> mutant was similar to that measured for the wild-type strain (<a href="#F6">Fig. 6</a>B), thus suggesting that LegK2 is not required for bacterial uptake but is more likely required for subsequent infectious cycle steps, namely, host cell vesicular trafficking control.</div><div role="paragraph">The unique intracellular fate of <span class="named-content" data-type="genus-species">L. pneumophila</span> involves ER recruitment to the LCV to evade the endocytic pathway and reside in a specialized vacuole, allowing intracellular replication. Recruitment of the ER by <span class="named-content" data-type="genus-species">L. pneumophila</span> was observed in the model amoeba <span class="named-content" data-type="genus-species">Dictyostelium discoideum</span> expressing the specific marker of ER, namely, calnexin fused to the GFP. Amoebae were infected at an MOI of 100 with wild-type or <i>legK2</i> mutant strains that express the red fluorescent protein (mCherry), and calnexin-positive LCVs were visually scored by confocal microscopy at 1 and 4 h postinfection. In <span class="named-content" data-type="genus-species">D. discoideum</span> infected with the parental <span class="named-content" data-type="genus-species">Legionella</span> strain Lens, ca. 70% of the LCVs stained positive for calnexin-GFP as early as 1 h after infection. On the contrary, only 12 and 13% (<i>P</i> &lt; 0.01) of the <i>legK2</i> mutant containing vacuoles were positive for calnexin at 1 and 4 h postinfection, respectively. The <i>legK2</i> ER recruitment defect is partially complemented by <i>legK2</i>-expressing RSF1010 derivative vector, since 49 and 51% (<i>P</i>= 0.01) of the <i>legK2</i>(p<i>legK2</i>) strain containing vacuoles were labeled by calnexin at 1 and 4 h postinfection, respectively. The incomplete complementation of the mutant could result from an elevated level of <i>legK2</i> gene expression on the plasmid, which may interfere with the translocation of other T4SS substrates required for ER recruitment. Together, these results indicate that in the absence of LegK2, ER is recruited to LCVs less efficiently. To identify whether the protein kinase activity of LegK2 is involved in ER recruitment, complementation assays of the deleted <i>legK2</i> mutant defect were performed with the substitution mutant <i>legK2</i>(<i>K112M</i>)-expressing vector. About 11% of the <i>legK2</i>(<i>plegK2<sub>K112M</sub></i>) mutant-containing vacuoles were stained with the ER marker calnexin at 1 and 4 h postinfection, which is very similar to that observed for the <i>legK2</i> mutant (<a href="#F6">Fig. 6</a>C). The lack of complementation by the <i>legK2</i> catalytic mutant points out the major role of LegK2 protein kinase activity in ER recruitment on the LCV.</div><div role="paragraph">To determine whether LegK2 anchors to the phagosomal membrane to recruit ER on the LCV, cellular localization of LegK2 during infection was investigated. <span class="named-content" data-type="genus-species">Legionella</span> cells producing the hybrid protein GFP-LegK2 were used to infect amoeba cells, and cellular localization of the hybrid protein was observed by confocal microscopy. Despite a good level of <i>gfp-legK2</i> expression in bacteria, we could not detect GFP-LegK2 protein either in the host cell cytoplasm or on the LCV, probably due to a low level of translocation into host cells (data not shown). To circumvent this difficulty, a strain of the amoeba <span class="named-content" data-type="genus-species">D. discoideum</span> that ectopically produces the C-terminal c-Myc-tagged LegK2(K112M) protein was constructed. Ectopic expression of the catalytic <i>legK2</i> mutant, rather than the wild-type <i>legK2</i> strain, was preferred to avoid possible aspecific effects of an active protein kinase overproduction on the amoeba physiology and to facilitate stable binding of LegK2 to its putative host cell interactants (<a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>, <a href="#B58" role="doc-biblioref" data-xml-rid="B58">58</a>), which may be crucial for LegK2 localization. Immunofluorescence observations suggested a diffused host cell cytoplasm localization of LegK2(K112M)-c-Myc (<a href="#F6">Fig. 6</a>D). To determine whether the localization of LegK2(K112M) could be dependent on other T4SS effectors, localization of LegK2(K112M)-c-Myc was investigated during infection of amoebae by a Δ<i>legK2</i> strain of <span class="named-content" data-type="genus-species">L. pneumophila</span>. Interestingly, infection triggered the localization of LegK2(K112M)-c-Myc into the plasma membrane (<a href="#F6">Fig. 6</a>D). Surprisingly, the same observation was made by incubating amoebae with <span class="named-content" data-type="genus-species">Escherichia coli</span>, suggesting that localization of LegK2(K112M) to the plasma membrane may be dependent on bacterial phagocytosis rather than on T4SS effector translocation and efficient <span class="named-content" data-type="genus-species">Legionella</span> infection (<a href="#F6">Fig. 6</a>D). Thus, when LegK2 is ectopically overproduced in amoebae, it is associated with the plasma membrane but not with the phagosomal membrane.</div></section></section><section id="sec-4" data-type="discussion"><h2>DISCUSSION</h2><div role="paragraph">The genomes of the five sequenced <span class="named-content" data-type="genus-species">L. pneumophila</span> strains are characterized by an unusual abundance of eucaryote-like proteins encoding genes; among them four protein kinases encoding genes have been identified (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>). Our extensive <i>in silico</i> analysis of the epidemic strain Lens genome identified one more protein kinase encoding gene, namely, <i>legK5</i> gene. It is noteworthy that <i>legK5</i> is specific to this strain. Interestingly, the LegK5 protein is homologous (50.99% identity) to a protein (UniprotKB entry number C6N1E1) from the strictly intracellular amoebal pathogen <span class="named-content" data-type="genus-species">Legionella drancourtii</span> (<a href="#B42" role="doc-biblioref" data-xml-rid="B42">42</a>). Moreover, <i>lpl2477</i>, located immediately downstream <i>legK5</i>, encodes a protein that displays 27% identity with an uncharacterized protein (UniProtKB entry number Q1RJU6) from <span class="named-content" data-type="genus-species">Rickettsia belii</span>, another intracellular bacteria that infects amoebae. Finally, the GC content of the <i>legK5</i> genomic region and the occurrence of a transposase-encoding gene strongly suggest that <i>legK5</i> has been acquired by a small insertion event associated with horizontal transfer during co-evolution with other bacteria in amoebae; it might represent a new instance of genomic recombination in amoebae, which have been considered a gene melting pot for evolution in a recent review (<a href="#B50" role="doc-biblioref" data-xml-rid="B50">50</a>). Multiple sequence comparisons of the kinase domains of LegK1 to LegK5 with eukaryotic and procaryotic protein kinase domains revealed that LegK1 and LegK3 cluster in the group of eukaryotic protein kinases, close to protein kinases from protozoa (data not shown), as previously reported (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>, <a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>). This observation raises the possibility of their initial acquisition by <span class="named-content" data-type="genus-species">L. pneumophila</span> from the distant eukaryotic organisms that are natural hosts for <span class="named-content" data-type="genus-species">Legionella</span>. In contrast, LegK2, LegK4, and LegK5 are not closely similar to eukaryotic protein kinases or prokaryotic enzymes. It is noteworthy that protein kinases have been identified in intracellular bacteria such as <span class="named-content" data-type="genus-species">Mycobacterium tuberculosis</span> (<a href="#B1" role="doc-biblioref" data-xml-rid="B1">1</a>, <a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>) and nonpathogenic species such as <span class="named-content" data-type="genus-species">Myxococcus xanthus</span> and <span class="named-content" data-type="genus-species">Anabaena</span> cyanobacteria (<a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>, <a href="#B57" role="doc-biblioref" data-xml-rid="B57">57</a>, <a href="#B74" role="doc-biblioref" data-xml-rid="B74">74</a>). The occurrence of protein kinase-encoding genes in free nonpathogenic bacterial species and in-depth analysis of GC content of various protein kinase encoding genes led researchers previously to conclude that bacterial protein kinases (“eukaryote-like”) existed before the divergence between eukaryotes and prokaryotes during evolution (<a href="#B27" role="doc-biblioref" data-xml-rid="B27">27</a>). Thus, even though the presence of horizontally acquired eukaryotic genes in amoeba pathogens such as <span class="named-content" data-type="genus-species">L. pneumophila</span> appears increasingly evident, the precise origin of protein kinase-encoding genes in <span class="named-content" data-type="genus-species">L. pneumophila</span> genomes has not been completely deciphered.</div><div role="paragraph">The eukaryotic kinase family is characterized by 11 conserved sequences, described as the Hanks subdomains, which are involved in ATP binding, and in phosphorylation reaction. LegK2 and LegK5 lack the Hanks subdomain VIII required for substrate recognition and phosphotransfer. However, both LegK2 and LegK5 are able to autophosphorylate and phosphorylate the eukaryotic protein kinase substrate myelin basic protein (MBP). More than 600 bacterial protein kinases have been identified through genome sequencing projects and homology-based comparisons with eukaryotic kinases and, less frequently, through direct experimental evidence (<a href="#B38" role="doc-biblioref" data-xml-rid="B38">38</a>, <a href="#B39" role="doc-biblioref" data-xml-rid="B39">39</a>). Generally, protein kinases from bacteria conserve the nucleotide binding region (subdomains I to III) and the core catalytic domain (subdomain VII known as Brenner's motif H-X-D-X<sub>4</sub>-N [<a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>]), but some of them display less homology as conserved enzymatic activity; for instance, SteC, which is required for host cell actin remodeling by <span class="named-content" data-type="genus-species">Salmonella</span>, is a functional protein kinase that exhibits only subdomains I to III involved in ATP binding (<a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a>). Moreover, although YihE from <span class="named-content" data-type="genus-species">E. coli</span> clearly possesses protein kinase activity, sequence analysis reveals little similarity to eukaryotic protein kinases, whereas tridimensional structure analysis reveals a kinase-like fold similar to that of choline kinase and aminoglycoside phosphotransferase (<a href="#B75" role="doc-biblioref" data-xml-rid="B75">75</a>). Likewise but to a lesser extent, LegK2 and LegK5 are functional in terms of phosphorylation even though they do not display all of the consensus domains of the eukaryotic protein kinase family. In contrast, LegK1 was not <i>in vitro</i> labeled by autophosphorylation and could not phosphorylate the MBP in standard <i>in vitro</i> assay conditions. It can be hypothesized that LegK1 must be activated to be functional. Indeed, it was previously shown that YpkA from <span class="named-content" data-type="genus-species">Y. pestis</span> is produced as an inactive protein kinase. This essential virulence factor is then translocated by a T3SS into the host cell, where it is activated by the host cell actin (<a href="#B33" role="doc-biblioref" data-xml-rid="B33">33</a>). On the other hand, we can assume that LegK1 cannot use the MBP as substrate but could phosphorylate other specific protein substrates. Indeed, it has been recently reported that LegK1 can specifically phosphorylate <i>in vitro</i> the eukaryotic IκBα protein but not MBP in cells, thus activating the NF-κB pathway (<a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>).</div><div role="paragraph">The bacterial protein kinases LegK1 to LegK4 are translocated into the host cell cytoplasm by the Dot/Icm T4SS during infection of <span class="named-content" data-type="genus-species">L. pneumophila</span>. Conversely, LegK5 appeared not to be translocated. It is noteworthy that another intracellular pathogen, <span class="named-content" data-type="genus-species">M. tuberculosis</span>, encodes 11 different protein kinases, among them one that is translocated into the host cell to subvert the host cell defenses (<a href="#B70" role="doc-biblioref" data-xml-rid="B70">70</a>, <a href="#B71" role="doc-biblioref" data-xml-rid="B71">71</a>), while others are tailored for bacterial signal transduction pathways (<a href="#B1" role="doc-biblioref" data-xml-rid="B1">1</a>, <a href="#B51" role="doc-biblioref" data-xml-rid="B51">51</a>). Likewise, LegK5 may be involved in bacterial transduction signaling pathways that control <span class="named-content" data-type="genus-species">Legionella</span> metabolism rather than in the control of host cell functions.</div><div role="paragraph">Interestingly, inactivation and/or kinase-dead substitution of the <i>legK2</i> gene resulted in a significant decrease in the virulence of <span class="named-content" data-type="genus-species">L. pneumophila</span> toward amoebae. It is noteworthy that T4SS effectors mutants are usually not altered in <span class="named-content" data-type="genus-species">Legionella</span> virulence, probably due to the high redundancy of these effectors genes (<a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a>). The <i>legK2</i> mutant virulence defect is not complemented by other <i>legK</i> gene expression vectors, thus showing a unique and key role for the LegK2 protein kinase in the control of the infectious cycle of <span class="named-content" data-type="genus-species">L. pneumophila</span>. Although the four other LegK proteins did not interfere individually with the overall ability of <span class="named-content" data-type="genus-species">Legionella</span> to replicate in the host, they might modulate subtle pathways of <span class="named-content" data-type="genus-species">Legionella</span> infectivity and/or have complementary or redundant cellular and functional targets. Indeed, although LegK1 has been reported to activate the NF-κB pathway and likely plays an important role in modulating macrophages defenses during <span class="named-content" data-type="genus-species">L. pneumophila</span> infection, deletion of the <i>legK1</i> gene has no notable effects on its intracellular replication (<a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>, <a href="#B45" role="doc-biblioref" data-xml-rid="B45">45</a>).</div><div role="paragraph">The <i>legK2</i> mutant strain is defective for recruiting the ER to the LCV, which results in delayed bacterial intracellular replication. Remodeling its phagosome by the recruitment of ER is a characteristic trait of the unique intracellular fate of <span class="named-content" data-type="genus-species">L. pneumophila</span> to evade the endocytic pathway (<a href="#B11" role="doc-biblioref" data-xml-rid="B11">11</a>, <a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>). Interestingly, two other Dot/Icm substrates, namely, SidC and SidJ, which do not display any homology with protein kinases, have been recently shown to be involved in ER recruitment (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B62" role="doc-biblioref" data-xml-rid="B62">62</a>). Mutation in <i>sidJ</i> results in a phenotype very similar to that of the <i>legK2</i> mutation, i.e., a <i>sidJ</i> mutant is considerably delayed in both ER recruitment on the LCV and intracellular growth. Moreover, SidJ was not associated with the phagosomal membrane. These results led researchers to conclude that SidJ modulates as-yet-unknown host cellular pathways to control the trafficking or the retention of ER-derived vesicles to the LCV (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>). It would be interesting to determine whether SidJ and LegK2 function synergistically and target the same or parallel pathways. On the other hand, SidC is an <span class="named-content" data-type="genus-species">L. pneumophila</span> effector protein that anchors to the host cell lipid phosphatidylinositol-4 phosphate and could act as a “tethering domain” for ER vesicle recruitment to the LCV (<a href="#B62" role="doc-biblioref" data-xml-rid="B62">62</a>). Indeed, as with <i>sidJ</i> and <i>legK2</i> mutants, an LCV harboring the <i>sidC</i> mutant recruits ER less efficiently than does the wild type. However, <i>sidC</i> mutant replicates at the same rate as the wild type, while localizing in calnexin-negative LCVs, thus indicating that the interaction with the ER might not be a prerequisite to form a <span class="named-content" data-type="genus-species">L. pneumophila</span> replication-permissive compartment. In a similar manner, <i>ralF</i> and <i>drrA</i> mutants replicate normally despite an inability to recruit the host GTPase ER-Golgi regulators Rab1 and Arf1 to the LCV, respectively (<a href="#B49" role="doc-biblioref" data-xml-rid="B49">49</a>, <a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>). This indicates that <span class="named-content" data-type="genus-species">L. pneumophila</span> simultaneously targets multiple and yet unknown functionally redundant pathways to contribute to the formation of replication-permissive LCVs, leading to the absence of a phenotype when any single pathway is disrupted. If we consider that a kinase-dead <i>legK2</i> mutant is altered both in ER recruitment and in intracellular <span class="named-content" data-type="genus-species">Legionella</span> replication, it can be proposed that the LegK2 protein kinase plays a key role for modulating one or several of the host cell signal transduction pathways that contribute to the formation of the LCV. Interestingly, this result highlights a new molecular mechanism, namely, protein phosphorylation, developed by <span class="named-content" data-type="genus-species">L. pneumophila</span> to subvert host cell defenses. The relationship between protein phosphorylation and virulence of <span class="named-content" data-type="genus-species">L. pneumophila</span> will be investigated further by characterizing the host cell pathways controlled by LegK2. Indeed, deciphering the individual contribution of each <span class="named-content" data-type="genus-species">Legionella</span> effector to the intracellular lifestyle of the bacteria remains a major and perhaps the principal challenge to understanding the molecular basis of <span class="named-content" data-type="genus-species">Legionella</span> virulence.</div></section><section id="acknowledgments" role="doc-acknowledgments"><h2>ACKNOWLEDGMENTS</h2><div data-type="financial-disclosure" role="paragraph">This study was supported by the PEPS program from the Centre National de la Recherche Scientifique and the Université Lyon 1. E.H. was supported by a fellowship from the Ministère de l'Enseignement Supérieur et de la Recherche. X.C. is supported by NIH grant AI023549 awarded to Howard Shuman.</div><div role="paragraph">We are grateful to N. Bailo for technical assistance and F. Letourneur for helpful discussions about <span class="named-content" data-type="genus-species">D. discoideum</span> manipulations. We thank the Dicty Stock Center of Columbia University for <span class="named-content" data-type="genus-species">D. discoideum</span> strains and Howard Shuman for the opportunity to perform translocation assays in his laboratory.</div></section></div></section><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="bibliography" role="doc-bibliography"><h2>REFERENCES</h2><div class="labeled" role="doc-biblioentry"><div class="label">1.</div><div id="B1" class="citations"><div class="citation"><div class="citation-content">Alber T. 2009. Signaling mechanisms of the <span class="named-content" data-type="genus-species">Mycobacterium tuberculosis</span> receptor Ser/Thr protein kinases. <em>Curr. Opin. Struct. Biol.</em> 19:650–657.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.sbi.2009.10.017">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19914822">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Signaling+mechanisms+of+the+Mycobacterium+tuberculosis+receptor+Ser%2FThr+protein+kinases&amp;author=T+Alber&amp;publication_year=2009&amp;pages=650-657&amp;doi=10.1016%2Fj.sbi.2009.10.017&amp;pmid=19914822">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">2.</div><div id="B2" class="citations"><div class="citation"><div class="citation-content">Amrein K. et al. 1995. Purification and characterization of recombinant human p50csk protein-tyrosine kinase from an <span class="named-content" data-type="genus-species">Escherichia coli</span> expression system overproducing the bacterial chaperones GroES and GroEL. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 92:1048–1052.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.92.4.1048">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7862631">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1995QG77000024">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Purification+and+characterization+of+recombinant+human+p50csk+protein-tyrosine+kinase+from+an+Escherichia+coli+expression+system+overproducing+the+bacterial+chaperones+GroES+and+GroEL&amp;author=K.+Amrein&amp;publication_year=1995&amp;pages=1048-1052&amp;doi=10.1073%2Fpnas.92.4.1048&amp;pmid=7862631">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">3.</div><div id="B3" class="citations"><div class="citation"><div class="citation-content">Andrews H. L., Vogel J. P., and Isberg R. R.. 1998. Identification of linked <span class="named-content" data-type="genus-species">Legionella pneumophila</span> genes essential for intracellular growth and evasion of the endocytic pathway. <em>Infect. Immun.</em> 66:950–958.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.66.3.950-958.1998">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9488381">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000072160900011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+linked+Legionella+pneumophila+genes+essential+for+intracellular+growth+and+evasion+of+the+endocytic+pathway&amp;author=H.+L.+Andrews&amp;author=J.+P.+Vogel&amp;author=R.+R.+Isberg&amp;publication_year=1998&amp;pages=950-958&amp;doi=10.1128%2FIAI.66.3.950-958.1998&amp;pmid=9488381">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">4.</div><div id="B4" class="citations"><div class="citation"><div class="citation-content">Barz C., Abahji T. N., Trulzsch K., and Heesemann J.. 2000. The <span class="named-content" data-type="genus-species">Yersinia</span> Ser/Thr protein kinase YpkA/YopO directly interacts with the small GTPases RhoA and Rac-1. <em>FEBS Lett.</em> 482:139–143.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0014-5793(00)02045-7">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11018537">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000089749100025">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Yersinia+Ser%2FThr+protein+kinase+YpkA%2FYopO+directly+interacts+with+the+small+GTPases+RhoA+and+Rac-1&amp;author=C.+Barz&amp;author=T.+N.+Abahji&amp;author=K.+Trulzsch&amp;author=J.+Heesemann&amp;publication_year=2000&amp;pages=139-143&amp;doi=10.1016%2FS0014-5793%2800%2902045-7&amp;pmid=11018537">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">5.</div><div id="B5" class="citations"><div class="citation"><div class="citation-content">Berger K. and Isberg R.. 1993. Two distinct defects in intracellular growth complemented by a single genetic locus in <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Mol. Microbiol.</em> 7:7–19.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.1993.tb01092.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8382332">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1993KE35300002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Two+distinct+defects+in+intracellular+growth+complemented+by+a+single+genetic+locus+in+Legionella+pneumophila&amp;author=K.+Berger&amp;author=R.+Isberg&amp;publication_year=1993&amp;pages=7-19&amp;doi=10.1111%2Fj.1365-2958.1993.tb01092.x&amp;pmid=8382332">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">6.</div><div id="B6" class="citations"><div class="citation"><div class="citation-content">Brenner S. 1987. Phosphotransferase sequence homology. <em>Nature</em> 329:21.</div><div class="external-links"><div><a href="https://doi.org/10.1038/329021a0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3041224">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Phosphotransferase+sequence+homology&amp;author=S+Brenner&amp;publication_year=1987&amp;pages=21&amp;doi=10.1038%2F329021a0&amp;pmid=3041224">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">7.</div><div id="B7" class="citations"><div class="citation"><div class="citation-content">Brombacher E. et al. 2009. Rab1 guanine nucleotide exchange factor SidM is a major phosphatidylinositol 4-phosphate-binding effector protein of <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>J. Biol. Chem.</em> 284:4846–4856.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.M807505200">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19095644">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000263416600013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rab1+guanine+nucleotide+exchange+factor+SidM+is+a+major+phosphatidylinositol+4-phosphate-binding+effector+protein+of+Legionella+pneumophila&amp;author=E.+Brombacher&amp;publication_year=2009&amp;pages=4846-4856&amp;doi=10.1074%2Fjbc.M807505200&amp;pmid=19095644">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">8.</div><div id="B8" class="citations"><div class="citation"><div class="citation-content">Cazalet C. et al. 2004. Evidence in the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> genome for exploitation of host cell functions and high genome plasticity. <em>Nat. Genet.</em> 36:1165–1173.</div><div class="external-links"><div><a href="https://doi.org/10.1038/ng1447">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15467720">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000224832800019">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Evidence+in+the+Legionella+pneumophila+genome+for+exploitation+of+host+cell+functions+and+high+genome+plasticity&amp;author=C.+Cazalet&amp;publication_year=2004&amp;pages=1165-1173&amp;doi=10.1038%2Fng1447&amp;pmid=15467720">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">9.</div><div id="B9" class="citations"><div class="citation"><div class="citation-content">Chao J. et al. 2010. Protein kinase and phosphatase signaling in <span class="named-content" data-type="genus-species">Mycobacterium tuberculosis</span> physiology and pathogenesis. <em>Biochim. Biophys. Acta</em> 1804:620–627.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.bbapap.2009.09.008">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19766738">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Protein+kinase+and+phosphatase+signaling+in+Mycobacterium+tuberculosis+physiology+and+pathogenesis&amp;author=J.+Chao&amp;publication_year=2010&amp;pages=620-627&amp;doi=10.1016%2Fj.bbapap.2009.09.008&amp;pmid=19766738">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">10.</div><div id="B10" class="citations"><div class="citation"><div class="citation-content">Charpentier X. et al. 2009. Chemical genetics reveals bacterial and host cell functions critical for type IV effector translocation by <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>PLoS Pathog.</em> 5:e1000501.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000501">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19578436">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269224500034">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Chemical+genetics+reveals+bacterial+and+host+cell+functions+critical+for+type+IV+effector+translocation+by+Legionella+pneumophila&amp;author=X.+Charpentier&amp;publication_year=2009&amp;pages=e1000501&amp;doi=10.1371%2Fjournal.ppat.1000501&amp;pmid=19578436">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">11.</div><div id="B11" class="citations"><div class="citation"><div class="citation-content">Coers J., Monahan C., and Roy C.. 1999. Modulation of phagosome biogenesis by <span class="named-content" data-type="genus-species">Legionella pneumophila</span> creates an organelle permissive for intracellular growth. <em>Nat. Cell Biol.</em> 1:451–453.</div><div class="external-links"><div><a href="https://doi.org/10.1038/15687">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10559990">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000083533100019">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Modulation+of+phagosome+biogenesis+by+Legionella+pneumophila+creates+an+organelle+permissive+for+intracellular+growth&amp;author=J.+Coers&amp;author=C.+Monahan&amp;author=C.+Roy&amp;publication_year=1999&amp;pages=451-453&amp;doi=10.1038%2F15687&amp;pmid=10559990">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">12.</div><div id="B12" class="citations"><div class="citation"><div class="citation-content">Cornillon S. et al. 2000. Phg1p is a nine-transmembrane protein superfamily member involved in <span class="named-content" data-type="genus-species">Dictyostelium</span> adhesion and phagocytosis. <em>J. Biol. Chem.</em> 275:34287–34292.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.M006725200">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10944536">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000165095300041">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Phg1p+is+a+nine-transmembrane+protein+superfamily+member+involved+in+Dictyostelium+adhesion+and+phagocytosis&amp;author=S.+Cornillon&amp;publication_year=2000&amp;pages=34287-34292&amp;doi=10.1074%2Fjbc.M006725200&amp;pmid=10944536">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">13.</div><div id="B13" class="citations"><div class="citation"><div class="citation-content">Cozzone A. J. 2005. Role of protein phosphorylation on serine/threonine and tyrosine in the virulence of bacterial pathogens. <em>J. Mol. Microbiol. Biotechnol.</em> 9:198–213.</div><div class="external-links"><div><a href="https://doi.org/10.1159/000089648">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16415593">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000234650900008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Role+of+protein+phosphorylation+on+serine%2Fthreonine+and+tyrosine+in+the+virulence+of+bacterial+pathogens&amp;author=A.+J+Cozzone&amp;publication_year=2005&amp;pages=198-213&amp;doi=10.1159%2F000089648&amp;pmid=16415593">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">14.</div><div id="B14" class="citations"><div class="citation"><div class="citation-content">Datsenko K. and Wanner B.. 2000. One-step inactivation of chromosomal genes in <span class="named-content" data-type="genus-species">Escherichia coli</span> K-12 using PCR products. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 97:6640–6645.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.120163297">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000087526300074">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products&amp;author=K.+Datsenko&amp;author=B.+Wanner&amp;publication_year=2000&amp;pages=6640-6645&amp;doi=10.1073%2Fpnas.120163297&amp;pmid=10829079">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">15.</div><div id="B15" class="citations"><div class="citation"><div class="citation-content">D'Auria G., Jiménez-Hernández N., Peris-Bondia F., Moya A., and Latorre A.. 2010. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> pangenome reveals strain-specific virulence factors. <em>BMC Genomics</em> 11:181.</div><div class="external-links"><div><a href="https://doi.org/10.1186/1471-2164-11-181">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20236513">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000277046700001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+pangenome+reveals+strain-specific+virulence+factors&amp;author=G.+D%27Auria&amp;author=N.+Jim%C3%A9nez-Hern%C3%A1ndez&amp;author=F.+Peris-Bondia&amp;author=A.+Moya&amp;author=A.+Latorre&amp;publication_year=2010&amp;pages=181&amp;doi=10.1186%2F1471-2164-11-181&amp;pmid=20236513">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">16.</div><div id="B16" class="citations"><div class="citation"><div class="citation-content">de Felipe K. et al. 2005. Evidence for acquisition of <span class="named-content" data-type="genus-species">Legionella</span> type IV secretion substrates via interdomain horizontal gene transfer. <em>J. Bacteriol.</em> 187:7716–7726.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.187.22.7716-7726.2005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16267296">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Evidence+for+acquisition+of+Legionella+type+IV+secretion+substrates+via+interdomain+horizontal+gene+transfer&amp;author=K.+de+Felipe&amp;publication_year=2005&amp;pages=7716-7726&amp;doi=10.1128%2FJB.187.22.7716-7726.2005&amp;pmid=16267296">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">17.</div><div id="B17" class="citations"><div class="citation"><div class="citation-content">de Felipe K. S. et al. 2008. <span class="named-content" data-type="genus-species">Legionella</span> eukaryotic-like type IV substrates interfere with organelle trafficking. <em>PLoS Pathog.</em> 4:e1000117.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000117">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18670632">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000259783100005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+eukaryotic-like+type+IV+substrates+interfere+with+organelle+trafficking&amp;author=K.+S.+de+Felipe&amp;publication_year=2008&amp;pages=e1000117&amp;doi=10.1371%2Fjournal.ppat.1000117&amp;pmid=18670632">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">18.</div><div id="B18" class="citations"><div class="citation"><div class="citation-content">Degtyar E., Zusman T., Ehrlich M., and Segal G.. 2009. A <span class="named-content" data-type="genus-species">Legionella</span> effector acquired from protozoa is involved in sphingolipids metabolism and is targeted to the host cell mitochondria. <em>Cell Microbiol.</em> 11:1219–1235.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2009.01328.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19438520">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000267658300007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+Legionella+effector+acquired+from+protozoa+is+involved+in+sphingolipids+metabolism+and+is+targeted+to+the+host+cell+mitochondria&amp;author=E.+Degtyar&amp;author=T.+Zusman&amp;author=M.+Ehrlich&amp;author=G.+Segal&amp;publication_year=2009&amp;pages=1219-1235&amp;doi=10.1111%2Fj.1462-5822.2009.01328.x&amp;pmid=19438520">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">19.</div><div id="B19" class="citations"><div class="citation"><div class="citation-content">Deutscher J. and Saier M. H.. 2005. Ser/Thr/Tyr protein phosphorylation in bacteria: for long time neglected, now well established. <em>J. Mol. Microbiol. Biotechnol.</em> 9:125–131.</div><div class="external-links"><div><a href="https://doi.org/10.1159/000089641">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16415586">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Ser%2FThr%2FTyr+protein+phosphorylation+in+bacteria%3A+for+long+time+neglected%2C+now+well+established&amp;author=J.+Deutscher&amp;author=M.+H.+Saier&amp;publication_year=2005&amp;pages=125-131&amp;doi=10.1159%2F000089641&amp;pmid=16415586">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">20.</div><div id="B20" class="citations"><div class="citation"><div class="citation-content">Echenique J., Kadioglu A., Romao S., Andrew P. W., and Trombe M. C.. 2004. Protein serine/threonine kinase StkP positively controls virulence and competence in <span class="named-content" data-type="genus-species">Streptococcus pneumoniae</span>. <em>Infect. Immun.</em> 72:2434–2437.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.72.4.2434-2437.2004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15039376">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000220481600074">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Protein+serine%2Fthreonine+kinase+StkP+positively+controls+virulence+and+competence+in+Streptococcus+pneumoniae&amp;author=J.+Echenique&amp;author=A.+Kadioglu&amp;author=S.+Romao&amp;author=P.+W.+Andrew&amp;author=M.+C.+Trombe&amp;publication_year=2004&amp;pages=2434-2437&amp;doi=10.1128%2FIAI.72.4.2434-2437.2004&amp;pmid=15039376">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">21.</div><div id="B21" class="citations"><div class="citation"><div class="citation-content">Ensminger A. and Isberg R.. 2010. E3 ubiquitin ligase activity and targeting of BAT3 by multiple <span class="named-content" data-type="genus-species">Legionella pneumophila</span> translocated substrates. <em>Infect. Immun.</em> 78:3905–3919.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.00344-10">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20547746">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=E3+ubiquitin+ligase+activity+and+targeting+of+BAT3+by+multiple+Legionella+pneumophila+translocated+substrates&amp;author=A.+Ensminger&amp;author=R.+Isberg&amp;publication_year=2010&amp;pages=3905-3919&amp;doi=10.1128%2FIAI.00344-10&amp;pmid=20547746">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">22.</div><div id="B22" class="citations"><div class="citation"><div class="citation-content">Ensminger A. W. and Isberg R. R.. 2009. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> Dot/Icm translocated substrates: a sum of parts. <em>Curr. Opin. Microbiol.</em> 12:67–73.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2008.12.004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19157961">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000264319100011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+Dot%2FIcm+translocated+substrates%3A+a+sum+of+parts&amp;author=A.+W.+Ensminger&amp;author=R.+R.+Isberg&amp;publication_year=2009&amp;pages=67-73&amp;doi=10.1016%2Fj.mib.2008.12.004&amp;pmid=19157961">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">23.</div><div id="B23" class="citations"><div class="citation"><div class="citation-content">Ferhat M. et al. 2009. The TolC protein of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> plays a major role in multidrug resistance and the early steps of host invasion. <em>PLoS One</em> 4:e7732.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.pone.0007732">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19888467">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+TolC+protein+of+Legionella+pneumophila+plays+a+major+role+in+multidrug+resistance+and+the+early+steps+of+host+invasion&amp;author=M.+Ferhat&amp;publication_year=2009&amp;pages=e7732&amp;doi=10.1371%2Fjournal.pone.0007732&amp;pmid=19888467">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">24.</div><div id="B24" class="citations"><div class="citation"><div class="citation-content">Gao L. Y., Harb O. S., and AbuKwaik Y.. 1997. Utilization of similar mechanisms by <span class="named-content" data-type="genus-species">Legionella pneumophila</span> to parasitize two evolutionarily distant host cells, mammalian macrophages and protozoa. <em>Infect. Immun.</em> 65:4738–4746.</div><div class="external-links"><div><a href="https://doi.org/10.1128/iai.65.11.4738-4746.1997">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9353059">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1997YD17600053">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Utilization+of+similar+mechanisms+by+Legionella+pneumophila+to+parasitize+two+evolutionarily+distant+host+cells%2C+mammalian+macrophages+and+protozoa&amp;author=L.+Y.+Gao&amp;author=O.+S.+Harb&amp;author=Y.+AbuKwaik&amp;publication_year=1997&amp;pages=4738-4746&amp;doi=10.1128%2Fiai.65.11.4738-4746.1997&amp;pmid=9353059">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">25.</div><div id="B25" class="citations"><div class="citation"><div class="citation-content">Ge J. et al. 2009. A <span class="named-content" data-type="genus-species">Legionella</span> type IV effector activates the NF-κB pathway by phosphorylating the IκB family of inhibitors. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 106:13725–13730.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0907200106">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19666608">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269078700020">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+Legionella+type+IV+effector+activates+the+NF-%CE%BAB+pathway+by+phosphorylating+the+I%CE%BAB+family+of+inhibitors&amp;author=J.+Ge&amp;publication_year=2009&amp;pages=13725-13730&amp;doi=10.1073%2Fpnas.0907200106&amp;pmid=19666608">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">26.</div><div id="B26" class="citations"><div class="citation"><div class="citation-content">Glöckner G. et al. 2008. Identification and characterization of a new conjugation/type IVA secretion system (<i>trb/tra</i>) of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> Corby localized on two mobile genomic islands. <em>Int. J. Med. Microbiol.</em> 298:411–428.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.ijmm.2007.07.012">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17888731">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000258027000006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+and+characterization+of+a+new+conjugation%2Ftype+IVA+secretion+system+%28trb%2Ftra%29+of+Legionella+pneumophila+Corby+localized+on+two+mobile+genomic+islands&amp;author=G.+Gl%C3%B6ckner&amp;publication_year=2008&amp;pages=411-428&amp;doi=10.1016%2Fj.ijmm.2007.07.012&amp;pmid=17888731">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">27.</div><div id="B27" class="citations"><div class="citation"><div class="citation-content">Han G. and Zhang C.. 2001. On the origin of Ser/Thr kinases in a prokaryote. <em>FEMS Microbiol. Lett.</em> 200:79–84.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1574-6968.2001.tb10696.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11410353">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=On+the+origin+of+Ser%2FThr+kinases+in+a+prokaryote&amp;author=G.+Han&amp;author=C.+Zhang&amp;publication_year=2001&amp;pages=79-84&amp;doi=10.1111%2Fj.1574-6968.2001.tb10696.x&amp;pmid=11410353">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">28.</div><div id="B28" class="citations"><div class="citation"><div class="citation-content">Hanks S. 2003. Genomic analysis of the eukaryotic protein kinase superfamily: a perspective. <em>Genome Biol.</em> 4:111.</div><div class="external-links"><div><a href="https://doi.org/10.1186/gb-2003-4-5-111">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12734000">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genomic+analysis+of+the+eukaryotic+protein+kinase+superfamily%3A+a+perspective&amp;author=S+Hanks&amp;publication_year=2003&amp;pages=111&amp;doi=10.1186%2Fgb-2003-4-5-111&amp;pmid=12734000">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">29.</div><div id="B29" class="citations"><div class="citation"><div class="citation-content">Harb O., Gao L., and Abu Kwaik Y.. 2000. From protozoa to mammalian cells: a new paradigm in the life cycle of intracellular bacterial pathogens. <em>Environ. Microbiol.</em> 2:251–265.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1462-2920.2000.00112.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11200426">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000087843500002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=From+protozoa+to+mammalian+cells%3A+a+new+paradigm+in+the+life+cycle+of+intracellular+bacterial+pathogens&amp;author=O.+Harb&amp;author=L.+Gao&amp;author=Y.+Abu+Kwaik&amp;publication_year=2000&amp;pages=251-265&amp;doi=10.1046%2Fj.1462-2920.2000.00112.x&amp;pmid=11200426">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">30.</div><div id="B30" class="citations"><div class="citation"><div class="citation-content">Ingmundson A., Delprato A., Lambright D., and Roy C.. 2007. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> proteins that regulate Rab1 membrane cycling. <em>Nature</em> 450:365–369.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nature06336">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17952054">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000250918600043">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+proteins+that+regulate+Rab1+membrane+cycling&amp;author=A.+Ingmundson&amp;author=A.+Delprato&amp;author=D.+Lambright&amp;author=C.+Roy&amp;publication_year=2007&amp;pages=365-369&amp;doi=10.1038%2Fnature06336&amp;pmid=17952054">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">31.</div><div id="B31" class="citations"><div class="citation"><div class="citation-content">Ish-Horowicz D. and Burke J.. 1981. Rapid and efficient cosmid cloning. <em>Nucleic Acids Res.</em> 9:2989–2998.</div><div class="external-links"><div><a href="https://doi.org/10.1093/nar/9.13.2989">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/6269067">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1981LX80600001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rapid+and+efficient+cosmid+cloning&amp;author=D.+Ish-Horowicz&amp;author=J.+Burke&amp;publication_year=1981&amp;pages=2989-2998&amp;doi=10.1093%2Fnar%2F9.13.2989&amp;pmid=6269067">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">32.</div><div id="B32" class="citations"><div class="citation"><div class="citation-content">Jin H. and Pancholi V.. 2006. Identification and biochemical characterization of a eukaryotic-type serine/threonine kinase and its cognate phosphatase in <span class="named-content" data-type="genus-species">Streptococcus pyogenes</span>: their biological functions and substrate identification. <em>J. Mol. Biol.</em> 357:1351–1372.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.jmb.2006.01.020">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16487973">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000236629400002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+and+biochemical+characterization+of+a+eukaryotic-type+serine%2Fthreonine+kinase+and+its+cognate+phosphatase+in+Streptococcus+pyogenes%3A+their+biological+functions+and+substrate+identification&amp;author=H.+Jin&amp;author=V.+Pancholi&amp;publication_year=2006&amp;pages=1351-1372&amp;doi=10.1016%2Fj.jmb.2006.01.020&amp;pmid=16487973">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">33.</div><div id="B33" class="citations"><div class="citation"><div class="citation-content">Juris S., Rudolph A., Huddler D., Orth K., and Dixon J.. 2000. A distinctive role for the <span class="named-content" data-type="genus-species">Yersinia</span> protein kinase: actin binding, kinase activation, and cytoskeleton disruption. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 97:9431–9436.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.170281997">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10920208">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000088840500019">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+distinctive+role+for+the+Yersinia+protein+kinase%3A+actin+binding%2C+kinase+activation%2C+and+cytoskeleton+disruption&amp;author=S.+Juris&amp;author=A.+Rudolph&amp;author=D.+Huddler&amp;author=K.+Orth&amp;author=J.+Dixon&amp;publication_year=2000&amp;pages=9431-9436&amp;doi=10.1073%2Fpnas.170281997&amp;pmid=10920208">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">34.</div><div id="B34" class="citations"><div class="citation"><div class="citation-content">Kagan J. and Roy C.. 2002. <span class="named-content" data-type="genus-species">Legionella</span> phagosomes intercept vesicular traffic from endoplasmic reticulum exit sites. <em>Nat. Cell Biol.</em> 4:945–954.</div><div class="external-links"><div><a href="https://doi.org/10.1038/ncb883">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12447391">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000179571800014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+phagosomes+intercept+vesicular+traffic+from+endoplasmic+reticulum+exit+sites&amp;author=J.+Kagan&amp;author=C.+Roy&amp;publication_year=2002&amp;pages=945-954&amp;doi=10.1038%2Fncb883&amp;pmid=12447391">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">35.</div><div id="B35" class="citations"><div class="citation"><div class="citation-content">Kawachi H., Fujikawa A., Maeda N., and Noda M.. 2001. Identification of GIT1/Cat-1 as a substrate molecule of protein tyrosine phosphatase zeta /beta by the yeast substrate-trapping system. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 98:6593–6598.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.041608698">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11381105">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+GIT1%2FCat-1+as+a+substrate+molecule+of+protein+tyrosine+phosphatase+zeta+%2Fbeta+by+the+yeast+substrate-trapping+system&amp;author=H.+Kawachi&amp;author=A.+Fujikawa&amp;author=N.+Maeda&amp;author=M.+Noda&amp;publication_year=2001&amp;pages=6593-6598&amp;doi=10.1073%2Fpnas.041608698&amp;pmid=11381105">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">36.</div><div id="B36" class="citations"><div class="citation"><div class="citation-content">Kim E., Charpentier X., Torres-Urquidy O., McEvoy M., and Rensing C.. 2009. The metal efflux island of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> is not required for survival in macrophages and amoebas. <em>FEMS Microbiol. Lett.</em> 301:164–170.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1574-6968.2009.01813.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19895645">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+metal+efflux+island+of+Legionella+pneumophila+is+not+required+for+survival+in+macrophages+and+amoebas&amp;author=E.+Kim&amp;author=X.+Charpentier&amp;author=O.+Torres-Urquidy&amp;author=M.+McEvoy&amp;author=C.+Rensing&amp;publication_year=2009&amp;pages=164-170&amp;doi=10.1111%2Fj.1574-6968.2009.01813.x&amp;pmid=19895645">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">37.</div><div id="B37" class="citations"><div class="citation"><div class="citation-content">Kristich C. J., Wells C. L., and Dunny G. M.. 2007. A eukaryotic-type Ser/Thr kinase in <span class="named-content" data-type="genus-species">Enterococcus faecalis</span> mediates antimicrobial resistance and intestinal persistence. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 104:3508–3513.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0608742104">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17360674">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000244661400086">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+eukaryotic-type+Ser%2FThr+kinase+in+Enterococcus+faecalis+mediates+antimicrobial+resistance+and+intestinal+persistence&amp;author=C.+J.+Kristich&amp;author=C.+L.+Wells&amp;author=G.+M.+Dunny&amp;publication_year=2007&amp;pages=3508-3513&amp;doi=10.1073%2Fpnas.0608742104&amp;pmid=17360674">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">38.</div><div id="B38" class="citations"><div class="citation"><div class="citation-content">Krupa A., Abhinandan K., and Srinivasan N.. 2004. KinG: a database of protein kinases in genomes. <em>Nucleic Acids Res.</em> 32:D153–D155.</div><div class="external-links"><div><a href="https://doi.org/10.1093/nar/gkh019">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14681382">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000188079000034">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=KinG%3A+a+database+of+protein+kinases+in+genomes&amp;author=A.+Krupa&amp;author=K.+Abhinandan&amp;author=N.+Srinivasan&amp;publication_year=2004&amp;pages=D153-D155&amp;doi=10.1093%2Fnar%2Fgkh019&amp;pmid=14681382">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">39.</div><div id="B39" class="citations"><div class="citation"><div class="citation-content">Krupa A. and Srinivasan N.. 2005. Diversity in domain architectures of Ser/Thr kinases and their homologues in prokaryotes. <em>BMC Genomics</em> 6:129.</div><div class="external-links"><div><a href="https://doi.org/10.1186/1471-2164-6-129">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16171520">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Diversity+in+domain+architectures+of+Ser%2FThr+kinases+and+their+homologues+in+prokaryotes&amp;author=A.+Krupa&amp;author=N.+Srinivasan&amp;publication_year=2005&amp;pages=129&amp;doi=10.1186%2F1471-2164-6-129&amp;pmid=16171520">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">40.</div><div id="B40" class="citations"><div class="citation"><div class="citation-content">Kubori T., Hyakutake A., and Nagai H.. 2008. <span class="named-content" data-type="genus-species">Legionella</span> translocates an E3 ubiquitin ligase that has multiple U-boxes with distinct functions. <em>Mol. Microbiol.</em> 67:1307–1319.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2008.06124.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18284575">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+translocates+an+E3+ubiquitin+ligase+that+has+multiple+U-boxes+with+distinct+functions&amp;author=T.+Kubori&amp;author=A.+Hyakutake&amp;author=H.+Nagai&amp;publication_year=2008&amp;pages=1307-1319&amp;doi=10.1111%2Fj.1365-2958.2008.06124.x&amp;pmid=18284575">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">41.</div><div id="B41" class="citations"><div class="citation"><div class="citation-content">Laemmli U. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. <em>Nature</em> 227:680–685.</div><div class="external-links"><div><a href="https://doi.org/10.1038/227680a0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/5432063">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1970H022600020">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Cleavage+of+structural+proteins+during+the+assembly+of+the+head+of+bacteriophage+T4&amp;author=U+Laemmli&amp;publication_year=1970&amp;pages=680-685&amp;doi=10.1038%2F227680a0&amp;pmid=5432063">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">42.</div><div id="B42" class="citations"><div class="citation"><div class="citation-content">La Scola B. et al. 2004. <span class="named-content" data-type="genus-species">Legionella drancourtii</span> sp. nov., a strictly intracellular amoebal pathogen. <em>Int. J. Syst. Evol. Microbiol.</em> 54:699–703.</div><div class="external-links"><div><a href="https://doi.org/10.1099/ijs.0.02455-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15143011">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+drancourtii+sp.+nov.%2C+a+strictly+intracellular+amoebal+pathogen&amp;author=B.+La+Scola&amp;publication_year=2004&amp;pages=699-703&amp;doi=10.1099%2Fijs.0.02455-0&amp;pmid=15143011">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">43.</div><div id="B43" class="citations"><div class="citation"><div class="citation-content">Liu Y. and Luo Z.. 2007. The <span class="named-content" data-type="genus-species">Legionella pneumophila</span> effector SidJ is required for efficient recruitment of endoplasmic reticulum proteins to the bacterial phagosome. <em>Infect. Immun.</em> 75:592–603.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.01278-06">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17101649">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+pneumophila+effector+SidJ+is+required+for+efficient+recruitment+of+endoplasmic+reticulum+proteins+to+the+bacterial+phagosome&amp;author=Y.+Liu&amp;author=Z.+Luo&amp;publication_year=2007&amp;pages=592-603&amp;doi=10.1128%2FIAI.01278-06&amp;pmid=17101649">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">44.</div><div id="B44" class="citations"><div class="citation"><div class="citation-content">Lomma M. et al. 2010. The <span class="named-content" data-type="genus-species">Legionella pneumophila</span> F-box protein Lpp2082 (AnkB) modulates ubiquitination of the host protein parvin B and promotes intracellular replication. <em>Cell Microbiol.</em> 12:1272–1291.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2010.01467.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20345489">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+pneumophila+F-box+protein+Lpp2082+%28AnkB%29+modulates+ubiquitination+of+the+host+protein+parvin+B+and+promotes+intracellular+replication&amp;author=M.+Lomma&amp;publication_year=2010&amp;pages=1272-1291&amp;doi=10.1111%2Fj.1462-5822.2010.01467.x&amp;pmid=20345489">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">45.</div><div id="B45" class="citations"><div class="citation"><div class="citation-content">Losick V., Haenssler E., Moy M., and Isberg R.. 2010. LnaB: a <span class="named-content" data-type="genus-species">Legionella pneumophila</span> activator of NF-κB. <em>Cell Microbiol</em>. 12:1083–1097.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2010.01452.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20148897">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000279721400006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=LnaB%3A+a+Legionella+pneumophila+activator+of+NF-%CE%BAB&amp;author=V.+Losick&amp;author=E.+Haenssler&amp;author=M.+Moy&amp;author=R.+Isberg&amp;publication_year=2010&amp;pages=1083-1097&amp;doi=10.1111%2Fj.1462-5822.2010.01452.x&amp;pmid=20148897">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">46.</div><div id="B46" class="citations"><div class="citation"><div class="citation-content">Lurie-Weinberger M. et al. 2010. The origins of eukaryotic-like proteins in <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Int. J. Med. Microbiol</em>. 300:470–481.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.ijmm.2010.04.016">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20537944">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+origins+of+eukaryotic-like+proteins+in+Legionella+pneumophila&amp;author=M.+Lurie-Weinberger&amp;publication_year=2010&amp;pages=470-481&amp;doi=10.1016%2Fj.ijmm.2010.04.016&amp;pmid=20537944">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">47.</div><div id="B47" class="citations"><div class="citation"><div class="citation-content">Lux R. and Shi W.. 2005. A novel bacterial signaling system with a combination of a Ser/Thr kinase cascade and a His/Asp two-component system. <em>Mol. Microbiol.</em> 58:345–348.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2005.04856.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16194223">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+novel+bacterial+signaling+system+with+a+combination+of+a+Ser%2FThr+kinase+cascade+and+a+His%2FAsp+two-component+system&amp;author=R.+Lux&amp;author=W.+Shi&amp;publication_year=2005&amp;pages=345-348&amp;doi=10.1111%2Fj.1365-2958.2005.04856.x&amp;pmid=16194223">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">48.</div><div id="B48" class="citations"><div class="citation"><div class="citation-content">Machner M. and Isberg R.. 2007. A bifunctional bacterial protein links GDI displacement to Rab1 activation. <em>Science</em> 318:974–977.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1149121">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17947549">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000250788500043">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+bifunctional+bacterial+protein+links+GDI+displacement+to+Rab1+activation&amp;author=M.+Machner&amp;author=R.+Isberg&amp;publication_year=2007&amp;pages=974-977&amp;doi=10.1126%2Fscience.1149121&amp;pmid=17947549">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">49.</div><div id="B49" class="citations"><div class="citation"><div class="citation-content">Machner M. and Isberg R.. 2006. Targeting of host Rab GTPase function by the intravacuolar pathogen <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Dev. Cell</em> 11:47–56.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.devcel.2006.05.013">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16824952">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000239128300008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Targeting+of+host+Rab+GTPase+function+by+the+intravacuolar+pathogen+Legionella+pneumophila&amp;author=M.+Machner&amp;author=R.+Isberg&amp;publication_year=2006&amp;pages=47-56&amp;doi=10.1016%2Fj.devcel.2006.05.013&amp;pmid=16824952">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">50.</div><div id="B50" class="citations"><div class="citation"><div class="citation-content">Moliner C., Fournier P., and Raoult D.. 2010. Genome analysis of microorganisms living in amoebae reveals a melting pot of evolution. <em>FEMS Microbiol. Rev</em>. 34:281–294.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1574-6976.2009.00209.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20132312">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000276246300003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genome+analysis+of+microorganisms+living+in+amoebae+reveals+a+melting+pot+of+evolution&amp;author=C.+Moliner&amp;author=P.+Fournier&amp;author=D.+Raoult&amp;publication_year=2010&amp;pages=281-294&amp;doi=10.1111%2Fj.1574-6976.2009.00209.x&amp;pmid=20132312">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">51.</div><div id="B51" class="citations"><div class="citation"><div class="citation-content">Molle V. and Kremer L.. 2010. Division and cell envelope regulation by Ser/Thr phosphorylation: <span class="named-content" data-type="genus-species">Mycobacterium</span> shows the way. <em>Mol. Microbiol</em>. 75:1064–1077.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2009.07041.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20487298">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Division+and+cell+envelope+regulation+by+Ser%2FThr+phosphorylation%3A+Mycobacterium+shows+the+way&amp;author=V.+Molle&amp;author=L.+Kremer&amp;publication_year=2010&amp;pages=1064-1077&amp;doi=10.1111%2Fj.1365-2958.2009.07041.x&amp;pmid=20487298">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">52.</div><div id="B52" class="citations"><div class="citation"><div class="citation-content">Molofsky A. and Swanson M.. 2004. Differentiate to thrive: lessons from the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> life cycle. <em>Mol. Microbiol.</em> 53:29–40.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2004.04129.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15225301">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000222208100004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Differentiate+to+thrive%3A+lessons+from+the+Legionella+pneumophila+life+cycle&amp;author=A.+Molofsky&amp;author=M.+Swanson&amp;publication_year=2004&amp;pages=29-40&amp;doi=10.1111%2Fj.1365-2958.2004.04129.x&amp;pmid=15225301">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">53.</div><div id="B53" class="citations"><div class="citation"><div class="citation-content">Moncrief N., Kretsinger R., and Goodman M.. 1990. Evolution of EF-hand calcium-modulated proteins. I. Relationships based on amino acid sequences. <em>J. Mol. Evol.</em> 30:522–562.</div><div class="external-links"><div><a href="https://doi.org/10.1007/BF02101108">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/2115931">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1990DF74600008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Evolution+of+EF-hand+calcium-modulated+proteins.+I.+Relationships+based+on+amino+acid+sequences&amp;author=N.+Moncrief&amp;author=R.+Kretsinger&amp;author=M.+Goodman&amp;publication_year=1990&amp;pages=522-562&amp;doi=10.1007%2FBF02101108&amp;pmid=2115931">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">54.</div><div id="B54" class="citations"><div class="citation"><div class="citation-content">Murata T. et al. 2006. The <span class="named-content" data-type="genus-species">Legionella pneumophila</span> effector protein DrrA is a Rab1 guanine nucleotide-exchange factor. <em>Nat. Cell Biol.</em> 8:971–977.</div><div class="external-links"><div><a href="https://doi.org/10.1038/ncb1463">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16906144">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000240241700013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+pneumophila+effector+protein+DrrA+is+a+Rab1+guanine+nucleotide-exchange+factor&amp;author=T.+Murata&amp;publication_year=2006&amp;pages=971-977&amp;doi=10.1038%2Fncb1463&amp;pmid=16906144">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">55.</div><div id="B55" class="citations"><div class="citation"><div class="citation-content">Müller-Taubenberger A. et al. 2001. Calreticulin and calnexin in the endoplasmic reticulum are important for phagocytosis. <em>EMBO J.</em> 20:6772–6782.</div><div class="external-links"><div><a href="https://doi.org/10.1093/emboj/20.23.6772">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11726513">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Calreticulin+and+calnexin+in+the+endoplasmic+reticulum+are+important+for+phagocytosis&amp;author=A.+M%C3%BCller-Taubenberger&amp;publication_year=2001&amp;pages=6772-6782&amp;doi=10.1093%2Femboj%2F20.23.6772&amp;pmid=11726513">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">56.</div><div id="B56" class="citations"><div class="citation"><div class="citation-content">Nagai H., Kagan J., Zhu X., Kahn R., and Roy C.. 2002. A bacterial guanine nucleotide exchange factor activates ARF on <span class="named-content" data-type="genus-species">Legionella</span> phagosomes. <em>Science</em> 295:679–682.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1067025">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11809974">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000173560900046">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+bacterial+guanine+nucleotide+exchange+factor+activates+ARF+on+Legionella+phagosomes&amp;author=H.+Nagai&amp;author=J.+Kagan&amp;author=X.+Zhu&amp;author=R.+Kahn&amp;author=C.+Roy&amp;publication_year=2002&amp;pages=679-682&amp;doi=10.1126%2Fscience.1067025&amp;pmid=11809974">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">57.</div><div id="B57" class="citations"><div class="citation"><div class="citation-content">Nariya H. and Inouye S.. 2006. A protein Ser/Thr kinase cascade negatively regulates the DNA-binding activity of MrpC, a smaller form of which may be necessary for the <span class="named-content" data-type="genus-species">Myxococcus xanthus</span> development. <em>Mol. Microbiol.</em> 60:1205–1217.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2006.05178.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16689796">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000237423300011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+protein+Ser%2FThr+kinase+cascade+negatively+regulates+the+DNA-binding+activity+of+MrpC%2C+a+smaller+form+of+which+may+be+necessary+for+the+Myxococcus+xanthus+development&amp;author=H.+Nariya&amp;author=S.+Inouye&amp;publication_year=2006&amp;pages=1205-1217&amp;doi=10.1111%2Fj.1365-2958.2006.05178.x&amp;pmid=16689796">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">58.</div><div id="B58" class="citations"><div class="citation"><div class="citation-content">Patel R. et al. 1999. DRBP76, a double-stranded RNA-binding nuclear protein, is phosphorylated by the interferon-induced protein kinase, PKR. <em>J. Biol. Chem.</em> 274:20432–20437.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.274.29.20432">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10400669">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=DRBP76%2C+a+double-stranded+RNA-binding+nuclear+protein%2C+is+phosphorylated+by+the+interferon-induced+protein+kinase%2C+PKR&amp;author=R.+Patel&amp;publication_year=1999&amp;pages=20432-20437&amp;doi=10.1074%2Fjbc.274.29.20432&amp;pmid=10400669">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">59.</div><div id="B59" class="citations"><div class="citation"><div class="citation-content">Poh J. et al. 2008. SteC is a <span class="named-content" data-type="genus-species">Salmonella</span> kinase required for SPI-2-dependent F-actin remodeling. <em>Cell. Microbiol.</em> 10:20–30.</div><div class="external-links"><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17645553">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000253096200003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=SteC+is+a+Salmonella+kinase+required+for+SPI-2-dependent+F-actin+remodeling&amp;author=J.+Poh&amp;publication_year=2008&amp;pages=20-30&amp;pmid=17645553">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">60.</div><div id="B60" class="citations"><div class="citation"><div class="citation-content">Pope C., Dhand L., and Cianciotto N.. 1994. Random mutagenesis of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> with mini-Tn10. <em>FEMS Microbiol. Lett.</em> 124:107–111.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1574-6968.1994.tb07269.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8001761">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Random+mutagenesis+of+Legionella+pneumophila+with+mini-Tn10&amp;author=C.+Pope&amp;author=L.+Dhand&amp;author=N.+Cianciotto&amp;publication_year=1994&amp;pages=107-111&amp;doi=10.1111%2Fj.1574-6968.1994.tb07269.x&amp;pmid=8001761">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">61.</div><div id="B61" class="citations"><div class="citation"><div class="citation-content">Price C. et al. 2009. Molecular mimicry by an F-box effector of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> hijacks a conserved polyubiquitination machinery within macrophages and protozoa. <em>PLoS Pathog.</em> 5:e1000704.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000704">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20041211">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Molecular+mimicry+by+an+F-box+effector+of+Legionella+pneumophila+hijacks+a+conserved+polyubiquitination+machinery+within+macrophages+and+protozoa&amp;author=C.+Price&amp;publication_year=2009&amp;pages=e1000704&amp;doi=10.1371%2Fjournal.ppat.1000704&amp;pmid=20041211">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">62.</div><div id="B62" class="citations"><div class="citation"><div class="citation-content">Ragaz C. et al. 2008. The <span class="named-content" data-type="genus-species">Legionella pneumophila</span> phosphatidylinositol-4 phosphate-binding type IV substrate SidC recruits endoplasmic reticulum vesicles to a replication-permissive vacuole. <em>Cell Microbiol.</em> 10:2416–2433.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2008.01219.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18673369">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Legionella+pneumophila+phosphatidylinositol-4+phosphate-binding+type+IV+substrate+SidC+recruits+endoplasmic+reticulum+vesicles+to+a+replication-permissive+vacuole&amp;author=C.+Ragaz&amp;publication_year=2008&amp;pages=2416-2433&amp;doi=10.1111%2Fj.1462-5822.2008.01219.x&amp;pmid=18673369">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">63.</div><div id="B63" class="citations"><div class="citation"><div class="citation-content">Rowbotham T. 1980. Preliminary report on the pathogenicity of <span class="named-content" data-type="genus-species">Legionella pneumophila</span> for freshwater and soil amoebae. <em>J. Clin. Pathol.</em> 33:1179–1183.</div><div class="external-links"><div><a href="https://doi.org/10.1136/jcp.33.12.1179">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7451664">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1980KY25200009">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Preliminary+report+on+the+pathogenicity+of+Legionella+pneumophila+for+freshwater+and+soil+amoebae&amp;author=T+Rowbotham&amp;publication_year=1980&amp;pages=1179-1183&amp;doi=10.1136%2Fjcp.33.12.1179&amp;pmid=7451664">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">64.</div><div id="B64" class="citations"><div class="citation"><div class="citation-content">Roy C. R., Berger K. H., and Isberg R. R.. 1998. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> DotA protein is required for early phagosome trafficking decisions that occur within minutes of bacterial uptake. <em>Mol. Microbiol.</em> 28:663–674.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.1998.00841.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9632267">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000073797900022">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+DotA+protein+is+required+for+early+phagosome+trafficking+decisions+that+occur+within+minutes+of+bacterial+uptake&amp;author=C.+R.+Roy&amp;author=K.+H.+Berger&amp;author=R.+R.+Isberg&amp;publication_year=1998&amp;pages=663-674&amp;doi=10.1046%2Fj.1365-2958.1998.00841.x&amp;pmid=9632267">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">65.</div><div id="B65" class="citations"><div class="citation"><div class="citation-content">Sahr T. et al. 2009. Two small ncRNAs jointly govern virulence and transmission in <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Mol. Microbiol</em>. 72:741–762.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2009.06677.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19400772">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000265410500014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Two+small+ncRNAs+jointly+govern+virulence+and+transmission+in+Legionella+pneumophila&amp;author=T.+Sahr&amp;publication_year=2009&amp;pages=741-762&amp;doi=10.1111%2Fj.1365-2958.2009.06677.x&amp;pmid=19400772">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">66.</div><div id="B66" class="citations"><div class="citation"><div class="citation-content">Saskova L., Novakova L., Basler M., and Branny P.. 2007. Eukaryotic-type serine/threonine protein kinase StkP is a global regulator of gene expression in <span class="named-content" data-type="genus-species">Streptococcus pneumoniae</span>. <em>J. Bacteriol.</em> 189:4168–4179.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.01616-06">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17416671">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000247285900027">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Eukaryotic-type+serine%2Fthreonine+protein+kinase+StkP+is+a+global+regulator+of+gene+expression+in+Streptococcus+pneumoniae&amp;author=L.+Saskova&amp;author=L.+Novakova&amp;author=M.+Basler&amp;author=P.+Branny&amp;publication_year=2007&amp;pages=4168-4179&amp;doi=10.1128%2FJB.01616-06&amp;pmid=17416671">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">67.</div><div id="B67" class="citations"><div class="citation"><div class="citation-content">Segal G., Purcell M., and Shuman H. A.. 1998. Host cell killing and bacterial conjugation require overlapping sets of genes within a 22-kb region of the <span class="named-content" data-type="genus-species">Legionella pneumophila</span> genome. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 95:1669–1674.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.95.4.1669">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9465074">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000072115900057">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Host+cell+killing+and+bacterial+conjugation+require+overlapping+sets+of+genes+within+a+22-kb+region+of+the+Legionella+pneumophila+genome&amp;author=G.+Segal&amp;author=M.+Purcell&amp;author=H.+A.+Shuman&amp;publication_year=1998&amp;pages=1669-1674&amp;doi=10.1073%2Fpnas.95.4.1669&amp;pmid=9465074">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">68.</div><div id="B68" class="citations"><div class="citation"><div class="citation-content">Shin S. and Roy C. R.. 2008. Host cell processes that influence the intracellular survival of <span class="named-content" data-type="genus-species">Legionella pneumophila</span>. <em>Cell. Microbiol.</em> 10:1209–1220.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2008.01145.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18363881">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000255820000001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Host+cell+processes+that+influence+the+intracellular+survival+of+Legionella+pneumophila&amp;author=S.+Shin&amp;author=C.+R.+Roy&amp;publication_year=2008&amp;pages=1209-1220&amp;doi=10.1111%2Fj.1462-5822.2008.01145.x&amp;pmid=18363881">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">69.</div><div id="B69" class="citations"><div class="citation"><div class="citation-content">Smith D. and Johnson K.. 1988. Single-step purification of polypeptides expressed in <span class="named-content" data-type="genus-species">Escherichia coli</span> as fusions with glutathione <i>S</i>-transferase. <em>Gene</em> 67:31–40.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0378-1119(88)90005-4">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3047011">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1988P255900004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Single-step+purification+of+polypeptides+expressed+in+Escherichia+coli+as+fusions+with+glutathione+S-transferase&amp;author=D.+Smith&amp;author=K.+Johnson&amp;publication_year=1988&amp;pages=31-40&amp;doi=10.1016%2F0378-1119%2888%2990005-4&amp;pmid=3047011">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">70.</div><div id="B70" class="citations"><div class="citation"><div class="citation-content">Tiwari D. et al. 2009. Key residues in <span class="named-content" data-type="genus-species">Mycobacterium tuberculosis</span> protein kinase G play a role in regulating kinase activity and survival in the host. <em>J. Biol. Chem.</em> 284:27467–27479.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.M109.036095">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19638631">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Key+residues+in+Mycobacterium+tuberculosis+protein+kinase+G+play+a+role+in+regulating+kinase+activity+and+survival+in+the+host&amp;author=D.+Tiwari&amp;publication_year=2009&amp;pages=27467-27479&amp;doi=10.1074%2Fjbc.M109.036095&amp;pmid=19638631">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">71.</div><div id="B71" class="citations"><div class="citation"><div class="citation-content">Walburger A. et al. 2004. Protein kinase G from pathogenic mycobacteria promotes survival within macrophages. <em>Science</em> 304:1800–1804.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1099384">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15155913">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000222089500045">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Protein+kinase+G+from+pathogenic+mycobacteria+promotes+survival+within+macrophages&amp;author=A.+Walburger&amp;publication_year=2004&amp;pages=1800-1804&amp;doi=10.1126%2Fscience.1099384&amp;pmid=15155913">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">72.</div><div id="B72" class="citations"><div class="citation"><div class="citation-content">Weber S., Ragaz C., Reus K., Nyfeler Y., and Hilbi H.. 2006. <span class="named-content" data-type="genus-species">Legionella pneumophila</span> exploits PI(4)P to anchor secreted effector proteins to the replicative vacuole. <em>PLoS Pathog.</em> 2:e46.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.0020046">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16710455">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Legionella+pneumophila+exploits+PI%284%29P+to+anchor+secreted+effector+proteins+to+the+replicative+vacuole&amp;author=S.+Weber&amp;author=C.+Ragaz&amp;author=K.+Reus&amp;author=Y.+Nyfeler&amp;author=H.+Hilbi&amp;publication_year=2006&amp;pages=e46&amp;doi=10.1371%2Fjournal.ppat.0020046&amp;pmid=16710455">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">73.</div><div id="B73" class="citations"><div class="citation"><div class="citation-content">Xu L. et al. 2010. Inhibition of host vacuolar H+-ATPase activity by a <span class="named-content" data-type="genus-species">Legionella pneumophila</span> effector. <em>PLoS Pathog.</em> 6:e1000822.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000822">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20333253">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Inhibition+of+host+vacuolar+H%2B-ATPase+activity+by+a+Legionella+pneumophila+effector&amp;author=L.+Xu&amp;publication_year=2010&amp;pages=e1000822&amp;doi=10.1371%2Fjournal.ppat.1000822&amp;pmid=20333253">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">74.</div><div id="B74" class="citations"><div class="citation"><div class="citation-content">Zhang C., Jang J., Sakr S., and Wang L.. 2005. Protein phosphorylation on Ser, Thr, and Tyr residues in cyanobacteria. <em>J. Mol. Microbiol. Biotechnol.</em> 9:154–166.</div><div class="external-links"><div><a href="https://doi.org/10.1159/000089644">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16415589">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Protein+phosphorylation+on+Ser%2C+Thr%2C+and+Tyr+residues+in+cyanobacteria&amp;author=C.+Zhang&amp;author=J.+Jang&amp;author=S.+Sakr&amp;author=L.+Wang&amp;publication_year=2005&amp;pages=154-166&amp;doi=10.1159%2F000089644&amp;pmid=16415589">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">75.</div><div id="B75" class="citations"><div class="citation"><div class="citation-content">Zheng J., He C., Singh V., Martin N., and Jia Z.. 2007. Crystal structure of a novel prokaryotic Ser/Thr kinase and its implication in the Cpx stress response pathway. <em>Mol. Microbiol.</em> 63:1360–1371.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2007.05611.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17302814">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Crystal+structure+of+a+novel+prokaryotic+Ser%2FThr+kinase+and+its+implication+in+the+Cpx+stress+response+pathway&amp;author=J.+Zheng&amp;author=C.+He&amp;author=V.+Singh&amp;author=N.+Martin&amp;author=Z.+Jia&amp;publication_year=2007&amp;pages=1360-1371&amp;doi=10.1111%2Fj.1365-2958.2007.05611.x&amp;pmid=17302814">Google Scholar</a></div></div></div></div></div></section></div></section><div class="core-collateral"><div id="core-collateral-info" role="tabpanel"><header><h2><i class="icon-info" aria-hidden="true"></i>Information &amp; Contributors</h2></header><section id="tab-information" aria-labelledby="information" role="tabpanel"><h3>Information</h3><section class="core-self-citation"><h4>Published In</h4><div class="core-journal-presentation">



        
        <div class="cover-image"><div class="cover-image__image"><img src="/cms/asset/353df96d-410c-47e2-bebe-342ca3a4f2d8/iai.2011.79.issue-5.cover.gif" alt="Infection and Immunity cover image"/></div></div>
<div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">Infection and Immunity</span></div><div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">79</span></span> • <span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">5</span></span> • <span property="datePublished">May 2011</span></div><div class="core-pagination"><span class="heading">Pages</span>: <span class="content"><span property="pageStart">1936</span> - <span property="pageEnd">1950</span></span></div><div class="core-editors"><span class="heading">Editor</span>: <span role="list"><span role="listitem" property="editor" typeof="Person"><span property="givenName">J. B.</span> <span property="familyName">Bliska</span></span></span></div><div class="core-pmid"><span class="heading">PubMed</span>: <a class="content" href="https://pubmed.ncbi.nlm.nih.gov/21321072/" property="sameAs">21321072</a></div></div></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">© 2011 American Society for Microbiology.</div></section><section class="core-history"><h4>History</h4><div>Received: 23 July 2010</div><div>Returned for modification: 30 August 2010</div><div>Accepted: 3 February 2011</div><div>Published online: 19 April 2011</div></section><section class="core-permissions"><h4>Permissions</h4><div>Request permissions for this article.</div><div><a href="https://marketplace.copyright.com/rs-ui-web/mp/search/all/10.1128%2FIAI.00805-10">Request Permissions</a></div></section><div class="core-download"><a href="/doi/pdf/10.1128/IAI.00805-10?download=true" class="btn btn-primary"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></div></section><section id="tab-contributors" aria-labelledby="contributors" role="tabpanel"><h3>Contributors</h3><section class="core-authors"><h4>Authors</h4><div id="con1" property="author" typeof="Person"><div class="heading"><span property="givenName">Eva</span> <span property="familyName">Hervet</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Université de Lyon, Université Lyon 1, CNRS UMR 5240 “Microbiologie, Adaptation et Pathogénie,” 69622 Villeurbanne, France</span></div></div><div class="core-author-link"><a href="/authored-by/Hervet/Eva">View all articles by this author</a></div></div></div><div id="con2" property="author" typeof="Person"><div class="heading"><span property="givenName">Xavier</span> <span property="familyName">Charpentier</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology, Columbia University Medical Center, New York, New York 10032</span></div></div><div class="author-notes"><div role="paragraph">Present address: Université Joseph Fourier, CNRS UMR5163 “Adaptation et Pathogénèse des Microorganismes,” Institut Jean Roget 38042 Grenoble, France.</div></div><div class="core-author-link"><a href="/authored-by/Charpentier/Xavier">View all articles by this author</a></div></div></div><div id="con3" property="author" typeof="Person"><div class="heading"><span property="givenName">Anne</span> <span property="familyName">Vianney</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Université de Lyon, Université Lyon 1, CNRS UMR 5240 “Microbiologie, Adaptation et Pathogénie,” 69622 Villeurbanne, France</span></div></div><div class="core-author-link"><a href="/authored-by/Vianney/Anne">View all articles by this author</a></div></div></div><div id="con4" property="author" typeof="Person"><div class="heading"><span property="givenName">Jean-Claude</span> <span property="familyName">Lazzaroni</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Université de Lyon, Université Lyon 1, CNRS UMR 5240 “Microbiologie, Adaptation et Pathogénie,” 69622 Villeurbanne, France</span></div></div><div class="core-author-link"><a href="/authored-by/Lazzaroni/Jean-Claude">View all articles by this author</a></div></div></div><div id="con5" property="author" typeof="Person"><div class="heading"><span property="givenName">Christophe</span> <span property="familyName">Gilbert</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Université de Lyon, Université Lyon 1, CNRS UMR 5240 “Microbiologie, Adaptation et Pathogénie,” 69622 Villeurbanne, France</span></div></div><div class="core-author-link"><a href="/authored-by/Gilbert/Christophe">View all articles by this author</a></div></div></div><div id="con6" property="author" typeof="Person"><div class="heading"><span property="givenName">Danièle</span> <span property="familyName">Atlan</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Université de Lyon, Université Lyon 1, CNRS UMR 5240 “Microbiologie, Adaptation et Pathogénie,” 69622 Villeurbanne, France</span></div></div><div class="core-author-link"><a href="/authored-by/Atlan/Dani%C3%A8le">View all articles by this author</a></div></div></div><div id="con7" property="author" typeof="Person"><div class="heading"><span property="givenName">Patricia</span> <span property="familyName">Doublet</span> <a href="/cdn-cgi/l/email-protection#06766772746f656f6728626973646a63724673686f702b6a7f696837286074" property="email"><span class="__cf_email__" data-cfemail="deaebfaaacb7bdb7bff0bab1abbcb2bbaa9eabb0b7a8f3b2a7b1b0eff0b8ac">[email&#160;protected]</span></a></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Université de Lyon, Université Lyon 1, CNRS UMR 5240 “Microbiologie, Adaptation et Pathogénie,” 69622 Villeurbanne, France</span></div></div><div class="core-author-link"><a href="/authored-by/Doublet/Patricia">View all articles by this author</a></div></div></div></section><section class="core-editors"><h4>Editor</h4><div id="editor" property="editor" typeof="Person"><div class="heading"><span property="givenName">J. B.</span> <span property="familyName">Bliska</span></div><div class="content"><div property="jobTitle">Editor</div></div></div></section></section></div><div id="core-collateral-metrics" role="tabpanel"><header><h2><i class="icon-timeline" aria-hidden="true"></i>Metrics &amp; Citations</h2></header><section id="tab-metrics-inner" aria-labelledby="metrics-inner" role="tabpanel"><h3>Metrics</h3><section>



        
        







                <div class="sidebar-metric-container">
                        <input class="sidebar-metric__data" type="hidden" name="json" value='{"data":{"dataSections":[{"total":"158","months":[{"month":"2022-2-1","count":10},{"month":"2022-1-1","count":14},{"month":"2021-12-1","count":10},{"month":"2021-11-1","count":36},{"month":"2021-10-1","count":32},{"month":"2021-9-1","count":10},{"month":"2021-8-1","count":18},{"month":"2021-7-1","count":25},{"month":"2021-6-1","count":3}],"color":"color-4","firstMonth":[],"id":"download","title":"Download","startDate":"2021-6-1","displayDefault":true},{"total":"57","months":[{"month":"2022-2-1","count":1},{"month":"2022-1-1","count":1},{"month":"2021-12-1","count":1},{"month":"2021-11-1","count":0},{"month":"2021-10-1","count":1},{"month":"2021-9-1","count":1},{"month":"2021-8-1","count":0},{"month":"2021-7-1","count":0},{"month":"2021-6-1","count":1},{"month":"2021-5-1","count":0},{"month":"2021-4-1","count":0},{"month":"2021-3-1","count":0},{"month":"2021-2-1","count":2},{"month":"2021-1-1","count":3},{"month":"2020-12-1","count":0},{"month":"2020-11-1","count":0},{"month":"2020-10-1","count":1},{"month":"2020-9-1","count":0},{"month":"2020-8-1","count":2},{"month":"2020-7-1","count":1},{"month":"2020-6-1","count":0},{"month":"2020-5-1","count":1},{"month":"2020-4-1","count":0},{"month":"2020-3-1","count":0},{"month":"2020-2-1","count":0},{"month":"2020-1-1","count":0},{"month":"2019-12-1","count":0},{"month":"2019-11-1","count":1},{"month":"2019-10-1","count":0},{"month":"2019-9-1","count":0},{"month":"2019-8-1","count":0},{"month":"2019-7-1","count":0},{"month":"2019-6-1","count":0},{"month":"2019-5-1","count":0},{"month":"2019-4-1","count":0},{"month":"2019-3-1","count":1},{"month":"2019-2-1","count":0},{"month":"2019-1-1","count":0},{"month":"2018-12-1","count":1},{"month":"2018-11-1","count":0},{"month":"2018-10-1","count":0},{"month":"2018-9-1","count":1},{"month":"2018-8-1","count":0},{"month":"2018-7-1","count":0},{"month":"2018-6-1","count":0},{"month":"2018-5-1","count":1},{"month":"2018-4-1","count":0},{"month":"2018-3-1","count":0},{"month":"2018-2-1","count":0},{"month":"2018-1-1","count":1},{"month":"2017-12-1","count":0},{"month":"2017-11-1","count":0},{"month":"2017-10-1","count":0},{"month":"2017-9-1","count":0},{"month":"2017-8-1","count":0},{"month":"2017-7-1","count":0},{"month":"2017-6-1","count":0},{"month":"2017-5-1","count":0},{"month":"2017-4-1","count":0},{"month":"2017-3-1","count":1},{"month":"2017-2-1","count":0},{"month":"2017-1-1","count":3},{"month":"2016-12-1","count":2},{"month":"2016-11-1","count":1},{"month":"2016-10-1","count":0},{"month":"2016-9-1","count":0},{"month":"2016-8-1","count":0},{"month":"2016-7-1","count":0},{"month":"2016-6-1","count":0},{"month":"2016-5-1","count":0},{"month":"2016-4-1","count":0},{"month":"2016-3-1","count":0},{"month":"2016-2-1","count":0},{"month":"2016-1-1","count":1},{"month":"2015-12-1","count":0},{"month":"2015-11-1","count":1},{"month":"2015-10-1","count":0},{"month":"2015-9-1","count":2},{"month":"2015-8-1","count":1},{"month":"2015-7-1","count":2},{"month":"2015-6-1","count":1},{"month":"2015-5-1","count":1},{"month":"2015-4-1","count":1},{"month":"2015-3-1","count":0},{"month":"2015-2-1","count":2},{"month":"2015-1-1","count":0},{"month":"2014-12-1","count":1},{"month":"2014-11-1","count":0},{"month":"2014-10-1","count":0},{"month":"2014-9-1","count":0},{"month":"2014-8-1","count":1},{"month":"2014-7-1","count":0},{"month":"2014-6-1","count":0},{"month":"2014-5-1","count":0},{"month":"2014-4-1","count":1},{"month":"2014-3-1","count":1},{"month":"2014-2-1","count":1},{"month":"2014-1-1","count":1},{"month":"2013-12-1","count":1},{"month":"2013-11-1","count":0},{"month":"2013-10-1","count":0},{"month":"2013-9-1","count":1},{"month":"2013-8-1","count":0},{"month":"2013-7-1","count":0},{"month":"2013-6-1","count":0},{"month":"2013-5-1","count":0},{"month":"2013-4-1","count":0},{"month":"2013-3-1","count":0},{"month":"2013-2-1","count":0},{"month":"2013-1-1","count":3},{"month":"2012-12-1","count":0},{"month":"2012-11-1","count":0},{"month":"2012-10-1","count":0},{"month":"2012-9-1","count":0},{"month":"2012-8-1","count":0},{"month":"2012-7-1","count":0},{"month":"2012-6-1","count":1},{"month":"2012-5-1","count":0},{"month":"2012-4-1","count":0},{"month":"2012-3-1","count":1},{"month":"2012-2-1","count":1},{"month":"2012-1-1","count":1},{"month":"2011-12-1","count":1}],"color":"color-0","id":"citation","title":"Citation","startDate":"2011-12-1","displayDefault":true}],"doi":"10.1128/IAI.00805-10","widgetTitle":"Article Metrics"}}'>
                </div>
        
</section><section></section><section class="text-right"><a href="/doi/metrics/10.1128/IAI.00805-10" class="btn btn--inverse open-in-viewer"><span>VIEW ALL METRICS</span><i class="icon-open_in_new"></i></a></section></section><section id="tab-citations" aria-labelledby="citations" role="tabpanel"><h3>Citations</h3><section></section><section>



        
        <div class="citation-download">
	<p class="citation-msg">
		If you have the appropriate software installed, you can download article citation data to the citation manager of your choice. Simply select your manager software from the list below and click Download.
	</p>
	<!-- download options -->
	<form action="/action/downloadCitation" name="frmCitmgr" class="citation-form" method="post" target="_self"><input type="hidden" name="doi" value="10.1128/IAI.00805-10"><input type="hidden" name="downloadFileName" value="csp_79_"><input type="hidden" name="include" value="abs">
		<fieldset class="format-select">
			<label for="slct_format">Format</label>
			<div class="select-container">
				<select id="slct_format" name="format" class="js__slcInclude">
					<option value="" selected="selected">Please select one from the list</option>
					<option value="ris">RIS (ProCite, Reference Manager)</option>
					<option value="endnote">EndNote</option>
					<option value="bibtex">BibTex</option>
					<option value="medlars">Medlars</option>
					<option value="refworks">RefWorks</option>
				</select>
			</div>
			<label class="label-direct" for="direct">
				<input id="direct" type="checkbox" name="direct" value="" checked="checked">
				<span class="round-check"><span class="check"></span></span>
				Direct import
			</label>
		</fieldset>           
		<footer class="form-footer">
			<input onclick="onCitMgrSubmit()" class="formbutton" type="submit" name="submit" value="Download">
		</footer>
	</form>
</div>

</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel"><header><h2><i class="icon-eye" aria-hidden="true"></i>View Options</h2></header><div tabindex="0" class="section--wrapper"><h3>View options</h3><section><h4>ePub</h4><a href="https://journals.asm.org/doi/epub/10.1128/IAI.00805-10" class="btn"><i aria-hidden="true" class="icon-epub"></i><span>Open ePub</span></a></section><section><h4>PDF</h4><a href="/doi/pdf/10.1128/IAI.00805-10?download=true" download="true" class="btn btn--pdf"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></section><section><h4>Full Text</h4><a href="https://journals.asm.org/doi/full/10.1128/IAI.00805-10" class="btn"><i aria-hidden="true" class="icon-subject"></i><span>Open Full Text</span></a></section></div><!-- Its needed to duplicate this for PB check Pill.js as well--><div tabindex="-1" class="section--wrapper"><h3>Get Access</h3><section>



        
        












    <div class="purchaseArea pill__item" id="purchaseArea">
        <div class="login-options" data-pb-dropzone="login-options">
            
        </div>
        <div class="purchaseOptions">
            <div class="purchaseOptions--header">
                <h4 class="purchaseArea--section-heading">
                    PURCHASE OPTIONS
                </h4>
                
                        <div class="save-for-later">
                            <a data-bind="saveItem" data-doi="10.1128/IAI.00805-10" class="purchaseOptions--save" href="/action/saveItem?doi=10.1128/IAI.00805-10">
                                <i aria-hidden="true" class="icon-heart mx-1"></i>
                                <span>Save for later</span>
                            </a>
                            <a class="saved-go-cart hidden"  href="/action/showCart?FlowID=2"><span class="icon-cart-icon-material"></span> Item saved, go to cart </a>
                        </div>
                    
            </div>
            <p class="purchaseArea--section-info">
                Subscribe and get full access to this article
            </p>

            <div class="purchaseOptions--options-list">
                











<div class="purchase-options-container">
    

   
        <div class="savedItem-info hidden">
            <p></p>
        </div>
    

     
            
                
                
                        <div class="purchaseMessage hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00805-10-500044">
                            <p class="errorMsgBox"></p>
                        </div>
                        <div class="add-journal-to-cart">
                            <div class="purchaseOptions--option mb-3">
                                 
                                    <h5 class="option-title">Buy Article</h5>
                                    <span class="option-info mb-1">Infection and Immunity</span>
                                    <span class="option-info">Vol.79 &bull; Issue 5 &bull; 
                                            ASM Journals Pay Per View, PPV 25
                                        </span>
                                    <div class="option-actions mt-3 pt-3">
                                        <div class="option-price">
                                            <b>$25.00</b>
                                        </div>
                                        <a href="/action/addToCart?id=ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00805-10-500044" data-bind="addItem"
                                           data-key="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00805-10-500044" data-entity="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00805-10-500044"
                                           class="btn btn-primary btn--borderless">
                                            <i aria-hidden="true" class="mx-2 icon-shopping_cart"> </i>
                                            <span>ADD TO CART</span>
                                        </a>
                                    </div>
                                 
                            <div class="option-actions mt-3 pt-3 hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00805-10-500044" style="justify-content: flex-end">
                                <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                    <span>Checkout</span>
                                </a>
                            </div>
                        </div>

                        <div class="purchaseOptions--option mb-3">
                                                             <h5 class="option-title">Buy Article</h5>
                                <span class="option-info mb-1">Infection and Immunity</span>
                                <span class="option-info">Vol.79 &bull; Issue 5 &bull; 
                                        ASM Journals Pay Per View, PPV 25
                                    
                                </span>
                             
                             <div class="option-actions mt-3 pt-3 hidden"
                                     data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.00805-10-500044" style="justify-content: flex-end">
                                    <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                        <span>Checkout</span>
                                    </a>
                             </div>
                        </div>
                        </div>
                    
            
        

    
</div>

                <div class="purchaseOptions--option">
                    <h5 class="option-title">Journal Subscription</h5>
                    <span class="option-info mb-1">Infection and Immunity</span>
                    <div class="option-actions mt-3 pt-3">
                        <div class="option-actions--info">ASM members can purchase subscriptions to journals.</div>
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FIAI.00805-10" redirectUri="/doi/10.1128/IAI.00805-10" class="btn btn-primary btn--borderless">
                            
                            <span>Join or renew</span>
                            <i aria-hidden="true" class="mx-2 icon-open_in_new"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>




</section></div></div><div id="core-collateral-media" role="tabpanel"><header><h2><i class="icon-photo" aria-hidden="true"></i>Figures and Media</h2></header><section id="tab-figures" aria-labelledby="figures" role="tabpanel"><h3>Figures</h3></section><section id="tab-other" aria-labelledby="other" role="tabpanel"><h3>Media</h3><section>



        
        
</section></section></div><div id="core-collateral-tables" role="tabpanel"><header><h2><i class="icon-tables" aria-hidden="true"></i>Tables</h2></header></div><div id="core-collateral-share" role="tabpanel"><header><h2><i class="icon-share" aria-hidden="true"></i>Share</h2></header><h3>Share</h3><div class="section--wrapper"><section><h4>Share the article link</h4><div id="share-self"><p><a href="#" target="_blank" class="share-self__source"></a></p><button class="btn btn--inverse share-self__action"><i aria-hidden="true" class="icon-copy-o"></i><span>Copy Link</span></button><div class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section><h4>Share with email</h4><p><a href="#" target="_blank" class="btn btn-primary share-email__source"><i class="icon-copy-o"></i><span>EMAIL A COLLEAGUE</span></a></p></section><section><h4>Share on social media</h4><div class="share-buttons"><a role="link" target="_blank" href="#" class="btn btn--facebook addthis_button_facebook"><i aria-hidden="true" class="icon-facebook-2"></i><span>facebook</span></a><a role="link" target="_blank" href="#" class="btn btn--twitter addthis_button_twitter"><i aria-hidden="true" class="icon-twitter"></i><span>twitter</span></a><a role="link" target="_blank" href="#" class="btn btn--linkedin addthis_button_linkedin"><i aria-hidden="true" class="icon-linkedin"></i><span>linkedin</span></a><a role="link" target="_blank" href="#" class="btn btn--email addthis_button_email"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a></div></section></div></div></div></article><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script id="contribution-meta" type="application/json">{
    "doi": "10.1128/IAI.00805-10",
    "toc": "/toc/iai/79/5",
    "issn": "0019-9567"
}</script><div class="article__side-ad hidden">



        
        <!-- /13678947/asm_box -->
<div class="boombox-wrapper">
	<span class="boombox-label">Advertisement</span>
<div id='div-gpt-ad-1594664702378-0' style='width: 300px; height: 250px;'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664702378-0'); });
  </script>
</div>
</div>
</div>




        
        <div class="article-footer-element">
    <div class="row">
        <div class="col-12">
            <section aria-label="we recommend" class="we-recommend">
               <div id="trendmd-suggestions"></div>
<script defer src='//js.trendmd.com/trendmd.min.js' data-trendmdconfig='{"journal_id":"35290", "element":"#trendmd-suggestions"}'></script>
            </section>
        </div>
    </div>
</div>










        <div data-widget-def="ux3-layout-widget" data-widget-id="9f4096e6-614d-46c7-a704-f706dfcdc172" class="related-tabs swipe-with-arrows">
        



        
        <div class="article-footer-element">









        <div data-widget-def="UX3Tabs" data-widget-id="3c1dc793-20ba-4336-a3d4-bb86a6fc2c43" class="tab  tab--flex" data-ctrl-res="screen-sm">
        



        
        <div data-ctrl-res="screen-md" class="tab"><ul role="tablist" class="tab__nav rlist"><li role="presentation" class="tab__nav__item  active "><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" aria-selected="true" class="tab__nav__item__link active"><span>Related Articles</span></a></li><li role="presentation" class="tab__nav__item"><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" aria-selected="false" class="tab__nav__item__link"><span>Related Articles From ASM</span></a></li></ul><div class="tab__content"><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" role="tabpanel" class="tab__pane active">



        
        <section class="related-articles"><ul class="rlist related-articles__items"><li class="related-articles__item"><a href="/doi/10.1128/IAI.00587-15" class="related-articles__item__title">Correction for Hervet et al., Protein Kinase LegK2 Is a Type IV Secretion System Effector Involved in Endoplasmic Reticulum Recruitment and Intracellular Replication of Legionella pneumophila</a><ul class="meta__authors rlist--inline loa mobile-authors" title="list of authors">
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Hervet%2C+Eva" title="Eva Hervet">Eva Hervet</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Charpentier%2C+Xavier" title="Xavier Charpentier">Xavier Charpentier</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Vianney%2C+Anne" title="Anne Vianney">Anne Vianney</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Lazzaroni%2C+Jean-Claude" title="Jean-Claude Lazzaroni">Jean-Claude Lazzaroni</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Gilbert%2C+Christophe" title="Christophe Gilbert">Christophe Gilbert</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Atlan%2C+Dani%C3%A8le" title="Danièle Atlan">Danièle Atlan</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Doublet%2C+Patricia" title="Patricia Doublet">Patricia Doublet</a></span></li>
</ul>
</li></ul></section>
</div><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" role="tabpanel" class="tab__pane">



        
        <!--content loaded via ajax-->
<div class="asmj-org general-rss-feed-reader aog" id="80036a5d-0f41-4828-9cd1-19399fe2560a" data-rss-url="https://asm.org/rss/content/">
    <div class="rss-body"></div>
</div>
</div></div></div>

        </div>
    
</div>

        </div>
    




        
        
</div>
</main>




        
        <footer class="footer"><div class="footer__wrapper container container--lg"><div class="footer__top row"><div class="sitemap"><div class="sitemap__column sitemap__column--media col-xs-12"><div class="footer__logos">



        
        <a href="https://asm.org/" title="American Society for Microbiology logo"><img id="" alt="American Society for Microbiology logo" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title="ASM Journals logo"><img id="" alt="ASM Journals logo" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div class="footer__social-media">



        
        <div class="footer__social-media">
    <ul class="rlist--inline">
        <li><a href="https://www.facebook.com/asmfan" target="_blank" rel="noopener"><i class="icon-facebook"></i></a></li>
        <li><a href="https://twitter.com/asmicrobiology" target="_blank" rel="noopener"><i class="icon-twitter"></i></a></li>
        <li><a href="https://www.linkedin.com/companies/45861" target="_blank" rel="noopener"><i class="icon-linkedin"></i></a></li>
        <li><a href="https://www.youtube.com/channel/UCBxWzf49gVHz0Ksgw_N8brw" target="_blank" rel="noopener"><i class="icon-youtube"></i></a></li>
    </ul>
</div>
</div></div><div class="sitemap__column sitemap__column--journals col-xs-12">



        
        <a href="#" data-toggle-target-id="footerJournals"><h6 class="footer__list-title">JOURNALS</h6></a>
<ul id="footerJournals" class="rlist footer__list footer__list--columns">
	<li><a href="/journal/aac">Antimicrobial Agents and Chemotherapy</a></li>
	<li><a href="/journal/aem">Applied and Environmental Microbiology</a></li>
	<li><a href="/journal/cvi">Clinical and Vaccine Immunology</a></li>
	<li><a href="/journal/cmr">Clinical Microbiology Reviews</a></li>
	<li><a href="/journal/ecosalplus">EcoSal Plus</a></li>
	<li><a href="/journal/ec">Eukaryotic Cell</a></li>
	<li><a href="/journal/iai">Infection and Immunity</a></li>
	<li><a href="/journal/jb">Journal of Bacteriology</a></li>
	<li><a href="/journal/jcm">Journal of Clinical Microbiology</a></li>
	<li><a href="/journal/jmbe">Journal of Microbiology &amp; Biology Education</a></li>
	<li><a href="/journal/jvi">Journal of Virology</a></li>
	<li><a href="/journal/mbio">mBio</a></li>
	<li><a href="/journal/mmbr">Microbiology and Molecular Biology Reviews</a></li>
	<li><a href="/journal/mra">Microbiology Resource Announcements</a></li>
	<li><a href="/journal/spectrum">Microbiology Spectrum</a></li>
	<li><a href="/journal/mcb">Molecular and Cellular Biology</a></li>
	<li><a href="/journal/msphere">mSphere</a></li>
	<li><a href="/journal/msystems">mSystems</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--connect col-xs-12">



        
        <ul class="rlist footer__list">
    <li><h6>RESOURCES</h6></li>
    <li><a href="/about-asm-journals" target="_blank" rel="noopener">About Us</a></li>
    <li><a href="/announcements" target="_blank" rel="noopener">Announcements</a></li>
    <li><a href="/asm-author-center" target="_blank" rel="noopener">Author Center</a></li>
    <li><a href="/collection" target="_blank" rel="noopener">ASM Journal Collections</a></li>
     <li><a href="/publishing-ethics" target="_blank" rel="noopener">Publishing Ethics</a></li>
    <li><a href="/migration-information" target="_blank" rel="noopener">Migration Information</a></li>
    <li><a href="/advertising" target="_blank" rel="noopener">Advertising</a></li>
    <li><a href="https://asm.org/Membership/Home" target="_blank" rel="noopener">ASM Membership</a></li>
    <li><a href="https://asm.org/cookie-policy" target="_blank" rel="noopener">Privacy Policy</a></li>
    <li><a href="/terms" target="_blank" rel="noopener">Terms &amp; Conditions</a></li>
    <li><a href="/proprietary-rights" target="_blank" rel="noopener">Proprietary Rights Notices</a></li>
    <li><a href="/non-commercial-tdm-license" target="_blank" rel="noopener">Non-Commercial TDM License</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--news col-xs-12">



        
        <a href="#" data-toggle-target-id="footerConnected"><h6 class="footer__list-title">STAY CONNECTED</h6></a>
<ul id="footerConnected" class="rlist footer__list">
    <li><a href="/action/showPreferences?menuTab=Alerts" target="_blank" rel="noopener">Alerts</a></li>
    <li><a href="/asm-journals-rss-feeds" target="_blank" rel="noopener">RSS Feeds</a></li>
    <li><a href="/contact-us" target="_blank" rel="noopener">Contact Us</a></li>
    <li><a href=" https://connect.asm.org/ " target="_blank" rel="noopener">ASM Communities</a></li>
    
</ul>
</div><div class="sitemap__column sitemap__column--other col-xs-12 ">



        
        <h6>NEWS</h6>
<ul class="rlist footer__list">
    <li><a href="https://asm.org/newsroom" target="_blank" rel="noopener">Press Releases</a></li>
    <li><a href="https://asm.org/Browse-By-Content-Type/Policy-Statement" target="_blank" rel="noopener">Policy Statements</a></li>
    
</ul>
</div></div></div><div class="footer__bottom row"><div class="col-xs-12"><div class="footer__copyright">



        
        Copyright © 2022 American Society for Microbiology
</div></div></div></div></footer>




        
        <div class="cookiePolicy-popup"><div class="container"><div class="row"><div class="col-sm-7 col-md-9 cookiePolicy-popup__content"><span><span><strong>American Society for Microbiology ("ASM")</strong> is committed to maintaining your confidence and trust with respect to the information we collect from you on websites owned and operated by <strong>ASM ("ASM Web Sites")</strong> and other sources. This Privacy Policy sets forth the information we collect about you, how we use this information and the choices you have about how we use such information.</span></span></div><div class="col-sm-5 col-md-3"><div class="cookiePolicy-popup__actions"><button id="accept-cookie-policy" class="btn btn-primary btn--borderless">ACCEPT &nbsp;<span class="sr-only">the cookie policy and </span><span aria-hidden="true">& &nbsp;</span>CONTINUE</button><a href="https://asm.org/cookie-policy" class="btn btn-secondary btn--darker-content">FIND OUT MORE <span class="sr-only">about the privacy policy</span></a></div></div></div></div></div>
</div>
        </div>
    </div>



        
        

            <script src="/products/asmj/releasedAssets/js/build.lazyload.bundle-9eae693cc49bd405a4bc.js"></script>

        <script src="/products/asmj/releasedAssets/js/main.bundle-f0f6c9d7da4e7fd5b03e.js" ></script>
        

















<script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"6e7ac17bda0e7ba4","token":"edc08be6f1e64730992376d7e16c0615","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>
</body>
</html>
