
<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="8e7144fe-5496-406e-82e0-e7b9e9a0dfd1"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:doi\:10.1128/iai.2013.81.issue-4;page:string:Article/Chapter View;ctype:string:Journal Content;requestedJournal:journal:iai;website:website:asmj;article:article:doi\:10.1128/IAI.01165-12;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text;journal:journal:iai"/>
<link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"></link><meta name="citation_journal_title" content="Infection and Immunity"></meta><meta name="dc.Title" content="The Type VI Secretion System Encoded in Salmonella Pathogenicity Island 19 Is Required for Salmonella enterica Serotype Gallinarum Survival within Infected Macrophages"></meta><meta name="dc.Creator" content="Carlos J. Blondel"></meta><meta name="dc.Creator" content="Juan C. Jiménez"></meta><meta name="dc.Creator" content="Lorenzo E. Leiva"></meta><meta name="dc.Creator" content="Sergio A. Álvarez"></meta><meta name="dc.Creator" content="Bernardo I. Pinto"></meta><meta name="dc.Creator" content="Francisca Contreras"></meta><meta name="dc.Creator" content="David Pezoa"></meta><meta name="dc.Creator" content="Carlos A. Santiviago"></meta><meta name="dc.Creator" content="Inés Contreras"></meta><meta name="dc.Description" content="ABSTRACT Salmonella enterica serotype Gallinarum is the causative agent of fowl typhoid, a disease characterized by high morbidity and mortality that causes major economic losses in poultry product..."></meta><meta name="Description" content="ABSTRACT Salmonella enterica serotype Gallinarum is the causative agent of fowl typhoid, a disease characterized by high morbidity and mortality that causes major economic losses in poultry product..."></meta><meta name="dc.Publisher" content="&#xA;American Society for Microbiology&#xA;1752 N St., N.W., Washington, DC&#xA;"></meta><meta name="dc.Contributor" content="A. Camilli"></meta><meta name="dc.Date" scheme="WTN8601" content="2013-1-28"></meta><meta name="dc.Type" content="research-article"></meta><meta name="dc.Format" content="text/HTML"></meta><meta name="dc.Identifier" scheme="publisher-id" content="01165-12"></meta><meta name="dc.Identifier" scheme="doi" content="10.1128/IAI.01165-12"></meta><meta name="dc.Language" content="EN"></meta><meta name="dc.Coverage" content="1752 N St., N.W., Washington, DC"></meta><meta name="dc.Rights" content="Copyright © 2013, American Society for Microbiology. All Rights Reserved."></meta><!-- ${citation_fulltext_world_readable:10.1128/IAI.01165-12} -->




<link rel="meta" type="application/atom+xml" href="https://doi.org/10.1128%2FIAI.01165-12"></link>
<link rel="meta" type="application/rdf+json" href="https://doi.org/10.1128%2FIAI.01165-12"></link>
<link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1128%2FIAI.01165-12"></link>



<meta charset="UTF-8">
















        <meta property="og:title" content="The Type VI Secretion System Encoded in Salmonella Pathogenicity Island 19 Is Required for Salmonella enterica Serotype Gallinarum Survival within Infected Macrophages | Infection and Immunity"/>
    


        <meta property="og:type" content="Article"/>
    


        <meta property="og:url" content="https://journals.asm.org/doi/abs/10.1128/IAI.01165-12"/>
    

        <meta property="og:image" content="https://journals.asm.org/cms/asset/c4308797-6ffd-4494-aec3-3bbfe515c342/iai.2013.81.issue-4.cover.gif" />
        <meta property="og:image:width" content="113" />
        <meta property="og:image:height" content="150" />
    


        <meta property="og:site_name" content="Infection and Immunity"/>
    


        <meta property="og:description" content="Salmonella enterica serotype Gallinarum is the causative agent of fowl typhoid, a disease characterized
by high morbidity and mortality that causes major economic losses in poultry production.
We have reported that S. Gallinarum harbors a type VI ..."/>
    
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=0"/>
<title>The Type VI Secretion System Encoded in Salmonella Pathogenicity Island 19 Is Required for Salmonella enterica Serotype Gallinarum Survival within Infected Macrophages | Infection and Immunity</title>










            <link rel="stylesheet" type="text/css" href="/wro/l0of~article-metrics-phase3.css">
        
<link rel="stylesheet" type="text/css" href="/wro/l0of~product.css">



    
    
                <link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-6742698955b5929bab3a.css" media="all" id="build-style-sheet">
                <script>
                    function isIE() { //won't check for edge browser
                        var ua = navigator.userAgent;
                        /* MSIE used to detect old browsers and Trident used to newer ones*/
                        var is_ie = ua.indexOf("MSIE ") > -1 || ua.indexOf("Trident/") > -1;
          
                        return is_ie; 
                    }
                    
                    if(isIE()) {
                        document.write('<link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-style-fallback-0e2f8eb0a5d072bb5fa3.css" media="all" />')
                        var buildCss = document.getElementById('build-style-sheet');
                        buildCss.parentNode.removeChild(buildCss);
                    }
                </script>
        <link rel="stylesheet" href="/products/asmj/releasedAssets/css/print-6742698955b5929bab3a.css"media="print">
    
        <script>
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    





<meta http-equiv="X-UA-Compatible" content="IE=edge">
<style>
:root{
	--journal-image: url('/pb-assets/images/Homepage Header Images/header-iai-desktop-1602282256940.png');
	--journal-mobile-image: url("/pb-assets/images/Homepage Header Images/header-iai-mobile-1605057095637.png");
	--landing-image: url("/pb-assets/images/Homepage Header Images/header-global-desktop-1603901826210.png");
	--landing-mobile-image: url("/pb-assets/images/Homepage Header Images/header-global-mobile-1605057223667.png");


}
</style>
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/Journals-Atypon-1641917642317.css">
<script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
<script>
	window.googletag = window.googletag || {cmd: []};
	googletag.cmd.push(function() {
    	googletag.defineSlot('/13678947/asm_leader-mob', [[728, 90], [320, 50]], 'div-gpt-ad-1594664599282-0').addService(googletag.pubads());
    	googletag.defineSlot('/13678947/asm_box', [300, 250], 'div-gpt-ad-1594664702378-0').addService(googletag.pubads());	
		googletag.pubads().enableSingleRequest();
		
			
				googletag.pubads().setTargeting('asmjournals', 'iai');
			
		
		googletag.enableServices();
	});
</script>



                    <link rel="canonical" href="https://journals.asm.org/doi/10.1128/IAI.01165-12">
                





    <script src="//scholar.google.com/scholar_js/casa.js" async></script>








    
    

    
    
    
            <script type="text/javascript" src="/wro/l0of~article-metrics-phase2.js"></script>
        


    <!-- Google Tag Manager -->
    













<script>
    var dataLayer = dataLayer ||[];
    
    dataLayer.push({"event":"pageview","page_domain":"journals.asm.org","page_path":"/doi/10.1128/IAI.01165-12","page_name":"The Type VI Secretion System Encoded in Salmonella Pathogenicity Island 19 Is Required for Salmonella enterica Serotype Gallinarum Survival within Infected Macrophages | Infection and Immunity","page_type":"article","article_title":"The Type VI Secretion System Encoded in Salmonella Pathogenicity Island 19 Is Required for Salmonella enterica Serotype Gallinarum Survival within Infected Macrophages","article_type":"Research Article","article_category":"Cellular Microbiology: Pathogen-Host Cell Molecular Interactions","article_doi":"10.1128/IAI.01165-12","article_access":"free","article_received_date":"12 November 2012","article_accepted_date":"19 January 2013","article_published_online_date":"18 March 2013","article_issue":"volume 81, issue 4","article_issue_date":"April 2013","journal":"Infection and Immunity","user_type":"Guest","basic_groups":["All users group","Unknown Users","All users no webcrawlers"],"smart_groups":["Institution- IP authentication"]});
    
</script>


    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MTLLM76');</script>
    <!-- End Google Tag Manager -->

</head>
<body class="pb-ui">


    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MTLLM76" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->




















<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>





    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">




        
        <div>



        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input type="hidden" name="requestUsername" value="requestUsername" />
                <div class="input-group">
                    <div class="label">
                        <label for="5a26c991-102b-4ded-a542-61c81c149bd5.email">Email*</label>
                    </div>
                    <input id="5a26c991-102b-4ded-a542-61c81c149bd5.email" class="email" type="text" name="email" value="" size="15" />
                </div>
                
                    <div class="username-recaptcha-ajax"></div>
                
                <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" />
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        





<div class="popup registration-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Create a new account</h2>
        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.01165-12" />
            <div class="input-group">
                <div class="label">
                    <label for="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email">Email</label>
                </div>
                <input id="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email" class="email" type="email" name="email" value="" />
            </div>
            <div class="submit">
                <input class="button submit primary" type="submit" value="Register" />
            </div></form>
        <div class="center">
            <a href="/action/showLogin" class="show-login">
                
                Returning user
            </a>
        </div>
    </div>
</div>




        
        



<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="form legacy-form">
            






<h2>Change Password </h2>
<form action="/action/changePassword" method="post"><div class="message error"></div>
    <input type="hidden" name="submit" value="submit" />
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-old">Old Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-old" class="old" type="password" name="old" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
    </div>
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-new">New Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-new" class="pass-hint new" type="password" name="new" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
        
            







    


    


    


<div class="password-strength-indicator" data-min="8" data-max="20" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
        
    </div>
    <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
        </div>
        <div class="success-template hidden">
            <h2>Password Changed Successfully</h2>
            <p>Your password has been changed</p>
        </div>
    </div>
</div>





        
        















<div class="popup login-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        

        <div class="login-form-container ">
            <div class="tab tab--flex">
                <ul class="rlist tab__nav" role="tablist">
                    <li class="active" role="presentation">
                        <a class="active " id="pane-m1x01con" href="#pane-m1x01" aria-controls="pane-m1x01" role="tab"
                           data-toggle="tab" title="login" aria-selected="true" data-simple-tab-id=""
                           aria-expanded="true">
                            <span>Login</span>
                        </a>
                    </li>
                    <li role="presentation">
                        <a id="pane-m1x11con" href="#pane-m1x11" aria-controls="pane-m1x11" role="tab"
                           data-toggle="tab" title="register" aria-selected="false">
                            <span>Register</span>
                        </a>
                    </li>
                </ul>
                <ul class="rlist tab__content">
                    <li class="tab__pane active" id="pane-m1x01" aria-labelledby="pane-m1x01con" role="tabpanel">
                        




























<div class="login-form">
    
    <form action="/action/doLogin" method="post"><input type="hidden" name="id" value="016a54f6-925d-4632-a5f6-22977f0dce11" />
        





        <input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.01165-12" />
    

<input type="hidden" name="loginUri" value="/doi/10.1128/IAI.01165-12" />
        
            <input type="hidden" name="popup" value="true" />
        
        
                

























    
        
    
    
        
    
    
    
        
    

        

        
    <div class="input-group">
        <div class="label ">

            
                    <label for="login">Email*</label>
                
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="login" class="login" type="text" name="login" value="" size="15" placeholder="Enter your email address" /><div class="icon-mail"></div></div>
        
                    <div class="actions">
                        
                    </div>
                
    </div>

                





















    



    
    
    <div class="input-group">
        <div class="label ">
            <label for="password">Password*</label>
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="password" class="password" type="password" name="password" value="" autocomplete="off" placeholder="Enter your password" /><div class="icon-lock"></div></div>
        <span class="password-eye-icon icon-eye hidden"></span>
        
                    <div class="actions reset-password">
                        
                        
                        <i class="icon-lock"></i>
                        <span>
                            Forgot password?
                            Reset it <a href="/action/requestResetPassword" class="link show-request-reset-password uppercase styled-link">here</a>
                        </span>
                    </div>
                
    </div>

            

        

        <div class="remember">
            <label class="checkbox--primary remember-me" for="016a54f6-925d-4632-a5f6-22977f0dce11-remember">
                <input id="016a54f6-925d-4632-a5f6-22977f0dce11-remember" class="cmn-toggle cmn-toggle-round-flat" type="checkbox" name="remember" value="true" />
                <span class="label-txt">Keep me logged in</span>
            </label>
        </div>
        
                <div class="submit-section">
                    <input class="button submit primary" type="submit" name="submit" value="Login" />
                    <span class="required-info">Fields with * are mandatory</span>
                </div>
            
        

























        

        <div class="login-tabs-footer">
            <div class="actions">
                
                
                <span>
                    Don&apos;t have an account? Create one <a href="/action/registration" class="uppercase styled-link">here</a>
                </span>
            </div>
        </div></form>
</div>
                    </li>
                    <li class="tab__pane" id="pane-m1x11" aria-labelledby="pane-m1x11con" role="tabpanel">
                        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/IAI.01165-12" />
                            <div class="input-group">
                                <div class="label">
                                    <label for="016a54f6-925d-4632-a5f6-22977f0dce11.email">Email*</label>
                                </div>
                                <div class="iconable__input">
                                    <input id="016a54f6-925d-4632-a5f6-22977f0dce11.email" class="email" type="email" name="email" value="" placeholder="Enter your email address" />
                                    <div class="icon-mail"></div>
                                </div>
                            </div>
                            <div class="submit-section">
                                <input class="button submit primary" type="submit" value="Register" />
                                <span class="required-info">Fields with * are mandatory</span>
                            </div>
                            <div class="login-tabs-footer">
                                <div class="actions">
                                    
                                    
                                    <span>
                                        Already have an account? Login <a href="/action/showLogin" class="uppercase styled-link">here</a>
                                    </span>
                                </div>
                            </div></form>
                    </li>
                </ul>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <a href="#" class="close" tabindex="0"><i class="icon-close_thin"></i></a>
        <div class="form">
            





<p>Can't sign in? Forgot your password?</p>
<p class="sub">Enter your email address below and we will send you the reset instructions</p>
<div class="message error"></div>
<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input type="hidden" name="requestResetPassword" value="true" />
    
    <div class="input-group">
        <div class="label">
            <label for="7660a830-973e-446c-89b4-6e6ce315b3e0.email">Email*</label>
        </div>
        <input id="7660a830-973e-446c-89b4-6e6ce315b3e0.email" class="email" type="text" name="email" value="" size="15" />
    </div>
    
        








    

    
    
    
    

    
            <div class="clearfix"></div>
<div class="label required">
    <label for="captchaCode" class="prompt" style="white-space: nowrap">Retype the code from the picture:</label>
</div>
<link type="text/css" rel="stylesheet" href="/botdetectcaptcha?get=layoutStyleSheet" />

<div class="LBD_CaptchaDiv" id="sampleCaptcha" style="width: 280px !important; height: 50px !important"><!--
   --><div class="LBD_CaptchaImageDiv" id="sampleCaptcha_CaptchaImageDiv" style="width: 250px !important; height: 50px !important"><!--
   --><a target="_blank" href="http://captcha.com/java-captcha-info.html" onclick="sampleCaptcha.OnHelpLinkClick(); return sampleCaptcha.FollowHelpLink;"><img class="LBD_CaptchaImage" id="sampleCaptcha_CaptchaImage" src="/botdetectcaptcha?get=image&amp;c=sampleCaptcha&amp;t=9ecb8f7b90584df9987a981e6b5480e9" alt="CAPTCHA" /></a></div><!--
   --><div class="LBD_CaptchaIconsDiv" id="sampleCaptcha_CaptchaIconsDiv" style="width: 24px !important;"><!--
   --><a class="LBD_ReloadLink" id="sampleCaptcha_ReloadLink" href="#" onclick="sampleCaptcha.ReloadImage(); this.blur(); return false;" title="Change the CAPTCHA code"> <img class="LBD_ReloadIcon" id="sampleCaptcha_ReloadIcon" src="/botdetectcaptcha?get=ReloadIcon" alt="Change the CAPTCHA code" /></a><!--
   --><a id="sampleCaptcha_SoundLink" class="LBD_SoundLink" href="/botdetectcaptcha?get=sound&amp;c=sampleCaptcha&amp;t=9ecb8f7b90584df9987a981e6b5480e9" onclick="sampleCaptcha.PlaySound(); this.blur(); return false;" title="Speak the CAPTCHA code"> <img class="LBD_SoundIcon" id="sampleCaptcha_SoundIcon" src="/botdetectcaptcha?get=SoundIcon" alt="Speak the CAPTCHA code" /></a><!--
   --><div class="LBD_Placeholder" id="sampleCaptcha_AudioPlaceholder">&nbsp;</div><!--
   --></div>

<script type="text/javascript" src="/botdetectcaptcha?get=clientScriptInclude"></script>
<script type="text/javascript">
    //<![CDATA[
     BotDetect.Init('sampleCaptcha', '9ecb8f7b90584df9987a981e6b5480e9', 'captchaCode', true, true, true, true, 1200, 7200, 0);
    //]]>
    </script>


<script type="text/javascript">
    //<![CDATA[
      try{(function(){var bdrsn = document.createElement('script'); bdrsn.type = 'text/javascript'; bdrsn.async = true; bdrsn.src = document.location.protocol + '//remote.captcha.com/include.js?i=ATABMwExATMBMAJBMhMRP3E_PygHKT8_Pz9AbXw_Yj8i'; var fsn = document.getElementsByTagName('script')[0]; fsn.parentNode.insertBefore(bdrsn, fsn);})();} catch(err){}
    //]]>
    </script>

   <input type="hidden" id="LBD_VCID_sampleCaptcha" name="LBD_VCID_sampleCaptcha" value="9ecb8f7b90584df9987a981e6b5480e9" />
</div>

<input type="hidden" value="0" id="LBD_BackWorkaround_sampleCaptcha">
<div class="validationDiv">
    <input type="hidden" name="loginBotCaptcha" value="">
    <input id="captchaCode" type="text" name="captchaCode"/>
</div>
        

    

    

    
    <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" /></form>
            
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <p>If the address matches an existing account you will receive an email with instructions to reset your password.</p>
            <div class="center">
                <a href="#" class="cancel">Close</a>
            </div>
        </div>
    </div>
</div>




        
        




<div class="top-drawer verify-phone-drawer hidden">
    <div class="content">
        <div class="form">
            <h2>Verify Phone</h2>
            <form action="/action/verifyPhoneNumber" method="post"><input type="hidden" name="submit" value="submit" />
                <div class="message error"></div>
                <div class="input-group">
                    <div class="label">
                        <label for="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code">Enter the verification code</label>
                    </div>
                    <input id="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code" class="verificationCode" type="text" name="verificationCode" value="" size="15" />
                </div>
                <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <h2>Congrats!</h2>
            <p>Your Phone has been verified</p>
            <div class="center">
                <a href="/action/showPreferences" class="button primary submit">close</a>
            </div>
        </div>
    </div>
</div>





        
        <div id="search-overlay" class="header__search-overlay search-overlay js--hidden"><div class="search-overlay__close"><a href="#" title="Close search panel" data-hide-target-id="search-overlay" class="search-overlay__close__btn"><span>CLOSE </span><i class="icon-close"></i></a></div><div class="container">



        
        <div class="quick-search quick-search--all"><div class="full-width quickSearchFormContainer"><div class="tab"><div class="quick-search__searchbox"><ul role="tablist" class="rlist tab__nav dropblock--tab"><li role="presentation" class="tab__nav__item active"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-controls="pane-0" role="tab" data-toggle="tab" title="This Journal" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" aria-expanded="false" class="tab__nav__item__link">This Journal</a></li><li role="presentation" class="tab__nav__item"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-controls="pane-1" role="tab" data-toggle="tab" title="Anywhere" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" aria-expanded="false" class="tab__nav__item__link">Anywhere</a></li></ul><div class="quick-search__search-items tab__content"><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-0 option-journal"><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Search here within this Journal" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/><input type="hidden" name="SeriesKey" value="iai"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-1 "><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Enter words / phrases / DOI / ISBN / keywords / authors / etc" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div>



        
        <div class="quick-search__links">
    <a href="/search/advanced" class="quick-search__advanced-link">Advanced search</a>
</div>
</div></div></div></div></div>
<div class="search-overlay__autocomplete"><div id="autocompleted-terms"></div></div><div class="search-overlay__suggestions"><div class="search-overlay__suggestions__title">Suggested Terms:</div><ul class="rlist search-overlay__suggestions__list">



        
        <li><a href="/asm-author-center" class="btn btn-secondary">author information</a></li>
<li><a href="/migration-information" class="btn btn-secondary">migration information</a></li>
<li><a href="/publication-fees" class="btn btn-secondary">publication fees</a></li>
<li><a href="/asm-covid-19" class="btn btn-secondary">COVID-19 collection</a></li>
<li><a href="/collection" class="btn btn-secondary">asm journals collections</a></li>
</ul></div></div></div><header class="header header--journal header--journal-page"><div class="header__row-1">



        
        <div class="journals-menu">
	<a data-toggle-target-id="journals-menu-content" href="#" title="Journals Menu" class="journals-menu__trigger js--target-hidden">
		<i class="icon-dehaze"></i>
		<i class="icon-close"></i>
	</a>
	<div id="journals-menu-content" class="journals-menu__content js--hidden" aria-expanded="true">
		<a href="#" data-hide-target-id="journals-menu-content" class="journals-menu__back">
			<i class="icon-arrow_left"></i>
			<span>BACK</span>
		</a>
		<ul class="rlist journals-menu__list">
			<li><a href="/journal/aac"><img src="/pb-assets/images/thumbnails/thumb-aac-1602006917653.jpg" alt="Antimicrobial Agents and Chemotherapy"><span>Antimicrobial Agents and Chemotherapy</span></a></li>
			<li><a href="/journal/aem"><img src="/pb-assets/images/thumbnails/thumb-aem-1602006917677.jpg" alt="Applied and Environmental Microbiology"><span>Applied and Environmental Microbiology</span></a></li>
			<li><a href="/journal/cvi"><img src="/pb-assets/images/thumbnails/thumb-cvi-1602006922190.jpg" alt="Clinical and Vaccine Immunology"><span>Clinical and Vaccine Immunology</span></a></li>
			<li><a href="/journal/cmr"><img src="/pb-assets/images/thumbnails/thumb-cmr-1602006918000.jpg" alt="Clinical Microbiology Reviews"><span>Clinical Microbiology Reviews</span></a></li>
			<li><a href="/journal/ecosalplus"><img src="/pb-assets/images/thumbnails/thumb-ecosalplus-1602678711627.jpg" alt="EcoSal Plus"><span>EcoSal Plus</span></a></li>
			<li><a href="/journal/ec"><img src="/pb-assets/images/thumbnails/thumb-ec-1602006922537.jpg" alt="Eukaryotic Cell"><span>Eukaryotic Cell</span></a></li>
			<li><a href="/journal/iai"><img src="/pb-assets/images/thumbnails/thumb-iai-1602006922193.jpg" alt="Infection and Immunity"><span>Infection and Immunity</span></a></li>
			<li><a href="/journal/jb"><img src="/pb-assets/images/thumbnails/thumb-jb-1602006930407.jpg" alt="Journal of Bacteriology"><span>Journal of Bacteriology</span></a></li>
			<li><a href="/journal/jcm"><img src="/pb-assets/images/thumbnails/thumb-jcm-1602006930870.jpg" alt="Journal of Clinical Microbiology"><span>Journal of Clinical Microbiology</span></a></li>
			<li><a href="/journal/jmbe"><img src="/pb-assets/images/thumbnails/thumb-jmbe-1606273579503.jpg" alt="Journal of Microbiology &amp; Biology Education"><span>Journal of Microbiology &amp; Biology Education</span></a></li>
			<li><a href="/journal/jvi"><img src="/pb-assets/images/thumbnails/thumb-jvi-1602006937397.jpg" alt="Journal of Virology"><span>Journal of Virology</span></a></li>
			<li><a href="/journal/mbio"><img src="/pb-assets/images/thumbnails/thumb-mbio-1602006937397.jpg" alt="mBio"><span>mBio</span></a></li>
			<li><a href="/journal/mmbr"><img src="/pb-assets/images/thumbnails/thumb-mmbr-1602006945363.jpg" alt="Microbiology and Molecular Biology Reviews"><span>Microbiology and Molecular Biology Reviews</span></a></li>
			<li><a href="/journal/mra"><img src="/pb-assets/images/thumbnails/thumb-mra-1602006946427.jpg" alt="Microbiology Resource Announcements"><span>Microbiology Resource Announcements</span></a></li>
			<li><a href="/journal/spectrum"><img src="/pb-assets/images/thumbnails/thumb-spectrum-1602006955410.jpg" alt="Microbiology Spectrum"><span>Microbiology Spectrum</span></a></li>
			<li><a href="/journal/mcb"><img src="/pb-assets/images/thumbnails/thumb-mcb-1602006937413.jpg" alt="Molecular and Cellular Biology"><span>Molecular and Cellular Biology</span></a></li>
			<li><a href="/journal/msphere"><img src="/pb-assets/images/thumbnails/thumb-msphere-1602006946427.jpg" alt="mSphere"><span>mSphere</span></a></li>
			<li><a href="/journal/msystems"><img src="/pb-assets/images/thumbnails/thumb-msystems-1602006955407.jpg" alt="mSystems"><span>mSystems</span></a></li>
		</ul>
	</div>
</div>
<div class="header__logos">



        
        <a href="https://asm.org/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div data-truncate="1" class="header__feed mx-3 js--truncate"></div><div class="header__attribution"></div><div class="header__cart">



        
        















<div class="eCommerceCartIndicatorWidget">
    
            <a href="/action/showCart?FlowID=1&amp;aria-label=Shopping+Cart" id="cartLabel" class="quick-menu__item" aria-label="Shopping Cart">
                <div class="cartLabel" title="Cart">
                    <i class="icon-shopping_cart" aria-hidden="true"></i>
                    <span class="filter-count filter-count-bubble shopping-cart hidden"
                          data-id="cart-size">0</span>
                        
                </div>
            </a>
        
</div>
</div><div class="header__login">



        
        




    








        




    













<div class="user-login-bar">
    <div title="Sign in" class="loginBar user-login-bar">
        <div class="user-login-bar">
            <div class="loginBar">
                <div class="loginBar__login">
                    <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FIAI.01165-12&amp;data-toggle-target-id=loginOptions" class="btn btn-primary btn--light btn--darker-content btn--borderless" redirectUri="/doi/10.1128/IAI.01165-12" data-toggle-target-id="loginOptions">
                        Login
                    </a>
                    <div id="loginOptions" class="loginBar__login__options">
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FIAI.01165-12">
                            
                            <img src="/specs/products/asmj/releasedAssets/images/logo-textless.svg" class="mr-2">
                            <span>ASM Member Login</span>
                            <i class="icon-open_in_new" aria-hidden="true"></i>
                        </a>
                        <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FIAI.01165-12">
                            <i class="ml-1 mr-2 icon-account_circle"></i>
                            Non-Member Login
                        </a>
                        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1128%2FIAI.01165-12">
                            <i class="ml-1 mr-2 icon-institution"></i>
                            Institutional Login
                        </a>
                        <a href="/action/registration?redirectUri=%2Fdoi%2F10.1128%2FIAI.01165-12" class="btn btn-secondary btn--inverse btn--borderless btn--darker-content btn-register" title="Register">
                            Register
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>



    

</div>



        
        <a href="#" data-show-target-id="search-overlay" class="hidden-md hidden-sm hidden-xs search-trigger js--target-shown"><i title="Open Search Panel" class="icon-search"></i></a>
</div><div class="header__row-2"><a data-toggle-target-id="responsive-menu" href="#" title="Journals Menu" class="header__mobile-trigger"><i class="icon-dehaze"></i><i class="icon-close"></i></a><div class="header__logos">



        
        <a href="/journal/iai" title="Infection and Immunity logo"><img id="" alt="Infection and Immunity logo" src="/pb-assets/images/pub-nav/IAI-logo-1601672963217.png"/></a>
</div><div class="hidden-sm hidden-xs hidden-md">



        
        <a href="#afterNav-0fo" class="sr-only-focusable">Skip main navigation</a><nav class="main-nav"><ul id="menubar" role="menubar" class="menubar rlist--inline"><li role="menuitem" aria-label="JOURNAL HOME" id="menu-item-pub_nav-0" class="menu-item is--current"><a href="/journal/iai"><span>JOURNAL HOME</span></a></li><li role="menuitem" aria-haspopup="true" aria-label="ARTICLES" id="menu-item-pub_nav-1" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-1" data-toggle="dropdown" class="dropdown__toggle"><span>ARTICLES</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-1" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="CURRENT ISSUE" class="menu-item is--current"><a href="/toc/iai/current"><span>CURRENT ISSUE</span></a></li><li role="menuitem" tabindex="-1" aria-label="ACCEPTED MANUSCRIPTS" class="menu-item is--current"><a href="/toc/iai/0/ja"><span>ACCEPTED MANUSCRIPTS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="COLLECTIONS" class="menu-item is--current"><a href="/journal/iai/collections"><span>COLLECTIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARCHIVE" class="menu-item is--current"><a href="/loi/iai"><span>ARCHIVE</span></a></li></ul></li><li role="menuitem" id="menu-item-pub_nav-2" class="menu-item">



        
        <li role="menuitem" aria-haspopup="true" aria-label="FOR AUTHORS" id="menu-item-pub_nav-custom" class="dropdown menu-parent is--current">
	<a href="#" id="pub_nav-custom" data-toggle="dropdown" class="dropdown__toggle"><span>FOR AUTHORS</span></a>
	<ul aria-labelledby="pub_nav-custom" role="menu" class="rlist dropdown__menu">
		<li role="menuitem" tabindex="-1" aria-label="SUBMITTING" class="menu-item is--current">
			<a href="/submitting?journal=iai"><span>SUBMITTING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="WRITING YOUR PAPER" class="menu-item is--current">
			<a href="/writing-your-paper?journal=iai"><span>WRITING YOUR PAPER</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="TRANSFERRING" class="menu-item is--current">
			<a href="/transferring?journal=iai"><span>TRANSFERRING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="ACCEPTED PAPERS" class="menu-item is--current">
			<a href="/accepted-papers?journal=iai"><span>ACCEPTED PAPERS</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="EDITORIAL POLICIES" class="menu-item is--current">
			<a href="/editorial-policies?journal=iai"><span>EDITORIAL POLICIES</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="PUBLISHING ETHICS" class="menu-item is--current">
			<a href="/publishing-ethics?journal=iai"><span>PUBLISHING ETHICS</span></a>
		</li>
	</ul>
</li>

</li><li role="menuitem" aria-haspopup="true" aria-label="ABOUT THE JOURNAL" id="menu-item-pub_nav-3" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-2" data-toggle="dropdown" class="dropdown__toggle"><span>ABOUT THE JOURNAL</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-2" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="ABOUT IAI" class="menu-item is--current"><a href="/journal/iai/about"><span>ABOUT IAI</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMIT" class="menu-item is--current"><a href="/journal/iai/submit"><span>SUBMIT</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="SCOPE" class="menu-item is--current"><a href="/journal/iai/scope"><span>SCOPE</span></a></li><li role="menuitem" tabindex="-1" aria-label="EDITORIAL BOARD" class="menu-item is--current"><a href="/journal/iai/board-editors"><span>EDITORIAL BOARD</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARTICLE TYPES" class="menu-item is--current"><a href="/journal/iai/article-types"><span>ARTICLE TYPES</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMISSION AND REVIEW PROCESS" class="menu-item is--current"><a href="/journal/iai/submission-review-process"><span>SUBMISSION AND REVIEW PROCESS</span></a></li><li role="menuitem" tabindex="-1" aria-label="EBM SUBMISSIONS" class="menu-item is--current"><a href="/journal/iai/ebm-submission"><span>EBM SUBMISSIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FOR REVIEWERS" class="menu-item is--current"><a href="/reviewers"><span>FOR REVIEWERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FAQ" class="menu-item is--current"><a href="/journal/iai/faq"><span>FAQ</span></a></li><li role="menuitem" tabindex="-1" aria-label="CONTACT US" class="menu-item is--current"><a href="/journal/iai/contact-us"><span>CONTACT US</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="SUBSCRIBE" id="menu-item-pub_nav-4" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-3" data-toggle="dropdown" class="dropdown__toggle"><span>SUBSCRIBE</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-3" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="MEMBERS" class="menu-item is--current"><a href="/members"><span>MEMBERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="INSTITUTIONS" class="menu-item is--current"><a href="/institutions"><span>INSTITUTIONS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="ALERTS" class="menu-item is--current"><a href="/action/showPreferences?menuTab=Alerts"><span>ALERTS</span></a></li><li role="menuitem" tabindex="-1" aria-label="RSS" class="menu-item is--current"><a href="/asm-journals-rss-feeds"><span>RSS</span></a></li></ul></li></ul><span id="afterNav-0fo" tabindex="-1"></span></nav>
</div><a href="#" data-show-target-id="search-overlay" class="hidden-lg search-trigger"><i title="Open Search Panel" class="icon-search"></i></a></div></header>




        
        <main class="content article-page">



        
        <div>









        <div data-widget-def="ux3-layout-widget" data-widget-id="62435c6d-56e5-45d8-8726-1c5a8298a1a6" class="article-container">
        



        
        <div class="mb-3">



        
        <ol aria-label="Breadcrumb" role="navigation" vocab="https://schema.org/" typeof="BreadcrumbList" class="breadcrumbs rlist--inline"><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="https://journals.asm.org/"><span property="name">ASM Journals</span></a><meta property="position" content="1"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/journal/iai"><span property="name">Infection and Immunity</span></a><meta property="position" content="2"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/toc/iai/81/4"><span property="name">Vol. 81, No. 4</span></a><meta property="position" content="3"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/doi/10.1128/IAI.01165-12" aria-current="page" class="breadcrumbs__current"><span property="name">The Type VI Secretion System Encoded in <named-content content-type='genus-species' xlink:type='simple'>Salmonella</named-content> Pathogenicity Island 19 Is Required for <named-content content-type='genus-species' xlink:type='simple'>Salmonella enterica</named-content> Serotype Gallinarum Survival within Infected Macrophages</span></a><meta property="position" content="4"/></li></ol>










        <div data-widget-def="general-html-asset" data-widget-id="b28fb475-91bf-4257-b86f-8120484c7481" class="container--center-content">
        



        
        <!-- /13678947/asm_leader-mob -->
<div class="leaderboard-wrapper">
<span class="leaderboard-label">Advertisement</span>
<div id='div-gpt-ad-1594664599282-0'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664599282-0'); });
  </script>
</div>
</div>

        </div>
    
</div>

        </div>
    




        
        <link rel="stylesheet" type="text/css" href="/specs/products/asmj/releasedAssets/css/build-article.css"/><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle"><section id="frontmatter" data-extent="frontmatter"><header><div class="core-container"><div data-article-access="free" data-article-access-type="limited" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__type">Research Article</div><div class="meta-panel__publication-data">18 March 2013</div></div><div class="meta-panel__right-content"><div class="meta-panel__share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share__block share__inline-links"><span class="sr-only">Share on</span><ul class="rlist--inline addthis addthis_toolbox addthis_default_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper  icon-facebook"></i></a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i></a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper  icon-linkedin"></i></a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper  icon-mail"></i></a></li></ul></div>
</div></div></div><h1 property="name">The Type VI Secretion System Encoded in <span class="named-content" data-type="genus-species">Salmonella</span> Pathogenicity Island 19 Is Required for <span class="named-content" data-type="genus-species">Salmonella enterica</span> Serotype Gallinarum Survival within Infected Macrophages</h1><div class="core-relations my-3"><div class="relation mb-3"><div class="relation--head"><i class="icon-info"></i><div>This article has been corrected.</div><a href="/doi/10.1128/IAI.00541-15">VIEW CORRECTION</a></div></div></div><div class="contributors"><span class="authors"><span class="heading">Authors</span>: <span role="list"><span property="author" typeof="Person" role="listitem"><a href="#con1"><span property="givenName">Carlos J.</span> <span property="familyName">Blondel</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con2"><span property="givenName">Juan C.</span> <span property="familyName">Jiménez</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con3"><span property="givenName">Lorenzo E.</span> <span property="familyName">Leiva</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con4"><span property="givenName">Sergio A.</span> <span property="familyName">Álvarez</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con5"><span property="givenName">Bernardo I.</span> <span property="familyName">Pinto</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con6"><span property="givenName">Francisca</span> <span property="familyName">Contreras</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con7"><span property="givenName">David</span> <span property="familyName">Pezoa</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con8"><span property="givenName">Carlos A.</span> <span property="familyName">Santiviago</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con9"><span property="givenName">Inés</span> <span property="familyName">Contreras</span></a></span></span></span></div><div class="self-citation"><span class="heading">DOI</span>: <a href="https://doi.org/10.1128/IAI.01165-12" property="sameAs">https://doi.org/10.1128/IAI.01165-12</a><span>&nbsp;&#8226;&nbsp;</span><span class="crossmark"><a data-target="crossmark" href="#" data-doi="10.1128/IAI.01165-12" class="crossmark__link"><img alt="Check for updates on crossmark" src="/specs/products/asmj/releasedAssets/images/cross-mark.png" class="crossmark__logo"/></a></span></div><div class="info-panel"><div class="info-panel__metrics info-panel__item">



        
        







        <div class="toolbar-metric-container">
            <input class="toolbar-metric__data" type="hidden" name="json" value='{"data":{"menuSections":[{"total":"196","id":"downloads","title":"downloads"},{"total":"38","id":"citations","title":"citations"}],"doi":"10.1128/IAI.01165-12"},"thisWidget":{"properties":{"identifier":"","headingLvl":"h2","wrapperClass":"","citationMetricId":"2","downloadMetricId":"1","headingClass":"","ux3/template":"","wrapperTag":"div","title":"","linkToFullPage":""}}}'>
        </div>
    

</div><div class="info-panel__favorite info-panel__item">



        
        <div class="article-tools"><a href="/action/addCitationAlert?doi=10.1128%2FIAI.01165-12" target="_blank" title="Track Citations" class="article-tools__citation btn btn--slim btn-secondary btn--inverse btn--transparent"><i class="icon-alert-bell"></i></a></div>
</div><div class="info-panel__alerts info-panel__item"></div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" title="View Citations" class="btn btn--slim btn--inverse"><i aria-hidden="true" class="icon-format_quote"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://journals.asm.org/doi/full/10.1128/IAI.01165-12" class="btn btn--slim btn--inverse"><span>Full Text</span></a><a href="/doi/reader/10.1128/IAI.01165-12" class="btn btn--slim btn--pdf"><span>PDF/EPUB</span></a></div></div></div></header><nav data-core-nav="article"><header><div class="core-self-citation">



        
        <a href="/journal/iai" class="sections-navbar__title">IAI</a>
<div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">81</span></span><span class="delimiter">, </span><span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">4</span></span><br /><span property="datePublished">April 2013</span></div></div><nav></nav></header><ul xmlns="http://www.w3.org/1999/xhtml"><li><a href="#abstract">ABSTRACT</a></li><li><a href="#sec-1">INTRODUCTION</a></li><li><a href="#sec-2">MATERIALS AND METHODS</a></li><li><a href="#sec-3">RESULTS</a></li><li><a href="#sec-4">DISCUSSION</a></li><li><a href="#acknowledgments">ACKNOWLEDGMENTS</a></li><li><a href="#supplementary-materials">Supplemental Material</a></li><li><a href="#bibliography">REFERENCES</a></li></ul><footer></footer></nav><div id="abstracts"><div class="core-container"><section id="abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">ABSTRACT</h2><div role="paragraph"><span class="named-content" data-type="genus-species">Salmonella enterica</span> serotype Gallinarum is the causative agent of fowl typhoid, a disease characterized by high morbidity and mortality that causes major economic losses in poultry production. We have reported that <i>S</i>. Gallinarum harbors a type VI secretion system (T6SS) encoded in <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 19 (SPI-19) that is required for efficient colonization of chicks. In the present study, we aimed to characterize the SPI-19 T6SS functionality and to investigate the mechanisms behind the phenotypes previously observed <i>in vivo</i>. Expression analyses revealed that SPI-19 T6SS core components are expressed and produced under <i>in vitro</i> bacterial growth conditions. However, secretion of the structural/secreted components Hcp1, Hcp2, and VgrG to the culture medium could not be determined, suggesting that additional signals are required for T6SS-dependent secretion of these proteins. <i>In vitro</i> bacterial competition assays failed to demonstrate a role for SPI-19 T6SS in interbacterial killing. In contrast, cell culture experiments with murine and avian macrophages (RAW264.7 and HD11, respectively) revealed production of a green fluorescent protein-tagged version of VgrG soon after <span class="named-content" data-type="genus-species">Salmonella</span> uptake. Furthermore, infection of RAW264.7 and HD11 macrophages with deletion mutants of SPI-19 or strains with genes encoding specific T6SS core components (<i>clpV</i> and <i>vgrG</i>) revealed that SPI-19 T6SS contributes to <i>S</i>. Gallinarum survival within macrophages at 20 h postuptake. SPI-19 T6SS function was not linked to <span class="named-content" data-type="genus-species">Salmonella</span>-induced cytotoxicity or cell death of infected macrophages, as has been described for other T6SS. Our data indicate that SPI-19 T6SS corresponds to a novel tool used by <span class="named-content" data-type="genus-species">Salmonella</span> to survive within host cells.</div></section></div></div></section><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1" data-type="intro"><h2>INTRODUCTION</h2><div role="paragraph">The <span class="named-content" data-type="genus-species">Salmonella</span> genus includes over 2,500 known serotypes distributed between two species, <span class="named-content" data-type="genus-species">S. enterica</span> and <span class="named-content" data-type="genus-species">S. bongori</span> (<a href="#B1" role="doc-biblioref" data-xml-rid="B1">1</a>). <span class="named-content" data-type="genus-species">Salmonella</span> infections can range from subclinical colonization to serious systemic disease, depending both on the serotype and the infected host. While some serotypes can infect and cause systemic disease in a wide range of animal hosts, others are restricted to a specific host. <span class="named-content" data-type="genus-species">S. enterica</span> serotype Gallinarum has a host range restricted to birds and causes a severe systemic disease called fowl typhoid. This disease is characterized by high morbidity and mortality, causing major economic losses in poultry production in several parts of the world (<a href="#B2" role="doc-biblioref" data-xml-rid="B2">2</a>, <a href="#B3" role="doc-biblioref" data-xml-rid="B3">3</a>). <i>S</i>. Gallinarum has been eradicated from most developed countries, such as the United States, Australia, and many Western European countries, as a result of improved surveillance and slaughter practices; however, it is still a major concern in developing countries (<a href="#B2" role="doc-biblioref" data-xml-rid="B2">2</a>).</div><div role="paragraph">The ability of <span class="named-content" data-type="genus-species">Salmonella</span> to invade and survive within host cells depends on the delivery of specialized effector proteins into the host cell cytosol by two different type III secretion systems (T3SS), each encoded in distinct genomic islands known as <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 1 (SPI-1) and SPI-2. The SPI-1 T3SS confers the ability to invade nonphagocytic cells and to colonize and breach the gastrointestinal tract of their hosts. On the other hand, SPI-2 T3SS allows <span class="named-content" data-type="genus-species">Salmonella</span> to survive within eukaryotic cells and spread to internal organs, such as the liver and spleen (<a href="#B4" role="doc-biblioref" data-xml-rid="B4">4</a>). Both T3SS are present in every <span class="named-content" data-type="genus-species">S. enterica</span> serotype and correspond to the most critical virulence determinants in <span class="named-content" data-type="genus-species">Salmonella</span>. Nevertheless, <span class="named-content" data-type="genus-species">S. enterica</span> serotypes differ greatly in terms of pathogenicity and host specificity, suggesting that other virulence factors are involved in the adaptation of different serotypes to specific hosts and/or environments.</div><div role="paragraph">The type VI secretion system (T6SS) corresponds to a newly described mechanism for protein translocation that is widespread among Gram-negative bacteria (<a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>) and has been linked to a wide variety of cellular processes, including virulence in several bacterial pathogens (<a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>). We have reported that the <span class="named-content" data-type="genus-species">Salmonella</span> genus comprises five phylogenetically distinct T6SS, each encoded in differentially distributed SPIs, including SPI-6, SPI-19, SPI-20, SPI-21, and SPI-22 (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>, <a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>). <i>S</i>. Gallinarum harbors one T6SS encoded in SPI-19 (SPI-19 T6SS), which is required to efficiently colonize the avian host (<a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>, <a href="#B10" role="doc-biblioref" data-xml-rid="B10">10</a>). The requirement of a fully functional SPI-19 T6SS for systemic infection suggested a role for this system in the interaction of <span class="named-content" data-type="genus-species">Salmonella</span> with cells of the reticuloendothelial system (<a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>).</div><div role="paragraph">Interaction of <i>S</i>. Gallinarum with the avian immune system has been reported to be critical for the ability of this serotype to colonize and cause disease in chicks (<a href="#B11" role="doc-biblioref" data-xml-rid="B11">11</a>). This interaction occurs in each of the different stages of the progression of avian systemic salmonellosis, from intestinal invasion and systemic infection to the immune clearance/persistence of the pathogen (reviewed in reference <a href="#B12" role="doc-biblioref" data-xml-rid="B12">12</a>). Nevertheless, despite the critical aspects of this <span class="named-content" data-type="genus-species">Salmonella</span>-immune cell relationship, there is little information regarding the bacterial virulence determinants important for the interaction of <i>S</i>. Gallinarum with macrophages, especially those of avian origin. In this context, even though several SPIs have been shown to contribute to infection <i>in vivo</i> (<a href="#B3" role="doc-biblioref" data-xml-rid="B3">3</a>, <a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>, <a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a>), only SPI-1 and SPI-2 T3SS have been evaluated <i>in vitro</i> (<a href="#B14" role="doc-biblioref" data-xml-rid="B14">14</a>, <a href="#B15" role="doc-biblioref" data-xml-rid="B15">15</a>).</div><div role="paragraph">In the present work, we aimed to further characterize SPI-19 T6SS and the molecular mechanisms behind the phenotypes previously observed <i>in vivo</i>. To accomplish this, we analyzed the expression, production, and secretion profiles of selected T6SS components under different <i>in vitro</i> growth conditions and analyzed the role played by SPI-19 T6SS in the interaction of <i>S</i>. Gallinarum with murine and avian macrophages. We determined that SPI-19 T6SS components are expressed and produced, but not secreted, under different growth conditions. On the other hand, we determined intracellular production of the cell-puncturing protein VgrG once the bacteria are internalized by macrophages and that SPI-19 T6SS contributes to <i>S</i>. Gallinarum survival within murine and avian macrophages.</div></section><section id="sec-2" data-type="materials methods"><h2>MATERIALS AND METHODS</h2><section id="sec-2-1"><h3>Bacterial strains and growth conditions.</h3><div role="paragraph">The bacterial strains used in the present study are listed in <a href="#T1">Table 1</a>. Bacteria were routinely grown in Luria-Bertani (LB) medium (10 g/liter tryptone, 5 g/liter yeast extract, 5 g/liter NaCl) at 37°C with aeration. When required, LB medium was supplemented with ampicillin (Amp; 100 mg/liter), chloramphenicol (Cam; 20 mg/liter), and kanamycin (Kan; 50 mg/liter). Media were solidified by the addition of agar (15 g/liter). For SPI-1-inducing conditions, bacteria were grown in LB medium supplemented to reach final concentrations of 0.171 M NaCl, 0.3 M NaCl, or 0.2 M sucrose, as described previously (<a href="#B16" role="doc-biblioref" data-xml-rid="B16">16</a>). LB medium (0.085 M NaCl) was used as a control condition. For SPI-2-inducing conditions, bacteria were grown in a modified N-minimal medium [5 mM KCl, 7.5 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.5 mM K<sub>2</sub>SO<sub>4</sub>, 1 mM KH<sub>2</sub>PO<sub>4</sub>, 0.1 mM Tris-HCl, pH 5.5, 38 mM glycerol, 0.1% Casamino Acids] adjusted to pH 5.5 and supplemented to reach a final concentration of 10 μM MgCl<sub>2</sub> (<a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>). N-minimal medium, adjusted to pH 7.4 and supplemented to achieve/reach a final concentration of 10 mM MgCl<sub>2</sub>, was used as a control.</div><div class="figure-wrap"><header><div class="label">Table 1</div></header><figure id="T1" class="table"><figcaption><span class="heading">Table 1</span> Strains used in this study</figcaption><div class="table-wrap"><table><thead data-xml-valign="bottom"><tr data-xml-align="left"><th>Strain</th><th>Feature(s)</th><th>Source or reference</th></tr></thead><tbody data-xml-valign="top"><tr data-xml-align="left"><td><span class="named-content" data-type="genus-species">E. coli</span></td><td> </td><td> </td></tr><tr data-xml-align="left"><td>    DH5α</td><td>F<sup>−</sup> Φ80<i>lacZ</i>ΔM15 Δ(<i>lacZYA-argF</i>)<i>U169 deoR recA1 endA1 hsdR17</i>(r<sub>k</sub><sup>−</sup> m<sub>k</sub><sup>+</sup>) <i>phoA supE44 thi-1 gyrA96 relA1</i> λ<sup>−</sup></td><td>Laboratory collection</td></tr><tr data-xml-align="left"><td><i>S</i>. Typhimurium</td><td> </td><td> </td></tr><tr data-xml-align="left"><td>    14028s</td><td>Wild-type, sequenced isolate</td><td>Laboratory collection</td></tr><tr data-xml-align="left"><td>    ΔSPI-1::Kan</td><td>ΔSTM2865-STM2899 mutant strain, Kan<sup>r</sup></td><td>This work</td></tr><tr data-xml-align="left"><td>    pSipA/FT pmCherry</td><td>Wild-type strain harboring plasmids pSipA/FT and pmCherry</td><td>This work</td></tr><tr data-xml-align="left"><td>    ΔSPI-1 pSipA/FT pmCherry</td><td>ΔSPI-1::Kan mutant strain harboring plasmids pSipA/FT and pmCherry</td><td>This work</td></tr><tr data-xml-align="left"><td><i>S</i>. Gallinarum</td><td> </td><td> </td></tr><tr data-xml-align="left"><td>    287/91</td><td>Wild-type, sequenced isolate</td><td>Laboratory collection</td></tr><tr data-xml-align="left"><td>    WT/pWRKC</td><td><i>attG1</i>::pWRKC</td><td>This work</td></tr><tr data-xml-align="left"><td>    Δ<i>ssaG</i></td><td>Δ<i>SG1710</i> mutant strain</td><td>This work</td></tr><tr data-xml-align="left"><td>    Δ<i>aroA</i></td><td>Δ<i>SG0290</i> mutant strain</td><td>This work</td></tr><tr data-xml-align="left"><td>    ΔSPI-19::Kan</td><td>Δ<i>SG1024-SG1054</i>::Kan mutant strain, Kan<sup>r</sup></td><td>This work</td></tr><tr data-xml-align="left"><td>    ΔSPI-19</td><td>Δ<i>SG1024-SG1054</i> mutant strain</td><td><a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a></td></tr><tr data-xml-align="left"><td>    Δ<i>clpV</i></td><td>Δ<i>SG1034</i> mutant strain</td><td><a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a></td></tr><tr data-xml-align="left"><td>    Δ<i>hcp1</i></td><td>Δ<i>SG1029</i> mutant strain</td><td>This work</td></tr><tr data-xml-align="left"><td>    Δ<i>hcp2</i></td><td>Δ<i>SG1043</i> mutant strain</td><td>This work</td></tr><tr data-xml-align="left"><td>    Δ<i>vgrG</i></td><td>Δ<i>SG1044</i> mutant strain</td><td>This work</td></tr><tr data-xml-align="left"><td>    Δ<i>icmF</i></td><td>Δ<i>SG1031</i> mutant strain</td><td>This work</td></tr><tr data-xml-align="left"><td>    Δ<i>clpV/clpV</i></td><td>Δ<i>SG1034</i> mutant strain harboring plasmid pGEM-T::<i>SG1034</i></td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>hcp1</i>-3×FLAG</td><td><i>SG1029</i>-3XFLAG gene fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>hcp2</i>-3×FLAG</td><td><i>SG1043</i>-3XFLAG gene fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>vgrG</i>-3×FLAG</td><td><i>SG1044</i>-3XFLAG gene fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>vgrG</i>-GFP/pmCherry</td><td><i>SG1044</i>-GFP gene fusion harboring plasmid pmCherry</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>hcp1-lacZ</i></td><td><i>SG1029-lacZ</i> operon fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>hcp2-lacZ</i></td><td><i>SG1043-lacZ</i> operon fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>vgrG-lacZ</i></td><td><i>SG1044-lacZ</i> operon fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>icmF-lacZ</i></td><td><i>SG1031-lacZ</i> operon fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    <i>clpV-lacZ</i></td><td><i>SG1034-lacZ</i> operon fusion</td><td>This work</td></tr><tr data-xml-align="left"><td>    pSipA/FT pmCherry</td><td>Wild-type strain harboring plasmids pSipA/FT and pmCherry</td><td>This work</td></tr><tr data-xml-align="left"><td>    pVgrG/FT pmCherry</td><td>Wild-type strain harboring plasmids pVgrG/FT and pmCherry</td><td>This work</td></tr></tbody></table></div></figure></div></section><section id="sec-2-2"><h3>Standard DNA techniques.</h3><div role="paragraph">Total genomic DNA was obtained from overnight bacterial cultures using the GenElute bacterial genomic DNA kit (Sigma-Aldrich, St. Louis, MO) according to the manufacturer's instructions. Plasmid DNA was obtained from overnight cultures using either the QIAprep Spin Miniprep kit or the QIAprep Spin plasmid Maxikit (Qiagen, Germantown, MD) according to the manufacturer's instructions. PCR products were purified using the QIAquick PCR purification kit (Qiagen, Germantown, MD) as recommended. The multicopy plasmid pGEM-T Easy (Promega, Madison, WI) was used for complementation studies. DNA samples were routinely analyzed by electrophoresis in 1% agarose gels (prepared in 1× Tris-acetate-EDTA buffer) and visualized under UV light after ethidium bromide staining.</div></section><section id="sec-2-3"><h3>Mutant constructions and analyses.</h3><div role="paragraph">Mutant strains with deletions of SPI-19 (<i>SG1024</i> to <i>SG1054</i>), <i>clpV</i> (<i>SG1034</i>), <i>icmF</i> (<i>SG1031</i>), <i>hcp1</i> (<i>SG1029</i>), <i>hcp2</i> (<i>SG1043</i>), and <i>vgrG</i> (<i>SG1044</i>) were constructed using the Red-swap method (<a href="#B18" role="doc-biblioref" data-xml-rid="B18">18</a>). PCR primers 60 bases long were synthesized with 40 nucleotides (nt) on the ends corresponding to the extreme of the desired deletion (<a href="#T2">Table 2</a>) and 20 nt that anneal to the 5′ or 3′ end of a Cam resistance cassette flanked by the FRT sites (FLP recombinase target sequence) present in plasmid pCLF2 (GenBank accession number <a href="http://www.ncbi.nlm.nih.gov/nuccore?term=HM047089">HM047089</a>), which was used as the template DNA. <i>S</i>. Gallinarum strain 287/91 (<a href="#B10" role="doc-biblioref" data-xml-rid="B10">10</a>), carrying the temperature-sensitive plasmid pKD46, which expresses the Lambda-Red recombinase system, was grown to an optical density at 600 nm (OD<sub>600</sub>) of 0.5 at 30°C in LB medium containing Amp and <span data-style="small-caps">l</span>-arabinose (10 mM). Bacteria were made electrocompetent by sequential washes with ice-cold, sterile 10% glycerol and transformed with approximately 500 ng of the corresponding purified PCR product. Transformants were selected on LB Cam agar at 37°C. The presence of each mutation was confirmed by PCR amplification using primers flanking the sites of substitution. Also, each mutant was assayed for Amp sensitivity to confirm the loss of pKD46. Finally, mutant alleles were transferred to the wild-type background by generalized transduction using P22 phage.</div><div class="figure-wrap"><header><div class="label">Table 2</div></header><figure id="T2" class="table"><figcaption><span class="heading">Table 2</span> Oligonucleotides used in this study</figcaption><div class="table-wrap"><table><thead data-xml-valign="bottom"><tr data-xml-align="left"><th>Primer</th><th>Sequence<i><a href="#TF2-1" role="doc-noteref"><sup>a</sup></a></i></th></tr></thead><tbody data-xml-valign="top"><tr data-xml-align="left"><td>SPI-19_(H1+P1)</td><td><span data-style="monospace">TAGCTGAATTGCAATATGCGAAAAAAGCCGAGCTTGATGACAAAC<i>GTGTAGGCTGGAGCTGCTTC</i></span></td></tr><tr data-xml-align="left"><td>SPI-19_(H2+P2)</td><td><span data-style="monospace">AAGCATCTTCAATAATCACGGGTATAAATGCTTACACTCTTTATCCATATGAATATCCTCCTTAG</span></td></tr><tr data-xml-align="left"><td>SEN0997_SEN1009_(H1+P1)</td><td><span data-style="monospace">TAGCTGAATTGCAATATGCGAAAAAAGCCGAGCTTGATGACAAACGTGTAGGCTGGAGCTGCTTC</span></td></tr><tr data-xml-align="left"><td>SEN0997_SEN1009_(H2+P2)</td><td><span data-style="monospace">AAGCATCTTCAATAATCACGGGTATAAATGCTTACACTCTTTATC<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>SG1029_(H1+P1)</td><td><span data-style="monospace">GGAGATTAACATGGCCAATTTAATTTATTTAACACTGAACGTGTAGGCTGGAGCTGCTTC</span></td></tr><tr data-xml-align="left"><td>SG1029_(H2+P2)</td><td><span data-style="monospace">AAAATTTTTACTAAAACACCCTCTCATCCCATAAACTGAA<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>SG1031_(H1+P1)</td><td><span data-style="monospace">GTGTTCAGATTACCGACCCCCCGATTATTCAGCACACTGAGTGTAGGCTGGAGCTGCTTC</span></td></tr><tr data-xml-align="left"><td>SG1031_(H2+P2)</td><td><span data-style="monospace">TCAGTACAGGGTATCCGGCAGCCGGAACTGACTGAACAGC<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>SG1034_(H1+P1)</td><td><span data-style="monospace">ATGATCCAGATTGACTTAGCCACCCTGGTAAAACGGCTTGGTGTAGGCTGGAGCTGCTTC</span></td></tr><tr data-xml-align="left"><td>SG1034_(H2+P2)</td><td><span data-style="monospace">TCATAGAACGGCTTCGTCCTCTGCTTCCGCTACGGGCTGA<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>SG1043_(H1+P1)</td><td><span data-style="monospace">GAGTTTATAAATGCCAACACCGTGTTACATCTCAATTAC<i>GGTGCAGGCTGGAGCTGCTTC</i></span></td></tr><tr data-xml-align="left"><td>SG1043_(H2+P2)</td><td><span data-style="monospace">TTGATTCCGTTTACGCTTCCAGCGGCGCGCGCCAGTCATC<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>SG1044_(H1+P1)</td><td><span data-style="monospace">GAGGTTATTCATGTCAACAGGATTACGTTTCACACTGGA<i>GGTGCAGGCTGGAGCTGCTTC</i></span></td></tr><tr data-xml-align="left"><td>SG1044_(H2+P2)</td><td><span data-style="monospace">ATTATGTTATTCAGTACATGATCAGATCCGATGGATCGGG<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>Ssag_(H1+P1)</td><td><span data-style="monospace">CCAGTATCCTTACGATGTATTTATTTTAAGGAAAAGCATTGCGTAGGCTGGAGCTGCTTC</span></td></tr><tr data-xml-align="left"><td>Ssag_(H2+P2)</td><td><span data-style="monospace">GATTTCCAGCAGCAACCGTCGAACATCGTCGCTAATAACT<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>Hcp1_FLAG_FWD</td><td><span data-style="monospace">CGGGACCAGTGCATTCAGTTTATGGGATGAGAGGGTGTTT<i>GACTACAAAGACCATGACGG</i></span></td></tr><tr data-xml-align="left"><td>Hcp1_FLAG_REV</td><td><span data-style="monospace">TTTACCATGTGCAACCCTCCATTCATTTGTAAAATTTTTA<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>Hcp2_FLAG_FWD</td><td><span data-style="monospace">CACCTCTGGCGCGGATGACTGGCGCGCGCCGCTGGAAGCG<i>GACTACAAAGACCATGACGG</i></span></td></tr><tr data-xml-align="left"><td>Hcp2_FLAG_REV</td><td><span data-style="monospace">TTTACCTCCGTCTCCAGGCAAAAACAGCACCCTGCCCGTC<i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>VgrG_FLAG_FWD</td><td><span data-style="monospace"><i>GACTACAAAGACCATGACGG</i></span></td></tr><tr data-xml-align="left"><td>VgrG_FLAG_REV</td><td><span data-style="monospace"><i>CATATGAATATCCTCCTTAG</i></span></td></tr><tr data-xml-align="left"><td>SG1044_GFP_fwd</td><td><span data-style="monospace">CAGCAGGATGACCCCGATCCATCGGATCTGATCATGTAC<i>AGTAAAGGAGAAGAACTTTTC</i></span></td></tr><tr data-xml-align="left"><td>SG1044_blatem_rev</td><td><span data-style="monospace">CCCGGTGGCTCCTGCTGTAGTCCGGTATTCATTATGTTAT<i>CGTGTAGGCTGGAGCTGCTTC</i></span></td></tr><tr data-xml-align="left"><td>SG1034_OUT5</td><td><span data-style="monospace">ATCCGGCATGTTCTTGCG</span></td></tr><tr data-xml-align="left"><td>SG1043_OUT5</td><td><span data-style="monospace">CATGGTGAATGAGGTTTACG</span></td></tr><tr data-xml-align="left"><td>SG1034_out3_BamHI</td><td><span data-style="monospace"><span data-style="underline">CGGGATCC</span>CAGGAGCGTCTGCATTCTGC</span></td></tr><tr data-xml-align="left"><td>SG1034_out5_BamHI</td><td><span data-style="monospace"><span data-style="underline">CGGGATCC</span>GGCTGCTTGCCAGGTAAGGA</span></td></tr><tr data-xml-align="left"><td>ssag_OUT5</td><td><span data-style="monospace">ATAGCATTAACAGTGCTAAG</span></td></tr><tr data-xml-align="left"><td>SP6</td><td><span data-style="monospace">ATTTAGGTGACACTATAGAAT</span></td></tr><tr data-xml-align="left"><td>T7</td><td><span data-style="monospace">TAATACGACTCACTATAGGG</span></td></tr><tr data-xml-align="left"><td>Lac36</td><td><span data-style="monospace">GACCATTTTCAATCCGCA</span></td></tr><tr data-xml-align="left"><td>SPI19_OUT_UP</td><td><span data-style="monospace">CGCCTTACATAGCTACGATCTCAGG</span></td></tr><tr data-xml-align="left"><td>SPI19_OUT_DOWN</td><td><span data-style="monospace">CAGCTCAGGCAAAGAACCTATGC</span></td></tr><tr data-xml-align="left"><td>FlagTEM_MCS_OUT5</td><td><span data-style="monospace">TGTTGCCCGTCTCACTGGTG</span></td></tr><tr data-xml-align="left"><td>FlagTEM_MCS_OUT3</td><td><span data-style="monospace">CCACTCGTGCACCCAACTGA</span></td></tr><tr data-xml-align="left"><td>SG1044_NdeI</td><td><span data-style="monospace"><span data-style="underline">CATATGAGGTTATTCATGTCAAC</span></span></td></tr><tr data-xml-align="left"><td>SG1044_XhoI_OUT3</td><td><span data-style="monospace"><span data-style="underline">CTCGAG</span>GTA CATGATCAGA TCCGATG</span></td></tr><tr data-xml-align="left"><td>LepA_F</td><td><span data-style="monospace">CGTAATCATGGTCATAGCTGTTTCC</span></td></tr><tr data-xml-align="left"><td>LepA_R</td><td><span data-style="monospace">ACTTCAAGATCCATTTCCATCG</span></td></tr></tbody></table></div><div class="notes"><div class="labeled" role="doc-footnote"><div class="label"><sup>a</sup></div><div id="TF2-1" role="paragraph">Italics indicate the region that anneals to the 5′ or 3′ end of the antibiotic resistance cassette used for the mutagenesis. Underlining indicates XbaI, BamHI, XhoI, or NdeI restriction sites.</div></div></div></figure></div><div role="paragraph">To obtain nonpolar deletions, the Cam resistance gene was removed by transforming each mutant with pCP20, which encodes the FLP recombinase. Transformants were plated on LB Amp agar at 30°C. Individual colonies were replica plated on LB agar, LB Amp agar, and LB Cam agar at 37°C. Transformants that had lost the resistance cassette and pCP20 were selected as those colonies that were unable to grow in the presence of Amp and Cam. The absence of the antibiotic resistance gene cassette was confirmed for each mutant by PCR analysis.</div><div role="paragraph">To obtain the <i>S</i>. Typhimurium 14028s ΔSPI-1 mutant strain, a P22 phage lysate from ΔSPI-1::Kan <i>S</i>. Typhimurium SL1344 (NCTC 13347) (<a href="#B19" role="doc-biblioref" data-xml-rid="B19">19</a>) was used to transduce this mutation to the 14028s background.</div></section><section id="sec-2-4"><h3>Construction of <i>lacZ</i> operon fusions.</h3><div role="paragraph">To construct <i>lacZ</i> operon fusions, the method developed by Ellermeier and collaborators was used (<a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a>). This method consists of the specific integration of a suicide plasmid (pCE36) that has an FRT site directly upstream of promoterless <i>lacZY</i> into a chromosomal FRT site from a previous Red-swap mutation. First, <i>S</i>. Gallinarum mutant strains Δ<i>hcp1</i>::FRT, Δ<i>hcp2</i>::FRT, Δ<i>icmF</i>::FRT, Δ<i>clpV</i>::FRT, and Δ<i>vgrG</i>::FRT were transformed with the pCP20 plasmid and selected for Amp resistance at 30°C. These strains were then grown at 30°C and transformed by electroporation with plasmid pCE36. Transformants were recovered in LB, and dilutions were plated on LB Kan 5-bromo-4-chloro-3-indolyl-β-<span data-style="small-caps">d</span>-galactopyranoside (X-Gal) agar at 37°C. Growth at 37°C allows for the optimal production of FLP recombinase and blocks the replication of the temperature-sensitive plasmid pCP20, resulting in the stable integration of pCE36 and the loss of pCP20. Fusions were checked by PCR with the appropriate set of primers (<a href="#T2">Table 2</a>) to ensure integration of pCE36 in the correct chromosomal location and to rule out the presence of multiple plasmid integrants.</div></section><section id="sec-2-5"><h3>Construction of 3×FLAG gene fusions.</h3><div role="paragraph">Epitope tagging of T6SS components was performed by constructing gene fusions between the <i>hcp1</i>, <i>hcp2</i>, and <i>vgrG</i> open reading frames (ORFs) and the 3×FLAG epitope by a modification of the Red-swap method described by Uzzau and collaborators (<a href="#B21" role="doc-biblioref" data-xml-rid="B21">21</a>) using primers listed in <a href="#T2">Table 2</a>. The PCR products were amplified from template plasmid pSUB11 (<a href="#T3">Table 3</a>), and the Red-swap protocol was similar to that described above. Each fusion was checked by PCR and transduced into a clean wild-type background, and then the resistance marker was excised from the chromosome as described above.</div><div class="figure-wrap"><header><div class="label">Table 3</div></header><figure id="T3" class="table"><figcaption><span class="heading">Table 3</span> Plasmids used in this study</figcaption><div class="table-wrap"><table><thead data-xml-valign="bottom"><tr data-xml-align="left"><th>Plasmid</th><th>Feature(s)</th><th>Source or reference</th></tr></thead><tbody data-xml-valign="top"><tr data-xml-align="left"><td>pKD46</td><td><i>bla</i> P<i><sub>BAD</sub> gam bet exo</i> pSC101 <i>ori</i>(Ts), Amp<sup>r</sup></td><td><a href="#B18" role="doc-biblioref" data-xml-rid="B18">18</a></td></tr><tr data-xml-align="left"><td>pGEM-T Easy</td><td>Cloning vector, Amp<sup>r</sup></td><td>Promega</td></tr><tr data-xml-align="left"><td>pGEM-T::<i>clpV</i></td><td>pGEM-T Easy harboring <i>clpV</i>, Amp<sup>r</sup></td><td>This work</td></tr><tr data-xml-align="left"><td>p3174</td><td>Plasmid encoding the <i>gfpmut3</i> gene</td><td><a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a></td></tr><tr data-xml-align="left"><td>pSUB11</td><td>3×FLAG gene fusions plasmid, oriR6K, Kan<sup>r</sup></td><td><a href="#B21" role="doc-biblioref" data-xml-rid="B21">21</a></td></tr><tr data-xml-align="left"><td>pCP20</td><td><i>bla cat cI</i>857 λP<sub>R</sub> <i>flp</i> pSC101 <i>ori</i>(Ts), Cam<sup>r</sup>, Amp<sup>r</sup></td><td><a href="#B69" role="doc-biblioref" data-xml-rid="B69">69</a></td></tr><tr data-xml-align="left"><td>pCE36</td><td><i>aph</i> FRT <i>lacZY</i><sup>+</sup> t<i><sub>his</sub></i> oriR6K, Kan<sup>r</sup></td><td><a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a></td></tr><tr data-xml-align="left"><td>pCLF2</td><td>Red-swap-redesigned vector, Cam<sup>r</sup></td><td><a href="#B70" role="doc-biblioref" data-xml-rid="B70">70</a></td></tr><tr data-xml-align="left"><td>pCLF4</td><td>Red-swap-redesigned vector, Kan<sup>r</sup></td><td><a href="#B71" role="doc-biblioref" data-xml-rid="B71">71</a></td></tr><tr data-xml-align="left"><td>pWRKC</td><td>Derivative of <i>pir</i>-dependent plasmid pNFB9, integrates at Gifsy-1 attachment site <i>attG1</i>, Cam<sup>r</sup></td><td>This work</td></tr><tr data-xml-align="left"><td>pmCherry</td><td>Constitutive production of the mCherry fluorescent protein, Amp<sup>r</sup></td><td>C. Marolda</td></tr><tr data-xml-align="left"><td>pFlagTEM-1</td><td>Plasmid used to construct translational fusions to the mature TEM-1 β-lactamase reporter</td><td><a href="#B26" role="doc-biblioref" data-xml-rid="B26">26</a></td></tr><tr data-xml-align="left"><td>pSipA/FT</td><td>Plasmid which encodes the translational fusion between the mature TEM-1 β-lactamase reporter and SipA (STM2882)</td><td><a href="#B26" role="doc-biblioref" data-xml-rid="B26">26</a></td></tr><tr data-xml-align="left"><td>pVgrG/FT</td><td>Plasmid which encodes the translational fusion between the mature TEM-1 β-lactamase reporter and VgrG (SG1044)</td><td>This work</td></tr></tbody></table></div></figure></div></section><section id="sec-2-6"><h3>Construction of a VgrG-GFP gene fusion.</h3><div role="paragraph">A gene fusion between the green fluorescent protein (GFP) ORF and the ORF of the VgrG T6SS component (termed VgrG-GFP) was constructed by a modification of the Red-swap method described by Gerlach and collaborators (<a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a>). The PCR product was amplified using primers listed in <a href="#T2">Table 2</a> and plasmid p3174 as the template (<a href="#T3">Table 3</a>) and included the ORF encoding the GFP reporter followed by a Kan resistance cassette flanked by FRT sites. Proper integration of the reporter cassette was confirmed by PCR. Positive clones were moved into a wild-type background by generalized transduction using P22, and the resistance marker was excised from the chromosome as described above.</div></section><section id="sec-2-7"><h3>Protein sample preparations.</h3><div role="paragraph">For whole-cell lysates, bacteria were grown overnight in an orbital shaker at 37°C and subcultured in the appropriate growth media. When the culture reached an OD<sub>600</sub> of ∼0.3 to 0.4, the cells were collected by centrifugation (13,000 rpm for 2 min). The supernatant was removed and the pellet was weighted in order to normalize samples. Each mg of bacterial pellet was resuspended in 10 μl of 1× protein loading buffer (62.5 mM Tris-HCl, pH 6.8, 2% SDS, 10% glycerol, 0.01% bromophenol blue, 5% β-mercaptoethanol). Finally, the samples were boiled for 10 min and stored at −20°C.</div><div role="paragraph">For protein secretion analyses, bacterial cells were removed by centrifugation (13,000 rpm for 2 min). The culture supernatant was filter sterilized and mixed with trichloroacetic acid (TCA) to a final concentration of 10%. The samples were chilled on ice for 30 min and centrifuged at 13,000 rpm for 15 min. The protein pellet was washed twice with ice-cold acetone, air dried, and resuspended in 1× protein loading buffer. Finally, the samples were boiled for 5 min and stored at −20°C.</div></section><section id="sec-2-8"><h3>SDS-PAGE analyses and immunoblotting.</h3><div role="paragraph">Polyacrylamide gels (12%) were prepared and mounted using a Mini-Protean III (Bio-Rad, Richmond, CA) PAGE system. The samples were run at 100 V (constant) with 1× protein running buffer (1.44% glycine, 0.3% Tris, 0.1% SDS). Transfer from the polyacrylamide gel was performed in a Mini Trans-Blot system (Bio-Rad, Richmond, CA). A polyvinylidene difluoride (PVDF) membrane was activated by immersion in 100% methanol for 15 s. The transfer was performed for 65 min at 230 mA in 1× transfer buffer (1.44% glycine, 0.3% Tris, 0.01% SDS, and 20% methanol). The membrane was blocked by incubation in a solution containing 5% nonfat milk in TBS (25 mM Tris, pH 8, 125 mM NaCl) for 90 min at room temperature. Later, the membrane was incubated at 4°C for 2 h with the M2 anti-FLAG primary antibody (Sigma-Aldrich, St. Louis, MO) diluted at 1:10,000 in TBS. For sigma 70 detection, an anti-RNA polymerase sigma 70 antibody (ab12088) was used (Abcam, Cambridge, United Kingdom) at a 1:1,000 dilution in TBS. After five washes with a TBS–0.05% Tween solution, the membrane was incubated at room temperature for 2 h with a 1:20,000 dilution of the secondary antibody conjugated with horseradish peroxidase. For immunodetection, the Supersignal West-Pico solution (Pierce, Rockford, IL) was used, exposing the membrane on a CL-Xposure (Pierce, Rockford, IL) photographic film.</div></section><section id="sec-2-9"><h3>β-Galactosidase assays.</h3><div role="paragraph">Bacteria were grown overnight in LB broth and then subcultured and grown in 50 ml of the appropriate medium at 37°C in an orbital shaker. Every 30 min, a 1-ml sample was withdrawn to evaluate bacterial growth (OD<sub>600</sub> and CFU/ml), and the β-galactosidase activity was measured as described previously (<a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a>). Enzyme activities (in Miller units), normalized for cell density (OD<sub>600</sub>), were calculated using the following equation: [<i>A</i><sub>420</sub> − (1.75 × <i>A</i><sub>550</sub>)] × 1,000/(reaction time × culture volume × OD<sub>600</sub>), where reaction time was measured in minutes and culture volume in ml. Each sample was analyzed in triplicate in at least three independent experiments.</div></section><section id="sec-2-10"><h3>Bacterial killing assays.</h3><div role="paragraph">Bacterial strains used for the interbacterial competition studies included <span class="named-content" data-type="genus-species">Enterobacter cloacae</span> (ATCC 700323), <span class="named-content" data-type="genus-species">Shigella sonnei</span> (ATCC 25931), <span class="named-content" data-type="genus-species">Escherichia coli</span> (ATCC 35218), <span class="named-content" data-type="genus-species">Escherichia coli</span> K-12 (ATCC 25922), <span class="named-content" data-type="genus-species">Serratia marcescens</span>, and <span class="named-content" data-type="genus-species">Staphylococcus saprofiticus</span> (ATCC BAA750). Each of these strains was tested for competition against antibiotic-resistant derivatives of the wild-type and ΔSPI-19 mutant strains. A Cam-resistant derivative of <i>S</i>. Gallinarum was generated by transformation with plasmid pWRKC (Cam<sup>r</sup>), a derivative of plasmid pNFB9 which can integrate at the Gifsy-1 attachment site (<i>attG1</i>) in the chromosome (<a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a>). The chromosomal insertion of pWRKC, flanking the <i>attG1</i> site, was confirmed by PCR amplification using primers LepA_F and LepA_R (<a href="#T2">Table 2</a>). For competition assays, each strain was grown overnight in LB media and then collected and washed three times in sterile phosphate-buffered saline (PBS; 13,000 rpm for 2 min). The cell suspensions were normalized to an OD<sub>600</sub> of 0.5 and mixed at a 1:1 ratio with either the <i>S</i>. Gallinarum pWRKC (Cam<sup>r</sup>) or the ΔSPI-19::Kan (Kan<sup>r</sup>) strain. Fifty μl of the mixture then was spotted onto a prewarmed brain heart infusion (BHI) agar plate and incubated for 1 week at 37°C. The initial inoculum was serially diluted and plated to determine the initial CFU/ml. After 1 week, cells were recovered from the plates and resuspended in 1 ml of sterile PBS, and serial dilutions were plated out on BHI agar with or without kanamycin. The amount of prey CFU was determined by subtracting the Kan-resistant CFU counts from the total CFU counts.</div></section><section id="sec-2-11"><h3>Cell culture conditions.</h3><div role="paragraph">The murine macrophage cell line RAW264.7 and the avian macrophage line HD11 were routinely maintained in Dulbecco's modified Eagle medium (DMEM) high glucose (high glucose is composed of 584 mg/liter <span data-style="small-caps">l</span>-glutamine, 4,500 mg/liter glucose, 110 mg/liter sodium pyruvate) supplemented with 10% inactivated bovine fetal serum (FBS) and antibiotic/antimycotic solution (10,000 nU/ml penicillin, 10,000 mg/liter streptomycin, 25 mg/liter amphotericin B) at 37°C in a 5% CO<sub>2</sub> atmosphere.</div></section><section id="sec-2-12"><h3><i>In vitro</i> infection assays.</h3><div role="paragraph">Gentamicin protection assays were performed according to Schwan and collaborators (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>). RAW264.7 and HD11 macrophages were seeded in 48-well cell culture plates at a density of ∼2.5 × 10<sup>5</sup> cells per well. Bacterial strains were grown overnight at 37°C in an orbital shaker, and on the day of the assay they were subcultured in LB at a 1:30 dilution and grown until they reached an OD<sub>600</sub> of ∼0.6. The bacterial cells were collected by centrifugation (13,000 rpm for 2 min), washed three times with sterile PBS, and suspended in prewarmed 10% FBS-DMEM to reach a final concentration of 2.5 × 10<sup>7</sup> CFU/ml. This suspension was serially diluted, and bacteria were quantified by plating on LB agar plates. Macrophages were washed once with sterile PBS, and then 100 μl of the bacterial suspension was added (multiplicity of infection [MOI] of 10). Plates were centrifuged at 250 × <i>g</i> for 5 min at room temperature to synchronize infection, and invasion was allowed to proceed for 45 min (time point 0). After this time, the wells were washed 3 times with warm, sterile PBS and the cells were further incubated for 1 h with 10% FBS-DMEM supplemented with gentamicin (200 μg/ml) to kill extracellular bacteria. The gentamicin concentration then was lowered to 10 μg/ml for the rest of the experiment. At 2 and 20 h postuptake, different wells were washed 3 times with warm, sterile PBS, and the macrophages were lysed with 0.5% sodium deoxycholate (DOC) in PBS. Serial dilutions were prepared and plated on LB agar plates to determine viable intracellular bacteria. For each assay, viable macrophages were determined by trypan blue exclusion. The intracellular survival of mutant strains was calculated as CFU per 10<sup>5</sup> viable macrophages.</div></section><section id="sec-2-13"><h3>Viability and cytotoxicity assays.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">Salmonella</span>-induced cytotoxicity was determined by the CytoTox 96 nonradioactive cytotoxicity assay (Promega, Madison, WI). For each assay, RAW264.7 and HD11 cells were seeded in 96-well plates at a density of ∼30,000 cells per well. The next day, the cells were infected with the bacterial strains as described for the infection assays. At 20 h postuptake, the supernatant of infected cells was collected and transferred to a new 96-well plate. The cytotoxicity reagent then was added following 30 min of incubation at room temperature protected from light. After incubation, absorbance at 562 nm was measured and the cytotoxicity levels were calculated as a percentage of lactate dehydrogenase (LDH) release normalized to the spontaneous release from noninfected cells.</div><div role="paragraph">For viability and sub-G<sub>1</sub> population analyses, RAW264.7 and HD11 macrophages were seeded at a density of ∼300,000 cells per well. Sub-G<sub>1</sub> analysis allows the detection of hypoploid cells, which are cells with lower DNA content due to apoptotic cell death. Gentamicin protection assays were performed as described above. At 20 h postuptake, cells from both the supernatant and the monolayer were recovered (to obtain a complete infected cell population), washed, and resuspended in 1 ml of warm 10% FBS-DMEM. For viability analysis, 5 μl of a 1 mg/ml stock of propidium iodide (PI) was added to 500 μl of the cell suspension, and the samples were analyzed by flow cytometry. For sub-G<sub>1</sub> analysis, a volume of 500 μl of the cell suspension was used. Cells were collected by centrifugation and then treated with 100% methanol for 1 h at −20°C. After incubation, cells were collected and resuspended in 500 μl of PBS containing RNase at a concentration of 50 μg/ml. After incubation for 1 h, 5 μl of a 1 mg/ml stock of PI was added. The levels of PI incorporation were quantified on a FACScan flow cytometer (Becton, Dickinson). Cell size was evaluated by forward-angle light scattering (FSC). PI-negative cells of normal size were considered alive. A total of 5,000 cells per sample were analyzed.</div></section><section id="sec-2-14"><h3>Epifluorescence microscopy.</h3><div role="paragraph">For epifluorescence analysis of VgrG-GFP production upon internalization of <i>S</i>. Gallinarum into RAW264.7 and HD11 macrophages, cells were seeded on 25-mm coverslips in 6-well plates at a density of ∼375,000 cells per well. After 20 h, the cells were infected with the bacterial strains as described for the infection assays. At either 45 min or 2 h postuptake, the cells were washed 3 times with warm, sterile PBS and fixed with a 4% formaldehyde solution for 20 min at room temperature. After incubation, the coverslips were washed with sterile PBS and mounted in a 4′,6-diamidino-2-phenylindole (DAPI)-containing solution from VectaShield (Vector Laboratories, Burlington, CA). For Cytochalasin D analysis, cells were incubated for 1 h with 10 μg/ml Cytochalasin D before infection and maintained with this concentration throughout the experiment. The samples were stored at 4°C and visualized under an epifluorescence microscope. Fluorescence microscopy analysis was performed using a Spinning Disk Olympus IX 81 fluorescence microscope, and fluorescence micrographs were captured using a Zeiss Axiocam MRC5 and CellR software.</div></section><section id="sec-2-15"><h3>VgrG translocation assays.</h3><div role="paragraph">The <i>vgrG</i> ORF (<i>SG1044</i>), without the corresponding stop codon, was amplified by PCR using as the template a preparation of <i>S</i>. Gallinarum 287/91 genomic DNA and cloned into plasmid pFlagTEM-1 (<a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>, <a href="#B26" role="doc-biblioref" data-xml-rid="B26">26</a>). The production of the VgrG-BlaTEM recombinant protein was assessed by SDS-PAGE and Western blot analysis using anti-3×FLAG monoclonal antibodies. To demonstrate T6SS-dependent VgrG translocation, a modified gentamicin protection assay was performed as described previously (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>). RAW264.7 and HD11 macrophages were seeded in 96-well optical plates (black walls and clear, flat bottoms) and infected with each of the bacterial strains expressing the corresponding BlaTEM fusions.</div><div role="paragraph">Bacteria were grown to an OD<sub>600</sub> of 0.3 and induced with 1 mM isopropyl-β-<span data-style="small-caps">d</span>-thiogalactopyranoside (IPTG) until they reached an OD<sub>600</sub> of 0.6. The cells were collected and washed three times with sterile PBS (13,000 rpm for 2 min) and resuspended in the appropriate volume of prewarmed 10% FBS-DMEM. Infections were performed for 45 min at an MOI of 50. Noninternalized bacteria were removed by three washes with sterile PBS, and cells were further incubated with 200 μl 10% FBS-DMEM supplemented with 1 mM IPTG and 200 μg/ml gentamicin. After 1 h of incubation, the cell supernatant was replaced with 100 μl Hanks' buffered salt solution (HBSS; Gibco) supplemented with 20 mM HEPES and 3 mM probenecid (pH 7.4; Sigma-Aldrich, St. Louis, MO) (designated HBSS-HP), and 20 μl freshly prepared CCF4-AM β-lactamase substrate (LiveBLAzer FRET-B/G loading kit; Invitrogen) was added. After 90 min of incubation at room temperature in the dark, the cells were washed five times with HBSS-HP. Fluorescence emission at 450 and 520 nm was measured from the bottom using a plate reader fluorometer (Synergy H1; BioTek) (410-nm excitation wavelength, 10-nm band pass). The translocation level was calculated as recommended in the LiveBLAzer FRET-B/G loading kit manual. Briefly, emission values were first corrected by subtraction of the average background signals recorded for empty wells, and the mean 450-nm/520-nm emission ratio of a triplicate of wells was calculated for each sample. The translocation level is expressed as fold increase of the mean 450-nm/520-nm emission ratio of each infected sample relative to the mean emission ratio of uninfected cells. Epifluorescence microscopy analysis was performed using a Spinning Disk Olympus IX 81 fluorescence microscope. Fluorescence micrographs were captured using a Zeiss Axiocam MRC5 and CellR software. Images were imported into the ImageJ software (v.1.44).</div></section><section id="sec-2-16"><h3>Statistical analysis.</h3><div role="paragraph">The statistical significance of differences in the data was determined using one-way analysis of variance (ANOVA) with Dunnett's posttest (GraphPad Prism v5.0).</div></section></section><section id="sec-3" data-type="results"><h2>RESULTS</h2><section id="sec-3-1"><h3>SPI-19 T6SS components are expressed and produced under <i>in vitro</i> bacterial growth conditions.</h3><div role="paragraph">A recent report from our group provided genetic evidence of SPI-19 T6SS functionality (<a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>). Nevertheless, no study regarding expression, production, and secretion of its components has been performed to date. In order to further characterize SPI-19 T6SS in <i>S</i>. Gallinarum, operon fusions to the <i>lacZ</i> reporter were constructed for the conserved core components <i>hcp1</i>, <i>hcp2</i>, <i>vgrG</i>, <i>clpV</i>, and <i>icmF</i>, and gene fusions to the 3×FLAG epitope were constructed for <i>hcp1</i>, <i>hcp2</i>, and <i>vgrG</i>, as they encode secreted components of T6SS (<a href="#F1">Fig. 1A</a> and <a href="#T1">Table 1</a>). Different <i>in vitro</i> growth conditions were tested to study the expression and production of these components, including (i) standard bacterial culture conditions (growth in LB medium at 37°C with agitation); (ii) SPI-1-inducing growth conditions modified to mimic the conditions found by <i>S</i>. Gallinarum in the intestinal lumen of the chicken (42°C, high osmolarity, anaerobiosis) (<a href="#B16" role="doc-biblioref" data-xml-rid="B16">16</a>); and (iii) SPI-2-inducing growth conditions intended to mimic those found by the bacterium inside macrophages (pH 5.5 and 10 μM MgCl<sub>2</sub>) (<a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>). Even though these media cannot fully replicate the conditions encountered by <span class="named-content" data-type="genus-species">Salmonella</span> in the host, they have proven useful for identifying underlying regulatory networks in <span class="named-content" data-type="genus-species">Salmonella</span> (<a href="#B16" role="doc-biblioref" data-xml-rid="B16">16</a>, <a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>, <a href="#B27" role="doc-biblioref" data-xml-rid="B27">27</a><a href="#B28" role="doc-biblioref" data-xml-rid="B28">–</a><a href="#B29" role="doc-biblioref" data-xml-rid="B29">29</a>).</div><div class="figure-wrap"><header><div class="label">Fig 1</div></header><figure id="F1" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/baf09c1c-acfd-4c3a-9c54-5718abcbbb52/assets/graphic/zii9990900630001.jpeg" /><figcaption><span class="heading">Fig 1</span> <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 19 and the T6SS-related toxin/antitoxin gene cluster. (A) Schematic representation of the SPI-19 T6SS gene cluster in <i>S</i>. Gallinarum. (B) The putative toxin/antitoxin module identified by Russell and colleagues (<a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>). T6SS core components are shown in gray. Components studied in this work are shown in black.</figcaption></figure></div><div role="paragraph">Expression analyses revealed a growth phase-dependent regulation of each gene analyzed under standard bacterial culture conditions, with the <i>hcp1</i> gene showing the highest expression levels of all the genes tested (<a href="#F2">Fig. 2A</a>). Gene expression analyses under SPI-1- and SPI-2-inducing conditions showed a significant increase only for <i>hcp1</i> under SPI-2-inducing conditions (data not shown).</div><div class="figure-wrap"><header><div class="label">Fig 2</div></header><figure id="F2" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/3cf7420a-1619-4828-881a-516abbd94440/assets/graphic/zii9990900630002.jpeg" /><figcaption><span class="heading">Fig 2</span> Expression, production, and secretion analysis of selected SPI-19 T6SS components under different <i>in vitro</i> conditions. (A) Activity of <i>lacZ</i> operon fusions to <i>hcp1</i>, <i>clpV</i>, <i>icmF</i>, <i>hcp2</i>, and <i>vgrG</i> during the bacterial growth curve. Expression was measured as β-galactosidase activity (expressed as Miller units). (B) Production and secretion analyses of 3×-FLAG epitope-tagged versions of Hcp1, Hcp2, and VgrG from whole-cell lysates and supernatant fractions. The asterisk indicates an unspecific band detected with the 3×-FLAG antibody. Relative quantification (RQ) of the Hcp1 protein was determined by densitometric analysis and is expressed as the ratio against the production levels of the nonspecific band detected with the 3×-FLAG antibody. Detection of the RNA polymerase by an α-sigma 70 antibody was used as a control to analyze protein secretion in the cell culture supernatants. Under standard bacterial growth conditions (LB), samples were acquired at both exponential (Ex; OD<sub>600</sub> of 0.3) and stationary (St; OD<sub>600</sub> of 1.8) phases of growth.</figcaption></figure></div><div role="paragraph">Production of the structural/secreted proteins Hcp1, Hcp2, and VgrG revealed that these three proteins are produced <i>in vitro</i> (<a href="#F2">Fig. 2B</a>). The Hcp1 protein was detected under all of the conditions tested, and relative quantification by gel densitometric analysis of its production revealed that increased levels of the protein were present under SPI-2-inducing conditions (<a href="#F2">Fig. 2B</a>). In contrast to expression analyses, the Hcp2 and VgrG proteins were detected only at the exponential phase of bacterial growth and were barely detected at stationary phase or under SPI-1- and SPI-2-inducing conditions (<a href="#F2">Fig. 2B</a>).</div><div role="paragraph">Regardless of the differences observed for the expression and production of T6SS core components, it was not possible to detect secretion of either Hcp1, Hcp2, or VgrG to the culture media under the different growth conditions tested (<a href="#F2">Fig. 2B</a>). These findings suggest that additional signals are required for full activation of SPI-19 T6SS in <i>S</i>. Gallinarum.</div></section><section id="sec-3-2"><h3>SPI-19 T6SS is not involved in interbacterial killing under different <i>in vitr</i>o growth conditions.</h3><div role="paragraph">An increasing number of T6SSs have been linked to interbacterial killing against different Gram-negative bacteria through the delivery of different toxins which target the peptidoglycan of susceptible bacterial species (<a href="#B30" role="doc-biblioref" data-xml-rid="B30">30</a><a href="#B31" role="doc-biblioref" data-xml-rid="B31 B32 B33">–</a><a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>). A heuristic informatic approach was recently developed for large-scale identification of T6SS antibacterial effectors (<a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>). In <span class="named-content" data-type="genus-species">Salmonella</span>, this approach identified several toxin/antitoxin modules inside the SPI-6 T6SS gene cluster and one additional module in a genomic region not associated with T6SS loci (<i>tae3</i> and <i>tai3</i> genes in <a href="#F1">Fig. 1B</a>) (<a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>). Interestingly, this module is present in every <span class="named-content" data-type="genus-species">Salmonella enterica</span> genome we have analyzed (data not shown), so it is tempting to speculate that even though the SPI-19 T6SS gene cluster does not seem to encode a toxin/antitoxin module, it could be involved in interbacterial killing through translocation of the toxin encoded in this novel cluster (<a href="#F1">Fig. 1B</a>).</div><div role="paragraph">To test this, an <i>in vitro</i> competition assay was performed using the wild type or a ΔSPI-19 derivative of <i>S</i>. Gallinarum against a panel of 6 Gram-negative bacteria, including bacteria previously shown to be susceptible to T6SS killing. A Gram-positive strain was included in the assay as a control. As shown in <a href="#F3">Fig. 3</a>, each prey strain tested grew at similar levels after coculture with either the wild-type or the SPI-19 mutant strain. When the same experiment was performed under different <i>in vitro</i> growth conditions (30, 37, and 42°C and SPI-1-inducing conditions), no differences were observed (data not shown). These results indicate that <i>S</i>. Gallinarum SPI-19 T6SS is not involved in interbacterial killing under the conditions studied.</div><div class="figure-wrap"><header><div class="label">Fig 3</div></header><figure id="F3" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/af559b8b-c1bd-4f14-8560-aa20d53fdb11/assets/graphic/zii9990900630003.jpeg" /><figcaption><span class="heading">Fig 3</span> SPI-19 T6SS is not involved in interbacterial killing. Survival of prey strains listed below the <i>x</i> axis was determined by measuring the corresponding CFU after 1 week of exposure to either the wild type or a ΔSPI-19 mutant of <i>S</i>. Gallinarum. Data represent CFU from three independent experiments. Bars represent mean values, and error bars denote standard deviations. No statistically significant differences between the mutant and the wild type were determined by one-way ANOVA with Dunnett's posttest.</figcaption></figure></div></section><section id="sec-3-3"><h3>VgrG is produced after <i>S</i>. Gallinarum internalization by murine and avian macrophages.</h3><div role="paragraph">The contribution of the SPI-19 T6SS to <i>S</i>. Gallinarum colonization of orally infected chickens suggested a role for this system in the interaction of <span class="named-content" data-type="genus-species">Salmonella</span> with eukaryotic cells, especially those of the reticuloendothelial system (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>). Because expression of T6SS components in other bacteria is induced within eukaryotic cells (reviewed in reference <a href="#B36" role="doc-biblioref" data-xml-rid="B36">36</a>), we analyzed the production of the VgrG protein within macrophages. To date, most studies regarding the ability of <span class="named-content" data-type="genus-species">Salmonella</span> to infect and survive within macrophages have been performed in murine macrophages. However, since <i>S</i>. Gallinarum corresponds to an avian-adapted serotype, we chose to use both murine (RAW264.7) and avian (HD11) macrophages and compare the results obtained for each cell line.</div><div role="paragraph">A chromosomal gene fusion was constructed between the <i>vgrG</i> ORF and the ORF encoding the GFP reporter using the method described by Gerlach and colleagues (<a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a>). The <i>vgrG</i> gene was chosen because (i) it is an essential T6SS component, (ii) production of VgrG was not detected under the conditions we tested, including those used prior to infection (either LB medium, 37°C, and an OD<sub>600</sub> of 0.6 or DMEM, 5% CO<sub>2</sub>, and 37°C) (<a href="#F4">Fig. 4A</a>), and (iii) it has been shown that VgrG C-terminal fusions can be constructed without affecting the functionality of the corresponding T6SS (<a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a>).</div><div class="figure-wrap"><header><div class="label">Fig 4</div></header><figure id="F4" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/38ad296e-d063-4f2e-837d-dcaf719a173a/assets/graphic/zii9990900630004.jpeg" /><figcaption><span class="heading">Fig 4</span> Intracellular production of the VgrG-GFP fusion protein. (A) Production of VgrG-GFP under cell culture (10% FBS-DMEM, 5% CO<sub>2</sub>, 37°C) and standard bacterial growth conditions (OD<sub>600</sub> of 0.6, LB, and 37°C). (B and C) Production of VgrG-GFP after interaction in RAW264.6 (B) and HD11 (C) macrophages. Samples were obtained after 45 min of infection and after 2 h postuptake. Samples were prepared in the presence or absence of the phagocytosis inhibitor Cytochalasin D. Bacterial cells were visualized through constitutive production of the mCherry fluorescent protein from plasmid pmCherry. Macrophages were visualized by DAPI staining and bright-field observation. The white scale bar represents 5 μm.</figcaption></figure></div><div role="paragraph">To analyze the production of the VgrG-GFP fusion protein after <span class="named-content" data-type="genus-species">Salmonella</span> internalization by macrophages, gentamicin protection assays were performed. RAW264.7 and HD11 cells were seeded onto sterile glass coverslips and infected with an <i>S</i>. Gallinarum strain harboring the VgrG-GFP fusion in its chromosome. This strain also carries the pmCherry plasmid, encoding the red fluorescent protein mCherry under the control of a constitutive promoter, to allow bacterial visualization. After 45 min of infection, cells were washed and cell culture media added with a final concentration of 100 μg/ml gentamicin to kill extracellular bacteria. Two hours postuptake, the cells were washed, fixed, and mounted for their analysis by epifluorescence microscopy. As shown in <a href="#F4">Fig. 4B</a> and <a href="#F4">C</a>, intracellular production of the VgrG-GFP fusion was detected in bacteria expressing cytosolic mCherry in both murine and avian macrophages at 2 h postuptake.</div><div role="paragraph">To determine whether production of VgrG was induced by the initial cell contact between <span class="named-content" data-type="genus-species">Salmonella</span> and the macrophages or occurred after bacterial internalization, cells were pretreated with Cytochalasin D to inhibit <span class="named-content" data-type="genus-species">Salmonella</span> uptake. As shown in <a href="#F4">Fig. 4B</a> and <a href="#F4">C</a>, pretreatment with Cytochalasin D prevented VgrG production in both murine and avian macrophages infected with <i>S</i>. Gallinarum. Interestingly, infections performed with the HD11 macrophage cell line revealed that <i>S</i>. Gallinarum was detected in granule/vacuolar structures after internalization. This was not seen in infected murine macrophages (<a href="#F4">Fig. 4C</a>). Further studies will be needed to fully characterize the intracellular fate of <i>S</i>. Gallinarum in HD11 macrophages compared to results from previous studies performed in murine macrophages. Altogether, our results indicate that production of VgrG is induced once <i>S</i>. Gallinarum is internalized by macrophages of either murine or avian origin.</div></section><section id="sec-3-4"><h3>SPI-19 T6SS contributes to <i>S</i>. Gallinarum intracellular survival within murine and avian macrophages.</h3><div role="paragraph">Intracellular production of VgrG in RAW264.7 and HD11 cells suggested that SPI-19 T6SS is involved in the interaction of <span class="named-content" data-type="genus-species">Salmonella</span> with macrophages. To test this, a set of defined <i>S</i>. Gallinarum mutants was constructed and tested in intracellular survival assays in both murine and avian macrophages. Gentamicin protection assays were performed, infecting each macrophage cell line with either the wild-type, ΔSPI-19, or Δ<i>clpV</i> mutant strain. As controls, Δ<i>ssaG</i> and Δ<i>aroA</i> mutant strains were also included, as both genes encode proteins required for <span class="named-content" data-type="genus-species">Salmonella</span> survival within macrophages. The <i>ssaG</i> gene encodes an essential SPI-2 T3SS structural component (<a href="#B38" role="doc-biblioref" data-xml-rid="B38">38</a>), while <i>aroA</i> corresponds to an auxotrophic mutant for aromatic amino acids with impaired ability to survive and grow within macrophages (<a href="#B39" role="doc-biblioref" data-xml-rid="B39">39</a>).</div><div role="paragraph">At 2 and 20 h postuptake, intracellular bacteria were recovered and bacterial loads were determined and expressed as CFU per 10<sup>5</sup> macrophages. As shown in <a href="#F5">Fig. 5</a>, none of the T6SS mutants showed a significant intracellular survival defect compared to the wild-type strain at 2 h postuptake. However, at 20 h postuptake, each T6SS mutant strain showed a statistically significant defect in intracellular survival compared to the wild-type strain (<a href="#F5">Fig. 5</a>). Bacterial loads per macrophage at 20 h postuptake were similar to those observed for the <i>ssaG</i> and <i>aroA</i> mutant strains.</div><div class="figure-wrap"><header><div class="label">Fig 5</div></header><figure id="F5" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/b4a4da21-0269-4847-8ccd-5e93548ee85c/assets/graphic/zii9990900630005.jpeg" /><figcaption><span class="heading">Fig 5</span> Contribution of SPI-19 T6SS to <span class="named-content" data-type="genus-species">Salmonella</span> intracellular survival. Gentamicin protection assays were performed using RAW264.7 and HD11 macrophages infected with the corresponding wild-type and mutant strains. At 2 and 20 h postuptake, cells were lysed and intracellular bacteria were determined as described in Materials and Methods. Data correspond to the averages from at least three independent assays performed in triplicate. Bars represent mean values, and error bars denote standard deviations. Asterisks indicate statistically significant differences (*, <i>P</i> &lt; 0.05; **, <i>P</i> &lt; 0.01; ***, <i>P</i> &lt; 0.001) between each mutant and the wild type as determined by one-way ANOVA with Dunnett's posttest.</figcaption></figure></div><div role="paragraph">To directly link the phenotype observed for T6SS mutant strains to T6SS functionality, the Δ<i>clpV</i> mutant was transformed with a plasmid harboring a wild-type copy of the <i>clpV</i> gene and gentamicin protection assays were performed. As shown in <a href="#F6">Fig. 6</a>, introduction of the <i>clpV</i> gene in <i>trans</i> restored the ability of the <i>S</i>. Gallinarum Δ<i>clpV</i> strain to survive within RAW264.7 and HD11 macrophages.</div><div class="figure-wrap"><header><div class="label">Fig 6</div></header><figure id="F6" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/aa166a9c-4d38-4d74-97f8-25ca3d1ea3a8/assets/graphic/zii9990900630006.jpeg" /><figcaption><span class="heading">Fig 6</span> Restoration of <i>clpV</i> gene in <i>trans</i> complements the survival defect of a Δ<i>clpV</i> mutant strain. Gentamicin protection assays were performed by infecting RAW264.7 and HD11 macrophages with the wild-type, Δ<i>clpV</i> mutant, and Δ<i>clpV</i> complemented strains. At 20 h postuptake, cells were lysed and intracellular bacteria determined as described in Materials and Methods. Data correspond to the averages from at least three independent assays performed in triplicate. Bars represent mean values, and error bars denote standard deviations. Asterisks indicate statistically significant differences (*, <i>P</i> &lt; 0.05) between the mutant and the wild type as determined by one-way ANOVA with Dunnett's posttest.</figcaption></figure></div><div role="paragraph">To determine if SPI-19 T6SS contributes to cytotoxicity and host cell death, as described for other T6SS (<a href="#B40" role="doc-biblioref" data-xml-rid="B40">40</a><a href="#B41" role="doc-biblioref" data-xml-rid="B41">–</a><a href="#B42" role="doc-biblioref" data-xml-rid="B42">42</a>), macrophages were infected with either the wild-type strain or the ΔSPI-19 mutant. Cytotoxicity was evaluated by release of lactate dehydrogenase (LDH) to the culture media, and cell viability was assayed by propidium iodide (PI) incorporation and analysis of the sub-G<sub>1</sub> population by flow cytometry. The sub-G<sub>1</sub> population is commonly used as an index of apoptotic cell death (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>). As shown in <a href="#F7">Fig. 7</a>, no differences in cytotoxicity, viability, and cell death were observed between the wild-type and mutant strains in either murine or avian macrophages. Altogether, these results clearly indicate that SPI-19 and the T6SS encoded therein contribute to <i>S</i>. Gallinarum intracellular survival in both murine and avian macrophages, and that this T6SS does not contribute to <span class="named-content" data-type="genus-species">Salmonella</span>-induced cytotoxicity and host cell death of infected macrophages.</div><div class="figure-wrap"><header><div class="label">Fig 7</div></header><figure id="F7" class="graphic"><img src="/cms/10.1128/IAI.01165-12/asset/e9a8ef06-0cbd-4cd5-90ae-b5ceb187ac46/assets/graphic/zii9990900630007.jpeg" /><figcaption><span class="heading">Fig 7</span> SPI-19 T6SS does not contribute to <span class="named-content" data-type="genus-species">Salmonella</span>-induced cell death of infected macrophages. RAW264.7 and HD11 macrophages were infected with either the wild-type or ΔSPI-19 mutant strain, and gentamicin protection assays were performed. After 20 h postuptake, cytotoxicity was evaluated by LDH release, and cell death by was determined by propidium iodide incorporation and cell cycle analysis of infected macrophages. NI, noninfected. Bars represent mean values, and error bars denote standard deviations. No statistical significant differences between the mutant and the wild type were determined by one-way ANOVA with Dunnett's posttest.</figcaption></figure></div><div role="paragraph">Finally, to assess if the VgrG protein is translocated into the cytosol of infected macrophages, a FRET-based fluorescent assay was performed using the CCF4/AM substrate. This method has been used recently to monitor VgrG translocation in <span class="named-content" data-type="genus-species">Vibrio cholerae</span> (<a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a>). A reporter gene fusion was constructed by cloning the <i>vgrG</i> ORF into plasmid pFlagTEM-1, which allows the production of β-lactamase fusion proteins under the control of an IPTG-inducible promoter (<a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>). VgrG-BlaTEM translocation into the host cytosol allows processing of the FRET fluorescent substrate CCF4/AM. RAW264.7 and HD11 macrophages were infected with <i>S</i>. Gallinarum strains harboring plasmids encoding either VgrG-BlaTEM or SipA-BlaTEM fusion proteins. The SipA-BlaTEM fusion was used as a positive control, as it corresponds to a known SPI-1 T3SS translocated effector protein (<a href="#B26" role="doc-biblioref" data-xml-rid="B26">26</a>). As a control, infection experiments were also performed using the <i>S</i>. Typhimurium 14028s strain. Translocation was assessed by both fluorometric plate reader quantification and epifluorescence microscopy. As expected, translocation of SipA was observed for <i>S</i>. Typhimurium in both macrophages by fluorometric quantification (see File S1 in the supplemental material). However, in the case of <i>S</i>. Gallinarum, we could not detect translocation of SipA or VgrG fusion proteins into either macrophage cell line (see File S1). In spite of this, epifluorescence microscopy analysis showed that a very small number of cells per field presented substrate processing in macrophages infected with the <i>S</i>. Gallinarum strains harboring pSipA/FT or pVgrG/FT (see File S1).</div></section></section><section id="sec-4" data-type="discussion"><h2>DISCUSSION</h2><div role="paragraph">The recent discovery of at least 5 different T6SSs in <span class="named-content" data-type="genus-species">Salmonella</span> has generated many questions of whether these systems contribute to <span class="named-content" data-type="genus-species">Salmonella</span> host and environmental adaptation. We have previously shown that SPI-19 T6SS contributes to efficient colonization of chicks infected with <i>S</i>. Gallinarum, suggesting a role for this T6SS in the interaction of <span class="named-content" data-type="genus-species">Salmonella</span> with cells of the reticuloendothelial system (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>). In this study, we characterized SPI-19 T6SS by analyzing the expression, production, and translocation of selected components/effectors and by defining its role in the interaction of <i>S</i>. Gallinarum with murine and avian macrophages.</div><div role="paragraph">Expression analysis of five SPI-19 T6SS core components revealed a growth phase-dependent regulation, with an increase in their expression when the bacteria reached the onset of stationary phase (<a href="#F2">Fig. 2A</a>). Interestingly, analysis of the production of the structural/secreted proteins Hcp1, Hcp2, and VgrG showed an inverse relationship regarding growth phase regulation, with increased protein levels detected at exponential phase (<a href="#F2">Fig. 2B</a>). This discrepancy between gene expression and protein production suggests that these components are under posttranscriptional regulatory mechanisms, as described for other T6SS (<a href="#B44" role="doc-biblioref" data-xml-rid="B44">44</a>, <a href="#B45" role="doc-biblioref" data-xml-rid="B45">45</a>). Growth phase-dependent expression and production of T6SS components was not unexpected. Core components of the T6SSs of <span class="named-content" data-type="genus-species">Yersinia pestis</span> and <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> have been shown to be regulated by the bacterial phase of growth (<a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>), and T6SSs of <span class="named-content" data-type="genus-species">Aeromonas hydrophila</span>, <span class="named-content" data-type="genus-species">Vibrio cholerae</span>, and <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> have been linked to quorum-sensing regulatory mechanisms (<a href="#B48" role="doc-biblioref" data-xml-rid="B48">48</a><a href="#B49" role="doc-biblioref" data-xml-rid="B49">–</a><a href="#B50" role="doc-biblioref" data-xml-rid="B50">50</a>).</div><div role="paragraph">Analysis of culture supernatants failed to demonstrate secretion of either Hcp1, Hcp2, or VgrG, even under the growth conditions that favored their production (<a href="#F2">Fig. 2B</a>). In this context, it is possible that secretion of the SPI-19 T6SS is regulated by a posttranslational mechanism responding to an unidentified environmental/host signal. In agreement with this notion, it has been reported that T6SS activation in <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> requires a serine-threonine kinase and a phosphatase that regulate the phosphorylation status of a forkhead-associated (FHA) inner membrane protein (<a href="#B51" role="doc-biblioref" data-xml-rid="B51">51</a>). Even though the SPI-19 T6SS locus does not encode putative serine-threonine kinase and a phosphatase, it does encode a putative FHA domain-containing protein (SG1038) (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>). Therefore, it is tempting to speculate that activation of the SPI-19 T6SS depends on a similar posttranslational regulatory mechanism.</div><div role="paragraph">In order to dissect the putative roles of the SPI-19 T6SS, we tested its contribution to interaction with bacterial cells and macrophages. Interbacterial killing assays failed to demonstrate antibacterial activity of SPI-19 T6SS under different growth conditions (<a href="#F3">Fig. 3</a>). Nevertheless, it is important to mention that this assay is not comprehensive, as other unidentified Gram-negative bacteria can be the target of SPI-19 T6SS, and other inducing signals might be important for activating SPI-19 T6SS-dependent killing. Furthermore, the presence of a toxin/antitoxin module (<a href="#F1">Fig. 1B</a>) suggests that the SPI-19 T6SS could indeed be involved in interbacterial killing.</div><div role="paragraph">Cell culture-based assays revealed that the VgrG protein was produced after <span class="named-content" data-type="genus-species">Salmonella</span> internalization by cultured murine and avian macrophages (<a href="#F4">Fig. 4</a>). Preferential production of VgrG upon phagocytosis was not unexpected, because expression of T6SS components within macrophages has been described for other T6SS, including those encoded in the <i>tss5</i> locus of <span class="named-content" data-type="genus-species">Burkholderia pseudomallei</span>, the FPI island of <span class="named-content" data-type="genus-species">Francisella tularensis</span>, the T6SS1 locus of <span class="named-content" data-type="genus-species">Burkholderia mallei</span>, and the <i>vas</i> cluster of <span class="named-content" data-type="genus-species">Vibrio cholerae</span>, all of which are upregulated upon phagocytosis (reviewed in reference <a href="#B36" role="doc-biblioref" data-xml-rid="B36">36</a>).</div><div role="paragraph">When intracellular production of the VgrG-GFP fusion protein was determined, GFP fluorescence was observed in the cytoplasm of infected macrophages, suggesting that the VgrG protein could be translocated into the host cell cytosol. To test this, a VgrG-BlaTEM fusion was constructed, and CCF4/AM processing was evaluated in murine and avian macrophages. Even though visual inspection through epifluorescence microscopy revealed translocation in a small percentage of the total cell population, fluorometric plate reader quantification did not determine a significant increase in the fluorescent substrate processing. Taking into account that <i>S</i>. Gallinarum is internalized significantly less efficiently by macrophages than <i>S</i>. Typhimurium, and that translocation of the positive-control SipA was observed by epifluorescence microscopy but not detected by fluorescence quantification in <i>S</i>. Gallinarum, it is plausible to think that VgrG is indeed translocated under these conditions. Nevertheless, further experiments will be needed to clarify this issue, including the construction of different reporter fusions attaching to the VgrG protein.</div><div role="paragraph">Unexpectedly, fluorescence microscopy analysis showed that the VgrG-GFP protein was detected throughout the bacterial cell and not in punctate foci in the bacterial membrane, as seen for other T6SS structural components (<a href="#B52" role="doc-biblioref" data-xml-rid="B52">52</a>). One possible explanation is that the GFP reporter affects the normal function of the VgrG protein, including its subcellular localization. To test this hypothesis, we performed gentamicin protection assays in murine and avian macrophages infected with either the wild-type or VgrG-GFP strain. Results from these assays suggest that the fusion protein is partially impaired in T6SS function (see File S2 in the supplemental material).</div><div role="paragraph">The SPI-19 T6SS expression data are reminiscent of what has been described for the <span class="named-content" data-type="genus-species">Salmonella</span> SPI-2 T3SS. The genes encoding structural components of the SPI-2 T3SS are upregulated upon phagocytosis (<a href="#B53" role="doc-biblioref" data-xml-rid="B53">53</a>, <a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>), including the <i>ssaG</i> gene, which encodes a translocon component of the needle complex. Expression of the <i>ssaG</i> gene responds to the DNA supercoiling status of the bacterial cell chromosome, as it adapts to the intracellular environment. This activation depends on the direct binding of the Fis protein to the <i>ssaG</i> promoter region (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>). Several Fis-regulated genes show a growth phase-dependent gene expression (<a href="#B56" role="doc-biblioref" data-xml-rid="B56">56</a>, <a href="#B57" role="doc-biblioref" data-xml-rid="B57">57</a>) due to enhanced levels of the Fis protein at exponential phase. Therefore, the increased levels of Hcp2 and VgrG proteins in the exponential phase of growth could be due to increased levels of Fis. In support of this notion, we have identified the presence of several Fis binding sites in the promoter regions of <i>hcp2</i> and <i>vgrG</i> (see File S3 in the supplemental material). Further work will be needed to clarify this hypothesis and to identify the global regulators that seem to be regulating the expression, production, and function of SPI-19 T6SS in <span class="named-content" data-type="genus-species">Salmonella</span> under <i>in vitro</i> growth and cell culture conditions.</div><div role="paragraph">Intracellular production of VgrG indicated a possible role of the SPI-19 T6SS in the interaction of <i>S</i>. Gallinarum with macrophages. Gentamicin protection assays revealed that SPI-9 T6SS contributes to <span class="named-content" data-type="genus-species">Salmonella</span> intracellular survival in both murine and avian macrophages. Upon phagocytosis, <span class="named-content" data-type="genus-species">Salmonella</span> is engulfed in a compartment known as the <span class="named-content" data-type="genus-species">Salmonella</span>-containing vacuole (SCV), and through the orchestrated action of SPI-1 and SPI-2 T3SS-secreted effector proteins, the bacterium ensures its survival by avoiding phagosome maturation (<a href="#B58" role="doc-biblioref" data-xml-rid="B58">58</a>). SCV biogenesis is usually divided in early (&lt;30 min), intermediate (30 min to 5 h), and late stages (&gt;5 h), with a defined set of T3SS effectors involved in each stage (<a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a>, <a href="#B60" role="doc-biblioref" data-xml-rid="B60">60</a>). By altering phagosome biogenesis, <span class="named-content" data-type="genus-species">Salmonella</span> is able to reside in mature SCVs that localize in a perinuclear region associated with the Golgi network late after infection (<a href="#B60" role="doc-biblioref" data-xml-rid="B60">60</a>, <a href="#B61" role="doc-biblioref" data-xml-rid="B61">61</a>).</div><div role="paragraph">In <i>S</i>. Gallinarum, the SPI-2 T3SS has been shown to be important for the ability of the bacteria to survive within murine and avian macrophages from 1 h postuptake (<a href="#B14" role="doc-biblioref" data-xml-rid="B14">14</a>, <a href="#B15" role="doc-biblioref" data-xml-rid="B15">15</a>). In agreement with this, we observed that the SPI-2 T3SS Δ<i>ssaG</i> mutant strain showed a significant defect on intracellular survival at 2 h postuptake. In contrast, the ΔSPI-19 and Δ<i>clpV</i> mutants showed a significant defect only at 20 h postuptake. These observations suggest that either (i) the T3SS and T6SS contribute to <span class="named-content" data-type="genus-species">Salmonella</span> survival by modulating different stages of SCV biogenesis, or (ii) the T6SS promotes replication of <span class="named-content" data-type="genus-species">Salmonella</span> once it has escaped from the phagolysosomal fate, as has been shown for other virulence-related loci, including the PhoP/Q regulon (<a href="#B62" role="doc-biblioref" data-xml-rid="B62">62</a>). Further experiments regarding the kinetics of phagolysosomal fusion and replication rates of T6SS mutant strains will be needed to clarify these issues.</div><div role="paragraph">Despite the role of SPI-19 in intracellular survival, there was no significant impact on host cell death and cytotoxicity of infected macrophages <i>in vitro</i>, as seen for other T6SSs that are involved in the interaction of the bacteria with cells of the immune system (<a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>). Notwithstanding this, it is plausible to think that the SPI-19 T6SS could influence other processes triggered after infection of macrophages, including the production and secretion of proinflammatory cytokines and chemokines. In this context, it has been shown that immune responses generated by infection with <i>S</i>. Gallinarum are critical for the virulence and pathogenicity of this serotype both <i>in vitro</i> and <i>in vivo</i> (<a href="#B11" role="doc-biblioref" data-xml-rid="B11">11</a>, <a href="#B12" role="doc-biblioref" data-xml-rid="B12">12</a>, <a href="#B63" role="doc-biblioref" data-xml-rid="B63">63</a>, <a href="#B64" role="doc-biblioref" data-xml-rid="B64">64</a>). Therefore, contribution of the SPI-19 T6SS to <i>in vivo</i> colonization of infected chicks is mediated by these processes in addition to intracellular survival within macrophages. Future work will be needed to determine these issues.</div><div role="paragraph">In agreement with our findings, other authors have pointed out a role for T6SSs in the interaction of <span class="named-content" data-type="genus-species">Salmonella</span> with host cells. Libby and colleagues (<a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>) identified several genes linked to the SPI-6 T6SS of <i>S</i>. Typhi which are required for systemic infection in a humanized mouse model. In addition, Wang and colleagues (<a href="#B66" role="doc-biblioref" data-xml-rid="B66">66</a>) recently reported that a mutant in an essential SPI-6 T6SS core component enhanced the ability of <i>S</i>. Typhi to invade epithelial cells and reduced <span class="named-content" data-type="genus-species">Salmonella</span>-induced cytotoxicity toward these cells. More recently, Liu and colleagues (<a href="#B67" role="doc-biblioref" data-xml-rid="B67">67</a>) have shown that lack of SPI-6 T6SS in <i>S</i>. Typhimurium leads to reduced <span class="named-content" data-type="genus-species">Salmonella</span> survival and proliferation within infected murine macrophages and a defect in mice colonization. Finally, Mulder and colleagues (<a href="#B68" role="doc-biblioref" data-xml-rid="B68">68</a>) reported that several SPI-6 T6SS core components and SPI-6 T6SS-associated gene clusters contribute to <i>S</i>. Typhimurium intracellular survival within murine macrophages and systemic dissemination in mice.</div><div role="paragraph">Since the evidence described above shows that both SPI-6 and SPI-19 T6SSs are involved in the ability of <span class="named-content" data-type="genus-species">Salmonella</span> to survive within macrophages, it is tempting to speculate that despite having distinct phylogenetic origins, these two T6SSs are functionally redundant.</div><div role="paragraph">In the context of <span class="named-content" data-type="genus-species">Salmonella</span> evolution, the functional implications of this potential redundancy are intriguing. Comparative genomic analysis of T6SSs in <span class="named-content" data-type="genus-species">Salmonella</span> have suggested that a common ancestor of SPI-19-containing serotypes harbored both SPI-6 and SPI-19 T6SSs in their genomes. During evolution, some serotypes maintained both T6SSs (e.g., Dublin and Weltevreden), others have lost SPI-6 T6SS while maintaining SPI-19 T6SS (e.g., Gallinarum and Agona), and still others lost both T6SS loci (e.g., Enteritidis) (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>). Therefore, it would be of interest to determine if these systems are indeed redundant or if there are subtle differences in their function that could explain why some <span class="named-content" data-type="genus-species">Salmonella</span> serotypes retained one of these systems over the other, and if they differentially contribute to the adaptation of <span class="named-content" data-type="genus-species">Salmonella</span> serotypes to different hosts and environments.</div><div role="paragraph">In conclusion, in this work we have demonstrated that SPI-19 T6SS contributes to <span class="named-content" data-type="genus-species">Salmonella</span> intracellular survival and that the VgrG protein is preferentially produced once the bacteria have been internalized by the cells. Altogether, our work expands the known set of virulence factors employed by <span class="named-content" data-type="genus-species">Salmonella</span> to subvert the immune system of its hosts.</div></section><section id="acknowledgments" role="doc-acknowledgments"><h2>ACKNOWLEDGMENTS</h2><div role="paragraph">We are indebted to R. Tsolis for the generous gift of plasmids pFlagTEM-1 and pSipA/FT. We thank C. Marolda for the generous gift of plasmid pmCherry.</div><div data-type="financial-disclosure" role="paragraph">This work was supported by FONDECYT grant 1100092. C.J.B. is supported by FONDECYT Postdoctoral Fellowship 3120175. C.A.S. is supported by FONDECYT grant 1110172. D.P. is supported by fellowships from Fulbright, Beca Doctorado Nacional 2009 CONICYT (no. 21090041), Beca de Apoyo a la Realización de Tesis Doctoral 2012 CONICYT (no. AT-21121297), and Beca de Pasantías Doctorales en el Extranjero 2011 (no. 75110062 BCH-3).</div></section></div></section><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="supplementary-materials" class="core-supplementary-materials"><h2>Supplemental Material</h2><div role="list"><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(zii999090063so1.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/IAI.01165-12/suppl_file/zii999090063so1.pdf" download="zii999090063so1.pdf">Download</a></div></div></div><footer><div role="paragraph">ASM does not own the copyrights to Supplemental Material that may be linked to, or accessed through, an article. The authors have granted ASM a non-exclusive, world-wide license to publish the Supplemental Material files. Please contact the corresponding author directly for reuse.</div></footer></section><section id="bibliography" role="doc-bibliography"><h2>REFERENCES</h2><div class="labeled" role="doc-biblioentry"><div class="label">1.</div><div id="B1" class="citations"><div class="citation"><div class="citation-content">Grimont PD and Weill FX. 2007. <em>Antigenic formulae of the Salmonella serovars</em>, 9th ed. WHO Collaborating Centre for Reference and Research on <span class="named-content" data-type="genus-species">Salmonella</span>, Institute Pasteur, Paris, France.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Antigenic+formulae+of+the+Salmonella+serovars&amp;author=PD+Grimont&amp;author=FX+Weill&amp;publication_year=2007">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">2.</div><div id="B2" class="citations"><div class="citation"><div class="citation-content">Shivaprasad HL. 2000. Fowl typhoid and pullorum disease. <em>Rev. Sci. Tech.</em> 19:405–424.</div><div class="external-links"><div><a href="https://doi.org/10.20506/rst.19.2.1222">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10935271">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000088194900006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Fowl+typhoid+and+pullorum+disease&amp;author=HL+Shivaprasad&amp;publication_year=2000&amp;pages=405-424&amp;doi=10.20506%2Frst.19.2.1222&amp;pmid=10935271">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">3.</div><div id="B3" class="citations"><div class="citation"><div class="citation-content">Barrow PA and Freitas Neto OC. 2011. Pullorum disease and fowl typhoid–new thoughts on old diseases: a review. <em>Avian Pathol.</em> 40:1–13.</div><div class="external-links"><div><a href="https://doi.org/10.1080/03079457.2010.542575">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21331943">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000287413700001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Pullorum+disease+and+fowl+typhoid%E2%80%93new+thoughts+on+old+diseases%3A+a+review&amp;author=PA+Barrow&amp;author=OC+Freitas+Neto&amp;publication_year=2011&amp;pages=1-13&amp;doi=10.1080%2F03079457.2010.542575&amp;pmid=21331943">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">4.</div><div id="B4" class="citations"><div class="citation"><div class="citation-content">Haraga A, Ohlson MB, and Miller SI. 2008. Salmonellae interplay with host cells. <em>Nat. Rev. Microbiol.</em> 6:53–66.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nrmicro1788">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18026123">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000252065900014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonellae+interplay+with+host+cells&amp;author=A+Haraga&amp;author=MB+Ohlson&amp;author=SI+Miller&amp;publication_year=2008&amp;pages=53-66&amp;doi=10.1038%2Fnrmicro1788&amp;pmid=18026123">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">5.</div><div id="B5" class="citations"><div class="citation"><div class="citation-content">Boyer F, Fichant G, Berthod J, Vandenbrouck Y, and Attree I. 2009. Dissecting the bacterial type VI secretion system by a genome wide <i>in silico</i> analysis: what can be learnt from available microbial genomic resources? <em>BMC Genomics</em> 10:104. doi:<a href="http://dx.doi.org/10.1186/1471-2164-10-104">10.1186/1471-2164-10-104</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1186/1471-2164-10-104">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19284603">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000264983700001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Dissecting+the+bacterial+type+VI+secretion+system+by+a+genome+wide+in+silico+analysis%3A+what+can+be+learnt+from+available+microbial+genomic+resources%3F&amp;author=F+Boyer&amp;author=G+Fichant&amp;author=J+Berthod&amp;author=Y+Vandenbrouck&amp;author=I+Attree&amp;publication_year=2009&amp;pages=104&amp;doi=10.1186%2F1471-2164-10-104&amp;pmid=19284603">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">6.</div><div id="B6" class="citations"><div class="citation"><div class="citation-content">Pukatzki S, McAuley SB, and Miyata ST. 2009. The type VI secretion system: translocation of effectors and effector-domains. <em>Curr. Opin. Microbiol.</em> 12:11–17.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2008.11.010">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19162533">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000264319100003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+type+VI+secretion+system%3A+translocation+of+effectors+and+effector-domains&amp;author=S+Pukatzki&amp;author=SB+McAuley&amp;author=ST+Miyata&amp;publication_year=2009&amp;pages=11-17&amp;doi=10.1016%2Fj.mib.2008.11.010&amp;pmid=19162533">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">7.</div><div id="B7" class="citations"><div class="citation"><div class="citation-content">Blondel CJ, Jimenez JC, Contreras I, and Santiviago CA. 2009. Comparative genomic analysis uncovers 3 novel loci encoding type six secretion systems differentially distributed in <span class="named-content" data-type="genus-species">Salmonella</span> serotypes. <em>BMC Genomics</em> 10:354. doi:<a href="http://dx.doi.org/10.1186/1471-2164-10-354">10.1186/1471-2164-10-354</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1186/1471-2164-10-354">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19653904">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269510300002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Comparative+genomic+analysis+uncovers+3+novel+loci+encoding+type+six+secretion+systems+differentially+distributed+in+Salmonella+serotypes&amp;author=CJ+Blondel&amp;author=JC+Jimenez&amp;author=I+Contreras&amp;author=CA+Santiviago&amp;publication_year=2009&amp;pages=354&amp;doi=10.1186%2F1471-2164-10-354&amp;pmid=19653904">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">8.</div><div id="B8" class="citations"><div class="citation"><div class="citation-content">Fookes M, Schroeder GN, Langridge GC, Blondel CJ, Mammina C, Connor TR, Seth-Smith H, Vernikos GS, Robinson KS, Sanders M, Petty NK, Kingsley RA, Baumler AJ, Nuccio SP, Contreras I, Santiviago CA, Maskell D, Barrow P, Humphrey T, Nastasi A, Roberts M, Frankel G, Parkhill J, Dougan G, and Thomson NR. 2011. <span class="named-content" data-type="genus-species">Salmonella bongori</span> provides insights into the evolution of the salmonellae. <em>PLoS Pathog.</em> 7:e1002191. doi:<a href="http://dx.doi.org/10.1371/journal.ppat.1002191">10.1371/journal.ppat.1002191</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1002191">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21876672">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000294298100027">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonella+bongori+provides+insights+into+the+evolution+of+the+salmonellae&amp;author=M+Fookes&amp;author=GN+Schroeder&amp;author=GC+Langridge&amp;author=CJ+Blondel&amp;author=C+Mammina&amp;author=TR+Connor&amp;author=H+Seth-Smith&amp;author=GS+Vernikos&amp;author=KS+Robinson&amp;author=M+Sanders&amp;author=NK+Petty&amp;author=RA+Kingsley&amp;publication_year=2011&amp;pages=e1002191&amp;doi=10.1371%2Fjournal.ppat.1002191&amp;pmid=21876672">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">9.</div><div id="B9" class="citations"><div class="citation"><div class="citation-content">Blondel CJ, Yang HJ, Castro B, Chiang S, Toro CS, Zaldivar M, Contreras I, Andrews-Polymenis HL, and Santiviago CA. 2010. Contribution of the type VI secretion system encoded in SPI-19 to chicken colonization by <span class="named-content" data-type="genus-species">Salmonella enterica</span> serotypes Gallinarum and Enteritidis. <em>PLoS One</em> 5:e11724. doi:<a href="http://dx.doi.org/10.1371/journal.pone.0011724">10.1371/journal.pone.0011724</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.pone.0011724">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20661437">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000280243300024">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Contribution+of+the+type+VI+secretion+system+encoded+in+SPI-19+to+chicken+colonization+by+Salmonella+enterica+serotypes+Gallinarum+and+Enteritidis&amp;author=CJ+Blondel&amp;author=HJ+Yang&amp;author=B+Castro&amp;author=S+Chiang&amp;author=CS+Toro&amp;author=M+Zaldivar&amp;author=I+Contreras&amp;author=HL+Andrews-Polymenis&amp;author=CA+Santiviago&amp;publication_year=2010&amp;pages=e11724&amp;doi=10.1371%2Fjournal.pone.0011724&amp;pmid=20661437">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">10.</div><div id="B10" class="citations"><div class="citation"><div class="citation-content">Thomson NR, Clayton DJ, Windhorst D, Vernikos G, Davidson S, Churcher C, Quail MA, Stevens M, Jones MA, Watson M, Barron A, Layton A, Pickard D, Kingsley RA, Bignell A, Clark L, Harris B, Ormond D, Abdellah Z, Brooks K, Cherevach I, Chillingworth T, Woodward J, Norberczak H, Lord A, Arrowsmith C, Jagels K, Moule S, Mungall K, Sanders M, Whitehead S, Chabalgoity JA, Maskell D, Humphrey T, Roberts M, Barrow PA, Dougan G, and Parkhill J. 2008. Comparative genome analysis of <span class="named-content" data-type="genus-species">Salmonella</span> Enteritidis PT4 and <span class="named-content" data-type="genus-species">Salmonella</span> Gallinarum 287/91 provides insights into evolutionary and host adaptation pathways. <em>Genome Res.</em> 18:1624–1637.</div><div class="external-links"><div><a href="https://doi.org/10.1101/gr.077404.108">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18583645">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000259700800009">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Comparative+genome+analysis+of+Salmonella+Enteritidis+PT4+and+Salmonella+Gallinarum+287%2F91+provides+insights+into+evolutionary+and+host+adaptation+pathways&amp;author=NR+Thomson&amp;author=DJ+Clayton&amp;author=D+Windhorst&amp;author=G+Vernikos&amp;author=S+Davidson&amp;author=C+Churcher&amp;author=MA+Quail&amp;author=M+Stevens&amp;author=MA+Jones&amp;author=M+Watson&amp;author=A+Barron&amp;author=A+Layton&amp;publication_year=2008&amp;pages=1624-1637&amp;doi=10.1101%2Fgr.077404.108&amp;pmid=18583645">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">11.</div><div id="B11" class="citations"><div class="citation"><div class="citation-content">Barrow PA, Huggins MB, and Lovell MA. 1994. Host specificity of <span class="named-content" data-type="genus-species">Salmonella</span> infection in chickens and mice is expressed <i>in vivo</i> primarily at the level of the reticuloendothelial system. <em>Infect. Immun.</em> 62:4602–4610.</div><div class="external-links"><div><a href="https://doi.org/10.1128/iai.62.10.4602-4610.1994">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7927727">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1994PH29800068">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Host+specificity+of+Salmonella+infection+in+chickens+and+mice+is+expressed+in+vivo+primarily+at+the+level+of+the+reticuloendothelial+system&amp;author=PA+Barrow&amp;author=MB+Huggins&amp;author=MA+Lovell&amp;publication_year=1994&amp;pages=4602-4610&amp;doi=10.1128%2Fiai.62.10.4602-4610.1994&amp;pmid=7927727">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">12.</div><div id="B12" class="citations"><div class="citation"><div class="citation-content">Chappell L, Kaiser P, Barrow P, Jones MA, Johnston C, and Wigley P. 2009. The immunobiology of avian systemic salmonellosis. <em>Vet. Immunol. Immunopathol.</em> 128:53–59.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.vetimm.2008.10.295">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19070366">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000264504600008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+immunobiology+of+avian+systemic+salmonellosis&amp;author=L+Chappell&amp;author=P+Kaiser&amp;author=P+Barrow&amp;author=MA+Jones&amp;author=C+Johnston&amp;author=P+Wigley&amp;publication_year=2009&amp;pages=53-59&amp;doi=10.1016%2Fj.vetimm.2008.10.295&amp;pmid=19070366">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">13.</div><div id="B13" class="citations"><div class="citation"><div class="citation-content">Shah DH, Lee MJ, Park JH, Lee JH, Eo SK, Kwon JT, and Chae JS. 2005. Identification of <span class="named-content" data-type="genus-species">Salmonella gallinarum</span> virulence genes in a chicken infection model using PCR-based signature-tagged mutagenesis. <em>Microbiology</em> 151:3957–3968.</div><div class="external-links"><div><a href="https://doi.org/10.1099/mic.0.28126-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16339940">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000234148100014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+Salmonella+gallinarum+virulence+genes+in+a+chicken+infection+model+using+PCR-based+signature-tagged+mutagenesis&amp;author=DH+Shah&amp;author=MJ+Lee&amp;author=JH+Park&amp;author=JH+Lee&amp;author=SK+Eo&amp;author=JT+Kwon&amp;author=JS+Chae&amp;publication_year=2005&amp;pages=3957-3968&amp;doi=10.1099%2Fmic.0.28126-0&amp;pmid=16339940">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">14.</div><div id="B14" class="citations"><div class="citation"><div class="citation-content">Jones MA, Wigley P, Page KL, Hulme SD, and Barrow PA. 2001. <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Gallinarum requires the <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 2 type III secretion system but not the <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 1 type III secretion system for virulence in chickens. <em>Infect. Immun.</em> 69:5471–5476.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.69.9.5471-5476.2001">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11500419">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000170540000032">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonella+enterica+serovar+Gallinarum+requires+the+Salmonella+pathogenicity+island+2+type+III+secretion+system+but+not+the+Salmonella+pathogenicity+island+1+type+III+secretion+system+for+virulence+in+chickens&amp;author=MA+Jones&amp;author=P+Wigley&amp;author=KL+Page&amp;author=SD+Hulme&amp;author=PA+Barrow&amp;publication_year=2001&amp;pages=5471-5476&amp;doi=10.1128%2FIAI.69.9.5471-5476.2001&amp;pmid=11500419">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">15.</div><div id="B15" class="citations"><div class="citation"><div class="citation-content">Jeong JH, Song M, Park SI, Cho KO, Rhee JH, and Choy HE. 2008. <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Gallinarum requires ppGpp for internalization and survival in animal cells. <em>J. Bacteriol.</em> 190:6340–6350.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.00385-08">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18621899">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000259212300006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonella+enterica+serovar+Gallinarum+requires+ppGpp+for+internalization+and+survival+in+animal+cells&amp;author=JH+Jeong&amp;author=M+Song&amp;author=SI+Park&amp;author=KO+Cho&amp;author=JH+Rhee&amp;author=HE+Choy&amp;publication_year=2008&amp;pages=6340-6350&amp;doi=10.1128%2FJB.00385-08&amp;pmid=18621899">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">16.</div><div id="B16" class="citations"><div class="citation"><div class="citation-content">Mizusaki H, Takaya A, Yamamoto T, and Aizawa S. 2008. Signal pathway in salt-activated expression of the <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 1 type III secretion system in <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium. <em>J. Bacteriol.</em> 190:4624–4631.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.01957-07">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18441068">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000257174400028">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Signal+pathway+in+salt-activated+expression+of+the+Salmonella+pathogenicity+island+1+type+III+secretion+system+in+Salmonella+enterica+serovar+Typhimurium&amp;author=H+Mizusaki&amp;author=A+Takaya&amp;author=T+Yamamoto&amp;author=S+Aizawa&amp;publication_year=2008&amp;pages=4624-4631&amp;doi=10.1128%2FJB.01957-07&amp;pmid=18441068">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">17.</div><div id="B17" class="citations"><div class="citation"><div class="citation-content">Groisman EA, Kayser J, and Soncini FC. 1997. Regulation of polymyxin resistance and adaptation to low-Mg2+ environments. <em>J. Bacteriol.</em> 179:7040–7045.</div><div class="external-links"><div><a href="https://doi.org/10.1128/jb.179.22.7040-7045.1997">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9371451">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1997YG07500023">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Regulation+of+polymyxin+resistance+and+adaptation+to+low-Mg2%2B+environments&amp;author=EA+Groisman&amp;author=J+Kayser&amp;author=FC+Soncini&amp;publication_year=1997&amp;pages=7040-7045&amp;doi=10.1128%2Fjb.179.22.7040-7045.1997&amp;pmid=9371451">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">18.</div><div id="B18" class="citations"><div class="citation"><div class="citation-content">Datsenko KA and Wanner BL. 2000. One-step inactivation of chromosomal genes in <span class="named-content" data-type="genus-species">Escherichia coli</span> K-12 using PCR products. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 97:6640–6645.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.120163297">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000087526300074">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products&amp;author=KA+Datsenko&amp;author=BL+Wanner&amp;publication_year=2000&amp;pages=6640-6645&amp;doi=10.1073%2Fpnas.120163297&amp;pmid=10829079">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">19.</div><div id="B19" class="citations"><div class="citation"><div class="citation-content">Bravo D, Blondel CJ, Hoare A, Leyton L, Valvano MA, and Contreras I. 2011. Type IV(B) pili are required for invasion but not for adhesion of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhi into BHK epithelial cells in a cystic fibrosis transmembrane conductance regulator-independent manner. <em>Microb. Pathog.</em> 51:373–377.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.micpath.2011.07.005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21782926">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000294985900010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+IV%28B%29+pili+are+required+for+invasion+but+not+for+adhesion+of+Salmonella+enterica+serovar+Typhi+into+BHK+epithelial+cells+in+a+cystic+fibrosis+transmembrane+conductance+regulator-independent+manner&amp;author=D+Bravo&amp;author=CJ+Blondel&amp;author=A+Hoare&amp;author=L+Leyton&amp;author=MA+Valvano&amp;author=I+Contreras&amp;publication_year=2011&amp;pages=373-377&amp;doi=10.1016%2Fj.micpath.2011.07.005&amp;pmid=21782926">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">20.</div><div id="B20" class="citations"><div class="citation"><div class="citation-content">Ellermeier CD, Janakiraman A, and Slauch JM. 2002. Construction of targeted single copy <i>lac</i> fusions using lambda Red and FLP-mediated site-specific recombination in bacteria. <em>Gene</em> 290:153–161.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0378-1119(02)00551-6">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12062810">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000176479500014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Construction+of+targeted+single+copy+lac+fusions+using+lambda+Red+and+FLP-mediated+site-specific+recombination+in+bacteria&amp;author=CD+Ellermeier&amp;author=A+Janakiraman&amp;author=JM+Slauch&amp;publication_year=2002&amp;pages=153-161&amp;doi=10.1016%2FS0378-1119%2802%2900551-6&amp;pmid=12062810">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">21.</div><div id="B21" class="citations"><div class="citation"><div class="citation-content">Uzzau S, Figueroa-Bossi N, Rubino S, and Bossi L. 2001. Epitope tagging of chromosomal genes in <span class="named-content" data-type="genus-species">Salmonella</span>. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 98:15264–15269.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.261348198">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11742086">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000172848800094">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Epitope+tagging+of+chromosomal+genes+in+Salmonella&amp;author=S+Uzzau&amp;author=N+Figueroa-Bossi&amp;author=S+Rubino&amp;author=L+Bossi&amp;publication_year=2001&amp;pages=15264-15269&amp;doi=10.1073%2Fpnas.261348198&amp;pmid=11742086">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">22.</div><div id="B22" class="citations"><div class="citation"><div class="citation-content">Gerlach RG, Holzer SU, Jackel D, and Hensel M. 2007. Rapid engineering of bacterial reporter gene fusions by using Red recombination. <em>Appl. Environ. Microbiol.</em> 73:4234–4242.</div><div class="external-links"><div><a href="https://doi.org/10.1128/AEM.00509-07">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17513596">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000248070000019">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rapid+engineering+of+bacterial+reporter+gene+fusions+by+using+Red+recombination&amp;author=RG+Gerlach&amp;author=SU+Holzer&amp;author=D+Jackel&amp;author=M+Hensel&amp;publication_year=2007&amp;pages=4234-4242&amp;doi=10.1128%2FAEM.00509-07&amp;pmid=17513596">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">23.</div><div id="B23" class="citations"><div class="citation"><div class="citation-content">Ho TD, Figueroa-Bossi N, Wang M, Uzzau S, Bossi L, and Slauch JM. 2002. Identification of GtgE, a novel virulence factor encoded on the Gifsy-2 bacteriophage of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium. <em>J. Bacteriol.</em> 184:5234–5239.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.184.19.5234-5239.2002">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12218008">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000177967000004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+GtgE%2C+a+novel+virulence+factor+encoded+on+the+Gifsy-2+bacteriophage+of+Salmonella+enterica+serovar+Typhimurium&amp;author=TD+Ho&amp;author=N+Figueroa-Bossi&amp;author=M+Wang&amp;author=S+Uzzau&amp;author=L+Bossi&amp;author=JM+Slauch&amp;publication_year=2002&amp;pages=5234-5239&amp;doi=10.1128%2FJB.184.19.5234-5239.2002&amp;pmid=12218008">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">24.</div><div id="B24" class="citations"><div class="citation"><div class="citation-content">Schwan WR, Huang XZ, Hu L, and Kopecko DJ. 2000. Differential bacterial survival, replication, and apoptosis-inducing ability of <span class="named-content" data-type="genus-species">Salmonella</span> serovars within human and murine macrophages. <em>Infect. Immun.</em> 68:1005–1013.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.68.3.1005-1013.2000">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10678900">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000085407400001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Differential+bacterial+survival%2C+replication%2C+and+apoptosis-inducing+ability+of+Salmonella+serovars+within+human+and+murine+macrophages&amp;author=WR+Schwan&amp;author=XZ+Huang&amp;author=L+Hu&amp;author=DJ+Kopecko&amp;publication_year=2000&amp;pages=1005-1013&amp;doi=10.1128%2FIAI.68.3.1005-1013.2000&amp;pmid=10678900">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">25.</div><div id="B25" class="citations"><div class="citation"><div class="citation-content">Sun YH, Rolan HG, and Tsolis RM. 2007. Injection of flagellin into the host cell cytosol by <span class="named-content" data-type="genus-species">Salmonella enterica</span> serotype Typhimurium. <em>J. Biol. Chem.</em> 282:33897–33901.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.C700181200">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17911114">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000251145700001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Injection+of+flagellin+into+the+host+cell+cytosol+by+Salmonella+enterica+serotype+Typhimurium&amp;author=YH+Sun&amp;author=HG+Rolan&amp;author=RM+Tsolis&amp;publication_year=2007&amp;pages=33897-33901&amp;doi=10.1074%2Fjbc.C700181200&amp;pmid=17911114">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">26.</div><div id="B26" class="citations"><div class="citation"><div class="citation-content">Raffatellu M, Sun YH, Wilson RP, Tran QT, Chessa D, Andrews-Polymenis HL, Lawhon SD, Figueiredo JF, Tsolis RM, Adams LG, and Baumler AJ. 2005. Host restriction of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serotype Typhi is not caused by functional alteration of SipA, SopB, or SopD. <em>Infect. Immun.</em> 73:7817–7826.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.73.12.7817-7826.2005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16299271">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000233480200004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Host+restriction+of+Salmonella+enterica+serotype+Typhi+is+not+caused+by+functional+alteration+of+SipA%2C+SopB%2C+or+SopD&amp;author=M+Raffatellu&amp;author=YH+Sun&amp;author=RP+Wilson&amp;author=QT+Tran&amp;author=D+Chessa&amp;author=HL+Andrews-Polymenis&amp;author=SD+Lawhon&amp;author=JF+Figueiredo&amp;author=RM+Tsolis&amp;author=LG+Adams&amp;author=AJ+Baumler&amp;publication_year=2005&amp;pages=7817-7826&amp;doi=10.1128%2FIAI.73.12.7817-7826.2005&amp;pmid=16299271">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">27.</div><div id="B27" class="citations"><div class="citation"><div class="citation-content">Hebrard M, Kroger C, Sivasankaran SK, Handler K, and Hinton JC. 2011. The challenge of relating gene expression to the virulence of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium. <em>Curr. Opin. Biotechnol.</em> 22:200–210.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.copbio.2011.02.007">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21388802">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000290505300011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+challenge+of+relating+gene+expression+to+the+virulence+of+Salmonella+enterica+serovar+Typhimurium&amp;author=M+Hebrard&amp;author=C+Kroger&amp;author=SK+Sivasankaran&amp;author=K+Handler&amp;author=JC+Hinton&amp;publication_year=2011&amp;pages=200-210&amp;doi=10.1016%2Fj.copbio.2011.02.007&amp;pmid=21388802">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">28.</div><div id="B28" class="citations"><div class="citation"><div class="citation-content">Ellermeier JR and Slauch JM. 2007. Adaptation to the host environment: regulation of the SPI1 type III secretion system in <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium. <em>Curr. Opin. Microbiol.</em> 10:24–29.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2006.12.002">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17208038">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000244809900005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Adaptation+to+the+host+environment%3A+regulation+of+the+SPI1+type+III+secretion+system+in+Salmonella+enterica+serovar+Typhimurium&amp;author=JR+Ellermeier&amp;author=JM+Slauch&amp;publication_year=2007&amp;pages=24-29&amp;doi=10.1016%2Fj.mib.2006.12.002&amp;pmid=17208038">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">29.</div><div id="B29" class="citations"><div class="citation"><div class="citation-content">Deiwick J, Nikolaus T, Erdogan S, and Hensel M. 1999. Environmental regulation of <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 2 gene expression. <em>Mol. Microbiol.</em> 31:1759–1773.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.1999.01312.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10209748">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000079511200014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Environmental+regulation+of+Salmonella+pathogenicity+island+2+gene+expression&amp;author=J+Deiwick&amp;author=T+Nikolaus&amp;author=S+Erdogan&amp;author=M+Hensel&amp;publication_year=1999&amp;pages=1759-1773&amp;doi=10.1046%2Fj.1365-2958.1999.01312.x&amp;pmid=10209748">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">30.</div><div id="B30" class="citations"><div class="citation"><div class="citation-content">Murdoch SL, Trunk K, English G, Fritsch MJ, Pourkarimi E, and Coulthurst SJ. 2011. The opportunistic pathogen <span class="named-content" data-type="genus-species">Serratia marcescens</span> utilizes type VI secretion to target bacterial competitors. <em>J. Bacteriol.</em> 193:6057–6069.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.05671-11">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21890705">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000296153400017">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+opportunistic+pathogen+Serratia+marcescens+utilizes+type+VI+secretion+to+target+bacterial+competitors&amp;author=SL+Murdoch&amp;author=K+Trunk&amp;author=G+English&amp;author=MJ+Fritsch&amp;author=E+Pourkarimi&amp;author=SJ+Coulthurst&amp;publication_year=2011&amp;pages=6057-6069&amp;doi=10.1128%2FJB.05671-11&amp;pmid=21890705">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">31.</div><div id="B31" class="citations"><div class="citation"><div class="citation-content">MacIntyre DL, Miyata ST, Kitaoka M, and Pukatzki S. 2010. The <span class="named-content" data-type="genus-species">Vibrio cholerae</span> type VI secretion system displays antimicrobial properties. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 107:19520–19524.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.1012931107">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20974937">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000283997800071">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Vibrio+cholerae+type+VI+secretion+system+displays+antimicrobial+properties&amp;author=DL+MacIntyre&amp;author=ST+Miyata&amp;author=M+Kitaoka&amp;author=S+Pukatzki&amp;publication_year=2010&amp;pages=19520-19524&amp;doi=10.1073%2Fpnas.1012931107&amp;pmid=20974937">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">32.</div><div id="B32" class="citations"><div class="citation"><div class="citation-content">Hood RD, Singh P, Hsu F, Guvener T, Carl MA, Trinidad RR, Silverman JM, Ohlson BB, Hicks KG, Plemel RL, Li M, Schwarz S, Wang WY, Merz AJ, Goodlett DR, and Mougous JD. 2010. A type VI secretion system of <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> targets a toxin to bacteria. <em>Cell Host Microbe</em> 7:25–37.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.chom.2009.12.007">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20114026">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000274808300007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+type+VI+secretion+system+of+Pseudomonas+aeruginosa+targets+a+toxin+to+bacteria&amp;author=RD+Hood&amp;author=P+Singh&amp;author=F+Hsu&amp;author=T+Guvener&amp;author=MA+Carl&amp;author=RR+Trinidad&amp;author=JM+Silverman&amp;author=BB+Ohlson&amp;author=KG+Hicks&amp;author=RL+Plemel&amp;author=M+Li&amp;author=S+Schwarz&amp;publication_year=2010&amp;pages=25-37&amp;doi=10.1016%2Fj.chom.2009.12.007&amp;pmid=20114026">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">33.</div><div id="B33" class="citations"><div class="citation"><div class="citation-content">English G, Trunk K, Rao VA, Srikannathasan V, Hunter WN, and Coulthurst SJ. 2012. New secreted toxins and immunity proteins encoded within the type VI secretion system gene cluster of <span class="named-content" data-type="genus-species">Serratia marcescens</span>. <em>Mol. Microbiol.</em> [Epub ahead of print.] doi:<a href="http://dx.doi.org/10.1111/mmi.12028">10.1111/mmi.12028</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1111/mmi.12028">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22957938">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000311605600014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=New+secreted+toxins+and+immunity+proteins+encoded+within+the+type+VI+secretion+system+gene+cluster+of+Serratia+marcescens&amp;author=G+English&amp;author=K+Trunk&amp;author=VA+Rao&amp;author=V+Srikannathasan&amp;author=WN+Hunter&amp;author=SJ+Coulthurst&amp;publication_year=2012&amp;doi=10.1111%2Fmmi.12028&amp;pmid=22957938">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">34.</div><div id="B34" class="citations"><div class="citation"><div class="citation-content">Basler M, Pilhofer M, Henderson GP, Jensen GJ, and Mekalanos JJ. 2012. Type VI secretion requires a dynamic contractile phage tail-like structure. <em>Nature</em> 483:182–186.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nature10846">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22367545">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000301174900031">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+VI+secretion+requires+a+dynamic+contractile+phage+tail-like+structure&amp;author=M+Basler&amp;author=M+Pilhofer&amp;author=GP+Henderson&amp;author=GJ+Jensen&amp;author=JJ+Mekalanos&amp;publication_year=2012&amp;pages=182-186&amp;doi=10.1038%2Fnature10846&amp;pmid=22367545">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">35.</div><div id="B35" class="citations"><div class="citation"><div class="citation-content">Russell AB, Singh P, Brittnacher M, Bui NK, Hood RD, Carl MA, Agnello DM, Schwarz S, Goodlett DR, Vollmer W, and Mougous JD. 2012. A widespread bacterial type VI secretion effector superfamily identified using a heuristic approach. <em>Cell Host Microbe</em> 11:538–549.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.chom.2012.04.007">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22607806">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000304794500014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+widespread+bacterial+type+VI+secretion+effector+superfamily+identified+using+a+heuristic+approach&amp;author=AB+Russell&amp;author=P+Singh&amp;author=M+Brittnacher&amp;author=NK+Bui&amp;author=RD+Hood&amp;author=MA+Carl&amp;author=DM+Agnello&amp;author=S+Schwarz&amp;author=DR+Goodlett&amp;author=W+Vollmer&amp;author=JD+Mougous&amp;publication_year=2012&amp;pages=538-549&amp;doi=10.1016%2Fj.chom.2012.04.007&amp;pmid=22607806">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">36.</div><div id="B36" class="citations"><div class="citation"><div class="citation-content">Bernard CS, Brunet YR, Gueguen E, and Cascales E. 2010. Nooks and crannies in type VI secretion regulation. <em>J. Bacteriol.</em> 192:3850–3860.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.00370-10">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20511495">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000279782000002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nooks+and+crannies+in+type+VI+secretion+regulation&amp;author=CS+Bernard&amp;author=YR+Brunet&amp;author=E+Gueguen&amp;author=E+Cascales&amp;publication_year=2010&amp;pages=3850-3860&amp;doi=10.1128%2FJB.00370-10&amp;pmid=20511495">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">37.</div><div id="B37" class="citations"><div class="citation"><div class="citation-content">Ma AT, McAuley S, Pukatzki S, and Mekalanos JJ. 2009. Translocation of a <span class="named-content" data-type="genus-species">Vibrio cholerae</span> type VI secretion effector requires bacterial endocytosis by host cells. <em>Cell Host Microbe</em> 5:234–243.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.chom.2009.02.005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19286133">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000264522500006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Translocation+of+a+Vibrio+cholerae+type+VI+secretion+effector+requires+bacterial+endocytosis+by+host+cells&amp;author=AT+Ma&amp;author=S+McAuley&amp;author=S+Pukatzki&amp;author=JJ+Mekalanos&amp;publication_year=2009&amp;pages=234-243&amp;doi=10.1016%2Fj.chom.2009.02.005&amp;pmid=19286133">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">38.</div><div id="B38" class="citations"><div class="citation"><div class="citation-content">Chakravortty D, Rohde M, Jager L, Deiwick J, and Hensel M. 2005. Formation of a novel surface structure encoded by <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 2. <em>EMBO J.</em> 24:2043–2052.</div><div class="external-links"><div><a href="https://doi.org/10.1038/sj.emboj.7600676">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15889142">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000229588100014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Formation+of+a+novel+surface+structure+encoded+by+Salmonella+pathogenicity+island+2&amp;author=D+Chakravortty&amp;author=M+Rohde&amp;author=L+Jager&amp;author=J+Deiwick&amp;author=M+Hensel&amp;publication_year=2005&amp;pages=2043-2052&amp;doi=10.1038%2Fsj.emboj.7600676&amp;pmid=15889142">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">39.</div><div id="B39" class="citations"><div class="citation"><div class="citation-content">Hoiseth SK and Stocker BA. 1981. Aromatic-dependent <span class="named-content" data-type="genus-species">Salmonella</span> typhimurium are non-virulent and effective as live vaccines. <em>Nature</em> 291:238–239.</div><div class="external-links"><div><a href="https://doi.org/10.1038/291238a0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7015147">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1981LQ44500038">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Aromatic-dependent+Salmonella+typhimurium+are+non-virulent+and+effective+as+live+vaccines&amp;author=SK+Hoiseth&amp;author=BA+Stocker&amp;publication_year=1981&amp;pages=238-239&amp;doi=10.1038%2F291238a0&amp;pmid=7015147">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">40.</div><div id="B40" class="citations"><div class="citation"><div class="citation-content">Pukatzki S, Ma AT, Sturtevant D, Krastins B, Sarracino D, Nelson WC, Heidelberg JF, and Mekalanos JJ. 2006. Identification of a conserved bacterial protein secretion system in <span class="named-content" data-type="genus-species">Vibrio cholerae</span> using the <span class="named-content" data-type="genus-species">Dictyostelium</span> host model system. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 103:1528–1533.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0510322103">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16432199">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000235094300067">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+a+conserved+bacterial+protein+secretion+system+in+Vibrio+cholerae+using+the+Dictyostelium+host+model+system&amp;author=S+Pukatzki&amp;author=AT+Ma&amp;author=D+Sturtevant&amp;author=B+Krastins&amp;author=D+Sarracino&amp;author=WC+Nelson&amp;author=JF+Heidelberg&amp;author=JJ+Mekalanos&amp;publication_year=2006&amp;pages=1528-1533&amp;doi=10.1073%2Fpnas.0510322103&amp;pmid=16432199">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">41.</div><div id="B41" class="citations"><div class="citation"><div class="citation-content">Suarez G, Sierra JC, Sha J, Wang S, Erova TE, Fadl AA, Foltz SM, Horneman AJ, and Chopra AK. 2008. Molecular characterization of a functional type VI secretion system from a clinical isolate of <span class="named-content" data-type="genus-species">Aeromonas hydrophila</span>. <em>Microb. Pathog.</em> 44:344–361.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.micpath.2007.10.005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18037263">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000262591000010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Molecular+characterization+of+a+functional+type+VI+secretion+system+from+a+clinical+isolate+of+Aeromonas+hydrophila&amp;author=G+Suarez&amp;author=JC+Sierra&amp;author=J+Sha&amp;author=S+Wang&amp;author=TE+Erova&amp;author=AA+Fadl&amp;author=SM+Foltz&amp;author=AJ+Horneman&amp;author=AK+Chopra&amp;publication_year=2008&amp;pages=344-361&amp;doi=10.1016%2Fj.micpath.2007.10.005&amp;pmid=18037263">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">42.</div><div id="B42" class="citations"><div class="citation"><div class="citation-content">Suarez G, Sierra JC, Erova TE, Sha J, Horneman AJ, and Chopra AK. 2010. A type VI secretion system effector protein VgrG1 from <span class="named-content" data-type="genus-species">Aeromonas hydrophila</span> that induces host cell toxicity by ADP-ribosylation of actin. <em>J. Bacteriol.</em> 192:155–168.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.01260-09">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19880608">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000272636600018">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+type+VI+secretion+system+effector+protein+VgrG1+from+Aeromonas+hydrophila+that+induces+host+cell+toxicity+by+ADP-ribosylation+of+actin&amp;author=G+Suarez&amp;author=JC+Sierra&amp;author=TE+Erova&amp;author=J+Sha&amp;author=AJ+Horneman&amp;author=AK+Chopra&amp;publication_year=2010&amp;pages=155-168&amp;doi=10.1128%2FJB.01260-09&amp;pmid=19880608">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">43.</div><div id="B43" class="citations"><div class="citation"><div class="citation-content">Hacker H, Furmann C, Wagner H, and Hacker G. 2002. Caspase-9/-3 activation and apoptosis are induced in mouse macrophages upon ingestion and digestion of <span class="named-content" data-type="genus-species">Escherichia coli</span> bacteria. <em>J. Immunol.</em> 169:3172–3179.</div><div class="external-links"><div><a href="https://doi.org/10.4049/jimmunol.169.6.3172">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12218135">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000177958200047">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Caspase-9%2F-3+activation+and+apoptosis+are+induced+in+mouse+macrophages+upon+ingestion+and+digestion+of+Escherichia+coli+bacteria&amp;author=H+Hacker&amp;author=C+Furmann&amp;author=H+Wagner&amp;author=G+Hacker&amp;publication_year=2002&amp;pages=3172-3179&amp;doi=10.4049%2Fjimmunol.169.6.3172&amp;pmid=12218135">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">44.</div><div id="B44" class="citations"><div class="citation"><div class="citation-content">Brencic A and Lory S. 2009. Determination of the regulon and identification of novel mRNA targets of <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> RsmA. <em>Mol. Microbiol.</em> 72:612–632.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2009.06670.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19426209">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000265410500005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Determination+of+the+regulon+and+identification+of+novel+mRNA+targets+of+Pseudomonas+aeruginosa+RsmA&amp;author=A+Brencic&amp;author=S+Lory&amp;publication_year=2009&amp;pages=612-632&amp;doi=10.1111%2Fj.1365-2958.2009.06670.x&amp;pmid=19426209">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">45.</div><div id="B45" class="citations"><div class="citation"><div class="citation-content">Brencic A, McFarland KA, McManus HR, Castang S, Mogno I, Dove SL, and Lory S. 2009. The GacS/GacA signal transduction system of <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> acts exclusively through its control over the transcription of the RsmY and RsmZ regulatory small RNAs. <em>Mol. Microbiol.</em> 73:434–445.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2009.06782.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19602144">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000268792700010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+GacS%2FGacA+signal+transduction+system+of+Pseudomonas+aeruginosa+acts+exclusively+through+its+control+over+the+transcription+of+the+RsmY+and+RsmZ+regulatory+small+RNAs&amp;author=A+Brencic&amp;author=KA+McFarland&amp;author=HR+McManus&amp;author=S+Castang&amp;author=I+Mogno&amp;author=SL+Dove&amp;author=S+Lory&amp;publication_year=2009&amp;pages=434-445&amp;doi=10.1111%2Fj.1365-2958.2009.06782.x&amp;pmid=19602144">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">46.</div><div id="B46" class="citations"><div class="citation"><div class="citation-content">Pieper R, Huang ST, Robinson JM, Clark DJ, Alami H, Parmar PP, Perry RD, Fleischmann RD, and Peterson SN. 2009. Temperature and growth phase influence the outer-membrane proteome and the expression of a type VI secretion system in <span class="named-content" data-type="genus-species">Yersinia pestis</span>. <em>Microbiology</em> 155:498–512.</div><div class="external-links"><div><a href="https://doi.org/10.1099/mic.0.022160-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19202098">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000263428900018">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Temperature+and+growth+phase+influence+the+outer-membrane+proteome+and+the+expression+of+a+type+VI+secretion+system+in+Yersinia+pestis&amp;author=R+Pieper&amp;author=ST+Huang&amp;author=JM+Robinson&amp;author=DJ+Clark&amp;author=H+Alami&amp;author=PP+Parmar&amp;author=RD+Perry&amp;author=RD+Fleischmann&amp;author=SN+Peterson&amp;publication_year=2009&amp;pages=498-512&amp;doi=10.1099%2Fmic.0.022160-0&amp;pmid=19202098">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">47.</div><div id="B47" class="citations"><div class="citation"><div class="citation-content">Wagner VE, Bushnell D, Passador L, Brooks AI, and Iglewski BH. 2003. Microarray analysis of <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> quorum-sensing regulons: effects of growth phase and environment. <em>J. Bacteriol.</em> 185:2080–2095.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.185.7.2080-2095.2003">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12644477">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000181703700003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Microarray+analysis+of+Pseudomonas+aeruginosa+quorum-sensing+regulons%3A+effects+of+growth+phase+and+environment&amp;author=VE+Wagner&amp;author=D+Bushnell&amp;author=L+Passador&amp;author=AI+Brooks&amp;author=BH+Iglewski&amp;publication_year=2003&amp;pages=2080-2095&amp;doi=10.1128%2FJB.185.7.2080-2095.2003&amp;pmid=12644477">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">48.</div><div id="B48" class="citations"><div class="citation"><div class="citation-content">Ishikawa T, Rompikuntal PK, Lindmark B, Milton DL, and Wai SN. 2009. Quorum sensing regulation of the two <i>hcp</i> alleles in <span class="named-content" data-type="genus-species">Vibrio cholerae</span> O1 strains. <em>PLoS One</em> 4:e6734. doi:<a href="http://dx.doi.org/10.1371/journal.pone.0006734">10.1371/journal.pone.0006734</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.pone.0006734">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19701456">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269268300011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Quorum+sensing+regulation+of+the+two+hcp+alleles+in+Vibrio+cholerae+O1+strains&amp;author=T+Ishikawa&amp;author=PK+Rompikuntal&amp;author=B+Lindmark&amp;author=DL+Milton&amp;author=SN+Wai&amp;publication_year=2009&amp;pages=e6734&amp;doi=10.1371%2Fjournal.pone.0006734&amp;pmid=19701456">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">49.</div><div id="B49" class="citations"><div class="citation"><div class="citation-content">Khajanchi BK, Sha J, Kozlova EV, Erova TE, Suarez G, Sierra JC, Popov VL, Horneman AJ, and Chopra AK. 2009. N-acylhomoserine lactones involved in quorum sensing control the type VI secretion system, biofilm formation, protease production, and in vivo virulence in a clinical isolate of <span class="named-content" data-type="genus-species">Aeromonas hydrophila</span>. <em>Microbiology</em> 155:3518–3531.</div><div class="external-links"><div><a href="https://doi.org/10.1099/mic.0.031575-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19729404">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000272121300007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=N-acylhomoserine+lactones+involved+in+quorum+sensing+control+the+type+VI+secretion+system%2C+biofilm+formation%2C+protease+production%2C+and+in+vivo+virulence+in+a+clinical+isolate+of+Aeromonas+hydrophila&amp;author=BK+Khajanchi&amp;author=J+Sha&amp;author=EV+Kozlova&amp;author=TE+Erova&amp;author=G+Suarez&amp;author=JC+Sierra&amp;author=VL+Popov&amp;author=AJ+Horneman&amp;author=AK+Chopra&amp;publication_year=2009&amp;pages=3518-3531&amp;doi=10.1099%2Fmic.0.031575-0&amp;pmid=19729404">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">50.</div><div id="B50" class="citations"><div class="citation"><div class="citation-content">Lesic B, Starkey M, He J, Hazan R, and Rahme LG. 2009. Quorum sensing differentially regulates <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> type VI secretion locus I and homologous loci II and III, which are required for pathogenesis. <em>Microbiology</em> 155:2845–2855.</div><div class="external-links"><div><a href="https://doi.org/10.1099/mic.0.029082-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19497948">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000270264300005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Quorum+sensing+differentially+regulates+Pseudomonas+aeruginosa+type+VI+secretion+locus+I+and+homologous+loci+II+and+III%2C+which+are+required+for+pathogenesis&amp;author=B+Lesic&amp;author=M+Starkey&amp;author=J+He&amp;author=R+Hazan&amp;author=LG+Rahme&amp;publication_year=2009&amp;pages=2845-2855&amp;doi=10.1099%2Fmic.0.029082-0&amp;pmid=19497948">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">51.</div><div id="B51" class="citations"><div class="citation"><div class="citation-content">Mougous JD, Gifford CA, Ramsdell TL, and Mekalanos JJ. 2007. Threonine phosphorylation post-translationally regulates protein secretion in <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span>. <em>Nat. Cell Biol.</em> 9:797–803.</div><div class="external-links"><div><a href="https://doi.org/10.1038/ncb1605">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17558395">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000247760700015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Threonine+phosphorylation+post-translationally+regulates+protein+secretion+in+Pseudomonas+aeruginosa&amp;author=JD+Mougous&amp;author=CA+Gifford&amp;author=TL+Ramsdell&amp;author=JJ+Mekalanos&amp;publication_year=2007&amp;pages=797-803&amp;doi=10.1038%2Fncb1605&amp;pmid=17558395">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">52.</div><div id="B52" class="citations"><div class="citation"><div class="citation-content">Mougous JD, Cuff ME, Raunser S, Shen A, Zhou M, Gifford CA, Goodman AL, Joachimiak G, Ordonez CL, Lory S, Walz T, Joachimiak A, and Mekalanos JJ. 2006. A virulence locus of <span class="named-content" data-type="genus-species">Pseudomonas aeruginosa</span> encodes a protein secretion apparatus. <em>Science</em> 312:1526–1530.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1128393">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16763151">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000238124100049">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+virulence+locus+of+Pseudomonas+aeruginosa+encodes+a+protein+secretion+apparatus&amp;author=JD+Mougous&amp;author=ME+Cuff&amp;author=S+Raunser&amp;author=A+Shen&amp;author=M+Zhou&amp;author=CA+Gifford&amp;author=AL+Goodman&amp;author=G+Joachimiak&amp;author=CL+Ordonez&amp;author=S+Lory&amp;author=T+Walz&amp;author=A+Joachimiak&amp;publication_year=2006&amp;pages=1526-1530&amp;doi=10.1126%2Fscience.1128393&amp;pmid=16763151">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">53.</div><div id="B53" class="citations"><div class="citation"><div class="citation-content">Valdivia RH and Falkow S. 1997. Fluorescence-based isolation of bacterial genes expressed within host cells. <em>Science</em> 277:2007–2011.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.277.5334.2007">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9302299">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1997XX84900049">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Fluorescence-based+isolation+of+bacterial+genes+expressed+within+host+cells&amp;author=RH+Valdivia&amp;author=S+Falkow&amp;publication_year=1997&amp;pages=2007-2011&amp;doi=10.1126%2Fscience.277.5334.2007&amp;pmid=9302299">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">54.</div><div id="B54" class="citations"><div class="citation"><div class="citation-content">Cirillo DM, Valdivia RH, Monack DM, and Falkow S. 1998. Macrophage-dependent induction of the <span class="named-content" data-type="genus-species">Salmonella</span> pathogenicity island 2 type III secretion system and its role in intracellular survival. <em>Mol. Microbiol.</em> 30:175–188.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.1998.01048.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9786194">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000076538500015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Macrophage-dependent+induction+of+the+Salmonella+pathogenicity+island+2+type+III+secretion+system+and+its+role+in+intracellular+survival&amp;author=DM+Cirillo&amp;author=RH+Valdivia&amp;author=DM+Monack&amp;author=S+Falkow&amp;publication_year=1998&amp;pages=175-188&amp;doi=10.1046%2Fj.1365-2958.1998.01048.x&amp;pmid=9786194">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">55.</div><div id="B55" class="citations"><div class="citation"><div class="citation-content">Lim S, Kim B, Choi HS, Lee Y, and Ryu S. 2006. Fis is required for proper regulation of ssaG expression in <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium. <em>Microb. Pathog.</em> 41:33–42.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.micpath.2006.03.005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16777370">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000205812300004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Fis+is+required+for+proper+regulation+of+ssaG+expression+in+Salmonella+enterica+serovar+Typhimurium&amp;author=S+Lim&amp;author=B+Kim&amp;author=HS+Choi&amp;author=Y+Lee&amp;author=S+Ryu&amp;publication_year=2006&amp;pages=33-42&amp;doi=10.1016%2Fj.micpath.2006.03.005&amp;pmid=16777370">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">56.</div><div id="B56" class="citations"><div class="citation"><div class="citation-content">Bradley MD, Beach MB, de Koning AP, Pratt TS, and Osuna R. 2007. Effects of Fis on <span class="named-content" data-type="genus-species">Escherichia coli</span> gene expression during different growth stages. <em>Microbiology</em> 153:2922–2940.</div><div class="external-links"><div><a href="https://doi.org/10.1099/mic.0.2007/008565-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17768236">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000250013400011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Effects+of+Fis+on+Escherichia+coli+gene+expression+during+different+growth+stages&amp;author=MD+Bradley&amp;author=MB+Beach&amp;author=AP+de+Koning&amp;author=TS+Pratt&amp;author=R+Osuna&amp;publication_year=2007&amp;pages=2922-2940&amp;doi=10.1099%2Fmic.0.2007%2F008565-0&amp;pmid=17768236">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">57.</div><div id="B57" class="citations"><div class="citation"><div class="citation-content">Xu J and Johnson RC. 1995. Identification of genes negatively regulated by Fis: Fis and RpoS comodulate growth-phase-dependent gene expression in <span class="named-content" data-type="genus-species">Escherichia coli</span>. <em>J. Bacteriol.</em> 177:938–947.</div><div class="external-links"><div><a href="https://doi.org/10.1128/jb.177.4.938-947.1995">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7860604">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1995QG25000011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+genes+negatively+regulated+by+Fis%3A+Fis+and+RpoS+comodulate+growth-phase-dependent+gene+expression+in+Escherichia+coli&amp;author=J+Xu&amp;author=RC+Johnson&amp;publication_year=1995&amp;pages=938-947&amp;doi=10.1128%2Fjb.177.4.938-947.1995&amp;pmid=7860604">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">58.</div><div id="B58" class="citations"><div class="citation"><div class="citation-content">McGhie EJ, Brawn LC, Hume PJ, Humphreys D, and Koronakis V. 2009. <span class="named-content" data-type="genus-species">Salmonella</span> takes control: effector-driven manipulation of the host. <em>Curr. Opin. Microbiol.</em> 12:117–124.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2008.12.001">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19157959">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000264319100018">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonella+takes+control%3A+effector-driven+manipulation+of+the+host&amp;author=EJ+McGhie&amp;author=LC+Brawn&amp;author=PJ+Hume&amp;author=D+Humphreys&amp;author=V+Koronakis&amp;publication_year=2009&amp;pages=117-124&amp;doi=10.1016%2Fj.mib.2008.12.001&amp;pmid=19157959">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">59.</div><div id="B59" class="citations"><div class="citation"><div class="citation-content">Steele-Mortimer O. 2008. The <span class="named-content" data-type="genus-species">Salmonella</span>-containing vacuole: moving with the times. <em>Curr. Opin. Microbiol.</em> 11:38–45.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2008.01.002">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18304858">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000255233400007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+Salmonella-containing+vacuole%3A+moving+with+the+times&amp;author=O+Steele-Mortimer&amp;publication_year=2008&amp;pages=38-45&amp;doi=10.1016%2Fj.mib.2008.01.002&amp;pmid=18304858">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">60.</div><div id="B60" class="citations"><div class="citation"><div class="citation-content">Brumell JH and Grinstein S. 2004. <span class="named-content" data-type="genus-species">Salmonella</span> redirects phagosomal maturation. <em>Curr. Opin. Microbiol.</em> 7:78–84.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2003.12.005">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15036145">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000189358700014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonella+redirects+phagosomal+maturation&amp;author=JH+Brumell&amp;author=S+Grinstein&amp;publication_year=2004&amp;pages=78-84&amp;doi=10.1016%2Fj.mib.2003.12.005&amp;pmid=15036145">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">61.</div><div id="B61" class="citations"><div class="citation"><div class="citation-content">Bakowski MA, Braun V, and Brumell JH. 2008. <span class="named-content" data-type="genus-species">Salmonella</span>-containing vacuoles: directing traffic and nesting to grow. <em>Traffic</em> 9:2022–2031.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1600-0854.2008.00827.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18778407">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000261086000002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Salmonella-containing+vacuoles%3A+directing+traffic+and+nesting+to+grow&amp;author=MA+Bakowski&amp;author=V+Braun&amp;author=JH+Brumell&amp;publication_year=2008&amp;pages=2022-2031&amp;doi=10.1111%2Fj.1600-0854.2008.00827.x&amp;pmid=18778407">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">62.</div><div id="B62" class="citations"><div class="citation"><div class="citation-content">Thompson JA, Liu M, Helaine S, and Holden DW. 2011. Contribution of the PhoP/Q regulon to survival and replication of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium in macrophages. <em>Microbiology</em> 157:2084–2093.</div><div class="external-links"><div><a href="https://doi.org/10.1099/mic.0.048926-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21511762">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000293204000020">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Contribution+of+the+PhoP%2FQ+regulon+to+survival+and+replication+of+Salmonella+enterica+serovar+Typhimurium+in+macrophages&amp;author=JA+Thompson&amp;author=M+Liu&amp;author=S+Helaine&amp;author=DW+Holden&amp;publication_year=2011&amp;pages=2084-2093&amp;doi=10.1099%2Fmic.0.048926-0&amp;pmid=21511762">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">63.</div><div id="B63" class="citations"><div class="citation"><div class="citation-content">Setta AM, Barrow PA, Kaiser P, and Jones MA. 2012. Early immune dynamics following infection with <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovars Enteritidis, Infantis, Pullorum and Gallinarum: cytokine and chemokine gene expression profile and cellular changes of chicken cecal tonsils. <em>Comp. Immunol. Microbiol. Infect. Dis.</em> 35:397–410.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.cimid.2012.03.004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22512820">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000307616000001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Early+immune+dynamics+following+infection+with+Salmonella+enterica+serovars+Enteritidis%2C+Infantis%2C+Pullorum+and+Gallinarum%3A+cytokine+and+chemokine+gene+expression+profile+and+cellular+changes+of+chicken+cecal+tonsils&amp;author=AM+Setta&amp;author=PA+Barrow&amp;author=P+Kaiser&amp;author=MA+Jones&amp;publication_year=2012&amp;pages=397-410&amp;doi=10.1016%2Fj.cimid.2012.03.004&amp;pmid=22512820">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">64.</div><div id="B64" class="citations"><div class="citation"><div class="citation-content">Setta A, Barrow PA, Kaiser P, and Jones MA. 2012. Immune dynamics following infection of avian macrophages and epithelial cells with typhoidal and non-typhoidal <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovars; bacterial invasion and persistence, nitric oxide and oxygen production, differential host gene expression, NF-kappaB signalling and cell cytotoxicity. <em>Vet. Immunol. Immunopathol.</em> 146:212–224.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.vetimm.2012.03.008">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22475571">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000304585200003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Immune+dynamics+following+infection+of+avian+macrophages+and+epithelial+cells+with+typhoidal+and+non-typhoidal+Salmonella+enterica+serovars%3B+bacterial+invasion+and+persistence%2C+nitric+oxide+and+oxygen+production%2C+differential+host+gene+expression%2C+NF-kappaB+signalling+and+cell+cytotoxicity&amp;author=A+Setta&amp;author=PA+Barrow&amp;author=P+Kaiser&amp;author=MA+Jones&amp;publication_year=2012&amp;pages=212-224&amp;doi=10.1016%2Fj.vetimm.2012.03.008&amp;pmid=22475571">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">65.</div><div id="B65" class="citations"><div class="citation"><div class="citation-content">Libby SJ, Brehm MA, Greiner DL, Shultz LD, McClelland M, Smith KD, Cookson BT, Karlinsey JE, Kinkel TL, Porwollik S, Canals R, Cummings LA, and Fang FC. 2010. Humanized nonobese diabetic-scid IL2rgamma null mice are susceptible to lethal <span class="named-content" data-type="genus-species">Salmonella</span> Typhi infection. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 107:15589–15594.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.1005566107">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20713716">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000281468500054">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Humanized+nonobese+diabetic-scid+IL2rgamma+null+mice+are+susceptible+to+lethal+Salmonella+Typhi+infection&amp;author=SJ+Libby&amp;author=MA+Brehm&amp;author=DL+Greiner&amp;author=LD+Shultz&amp;author=M+McClelland&amp;author=KD+Smith&amp;author=BT+Cookson&amp;author=JE+Karlinsey&amp;author=TL+Kinkel&amp;author=S+Porwollik&amp;author=R+Canals&amp;author=LA+Cummings&amp;publication_year=2010&amp;pages=15589-15594&amp;doi=10.1073%2Fpnas.1005566107&amp;pmid=20713716">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">66.</div><div id="B66" class="citations"><div class="citation"><div class="citation-content">Wang M, Luo Z, Du H, Xu S, Ni B, Zhang H, Sheng X, Xu H, and Huang X. 2011. Molecular characterization of a functional type VI secretion system in <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhi. <em>Curr. Microbiol.</em> 63:22–31.</div><div class="external-links"><div><a href="https://doi.org/10.1007/s00284-011-9935-z">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21487806">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000292561500004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Molecular+characterization+of+a+functional+type+VI+secretion+system+in+Salmonella+enterica+serovar+Typhi&amp;author=M+Wang&amp;author=Z+Luo&amp;author=H+Du&amp;author=S+Xu&amp;author=B+Ni&amp;author=H+Zhang&amp;author=X+Sheng&amp;author=H+Xu&amp;author=X+Huang&amp;publication_year=2011&amp;pages=22-31&amp;doi=10.1007%2Fs00284-011-9935-z&amp;pmid=21487806">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">67.</div><div id="B67" class="citations"><div class="citation"><div class="citation-content">Liu J, Guo JT, Li YG, Johnston RN, Liu GR, and Liu SL. 2012. The type VI secretion system gene cluster of <span class="named-content" data-type="genus-species">Salmonella</span> typhimurium: required for full virulence in mice. <em>J. Basic Microbiol.</em> [Epub ahead of print.] doi:<a href="http://dx.doi.org/10.1002/jobm.201200047">10.1002/jobm.201200047</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1002/jobm.201200047">Crossref</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000323227300007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+type+VI+secretion+system+gene+cluster+of+Salmonella+typhimurium%3A+required+for+full+virulence+in+mice&amp;author=J+Liu&amp;author=JT+Guo&amp;author=YG+Li&amp;author=RN+Johnston&amp;author=GR+Liu&amp;author=SL+Liu&amp;publication_year=2012&amp;doi=10.1002%2Fjobm.201200047">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">68.</div><div id="B68" class="citations"><div class="citation"><div class="citation-content">Mulder DT, Cooper CA, and Coombes BK. 2012. Type VI secretion system-associated gene clusters contribute to pathogenesis of <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Typhimurium. <em>Infect. Immun.</em> 80:1996–2007.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.06205-11">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22493086">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000304387700006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Type+VI+secretion+system-associated+gene+clusters+contribute+to+pathogenesis+of+Salmonella+enterica+serovar+Typhimurium&amp;author=DT+Mulder&amp;author=CA+Cooper&amp;author=BK+Coombes&amp;publication_year=2012&amp;pages=1996-2007&amp;doi=10.1128%2FIAI.06205-11&amp;pmid=22493086">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">69.</div><div id="B69" class="citations"><div class="citation"><div class="citation-content">Cherepanov PP and Wackernagel W. 1995. Gene disruption in <span class="named-content" data-type="genus-species">Escherichia coli</span>: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant. <em>Gene</em> 158:9–14.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0378-1119(95)00193-A">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7789817">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1995RE87900002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Gene+disruption+in+Escherichia+coli%3A+TcR+and+KmR+cassettes+with+the+option+of+Flp-catalyzed+excision+of+the+antibiotic-resistance+determinant&amp;author=PP+Cherepanov&amp;author=W+Wackernagel&amp;publication_year=1995&amp;pages=9-14&amp;doi=10.1016%2F0378-1119%2895%2900193-A&amp;pmid=7789817">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">70.</div><div id="B70" class="citations"><div class="citation"><div class="citation-content">Santiviago CA, Blondel CJ, Quezada CP, Silva CA, Tobar PM, Porwollik S, McClelland M, Andrews-Polymenis HL, Toro CS, Zaldivar M, and Contreras I. 2010. Spontaneous excision of the <span class="named-content" data-type="genus-species">Salmonella enterica</span> serovar Enteritidis-specific defective prophage-like element ϕSE14. <em>J. Bacteriol.</em> 192:2246–2254.</div><div class="external-links"><div><a href="https://doi.org/10.1128/JB.00270-09">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/20172996">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000276048300023">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Spontaneous+excision+of+the+Salmonella+enterica+serovar+Enteritidis-specific+defective+prophage-like+element+%CF%95SE14&amp;author=CA+Santiviago&amp;author=CJ+Blondel&amp;author=CP+Quezada&amp;author=CA+Silva&amp;author=PM+Tobar&amp;author=S+Porwollik&amp;author=M+McClelland&amp;author=HL+Andrews-Polymenis&amp;author=CS+Toro&amp;author=M+Zaldivar&amp;author=I+Contreras&amp;publication_year=2010&amp;pages=2246-2254&amp;doi=10.1128%2FJB.00270-09&amp;pmid=20172996">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">71.</div><div id="B71" class="citations"><div class="citation"><div class="citation-content">Santiviago CA, Reynolds MM, Porwollik S, Choi SH, Long F, Andrews-Polymenis HL, and McClelland M. 2009. Analysis of pools of targeted <span class="named-content" data-type="genus-species">Salmonella</span> deletion mutants identifies novel genes affecting fitness during competitive infection in mice. <em>PLoS Pathog.</em> 5:e1000477. doi:<a href="http://dx.doi.org/10.1371/journal.ppat.1000477">10.1371/journal.ppat.1000477</a>.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.ppat.1000477">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19578432">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000269224500030">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Analysis+of+pools+of+targeted+Salmonella+deletion+mutants+identifies+novel+genes+affecting+fitness+during+competitive+infection+in+mice&amp;author=CA+Santiviago&amp;author=MM+Reynolds&amp;author=S+Porwollik&amp;author=SH+Choi&amp;author=F+Long&amp;author=HL+Andrews-Polymenis&amp;author=M+McClelland&amp;publication_year=2009&amp;pages=e1000477&amp;doi=10.1371%2Fjournal.ppat.1000477&amp;pmid=19578432">Google Scholar</a></div></div></div></div></div></section></div></section><div class="core-collateral"><div id="core-collateral-info" role="tabpanel"><header><h2><i class="icon-info" aria-hidden="true"></i>Information &amp; Contributors</h2></header><section id="tab-information" aria-labelledby="information" role="tabpanel"><h3>Information</h3><section class="core-self-citation"><h4>Published In</h4><div class="core-journal-presentation">



        
        <div class="cover-image"><div class="cover-image__image"><img src="/cms/asset/c4308797-6ffd-4494-aec3-3bbfe515c342/iai.2013.81.issue-4.cover.gif" alt="Infection and Immunity cover image"/></div></div>
<div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">Infection and Immunity</span></div><div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">81</span></span> • <span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">4</span></span> • <span property="datePublished">April 2013</span></div><div class="core-pagination"><span class="heading">Pages</span>: <span class="content"><span property="pageStart">1207</span> - <span property="pageEnd">1220</span></span></div><div class="core-editors"><span class="heading">Editor</span>: <span role="list"><span role="listitem" property="editor" typeof="Person"><span property="givenName">A.</span> <span property="familyName">Camilli</span></span></span></div><div class="core-pmid"><span class="heading">PubMed</span>: <a class="content" href="https://pubmed.ncbi.nlm.nih.gov/23357385/" property="sameAs">23357385</a></div></div></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">© 2013 American Society for Microbiology.</div></section><section class="core-history"><h4>History</h4><div>Received: 12 November 2012</div><div>Returned for modification: 16 December 2012</div><div>Accepted: 19 January 2013</div><div>Published online: 18 March 2013</div></section><section class="core-permissions"><h4>Permissions</h4><div>Request permissions for this article.</div><div><a href="https://marketplace.copyright.com/rs-ui-web/mp/search/all/10.1128%2FIAI.01165-12">Request Permissions</a></div></section><div class="core-download"><a href="/doi/pdf/10.1128/IAI.01165-12?download=true" class="btn btn-primary"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></div></section><section id="tab-contributors" aria-labelledby="contributors" role="tabpanel"><h3>Contributors</h3><section class="core-authors"><h4>Authors</h4><div id="con1" property="author" typeof="Person"><div class="heading"><span property="givenName">Carlos J.</span> <span property="familyName">Blondel</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Blondel/Carlos+J">View all articles by this author</a></div></div></div><div id="con2" property="author" typeof="Person"><div class="heading"><span property="givenName">Juan C.</span> <span property="familyName">Jiménez</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Jim%C3%A9nez/Juan+C">View all articles by this author</a></div></div></div><div id="con3" property="author" typeof="Person"><div class="heading"><span property="givenName">Lorenzo E.</span> <span property="familyName">Leiva</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Leiva/Lorenzo+E">View all articles by this author</a></div></div></div><div id="con4" property="author" typeof="Person"><div class="heading"><span property="givenName">Sergio A.</span> <span property="familyName">Álvarez</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/%C3%81lvarez/Sergio+A">View all articles by this author</a></div></div></div><div id="con5" property="author" typeof="Person"><div class="heading"><span property="givenName">Bernardo I.</span> <span property="familyName">Pinto</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Pinto/Bernardo+I">View all articles by this author</a></div></div></div><div id="con6" property="author" typeof="Person"><div class="heading"><span property="givenName">Francisca</span> <span property="familyName">Contreras</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Contreras/Francisca">View all articles by this author</a></div></div></div><div id="con7" property="author" typeof="Person"><div class="heading"><span property="givenName">David</span> <span property="familyName">Pezoa</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Pezoa/David">View all articles by this author</a></div></div></div><div id="con8" property="author" typeof="Person"><div class="heading"><span property="givenName">Carlos A.</span> <span property="familyName">Santiviago</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Santiviago/Carlos+A">View all articles by this author</a></div></div></div><div id="con9" property="author" typeof="Person"><div class="heading"><span property="givenName">Inés</span> <span property="familyName">Contreras</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Departamento de Bioquímica y Biología Molecular, Facultad de Ciencias Químicas y Farmacéuticas, Universidad de Chile, Santiago, Chile</span></div></div><div class="core-author-link"><a href="/authored-by/Contreras/In%C3%A9s">View all articles by this author</a></div></div></div></section><section class="core-editors"><h4>Editor</h4><div id="editor" property="editor" typeof="Person"><div class="heading"><span property="givenName">A.</span> <span property="familyName">Camilli</span></div><div class="content"><div property="jobTitle">Editor</div></div></div></section><section class="core-authors-notes"><h4>Notes</h4><div role="doc-footnote">Address correspondence to Inés Contreras, <a href="/cdn-cgi/l/email-protection#afc6ccc0c1dbddcaddefdaccc7c6c3ca81ccc3"><span class="__cf_email__" data-cfemail="96fff5f9f8e2e4f3e4d6e3f5fefffaf3b8f5fa">[email&#160;protected]</span></a>.</div></section></section></div><div id="core-collateral-metrics" role="tabpanel"><header><h2><i class="icon-timeline" aria-hidden="true"></i>Metrics &amp; Citations</h2></header><section id="tab-metrics-inner" aria-labelledby="metrics-inner" role="tabpanel"><h3>Metrics</h3><section>



        
        







                <div class="sidebar-metric-container">
                        <input class="sidebar-metric__data" type="hidden" name="json" value='{"data":{"dataSections":[{"total":"196","months":[{"month":"2022-2-1","count":28},{"month":"2022-1-1","count":10},{"month":"2021-12-1","count":25},{"month":"2021-11-1","count":17},{"month":"2021-10-1","count":39},{"month":"2021-9-1","count":18},{"month":"2021-8-1","count":20},{"month":"2021-7-1","count":19},{"month":"2021-6-1","count":20}],"color":"color-4","firstMonth":[],"id":"download","title":"Download","startDate":"2021-6-1","displayDefault":true},{"total":"38","months":[{"month":"2022-2-1","count":1},{"month":"2022-1-1","count":0},{"month":"2021-12-1","count":0},{"month":"2021-11-1","count":1},{"month":"2021-10-1","count":0},{"month":"2021-9-1","count":0},{"month":"2021-8-1","count":0},{"month":"2021-7-1","count":0},{"month":"2021-6-1","count":0},{"month":"2021-5-1","count":0},{"month":"2021-4-1","count":0},{"month":"2021-3-1","count":0},{"month":"2021-2-1","count":0},{"month":"2021-1-1","count":1},{"month":"2020-12-1","count":1},{"month":"2020-11-1","count":2},{"month":"2020-10-1","count":0},{"month":"2020-9-1","count":0},{"month":"2020-8-1","count":1},{"month":"2020-7-1","count":1},{"month":"2020-6-1","count":0},{"month":"2020-5-1","count":0},{"month":"2020-4-1","count":0},{"month":"2020-3-1","count":1},{"month":"2020-2-1","count":0},{"month":"2020-1-1","count":0},{"month":"2019-12-1","count":1},{"month":"2019-11-1","count":0},{"month":"2019-10-1","count":0},{"month":"2019-9-1","count":0},{"month":"2019-8-1","count":1},{"month":"2019-7-1","count":0},{"month":"2019-6-1","count":2},{"month":"2019-5-1","count":0},{"month":"2019-4-1","count":1},{"month":"2019-3-1","count":1},{"month":"2019-2-1","count":0},{"month":"2019-1-1","count":1},{"month":"2018-12-1","count":0},{"month":"2018-11-1","count":0},{"month":"2018-10-1","count":1},{"month":"2018-9-1","count":0},{"month":"2018-8-1","count":1},{"month":"2018-7-1","count":0},{"month":"2018-6-1","count":0},{"month":"2018-5-1","count":1},{"month":"2018-4-1","count":0},{"month":"2018-3-1","count":0},{"month":"2018-2-1","count":0},{"month":"2018-1-1","count":0},{"month":"2017-12-1","count":1},{"month":"2017-11-1","count":0},{"month":"2017-10-1","count":0},{"month":"2017-9-1","count":0},{"month":"2017-8-1","count":0},{"month":"2017-7-1","count":0},{"month":"2017-6-1","count":0},{"month":"2017-5-1","count":0},{"month":"2017-4-1","count":0},{"month":"2017-3-1","count":2},{"month":"2017-2-1","count":0},{"month":"2017-1-1","count":1},{"month":"2016-12-1","count":1},{"month":"2016-11-1","count":0},{"month":"2016-10-1","count":0},{"month":"2016-9-1","count":0},{"month":"2016-8-1","count":2},{"month":"2016-7-1","count":0},{"month":"2016-6-1","count":0},{"month":"2016-5-1","count":1},{"month":"2016-4-1","count":0},{"month":"2016-3-1","count":1},{"month":"2016-2-1","count":1},{"month":"2016-1-1","count":1},{"month":"2015-12-1","count":0},{"month":"2015-11-1","count":0},{"month":"2015-10-1","count":1},{"month":"2015-9-1","count":1},{"month":"2015-8-1","count":0},{"month":"2015-7-1","count":0},{"month":"2015-6-1","count":0},{"month":"2015-5-1","count":0},{"month":"2015-4-1","count":0},{"month":"2015-3-1","count":0},{"month":"2015-2-1","count":0},{"month":"2015-1-1","count":1},{"month":"2014-12-1","count":2},{"month":"2014-11-1","count":0},{"month":"2014-10-1","count":0},{"month":"2014-9-1","count":0},{"month":"2014-8-1","count":0},{"month":"2014-7-1","count":1},{"month":"2014-6-1","count":1},{"month":"2014-5-1","count":1}],"color":"color-0","id":"citation","title":"Citation","startDate":"2014-5-1","displayDefault":true}],"doi":"10.1128/IAI.01165-12","widgetTitle":"Article Metrics"}}'>
                </div>
        
</section><section></section><section class="text-right"><a href="/doi/metrics/10.1128/IAI.01165-12" class="btn btn--inverse open-in-viewer"><span>VIEW ALL METRICS</span><i class="icon-open_in_new"></i></a></section></section><section id="tab-citations" aria-labelledby="citations" role="tabpanel"><h3>Citations</h3><section></section><section>



        
        <div class="citation-download">
	<p class="citation-msg">
		If you have the appropriate software installed, you can download article citation data to the citation manager of your choice. Simply select your manager software from the list below and click Download.
	</p>
	<!-- download options -->
	<form action="/action/downloadCitation" name="frmCitmgr" class="citation-form" method="post" target="_self"><input type="hidden" name="doi" value="10.1128/IAI.01165-12"><input type="hidden" name="downloadFileName" value="csp_81_"><input type="hidden" name="include" value="abs">
		<fieldset class="format-select">
			<label for="slct_format">Format</label>
			<div class="select-container">
				<select id="slct_format" name="format" class="js__slcInclude">
					<option value="" selected="selected">Please select one from the list</option>
					<option value="ris">RIS (ProCite, Reference Manager)</option>
					<option value="endnote">EndNote</option>
					<option value="bibtex">BibTex</option>
					<option value="medlars">Medlars</option>
					<option value="refworks">RefWorks</option>
				</select>
			</div>
			<label class="label-direct" for="direct">
				<input id="direct" type="checkbox" name="direct" value="" checked="checked">
				<span class="round-check"><span class="check"></span></span>
				Direct import
			</label>
		</fieldset>           
		<footer class="form-footer">
			<input onclick="onCitMgrSubmit()" class="formbutton" type="submit" name="submit" value="Download">
		</footer>
	</form>
</div>

</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel"><header><h2><i class="icon-eye" aria-hidden="true"></i>View Options</h2></header><div tabindex="0" class="section--wrapper"><h3>View options</h3><section><h4>ePub</h4><a href="https://journals.asm.org/doi/epub/10.1128/IAI.01165-12" class="btn"><i aria-hidden="true" class="icon-epub"></i><span>Open ePub</span></a></section><section><h4>PDF</h4><a href="/doi/pdf/10.1128/IAI.01165-12?download=true" download="true" class="btn btn--pdf"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></section><section><h4>Full Text</h4><a href="https://journals.asm.org/doi/full/10.1128/IAI.01165-12" class="btn"><i aria-hidden="true" class="icon-subject"></i><span>Open Full Text</span></a></section></div><!-- Its needed to duplicate this for PB check Pill.js as well--><div tabindex="-1" class="section--wrapper"><h3>Get Access</h3><section>



        
        












    <div class="purchaseArea pill__item" id="purchaseArea">
        <div class="login-options" data-pb-dropzone="login-options">
            
        </div>
        <div class="purchaseOptions">
            <div class="purchaseOptions--header">
                <h4 class="purchaseArea--section-heading">
                    PURCHASE OPTIONS
                </h4>
                
                        <div class="save-for-later">
                            <a data-bind="saveItem" data-doi="10.1128/IAI.01165-12" class="purchaseOptions--save" href="/action/saveItem?doi=10.1128/IAI.01165-12">
                                <i aria-hidden="true" class="icon-heart mx-1"></i>
                                <span>Save for later</span>
                            </a>
                            <a class="saved-go-cart hidden"  href="/action/showCart?FlowID=2"><span class="icon-cart-icon-material"></span> Item saved, go to cart </a>
                        </div>
                    
            </div>
            <p class="purchaseArea--section-info">
                Subscribe and get full access to this article
            </p>

            <div class="purchaseOptions--options-list">
                











<div class="purchase-options-container">
    

   
        <div class="savedItem-info hidden">
            <p></p>
        </div>
    

     
            
                
                
                        <div class="purchaseMessage hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.01165-12-500044">
                            <p class="errorMsgBox"></p>
                        </div>
                        <div class="add-journal-to-cart">
                            <div class="purchaseOptions--option mb-3">
                                 
                                    <h5 class="option-title">Buy Article</h5>
                                    <span class="option-info mb-1">Infection and Immunity</span>
                                    <span class="option-info">Vol.81 &bull; Issue 4 &bull; 
                                            ASM Journals Pay Per View, PPV 25
                                        </span>
                                    <div class="option-actions mt-3 pt-3">
                                        <div class="option-price">
                                            <b>$25.00</b>
                                        </div>
                                        <a href="/action/addToCart?id=ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.01165-12-500044" data-bind="addItem"
                                           data-key="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.01165-12-500044" data-entity="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.01165-12-500044"
                                           class="btn btn-primary btn--borderless">
                                            <i aria-hidden="true" class="mx-2 icon-shopping_cart"> </i>
                                            <span>ADD TO CART</span>
                                        </a>
                                    </div>
                                 
                            <div class="option-actions mt-3 pt-3 hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.01165-12-500044" style="justify-content: flex-end">
                                <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                    <span>Checkout</span>
                                </a>
                            </div>
                        </div>

                        <div class="purchaseOptions--option mb-3">
                                                             <h5 class="option-title">Buy Article</h5>
                                <span class="option-info mb-1">Infection and Immunity</span>
                                <span class="option-info">Vol.81 &bull; Issue 4 &bull; 
                                        ASM Journals Pay Per View, PPV 25
                                    
                                </span>
                             
                             <div class="option-actions mt-3 pt-3 hidden"
                                     data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/IAI.01165-12-500044" style="justify-content: flex-end">
                                    <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                        <span>Checkout</span>
                                    </a>
                             </div>
                        </div>
                        </div>
                    
            
        

    
</div>

                <div class="purchaseOptions--option">
                    <h5 class="option-title">Journal Subscription</h5>
                    <span class="option-info mb-1">Infection and Immunity</span>
                    <div class="option-actions mt-3 pt-3">
                        <div class="option-actions--info">ASM members can purchase subscriptions to journals.</div>
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FIAI.01165-12" redirectUri="/doi/10.1128/IAI.01165-12" class="btn btn-primary btn--borderless">
                            
                            <span>Join or renew</span>
                            <i aria-hidden="true" class="mx-2 icon-open_in_new"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>




</section></div></div><div id="core-collateral-media" role="tabpanel"><header><h2><i class="icon-photo" aria-hidden="true"></i>Figures and Media</h2></header><section id="tab-figures" aria-labelledby="figures" role="tabpanel"><h3>Figures</h3></section><section id="tab-other" aria-labelledby="other" role="tabpanel"><h3>Media</h3><section>



        
        
</section></section></div><div id="core-collateral-tables" role="tabpanel"><header><h2><i class="icon-tables" aria-hidden="true"></i>Tables</h2></header></div><div id="core-collateral-share" role="tabpanel"><header><h2><i class="icon-share" aria-hidden="true"></i>Share</h2></header><h3>Share</h3><div class="section--wrapper"><section><h4>Share the article link</h4><div id="share-self"><p><a href="#" target="_blank" class="share-self__source"></a></p><button class="btn btn--inverse share-self__action"><i aria-hidden="true" class="icon-copy-o"></i><span>Copy Link</span></button><div class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section><h4>Share with email</h4><p><a href="#" target="_blank" class="btn btn-primary share-email__source"><i class="icon-copy-o"></i><span>EMAIL A COLLEAGUE</span></a></p></section><section><h4>Share on social media</h4><div class="share-buttons"><a role="link" target="_blank" href="#" class="btn btn--facebook addthis_button_facebook"><i aria-hidden="true" class="icon-facebook-2"></i><span>facebook</span></a><a role="link" target="_blank" href="#" class="btn btn--twitter addthis_button_twitter"><i aria-hidden="true" class="icon-twitter"></i><span>twitter</span></a><a role="link" target="_blank" href="#" class="btn btn--linkedin addthis_button_linkedin"><i aria-hidden="true" class="icon-linkedin"></i><span>linkedin</span></a><a role="link" target="_blank" href="#" class="btn btn--email addthis_button_email"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a></div></section></div></div></div></article><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script id="contribution-meta" type="application/json">{
    "doi": "10.1128/IAI.01165-12",
    "toc": "/toc/iai/81/4",
    "issn": "0019-9567"
}</script><div class="article__side-ad hidden">



        
        <!-- /13678947/asm_box -->
<div class="boombox-wrapper">
	<span class="boombox-label">Advertisement</span>
<div id='div-gpt-ad-1594664702378-0' style='width: 300px; height: 250px;'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664702378-0'); });
  </script>
</div>
</div>
</div>




        
        <div class="article-footer-element">
    <div class="row">
        <div class="col-12">
            <section aria-label="we recommend" class="we-recommend">
               <div id="trendmd-suggestions"></div>
<script defer src='//js.trendmd.com/trendmd.min.js' data-trendmdconfig='{"journal_id":"35290", "element":"#trendmd-suggestions"}'></script>
            </section>
        </div>
    </div>
</div>










        <div data-widget-def="ux3-layout-widget" data-widget-id="9f4096e6-614d-46c7-a704-f706dfcdc172" class="related-tabs swipe-with-arrows">
        



        
        <div class="article-footer-element">









        <div data-widget-def="UX3Tabs" data-widget-id="3c1dc793-20ba-4336-a3d4-bb86a6fc2c43" class="tab  tab--flex" data-ctrl-res="screen-sm">
        



        
        <div data-ctrl-res="screen-md" class="tab"><ul role="tablist" class="tab__nav rlist"><li role="presentation" class="tab__nav__item  active "><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" aria-selected="true" class="tab__nav__item__link active"><span>Related Articles</span></a></li><li role="presentation" class="tab__nav__item"><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" aria-selected="false" class="tab__nav__item__link"><span>Related Articles From ASM</span></a></li></ul><div class="tab__content"><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" role="tabpanel" class="tab__pane active">



        
        <section class="related-articles"><ul class="rlist related-articles__items"><li class="related-articles__item"><a href="/doi/10.1128/IAI.00541-15" class="related-articles__item__title">Correction for Blondel et al., The Type VI Secretion System Encoded in Salmonella Pathogenicity Island 19 Is Required for Salmonella enterica Serotype Gallinarum Survival within Infected Macrophages</a><ul class="meta__authors rlist--inline loa mobile-authors" title="list of authors">
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Blondel%2C+Carlos+J" title="Carlos J. Blondel">Carlos J. Blondel</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Jim%C3%A9nez%2C+Juan+C" title="Juan C. Jiménez">Juan C. Jiménez</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Leiva%2C+Lorenzo+E" title="Lorenzo E. Leiva">Lorenzo E. Leiva</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=%C3%81lvarez%2C+Sergio+A" title="Sergio A. Álvarez">Sergio A. Álvarez</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Pinto%2C+Bernardo+I" title="Bernardo I. Pinto">Bernardo I. Pinto</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Contreras%2C+Francisca" title="Francisca Contreras">Francisca Contreras</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Pezoa%2C+David" title="David Pezoa">David Pezoa</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Santiviago%2C+Carlos+A" title="Carlos A. Santiviago">Carlos A. Santiviago</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Contreras%2C+In%C3%A9s" title="Inés Contreras">Inés Contreras</a></span></li>
</ul>
</li></ul></section>
</div><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" role="tabpanel" class="tab__pane">



        
        <!--content loaded via ajax-->
<div class="asmj-org general-rss-feed-reader aog" id="80036a5d-0f41-4828-9cd1-19399fe2560a" data-rss-url="https://asm.org/rss/content/">
    <div class="rss-body"></div>
</div>
</div></div></div>

        </div>
    
</div>

        </div>
    




        
        
</div>
</main>




        
        <footer class="footer"><div class="footer__wrapper container container--lg"><div class="footer__top row"><div class="sitemap"><div class="sitemap__column sitemap__column--media col-xs-12"><div class="footer__logos">



        
        <a href="https://asm.org/" title="American Society for Microbiology logo"><img id="" alt="American Society for Microbiology logo" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title="ASM Journals logo"><img id="" alt="ASM Journals logo" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div class="footer__social-media">



        
        <div class="footer__social-media">
    <ul class="rlist--inline">
        <li><a href="https://www.facebook.com/asmfan" target="_blank" rel="noopener"><i class="icon-facebook"></i></a></li>
        <li><a href="https://twitter.com/asmicrobiology" target="_blank" rel="noopener"><i class="icon-twitter"></i></a></li>
        <li><a href="https://www.linkedin.com/companies/45861" target="_blank" rel="noopener"><i class="icon-linkedin"></i></a></li>
        <li><a href="https://www.youtube.com/channel/UCBxWzf49gVHz0Ksgw_N8brw" target="_blank" rel="noopener"><i class="icon-youtube"></i></a></li>
    </ul>
</div>
</div></div><div class="sitemap__column sitemap__column--journals col-xs-12">



        
        <a href="#" data-toggle-target-id="footerJournals"><h6 class="footer__list-title">JOURNALS</h6></a>
<ul id="footerJournals" class="rlist footer__list footer__list--columns">
	<li><a href="/journal/aac">Antimicrobial Agents and Chemotherapy</a></li>
	<li><a href="/journal/aem">Applied and Environmental Microbiology</a></li>
	<li><a href="/journal/cvi">Clinical and Vaccine Immunology</a></li>
	<li><a href="/journal/cmr">Clinical Microbiology Reviews</a></li>
	<li><a href="/journal/ecosalplus">EcoSal Plus</a></li>
	<li><a href="/journal/ec">Eukaryotic Cell</a></li>
	<li><a href="/journal/iai">Infection and Immunity</a></li>
	<li><a href="/journal/jb">Journal of Bacteriology</a></li>
	<li><a href="/journal/jcm">Journal of Clinical Microbiology</a></li>
	<li><a href="/journal/jmbe">Journal of Microbiology &amp; Biology Education</a></li>
	<li><a href="/journal/jvi">Journal of Virology</a></li>
	<li><a href="/journal/mbio">mBio</a></li>
	<li><a href="/journal/mmbr">Microbiology and Molecular Biology Reviews</a></li>
	<li><a href="/journal/mra">Microbiology Resource Announcements</a></li>
	<li><a href="/journal/spectrum">Microbiology Spectrum</a></li>
	<li><a href="/journal/mcb">Molecular and Cellular Biology</a></li>
	<li><a href="/journal/msphere">mSphere</a></li>
	<li><a href="/journal/msystems">mSystems</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--connect col-xs-12">



        
        <ul class="rlist footer__list">
    <li><h6>RESOURCES</h6></li>
    <li><a href="/about-asm-journals" target="_blank" rel="noopener">About Us</a></li>
    <li><a href="/announcements" target="_blank" rel="noopener">Announcements</a></li>
    <li><a href="/asm-author-center" target="_blank" rel="noopener">Author Center</a></li>
    <li><a href="/collection" target="_blank" rel="noopener">ASM Journal Collections</a></li>
     <li><a href="/publishing-ethics" target="_blank" rel="noopener">Publishing Ethics</a></li>
    <li><a href="/migration-information" target="_blank" rel="noopener">Migration Information</a></li>
    <li><a href="/advertising" target="_blank" rel="noopener">Advertising</a></li>
    <li><a href="https://asm.org/Membership/Home" target="_blank" rel="noopener">ASM Membership</a></li>
    <li><a href="https://asm.org/cookie-policy" target="_blank" rel="noopener">Privacy Policy</a></li>
    <li><a href="/terms" target="_blank" rel="noopener">Terms &amp; Conditions</a></li>
    <li><a href="/proprietary-rights" target="_blank" rel="noopener">Proprietary Rights Notices</a></li>
    <li><a href="/non-commercial-tdm-license" target="_blank" rel="noopener">Non-Commercial TDM License</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--news col-xs-12">



        
        <a href="#" data-toggle-target-id="footerConnected"><h6 class="footer__list-title">STAY CONNECTED</h6></a>
<ul id="footerConnected" class="rlist footer__list">
    <li><a href="/action/showPreferences?menuTab=Alerts" target="_blank" rel="noopener">Alerts</a></li>
    <li><a href="/asm-journals-rss-feeds" target="_blank" rel="noopener">RSS Feeds</a></li>
    <li><a href="/contact-us" target="_blank" rel="noopener">Contact Us</a></li>
    <li><a href=" https://connect.asm.org/ " target="_blank" rel="noopener">ASM Communities</a></li>
    
</ul>
</div><div class="sitemap__column sitemap__column--other col-xs-12 ">



        
        <h6>NEWS</h6>
<ul class="rlist footer__list">
    <li><a href="https://asm.org/newsroom" target="_blank" rel="noopener">Press Releases</a></li>
    <li><a href="https://asm.org/Browse-By-Content-Type/Policy-Statement" target="_blank" rel="noopener">Policy Statements</a></li>
    
</ul>
</div></div></div><div class="footer__bottom row"><div class="col-xs-12"><div class="footer__copyright">



        
        Copyright © 2022 American Society for Microbiology
</div></div></div></div></footer>
</div>
        </div>
    </div>



        
        

            <script src="/products/asmj/releasedAssets/js/build.lazyload.bundle-9eae693cc49bd405a4bc.js"></script>

        <script src="/products/asmj/releasedAssets/js/main.bundle-f0f6c9d7da4e7fd5b03e.js" ></script>
        

















<script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"6e7ac89e39477ea1","token":"edc08be6f1e64730992376d7e16c0615","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>
</body>
</html>
