
<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="0b517094-2416-4bf6-8eb1-4fd695800d59"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:mbio;page:string:Article/Chapter View;journal:journal:mbio;ctype:string:Journal Content;issue:issue:doi\:10.1128/mbio.2012.3.issue-6;website:website:asmj;wgroup:string:Publication Websites;article:article:doi\:10.1128/mBio.00495-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/>
<link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"></link><meta name="citation_journal_title" content="mBio"></meta><meta name="dc.Title" content="The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast Candida albicans"></meta><meta name="dc.Creator" content="Doblin Sandai"></meta><meta name="dc.Creator" content=" (Zhikang Yin)"></meta><meta name="dc.Creator" content="Laura Selway"></meta><meta name="dc.Creator" content="David Stead"></meta><meta name="dc.Creator" content="Janet Walker"></meta><meta name="dc.Creator" content="Michelle D. Leach"></meta><meta name="dc.Creator" content="Iryna Bohovych"></meta><meta name="dc.Creator" content="Iuliana V. Ene"></meta><meta name="dc.Creator" content="Stavroula Kastora"></meta><meta name="dc.Creator" content="Susan Budge"></meta><meta name="dc.Creator" content="Carol A. Munro"></meta><meta name="dc.Creator" content="Frank C. Odds"></meta><meta name="dc.Creator" content="Neil A. R. Gow"></meta><meta name="dc.Creator" content="Alistair J. P. Brown"></meta><meta name="dc.Description" content="ABSTRACT Microbes must assimilate carbon to grow and colonize their niches. Transcript profiling has suggested that Candida albicans, a major pathogen of humans, regulates its carbon assimilation i..."></meta><meta name="Description" content="ABSTRACT Microbes must assimilate carbon to grow and colonize their niches. Transcript profiling has suggested that Candida albicans, a major pathogen of humans, regulates its carbon assimilation i..."></meta><meta name="dc.Description" content="Pathogenic microbes must assimilate a range of carbon sources to grow and colonize their hosts. Current views about carbon assimilation in the pathogenic yeast Candida albicans are strongly influen..."></meta><meta name="Description" content="Pathogenic microbes must assimilate a range of carbon sources to grow and colonize their hosts. Current views about carbon assimilation in the pathogenic yeast Candida albicans are strongly influen..."></meta><meta name="dc.Publisher" content="&#xA;American Society for Microbiology&#xA;1752 N St., N.W., Washington, DC"></meta><meta name="dc.Contributor" content="John W. Taylor"></meta><meta name="dc.Date" scheme="WTN8601" content="2012-12-11"></meta><meta name="dc.Type" content="research-article"></meta><meta name="dc.Format" content="text/HTML"></meta><meta name="dc.Identifier" scheme="publisher-id" content="mBio00495-12"></meta><meta name="dc.Identifier" scheme="doi" content="10.1128/mBio.00495-12"></meta><meta name="dc.Language" content="EN"></meta><meta name="dc.Coverage" content="1752 N St., N.W., Washington, DC"></meta><meta name="dc.Rights" content="Copyright Â© 2012 Sandai et al."></meta><!-- ${citation_fulltext_world_readable:10.1128/mBio.00495-12} -->




<link rel="meta" type="application/atom+xml" href="https://doi.org/10.1128%2FmBio.00495-12"></link>
<link rel="meta" type="application/rdf+json" href="https://doi.org/10.1128%2FmBio.00495-12"></link>
<link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1128%2FmBio.00495-12"></link>



<meta charset="UTF-8">
















        <meta property="og:title" content="The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast Candida albicans | mBio"/>
    


        <meta property="og:type" content="Article"/>
    


        <meta property="og:url" content="https://journals.asm.org/doi/abs/10.1128/mBio.00495-12"/>
    

        <meta property="og:image" content="https://journals.asm.org/cms/asset/38c2aa36-eca6-463f-a637-9dc5ff5538b0/mbio.2012.3.issue-6.cover.gif" />
        <meta property="og:image:width" content="137" />
        <meta property="og:image:height" content="137" />
    


        <meta property="og:site_name" content="mBio"/>
    


        <meta property="og:description" content="Pathogenic microbes must assimilate a range of carbon sources to grow and colonize
their hosts. Current views about carbon assimilation in the pathogenic yeast Candida albicans are strongly influenced by the Saccharomyces cerevisiae paradigm in which ..."/>
    
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=0"/>
        <meta name="citation_fulltext_world_readable" content=""/>
    
<title>The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast Candida albicans | mBio</title>










            <link rel="stylesheet" type="text/css" href="/wro/l0of~article-metrics-phase3.css">
        
<link rel="stylesheet" type="text/css" href="/wro/l0of~product.css">



    
    
                <link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-6742698955b5929bab3a.css" media="all" id="build-style-sheet">
                <script>
                    function isIE() { //won't check for edge browser
                        var ua = navigator.userAgent;
                        /* MSIE used to detect old browsers and Trident used to newer ones*/
                        var is_ie = ua.indexOf("MSIE ") > -1 || ua.indexOf("Trident/") > -1;
          
                        return is_ie; 
                    }
                    
                    if(isIE()) {
                        document.write('<link rel="stylesheet" href="/products/asmj/releasedAssets/css/build-style-fallback-0e2f8eb0a5d072bb5fa3.css" media="all" />')
                        var buildCss = document.getElementById('build-style-sheet');
                        buildCss.parentNode.removeChild(buildCss);
                    }
                </script>
        <link rel="stylesheet" href="/products/asmj/releasedAssets/css/print-6742698955b5929bab3a.css"media="print">
    
        <script>
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    





<meta http-equiv="X-UA-Compatible" content="IE=edge">
<style>
:root{
	--journal-image: url('/pb-assets/images/Homepage Header Images/header-mbio-desktop-1602282261493.png');
	--journal-mobile-image: url("/pb-assets/images/Homepage Header Images/header-mbio-mobile-1605057112513.png");
	--landing-image: url("/pb-assets/images/Homepage Header Images/header-global-desktop-1603901826210.png");
	--landing-mobile-image: url("/pb-assets/images/Homepage Header Images/header-global-mobile-1605057223667.png");


}
</style>
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/Journals-Atypon-1641917642317.css">
<script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
<script>
	window.googletag = window.googletag || {cmd: []};
	googletag.cmd.push(function() {
    	googletag.defineSlot('/13678947/asm_leader-mob', [[728, 90], [320, 50]], 'div-gpt-ad-1594664599282-0').addService(googletag.pubads());
    	googletag.defineSlot('/13678947/asm_box', [300, 250], 'div-gpt-ad-1594664702378-0').addService(googletag.pubads());	
		googletag.pubads().enableSingleRequest();
		
			
				googletag.pubads().setTargeting('asmjournals', 'mbio');
			
		
		googletag.enableServices();
	});
</script>



                    <link rel="canonical" href="https://journals.asm.org/doi/10.1128/mBio.00495-12">
                





    <script src="//scholar.google.com/scholar_js/casa.js" async></script>








    
    

    
    
    
            <script type="text/javascript" src="/wro/l0of~article-metrics-phase2.js"></script>
        


    <!-- Google Tag Manager -->
    













<script>
    var dataLayer = dataLayer ||[];
    
    dataLayer.push({"event":"pageview","page_domain":"journals.asm.org","page_path":"/doi/10.1128/mBio.00495-12","page_name":"The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast Candida albicans | mBio","page_type":"article","article_title":"The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast Candida albicans","article_type":"Research Article","article_category":"Research Article","article_doi":"10.1128/mBio.00495-12","article_access":"open","article_received_date":"05 November 2012","article_accepted_date":"13 November 2012","article_published_online_date":"11 December 2012","article_issue":"volume 3, issue 6","article_issue_date":"December 2012","journal":"mBio","user_type":"Guest","basic_groups":["All users group","Unknown Users","All users no webcrawlers"],"smart_groups":["Institution- IP authentication"]});
    
</script>


    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MTLLM76');</script>
    <!-- End Google Tag Manager -->

</head>
<body class="pb-ui">


    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MTLLM76" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->




















<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>





    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">




        
        <div>



        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input type="hidden" name="requestUsername" value="requestUsername" />
                <div class="input-group">
                    <div class="label">
                        <label for="5a26c991-102b-4ded-a542-61c81c149bd5.email">Email*</label>
                    </div>
                    <input id="5a26c991-102b-4ded-a542-61c81c149bd5.email" class="email" type="text" name="email" value="" size="15" />
                </div>
                
                    <div class="username-recaptcha-ajax"></div>
                
                <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" />
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        





<div class="popup registration-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        <h2>Create a new account</h2>
        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/mBio.00495-12" />
            <div class="input-group">
                <div class="label">
                    <label for="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email">Email</label>
                </div>
                <input id="68b11fd0-c69c-4e98-9f68-782a66b3c94a.email" class="email" type="email" name="email" value="" />
            </div>
            <div class="submit">
                <input class="button submit primary" type="submit" value="Register" />
            </div></form>
        <div class="center">
            <a href="/action/showLogin" class="show-login">
                
                Returning user
            </a>
        </div>
    </div>
</div>




        
        



<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="form legacy-form">
            






<h2>Change Password </h2>
<form action="/action/changePassword" method="post"><div class="message error"></div>
    <input type="hidden" name="submit" value="submit" />
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-old">Old Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-old" class="old" type="password" name="old" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
    </div>
    <div class="input-group">
        <div class="label">
            <label for="2671405c-5390-4992-8d19-1189df090272-new">New Password</label>
        </div>
        <input id="2671405c-5390-4992-8d19-1189df090272-new" class="pass-hint new" type="password" name="new" value="" autocomplete="off" />
        <span class="password-eye-icon icon-eye hidden"></span>
        
            







    


    


    


<div class="password-strength-indicator" data-min="8" data-max="20" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
        
    </div>
    <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
        </div>
        <div class="success-template hidden">
            <h2>Password Changed Successfully</h2>
            <p>Your password has been changed</p>
        </div>
    </div>
</div>





        
        















<div class="popup login-popup hidden">
    <div class="content">
        <a href="#" class="close"><i class="icon-close_thin"></i></a>
        

        <div class="login-form-container ">
            <div class="tab tab--flex">
                <ul class="rlist tab__nav" role="tablist">
                    <li class="active" role="presentation">
                        <a class="active " id="pane-m1x01con" href="#pane-m1x01" aria-controls="pane-m1x01" role="tab"
                           data-toggle="tab" title="login" aria-selected="true" data-simple-tab-id=""
                           aria-expanded="true">
                            <span>Login</span>
                        </a>
                    </li>
                    <li role="presentation">
                        <a id="pane-m1x11con" href="#pane-m1x11" aria-controls="pane-m1x11" role="tab"
                           data-toggle="tab" title="register" aria-selected="false">
                            <span>Register</span>
                        </a>
                    </li>
                </ul>
                <ul class="rlist tab__content">
                    <li class="tab__pane active" id="pane-m1x01" aria-labelledby="pane-m1x01con" role="tabpanel">
                        




























<div class="login-form">
    
    <form action="/action/doLogin" method="post"><input type="hidden" name="id" value="016a54f6-925d-4632-a5f6-22977f0dce11" />
        





        <input type="hidden" name="redirectUri" value="/doi/10.1128/mBio.00495-12" />
    

<input type="hidden" name="loginUri" value="/doi/10.1128/mBio.00495-12" />
        
            <input type="hidden" name="popup" value="true" />
        
        
                

























    
        
    
    
        
    
    
    
        
    

        

        
    <div class="input-group">
        <div class="label ">

            
                    <label for="login">Email*</label>
                
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="login" class="login" type="text" name="login" value="" size="15" placeholder="Enter your email address" /><div class="icon-mail"></div></div>
        
                    <div class="actions">
                        
                    </div>
                
    </div>

                





















    



    
    
    <div class="input-group">
        <div class="label ">
            <label for="password">Password*</label>
            
        </div>
        
        
            
            
                
            
        
        <div class="iconable__input"><input id="password" class="password" type="password" name="password" value="" autocomplete="off" placeholder="Enter your password" /><div class="icon-lock"></div></div>
        <span class="password-eye-icon icon-eye hidden"></span>
        
                    <div class="actions reset-password">
                        
                        
                        <i class="icon-lock"></i>
                        <span>
                            Forgot password?
                            Reset it <a href="/action/requestResetPassword" class="link show-request-reset-password uppercase styled-link">here</a>
                        </span>
                    </div>
                
    </div>

            

        

        <div class="remember">
            <label class="checkbox--primary remember-me" for="016a54f6-925d-4632-a5f6-22977f0dce11-remember">
                <input id="016a54f6-925d-4632-a5f6-22977f0dce11-remember" class="cmn-toggle cmn-toggle-round-flat" type="checkbox" name="remember" value="true" />
                <span class="label-txt">Keep me logged in</span>
            </label>
        </div>
        
                <div class="submit-section">
                    <input class="button submit primary" type="submit" name="submit" value="Login" />
                    <span class="required-info">Fields with * are mandatory</span>
                </div>
            
        

























        

        <div class="login-tabs-footer">
            <div class="actions">
                
                
                <span>
                    Don&apos;t have an account? Create one <a href="/action/registration" class="uppercase styled-link">here</a>
                </span>
            </div>
        </div></form>
</div>
                    </li>
                    <li class="tab__pane" id="pane-m1x11" aria-labelledby="pane-m1x11con" role="tabpanel">
                        <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1128/mBio.00495-12" />
                            <div class="input-group">
                                <div class="label">
                                    <label for="016a54f6-925d-4632-a5f6-22977f0dce11.email">Email*</label>
                                </div>
                                <div class="iconable__input">
                                    <input id="016a54f6-925d-4632-a5f6-22977f0dce11.email" class="email" type="email" name="email" value="" placeholder="Enter your email address" />
                                    <div class="icon-mail"></div>
                                </div>
                            </div>
                            <div class="submit-section">
                                <input class="button submit primary" type="submit" value="Register" />
                                <span class="required-info">Fields with * are mandatory</span>
                            </div>
                            <div class="login-tabs-footer">
                                <div class="actions">
                                    
                                    
                                    <span>
                                        Already have an account? Login <a href="/action/showLogin" class="uppercase styled-link">here</a>
                                    </span>
                                </div>
                            </div></form>
                    </li>
                </ul>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <a href="#" class="close" tabindex="0"><i class="icon-close_thin"></i></a>
        <div class="form">
            





<p>Can't sign in? Forgot your password?</p>
<p class="sub">Enter your email address below and we will send you the reset instructions</p>
<div class="message error"></div>
<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input type="hidden" name="requestResetPassword" value="true" />
    
    <div class="input-group">
        <div class="label">
            <label for="7660a830-973e-446c-89b4-6e6ce315b3e0.email">Email*</label>
        </div>
        <input id="7660a830-973e-446c-89b4-6e6ce315b3e0.email" class="email" type="text" name="email" value="" size="15" />
    </div>
    
        








    

    
    
    
    

    
            <div class="clearfix"></div>
<div class="label required">
    <label for="captchaCode" class="prompt" style="white-space: nowrap">Retype the code from the picture:</label>
</div>
<link type="text/css" rel="stylesheet" href="/botdetectcaptcha?get=layoutStyleSheet" />

<div class="LBD_CaptchaDiv" id="sampleCaptcha" style="width: 280px !important; height: 50px !important"><!--
   --><div class="LBD_CaptchaImageDiv" id="sampleCaptcha_CaptchaImageDiv" style="width: 250px !important; height: 50px !important"><!--
   --><a target="_blank" href="http://captcha.com/java-captcha-info.html" onclick="sampleCaptcha.OnHelpLinkClick(); return sampleCaptcha.FollowHelpLink;"><img class="LBD_CaptchaImage" id="sampleCaptcha_CaptchaImage" src="/botdetectcaptcha?get=image&amp;c=sampleCaptcha&amp;t=57dc0bb191b74553a5723d331feea874" alt="CAPTCHA" /></a></div><!--
   --><div class="LBD_CaptchaIconsDiv" id="sampleCaptcha_CaptchaIconsDiv" style="width: 24px !important;"><!--
   --><a class="LBD_ReloadLink" id="sampleCaptcha_ReloadLink" href="#" onclick="sampleCaptcha.ReloadImage(); this.blur(); return false;" title="Change the CAPTCHA code"> <img class="LBD_ReloadIcon" id="sampleCaptcha_ReloadIcon" src="/botdetectcaptcha?get=ReloadIcon" alt="Change the CAPTCHA code" /></a><!--
   --><a id="sampleCaptcha_SoundLink" class="LBD_SoundLink" href="/botdetectcaptcha?get=sound&amp;c=sampleCaptcha&amp;t=57dc0bb191b74553a5723d331feea874" onclick="sampleCaptcha.PlaySound(); this.blur(); return false;" title="Speak the CAPTCHA code"> <img class="LBD_SoundIcon" id="sampleCaptcha_SoundIcon" src="/botdetectcaptcha?get=SoundIcon" alt="Speak the CAPTCHA code" /></a><!--
   --><div class="LBD_Placeholder" id="sampleCaptcha_AudioPlaceholder">&nbsp;</div><!--
   --></div>

<script type="text/javascript" src="/botdetectcaptcha?get=clientScriptInclude"></script>
<script type="text/javascript">
    //<![CDATA[
     BotDetect.Init('sampleCaptcha', '57dc0bb191b74553a5723d331feea874', 'captchaCode', true, true, true, true, 1200, 7200, 0);
    //]]>
    </script>


<script type="text/javascript">
    //<![CDATA[
      try{(function(){var bdrsn = document.createElement('script'); bdrsn.type = 'text/javascript'; bdrsn.async = true; bdrsn.src = document.location.protocol + '//remote.captcha.com/include.js?i=ATABMwExATMBMAJBMhMRP3E_PygHKT8_Pz9AbXw_Yj8i'; var fsn = document.getElementsByTagName('script')[0]; fsn.parentNode.insertBefore(bdrsn, fsn);})();} catch(err){}
    //]]>
    </script>

   <input type="hidden" id="LBD_VCID_sampleCaptcha" name="LBD_VCID_sampleCaptcha" value="57dc0bb191b74553a5723d331feea874" />
</div>

<input type="hidden" value="0" id="LBD_BackWorkaround_sampleCaptcha">
<div class="validationDiv">
    <input type="hidden" name="loginBotCaptcha" value="">
    <input id="captchaCode" type="text" name="captchaCode"/>
</div>
        

    

    

    
    <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" /></form>
            
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <p>If the address matches an existing account you will receive an email with instructions to reset your password.</p>
            <div class="center">
                <a href="#" class="cancel">Close</a>
            </div>
        </div>
    </div>
</div>




        
        




<div class="top-drawer verify-phone-drawer hidden">
    <div class="content">
        <div class="form">
            <h2>Verify Phone</h2>
            <form action="/action/verifyPhoneNumber" method="post"><input type="hidden" name="submit" value="submit" />
                <div class="message error"></div>
                <div class="input-group">
                    <div class="label">
                        <label for="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code">Enter the verification code</label>
                    </div>
                    <input id="6f5823be-1bcb-45d4-9e6e-3734bacc9f87-code" class="verificationCode" type="text" name="verificationCode" value="" size="15" />
                </div>
                <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
            <div class="center">
                <a href="#" class="cancel">Cancel</a>
            </div>
        </div>
        <div class="success-template hidden">
            <h2>Congrats!</h2>
            <p>Your Phone has been verified</p>
            <div class="center">
                <a href="/action/showPreferences" class="button primary submit">close</a>
            </div>
        </div>
    </div>
</div>





        
        <div id="search-overlay" class="header__search-overlay search-overlay js--hidden"><div class="search-overlay__close"><a href="#" title="Close search panel" data-hide-target-id="search-overlay" class="search-overlay__close__btn"><span>CLOSE </span><i class="icon-close"></i></a></div><div class="container">



        
        <div class="quick-search quick-search--all"><div class="full-width quickSearchFormContainer"><div class="tab"><div class="quick-search__searchbox"><ul role="tablist" class="rlist tab__nav dropblock--tab"><li role="presentation" class="tab__nav__item active"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-controls="pane-0" role="tab" data-toggle="tab" title="This Journal" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" aria-expanded="false" class="tab__nav__item__link">This Journal</a></li><li role="presentation" class="tab__nav__item"><a href="#pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-controls="pane-1" role="tab" data-toggle="tab" title="Anywhere" id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" aria-expanded="false" class="tab__nav__item__link">Anywhere</a></li></ul><div class="quick-search__search-items tab__content"><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856910" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856910-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-0 option-journal"><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856910" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Search here within this Journal" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/><input type="hidden" name="SeriesKey" value="mbio"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div><div id="pane-2037e191-2746-4af8-ae16-bb4d0b3856911" aria-labelledby="pane-2037e191-2746-4af8-ae16-bb4d0b3856911-con" role="tabpanel" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" class="quick-search__form"><div class="quick-search__input input-group option-1 "><label for="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input autocomplete="off" type="search" id="AllField2037e191-2746-4af8-ae16-bb4d0b3856911" name="AllField" aria-label="Enter words / phrases / DOI / ISBN / keywords / authors / etc" placeholder="Enter words / phrases / DOI / ISBN / keywords / authors / etc" data-display-labels="true" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="autocomplete"/></div><button id="quick-search-btn" type="submit" title="Search" class="btn btn-primary btn--borderless quick-search__button"><span class="sr-only">Search</span><i aria-hidden="true" class="icon-search"></i><span>Search</span></button></form></div>



        
        <div class="quick-search__links">
    <a href="/search/advanced" class="quick-search__advanced-link">Advanced search</a>
</div>
</div></div></div></div></div>
<div class="search-overlay__autocomplete"><div id="autocompleted-terms"></div></div><div class="search-overlay__suggestions"><div class="search-overlay__suggestions__title">Suggested Terms:</div><ul class="rlist search-overlay__suggestions__list">



        
        <li><a href="/asm-author-center" class="btn btn-secondary">author information</a></li>
<li><a href="/migration-information" class="btn btn-secondary">migration information</a></li>
<li><a href="/publication-fees" class="btn btn-secondary">publication fees</a></li>
<li><a href="/asm-covid-19" class="btn btn-secondary">COVID-19 collection</a></li>
<li><a href="/collection" class="btn btn-secondary">asm journals collections</a></li>
</ul></div></div></div><header class="header header--journal header--journal-page"><div class="header__row-1">



        
        <div class="journals-menu">
	<a data-toggle-target-id="journals-menu-content" href="#" title="Journals Menu" class="journals-menu__trigger js--target-hidden">
		<i class="icon-dehaze"></i>
		<i class="icon-close"></i>
	</a>
	<div id="journals-menu-content" class="journals-menu__content js--hidden" aria-expanded="true">
		<a href="#" data-hide-target-id="journals-menu-content" class="journals-menu__back">
			<i class="icon-arrow_left"></i>
			<span>BACK</span>
		</a>
		<ul class="rlist journals-menu__list">
			<li><a href="/journal/aac"><img src="/pb-assets/images/thumbnails/thumb-aac-1602006917653.jpg" alt="Antimicrobial Agents and Chemotherapy"><span>Antimicrobial Agents and Chemotherapy</span></a></li>
			<li><a href="/journal/aem"><img src="/pb-assets/images/thumbnails/thumb-aem-1602006917677.jpg" alt="Applied and Environmental Microbiology"><span>Applied and Environmental Microbiology</span></a></li>
			<li><a href="/journal/cvi"><img src="/pb-assets/images/thumbnails/thumb-cvi-1602006922190.jpg" alt="Clinical and Vaccine Immunology"><span>Clinical and Vaccine Immunology</span></a></li>
			<li><a href="/journal/cmr"><img src="/pb-assets/images/thumbnails/thumb-cmr-1602006918000.jpg" alt="Clinical Microbiology Reviews"><span>Clinical Microbiology Reviews</span></a></li>
			<li><a href="/journal/ecosalplus"><img src="/pb-assets/images/thumbnails/thumb-ecosalplus-1602678711627.jpg" alt="EcoSal Plus"><span>EcoSal Plus</span></a></li>
			<li><a href="/journal/ec"><img src="/pb-assets/images/thumbnails/thumb-ec-1602006922537.jpg" alt="Eukaryotic Cell"><span>Eukaryotic Cell</span></a></li>
			<li><a href="/journal/iai"><img src="/pb-assets/images/thumbnails/thumb-iai-1602006922193.jpg" alt="Infection and Immunity"><span>Infection and Immunity</span></a></li>
			<li><a href="/journal/jb"><img src="/pb-assets/images/thumbnails/thumb-jb-1602006930407.jpg" alt="Journal of Bacteriology"><span>Journal of Bacteriology</span></a></li>
			<li><a href="/journal/jcm"><img src="/pb-assets/images/thumbnails/thumb-jcm-1602006930870.jpg" alt="Journal of Clinical Microbiology"><span>Journal of Clinical Microbiology</span></a></li>
			<li><a href="/journal/jmbe"><img src="/pb-assets/images/thumbnails/thumb-jmbe-1606273579503.jpg" alt="Journal of Microbiology &amp; Biology Education"><span>Journal of Microbiology &amp; Biology Education</span></a></li>
			<li><a href="/journal/jvi"><img src="/pb-assets/images/thumbnails/thumb-jvi-1602006937397.jpg" alt="Journal of Virology"><span>Journal of Virology</span></a></li>
			<li><a href="/journal/mbio"><img src="/pb-assets/images/thumbnails/thumb-mbio-1602006937397.jpg" alt="mBio"><span>mBio</span></a></li>
			<li><a href="/journal/mmbr"><img src="/pb-assets/images/thumbnails/thumb-mmbr-1602006945363.jpg" alt="Microbiology and Molecular Biology Reviews"><span>Microbiology and Molecular Biology Reviews</span></a></li>
			<li><a href="/journal/mra"><img src="/pb-assets/images/thumbnails/thumb-mra-1602006946427.jpg" alt="Microbiology Resource Announcements"><span>Microbiology Resource Announcements</span></a></li>
			<li><a href="/journal/spectrum"><img src="/pb-assets/images/thumbnails/thumb-spectrum-1602006955410.jpg" alt="Microbiology Spectrum"><span>Microbiology Spectrum</span></a></li>
			<li><a href="/journal/mcb"><img src="/pb-assets/images/thumbnails/thumb-mcb-1602006937413.jpg" alt="Molecular and Cellular Biology"><span>Molecular and Cellular Biology</span></a></li>
			<li><a href="/journal/msphere"><img src="/pb-assets/images/thumbnails/thumb-msphere-1602006946427.jpg" alt="mSphere"><span>mSphere</span></a></li>
			<li><a href="/journal/msystems"><img src="/pb-assets/images/thumbnails/thumb-msystems-1602006955407.jpg" alt="mSystems"><span>mSystems</span></a></li>
		</ul>
	</div>
</div>
<div class="header__logos">



        
        <a href="https://asm.org/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title=""><img id="" alt="" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div data-truncate="1" class="header__feed mx-3 js--truncate"></div><div class="header__attribution"></div><div class="header__cart">



        
        















<div class="eCommerceCartIndicatorWidget">
    
            <a href="/action/showCart?FlowID=1&amp;aria-label=Shopping+Cart" id="cartLabel" class="quick-menu__item" aria-label="Shopping Cart">
                <div class="cartLabel" title="Cart">
                    <i class="icon-shopping_cart" aria-hidden="true"></i>
                    <span class="filter-count filter-count-bubble shopping-cart hidden"
                          data-id="cart-size">0</span>
                        
                </div>
            </a>
        
</div>
</div><div class="header__login">



        
        




    








        




    













<div class="user-login-bar">
    <div title="Sign in" class="loginBar user-login-bar">
        <div class="user-login-bar">
            <div class="loginBar">
                <div class="loginBar__login">
                    <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FmBio.00495-12&amp;data-toggle-target-id=loginOptions" class="btn btn-primary btn--light btn--darker-content btn--borderless" redirectUri="/doi/10.1128/mBio.00495-12" data-toggle-target-id="loginOptions">
                        Login
                    </a>
                    <div id="loginOptions" class="loginBar__login__options">
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FmBio.00495-12">
                            
                            <img src="/specs/products/asmj/releasedAssets/images/logo-textless.svg" class="mr-2">
                            <span>ASM Member Login</span>
                            <i class="icon-open_in_new" aria-hidden="true"></i>
                        </a>
                        <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1128%2FmBio.00495-12">
                            <i class="ml-1 mr-2 icon-account_circle"></i>
                            Non-Member Login
                        </a>
                        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1128%2FmBio.00495-12">
                            <i class="ml-1 mr-2 icon-institution"></i>
                            Institutional Login
                        </a>
                        <a href="/action/registration?redirectUri=%2Fdoi%2F10.1128%2FmBio.00495-12" class="btn btn-secondary btn--inverse btn--borderless btn--darker-content btn-register" title="Register">
                            Register
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>



    

</div>



        
        <a href="#" data-show-target-id="search-overlay" class="hidden-md hidden-sm hidden-xs search-trigger js--target-shown"><i title="Open Search Panel" class="icon-search"></i></a>
</div><div class="header__row-2"><a data-toggle-target-id="responsive-menu" href="#" title="Journals Menu" class="header__mobile-trigger"><i class="icon-dehaze"></i><i class="icon-close"></i></a><div class="header__logos">



        
        <a href="/journal/mbio" title="mBio logo"><img id="" alt="mBio logo" src="/pb-assets/images/pub-nav/MBIO-logo-1601672967533.png"/></a>
</div><div class="hidden-sm hidden-xs hidden-md">



        
        <a href="#afterNav-xob" class="sr-only-focusable">Skip main navigation</a><nav class="main-nav"><ul id="menubar" role="menubar" class="menubar rlist--inline"><li role="menuitem" aria-label="JOURNAL HOME" id="menu-item-pub_nav-0" class="menu-item is--current"><a href="/journal/mbio"><span>JOURNAL HOME</span></a></li><li role="menuitem" aria-haspopup="true" aria-label="ARTICLES" id="menu-item-pub_nav-1" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-1" data-toggle="dropdown" class="dropdown__toggle"><span>ARTICLES</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-1" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="LATEST ARTICLES" class="menu-item is--current"><a href="/toc/mbio/0/0"><span>LATEST ARTICLES</span></a></li><li role="menuitem" tabindex="-1" aria-label="CURRENT ISSUE" class="menu-item is--current"><a href="/toc/mbio/current"><span>CURRENT ISSUE</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="COLLECTIONS" class="menu-item is--current"><a href="/journal/mbio/collections"><span>COLLECTIONS</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARCHIVE" class="menu-item is--current"><a href="/loi/mbio"><span>ARCHIVE</span></a></li></ul></li><li role="menuitem" id="menu-item-pub_nav-2" class="menu-item">



        
        <li role="menuitem" aria-haspopup="true" aria-label="FOR AUTHORS" id="menu-item-pub_nav-custom" class="dropdown menu-parent is--current">
	<a href="#" id="pub_nav-custom" data-toggle="dropdown" class="dropdown__toggle"><span>FOR AUTHORS</span></a>
	<ul aria-labelledby="pub_nav-custom" role="menu" class="rlist dropdown__menu">
		<li role="menuitem" tabindex="-1" aria-label="SUBMITTING" class="menu-item is--current">
			<a href="/submitting?journal=mbio"><span>SUBMITTING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="WRITING YOUR PAPER" class="menu-item is--current">
			<a href="/writing-your-paper?journal=mbio"><span>WRITING YOUR PAPER</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="TRANSFERRING" class="menu-item is--current">
			<a href="/transferring?journal=mbio"><span>TRANSFERRING</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="ACCEPTED PAPERS" class="menu-item is--current">
			<a href="/accepted-papers?journal=mbio"><span>ACCEPTED PAPERS</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="EDITORIAL POLICIES" class="menu-item is--current">
			<a href="/editorial-policies?journal=mbio"><span>EDITORIAL POLICIES</span></a>
		</li>
		<li role="menuitem" tabindex="-1" aria-label="PUBLISHING ETHICS" class="menu-item is--current">
			<a href="/publishing-ethics?journal=mbio"><span>PUBLISHING ETHICS</span></a>
		</li>
	</ul>
</li>

</li><li role="menuitem" aria-haspopup="true" aria-label="ABOUT THE JOURNAL" id="menu-item-pub_nav-3" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-2" data-toggle="dropdown" class="dropdown__toggle"><span>ABOUT THE JOURNAL</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-2" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="ABOUT MBIO" class="menu-item is--current"><a href="/journal/mbio/about"><span>ABOUT MBIO</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMIT" class="menu-item is--current"><a href="/journal/mbio/submit"><span>SUBMIT</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="SCOPE" class="menu-item is--current"><a href="/journal/mbio/scope"><span>SCOPE</span></a></li><li role="menuitem" tabindex="-1" aria-label="BOARD OF EDITORS" class="menu-item is--current"><a href="/journal/mbio/board-editors"><span>BOARD OF EDITORS</span></a></li><li role="menuitem" tabindex="-1" aria-label="ARTICLE TYPES" class="menu-item is--current"><a href="/journal/mbio/article-types"><span>ARTICLE TYPES</span></a></li><li role="menuitem" tabindex="-1" aria-label="SUBMISSION AND REVIEW PROCESS" class="menu-item is--current"><a href="/journal/mbio/submission-review-process"><span>SUBMISSION AND REVIEW PROCESS</span></a></li><li role="menuitem" tabindex="-1" aria-label="AAM FELLOWS" class="menu-item is--current"><a href="/journal/mbio/fellows"><span>AAM FELLOWS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FOR REVIEWERS" class="menu-item is--current"><a href="/reviewers"><span>FOR REVIEWERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="FAQ" class="menu-item is--current"><a href="/journal/mbio/faq"><span>FAQ</span></a></li><li role="menuitem" tabindex="-1" aria-label="CONTACT US" class="menu-item is--current"><a href="/journal/mbio/contact-us"><span>CONTACT US</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="SUBSCRIBE" id="menu-item-pub_nav-4" class="dropdown menu-parent is--current"><a href="#" id="pub_nav-3" data-toggle="dropdown" class="dropdown__toggle"><span>SUBSCRIBE</span><i aria-hidden="true" class="icon-triangle-bottom"></i></a><ul aria-labelledby="pub_nav-3" role="menu" class="rlist dropdown__menu"><li role="menuitem" tabindex="-1" aria-label="MEMBERS" class="menu-item is--current"><a href="/members"><span>MEMBERS</span></a></li><li role="menuitem" tabindex="-1" aria-label="INSTITUTIONS" class="menu-item is--current"><a href="/institutions"><span>INSTITUTIONS</span></a></li><li role="menuitem" tabindex="-1" class="menu-item"></li><li role="menuitem" tabindex="-1" aria-label="ALERTS" class="menu-item is--current"><a href="/action/showPreferences?menuTab=Alerts"><span>ALERTS</span></a></li><li role="menuitem" tabindex="-1" aria-label="RSS" class="menu-item is--current"><a href="/asm-journals-rss-feeds"><span>RSS</span></a></li></ul></li></ul><span id="afterNav-xob" tabindex="-1"></span></nav>
</div><a href="#" data-show-target-id="search-overlay" class="hidden-lg search-trigger"><i title="Open Search Panel" class="icon-search"></i></a></div></header>




        
        <main class="content article-page">



        
        <div>









        <div data-widget-def="ux3-layout-widget" data-widget-id="62435c6d-56e5-45d8-8726-1c5a8298a1a6" class="article-container">
        



        
        <div class="mb-3">



        
        <ol aria-label="Breadcrumb" role="navigation" vocab="https://schema.org/" typeof="BreadcrumbList" class="breadcrumbs rlist--inline"><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="https://journals.asm.org/"><span property="name">ASM Journals</span></a><meta property="position" content="1"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/journal/mbio"><span property="name">mBio</span></a><meta property="position" content="2"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/toc/mbio/3/6"><span property="name">Vol. 3, No. 6</span></a><meta property="position" content="3"/></li><li property="itemListElement" typeof="ListItem"><a property="item" typeof="WebPage" href="/doi/10.1128/mBio.00495-12" aria-current="page" class="breadcrumbs__current"><span property="name">The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast <named-content content-type='genus-species'>Candida albicans</named-content></span></a><meta property="position" content="4"/></li></ol>










        <div data-widget-def="general-html-asset" data-widget-id="b28fb475-91bf-4257-b86f-8120484c7481" class="container--center-content">
        



        
        <!-- /13678947/asm_leader-mob -->
<div class="leaderboard-wrapper">
<span class="leaderboard-label">Advertisement</span>
<div id='div-gpt-ad-1594664599282-0'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664599282-0'); });
  </script>
</div>
</div>

        </div>
    
</div>

        </div>
    




        
        <link rel="stylesheet" type="text/css" href="/specs/products/asmj/releasedAssets/css/build-article.css"/><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle"><section id="frontmatter" data-extent="frontmatter"><header><div class="core-container"><div data-article-access="free" data-article-access-type="limited" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__access meta-panel__access--open"><i aria-hidden="true" class="icon-open_access"></i></div><div class="meta-panel__editor-award">Editor's Pick</div><div class="meta-panel__type">Research Article</div><div class="meta-panel__publication-data">11 December 2012</div></div><div class="meta-panel__right-content"><div class="meta-panel__share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share__block share__inline-links"><span class="sr-only">Share on</span><ul class="rlist--inline addthis addthis_toolbox addthis_default_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper  icon-facebook"></i></a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i></a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper  icon-linkedin"></i></a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper  icon-mail"></i></a></li></ul></div>
</div></div></div><h1 property="name">The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast <span class="named-content" data-type="genus-species">Candida albicans</span></h1><div class="core-relations my-3"><div class="relation mb-3"><div class="relation--head"><i class="icon-info"></i><div>This article has been corrected.</div><a href="/doi/10.1128/mBio.02489-14">VIEW CORRECTION</a></div></div></div><div class="contributors"><span class="authors"><span class="heading">Authors</span>: <span role="list"><span property="author" typeof="Person" role="listitem"><a href="#con1"><span property="givenName">Doblin</span> <span property="familyName">Sandai</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con2"> (<span property="givenName">Zhikang</span> <span property="familyName">Yin</span>)</a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con3"><span property="givenName">Laura</span> <span property="familyName">Selway</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con4"><span property="givenName">David</span> <span property="familyName">Stead</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con5"><span property="givenName">Janet</span> <span property="familyName">Walker</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con6"><span property="givenName">Michelle D.</span> <span property="familyName">Leach</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con7"><span property="givenName">Iryna</span> <span property="familyName">Bohovych</span></a></span><span data-control-for="ui-revealable" aria-hidden="true">, <span>â¦ <span data-action-for="ui-revealable" data-action="show">Show All</span> â¦</span> </span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con8"><span property="givenName">Iuliana V.</span> <span property="familyName">Ene</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con9"><span property="givenName">Stavroula</span> <span property="familyName">Kastora</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con10"><span property="givenName">Susan</span> <span property="familyName">Budge</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con11"><span property="givenName">Carol A.</span> <span property="familyName">Munro</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con12"><span property="givenName">Frank C.</span> <span property="familyName">Odds</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con13"><span property="givenName">Neil A. R.</span> <span property="familyName">Gow</span></a></span></span>, <span property="author" typeof="Person" role="listitem"><a href="#con14"><span property="givenName">Alistair J. P.</span> <span property="familyName">Brown</span></a></span><span class="ui-hidden" data-control-for="ui-revealable" aria-hidden="true"> <span data-action-for="ui-revealable" data-action="hide">Show Fewer</span></span></span></span></div><div class="self-citation"><span class="heading">DOI</span>: <a href="https://doi.org/10.1128/mBio.00495-12" property="sameAs">https://doi.org/10.1128/mBio.00495-12</a><span>&nbsp;&#8226;&nbsp;</span><span class="crossmark"><a data-target="crossmark" href="#" data-doi="10.1128/mBio.00495-12" class="crossmark__link"><img alt="Check for updates on crossmark" src="/specs/products/asmj/releasedAssets/images/cross-mark.png" class="crossmark__logo"/></a></span></div><div class="info-panel"><div class="info-panel__metrics info-panel__item">



        
        







        <div class="toolbar-metric-container">
            <input class="toolbar-metric__data" type="hidden" name="json" value='{"data":{"menuSections":[{"total":"319","id":"downloads","title":"downloads"},{"total":"64","id":"citations","title":"citations"}],"doi":"10.1128/mBio.00495-12"},"thisWidget":{"properties":{"identifier":"","headingLvl":"h2","wrapperClass":"","citationMetricId":"2","downloadMetricId":"1","headingClass":"","ux3/template":"","wrapperTag":"div","title":"","linkToFullPage":""}}}'>
        </div>
    

</div><div class="info-panel__favorite info-panel__item">



        
        <div class="article-tools"><a href="/action/addCitationAlert?doi=10.1128%2FmBio.00495-12" target="_blank" title="Track Citations" class="article-tools__citation btn btn--slim btn-secondary btn--inverse btn--transparent"><i class="icon-alert-bell"></i></a></div>
</div><div class="info-panel__alerts info-panel__item"></div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" title="View Citations" class="btn btn--slim btn--inverse"><i aria-hidden="true" class="icon-format_quote"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://journals.asm.org/doi/full/10.1128/mBio.00495-12" class="btn btn--slim btn--inverse"><span>Full Text</span></a><a href="/doi/reader/10.1128/mBio.00495-12" class="btn btn--slim btn--pdf"><span>PDF/EPUB</span></a></div></div></div></header><nav data-core-nav="article"><header><div class="core-self-citation">



        
        <a href="/journal/mbio" class="sections-navbar__title">mBio</a>
<div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">3</span></span><span class="delimiter">, </span><span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">6</span></span><br /><span property="datePublished">31 December 2012</span></div></div><nav></nav></header><ul xmlns="http://www.w3.org/1999/xhtml"><li><a href="#abstract">ABSTRACT</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">RESULTS</a></li><li><a href="#sec-3">DISCUSSION</a></li><li><a href="#sec-4">MATERIALS AND METHODS</a></li><li><a href="#acknowledgments">ACKNOWLEDGMENTS</a></li><li><a href="#supplementary-materials">Supplemental Material</a></li><li><a href="#bibliography">REFERENCES</a></li></ul><footer></footer></nav><div id="abstracts"><div class="core-container"><section id="abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">ABSTRACT</h2><div role="paragraph">Microbes must assimilate carbon to grow and colonize their niches. Transcript profiling has suggested that <span class="named-content" data-type="genus-species">Candida albicans</span>, a major pathogen of humans, regulates its carbon assimilation in an analogous fashion to the model yeast <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>, repressing metabolic pathways required for the use of alterative nonpreferred carbon sources when sugars are available. However, we show that there is significant dislocation between the proteome and transcriptome in <span class="named-content" data-type="genus-species">C. albicans</span>. Glucose triggers the degradation of the <i>ICL1</i> and <i>PCK1</i> transcripts in <span class="named-content" data-type="genus-species">C. albicans</span>, yet isocitrate lyase (Icl1) and phosphoenolpyruvate carboxykinase (Pck1) are stable and are retained. Indeed, numerous enzymes required for the assimilation of carboxylic and fatty acids are not degraded in response to glucose. However, when expressed in <span class="named-content" data-type="genus-species">C. albicans</span>, <span class="named-content" data-type="genus-species">S. cerevisiae</span> Icl1 (ScIcl1) is subjected to glucose-accelerated degradation, indicating that like <span class="named-content" data-type="genus-species">S. cerevisiae</span>, this pathogen has the molecular apparatus required to execute ubiquitin-dependent catabolite inactivation. <span class="named-content" data-type="genus-species">C. albicans</span> Icl1 (CaIcl1) lacks analogous ubiquitination sites and is stable under these conditions, but the addition of a ubiquitination site programs glucose-accelerated degradation of CaIcl1. Also, catabolite inactivation is slowed in <i>C. albicans ubi4</i> cells. Ubiquitination sites are present in gluconeogenic and glyoxylate cycle enzymes from <span class="named-content" data-type="genus-species">S. cerevisiae</span> but absent from their <span class="named-content" data-type="genus-species">C. albicans</span> homologues. We conclude that evolutionary rewiring of ubiquitination targets has meant that following glucose exposure, <span class="named-content" data-type="genus-species">C. albicans</span> retains key metabolic functions, allowing it to continue to assimilate alternative carbon sources. This metabolic flexibility may be critical during infection, facilitating the rapid colonization of dynamic host niches containing complex arrays of nutrients.</div><div role="paragraph"><b>IMPORTANCE</b> Pathogenic microbes must assimilate a range of carbon sources to grow and colonize their hosts. Current views about carbon assimilation in the pathogenic yeast <span class="named-content" data-type="genus-species">Candida albicans</span> are strongly influenced by the <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span> paradigm in which cells faced with choices of nutrients first use energetically favorable sugars, degrading enzymes required for the assimilation of less favorable alternative carbon sources. We show that this is not the case in <span class="named-content" data-type="genus-species">C. albicans</span> because there has been significant evolutionary rewiring of the molecular signals that promote enzyme degradation in response to glucose. As a result, this major pathogen of humans retains enzymes required for the utilization of physiologically relevant carbon sources such as lactic acid and fatty acids, allowing it to continue to use these host nutrients even when glucose is available. This phenomenon probably enhances efficient colonization of host niches where sugars are only transiently available.</div></section></div></div></section><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1" data-type="intro"><h2>Introduction</h2><div role="paragraph">Carbon assimilation is fundamentally important for all organisms. When faced with choices of carbon source, microbes often assimilate preferred carbon sources to support the first phase of growth and then, having exhausted these carbon sources, turn to alternative energetically less favorable carbon sources to drive subsequent phases of diauxic growth. This selective carbon utilization is reflected in the differential regulation of genes and enzymes that support the uptake and catabolism of specific carbon sources. The <span class="named-content" data-type="genus-species">Escherichia coli</span> <i>lac</i> operon provides a classic example of this, mediating lactose utilization only after the preferred carbon source, glucose, is exhausted (<a href="#B1" role="doc-biblioref" data-xml-rid="B1">1</a>, <a href="#B2" role="doc-biblioref" data-xml-rid="B2">2</a>). In <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>, glucose limits the assimilation of alternative carbon sources and represses respiration under aerobic conditions, promoting fermentative metabolism (the Crabtree effect [<a href="#B3" role="doc-biblioref" data-xml-rid="B3">3</a>, <a href="#B4" role="doc-biblioref" data-xml-rid="B4">4</a>]).</div><div role="paragraph"><span class="named-content" data-type="genus-species">S. cerevisiae</span> is exquisitely sensitive to sugars: even glucose concentrations as low as 0.01% trigger the major redirection of cellular resources (<a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>, <a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>). Glucose exerts its dramatic effects upon <span class="named-content" data-type="genus-species">S. cerevisiae</span> physiology via signaling pathways that include the glucose repression (Snf1 AMP kinase) pathway, cyclic AMP (cAMP)-protein kinase A signaling, and the sugar receptor repressor (Snf3-Rgt2) pathway (for reviews, see references <a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a> to <a href="#B12" role="doc-biblioref" data-xml-rid="B12">12</a>). The cAMP-protein kinase A pathway activates ribosome biogenesis and downregulates stress responses in response to glucose (<a href="#B13" role="doc-biblioref" data-xml-rid="B13">13</a><a href="#B14" role="doc-biblioref" data-xml-rid="B14 B15">â</a><a href="#B16" role="doc-biblioref" data-xml-rid="B16">16</a>). The sugar receptor repressor pathway modulates the expression of hexose transporters (<a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>, <a href="#B17" role="doc-biblioref" data-xml-rid="B17">17</a>, <a href="#B18" role="doc-biblioref" data-xml-rid="B18">18</a>). Meanwhile, the glucose repression pathway represses the transcription of genes involved in the assimilation of alternative carbon sources, such as galactose, ethanol, and fatty acids (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a>, <a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>).</div><div role="paragraph">These sugar signaling mechanisms comprise an interlinked network rather than parallel signaling pathways (<a href="#B8" role="doc-biblioref" data-xml-rid="B8">8</a>, <a href="#B10" role="doc-biblioref" data-xml-rid="B10">10</a>, <a href="#B12" role="doc-biblioref" data-xml-rid="B12">12</a>, <a href="#B18" role="doc-biblioref" data-xml-rid="B18">18</a>, <a href="#B19" role="doc-biblioref" data-xml-rid="B19">19</a>). Furthermore, glucose regulation is imposed at multiple levels in <span class="named-content" data-type="genus-species">S. cerevisiae</span>. They include transcriptional (<a href="#B7" role="doc-biblioref" data-xml-rid="B7">7</a><a href="#B8" role="doc-biblioref" data-xml-rid="B8">â</a><a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>), posttranscriptional (<a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>, <a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>, <a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a>, <a href="#B21" role="doc-biblioref" data-xml-rid="B21">21</a>), translational (<a href="#B22" role="doc-biblioref" data-xml-rid="B22">22</a>), and posttranslational mechanisms (<a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a><a href="#B24" role="doc-biblioref" data-xml-rid="B24 B25 B26 B27">â</a><a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>). For example, glucose represses the transcription of genes encoding the gluconeogenic enzymes fructose 1,6-bisphosphatase (<i>FBP1</i>) and phosphoenolpyruvate carboxykinase (<i>PCK1</i>) (<a href="#B29" role="doc-biblioref" data-xml-rid="B29">29</a>, <a href="#B30" role="doc-biblioref" data-xml-rid="B30">30</a>) and triggers the accelerated degradation of the <i>FBP1</i> and <i>PCK1</i> mRNAs (<a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>, <a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a>). Furthermore, glucose triggers the phosphorylation and inactivation of fructose 1,6-bisphosphatase (Fbp1) as well as its proteolytic degradation (<a href="#B31" role="doc-biblioref" data-xml-rid="B31">31</a>). The accelerated degradation of Fbp1 is mediated by vacuolar and ubiquitin-dependent mechanisms (<a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>). Following glucose addition, Fbp1 is ubiquitinated and degraded via the proteasome (<a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>, <a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>, <a href="#B32" role="doc-biblioref" data-xml-rid="B32">32</a>). Like Fbp1, the glyoxylate cycle enzyme, isocitrate lyase (Icl1) is also subject to catabolite inactivation in <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>). This tight control of central carbon metabolism is thought to reflect the evolution of this model yeast under conditions of âfeast or famineâ and to enhance the competitiveness of <span class="named-content" data-type="genus-species">S. cerevisiae</span> in sugar-rich niches containing complex microflora (<a href="#B9" role="doc-biblioref" data-xml-rid="B9">9</a>).</div><div role="paragraph"><span class="named-content" data-type="genus-species">S. cerevisiae</span> is often viewed as a paradigm for other yeasts (<a href="#B33" role="doc-biblioref" data-xml-rid="B33">33</a>). However, yeast species inhabit diverse niches and have evolved under contrasting selective pressures leading to differing strategies of carbon utilization (<a href="#B34" role="doc-biblioref" data-xml-rid="B34">34</a>). For example, the major systemic fungal pathogen of humans, <span class="named-content" data-type="genus-species">Candida albicans</span>, inhabits niches that contain complex mixtures of carbon sources. During commensalism and mucosal infection, <span class="named-content" data-type="genus-species">C. albicans</span> colonizes the oral cavity and the gastrointestinal and urogenital tracts, and during systemic infection, this pathogen can thrive in the bloodstream and most internal organs (<a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>, <a href="#B36" role="doc-biblioref" data-xml-rid="B36">36</a>). Few of these niches are rich in sugar. Blood glucose levels range from 4 to 7 mM (0.07 to 0.13%), whereas concentrations of about 111 mM (2%) are often used to impose glucose repression in <i>in vitro</i> experiments. Many niches are rich in alternative carbon sources, such as lactate, fatty acids, and amino acids. For example, lactic acid is found in ingested foods, is produced by host metabolic activity and by lactic acid bacteria in the gastrointestinal and urogenital tracts (<a href="#B37" role="doc-biblioref" data-xml-rid="B37">37</a>), and is essential for the proliferation of <span class="named-content" data-type="genus-species">Candida glabrata</span> in the intestinal tract (<a href="#B38" role="doc-biblioref" data-xml-rid="B38">38</a>). Also, glyoxylate cycle and fatty acid Î²-oxidation genes are expressed in the host and are required for the full virulence of <span class="named-content" data-type="genus-species">C. albicans</span> during systemic infections (<a href="#B39" role="doc-biblioref" data-xml-rid="B39">39</a><a href="#B40" role="doc-biblioref" data-xml-rid="B40 B41">â</a><a href="#B42" role="doc-biblioref" data-xml-rid="B42">42</a>).</div><div role="paragraph">Although <span class="named-content" data-type="genus-species">C. albicans</span> occupies contrasting niches from <span class="named-content" data-type="genus-species">S. cerevisiae</span>, analogous sugar signaling pathways are thought to exist in these yeasts (<a href="#B33" role="doc-biblioref" data-xml-rid="B33">33</a>). Although there has been considerable rewiring of the regulatory circuitry that controls carbon metabolism (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a><a href="#B44" role="doc-biblioref" data-xml-rid="B44">â</a><a href="#B45" role="doc-biblioref" data-xml-rid="B45">45</a>), microarray experiments have revealed that <span class="named-content" data-type="genus-species">C. albicans</span> genes involved in the assimilation of alternative carbon sources are exquisitely sensitive to low concentrations of glucose (<a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>), like their orthologs in <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>). However, unlike <span class="named-content" data-type="genus-species">S. cerevisiae</span>, <span class="named-content" data-type="genus-species">C. albicans</span> continues to respire in the presence of glucose, leading to its classification as a Crabtree-negative yeast (<a href="#B48" role="doc-biblioref" data-xml-rid="B48">48</a>).</div><div role="paragraph">These observations create an interesting conundrum relating to the carbon assimilation and pathogenicity of <span class="named-content" data-type="genus-species">C. albicans</span>: how can this yeast rapidly colonize niches that contain small amounts of glucose if many of the metabolic genes required for efficient growth in these niches are repressed by glucose? Could glucose regulation be relaxed at a posttranscriptional level in <span class="named-content" data-type="genus-species">C. albicans</span>, thereby facilitating simultaneous assimilation of sugars and alternative carbon sources <i>in vivo</i>? We have addressed these questions first by performing proteomic screens to identify proteins that are regulated in response to physiologically relevant carbon sources, revealing that gluconeogenic and glyoxylate cycle enzymes remain at high levels hours after glucose exposure. We then showed that <span class="named-content" data-type="genus-species">C. albicans</span> Pck1 and Icl1 are not destabilized by glucose, in contrast to <span class="named-content" data-type="genus-species">S. cerevisiae</span> Pck1 and Icl1, even though <span class="named-content" data-type="genus-species">C. albicans</span> has retained the molecular apparatus to program the accelerated, ubiquitin-mediated degradation of target proteins following glucose exposure. <span class="named-content" data-type="genus-species">C. albicans</span> Icl1 escapes degradation following glucose addition, because this enzyme lacks key ubiquitination sites required to target it for accelerated degradation. Our data show that there has been significant posttranscriptional rewiring during the evolution of this pathogen, thereby allowing <span class="named-content" data-type="genus-species">C. albicans</span> to continue to assimilate alternative carbon sources in the presence of glucose.</div></section><section id="sec-2" data-type="results"><h2>RESULTS</h2><section id="sec-2-1"><h3>Carbon source has a major impact on the <span class="named-content" data-type="genus-species">C. albicans</span> proteome.</h3><div role="paragraph">Microarray studies have shown that the <span class="named-content" data-type="genus-species">C. albicans</span> transcriptome is exquisitely sensitive to glucose and that genes involved in the assimilation of alternative carbon sources are subject to glucose repression (<a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). Our first aim was to establish whether this transcriptional regulation was reflected in the <span class="named-content" data-type="genus-species">C. albicans</span> proteome. To test this, we grew prototrophic <span class="named-content" data-type="genus-species">C. albicans</span> NGY152 cells (see <a href="#tabS1">Table S1</a> in the supplemental material) for 20 h in minimal medium containing glucose, lactate, oleate, or amino acids as the sole carbon source, harvested them in mid-exponential phase, prepared protein extracts, and subjected them to two-dimensional (2D) gel electrophoresis (Materials and Methods). Principal component analysis confirmed the reproducibility of the 2D gels from the independent replicate experiments and showed that the carbon source had a significant impact upon the <span class="named-content" data-type="genus-species">C. albicans</span> proteome (see <a href="#figS1">Fig. S1A</a> in the supplemental material). Proteins that displayed statistically significant changes in level on the basis of three independent experiments (<a href="#fig1">Fig. 1A</a>) were identified by tryptic digestion and matrix-assisted laser desorption ionizationâtime of flight (MALDI-ToF) mass spectrometry. Positive identifications were obtained for 206 2D gel features, representing 152 different <span class="named-content" data-type="genus-species">C. albicans</span> proteins, some distinct features representing isoforms of the same protein. The list of <span class="named-content" data-type="genus-species">C. albicans</span> proteins identified is presented in <a href="#tabS2">Table S2</a> and submitted to the PRIDE (proteomics identifications database) proteomic data repository (<a href="http://www.ebi.ac.uk/pride/www.ebi.ac.uk/pride/www.ebi.ac.uk/pride/">http://www.ebi.ac.uk/pride/</a>) (accession numbers 3186 to 3192).</div><aside id="figS1" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S1</header><div role="paragraph">(A) Principal component analysis confirms the significant impact of carbon source on the <span class="named-content" data-type="genus-species">C. albicans</span> proteome and the reproducibility of the replicate 2D gel profiles. (B) Correlation between the long-term and short-term effects of glucose on protein levels in <span class="named-content" data-type="genus-species">C. albicans</span> cells grown on oleic acid for all proteins for which changes in expression were detected. (C) Correlation for glycolytic enzymes. (D) Correlation for fatty acid Î²-oxidation enzymes. (E) Correlation for gluconeogenic, glyoxylate cycle, and TCA cycle enzymes. Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf01.pdf" download="mbo006121395sf01.pdf">Figure S1, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><aside id="tabS1" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Table S1</header><div role="paragraph"><span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> strains <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395st1.pdf" download="mbo006121395st1.pdf">Table S1, PDF file, 0.1 MB.</a></div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><aside id="tabS2" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Table S2</header><div role="paragraph">List of changes in the <span class="named-content" data-type="genus-species">C. albicans</span> proteome detected in response to alterations in carbon source <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395st2.pdf" download="mbo006121395st2.pdf">Table S2, PDF file, 0.1 MB.</a></div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div class="figure-wrap"><header><div class="label">FIG 1 </div></header><figure id="fig1" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/c4483bf2-85f6-48b4-9c17-8739653c5a67/assets/graphic/mbo0061213950001.jpeg" /><figcaption><span class="heading">FIG 1</span> Impact of carbon source on the <span class="named-content" data-type="genus-species">C. albicans</span> proteome. (A) Replicate 2D gels showing that Pox4 is more abundant during growth on oleic acid than on glucose and that Pox4 is retained after 2 h of exposure to glucose. (B) Network of <span class="named-content" data-type="genus-species">C. albicans</span> proteins regulated in response to carbon source. Nodes are connected by edges to the one or more conditions under which it was identified: upregulated â¥2-fold relative to growth on glucose (upward arrowhead); downregulated â¥2-fold (downward arrowhead). Functions that were differentially regulated on different carbon sources (rounded rectangle) have color-coded connecting lines: green for upregulated and red for downregulated. Functions regulated under all three conditions lie in three blocks in the center of the network interactions.</figcaption></figure></div><div role="paragraph">The network of proteins that displayed statistically significant changes in response to carbon source mainly comprised metabolic enzymes (<a href="#fig1">Fig. 1B</a>). Significant overlap was observed between the sets of proteins that were regulated in response to growth on lactate, oleate, or amino acids compared to the glucose condition. During growth on these organic acids, glycolytic enzymes were downregulated, and enzymes involved in gluconeogenesis, the glycoxylate cycle, the tricarboxylic acid (TCA) cycle and pathways involved in the assimilation of alternative carbon sources were upregulated (<a href="#fig1">Fig. 1B and 2B</a>; see <a href="#figS2">Fig. S2</a> in the supplemental material). For example, gluconeogenic (Pck1 and Fbp1) and tricarboxylic acid cycle enzymes (Aco1, Kgd1, and Mdh11) displayed reduced levels, and the glycolysis-specific enzyme pyruvate kinase (Cdc19) was at elevated levels (see <a href="#figS2">Fig. S2</a> and <a href="#tabS2">Table S2</a> in the supplemental material) during growth on glucose compared to lactate-, oleic acid-, and amino acid-grown cells. During growth on oleate, fatty acid Î²-oxidation (Faa21/23/24, Fox2, Pot11, Pox4, and Tes1), glyoxylate cycle (Icl1), and additional TCA cycle enzymes (Cit1 and Fum11) were present at elevated levels compared to the levels in glucose-grown cells (<a href="#fig2">Fig. 2</a>; <a href="#tabS2">Table S2</a>). Enzymes on many amino acid biosynthetic pathways were downregulated during growth on amino acids (Arg5,6, Bat21, Cpa2, His1, Hom6, Ilv5, Lys12, and Met6/14/15), and this was also the case during growth on lactate (Arg5,6, Bat21, His1, and Met6) or oleic acid (Arg5,6, Bat21, His1, and Met6) (<a href="#fig1">Fig. 1</a>; <a href="#tabS2">Table S2</a>). These data largely confirmed expectations based on our understanding of metabolism in other yeasts and the limited experimental data on <span class="named-content" data-type="genus-species">C. albicans</span> metabolic regulation (<a href="#B35" role="doc-biblioref" data-xml-rid="B35">35</a>, <a href="#B49" role="doc-biblioref" data-xml-rid="B49">49</a>). The observed changes in Pck1 and Icl1 levels were validated by Western blot analysis of these proteins following Myc tagging in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#fig2">Fig. 2A</a>). Note that Icl1 levels were below the limits of detection in the lactate proteome (<a href="#tabS2">Table S2</a>).</div><aside id="figS2" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S2</header><div role="paragraph">Impact of carbon sources on central carbon metabolism in <span class="named-content" data-type="genus-species">C. albicans</span>. (A) Glucose; (B) lactate; (C) amino acids. Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf02.pdf" download="mbo006121395sf02.pdf">Figure S2, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div class="figure-wrap"><header><div class="label">FIG 2 </div></header><figure id="fig2" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/4d870fa3-d67d-4ca5-9041-266a9e8974d9/assets/graphic/mbo0061213950002.jpeg" /><figcaption><span class="heading">FIG 2</span> Effect of carbon source on central carbon metabolic enzymes in <span class="named-content" data-type="genus-species">C. albicans</span>. (A) Western blots demonstrating the effects of overnight growth on different carbon sources on the levels of Myc-tagged Icl1 and Pck1 in <span class="named-content" data-type="genus-species">C. albicans</span> cells (CA1395 and CA1431 [see <a href="#tabS1">Table S1</a> in the supplemental material]). Control lanes contain extracts from the untagged parental strain (RM1000) control grown on lactate. (B) Effects of growth on oleic acid versus glucose on the levels of enzymes involved in central carbon metabolism. All enzymes shown were identified on the 2D gels. The effects of growth on oleic acid versus glucose on the levels of enzymes are indicated as follows: white, no significant change in the levels of glucose- and oleic acid-grown cells; red, protein level significantly elevated in oleic acid-grown cells; cyan, protein level significantly elevated in glucose-grown cells. For abbreviations, see the <i>Candida</i> Genome Database (<a href="http://www.candidagenome.org">http://www.candidagenome.org</a>).</figcaption></figure></div><div role="paragraph">The levels of other types of protein were modulated in response to carbon source. These proteins included proteins involved in growth and cell polarity (Arp2, Cap2, Pfy1, Cof1, Crn1, and Rbp1), nuclear transport (Ntf2), DNA repair (Rad23), and protein folding (Pdi1 and Cyp51) (see <a href="#tabS2">Table S2</a> in the supplemental material). Interestingly, proteins involved in drug resistance (Pdr13 and Erg13) were also affected by carbon source, which is consistent with the recent finding that carbon source affects the antifungal drug resistance of <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#B50" role="doc-biblioref" data-xml-rid="B50">50</a>). Also, a number of stress functions were affected by carbon source (<a href="#tabS2">Table S2</a>). The levels of Hsp70 family members (Hsp70/Ssa4, Ssa1, and Kar2) were differentially regulated in response to carbon source. The Trx1 thioredoxin was also dramatically upregulated during growth on lactate, oleic acid, or amino acids, while the glutaredoxin-like protein Grx3 was downregulated on lactate and oleic acid. The levels of catalase glutathione peroxidize (Gpx1) were also upregulated on lactate. These data were consistent with the observation that host carbon source affects stress resistance in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#B50" role="doc-biblioref" data-xml-rid="B50">50</a>).</div><div role="paragraph">Transcript profiling data are available for the effects of glucose on lactate-grown <span class="named-content" data-type="genus-species">C. albicans</span> cells (<a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). We found the correlation between these transcript profiling data and our proteomic data for lactate- and glucose-grown cells to be modest at best (correlation coefficient = 0.25) (<a href="#fig3">Fig. 3A</a>). No doubt differential protein stabilities, alterations in posttranslational modifications, and the existence of multiple 2D gel features for some proteins contributed to this. Nevertheless, this was consistent with other comparisons of transcriptomic and proteomic data sets, which vary considerably in their degree of correlation (<a href="#B51" role="doc-biblioref" data-xml-rid="B51">51</a><a href="#B52" role="doc-biblioref" data-xml-rid="B52 B53">â</a><a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>).</div><div class="figure-wrap"><header><div class="label">FIG 3 </div></header><figure id="fig3" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/bb843420-1a79-46c3-9548-807105a00bd7/assets/graphic/mbo0061213950003.jpeg" /><figcaption><span class="heading">FIG 3</span> Comparisons of the <span class="named-content" data-type="genus-species">C. albicans</span> transcriptome and proteome. (A) The <span class="named-content" data-type="genus-species">C. albicans</span> transcriptome and proteome display limited correlation with respect to the observed log fold changes between cells grown on glucose or lactic acid. The transcriptomic data were taken from reference <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>, and proteomic data were taken from <a href="#tabS2">Table S2</a> in the supplemental material. (B) Correlation between the short-term and long-term effects of glucose on central metabolic enzymes. Glycolytic enzymes are shown in blue. Gluconeogenic, glyoxylate cycle, and tricarboxylic acid (TCA) cycle enzymes are shown in red.</figcaption></figure></div></section><section id="sec-2-2"><h3>Differential effects of glucose on the <span class="named-content" data-type="genus-species">C. albicans</span> proteome in the short term and longer term.</h3><div role="paragraph">Having tested the impact on the <span class="named-content" data-type="genus-species">C. albicans</span> proteome of growth on different carbon sources, we examined the effects of glucose addition to cells growing on alternative carbon sources. A similar experimental approach was taken except that <span class="named-content" data-type="genus-species">C. albicans</span> NGY152 cells were grown for 20 h to mid-exponential phase on lactate, oleate, or amino acids as the sole carbon source, and then 2% glucose was added 2 h before the cells were harvested for proteomic analyses. Once again, proteins that displayed statistically significant changes across three independent experiments (<a href="#fig1">Fig. 1A</a>) were identified by mass spectrometry (see <a href="#tabS2">Table S2</a> in the supplemental material; PRIDE accession numbers 3186 to 3192). Principal component analysis indicated that glucose addition exerted significant effects on the lactate, oleate, and amino acid proteomes and that these effects were distinct from long-term growth on glucose (<a href="#fig1">Fig. 1B</a>), although similar subsets of proteins were affected (<a href="#tabS2">Table S2</a>).</div><div role="paragraph">We compared the short- and longer-term effects of glucose on the <span class="named-content" data-type="genus-species">C. albicans</span> proteome (<a href="#fig3">Fig. 3B</a>; see <a href="#figS1">Fig. S1</a> in the supplemental material). Interestingly, the extent of correlation between the short- and long-term changes differed for different sets of functionally related proteins. Glycolytic enzymes showed a strong correlation, generally displaying glucose induction in both the short and longer terms (<a href="#fig3">Fig. 3B</a>). In contrast, gluconeogenic, glycoxylate cycle, TCA cycle, and fatty acid Î²-oxidation enzymes displayed a poor correlation. The levels of many of these enzymes were generally lower in glucose-grown cells, and their levels did not decline significantly after exposure to glucose for 2 h (<a href="#figS1">Fig. S1</a>).</div></section><section id="sec-2-3"><h3>Icl1 and Pck1 are not subject to catabolite inactivation in <span class="named-content" data-type="genus-species">C. albicans</span>.</h3><div role="paragraph">The apparent stability of gluconeogenic and glyoxylate cycle enzymes following glucose exposure in <span class="named-content" data-type="genus-species">C. albicans</span> contrasted with observations in <span class="named-content" data-type="genus-species">S. cerevisiae</span> where glucose triggers catabolite inactivation of these enzymes (<a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a><a href="#B24" role="doc-biblioref" data-xml-rid="B24">â</a><a href="#B25" role="doc-biblioref" data-xml-rid="B25">25</a>, <a href="#B31" role="doc-biblioref" data-xml-rid="B31">31</a>). Therefore, we examined the impact of glucose on the stability of <span class="named-content" data-type="genus-species">C. albicans</span> Icl1 and Pck1 by Western blotting of Myc-tagged versions of these proteins (<a href="#fig4">Fig. 4</a>). <span class="named-content" data-type="genus-species">C. albicans</span> CA1395 (<i>ICL1-Myc<sub>3</sub></i>) and CA1431 (<i>PCK1-Myc<sub>3</sub></i>) (see <a href="#tabS1">Table S1</a> in the supplemental material) were grown to mid-exponential phase in medium containing lactate as the sole carbon source, and then 2% glucose was added. Protein extracts were prepared at various times thereafter and subjected to Western blotting, revealing that Icl1 and Pck1 were stable following glucose addition (<a href="#fig4">Fig. 4B</a>). Similar observations were made in oleic acid-grown cells exposed to glucose (see <a href="#figS3">Fig. S3</a> in the supplemental material). These data were consistent with our observation that Icl1 and Pck1 are retained at high levels in <span class="named-content" data-type="genus-species">C. albicans</span> cells 2 h after glucose addition (<a href="#tabS2">Table S2</a>).</div><aside id="figS3" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S3</header><div role="paragraph">Icl1 and Pck1 protein stability and <i>ICL1</i> and <i>PCK1</i> mRNA turnover in oleic acid-grown <span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> following the addition of glucose. (A and B) CaIcl1-Myc and CaPck1-Myc protein levels (A) and <i>CaICL1</i> and <i>CaPCK1</i> mRNA levels (B) after the addition of glucose to <span class="named-content" data-type="genus-species">C. albicans</span> cells grown on oleic acid. (C and D) ScIcl1-Myc and ScPck1-HA protein levels (C) and <i>ScICL1</i> and <i>ScPCK1</i> mRNA levels (D) after glucose addition to <span class="named-content" data-type="genus-species">S. cerevisiae</span> cells grown on oleic acid. Similar data were obtained from two independent replicate experiments. Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf03.pdf" download="mbo006121395sf03.pdf">Figure S3, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div class="figure-wrap"><header><div class="label">FIG 4 </div></header><figure id="fig4" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/5eff0bf0-7f30-4c4f-a620-4815084b2ed1/assets/graphic/mbo0061213950004.jpeg" /><figcaption><span class="heading">FIG 4</span> Icl1 and Pck1 protein stability and <i>ICL1</i> and <i>PCK1</i> mRNA turnover in <span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> following the addition of glucose. (A) CaIcl1-Myc and CaPck1-Myc protein levels were measured by Western blotting at various times after the addition of glucose to <span class="named-content" data-type="genus-species">C. albicans</span> cells grown on lactate. (B) CaIcl1-Myc and CaPck1-Myc were quantified by Western blotting relative to the Act1 internal control following glucose addition. Also <i>CaICL1</i> and <i>CaPCK1</i> mRNA levels were assayed in the same cultures by qRT-PCR and compared to the level in the <i>ACT1</i> mRNA internal control after glucose addition. The relative levels of these mRNAs and proteins were expressed as a percentage of their abundance at time zero, which was set at 100%. (C) ScIcl1-Myc and ScPck1-HA protein levels were measured by Western blotting, and <i>ScICL1</i> and <i>ScPCK1</i> mRNA levels were assayed by qRT-PCR in <span class="named-content" data-type="genus-species">S. cerevisiae</span> cells grown on lactate and exposed to glucose at time zero. The relative levels of these mRNAs and proteins were expressed as a percentage of their abundance at time zero (which was set at 100%): circles, Icl1; squares, Pck1; black symbols, Icl1 and Pck1 proteins in lactate-grown cells after glucose addition; gray symbols, Icl1 and Pck1 proteins in control lactate-grown cells with no glucose addition; open symbols, <i>ICL1</i> and <i>PCK1</i> mRNAs in lactate-grown cells after glucose addition. Similar data were obtained from two independent replicate experiments.</figcaption></figure></div><div role="paragraph">We then measured the stability of the corresponding mRNAs under equivalent growth conditions by quantitative reverse transcription-PCR (qRT-PCR) relative to the internal <i>ACT1</i> mRNA control. The <i>ICL1</i> and <i>PCK1</i> mRNAs were rapidly degraded following glucose addition to lactate-grown cells (<a href="#fig4">Fig. 4B</a>), and similar data were obtained for oleic acid-grown cells (see <a href="#figS3">Fig. S3</a> in the supplemental material). These observations were entirely consistent with the results of previous microarray experiments, suggesting that these transcripts are strongly repressed by glucose in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). Therefore, <i>ICL1</i> and <i>PCK1</i> transcription is strongly regulated by glucose, but this regulation is not reflected in the proteome.</div><div role="paragraph">These experiments were replicated in <span class="named-content" data-type="genus-species">S. cerevisiae</span> to exclude the possibility that the lack of catabolite inactivation in <span class="named-content" data-type="genus-species">C. albicans</span> was an experimental artifact. <span class="named-content" data-type="genus-species">S. cerevisiae</span> DS1-W10 (<i>ICL1-Myc<sub>9</sub></i>) and DS1-W20 (<i>PCK1-HA<sub>6</sub></i>) (see <a href="#tabS1">Table S1</a> in the supplemental material) were grown on lactate and exposed to 2% glucose, and at various times thereafter, Icl1 and Pck1 levels were examined by Western blotting (<a href="#fig4">Fig. 4C</a>). The Icl1 and Pck1 enzymes were stable in <span class="named-content" data-type="genus-species">S. cerevisiae</span> in the absence of glucose but were destabilized following glucose addition. Similar data were obtained for oleic acid-grown <span class="named-content" data-type="genus-species">S. cerevisiae</span> cells (see <a href="#figS3">Fig. S3</a> in the supplemental material). These data recapitulate earlier observations (<a href="#B23" role="doc-biblioref" data-xml-rid="B23">23</a>, <a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>) and confirm that these gluconeogenic and glyoxylate cycle enzymes are subject to catabolite inactivation in <span class="named-content" data-type="genus-species">S. cerevisiae</span>. Furthermore, the <i>ICL1</i> and <i>PCK1</i> transcripts were rapidly degraded in <span class="named-content" data-type="genus-species">S. cerevisiae</span> following glucose addition (<a href="#fig4">Fig. 4C</a>), confirming earlier reports to this effect (<a href="#B5" role="doc-biblioref" data-xml-rid="B5">5</a>, <a href="#B6" role="doc-biblioref" data-xml-rid="B6">6</a>, <a href="#B20" role="doc-biblioref" data-xml-rid="B20">20</a>). Therefore, <span class="named-content" data-type="genus-species">C. albicans</span> differs significantly from <span class="named-content" data-type="genus-species">S. cerevisiae</span> in that Icl1 and Pck1 are not subject to catabolite inactivation in response to glucose in the pathogenic yeast.</div></section><section id="sec-2-4"><h3><span class="named-content" data-type="genus-species">C. albicans</span> has retained the apparatus for catabolite inactivation.</h3><div role="paragraph">In principle, the lack of glucose-accelerated degradation of Icl1 and Pck1 in <span class="named-content" data-type="genus-species">C. albicans</span> could have been due to the evolutionary loss of the catabolite inactivation apparatus in this pathogen. To test this, we asked whether <span class="named-content" data-type="genus-species">S. cerevisiae</span> Icl1 is subject to glucose-accelerated degradation in <span class="named-content" data-type="genus-species">C. albicans</span>.</div><div role="paragraph"><span class="named-content" data-type="genus-species">C. albicans</span> strains were constructed in which the <i>S. cerevisiae ICL1</i> (<i>ScICL1</i>) open reading frame (ORF) was expressed from the <i>C. albicans ICL1</i> (<i>CaICL1</i>) promoter at the native <i>ICL1</i> locus. The functionality of three-Myc (Myc<sub>3</sub>)-tagged <span class="named-content" data-type="genus-species">S. cerevisiae</span> Icl1 (ScIcl1) in <span class="named-content" data-type="genus-species">C. albicans</span> was first confirmed in DSCO2 cells (<i>icl1</i>/<i>ScICL1</i>) (see <a href="#tabS1">Table S1</a> in the supplemental material), the <i>ScICL1-Myc</i><sub><i>3</i></sub> gene suppressing the growth defect of <i>C. albicans icl1</i> cells on lactate, oleic acid, pyruvate, and acetate as the sole carbon source (<a href="#fig5">Fig. 5A</a>). The stability of the Myc<sub>3</sub>-tagged ScIcl1 was then examined in lactate-grown <span class="named-content" data-type="genus-species">C. albicans</span> DSCO1 cells (<i>ICL1</i>/<i>ScICL1-Myc</i><sub><i>3</i></sub>). ScIcl1 was stable in <span class="named-content" data-type="genus-species">C. albicans</span> in the absence of glucose but was rapidly degraded following glucose addition (<a href="#fig5">Fig. 5B</a>). These observations were reproducible, and analogous observations were made in oleic acid-grown <span class="named-content" data-type="genus-species">C. albicans</span> DSCO1 cells (see <a href="#figS4">Fig. S4</a> in the supplemental material). We conclude that ScIcl1 is subject to glucose-accelerated degradation in <span class="named-content" data-type="genus-species">C. albicans</span>, and hence that during evolution <span class="named-content" data-type="genus-species">C. albicans</span> has retained the apparatus for catabolite inactivation.</div><aside id="figS4" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S4</header><div role="paragraph">ScIcl1 is degraded in <span class="named-content" data-type="genus-species">C. albicans</span>, but CaIcl1 is stable in <span class="named-content" data-type="genus-species">S. cerevisiae</span>. (A) ScIcl1 is rapidly degraded in oleic acid-grown <span class="named-content" data-type="genus-species">C. albicans</span> DSCO1 cells after glucose exposure. Similar data were obtained from two independent replicate experiments. (B) CaIcl1 is stable in oleic acid-grown <span class="named-content" data-type="genus-species">S. cerevisiae</span> DS4-Y40 cells after glucose exposure. Similar data were obtained from two independent replicate experiments Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf04.pdf" download="mbo006121395sf04.pdf">Figure S4, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div class="figure-wrap"><header><div class="label">FIG 5 </div></header><figure id="fig5" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/25b81059-6daa-45bb-8fba-7ddb7d93c437/assets/graphic/mbo0061213950005.jpeg" /><figcaption><span class="heading">FIG 5</span> ScIcl1 is functional in <span class="named-content" data-type="genus-species">C. albicans</span> and rapidly degraded in response to glucose. (A) Expression of <i>ScICL1</i> in <span class="named-content" data-type="genus-species">C. albicans</span> suppresses the carbon source conditional phenotypes of an <i>icl1</i> mutant: <i>ICL1</i>, <span class="named-content" data-type="genus-species">C. albicans</span> CA510 (<i>ICL1/icl1</i>) (see <a href="#tabS1">Table S1</a> in the supplemental material); <i>icl1</i>, <span class="named-content" data-type="genus-species">C. albicans</span> CA517 (<i>icl1/icl1</i>); <i>ScICL1</i>, <span class="named-content" data-type="genus-species">C. albicans</span> DSCO1 (<i>ScICL1/icl1</i>). (B) Impact of glucose on the levels of the ScIcl1-Myc protein expressed in lactate-grown <span class="named-content" data-type="genus-species">C. albicans</span> DSCO1 cells relative to the abundance at time zero. (Top) Western blotting of ScIcl1-Myc levels. (Bottom) Quantification of ScIcl1-Myc levels expressed as a percentage of the abundance at time zero (100%): closed symbols, plus glucose; open symbols, control cells lacking glucose. Similar data were obtained from two independent replicate experiments.</figcaption></figure></div><div role="paragraph">In this case, has CaIcl1 lost the signal that triggers glucose-accelerated degradation? The stability of the CaIcl1-Myc<sub>3</sub> protein was assayed in the <span class="named-content" data-type="genus-species">S. cerevisiae</span> strain DS4-Y40 in which the Myc<sub>3</sub>-tagged <i>CaICL1</i> open reading frame was expressed from the <i>ScICL1</i> promoter at the native <i>ICL1</i>/<i>YDR477w</i> locus (see <a href="#tabS1">Table S1</a> in the supplemental material). The CaIcl1-Myc<sub>3</sub> protein was stable in lactate-grown DS4-Y40 cells and was not destabilized by glucose addition relative to internal loading controls (<a href="#fig6">Fig. 6</a>). Once again, these observations were reproducible, and analogous observations were made for oleic acid-grown <span class="named-content" data-type="genus-species">S. cerevisiae</span> DS4-Y40 cells (see <a href="#figS4">Fig. S4</a> in the supplemental material). Therefore, CaIcl1 appears to have lost the signal that triggers glucose-accelerated degradation.</div><div class="figure-wrap"><header><div class="label">FIG 6 </div></header><figure id="fig6" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/a7f85416-699e-4119-a003-6337c6173ce4/assets/graphic/mbo0061213950006.jpeg" /><figcaption><span class="heading">FIG 6</span> CaIcl1 is stable in <span class="named-content" data-type="genus-species">S. cerevisiae</span> following the addition of glucose. <i>CaICL1</i>-<i>Myc</i><sub><i>3</i></sub> was expressed in <span class="named-content" data-type="genus-species">S. cerevisiae</span> DS4-Y40 cells (see <a href="#tabS1">Table S1</a> in the supplemental material) grown on lactate, and CaIcl1-Myc protein levels were assayed by Western blotting after glucose addition. CaIcl1-Myc levels are expressed as a percentage of the abundance at time zero (which was set at 100%): closed symbols, plus glucose; open symbols, control cells lacking glucose. Similar data were obtained from two independent replicate experiments.</figcaption></figure></div></section><section id="sec-2-5"><h3>Ubiquitination contributes to glucose-accelerated protein degradation in <span class="named-content" data-type="genus-species">C. albicans</span>.</h3><div role="paragraph">Ubiquitination contributes to glucose-mediated protein destabilization in <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>, <a href="#B32" role="doc-biblioref" data-xml-rid="B32">32</a>). Therefore, we tested whether ubiquitination contributes to glucose-accelerated degradation of ScIcl1 in <span class="named-content" data-type="genus-species">C. albicans</span>. The <i>ScICL1-Myc</i><sub><i>3</i></sub> cassette was integrated at the native <i>ICL1</i> locus in a <i>C. albicans ubi4</i> mutant that lacks polyubiquitin. Ubiquitination is significantly reduced in this <i>ubi4</i> mutant, residual ubiquitin being expressed only from the <i>UBI3</i> locus (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>). The stability of the ScIcl1-Myc<sub>3</sub> protein in these <span class="named-content" data-type="genus-species">C. albicans</span> DSCO3 cells (<i>ubi4/ubi4 ICL1/ScICL1-Myc<sub>3</sub></i>) (see <a href="#tabS1">Table S1</a> in the supplemental material]) was compared to ScIcl1-Myc<sub>3</sub> stability in DSCO1 cells (<i>UBI4/UBI4 ICL1/ScICL1-Myc<sub>3</sub></i>) and the stability of the CaIcl1-Myc<sub>3</sub> protein in CA1395 cells (<i>UBI4/UBI4 ICL1/ICL1-Myc<sub>3</sub></i>). ScIcl1-Myc<sub>3</sub> and CaIcl1-Myc<sub>3</sub> were stable in these strains during growth on lactate in the absence of glucose (not shown). Once again, ScIcl1-Myc<sub>3</sub> was destabilized following glucose addition to wild-type DSCO1 cells (<a href="#fig7">Fig. 7</a>). However, ScIcl1-Myc<sub>3</sub> was partially stabilized in the polyubiquitin mutant, suggesting that glucose-accelerated degradation of ScIcl1 in <span class="named-content" data-type="genus-species">C. albicans</span> is at least partially dependent upon ubiquitination. This observation was reproducible in independent experiments, and furthermore, similar observations were made when the same strains were grown on oleic acid rather than lactate (see <a href="#figS5">Fig. S5</a> in the supplemental material). The residual ScIcl1-Myc<sub>3</sub> destabilization observed in lactate- or oleic acid-grown <i>ubi4</i> cells could have been due either to the involvement of a second degradation pathway in this response (<a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>) or to the residual ubiquitination that is observed in the <span class="named-content" data-type="genus-species">C. albicans</span> polyubiquitin mutant (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>). We conclude that ubiquitination contributes to glucose-accelerated protein degradation in <i>C. albicans.</i></div><aside id="figS5" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S5</header><div role="paragraph">Inactivation of polyubiquitin inhibits glucose-accelerated ScIcl1 degradation in oleic acid-grown <span class="named-content" data-type="genus-species">C. albicans</span> DSCO3 cells. Similar data were obtained from two independent replicate experiments. Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf05.pdf" download="mbo006121395sf05.pdf">Figure S5, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div class="figure-wrap"><header><div class="label">FIG 7 </div></header><figure id="fig7" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/307e6d96-4140-435b-81df-70bc29468dd6/assets/graphic/mbo0061213950007.jpeg" /><figcaption><span class="heading">FIG 7</span> Inactivation of polyubiquitin inhibits glucose-accelerated ScIcl1 degradation in <i>C. albicans. ScICL1</i>-<i>Myc</i><sub><i>3</i></sub> was expressed in <span class="named-content" data-type="genus-species">C. albicans</span> DSCO3 (<i>ubi4/ubi4</i>) (see <a href="#tabS1">Table S1</a> in the supplemental material) and DSCO1 (wild type [wt]) (<i>UBI4/UBI4</i>) cells grown on lactate, and then ScIcl1-Myc protein levels were assayed after the addition of glucose by Western blotting. For a control, CaIcI1-Myc levels were assayed in <span class="named-content" data-type="genus-species">C. albicans</span> DSCO3 (<i>ubi4/ubi4</i>) cells grown on lactate and following glucose addition. ScIcl1-Myc and CaIcl1-Myc levels are expressed as a percentage of their abundance at time zero (which was set at 100%). Similar data were obtained from two independent replicate experiments.</figcaption></figure></div></section><section id="sec-2-6"><h3><span class="named-content" data-type="genus-species">C. albicans</span> Icl1 lacks ubiquitination sites that trigger glucose-accelerated protein degradation.</h3><div role="paragraph">Why does CaIcl1 escape glucose-accelerated ubiquitin-mediated protein degradation? We screened for consensus ubiquitination target sites in CaIcl1 and ScIcl1 using UbPred (predictor of protein ubiquitination sites) (<a href="http://www.ubpred.org/index.htmlwww.ubpred.org/index.htmlwww.ubpred.org/index.html">http://www.ubpred.org/index.html</a>) (<a href="#B56" role="doc-biblioref" data-xml-rid="B56">56</a>). The UbPred software, which was trained on 272 ubiquitination sites in <span class="named-content" data-type="genus-species">S. cerevisiae</span>, predicts ubiquitination sites based on numerous properties of the target amino acid sequence, including the net and total charge, aromatic content, charge/hydrophobicity ratio, sequence complexity, flexibility, amphipathic moment, and intrinsic disorder (<a href="#B56" role="doc-biblioref" data-xml-rid="B56">56</a>). The ScIcl1 and CaIcl1 proteins display strong amino acid sequence similarity (78% similarity and 67% identity). Yet while ScIcl1 contains two strong consensus ubiquitination sites at residues 158 and 551 according to UbPred, CaIcl1 carries no high-confidence ubiquitination targets (<a href="#fig8">Fig. 8A</a>). Therefore, we reasoned that the lack of such a ubiquitination site protected CaIcl1 from glucose-accelerated degradation.</div><div class="figure-wrap"><header><div class="label">FIG 8 </div></header><figure id="fig8" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/01a2f814-3be7-47c1-9b06-fa5ab4db3a82/assets/graphic/mbo0061213950008.jpeg" /><figcaption><span class="heading">FIG 8</span> Addition of a consensus ubiquitin site stimulates glucose-accelerated degradation of CaICl1 in <span class="named-content" data-type="genus-species">C. albicans</span>. (A) Schematic representation illustrating the existence of high-confidence ubiquitination sites in ScIcl1 and the lack of such sites in CaIcl1 as predicted by UbPred (<a href="http://www.ubpred.org/index.html">http://www.ubpred.org/index.html</a>) (<a href="#B56" role="doc-biblioref" data-xml-rid="B56">56</a>). (B) The carboxy-terminal ubiquitination site from ScIcl1 was fused to CaIcl1 to create CaIcl1-Ubi-Myc in <span class="named-content" data-type="genus-species">C. albicans</span> DSCO4 (see <a href="#tabS1">Table S1</a> in the supplemental material). These cells were grown on lactate, and the levels of CaIcl1-Ubi-Myc were assayed by Western blotting after the addition of glucose. For controls, the stabilities of CaIcl1-Myc (CA1395) and ScIcl1-Myc (DSCO1) (gray squares) in <span class="named-content" data-type="genus-species">C. albicans</span> were compared under equivalent conditions. CaIcl1-Ubi-Myc, ScIcl1-Myc, and CaIcl1-Myc levels are expressed as a percentage of their abundance at time zero (which was set at 100%). Similar data were obtained from two independent replicate experiments.</figcaption></figure></div><div role="paragraph">To test this, we fused the carboxy-terminal ubiquitination site from ScIcl1 onto the carboxy terminus of CaIcl1 along with a Myc<sub>3</sub> epitope tag and then compared the stability of this CaIcl1-Ubi-Myc<sub>3</sub> protein with control CaIcl1-Myc<sub>3</sub> and ScIcl1-Myc<sub>3</sub> proteins in <span class="named-content" data-type="genus-species">C. albicans</span>. The stabilities of these proteins were first measured in mid-exponential <span class="named-content" data-type="genus-species">C. albicans</span> CA1395 (<i>ICL1/ICL1-Myc<sub>3</sub></i>), DSCO1 (<i>ICL1/ScICL1-Myc<sub>3</sub></i>), and DSCO4 (<i>ICL1/ICL1-Ubi-Myc<sub>3</sub></i>) cells grown on lactate. All of the proteins were stable under these conditions (not shown). Once again, ScIcl1-Myc<sub>3</sub> was destabilized following glucose addition, whereas CaIcl1-Myc<sub>3</sub> remained stable (<a href="#fig8">Fig. 8B</a>). Interestingly, the CaIcl1-Ubi-Myc<sub>3</sub> protein was reproducibly destabilized following glucose addition. Furthermore, analogous observations were made in <span class="named-content" data-type="genus-species">C. albicans</span> CA1395, DSCO1, and DSCO4 cells grown on oleic acid (see <a href="#figS6">Fig. S6</a> in the supplemental material). Therefore, the addition of a ubiquitination site was sufficient to trigger glucose-accelerated degradation of the normally stable CaIcl1-Myc<sub>3</sub> protein in <span class="named-content" data-type="genus-species">C. albicans</span>. These observations reinforced the view that <span class="named-content" data-type="genus-species">C. albicans</span> has retained the apparatus for glucose-accelerated protein degradation and suggested that the lack of an appropriate ubiquitination site in CaIcl1 prevents this protein from entering this degradation pathway.</div><aside id="figS6" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S6</header><div role="paragraph">Addition of a consensus ubiquitin site stimulates glucose-accelerated degradation of the CaIcl1-Ubi-Myc protein in oleic acid-grown <span class="named-content" data-type="genus-species">C. albicans</span>. Similar data were obtained from two independent replicate experiments. Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf06.pdf" download="mbo006121395sf06.pdf">Figure S6, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside></section><section id="sec-2-7"><h3><span class="named-content" data-type="genus-species">C. albicans</span> continues to assimilate alternative carbon sources following glucose exposure.</h3><div role="paragraph">Is this evolutionary rewiring of ubiquitination targets between <span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> reflected in the metabolic activities of these yeasts? To test this, we compared [<sup>14</sup>C]lactate assimilation by mid-exponential <span class="named-content" data-type="genus-species">C. albicans</span> RM1000 and <span class="named-content" data-type="genus-species">S. cerevisiae</span> W303-1B cells that were grown on lactate in the presence and absence of glucose (<a href="#fig9">Fig. 9</a>). As expected, the presence of glucose inhibited the assimilation of [<sup>14</sup>C]lactate by <span class="named-content" data-type="genus-species">S. cerevisiae</span>. However, glucose did not inhibit [<sup>14</sup>C]lactate by <span class="named-content" data-type="genus-species">C. albicans</span> over the 4-h period examined, which was consistent with the retention by this pathogen of glyoxylate cycle and gluconeogenic enzymes after glucose addition. These reproducible observations were reinforced by analogous data on oleic acid assimilation by oleic acid-grown <span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> cells. Glucose inhibited [<sup>3</sup>H]oleic acid assimilation in <span class="named-content" data-type="genus-species">S. cerevisiae</span> W303-1B cells, but not in <span class="named-content" data-type="genus-species">C. albicans</span> RM1000 cells (see <a href="#figS7">Fig. S7</a> in the supplemental material). These data indicate that, unlike <span class="named-content" data-type="genus-species">S. cerevisiae</span>, <span class="named-content" data-type="genus-species">C. albicans</span> continues to assimilate alternative carbon sources after exposure to glucose.</div><aside id="figS7" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Figure S7</header><div role="paragraph">Glucose does not inhibit [<sup>3</sup>H]oleic acid assimilation by <span class="named-content" data-type="genus-species">C. albicans</span> RM1000 but does block [<sup>3</sup>H]oleic acid assimilation in <span class="named-content" data-type="genus-species">S. cerevisiae</span> W303-1B. Similar data were obtained from two independent replicate experiments. Download <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf07.pdf" download="mbo006121395sf07.pdf">Figure S7, PDF file, 0.1 MB</a>.</div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div class="figure-wrap"><header><div class="label">FIG 9 </div></header><figure id="fig9" class="graphic"><img src="/cms/10.1128/mBio.00495-12/asset/66facf92-676c-4b43-a537-5132bc9f3da6/assets/graphic/mbo0061213950009.jpeg" /><figcaption><span class="heading">FIG 9</span> Glucose does not inhibit lactate assimilation by <span class="named-content" data-type="genus-species">C. albicans</span>. Exponential <span class="named-content" data-type="genus-species">C. albicans</span> RM1000 and <span class="named-content" data-type="genus-species">S. cerevisiae</span> W303-1B cells (see <a href="#tabS1">Table S1</a> in the supplemental material) grown on lactate were suspended in fresh medium containing [<sup>14</sup>C]lactate and 2% glucose (gray bars) or with no glucose (black bars). The assimilation of radiolabeled lactate by cells was assayed at various times thereafter (Materials and Methods). Similar data were obtained from two independent replicate experiments.</figcaption></figure></div></section></section><section id="sec-3" data-type="discussion"><h2>DISCUSSION</h2><div role="paragraph">The prevailing view is that <span class="named-content" data-type="genus-species">C. albicans</span> fits the <span class="named-content" data-type="genus-species">S. cerevisiae</span> paradigm with regard to the impact of glucose upon central carbon metabolism. This influences current thinking about nutrient adaptation during infection and host-fungus interactions (<a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>, <a href="#B57" role="doc-biblioref" data-xml-rid="B57">57</a>). However, this view does not resonate well with the evolution of these yeasts in contrasting niches that differ significantly with regard to carbon source availability (<a href="#B40" role="doc-biblioref" data-xml-rid="B40">40</a>) and with an early report that <span class="named-content" data-type="genus-species">C. albicans</span> is a Crabtree-negative yeast (<a href="#B48" role="doc-biblioref" data-xml-rid="B48">48</a>). We now show that, despite the similar effects that glucose exerts on the <span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> transcriptomes (<a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>), these yeasts display fundamental differences in the effects of glucose on central metabolic functions and carbon assimilation. The levels of many gluconeogenic, glyoxylate cycle, and fatty acid Î²-oxidation enzymes do decline in <span class="named-content" data-type="genus-species">C. albicans</span> after protracted growth on glucose, and the changes in the proteomic network (<a href="#fig1">Fig. 1</a>) reflect those observed previously during a comprehensive comparison of exponential- and stationary-phase <span class="named-content" data-type="genus-species">C. albicans</span> cells (<a href="#B58" role="doc-biblioref" data-xml-rid="B58">58</a>). However, these enzymes are stable during short-term exposure to glucose (<a href="#fig2">Fig. 2 and 4</a>; see <a href="#tabS2">Table S2</a> in the supplemental material), and this allows <span class="named-content" data-type="genus-species">C. albicans</span> cells to continue to assimilate alternative carbon sources even in the presence of glucose, unlike <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#fig9">Fig. 9</a>).</div><div role="paragraph">Our analyses of Icl1 and Pck1 turnover have suggested that enzymes involved in the assimilation of alternative carbon sources are retained in <span class="named-content" data-type="genus-species">C. albicans</span> following glucose addition because these enzymes remain stable while their transcripts are degraded (<a href="#fig4">Fig. 4</a>). We show that this contrasts with <span class="named-content" data-type="genus-species">S. cerevisiae</span> cells that degrade these enzymes following glucose addition and their transcripts (<a href="#fig4">Fig. 4</a>). It is conceivable that carboxy-terminal tagging of CaIcl1 and ScIcl1 affected their localization and stability. Nevertheless, our data were consistent with the study of LÃ³pez-Boado et al. (<a href="#B24" role="doc-biblioref" data-xml-rid="B24">24</a>) who reported that Icl1 is subject to catabolite inactivation in <span class="named-content" data-type="genus-species">S. cerevisiae</span>. We then showed that, although CaIcl1 and CaPck1 are not subjected to glucose-accelerated protein degradation in <span class="named-content" data-type="genus-species">C. albicans</span>, this pathogen has retained the molecular apparatus to execute this function, as ScIcl1 is degraded following glucose addition when expressed as a functional enzyme in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#fig5">Fig. 5</a>). Rather, CaIcl1 lacks the signal that triggers glucose-accelerated protein degradation in <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#fig6">Fig. 6</a>).</div><div role="paragraph">Ubiquitin-dependent mechanisms contribute to the catabolite inactivation of Fbp1 in <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#B28" role="doc-biblioref" data-xml-rid="B28">28</a>), and protein ubiquitination is known to be important for nutrient adaptation as well as growth, morphogenesis, and stress responses in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>, <a href="#B59" role="doc-biblioref" data-xml-rid="B59">59</a><a href="#B60" role="doc-biblioref" data-xml-rid="B60 B61 B62 B63">â</a><a href="#B64" role="doc-biblioref" data-xml-rid="B64">64</a>). Therefore, we reasoned that ubiquitination might also contribute to glucose-accelerated protein degradation in <span class="named-content" data-type="genus-species">C. albicans</span> and that CaIcl1 might lack critical ubiquitination sites required to target this protein for catabolite inactivation. Three observations support this hypothesis. First, the inactivation of polyubiquitin (Ubi4) lowered the rate of ScIcl1 degradation in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#fig7">Fig. 7</a>). ScIcl1 degradation was not completely blocked because the <i>UBI3</i> gene would provide residual ubiquitin in <i>C. albicans ubi4</i> cells (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>, <a href="#B65" role="doc-biblioref" data-xml-rid="B65">65</a>). Second, <i>in silico</i> analyses of the ScIcl1 and CaIcl1 sequences revealed that while ScIcl1 contains two high-confidence ubiquitination sites, CaIcl1 contains none (<a href="#fig8">Fig. 8A</a>) despite these proteins displaying a high degree of overall sequence identity (67%). Third, the addition of a carboxy-terminal ubiquitination site to the CaIcl1 protein was sufficient to program this protein for glucose-accelerated protein degradation in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#fig8">Fig. 8</a>). Therefore, during the evolution of <span class="named-content" data-type="genus-species">C. albicans</span>, this pathogen has retained the molecular apparatus that mediates glucose-accelerated protein degradation, and CaIcl1 has evolved to escape this process.</div><div role="paragraph">Our proteomic analyses indicate that other <span class="named-content" data-type="genus-species">C. albicans</span> enzymes involved in the assimilation of alternative carbon sources lack ubiquitination sites are retained following glucose exposure (<a href="#fig3">Fig. 3B</a>; see <a href="#tabS1">Table S1</a> in the supplemental material). Furthermore, our <i>in silico</i> comparisons have suggested that enzymes specific for gluconeogenesis and the glyoxylate cycle lack the requisite ubiquitination sites required for glucose-accelerated protein degradation. <span class="named-content" data-type="genus-species">S. cerevisiae</span> malate synthase (ScMls1) contains three low-confidence ubiquitination sites according to UbPred (<a href="http://www.ubpred.org/index.htmlwww.ubpred.org/index.htmlwww.ubpred.org/index.html">http://www.ubpred.org/index.html</a>) (56), whereas CaMls1 contains no putative ubiquitination sites. UbPred also predicts that both ScFbp1 and ScPck1 contain high-confidence ubiquitination sites, whereas CaFbp1 and CaPck1 do not. This was consistent with our observation that, following glucose addition, ScPck1 is degraded in <span class="named-content" data-type="genus-species">S. cerevisiae</span>, but CaPck1 is not degraded in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#fig4">Fig. 4</a>). Meanwhile, our previous work has indicated that a number of glycolytic enzymes are ubiquitinated in <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>). Therefore, significant evolutionary rewiring of ubiquitination targets appears to have occurred in this pathogen, allowing it to assimilate alternative carbon sources in the presence of glucose, unlike <span class="named-content" data-type="genus-species">S. cerevisiae</span> (<a href="#fig9">Fig. 9</a>).</div><div role="paragraph">These observations have several important implications for our understanding of <span class="named-content" data-type="genus-species">C. albicans</span> as a pathogen. First, our data indicate that under some conditions there is significant dislocation between the transcriptome and proteome. Microarray studies have contributed significantly to our understanding of <span class="named-content" data-type="genus-species">C. albicans</span> pathobiology (<a href="#B43" role="doc-biblioref" data-xml-rid="B43">43</a>, <a href="#B44" role="doc-biblioref" data-xml-rid="B44">44</a>, <a href="#B46" role="doc-biblioref" data-xml-rid="B46">46</a>, <a href="#B66" role="doc-biblioref" data-xml-rid="B66">66</a><a href="#B67" role="doc-biblioref" data-xml-rid="B67">â</a><a href="#B68" role="doc-biblioref" data-xml-rid="B68">68</a>), and it is frequently presumed that changes in the transcriptome are reflected in corresponding changes in the <span class="named-content" data-type="genus-species">C. albicans</span> proteome and physiology. In some cases, there is a good correlation between the transcriptome and proteome, for example with regard to the induction of glycolytic enzymes in response to glucose (<a href="#fig3">Fig. 3</a>) and during amino acid starvation (<a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>). However, this is not always the case (<a href="#B69" role="doc-biblioref" data-xml-rid="B69">69</a>), and this study demonstrates clearly that caution is required in making presumptions about the proteome based on the transcriptome. Indeed, this explains the conundrum as to why <span class="named-content" data-type="genus-species">C. albicans</span> is a Crabtree-negative yeast (<a href="#B48" role="doc-biblioref" data-xml-rid="B48">48</a>) even though its transcriptome is exquisitely sensitive to glucose (<a href="#B47" role="doc-biblioref" data-xml-rid="B47">47</a>). Thankfully, many studies provide experimental confirmation of working hypotheses on the basis of their microarray observations.</div><div role="paragraph">The second important implication relates to fungal nutrient assimilation during colonization and disease progression. <span class="named-content" data-type="genus-species">C. albicans</span> occupies dynamic host niches that contain complex mixtures of carbon sources that change over time through a combination of host and fungal metabolic activities. Many niches contain minimal sugar concentrations (e.g., the urogenital tract), while in others, <span class="named-content" data-type="genus-species">C. albicans</span> is transiently exposed to sugars (e.g., the oral cavity and gastrointestinal tract). This opportunistic pathogen probably evolved in niches such as these. Also, the evidence suggests that a proportion of <span class="named-content" data-type="genus-species">C. albicans</span> cells that infect internal organs actively assimilate alternative carbon sources rather than sugars (<a href="#B40" role="doc-biblioref" data-xml-rid="B40">40</a>, <a href="#B70" role="doc-biblioref" data-xml-rid="B70">70</a>). Furthermore, lactate assimilation is essential for the proliferation of <span class="named-content" data-type="genus-species">C. glabrata</span> in the intestine (<a href="#B38" role="doc-biblioref" data-xml-rid="B38">38</a>). Our data now suggest that transient exposure to sugar does not prejudice the assimilation of alternative carbon sources and, as a result, the growth of the fungus in these niches. This explains why both gluconeogenic and glycolytic functions can be expressed in <span class="named-content" data-type="genus-species">C. albicans</span> cells infecting the kidney (<a href="#B40" role="doc-biblioref" data-xml-rid="B40">40</a>) and why mutations that block gluconeogenesis or the glyoxylate cycle partially attenuate the virulence of <span class="named-content" data-type="genus-species">C. albicans</span> (<a href="#B39" role="doc-biblioref" data-xml-rid="B39">39</a><a href="#B40" role="doc-biblioref" data-xml-rid="B40 B41">â</a><a href="#B42" role="doc-biblioref" data-xml-rid="B42">42</a>). Clearly, <span class="named-content" data-type="genus-species">C. albicans</span> does not conform to the <span class="named-content" data-type="genus-species">S. cerevisiae</span> paradigm of glucose regulation, and this is important for the pathogenicity of this yeast.</div></section><section id="sec-4" data-type="materials methods"><h2>MATERIALS AND METHODS</h2><section id="sec-4-1"><h3>Strains and growth conditions.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> strains (see <a href="#tabS1">Table S1</a> in the supplemental material) were grown at 30Â°C in minimal medium (0.67% yeast nitrogen base) containing glucose (2%) (SD), lactic acid (2%), oleic acid (0.2%), or mixed amino acids (2%) as the sole carbon source (<a href="#B71" role="doc-biblioref" data-xml-rid="B71">71</a>).</div></section><section id="sec-4-2"><h3>Strain construction.</h3><div role="paragraph"><span class="named-content" data-type="genus-species">C. albicans</span> strains expressing Icl1-Myc<sub>3</sub> (CA1395) or Pck1-Myc<sub>3</sub> (CA1431) were made by PCR amplification of a <i>Myc</i><sub><i>3</i></sub><i>-URA3</i> cassette and transforming these products into <span class="named-content" data-type="genus-species">C. albicans</span> RM1000 (see <a href="#tabS1">Table S1</a> in the supplemental material). Insertion at the correct locus was confirmed by diagnostic PCR, and expression of Icl1-Myc<sub>3</sub> and Pck1-Myc<sub>3</sub> was confirmed by Western blotting. Analogous controls were performed for all the following strain constructions.</div><div role="paragraph">To replace a <i>CaICL1</i> allele in <span class="named-content" data-type="genus-species">C. albicans</span> with a Myc<sub>3</sub>-tagged <i>ScICL1</i> open reading frame (ORF) from <span class="named-content" data-type="genus-species">S. cerevisiae</span>, the <i>ScICL1</i> locus was first Myc<sub>3</sub> tagged by PCR amplifying the <i>Myc</i><sub><i>3</i></sub><i>-URA3</i> cassette with primers DS_3F (F stands for forward) and DS_3R (R stands for reverse) (see <a href="#tabS3">Table S3</a> in the supplemental material) and transforming this cassette into <span class="named-content" data-type="genus-species">S. cerevisiae</span> BY4743 to create strain DS3-Y30 (<a href="#tabS1">Table S1</a>). The new <i>ScICL1-Myc</i><sub><i>3</i></sub><i>-URA3</i> locus was then PCR amplified using primers DS_5F and DS_5R, and this <i>CaICL1p-ScICL1-MYC</i><sub><i>3</i></sub><i>-URA3</i> cassette was then transformed into <span class="named-content" data-type="genus-species">C. albicans</span> RM1000 to create strain DSCO1 (<i>CaICL1/CaICL1p-ScICL1-MYC<sub>3</sub>-URA3</i>) (<a href="#tabS1">Table S1</a>) and also transformed into <span class="named-content" data-type="genus-species">C. albicans</span> CA510 to create strain DSCO1 (<i>Caicl1/CaICL1p-ScICL1-MYC<sub>3</sub>-URA3</i>) (<a href="#tabS1">Table S1</a>).</div><aside id="tabS3" class="core-supplementary-material"><header>Supplemental Material</header><div class="core-content"><header>Table S3</header><div role="paragraph">Primers used in this study <a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395st3.pdf" download="mbo006121395st3.pdf">Table S3, PDF file, 0.1 MB.</a></div><footer><div class="core-copyright">Copyright Â© 2012 Sandai et al.</div><div>This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></footer></div></aside><div role="paragraph">To introduce the <i>CaICl1-ScICL1-Myc</i><sub><i>3</i></sub>-<i>URA3</i> allele into the <i>C. albicans ubi4/ubi4</i> background (<a href="#B55" role="doc-biblioref" data-xml-rid="B55">55</a>), the <i>CaICl1-ScICL1-Myc</i><sub><i>3</i></sub>-<i>URA3</i> cassette was PCR amplified using the DS_5F and DS_5R primers (see <a href="#tabS3">Table S3</a> in the supplemental material) and transformed into <span class="named-content" data-type="genus-species">C. albicans</span> DSCO to create strain DSCO3 (<i>ubi4/ubi4 CaICL1/CaICl1-ScICL1-Myc</i><sub><i>3</i></sub>-<i>URA3</i>) (<a href="#tabS1">Table S1</a>).</div><div role="paragraph"><span class="named-content" data-type="genus-species">C. albicans</span> DSCO4 expresses CaIcl1 with the carboxy-terminal ubiquitination site from ScIcl1 at its carboxy terminus (<i>CaICL1/CaICl1-Ubi-Myc</i><sub><i>3</i></sub>-<i>URA3</i>) (see <a href="#tabS1">Table S1</a> in the supplemental material). This was achieved by PCR amplifying the 3â² end of the <i>ScICL1</i> ORF along with the conjoined <i>MYC3-URA3</i> sequences from <span class="named-content" data-type="genus-species">S. cerevisiae</span> DSCO3 cells using the DS_6F and DS_6Rprimers (<a href="#tabS3">Table S3</a>) and transforming this <i>CaICl1-Ubi-Myc</i><sub><i>3</i></sub>-<i>URA3</i> cassette into <span class="named-content" data-type="genus-species">C. albicans</span> RM1000.</div><div role="paragraph">The <i>ScICL1</i> and <i>ScPCK1</i> loci were epitope tagged in <i>S. cerevisiae. ScICL1</i> was tagged with Myc<sub>9</sub> at its 3â² end by PCR amplification of the <i>MYC</i><sub>9</sub><i>-NAT</i> cassette in pYM21 (<a href="#B72" role="doc-biblioref" data-xml-rid="B72">72</a>) with primers DS_1F and DS_1R (see <a href="#tabS3">Table S3</a> in the supplemental material) and transformation of <span class="named-content" data-type="genus-species">S. cerevisiae</span> W303-1B to generate strain DS1-W10 (<i>ScICL1-MYC</i><sub>9</sub><i>-NAT</i>) (<a href="#tabS1">Table S1</a>). <i>ScPCK1</i> was tagged with hemagglutinin (a six-hemagglutinin tag [HA<sub>6</sub>]) by PCR amplification of the <i>HA</i><sub>6</sub><i>-KlTRP1</i> (<i>KlTRP1</i> stands for <span class="named-content" data-type="genus-species">Kluyveromyces lactis</span> <i>TRP1</i>) cassette in pYM3 (<a href="#B73" role="doc-biblioref" data-xml-rid="B73">73</a>) with primers DS_2F and DS_2R (<a href="#tabS3">Table S3</a>) and transformation of <span class="named-content" data-type="genus-species">S. cerevisiae</span> W303-1B to create strain DS1-W20 (<i>ScPCK1-HA</i><sub>6</sub><i>-KlTRP1</i>) (<a href="#tabS1">Table S1</a>).</div><div role="paragraph">The <i>ScICl1</i> ORF in <span class="named-content" data-type="genus-species">S. cerevisiae</span> BY4743 was replaced with <i>CaICL1-Myc</i><sub><i>3</i></sub> to create the <span class="named-content" data-type="genus-species">S. cerevisiae</span> strain DS4-Y40 (<i>ScICl1-CaICL1-Myc<sub>3</sub>-URA3</i>) (see <a href="#tabS1">Table S1</a> in the supplemental material). The <i>CaICL1-Myc</i><sub><i>3</i></sub><i>-URA3</i> ORF was PCR amplified from <span class="named-content" data-type="genus-species">C. albicans</span> CA1395 (<a href="#tabS1">Table S1</a>) using the DS_4F and DS_4R primers (<a href="#tabS3">Table S3</a>).</div></section><section id="sec-4-3"><h3>Proteomics.</h3><div role="paragraph">Replicate cultures of <span class="named-content" data-type="genus-species">C. albicans</span> NGY152 (CAI4 containing CIp10 [<a href="#B74" role="doc-biblioref" data-xml-rid="B74">74</a>]) were grown in minimal medium containing glucose (2%), lactic acid (2%), oleic acid (0.2%), or mixed amino acids (2%) as the sole carbon source. Samples from the cultures were taken, then subcultured in the same medium, grown for a further 20 h, and harvested in exponential phase (optical density at 600 nm [OD<sub>600</sub>] of 0.8). Glucose (2%) was added to some cultures, and the cells were harvested after 2 h. Protein extracts were prepared, subjected to 2D gel electrophoresis, and stained with Coomassie blue as described previously (<a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>). Independent triplicate experiments were done for each growth condition. Gel features were compared using <i>Phoretix</i> 2D Expression (Nonlinear Dynamics, Newcastle upon Tyne, United Kingdom) and GeneSpring (Silicon Genetics, San Carlos, CA). Principal component analysis scores were plotted in 3D (SIMCA-P version 11.0: Umetrics AB, Sweden). Proteins displaying statistically significant differences in mean spot volume (<i>P</i> â¤ 0.05) were identified by MALDI-ToF mass spectrometry of tryptic peptides (<a href="#B54" role="doc-biblioref" data-xml-rid="B54">54</a>) using MS-Fit and MASCOT (Matrix Science, Boston, MA). Protein annotations were from the <i>Candida</i> Genome Database (<a href="http://www.candidagenome.org/www.candidagenome.org/www.candidagenome.org/">http://www.candidagenome.org/</a>). The proteomic data set is available in the supplemental material (see <a href="#tabS2">Table S2</a> in the supplemental material) and at the PRIDE proteomic data repository (<a href="http://www.ebi.ac.uk/pride/www.ebi.ac.uk/pride/www.ebi.ac.uk/pride/">http://www.ebi.ac.uk/pride/</a>) (accession numbers 3186 to 3192). The carbon source network was constructed using Cytoscape version (<a href="http://www.cytoscape.org/www.cytoscape.org/www.cytoscape.org/">http://www.cytoscape.org/</a>) (<a href="#B75" role="doc-biblioref" data-xml-rid="B75">75</a>).</div></section><section id="sec-4-4"><h3>Western blotting.</h3><div role="paragraph">Protein extracts were subjected to Western blotting as described previously (<a href="#B76" role="doc-biblioref" data-xml-rid="B76">76</a>). Membranes were probed with a mouse anti-Myc antibody (diluted 1:10,000) (Sigma). The secondary antibody was peroxidase-conjugated rabbit anti-mouse IgG. Signals were detected with an enhanced chemiluminescence (ECL) Western blotting kit (Amersham, United Kingdom) and quantified using a Fuji FLA-3000 imager.</div></section><section id="sec-4-5"><h3>RNA analyses.</h3><div role="paragraph">RNA was prepared (<a href="#B77" role="doc-biblioref" data-xml-rid="B77">77</a>, <a href="#B78" role="doc-biblioref" data-xml-rid="B78">78</a>), yields were quantified using an RNA 6000 Nano assay, and RNA integrity was assessed using an Agilent 2100 bioanalyzer (<a href="#B79" role="doc-biblioref" data-xml-rid="B79">79</a>). <i>ICL1</i> and <i>PCK1</i> transcript levels were quantified to the internal <i>ACT1</i> mRNA by qRT-PCR using primers (see <a href="#tabS3">Table S3</a> in the supplemental material). RNA samples (2 Âµg) were incubated in 20-Âµl reactions with DNase I (1.5 Âµl), RNase OUT (1.5 Âµl), and DNase I buffer (2 Âµl) (Invitrogen, United Kingdom) at room temperature for 15 min. cDNA was then made with SuperScript II reverse transcriptase (Invitrogen, United Kingdom) following the manufacturerâs protocols. Real-time RT-PCR SYBR green (Roche, United Kingdom) was performed using the manufacturerâs instructions with a LightCycler 480 real-time PCR system (Roche).</div></section><section id="sec-4-6"><h3>Lactate and oleic acid assimilation.</h3><div role="paragraph">Lactate and oleic acid assimilation was assayed by measuring the incorporation by <span class="named-content" data-type="genus-species">C. albicans</span> and <span class="named-content" data-type="genus-species">S. cerevisiae</span> of [<sup>14</sup>C]lactate and [<sup>3</sup>H]oleic acid into trichloroacetic acid-precipitable material. Yeast cells were grown on YPLactate (2% Bacto peptone and 1% yeast extract containing 2% lactate) or YPOleic acid (2% Bacto peptone and 1% yeast extract containing 0.2% oleic acid) at 30Â°C to an OD<sub>600</sub> of 1, harvested, and resuspended in 1 ml fresh prewarmed YPLactate or YPOleic acid. Glucose (2%) was added to half of the cells, and no glucose was added to control cells. Then, 1.85 MBq of [<sup>14</sup>C]lactic acid or 37 MBq of [<sup>3</sup>H]oleic acid was added at time zero, and samples were taken at various times thereafter. The samples were precipitated in 5% trichloroacetic acid at 0Â°C, washed at 0Â°C in a series of solutions [(i) fresh 5% trichloroacetic acid containing 0.1% SDS, (ii) 50% ethanol, and (iii) 100% ethanol], dried, and subjected to scintillation counting (Packard BioScience). The results are means and standard deviations from triplicate assays. Similar results were obtained in triplicate independent experiments.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>ACKNOWLEDGMENTS</h2><div role="paragraph">We thank Esperanza Lopez-Franco for help with the construction of <span class="named-content" data-type="genus-species">C. albicans</span> Pck1-Myc and Icl1-Myc strains. We thank Phil Cash, Mike Lorenz, and Ben Distel for stimulating discussions and helpful advice. D.S. was supported by a scholarship from Universiti Sains, Malaysia. M.D.L. was the recipient of a Carnegie/Caledonian scholarship from the Carnegie Trust, and a Sir Henry Wellcome postdoctoral fellowship from the Wellcome Trust (096072). This work was also supported by the United Kingdom Biotechnology and Biological Research Council (BBS/B/06679 and BB/F00513X/1), the Wellcome Trust (080088 and 097377), and the European Commission (PITN-GA-2008-214004 and ERC-2009-AdG-249793).</div></section></div></section><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="supplementary-materials" class="core-supplementary-materials"><h2>Supplemental Material</h2><div role="list"><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf01.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf01.pdf" download="mbo006121395sf01.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf02.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf02.pdf" download="mbo006121395sf02.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf03.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf03.pdf" download="mbo006121395sf03.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf04.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf04.pdf" download="mbo006121395sf04.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf05.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf05.pdf" download="mbo006121395sf05.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf06.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf06.pdf" download="mbo006121395sf06.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395sf07.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395sf07.pdf" download="mbo006121395sf07.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395st1.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395st1.pdf" download="mbo006121395st1.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395st2.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395st2.pdf" download="mbo006121395st2.pdf">Download</a></div></div><div class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">File</span> <span class="core-filename">(mbo006121395st3.pdf)</span></div><div class="core-link"><a href="/doi/suppl/10.1128/mBio.00495-12/suppl_file/mbo006121395st3.pdf" download="mbo006121395st3.pdf">Download</a></div></div></div><footer><div role="paragraph">ASM does not own the copyrights to Supplemental Material that may be linked to, or accessed through, an article. The authors have granted ASM a non-exclusive, world-wide license to publish the Supplemental Material files. Please contact the corresponding author directly for reuse.</div></footer></section><section id="bibliography" role="doc-bibliography"><h2>REFERENCES</h2><div class="labeled" role="doc-biblioentry"><div class="label">1.</div><div id="B1" class="citations"><div class="citation"><div class="citation-content">Jacob F and Monod J. 1961. Genetic regulatory mechanisms in the synthesis of proteins. <em>J. Mol. Biol.</em> 3:318â356.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0022-2836(61)80072-7">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/13718526">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A19615792B00007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genetic+regulatory+mechanisms+in+the+synthesis+of+proteins&amp;author=F+Jacob&amp;author=J+Monod&amp;publication_year=1961&amp;pages=318-356&amp;doi=10.1016%2FS0022-2836%2861%2980072-7&amp;pmid=13718526">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">2.</div><div id="B2" class="citations"><div class="citation"><div class="citation-content">Beckwith JR. 1967. Regulation of the lac operon. Recent studies on the regulation of lactose metabolism in <span class="named-content" data-type="genus-species">Escherichia coli</span> support the operon model. <em>Science</em> 156:597â604.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.156.3775.597">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/5337175">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A19679280100006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Regulation+of+the+lac+operon.+Recent+studies+on+the+regulation+of+lactose+metabolism+in+Escherichia+coli+support+the+operon+model&amp;author=JR+Beckwith&amp;publication_year=1967&amp;pages=597-604&amp;doi=10.1126%2Fscience.156.3775.597&amp;pmid=5337175">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">3.</div><div id="B3" class="citations"><div class="citation"><div class="citation-content">Crabtree HG. 1928. The carbohydrate metabolism of certain pathological overgrowths. <em>Biochem. J.</em> 22:1289â1298.</div><div class="external-links"><div><a href="https://doi.org/10.1042/bj0221289">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16744142">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+carbohydrate+metabolism+of+certain+pathological+overgrowths&amp;author=HG+Crabtree&amp;publication_year=1928&amp;pages=1289-1298&amp;doi=10.1042%2Fbj0221289&amp;pmid=16744142">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">4.</div><div id="B4" class="citations"><div class="citation"><div class="citation-content">Barford JP and Hall RJ. 1978. An examination of the Crabtree effect in <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>: the role of respiratory adaptation. <em>J. Gen. Microbiol.</em> 114:267â275.</div><div class="external-links"><div><a href="https://doi.org/10.1099/00221287-114-2-267">Crossref</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=An+examination+of+the+Crabtree+effect+in+Saccharomyces+cerevisiae%3A+the+role+of+respiratory+adaptation&amp;author=JP+Barford&amp;author=RJ+Hall&amp;publication_year=1978&amp;pages=267-275&amp;doi=10.1099%2F00221287-114-2-267">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">5.</div><div id="B5" class="citations"><div class="citation"><div class="citation-content">Yin Z, Smith RJ, and Brown AJ. 1996. Multiple signalling pathways trigger the exquisite sensitivity of yeast gluconeogenic mRNAs to glucose. <em>Mol. Microbiol.</em> 20:751â764.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.1996.tb02514.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8793872">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1996UN98800007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Multiple+signalling+pathways+trigger+the+exquisite+sensitivity+of+yeast+gluconeogenic+mRNAs+to+glucose&amp;author=Z+Yin&amp;author=RJ+Smith&amp;author=AJ+Brown&amp;publication_year=1996&amp;pages=751-764&amp;doi=10.1111%2Fj.1365-2958.1996.tb02514.x&amp;pmid=8793872">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">6.</div><div id="B6" class="citations"><div class="citation"><div class="citation-content">Yin Z et al. 2003. Glucose triggers different global responses in yeast, depending on the strength of the signal, and transiently stabilizes ribosomal protein mRNAs. <em>Mol. Microbiol.</em> 48:713â724.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.2003.03478.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12694616">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000182191700011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Glucose+triggers+different+global+responses+in+yeast%2C+depending+on+the+strength+of+the+signal%2C+and+transiently+stabilizes+ribosomal+protein+mRNAs&amp;author=Z+Yin&amp;publication_year=2003&amp;pages=713-724&amp;doi=10.1046%2Fj.1365-2958.2003.03478.x&amp;pmid=12694616">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">7.</div><div id="B7" class="citations"><div class="citation"><div class="citation-content">Gancedo JM. 1998. Yeast carbon catabolite repression. <em>Microbiol. Mol. Biol. Rev.</em> 62:334â361.</div><div class="external-links"><div><a href="https://doi.org/10.1128/MMBR.62.2.334-361.1998">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9618445">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000074114800005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Yeast+carbon+catabolite+repression&amp;author=JM+Gancedo&amp;publication_year=1998&amp;pages=334-361&amp;doi=10.1128%2FMMBR.62.2.334-361.1998&amp;pmid=9618445">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">8.</div><div id="B8" class="citations"><div class="citation"><div class="citation-content">Carlson M. 1999. Glucose repression in yeast. <em>Curr. Opin. Microbiol.</em> 2:202â207.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S1369-5274(99)80035-6">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10322167">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000079526800015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Glucose+repression+in+yeast&amp;author=M+Carlson&amp;publication_year=1999&amp;pages=202-207&amp;doi=10.1016%2FS1369-5274%2899%2980035-6&amp;pmid=10322167">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">9.</div><div id="B9" class="citations"><div class="citation"><div class="citation-content">Johnston M. 1999. Feasting, fasting and fermenting. Glucose sensing in yeast and other cells. <em>Trends Genet.</em> 15:29â33.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0168-9525(98)01637-0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10087931">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000079419200010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Feasting%2C+fasting+and+fermenting.+Glucose+sensing+in+yeast+and+other+cells&amp;author=M+Johnston&amp;publication_year=1999&amp;pages=29-33&amp;doi=10.1016%2FS0168-9525%2898%2901637-0&amp;pmid=10087931">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">10.</div><div id="B10" class="citations"><div class="citation"><div class="citation-content">Thevelein JM and de Winde JH. 1999. Novel sensing mechanisms and targets for the cAMP-protein kinase A pathway in the yeast <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Mol. Microbiol.</em> 33:904â918.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.1999.01538.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10476026">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000082779700002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Novel+sensing+mechanisms+and+targets+for+the+cAMP-protein+kinase+A+pathway+in+the+yeast+Saccharomyces+cerevisiae&amp;author=JM+Thevelein&amp;author=JH+de+Winde&amp;publication_year=1999&amp;pages=904-918&amp;doi=10.1046%2Fj.1365-2958.1999.01538.x&amp;pmid=10476026">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">11.</div><div id="B11" class="citations"><div class="citation"><div class="citation-content">Rolland F, Winderickx J, and Thevelein JM. 2001. Glucose-sensing mechanisms in eukaryotic cells. <em>Trends Biochem. Sci.</em> 26:310â317.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0968-0004(01)01805-9">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11343924">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000168720000014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Glucose-sensing+mechanisms+in+eukaryotic+cells&amp;author=F+Rolland&amp;author=J+Winderickx&amp;author=JM+Thevelein&amp;publication_year=2001&amp;pages=310-317&amp;doi=10.1016%2FS0968-0004%2801%2901805-9&amp;pmid=11343924">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">12.</div><div id="B12" class="citations"><div class="citation"><div class="citation-content">Gancedo JM. 2008. The early steps of glucose signalling in yeast. <em>FEMS Microbiol. Rev.</em> 32:673â704.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1574-6976.2008.00117.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/18559076">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000256538900006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+early+steps+of+glucose+signalling+in+yeast&amp;author=JM+Gancedo&amp;publication_year=2008&amp;pages=673-704&amp;doi=10.1111%2Fj.1574-6976.2008.00117.x&amp;pmid=18559076">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">13.</div><div id="B13" class="citations"><div class="citation"><div class="citation-content">Gounalaki N and Thireos G. 1994. Yap1p, a yeast transcriptional activator that mediates multidrug resistance, regulates the metabolic stress response. <em>EMBO J.</em> 13:4036â4041.</div><div class="external-links"><div><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06720.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/8076599">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1994PF60600013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Yap1p%2C+a+yeast+transcriptional+activator+that+mediates+multidrug+resistance%2C+regulates+the+metabolic+stress+response&amp;author=N+Gounalaki&amp;author=G+Thireos&amp;publication_year=1994&amp;pages=4036-4041&amp;doi=10.1002%2Fj.1460-2075.1994.tb06720.x&amp;pmid=8076599">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">14.</div><div id="B14" class="citations"><div class="citation"><div class="citation-content">GÃ¶rner W et al. 1998. Nuclear localization of the C2H2 zinc finger protein Msn2p is regulated by stress and protein kinase A activity. <em>Genes Dev.</em> 12:586â597.</div><div class="external-links"><div><a href="https://doi.org/10.1101/gad.12.4.586">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9472026">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000072171100014">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nuclear+localization+of+the+C2H2+zinc+finger+protein+Msn2p+is+regulated+by+stress+and+protein+kinase+A+activity&amp;author=W+G%C3%B6rner&amp;publication_year=1998&amp;pages=586-597&amp;doi=10.1101%2Fgad.12.4.586&amp;pmid=9472026">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">15.</div><div id="B15" class="citations"><div class="citation"><div class="citation-content">Stanhill A, Schick N, and Engelberg D. 1999. The yeast ras/cyclic AMP pathway induces invasive growth by suppressing the cellular stress response. <em>Mol. Cell. Biol.</em> 19:7529â7538.</div><div class="external-links"><div><a href="https://doi.org/10.1128/MCB.19.11.7529">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10523641">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000083203100024">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+yeast+ras%2Fcyclic+AMP+pathway+induces+invasive+growth+by+suppressing+the+cellular+stress+response&amp;author=A+Stanhill&amp;author=N+Schick&amp;author=D+Engelberg&amp;publication_year=1999&amp;pages=7529-7538&amp;doi=10.1128%2FMCB.19.11.7529&amp;pmid=10523641">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">16.</div><div id="B16" class="citations"><div class="citation"><div class="citation-content">Garreau H et al. 2000. Hyperphosphorylation of Msn2p and Msn4p in response to heat shock and the diauxic shift is inhibited by cAMP in <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Microbiology</em> 146(Part 9):2113â2120.</div><div class="external-links"><div><a href="https://doi.org/10.1099/00221287-146-9-2113">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10974099">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000089268600006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Hyperphosphorylation+of+Msn2p+and+Msn4p+in+response+to+heat+shock+and+the+diauxic+shift+is+inhibited+by+cAMP+in+Saccharomyces+cerevisiae&amp;author=H+Garreau&amp;publication_year=2000&amp;pages=2113-2120&amp;doi=10.1099%2F00221287-146-9-2113&amp;pmid=10974099">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">17.</div><div id="B17" class="citations"><div class="citation"><div class="citation-content">Ozcan S and Johnston M. 1999. Function and regulation of yeast hexose transporters. <em>Microbiol. Mol. Biol. Rev.</em> 63:554â569.</div><div class="external-links"><div><a href="https://doi.org/10.1128/MMBR.63.3.554-569.1999">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10477308">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000082409400003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Function+and+regulation+of+yeast+hexose+transporters&amp;author=S+Ozcan&amp;author=M+Johnston&amp;publication_year=1999&amp;pages=554-569&amp;doi=10.1128%2FMMBR.63.3.554-569.1999&amp;pmid=10477308">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">18.</div><div id="B18" class="citations"><div class="citation"><div class="citation-content">Kaniak A, Xue Z, Macool D, Kim JH, and Johnston M. 2004. Regulatory network connecting two glucose signal transduction pathways in <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Eukaryot. Cell</em> 3:221â231.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.3.1.221-231.2004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14871952">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000189092500021">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Regulatory+network+connecting+two+glucose+signal+transduction+pathways+in+Saccharomyces+cerevisiae&amp;author=A+Kaniak&amp;author=Z+Xue&amp;author=D+Macool&amp;author=JH+Kim&amp;author=M+Johnston&amp;publication_year=2004&amp;pages=221-231&amp;doi=10.1128%2FEC.3.1.221-231.2004&amp;pmid=14871952">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">19.</div><div id="B19" class="citations"><div class="citation"><div class="citation-content">Usaite R et al. 2009. Reconstruction of the yeast Snf1 kinase regulatory network reveals its role as a global energy regulator. <em>Mol. Syst. Biol.</em> 5:319.</div><div class="external-links"><div><a href="https://doi.org/10.1038/msb.2009.67">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19888214">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000272308900001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction+of+the+yeast+Snf1+kinase+regulatory+network+reveals+its+role+as+a+global+energy+regulator&amp;author=R+Usaite&amp;publication_year=2009&amp;pages=319&amp;doi=10.1038%2Fmsb.2009.67&amp;pmid=19888214">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">20.</div><div id="B20" class="citations"><div class="citation"><div class="citation-content">Mercado JJ, Smith R, Sagliocco FA, Brown AJ, and Gancedo JM. 1994. The levels of yeast gluconeogenic mRNAs respond to environmental factors. <em>Eur. J. Biochem.</em> 224:473â481.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1432-1033.1994.00473.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/7925362">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+levels+of+yeast+gluconeogenic+mRNAs+respond+to+environmental+factors&amp;author=JJ+Mercado&amp;author=R+Smith&amp;author=FA+Sagliocco&amp;author=AJ+Brown&amp;author=JM+Gancedo&amp;publication_year=1994&amp;pages=473-481&amp;doi=10.1111%2Fj.1432-1033.1994.00473.x&amp;pmid=7925362">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">21.</div><div id="B21" class="citations"><div class="citation"><div class="citation-content">Scheffler IE, de la Cruz BJ, and Prieto S. 1998. Control of mRNA turnover as a mechanism of glucose repression in <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Int. J. Biochem. Cell Biol.</em> 30:1175â1193.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S1357-2725(98)00086-7">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9839444">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000077021400005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Control+of+mRNA+turnover+as+a+mechanism+of+glucose+repression+in+Saccharomyces+cerevisiae&amp;author=IE+Scheffler&amp;author=BJ+de+la+Cruz&amp;author=S+Prieto&amp;publication_year=1998&amp;pages=1175-1193&amp;doi=10.1016%2FS1357-2725%2898%2900086-7&amp;pmid=9839444">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">22.</div><div id="B22" class="citations"><div class="citation"><div class="citation-content">Ashe MP, De Long SK, and Sachs AB. 2000. Glucose depletion rapidly inhibits translation initiation in yeast. <em>Mol. Biol. Cell</em> 11:833â848.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.11.3.833">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10712503">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000085929500005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Glucose+depletion+rapidly+inhibits+translation+initiation+in+yeast&amp;author=MP+Ashe&amp;author=SK+De+Long&amp;author=AB+Sachs&amp;publication_year=2000&amp;pages=833-848&amp;doi=10.1091%2Fmbc.11.3.833&amp;pmid=10712503">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">23.</div><div id="B23" class="citations"><div class="citation"><div class="citation-content">Entian KD, DrÃ¶ll L, and Mecke D. 1983. Studies on rapid reversible and non-reversible inactivation of fructose-1,6-bisphosphatase and malate dehydrogenase in wild-type and glycolytic block mutants of <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Arch. Microbiol.</em> 134:187â192.</div><div class="external-links"><div><a href="https://doi.org/10.1007/BF00407756">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/6311131">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1983QV82900004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Studies+on+rapid+reversible+and+non-reversible+inactivation+of+fructose-1%2C6-bisphosphatase+and+malate+dehydrogenase+in+wild-type+and+glycolytic+block+mutants+of+Saccharomyces+cerevisiae&amp;author=KD+Entian&amp;author=L+Dr%C3%B6ll&amp;author=D+Mecke&amp;publication_year=1983&amp;pages=187-192&amp;doi=10.1007%2FBF00407756&amp;pmid=6311131">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">24.</div><div id="B24" class="citations"><div class="citation"><div class="citation-content">LÃ³pez-Boado YS, Herrero P, GascÃ³n S, and Moreno F. 1987. Catabolite inactivation of isocitrate lyase from <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Arch. Microbiol.</em> 147:231â234.</div><div class="external-links"><div><a href="https://doi.org/10.1007/BF00463480">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3036035">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1987G794500005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Catabolite+inactivation+of+isocitrate+lyase+from+Saccharomyces+cerevisiae&amp;author=YS+L%C3%B3pez-Boado&amp;author=P+Herrero&amp;author=S+Gasc%C3%B3n&amp;author=F+Moreno&amp;publication_year=1987&amp;pages=231-234&amp;doi=10.1007%2FBF00463480&amp;pmid=3036035">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">25.</div><div id="B25" class="citations"><div class="citation"><div class="citation-content">HÃ¤mmerle M et al. 1998. Proteins of newly isolated mutants and the amino-terminal proline are essential for ubiquitin-proteasome-catalyzed catabolite degradation of fructose-1,6-bisphosphatase of <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>J. Biol. Chem.</em> 273:25000â25005.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.273.39.25000">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/9737955">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000076085400005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proteins+of+newly+isolated+mutants+and+the+amino-terminal+proline+are+essential+for+ubiquitin-proteasome-catalyzed+catabolite+degradation+of+fructose-1%2C6-bisphosphatase+of+Saccharomyces+cerevisiae&amp;author=M+H%C3%A4mmerle&amp;publication_year=1998&amp;pages=25000-25005&amp;doi=10.1074%2Fjbc.273.39.25000&amp;pmid=9737955">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">26.</div><div id="B26" class="citations"><div class="citation"><div class="citation-content">Jiang H, Tatchell K, Liu S, and Michels CA. 2000. Protein phosphatase type-1 regulatory subunits Reg1p and Reg2p act as signal transducers in the glucose-induced inactivation of maltose permease in <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Mol. Gen. Genet.</em> 263:411â422.</div><div class="external-links"><div><a href="https://doi.org/10.1007/s004380051185">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10821175">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Protein+phosphatase+type-1+regulatory+subunits+Reg1p+and+Reg2p+act+as+signal+transducers+in+the+glucose-induced+inactivation+of+maltose+permease+in+Saccharomyces+cerevisiae&amp;author=H+Jiang&amp;author=K+Tatchell&amp;author=S+Liu&amp;author=CA+Michels&amp;publication_year=2000&amp;pages=411-422&amp;doi=10.1007%2Fs004380051185&amp;pmid=10821175">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">27.</div><div id="B27" class="citations"><div class="citation"><div class="citation-content">Horak J, Regelmann J, and Wolf DH. 2002. Two distinct proteolytic systems responsible for glucose-induced degradation of fructose-1,6-bisphosphatase and the Gal2p transporter in the yeast <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span> share the same protein components of the glucose signaling pathway. <em>J. Biol. Chem.</em> 277:8248â8254.</div><div class="external-links"><div><a href="https://doi.org/10.1074/jbc.M107255200">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11773046">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000174268000080">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Two+distinct+proteolytic+systems+responsible+for+glucose-induced+degradation+of+fructose-1%2C6-bisphosphatase+and+the+Gal2p+transporter+in+the+yeast+Saccharomyces+cerevisiae+share+the+same+protein+components+of+the+glucose+signaling+pathway&amp;author=J+Horak&amp;author=J+Regelmann&amp;author=DH+Wolf&amp;publication_year=2002&amp;pages=8248-8254&amp;doi=10.1074%2Fjbc.M107255200&amp;pmid=11773046">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">28.</div><div id="B28" class="citations"><div class="citation"><div class="citation-content">Regelmann J et al. 2003. Catabolite degradation of fructose-1,6-bisphosphatase in the yeast <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>: a genome-wide screen identifies eight novel GID genes and indicates the existence of two degradation pathways. <em>Mol. Biol. Cell</em> 14:1652â1663.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e02-08-0456">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12686616">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000182185200033">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Catabolite+degradation+of+fructose-1%2C6-bisphosphatase+in+the+yeast+Saccharomyces+cerevisiae%3A+a+genome-wide+screen+identifies+eight+novel+GID+genes+and+indicates+the+existence+of+two+degradation+pathways&amp;author=J+Regelmann&amp;publication_year=2003&amp;pages=1652-1663&amp;doi=10.1091%2Fmbc.e02-08-0456&amp;pmid=12686616">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">29.</div><div id="B29" class="citations"><div class="citation"><div class="citation-content">Mercado JJ, Vincent O, and Gancedo JM. 1991. Regions in the promoter of the yeast <i>FBP1</i> gene implicated in catabolite repression may bind the product of the regulatory gene <i>MIG1</i>. <em>FEBS Lett.</em> 291:97â100.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0014-5793(91)81112-L">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/1657641">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1991GL30000024">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Regions+in+the+promoter+of+the+yeast+FBP1+gene+implicated+in+catabolite+repression+may+bind+the+product+of+the+regulatory+gene+MIG1&amp;author=JJ+Mercado&amp;author=O+Vincent&amp;author=JM+Gancedo&amp;publication_year=1991&amp;pages=97-100&amp;doi=10.1016%2F0014-5793%2891%2981112-L&amp;pmid=1657641">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">30.</div><div id="B30" class="citations"><div class="citation"><div class="citation-content">Mercado JJ and Gancedo JM. 1992. Regulatory regions in the yeast <i>FBP1</i> and <i>PCK1</i> genes. <em>FEBS Lett.</em> 311:110â114.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0014-5793(92)81379-Z">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/1327878">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1992JT76500008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Regulatory+regions+in+the+yeast+FBP1+and+PCK1+genes&amp;author=JJ+Mercado&amp;author=JM+Gancedo&amp;publication_year=1992&amp;pages=110-114&amp;doi=10.1016%2F0014-5793%2892%2981379-Z&amp;pmid=1327878">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">31.</div><div id="B31" class="citations"><div class="citation"><div class="citation-content">Gancedo JM and Gancedo C. 1997. Gluconeogenesis and catabolite inactivation, p 359â377. <i>In</i> Zimmermann FK and Entian KD, <em>Yeast sugar metabolism</em>. Technomic Publishing, Basel, Switzerland.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Gluconeogenesis+and+catabolite+inactivation&amp;author=JM+Gancedo&amp;author=C+Gancedo&amp;publication_year=1997&amp;pages=359-377">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">32.</div><div id="B32" class="citations"><div class="citation"><div class="citation-content">SchÃ¼le T, Rose M, Entian KD, Thumm M, and Wolf DH. 2000. Ubc8p functions in catabolite degradation of fructose-1,6-bisphosphatase in yeast. <em>EMBO J.</em> 19:2161â2167.</div><div class="external-links"><div><a href="https://doi.org/10.1093/emboj/19.10.2161">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10811607">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000087140800003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Ubc8p+functions+in+catabolite+degradation+of+fructose-1%2C6-bisphosphatase+in+yeast&amp;author=T+Sch%C3%BCle&amp;author=M+Rose&amp;author=KD+Entian&amp;author=M+Thumm&amp;author=DH+Wolf&amp;publication_year=2000&amp;pages=2161-2167&amp;doi=10.1093%2Femboj%2F19.10.2161&amp;pmid=10811607">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">33.</div><div id="B33" class="citations"><div class="citation"><div class="citation-content">Sabina J and Brown V. 2009. Glucose sensing network in <span class="named-content" data-type="genus-species">Candida albicans</span>: a sweet spot for fungal morphogenesis. <em>Eukaryot. Cell</em> 8:1314â1320.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.00138-09">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19617394">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000270399000001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Glucose+sensing+network+in+Candida+albicans%3A+a+sweet+spot+for+fungal+morphogenesis&amp;author=J+Sabina&amp;author=V+Brown&amp;publication_year=2009&amp;pages=1314-1320&amp;doi=10.1128%2FEC.00138-09&amp;pmid=19617394">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">34.</div><div id="B34" class="citations"><div class="citation"><div class="citation-content">Merico A, Sulo P, Piskur J, and Compagno C. 2007. Fermentative lifestyle in yeasts belonging to the <i>Saccharomyces</i> complex. <em>FEBS J.</em> 274:976â989.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1742-4658.2007.05645.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17239085">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000243940800008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Fermentative+lifestyle+in+yeasts+belonging+to+the+Saccharomyces+complex&amp;author=A+Merico&amp;author=P+Sulo&amp;author=J+Piskur&amp;author=C+Compagno&amp;publication_year=2007&amp;pages=976-989&amp;doi=10.1111%2Fj.1742-4658.2007.05645.x&amp;pmid=17239085">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">35.</div><div id="B35" class="citations"><div class="citation"><div class="citation-content">Odds FC. 1988. <em>Candida and candidosis</em>. Bailliere Tindall, London, United Kingdom.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Candida+and+candidosis&amp;author=FC+Odds&amp;publication_year=1988">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">36.</div><div id="B36" class="citations"><div class="citation"><div class="citation-content">Calderone RA. 2002. <em>Candida and candidiasis</em>. ASM Press, Washington, DC.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Candida+and+candidiasis&amp;author=RA+Calderone&amp;publication_year=2002">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">37.</div><div id="B37" class="citations"><div class="citation"><div class="citation-content">Buchalter SE, Crain MR, and Kreisberg R. 1989. Regulation of lactate metabolism in vivo. <em>Diabetes Metab. Rev.</em> 5:379â391.</div><div class="external-links"><div><a href="https://doi.org/10.1002/dmr.5610050405">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/2656161">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1989U661100004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Regulation+of+lactate+metabolism+in+vivo&amp;author=SE+Buchalter&amp;author=MR+Crain&amp;author=R+Kreisberg&amp;publication_year=1989&amp;pages=379-391&amp;doi=10.1002%2Fdmr.5610050405&amp;pmid=2656161">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">38.</div><div id="B38" class="citations"><div class="citation"><div class="citation-content">Ueno K et al. 2011. Intestinal resident yeast <span class="named-content" data-type="genus-species">Candida glabrata</span> requires Cyb2p-mediated lactate assimilation to adapt in mouse intestine. <em>PLoS One</em> 6:e24759.</div><div class="external-links"><div><a href="https://doi.org/10.1371/journal.pone.0024759">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21931845">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000294803100065">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Intestinal+resident+yeast+Candida+glabrata+requires+Cyb2p-mediated+lactate+assimilation+to+adapt+in+mouse+intestine&amp;author=K+Ueno&amp;publication_year=2011&amp;pages=e24759&amp;doi=10.1371%2Fjournal.pone.0024759&amp;pmid=21931845">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">39.</div><div id="B39" class="citations"><div class="citation"><div class="citation-content">Lorenz MC and Fink GR. 2001. The glyoxylate cycle is required for fungal virulence. <em>Nature</em> 412:83â86.</div><div class="external-links"><div><a href="https://doi.org/10.1038/35083594">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11452311">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000169644900048">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+glyoxylate+cycle+is+required+for+fungal+virulence&amp;author=MC+Lorenz&amp;author=GR+Fink&amp;publication_year=2001&amp;pages=83-86&amp;doi=10.1038%2F35083594&amp;pmid=11452311">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">40.</div><div id="B40" class="citations"><div class="citation"><div class="citation-content">Barelle CJ et al. 2006. Niche-specific regulation of central metabolic pathways in a fungal pathogen. <em>Cell. Microbiol.</em> 8:961â971.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2005.00676.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16681837">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000237394900006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Niche-specific+regulation+of+central+metabolic+pathways+in+a+fungal+pathogen&amp;author=CJ+Barelle&amp;publication_year=2006&amp;pages=961-971&amp;doi=10.1111%2Fj.1462-5822.2005.00676.x&amp;pmid=16681837">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">41.</div><div id="B41" class="citations"><div class="citation"><div class="citation-content">Piekarska K et al. 2006. Peroxisomal fatty acid beta-oxidation is not essential for virulence of <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Eukaryot. Cell</em> 5:1847â1856.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.00093-06">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16963628">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000242398100003">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Peroxisomal+fatty+acid+beta-oxidation+is+not+essential+for+virulence+of+Candida+albicans&amp;author=K+Piekarska&amp;publication_year=2006&amp;pages=1847-1856&amp;doi=10.1128%2FEC.00093-06&amp;pmid=16963628">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">42.</div><div id="B42" class="citations"><div class="citation"><div class="citation-content">RamÃ­rez MA and Lorenz MC. 2007. Mutations in alternative carbon utilization pathways in <span class="named-content" data-type="genus-species">Candida albicans</span> attenuate virulence and confer pleiotropic phenotypes. <em>Eukaryot. Cell</em> 6:280â290.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.00372-06">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17158734">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000244937800015">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Mutations+in+alternative+carbon+utilization+pathways+in+Candida+albicans+attenuate+virulence+and+confer+pleiotropic+phenotypes&amp;author=MA+Ram%C3%ADrez&amp;author=MC+Lorenz&amp;publication_year=2007&amp;pages=280-290&amp;doi=10.1128%2FEC.00372-06&amp;pmid=17158734">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">43.</div><div id="B43" class="citations"><div class="citation"><div class="citation-content">Ihmels J et al. 2005. Rewiring of the yeast transcriptional network through the evolution of motif usage. <em>Science</em> 309:938â940.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1113833">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16081737">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000231101400048">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rewiring+of+the+yeast+transcriptional+network+through+the+evolution+of+motif+usage&amp;author=J+Ihmels&amp;publication_year=2005&amp;pages=938-940&amp;doi=10.1126%2Fscience.1113833&amp;pmid=16081737">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">44.</div><div id="B44" class="citations"><div class="citation"><div class="citation-content">Martchenko M, Levitin A, Hogues H, Nantel A, and Whiteway M. 2007. Transcriptional rewiring of fungal galactose-metabolism circuitry. <em>Curr. Biol.</em> 17:1007â1013.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.cub.2007.05.017">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17540568">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000247409000021">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional+rewiring+of+fungal+galactose-metabolism+circuitry&amp;author=M+Martchenko&amp;author=A+Levitin&amp;author=H+Hogues&amp;author=A+Nantel&amp;author=M+Whiteway&amp;publication_year=2007&amp;pages=1007-1013&amp;doi=10.1016%2Fj.cub.2007.05.017&amp;pmid=17540568">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">45.</div><div id="B45" class="citations"><div class="citation"><div class="citation-content">Lavoie H, Hogues H, and Whiteway M. 2009. Rearrangements of the transcriptional regulatory networks of metabolic pathways in fungi. <em>Curr. Opin. Microbiol.</em> 12:655â663.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.mib.2009.09.015">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19875326">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000272479400011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rearrangements+of+the+transcriptional+regulatory+networks+of+metabolic+pathways+in+fungi&amp;author=H+Lavoie&amp;author=H+Hogues&amp;author=M+Whiteway&amp;publication_year=2009&amp;pages=655-663&amp;doi=10.1016%2Fj.mib.2009.09.015&amp;pmid=19875326">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">46.</div><div id="B46" class="citations"><div class="citation"><div class="citation-content">Lorenz MC, Bender JA, and Fink GR. 2004. Transcriptional response of <span class="named-content" data-type="genus-species">Candida albicans</span> upon internalization by macrophages. <em>Eukaryot. Cell</em> 3:1076â1087.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.3.5.1076-1087.2004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15470236">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000224822300002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional+response+of+Candida+albicans+upon+internalization+by+macrophages&amp;author=MC+Lorenz&amp;author=JA+Bender&amp;author=GR+Fink&amp;publication_year=2004&amp;pages=1076-1087&amp;doi=10.1128%2FEC.3.5.1076-1087.2004&amp;pmid=15470236">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">47.</div><div id="B47" class="citations"><div class="citation"><div class="citation-content">Rodaki A et al. 2009. Glucose promotes stress resistance in the fungal pathogen <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Mol. Biol. Cell</em> 20:4845â4855.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e09-01-0002">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19759180">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000271769900020">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Glucose+promotes+stress+resistance+in+the+fungal+pathogen+Candida+albicans&amp;author=A+Rodaki&amp;publication_year=2009&amp;pages=4845-4855&amp;doi=10.1091%2Fmbc.e09-01-0002&amp;pmid=19759180">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">48.</div><div id="B48" class="citations"><div class="citation"><div class="citation-content">Niimi M, Kamiyama A, and Tokunaga M. 1988. Respiration of medically important <i>Candida</i> species and <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span> in relation to glucose effect. <em>J. Med. Vet. Mycol.</em> 26:195â198.</div><div class="external-links"><div><a href="https://doi.org/10.1080/02681218880000271">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/3050010">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Respiration+of+medically+important+Candida+species+and+Saccharomyces+cerevisiae+in+relation+to+glucose+effect&amp;author=M+Niimi&amp;author=A+Kamiyama&amp;author=M+Tokunaga&amp;publication_year=1988&amp;pages=195-198&amp;doi=10.1080%2F02681218880000271&amp;pmid=3050010">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">49.</div><div id="B49" class="citations"><div class="citation"><div class="citation-content">Brown AJ. 2005. Integration of metabolism with virulence in <span class="named-content" data-type="genus-species">Candida albicans</span>, p 185â203. <i>In</i> Brown AJ, <em>Fungal genomics. Mycota XIII</em>. Springer-Verlag, Heidelberg, Germany.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Integration+of+metabolism+with+virulence+in+Candida+albicans&amp;author=AJ+Brown&amp;publication_year=2005&amp;pages=185-203">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">50.</div><div id="B50" class="citations"><div class="citation"><div class="citation-content">Ene IV et al. 2012. Host carbon sources modulate cell wall architecture, drug resistance and virulence in a fungal pathogen. <em>Cell. Microbiol.</em> 14:1319â1335.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1462-5822.2012.01813.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22587014">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000307844700001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Host+carbon+sources+modulate+cell+wall+architecture%2C+drug+resistance+and+virulence+in+a+fungal+pathogen&amp;author=IV+Ene&amp;publication_year=2012&amp;pages=1319-1335&amp;doi=10.1111%2Fj.1462-5822.2012.01813.x&amp;pmid=22587014">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">51.</div><div id="B51" class="citations"><div class="citation"><div class="citation-content">Futcher B, Latter GI, Monardo P, McLaughlin CS, and Garrels JI. 1999. A sampling of the yeast proteome. <em>Mol. Cell. Biol.</em> 19:7357â7368.</div><div class="external-links"><div><a href="https://doi.org/10.1128/MCB.19.11.7357">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10523624">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000083203100007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+sampling+of+the+yeast+proteome&amp;author=B+Futcher&amp;author=GI+Latter&amp;author=P+Monardo&amp;author=CS+McLaughlin&amp;author=JI+Garrels&amp;publication_year=1999&amp;pages=7357-7368&amp;doi=10.1128%2FMCB.19.11.7357&amp;pmid=10523624">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">52.</div><div id="B52" class="citations"><div class="citation"><div class="citation-content">Gygi SP, Rochon Y, Franza BR, and Aebersold R. 1999. Correlation between protein and mRNA abundance in yeast. <em>Mol. Cell. Biol.</em> 19:1720â1730.</div><div class="external-links"><div><a href="https://doi.org/10.1128/MCB.19.3.1720">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10022859">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000078697900010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Correlation+between+protein+and+mRNA+abundance+in+yeast&amp;author=SP+Gygi&amp;author=Y+Rochon&amp;author=BR+Franza&amp;author=R+Aebersold&amp;publication_year=1999&amp;pages=1720-1730&amp;doi=10.1128%2FMCB.19.3.1720&amp;pmid=10022859">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">53.</div><div id="B53" class="citations"><div class="citation"><div class="citation-content">de Nobel H et al. 2001. Parallel and comparative analysis of the proteome and transcriptome of sorbic acid-stressed <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Yeast</em> 18:1413â1428.</div><div class="external-links"><div><a href="https://doi.org/10.1002/yea.793">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11746603">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000172193000006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Parallel+and+comparative+analysis+of+the+proteome+and+transcriptome+of+sorbic+acid-stressed+Saccharomyces+cerevisiae&amp;author=H+de+Nobel&amp;publication_year=2001&amp;pages=1413-1428&amp;doi=10.1002%2Fyea.793&amp;pmid=11746603">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">54.</div><div id="B54" class="citations"><div class="citation"><div class="citation-content">Yin Z et al. 2004. Proteomic response to amino acid starvation in <span class="named-content" data-type="genus-species">Candida albicans</span> and <span class="named-content" data-type="genus-species">Saccharomyces cerevisiae</span>. <em>Proteomics</em> 4:2425â2436.</div><div class="external-links"><div><a href="https://doi.org/10.1002/pmic.200300760">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15274137">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000223137300025">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proteomic+response+to+amino+acid+starvation+in+Candida+albicans+and+Saccharomyces+cerevisiae&amp;author=Z+Yin&amp;publication_year=2004&amp;pages=2425-2436&amp;doi=10.1002%2Fpmic.200300760&amp;pmid=15274137">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">55.</div><div id="B55" class="citations"><div class="citation"><div class="citation-content">Leach MD, Stead DA, Argo E, MacCallum DM, and Brown AJ. 2011. Molecular and proteomic analyses highlight the importance of ubiquitination for the stress resistance, metabolic adaptation, morphogenetic regulation and virulence of <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Mol. Microbiol.</em> 79:1574â1593.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2011.07542.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/21269335">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000288164900013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Molecular+and+proteomic+analyses+highlight+the+importance+of+ubiquitination+for+the+stress+resistance%2C+metabolic+adaptation%2C+morphogenetic+regulation+and+virulence+of+Candida+albicans&amp;author=MD+Leach&amp;author=DA+Stead&amp;author=E+Argo&amp;author=DM+MacCallum&amp;author=AJ+Brown&amp;publication_year=2011&amp;pages=1574-1593&amp;doi=10.1111%2Fj.1365-2958.2011.07542.x&amp;pmid=21269335">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">56.</div><div id="B56" class="citations"><div class="citation"><div class="citation-content">Radivojac P et al. 2010. Identification, analysis, and prediction of protein ubiquitination sites. <em>Proteins</em> 78:365â380.</div><div class="external-links"><div><a href="https://doi.org/10.1002/prot.22555">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19722269">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000273546600013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Identification%2C+analysis%2C+and+prediction+of+protein+ubiquitination+sites&amp;author=P+Radivojac&amp;publication_year=2010&amp;pages=365-380&amp;doi=10.1002%2Fprot.22555&amp;pmid=19722269">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">57.</div><div id="B57" class="citations"><div class="citation"><div class="citation-content">Fradin C et al. 2005. Granulocytes govern the transcriptional response, morphology and proliferation of <span class="named-content" data-type="genus-species">Candida albicans</span> in human blood. <em>Mol. Microbiol.</em> 56:397â415.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2005.04557.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15813733">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000227897400010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Granulocytes+govern+the+transcriptional+response%2C+morphology+and+proliferation+of+Candida+albicans+in+human+blood&amp;author=C+Fradin&amp;publication_year=2005&amp;pages=397-415&amp;doi=10.1111%2Fj.1365-2958.2005.04557.x&amp;pmid=15813733">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">58.</div><div id="B58" class="citations"><div class="citation"><div class="citation-content">Kusch H et al. 2008. A proteomic view of <span class="named-content" data-type="genus-species">Candida albicans</span> yeast cell metabolism in exponential and stationary growth phases. <em>Int. J. Med. Microbiol.</em> 298:291â318.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.ijmm.2007.03.020">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17588813">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000254839600011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+proteomic+view+of+Candida+albicans+yeast+cell+metabolism+in+exponential+and+stationary+growth+phases&amp;author=H+Kusch&amp;publication_year=2008&amp;pages=291-318&amp;doi=10.1016%2Fj.ijmm.2007.03.020&amp;pmid=17588813">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">59.</div><div id="B59" class="citations"><div class="citation"><div class="citation-content">Leng P, Sudbery PE, and Brown AJ. 2000. Rad6p represses yeast-hypha morphogenesis in the human fungal pathogen <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Mol. Microbiol.</em> 35:1264â1275.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.2000.01801.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10712706">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000085625800027">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Rad6p+represses+yeast-hypha+morphogenesis+in+the+human+fungal+pathogen+Candida+albicans&amp;author=P+Leng&amp;author=PE+Sudbery&amp;author=AJ+Brown&amp;publication_year=2000&amp;pages=1264-1275&amp;doi=10.1046%2Fj.1365-2958.2000.01801.x&amp;pmid=10712706">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">60.</div><div id="B60" class="citations"><div class="citation"><div class="citation-content">Roig P and Gozalbo D. 2003. Depletion of polyubiquitin encoded by the UBI4 gene confers pleiotropic phenotype to <span class="named-content" data-type="genus-species">Candida albicans</span> cells. <em>Fungal Genet. Biol.</em> 39:70â81.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S1087-1845(03)00004-5">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12742065">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000182941400007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Depletion+of+polyubiquitin+encoded+by+the+UBI4+gene+confers+pleiotropic+phenotype+to+Candida+albicans+cells&amp;author=P+Roig&amp;author=D+Gozalbo&amp;publication_year=2003&amp;pages=70-81&amp;doi=10.1016%2FS1087-1845%2803%2900004-5&amp;pmid=12742065">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">61.</div><div id="B61" class="citations"><div class="citation"><div class="citation-content">Atir-Lande A, Gildor T, and Kornitzer D. 2005. Role for the SCFCDC4 ubiquitin ligase in <span class="named-content" data-type="genus-species">Candida albicans</span> morphogenesis. <em>Mol. Biol. Cell</em> 16:2772â2785.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e05-01-0079">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15814839">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000229468400016">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Role+for+the+SCFCDC4+ubiquitin+ligase+in+Candida+albicans+morphogenesis&amp;author=A+Atir-Lande&amp;author=T+Gildor&amp;author=D+Kornitzer&amp;publication_year=2005&amp;pages=2772-2785&amp;doi=10.1091%2Fmbc.e05-01-0079&amp;pmid=15814839">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">62.</div><div id="B62" class="citations"><div class="citation"><div class="citation-content">Li WJ et al. 2006. The F-box protein Grr1 regulates the stability of Ccn1, Cln3 and Hof1 and cell morphogenesis in <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Mol. Microbiol.</em> 62:212â226.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2006.05361.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16987179">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000240440700017">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=The+F-box+protein+Grr1+regulates+the+stability+of+Ccn1%2C+Cln3+and+Hof1+and+cell+morphogenesis+in+Candida+albicans&amp;author=WJ+Li&amp;publication_year=2006&amp;pages=212-226&amp;doi=10.1111%2Fj.1365-2958.2006.05361.x&amp;pmid=16987179">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">63.</div><div id="B63" class="citations"><div class="citation"><div class="citation-content">Trunk K et al. 2009. Depletion of the cullin Cdc53p induces morphogenetic changes in <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Eukaryot. Cell</em> 8:756â767.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.00332-08">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19270112">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000265906000009">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Depletion+of+the+cullin+Cdc53p+induces+morphogenetic+changes+in+Candida+albicans&amp;author=K+Trunk&amp;publication_year=2009&amp;pages=756-767&amp;doi=10.1128%2FEC.00332-08&amp;pmid=19270112">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">64.</div><div id="B64" class="citations"><div class="citation"><div class="citation-content">Leach MD and Brown AJ. 2012. Posttranslational modifications of proteins in the pathobiology of medically relevant fungi. <em>Eukaryot. Cell</em> 11:98â108.</div><div class="external-links"><div><a href="https://doi.org/10.1128/EC.05238-11">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/22158711">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000300045800002">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Posttranslational+modifications+of+proteins+in+the+pathobiology+of+medically+relevant+fungi&amp;author=MD+Leach&amp;author=AJ+Brown&amp;publication_year=2012&amp;pages=98-108&amp;doi=10.1128%2FEC.05238-11&amp;pmid=22158711">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">65.</div><div id="B65" class="citations"><div class="citation"><div class="citation-content">Roig P, MartÃ­nez JP, Gil ML, and Gozalbo D. 2000. Molecular cloning and characterization of the <span class="named-content" data-type="genus-species">Candida albicans</span> <i>UBI3</i> gene coding for a ubiquitin-hybrid protein. <em>Yeast</em> 16:1413â1419.</div><div class="external-links"><div><a href="https://doi.org/10.1002/1097-0061(200011)16:15%3C1413::AID-YEA632%3E3.0.CO;2-U">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11054822">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000165780900007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Molecular+cloning+and+characterization+of+the+Candida+albicans+UBI3+gene+coding+for+a+ubiquitin-hybrid+protein&amp;author=P+Roig&amp;author=JP+Mart%C3%ADnez&amp;author=ML+Gil&amp;author=D+Gozalbo&amp;publication_year=2000&amp;pages=1413-1419&amp;doi=10.1002%2F1097-0061%28200011%2916%3A15%253C1413%3A%3AAID-YEA632%253E3.0.CO%3B2-U&amp;pmid=11054822">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">66.</div><div id="B66" class="citations"><div class="citation"><div class="citation-content">Nantel A et al. 2002. Transcription profiling of <span class="named-content" data-type="genus-species">Candida albicans</span> cells undergoing the yeast-to-hyphal transition. <em>Mol. Biol. Cell</em> 13:3452â3465.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e02-05-0272">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12388749">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000178784300008">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Transcription+profiling+of+Candida+albicans+cells+undergoing+the+yeast-to-hyphal+transition&amp;author=A+Nantel&amp;publication_year=2002&amp;pages=3452-3465&amp;doi=10.1091%2Fmbc.e02-05-0272&amp;pmid=12388749">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">67.</div><div id="B67" class="citations"><div class="citation"><div class="citation-content">Enjalbert B, Nantel A, and Whiteway M. 2003. Stress-induced gene expression in <span class="named-content" data-type="genus-species">Candida albicans</span>: absence of a general stress response. <em>Mol. Biol. Cell</em> 14:1460â1467.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e02-08-0546">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12686601">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000182185200018">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Stress-induced+gene+expression+in+Candida+albicans%3A+absence+of+a+general+stress+response&amp;author=B+Enjalbert&amp;author=A+Nantel&amp;author=M+Whiteway&amp;publication_year=2003&amp;pages=1460-1467&amp;doi=10.1091%2Fmbc.e02-08-0546&amp;pmid=12686601">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">68.</div><div id="B68" class="citations"><div class="citation"><div class="citation-content">Tsong AE, Miller MG, Raisner RM, and Johnson AD. 2003. Evolution of a combinatorial transcriptional circuit: a case study in yeasts. <em>Cell</em> 115:389â399.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0092-8674(03)00885-7">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14622594">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000186627100006">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Evolution+of+a+combinatorial+transcriptional+circuit%3A+a+case+study+in+yeasts&amp;author=AE+Tsong&amp;author=MG+Miller&amp;author=RM+Raisner&amp;author=AD+Johnson&amp;publication_year=2003&amp;pages=389-399&amp;doi=10.1016%2FS0092-8674%2803%2900885-7&amp;pmid=14622594">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">69.</div><div id="B69" class="citations"><div class="citation"><div class="citation-content">Yin Z et al. 2009. A proteomic analysis of the salt, cadmium and peroxide stress responses in <span class="named-content" data-type="genus-species">Candida albicans</span> and the role of the Hog1 stress-activated MAPK in regulating the stress-induced proteome. <em>Proteomics</em> 9:4686â4703.</div><div class="external-links"><div><a href="https://doi.org/10.1002/pmic.200800958">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19824012">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000271554100013">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+proteomic+analysis+of+the+salt%2C+cadmium+and+peroxide+stress+responses+in+Candida+albicans+and+the+role+of+the+Hog1+stress-activated+MAPK+in+regulating+the+stress-induced+proteome&amp;author=Z+Yin&amp;publication_year=2009&amp;pages=4686-4703&amp;doi=10.1002%2Fpmic.200800958&amp;pmid=19824012">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">70.</div><div id="B70" class="citations"><div class="citation"><div class="citation-content">Thewes S et al. 2007. <i>In vivo</i> and <i>ex vivo</i> comparative transcriptional profiling of invasive and non-invasive <span class="named-content" data-type="genus-species">Candida albicans</span> isolates identifies genes associated with tissue invasion. <em>Mol. Microbiol.</em> 63:1606â1628.</div><div class="external-links"><div><a href="https://doi.org/10.1111/j.1365-2958.2007.05614.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17367383">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000244886000005">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=In+vivo+and+ex+vivo+comparative+transcriptional+profiling+of+invasive+and+non-invasive+Candida+albicans+isolates+identifies+genes+associated+with+tissue+invasion&amp;author=S+Thewes&amp;publication_year=2007&amp;pages=1606-1628&amp;doi=10.1111%2Fj.1365-2958.2007.05614.x&amp;pmid=17367383">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">71.</div><div id="B71" class="citations"><div class="citation"><div class="citation-content">Sherman F. 1991. Getting started with yeast. <em>Methods Enzymol.</em> 194:3â21.</div><div class="external-links"><div><a href="https://doi.org/10.1016/0076-6879(91)94004-V">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/2005794">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=A1991FN84200001">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Getting+started+with+yeast&amp;author=F+Sherman&amp;publication_year=1991&amp;pages=3-21&amp;doi=10.1016%2F0076-6879%2891%2994004-V&amp;pmid=2005794">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">72.</div><div id="B72" class="citations"><div class="citation"><div class="citation-content">Janke C et al. 2004. A versatile toolbox for PCR-based tagging of yeast genes: new fluorescent proteins, more markers and promoter substitution cassettes. <em>Yeast</em> 21:947â962.</div><div class="external-links"><div><a href="https://doi.org/10.1002/yea.1142">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15334558">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000223564100004">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+versatile+toolbox+for+PCR-based+tagging+of+yeast+genes%3A+new+fluorescent+proteins%2C+more+markers+and+promoter+substitution+cassettes&amp;author=C+Janke&amp;publication_year=2004&amp;pages=947-962&amp;doi=10.1002%2Fyea.1142&amp;pmid=15334558">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">73.</div><div id="B73" class="citations"><div class="citation"><div class="citation-content">Knop M et al. 1999. Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines. <em>Yeast</em> 15:963â972.</div><div class="external-links"><div><a href="https://doi.org/10.1002/(SICI)1097-0061(199907)15:10B%3C963::AID-YEA399%3E3.0.CO;2-W">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10407276">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000081644400007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Epitope+tagging+of+yeast+genes+using+a+PCR-based+strategy%3A+more+tags+and+improved+practical+routines&amp;author=M+Knop&amp;publication_year=1999&amp;pages=963-972&amp;doi=10.1002%2F%28SICI%291097-0061%28199907%2915%3A10B%253C963%3A%3AAID-YEA399%253E3.0.CO%3B2-W&amp;pmid=10407276">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">74.</div><div id="B74" class="citations"><div class="citation"><div class="citation-content">Walker LA et al. 2009. Genome-wide analysis of <span class="named-content" data-type="genus-species">Candida albicans</span> gene expression patterns during infection of the mammalian kidney. <em>Fungal Genet. Biol.</em> 46:210â219.</div><div class="external-links"><div><a href="https://doi.org/10.1016/j.fgb.2008.10.012">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/19032986">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000263395800011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide+analysis+of+Candida+albicans+gene+expression+patterns+during+infection+of+the+mammalian+kidney&amp;author=LA+Walker&amp;publication_year=2009&amp;pages=210-219&amp;doi=10.1016%2Fj.fgb.2008.10.012&amp;pmid=19032986">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">75.</div><div id="B75" class="citations"><div class="citation"><div class="citation-content">Cline MS et al. 2007. Integration of biological networks and gene expression data using cytoscape. <em>Nat. Protoc.</em> 2:2366â2382.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nprot.2007.324">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17947979">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000253139900010">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Integration+of+biological+networks+and+gene+expression+data+using+cytoscape&amp;author=MS+Cline&amp;publication_year=2007&amp;pages=2366-2382&amp;doi=10.1038%2Fnprot.2007.324&amp;pmid=17947979">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">76.</div><div id="B76" class="citations"><div class="citation"><div class="citation-content">Smith DA, Nicholls S, Morgan BA, Brown AJ, and Quinn J. 2004. A conserved stress-activated protein kinase regulates a core stress response in the human pathogen <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Mol. Biol. Cell</em> 15:4179â4190.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e04-03-0181">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15229284">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000223567100020">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=A+conserved+stress-activated+protein+kinase+regulates+a+core+stress+response+in+the+human+pathogen+Candida+albicans&amp;author=DA+Smith&amp;author=S+Nicholls&amp;author=BA+Morgan&amp;author=AJ+Brown&amp;author=J+Quinn&amp;publication_year=2004&amp;pages=4179-4190&amp;doi=10.1091%2Fmbc.e04-03-0181&amp;pmid=15229284">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">77.</div><div id="B77" class="citations"><div class="citation"><div class="citation-content">Enjalbert B et al. 2006. Role of the Hog1 stress-activated protein kinase in the global transcriptional response to stress in the fungal pathogen <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Mol. Biol. Cell</em> 17:1018â1032.</div><div class="external-links"><div><a href="https://doi.org/10.1091/mbc.e05-06-0501">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16339080">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000235117300041">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Role+of+the+Hog1+stress-activated+protein+kinase+in+the+global+transcriptional+response+to+stress+in+the+fungal+pathogen+Candida+albicans&amp;author=B+Enjalbert&amp;publication_year=2006&amp;pages=1018-1032&amp;doi=10.1091%2Fmbc.e05-06-0501&amp;pmid=16339080">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">78.</div><div id="B78" class="citations"><div class="citation"><div class="citation-content">Enjalbert B, MacCallum DM, Odds FC, and Brown AJ. 2007. Niche-specific activation of the oxidative stress response by the pathogenic fungus <span class="named-content" data-type="genus-species">Candida albicans</span>. <em>Infect. Immun.</em> 75:2143â2151.</div><div class="external-links"><div><a href="https://doi.org/10.1128/IAI.01680-06">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/17339352">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000246345000007">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Niche-specific+activation+of+the+oxidative+stress+response+by+the+pathogenic+fungus+Candida+albicans&amp;author=B+Enjalbert&amp;author=DM+MacCallum&amp;author=FC+Odds&amp;author=AJ+Brown&amp;publication_year=2007&amp;pages=2143-2151&amp;doi=10.1128%2FIAI.01680-06&amp;pmid=17339352">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">79.</div><div id="B79" class="citations"><div class="citation"><div class="citation-content">Imbeaud S and Auffray C. 2005. Functional annotation: extracting functional and regulatory order from microarrays. <em>Mol. Syst. Biol.</em> 1:2005.0009.</div><div class="external-links"><div><a href="https://doi.org/10.1038/msb4100013">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/16729044">PubMed</a></div><div><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000243244500011">ISI</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Functional+annotation%3A+extracting+functional+and+regulatory+order+from+microarrays&amp;author=S+Imbeaud&amp;author=C+Auffray&amp;publication_year=2005&amp;pages=2005&amp;doi=10.1038%2Fmsb4100013&amp;pmid=16729044">Google Scholar</a></div></div></div></div></div></section></div></section><div class="core-collateral"><div id="core-collateral-info" role="tabpanel"><header><h2><i class="icon-info" aria-hidden="true"></i>Information &amp; Contributors</h2></header><section id="tab-information" aria-labelledby="information" role="tabpanel"><h3>Information</h3><section class="core-self-citation"><h4>Published In</h4><div class="core-journal-presentation">



        
        <div class="cover-image"><div class="cover-image__image"><img src="/cms/asset/38c2aa36-eca6-463f-a637-9dc5ff5538b0/mbio.2012.3.issue-6.cover.gif" alt="mBio cover image"/></div></div>
<div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">mBio</span></div><div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">3</span></span> â¢ <span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">6</span></span> â¢ <span property="datePublished">31 December 2012</span></div><div class="core-elocator"><span class="heading">eLocator</span>: <span class="content" property="identifier" typeof="Text">e00495-12</span></div><div class="core-editors"><span class="heading">Editor</span>: <span role="list"><span role="listitem" property="editor" typeof="Person"><span property="givenName">John W.</span> <span property="familyName">Taylor</span>, <span property="affiliation" typeof="Organization"><span property="name">University of California</span></span></span></span></div><div class="core-pmid"><span class="heading">PubMed</span>: <a class="content" href="https://pubmed.ncbi.nlm.nih.gov/23232717/" property="sameAs">23232717</a></div></div></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">Â© 2012 Sandai et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported</a> license, which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></section><section class="core-history"><h4>History</h4><div>Received: 5 November 2012</div><div>Accepted: 13 November 2012</div><div>Published online: 11 December 2012</div></section><section class="core-permissions"><h4>Permissions</h4><div>Request permissions for this article.</div><div><a href="https://marketplace.copyright.com/rs-ui-web/mp/search/all/10.1128%2FmBio.00495-12">Request Permissions</a></div></section><div class="core-download"><a href="/doi/pdf/10.1128/mBio.00495-12?download=true" class="btn btn-primary"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></div></section><section id="tab-contributors" aria-labelledby="contributors" role="tabpanel"><h3>Contributors</h3><section class="core-authors"><h4>Authors</h4><div id="con1" property="author" typeof="Person"><div class="heading"><span property="givenName">Doblin</span> <span property="familyName">Sandai</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="author-notes"><div role="paragraph">Present address: Doblin Sandai, Institut Perubatan &amp; Pergigian Termaju, Universiti Sains Malaysia, Pulau Pinang, Malaysia; Michelle D. Leach, Department of Molecular Genetics, University of Toronto, Toronto, Canada; Iryna Bohovych, Nebraska Redox Biology Center, University of NebraskaâLincoln, Lincoln, Nebraska, USA</div></div><div class="core-author-link"><a href="/authored-by/Sandai/Doblin">View all articles by this author</a></div></div></div><div id="con2" property="author" typeof="Person"><div class="heading"> (<span property="givenName">Zhikang</span> <span property="familyName">Yin</span>)</div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Yin/Zhikang">View all articles by this author</a></div></div></div><div id="con3" property="author" typeof="Person"><div class="heading"><span property="givenName">Laura</span> <span property="familyName">Selway</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Selway/Laura">View all articles by this author</a></div></div></div><div id="con4" property="author" typeof="Person"><div class="heading"><span property="givenName">David</span> <span property="familyName">Stead</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Stead/David">View all articles by this author</a></div></div></div><div id="con5" property="author" typeof="Person"><div class="heading"><span property="givenName">Janet</span> <span property="familyName">Walker</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Walker/Janet">View all articles by this author</a></div></div></div><div id="con6" property="author" typeof="Person"><div class="heading"><span property="givenName">Michelle D.</span> <span property="familyName">Leach</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="author-notes"><div role="paragraph">Present address: Doblin Sandai, Institut Perubatan &amp; Pergigian Termaju, Universiti Sains Malaysia, Pulau Pinang, Malaysia; Michelle D. Leach, Department of Molecular Genetics, University of Toronto, Toronto, Canada; Iryna Bohovych, Nebraska Redox Biology Center, University of NebraskaâLincoln, Lincoln, Nebraska, USA</div></div><div class="core-author-link"><a href="/authored-by/Leach/Michelle+D">View all articles by this author</a></div></div></div><div id="con7" property="author" typeof="Person"><div class="heading"><span property="givenName">Iryna</span> <span property="familyName">Bohovych</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="author-notes"><div role="paragraph">Present address: Doblin Sandai, Institut Perubatan &amp; Pergigian Termaju, Universiti Sains Malaysia, Pulau Pinang, Malaysia; Michelle D. Leach, Department of Molecular Genetics, University of Toronto, Toronto, Canada; Iryna Bohovych, Nebraska Redox Biology Center, University of NebraskaâLincoln, Lincoln, Nebraska, USA</div></div><div class="core-author-link"><a href="/authored-by/Bohovych/Iryna">View all articles by this author</a></div></div></div><div id="con8" property="author" typeof="Person"><div class="heading"><span property="givenName">Iuliana V.</span> <span property="familyName">Ene</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Ene/Iuliana+V">View all articles by this author</a></div></div></div><div id="con9" property="author" typeof="Person"><div class="heading"><span property="givenName">Stavroula</span> <span property="familyName">Kastora</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Kastora/Stavroula">View all articles by this author</a></div></div></div><div id="con10" property="author" typeof="Person"><div class="heading"><span property="givenName">Susan</span> <span property="familyName">Budge</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Budge/Susan">View all articles by this author</a></div></div></div><div id="con11" property="author" typeof="Person"><div class="heading"><span property="givenName">Carol A.</span> <span property="familyName">Munro</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Munro/Carol+A">View all articles by this author</a></div></div></div><div id="con12" property="author" typeof="Person"><div class="heading"><span property="givenName">Frank C.</span> <span property="familyName">Odds</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Odds/Frank+C">View all articles by this author</a></div></div></div><div id="con13" property="author" typeof="Person"><div class="heading"><span property="givenName">Neil A. R.</span> <span property="familyName">Gow</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Gow/Neil+A+R">View all articles by this author</a></div></div></div><div id="con14" property="author" typeof="Person"><div class="heading"><span property="givenName">Alistair J. P.</span> <span property="familyName">Brown</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">School of Medical Sciences, University of Aberdeen, Institute of Medical Sciences, Foresterhill, Aberdeen, United Kingdom</span></div></div><div class="core-author-link"><a href="/authored-by/Brown/Alistair+J+P">View all articles by this author</a></div></div></div></section><section class="core-editors"><h4>Editor</h4><div id="editor" property="editor" typeof="Person"><div class="heading"><span property="givenName">John W.</span> <span property="familyName">Taylor</span></div><div class="content"><div property="jobTitle">Editor</div><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">University of California</span></div></div></div></div></section><section class="core-authors-notes"><h4>Notes</h4><div role="doc-footnote">Address correspondence to Alistair J. P. Brown, al.<a href="/cdn-cgi/l/email-protection#c5a7b7aab2ab85a4a7a1abeba4a6ebb0ae"><span class="__cf_email__" data-cfemail="2f4d5d4058416f4e4d4b41014e4c015a44">[email&#160;protected]</span></a>.</div></section></section></div><div id="core-collateral-metrics" role="tabpanel"><header><h2><i class="icon-timeline" aria-hidden="true"></i>Metrics &amp; Citations</h2></header><section id="tab-metrics-inner" aria-labelledby="metrics-inner" role="tabpanel"><h3>Metrics</h3><section>



        
        







                <div class="sidebar-metric-container">
                        <input class="sidebar-metric__data" type="hidden" name="json" value='{"data":{"dataSections":[{"total":"319","months":[{"month":"2022-2-1","count":37},{"month":"2022-1-1","count":25},{"month":"2021-12-1","count":43},{"month":"2021-11-1","count":29},{"month":"2021-10-1","count":31},{"month":"2021-9-1","count":33},{"month":"2021-8-1","count":36},{"month":"2021-7-1","count":65},{"month":"2021-6-1","count":20}],"color":"color-4","firstMonth":[],"id":"download","title":"Download","startDate":"2021-6-1","displayDefault":true},{"total":"64","months":[{"month":"2022-2-1","count":2},{"month":"2022-1-1","count":1},{"month":"2021-12-1","count":1},{"month":"2021-11-1","count":0},{"month":"2021-10-1","count":1},{"month":"2021-9-1","count":1},{"month":"2021-8-1","count":2},{"month":"2021-7-1","count":0},{"month":"2021-6-1","count":0},{"month":"2021-5-1","count":2},{"month":"2021-4-1","count":0},{"month":"2021-3-1","count":1},{"month":"2021-2-1","count":2},{"month":"2021-1-1","count":4},{"month":"2020-12-1","count":0},{"month":"2020-11-1","count":0},{"month":"2020-10-1","count":0},{"month":"2020-9-1","count":1},{"month":"2020-8-1","count":0},{"month":"2020-7-1","count":1},{"month":"2020-6-1","count":1},{"month":"2020-5-1","count":1},{"month":"2020-4-1","count":0},{"month":"2020-3-1","count":0},{"month":"2020-2-1","count":0},{"month":"2020-1-1","count":0},{"month":"2019-12-1","count":3},{"month":"2019-11-1","count":0},{"month":"2019-10-1","count":0},{"month":"2019-9-1","count":1},{"month":"2019-8-1","count":0},{"month":"2019-7-1","count":2},{"month":"2019-6-1","count":0},{"month":"2019-5-1","count":0},{"month":"2019-4-1","count":1},{"month":"2019-3-1","count":0},{"month":"2019-2-1","count":3},{"month":"2019-1-1","count":0},{"month":"2018-12-1","count":0},{"month":"2018-11-1","count":0},{"month":"2018-10-1","count":0},{"month":"2018-9-1","count":1},{"month":"2018-8-1","count":1},{"month":"2018-7-1","count":0},{"month":"2018-6-1","count":0},{"month":"2018-5-1","count":1},{"month":"2018-4-1","count":0},{"month":"2018-3-1","count":0},{"month":"2018-2-1","count":2},{"month":"2018-1-1","count":1},{"month":"2017-12-1","count":0},{"month":"2017-11-1","count":0},{"month":"2017-10-1","count":0},{"month":"2017-9-1","count":0},{"month":"2017-8-1","count":1},{"month":"2017-7-1","count":0},{"month":"2017-6-1","count":1},{"month":"2017-5-1","count":0},{"month":"2017-4-1","count":2},{"month":"2017-3-1","count":1},{"month":"2017-2-1","count":2},{"month":"2017-1-1","count":1},{"month":"2016-12-1","count":1},{"month":"2016-11-1","count":1},{"month":"2016-10-1","count":0},{"month":"2016-9-1","count":2},{"month":"2016-8-1","count":1},{"month":"2016-7-1","count":0},{"month":"2016-6-1","count":0},{"month":"2016-5-1","count":0},{"month":"2016-4-1","count":1},{"month":"2016-3-1","count":0},{"month":"2016-2-1","count":0},{"month":"2016-1-1","count":1},{"month":"2015-12-1","count":1},{"month":"2015-11-1","count":0},{"month":"2015-10-1","count":0},{"month":"2015-9-1","count":1},{"month":"2015-8-1","count":1},{"month":"2015-7-1","count":0},{"month":"2015-6-1","count":0},{"month":"2015-5-1","count":0},{"month":"2015-4-1","count":0},{"month":"2015-3-1","count":0},{"month":"2015-2-1","count":0},{"month":"2015-1-1","count":0},{"month":"2014-12-1","count":2},{"month":"2014-11-1","count":0},{"month":"2014-10-1","count":0},{"month":"2014-9-1","count":0},{"month":"2014-8-1","count":0},{"month":"2014-7-1","count":0},{"month":"2014-6-1","count":0},{"month":"2014-5-1","count":0},{"month":"2014-4-1","count":0},{"month":"2014-3-1","count":0},{"month":"2014-2-1","count":0},{"month":"2014-1-1","count":3},{"month":"2013-12-1","count":0},{"month":"2013-11-1","count":1},{"month":"2013-10-1","count":0},{"month":"2013-9-1","count":0},{"month":"2013-8-1","count":1},{"month":"2013-7-1","count":0},{"month":"2013-6-1","count":0},{"month":"2013-5-1","count":0},{"month":"2013-4-1","count":0},{"month":"2013-3-1","count":2}],"color":"color-0","id":"citation","title":"Citation","startDate":"2013-3-1","displayDefault":true}],"doi":"10.1128/mBio.00495-12","widgetTitle":"Article Metrics"}}'>
                </div>
        
</section><section></section><section class="text-right"><a href="/doi/metrics/10.1128/mBio.00495-12" class="btn btn--inverse open-in-viewer"><span>VIEW ALL METRICS</span><i class="icon-open_in_new"></i></a></section></section><section id="tab-citations" aria-labelledby="citations" role="tabpanel"><h3>Citations</h3><section></section><section>



        
        <div class="citation-download">
	<p class="citation-msg">
		If you have the appropriate software installed, you can download article citation data to the citation manager of your choice. Simply select your manager software from the list below and click Download.
	</p>
	<!-- download options -->
	<form action="/action/downloadCitation" name="frmCitmgr" class="citation-form" method="post" target="_self"><input type="hidden" name="doi" value="10.1128/mBio.00495-12"><input type="hidden" name="downloadFileName" value="csp_3_"><input type="hidden" name="include" value="abs">
		<fieldset class="format-select">
			<label for="slct_format">Format</label>
			<div class="select-container">
				<select id="slct_format" name="format" class="js__slcInclude">
					<option value="" selected="selected">Please select one from the list</option>
					<option value="ris">RIS (ProCite, Reference Manager)</option>
					<option value="endnote">EndNote</option>
					<option value="bibtex">BibTex</option>
					<option value="medlars">Medlars</option>
					<option value="refworks">RefWorks</option>
				</select>
			</div>
			<label class="label-direct" for="direct">
				<input id="direct" type="checkbox" name="direct" value="" checked="checked">
				<span class="round-check"><span class="check"></span></span>
				Direct import
			</label>
		</fieldset>           
		<footer class="form-footer">
			<input onclick="onCitMgrSubmit()" class="formbutton" type="submit" name="submit" value="Download">
		</footer>
	</form>
</div>

</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel"><header><h2><i class="icon-eye" aria-hidden="true"></i>View Options</h2></header><div tabindex="0" class="section--wrapper"><h3>View options</h3><section><h4>ePub</h4><a href="https://journals.asm.org/doi/epub/10.1128/mBio.00495-12" class="btn"><i aria-hidden="true" class="icon-epub"></i><span>Open ePub</span></a></section><section><h4>PDF</h4><a href="/doi/pdf/10.1128/mBio.00495-12?download=true" download="true" class="btn btn--pdf"><i aria-hidden="true" class="icon-PDF"></i><span>Download PDF</span></a></section><section><h4>Full Text</h4><a href="https://journals.asm.org/doi/full/10.1128/mBio.00495-12" class="btn"><i aria-hidden="true" class="icon-subject"></i><span>Open Full Text</span></a></section></div><!-- Its needed to duplicate this for PB check Pill.js as well--><div tabindex="-1" class="section--wrapper"><h3>Get Access</h3><section>



        
        












    <div class="purchaseArea pill__item" id="purchaseArea">
        <div class="login-options" data-pb-dropzone="login-options">
            
        </div>
        <div class="purchaseOptions">
            <div class="purchaseOptions--header">
                <h4 class="purchaseArea--section-heading">
                    PURCHASE OPTIONS
                </h4>
                
                        <div class="save-for-later">
                            <a data-bind="saveItem" data-doi="10.1128/mBio.00495-12" class="purchaseOptions--save" href="/action/saveItem?doi=10.1128/mBio.00495-12">
                                <i aria-hidden="true" class="icon-heart mx-1"></i>
                                <span>Save for later</span>
                            </a>
                            <a class="saved-go-cart hidden"  href="/action/showCart?FlowID=2"><span class="icon-cart-icon-material"></span> Item saved, go to cart </a>
                        </div>
                    
            </div>
            <p class="purchaseArea--section-info">
                Subscribe and get full access to this article
            </p>

            <div class="purchaseOptions--options-list">
                











<div class="purchase-options-container">
    

   
        <div class="savedItem-info hidden">
            <p></p>
        </div>
    

     
            
                
                
                        <div class="purchaseMessage hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/MBIO.00495-12-500044">
                            <p class="errorMsgBox"></p>
                        </div>
                        <div class="add-journal-to-cart">
                            <div class="purchaseOptions--option mb-3">
                                 
                                    <h5 class="option-title">Buy Article</h5>
                                    <span class="option-info mb-1">mBio</span>
                                    <span class="option-info">Vol.3 &bull; Issue 6 &bull; 
                                            ASM Journals Pay Per View, PPV 25
                                        </span>
                                    <div class="option-actions mt-3 pt-3">
                                        <div class="option-price">
                                            <b>$25.00</b>
                                        </div>
                                        <a href="/action/addToCart?id=ASMJ_PPV-ELECTRONIC-USD-10.1128/MBIO.00495-12-500044" data-bind="addItem"
                                           data-key="ASMJ_PPV-ELECTRONIC-USD-10.1128/MBIO.00495-12-500044" data-entity="ASMJ_PPV-ELECTRONIC-USD-10.1128/MBIO.00495-12-500044"
                                           class="btn btn-primary btn--borderless">
                                            <i aria-hidden="true" class="mx-2 icon-shopping_cart"> </i>
                                            <span>ADD TO CART</span>
                                        </a>
                                    </div>
                                 
                            <div class="option-actions mt-3 pt-3 hidden" data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/MBIO.00495-12-500044" style="justify-content: flex-end">
                                <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                    <span>Checkout</span>
                                </a>
                            </div>
                        </div>

                        <div class="purchaseOptions--option mb-3">
                                                             <h5 class="option-title">Buy Article</h5>
                                <span class="option-info mb-1">mBio</span>
                                <span class="option-info">Vol.3 &bull; Issue 6 &bull; 
                                        ASM Journals Pay Per View, PPV 25
                                    
                                </span>
                             
                             <div class="option-actions mt-3 pt-3 hidden"
                                     data-item="ASMJ_PPV-ELECTRONIC-USD-10.1128/MBIO.00495-12-500044" style="justify-content: flex-end">
                                    <a href="/action/showCart?FlowID=2" class="btn btn-primary btn--borderless">
                                        <span>Checkout</span>
                                    </a>
                             </div>
                        </div>
                        </div>
                    
            
        

    
</div>

                <div class="purchaseOptions--option">
                    <h5 class="option-title">Journal Subscription</h5>
                    <span class="option-info mb-1">mBio</span>
                    <div class="option-actions mt-3 pt-3">
                        <div class="option-actions--info">ASM members can purchase subscriptions to journals.</div>
                        <a href="/action/ssostart?idp=https%3A%2F%2Flogin.asm.org%2Fidp%2Fshibboleth&amp;redirectUri=%2Fdoi%2F10.1128%2FmBio.00495-12" redirectUri="/doi/10.1128/mBio.00495-12" class="btn btn-primary btn--borderless">
                            
                            <span>Join or renew</span>
                            <i aria-hidden="true" class="mx-2 icon-open_in_new"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </div>




</section></div></div><div id="core-collateral-media" role="tabpanel"><header><h2><i class="icon-photo" aria-hidden="true"></i>Figures and Media</h2></header><section id="tab-figures" aria-labelledby="figures" role="tabpanel"><h3>Figures</h3></section><section id="tab-other" aria-labelledby="other" role="tabpanel"><h3>Media</h3><section>



        
        
</section></section></div><div id="core-collateral-tables" role="tabpanel"><header><h2><i class="icon-tables" aria-hidden="true"></i>Tables</h2></header></div><div id="core-collateral-share" role="tabpanel"><header><h2><i class="icon-share" aria-hidden="true"></i>Share</h2></header><h3>Share</h3><div class="section--wrapper"><section><h4>Share the article link</h4><div id="share-self"><p><a href="#" target="_blank" class="share-self__source"></a></p><button class="btn btn--inverse share-self__action"><i aria-hidden="true" class="icon-copy-o"></i><span>Copy Link</span></button><div class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section><h4>Share with email</h4><p><a href="#" target="_blank" class="btn btn-primary share-email__source"><i class="icon-copy-o"></i><span>EMAIL A COLLEAGUE</span></a></p></section><section><h4>Share on social media</h4><div class="share-buttons"><a role="link" target="_blank" href="#" class="btn btn--facebook addthis_button_facebook"><i aria-hidden="true" class="icon-facebook-2"></i><span>facebook</span></a><a role="link" target="_blank" href="#" class="btn btn--twitter addthis_button_twitter"><i aria-hidden="true" class="icon-twitter"></i><span>twitter</span></a><a role="link" target="_blank" href="#" class="btn btn--linkedin addthis_button_linkedin"><i aria-hidden="true" class="icon-linkedin"></i><span>linkedin</span></a><a role="link" target="_blank" href="#" class="btn btn--email addthis_button_email"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a></div></section></div></div></div></article><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script id="contribution-meta" type="application/json">{
    "doi": "10.1128/mBio.00495-12",
    "toc": "/toc/mbio/3/6",
    "issn": ""
}</script><div class="article__side-ad hidden">



        
        <!-- /13678947/asm_box -->
<div class="boombox-wrapper">
	<span class="boombox-label">Advertisement</span>
<div id='div-gpt-ad-1594664702378-0' style='width: 300px; height: 250px;'>
  <script>
    googletag.cmd.push(function() { googletag.display('div-gpt-ad-1594664702378-0'); });
  </script>
</div>
</div>
</div>




        
        <div class="article-footer-element">
    <div class="row">
        <div class="col-12">
            <section aria-label="we recommend" class="we-recommend">
               <div id="trendmd-suggestions"></div>
<script defer src='//js.trendmd.com/trendmd.min.js' data-trendmdconfig='{"journal_id":"2721", "element":"#trendmd-suggestions"}'></script>
            </section>
        </div>
    </div>
</div>










        <div data-widget-def="ux3-layout-widget" data-widget-id="9f4096e6-614d-46c7-a704-f706dfcdc172" class="related-tabs swipe-with-arrows">
        



        
        <div class="article-footer-element">









        <div data-widget-def="UX3Tabs" data-widget-id="3c1dc793-20ba-4336-a3d4-bb86a6fc2c43" class="tab  tab--flex" data-ctrl-res="screen-sm">
        



        
        <div data-ctrl-res="screen-md" class="tab"><ul role="tablist" class="tab__nav rlist"><li role="presentation" class="tab__nav__item  active "><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" aria-selected="true" class="tab__nav__item__link active"><span>Related Articles</span></a></li><li role="presentation" class="tab__nav__item"><a href="#pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-controls="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" role="tab" data-toggle="tab" id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" aria-selected="false" class="tab__nav__item__link"><span>Related Articles From ASM</span></a></li></ul><div class="tab__content"><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4301con" role="tabpanel" class="tab__pane active">



        
        <section class="related-articles"><ul class="rlist related-articles__items"><li class="related-articles__item"><a href="/doi/10.1128/mBio.02489-14" class="related-articles__item__title">Correction for Sandai et al., The Evolutionary Rewiring of Ubiquitination Targets Has Reprogrammed the Regulation of Carbon Assimilation in the Pathogenic Yeast Candida albicans</a><ul class="meta__authors rlist--inline loa mobile-authors" title="list of authors">
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Sandai%2C+Doblin" title="Doblin Sandai">Doblin Sandai</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Yin%2C+Zhikang" title="Zhikang Yin">Zhikang Yin</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Selway%2C+Laura" title="Laura Selway">Laura Selway</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Stead%2C+David" title="David Stead">David Stead</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Walker%2C+Janet" title="Janet Walker">Janet Walker</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Leach%2C+Michelle+D" title="Michelle D. Leach">Michelle D. Leach</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Bohovych%2C+Iryna" title="Iryna Bohovych">Iryna Bohovych</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Ene%2C+Iuliana+V" title="Iuliana V. Ene">Iuliana V. Ene</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Kastora%2C+Stavroula" title="Stavroula Kastora">Stavroula Kastora</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Budge%2C+Susan" title="Susan Budge">Susan Budge</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Munro%2C+Carol+A" title="Carol A. Munro">Carol A. Munro</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Odds%2C+Frank+C" title="Frank C. Odds">Frank C. Odds</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Gow%2C+Neil+A+R" title="Neil A. R. Gow">Neil A. R. Gow</a></span>, </li>
   <li><span class="hlFld-ContribAuthor"><a href="/action/doSearch?ContribAuthorRaw=Brown%2C+Alistair+J+P" title="Alistair J. P. Brown">Alistair J. P. Brown</a></span></li>
</ul>
</li></ul></section>
</div><div id="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311" aria-labelledby="pane-3c1dc793-20ba-4336-a3d4-bb86a6fc2c4311con" role="tabpanel" class="tab__pane">



        
        <!--content loaded via ajax-->
<div class="asmj-org general-rss-feed-reader aog" id="80036a5d-0f41-4828-9cd1-19399fe2560a" data-rss-url="https://asm.org/rss/content/">
    <div class="rss-body"></div>
</div>
</div></div></div>

        </div>
    
</div>

        </div>
    




        
        
</div>
</main>




        
        <footer class="footer"><div class="footer__wrapper container container--lg"><div class="footer__top row"><div class="sitemap"><div class="sitemap__column sitemap__column--media col-xs-12"><div class="footer__logos">



        
        <a href="https://asm.org/" title="American Society for Microbiology logo"><img id="" alt="American Society for Microbiology logo" src="/specs/products/asmj/releasedAssets/images/asm-white.svg"/></a>




        
        <a href="/" title="ASM Journals logo"><img id="" alt="ASM Journals logo" src="/specs/products/asmj/releasedAssets/images/asm-logotype.svg"/></a>
</div><div class="footer__social-media">



        
        <div class="footer__social-media">
    <ul class="rlist--inline">
        <li><a href="https://www.facebook.com/asmfan" target="_blank" rel="noopener"><i class="icon-facebook"></i></a></li>
        <li><a href="https://twitter.com/mbiojournal" target="_blank" rel="noopener"><i class="icon-twitter"></i></a></li>
        <li><a href="https://www.linkedin.com/companies/45861" target="_blank" rel="noopener"><i class="icon-linkedin"></i></a></li>
        <li><a href="https://www.youtube.com/channel/UCBxWzf49gVHz0Ksgw_N8brw" target="_blank" rel="noopener"><i class="icon-youtube"></i></a></li>
    </ul>
</div>
</div></div><div class="sitemap__column sitemap__column--journals col-xs-12">



        
        <a href="#" data-toggle-target-id="footerJournals"><h6 class="footer__list-title">JOURNALS</h6></a>
<ul id="footerJournals" class="rlist footer__list footer__list--columns">
	<li><a href="/journal/aac">Antimicrobial Agents and Chemotherapy</a></li>
	<li><a href="/journal/aem">Applied and Environmental Microbiology</a></li>
	<li><a href="/journal/cvi">Clinical and Vaccine Immunology</a></li>
	<li><a href="/journal/cmr">Clinical Microbiology Reviews</a></li>
	<li><a href="/journal/ecosalplus">EcoSal Plus</a></li>
	<li><a href="/journal/ec">Eukaryotic Cell</a></li>
	<li><a href="/journal/iai">Infection and Immunity</a></li>
	<li><a href="/journal/jb">Journal of Bacteriology</a></li>
	<li><a href="/journal/jcm">Journal of Clinical Microbiology</a></li>
	<li><a href="/journal/jmbe">Journal of Microbiology &amp; Biology Education</a></li>
	<li><a href="/journal/jvi">Journal of Virology</a></li>
	<li><a href="/journal/mbio">mBio</a></li>
	<li><a href="/journal/mmbr">Microbiology and Molecular Biology Reviews</a></li>
	<li><a href="/journal/mra">Microbiology Resource Announcements</a></li>
	<li><a href="/journal/spectrum">Microbiology Spectrum</a></li>
	<li><a href="/journal/mcb">Molecular and Cellular Biology</a></li>
	<li><a href="/journal/msphere">mSphere</a></li>
	<li><a href="/journal/msystems">mSystems</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--connect col-xs-12">



        
        <ul class="rlist footer__list">
    <li><h6>RESOURCES</h6></li>
    <li><a href="/about-asm-journals" target="_blank" rel="noopener">About Us</a></li>
    <li><a href="/announcements" target="_blank" rel="noopener">Announcements</a></li>
    <li><a href="/asm-author-center" target="_blank" rel="noopener">Author Center</a></li>
    <li><a href="/collection" target="_blank" rel="noopener">ASM Journal Collections</a></li>
     <li><a href="/publishing-ethics" target="_blank" rel="noopener">Publishing Ethics</a></li>
    <li><a href="/migration-information" target="_blank" rel="noopener">Migration Information</a></li>
    <li><a href="/advertising" target="_blank" rel="noopener">Advertising</a></li>
    <li><a href="https://asm.org/Membership/Home" target="_blank" rel="noopener">ASM Membership</a></li>
    <li><a href="https://asm.org/cookie-policy" target="_blank" rel="noopener">Privacy Policy</a></li>
    <li><a href="/terms" target="_blank" rel="noopener">Terms &amp; Conditions</a></li>
    <li><a href="/proprietary-rights" target="_blank" rel="noopener">Proprietary Rights Notices</a></li>
    <li><a href="/non-commercial-tdm-license" target="_blank" rel="noopener">Non-Commercial TDM License</a></li>
</ul>
</div><div class="sitemap__column sitemap__column--news col-xs-12">



        
        <a href="#" data-toggle-target-id="footerConnected"><h6 class="footer__list-title">STAY CONNECTED</h6></a>
<ul id="footerConnected" class="rlist footer__list">
    <li><a href="/action/showPreferences?menuTab=Alerts" target="_blank" rel="noopener">Alerts</a></li>
    <li><a href="/asm-journals-rss-feeds" target="_blank" rel="noopener">RSS Feeds</a></li>
    <li><a href="/contact-us" target="_blank" rel="noopener">Contact Us</a></li>
    <li><a href=" https://connect.asm.org/ " target="_blank" rel="noopener">ASM Communities</a></li>
    
</ul>
</div><div class="sitemap__column sitemap__column--other col-xs-12 ">



        
        <h6>NEWS</h6>
<ul class="rlist footer__list">
    <li><a href="https://asm.org/newsroom" target="_blank" rel="noopener">Press Releases</a></li>
    <li><a href="https://asm.org/Browse-By-Content-Type/Policy-Statement" target="_blank" rel="noopener">Policy Statements</a></li>
    
</ul>
</div></div></div><div class="footer__bottom row"><div class="col-xs-12"><div class="footer__copyright">



        
        Copyright Â© 2022 American Society for Microbiology
</div></div></div></div></footer>




        
        <div class="cookiePolicy-popup"><div class="container"><div class="row"><div class="col-sm-7 col-md-9 cookiePolicy-popup__content"><span><span><strong>American Society for Microbiology ("ASM")</strong> is committed to maintaining your confidence and trust with respect to the information we collect from you on websites owned and operated by <strong>ASM ("ASM Web Sites")</strong> and other sources. This Privacy Policy sets forth the information we collect about you, how we use this information and the choices you have about how we use such information.</span></span></div><div class="col-sm-5 col-md-3"><div class="cookiePolicy-popup__actions"><button id="accept-cookie-policy" class="btn btn-primary btn--borderless">ACCEPT &nbsp;<span class="sr-only">the cookie policy and </span><span aria-hidden="true">& &nbsp;</span>CONTINUE</button><a href="https://asm.org/cookie-policy" class="btn btn-secondary btn--darker-content">FIND OUT MORE <span class="sr-only">about the privacy policy</span></a></div></div></div></div></div>
</div>
        </div>
    </div>



        
        

            <script src="/products/asmj/releasedAssets/js/build.lazyload.bundle-9eae693cc49bd405a4bc.js"></script>

        <script src="/products/asmj/releasedAssets/js/main.bundle-f0f6c9d7da4e7fd5b03e.js" ></script>
        

















<script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"6e7ab7884e577aba","token":"edc08be6f1e64730992376d7e16c0615","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>
</body>
</html>
